{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Intialize RDFGraph class\n",
    "This was originally intended to operate on pulled abstracts from the arxiv and pubmed APIs. The model is now adjusted to operate directly on full text. Simply pull out the full text into the nested list.\n",
    "\n",
    "There are several parameters that need tuning in the RDFGraph, for now, we use the default parameters that worked best for the DTRA model."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# change base dir to multivac dir\n",
    "import os\n",
    "import json\n",
    "os.chdir('/Users/taylor_bolt/PycharmProjects/multivac/multivac')\n",
    "\n",
    "# Pull in full text and put into nested list\n",
    "article_json = json.load(open('data/cleanedArticles-03042019.json'))\n",
    "all_article_text = [ article['text'] for article in article_json.values() ]\n",
    "\n",
    "# Initialize RDFGraph object\n",
    "from src.data.rdf_graph import RDFGraph\n",
    "rdf_obj = RDFGraph(all_article_text[:100]) # subset to 100 articles for now"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extract relation tuples from full text w/corenlp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 articles out of 100 processed\n",
      "Article is too large for parser, need to split up\n",
      "Article is too large for parser, need to split up\n",
      "Article is too large for parser, need to split up\n",
      "Article is too large for parser, need to split up\n",
      "Article is too large for parser, need to split up\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{0: [[('Recurrent epidemics',\n",
       "    'be in',\n",
       "    'small world networks J. A. Verdascaa')],\n",
       "  [('patterns', 'be with', 'slightly irregular periods')],\n",
       "  [('importance', 'be in', 'dynamics of childhood diseases')],\n",
       "  [('50', 'in', 'Bartlett')],\n",
       "  [('relative ﬂuctuations', 'lead to', 'Citation')],\n",
       "  [('spatial correlations', 'greatly enhance', 'stochastic ﬂuctuations')],\n",
       "  [('small world networks', 'be with', 'SIR')],\n",
       "  [('square lattice', 'be with', 'small world interaction rules')],\n",
       "  [('increase', 'be in', 'spatial correlations')],\n",
       "  [('simple deterministic models', 'be with', 'seasonal forcing'),\n",
       "   ('pre-vaccination pattern',\n",
       "    'be',\n",
       "    'For measles accurately reproduced by simple deterministic models with seasonal forcing')],\n",
       "  [('pertussis', 'be in', 'pre-vaccination era'),\n",
       "   ('pronounced 3.5 year period', 'be in', 'line with epidemiological data'),\n",
       "   ('epidemics', 'be with', 'pronounced 3.5 year period in line')],\n",
       "  [('model independent parameters', 'be', 'when taken for measles')],\n",
       "  [('populations', 'be with', 'similar sizes')],\n",
       "  [('behaviour', 'be in', 'limit')],\n",
       "  [('infectiousness parameter', 'be in', 'our model'),\n",
       "   ('recurrent epidemics', 'be in', 'unforced spatially extended models')],\n",
       "  [('sustained oscillations', 'be with', 'periods')],\n",
       "  [],\n",
       "  [('network connections', 'of importance be', 'ideia'),\n",
       "   ('real epidemics', 'in importance be', 'ideia'),\n",
       "   ('importance', 'be in', 'real epidemics')],\n",
       "  [('birth/death rates', 'be', 'assumed')],\n",
       "  [('basic diﬀerential equations', 'be in', 'terms of densities')],\n",
       "  [('we', 'obtain for', 'period of damped oscillations')],\n",
       "  [('sum', 'be in', 'time'),\n",
       "   ('sum', 'be with', 'mean Nβ si ∆ t'),\n",
       "   ('stochastic change', 'be in', 'number of susceptibles'),\n",
       "   ('stochastic change', 'be in', 'number of infectives')],\n",
       "  [('stochastic changes', 'be in', 's')],\n",
       "  [('role', 'be with', 'zero birth rate')],\n",
       "  [('We', 'shall consider', 'small world contact network')],\n",
       "  [('random one', 'be', 'when established')],\n",
       "  [('local neighbourhood', 'be in', 'as lattices'),\n",
       "   ('short average path length', 'be in', 'as random graphs')],\n",
       "  [('existence',\n",
       "    'be in',\n",
       "    'many interaction networks including networks of social contacts')],\n",
       "  [('square lattice', 'on automaton be', 'CA')],\n",
       "  [('other site', 'with probability be', 'p'),\n",
       "   ('interactions', 'be with', 'k neighbouring sites'),\n",
       "   ('small world probability', 'be with', 'other site on lattice')],\n",
       "  [('disease dependent period', 'set', 'time scale of model')],\n",
       "  [('it', 'be', 'found'),\n",
       "   ('it', 'be', 'then changed into susceptible S state')],\n",
       "  [('counters', 'be', 'set'),\n",
       "   ('counters', 'be set to', '-1'),\n",
       "   ('counters', 'account for', 'deterministic recovery')],\n",
       "  [('sites', 'be in', 'I state change into R state')],\n",
       "  [('spatial SIR model', 'depend on', 'only three additional parameters')],\n",
       "  [('persistence', 'fraction of', 'simulations surviving')],\n",
       "  [('µ', 'at ﬁxed be', 'γ =')],\n",
       "  [('ﬁnite size eﬀects',\n",
       "    'broaden transition region considerably in',\n",
       "    'addition')],\n",
       "  [('underlying small world network', 'of coeﬃcient be', 'Citation')],\n",
       "  [('eﬀective transmission rate', 'number of', 'new infectives'),\n",
       "   ('eﬀective transmission rate',\n",
       "    'number per',\n",
       "    'time step divided by product of instantaneous densities of susceptibles')],\n",
       "  [('signiﬁcant variation', 'be with', 'p')],\n",
       "  [('clustering', 'be in', 'general')],\n",
       "  [('small world network', 'be in', 'section')],\n",
       "  [('SEIR model', 'oscillation with', 'periods'),\n",
       "   ('sustained oscillations', 'be with', 'periods')],\n",
       "  [('order', 'be in', 'models')],\n",
       "  [('as spatial SIR model', 'be with', 'following modiﬁcations'),\n",
       "   ('rules', 'be as', 'as spatial SIR model with following modiﬁcations')],\n",
       "  [('second site', 'be infective in', 'I'),\n",
       "   ('S', 'be', 'in case changed into exposed E state')],\n",
       "  [('population size =', 'be in', 'simulations')],\n",
       "  [('network model', 'be with', 'p =')],\n",
       "  [('p', '=', '1'), ('network model', 'be with', 'p =')],\n",
       "  [('measles', 'for series be', 'p ='),\n",
       "   ('measles', 'be in', 'homogeneously mixed populations')],\n",
       "  [('measles', 'of oscillation be', 'Figure 4c'),\n",
       "   ('measles', 'be in', 'Birmingham'),\n",
       "   ('similar population size', 'with oscillation be', 'Figure 4c'),\n",
       "   ('incidence oscillations', 'be with', 'similar population size')],\n",
       "  [('sharp contrast', 'be with', 'homogeneously mixed stochastic models')],\n",
       "  [('incidence oscillations', 'be with', 'average period of two years'),\n",
       "   ('other parameters', 'be in', 'as Figure 4a'),\n",
       "   ('SEIR simulations', 'be with', 'p =')],\n",
       "  [('11 increase', 'be in', 'line with real data'),\n",
       "   ('line', 'be with', 'real data')],\n",
       "  [('line', 'be with', 'epidemiological data'),\n",
       "   ('small world model', 'be with', 'p =')],\n",
       "  [('result', 'provide', 'additional support of network model')],\n",
       "  [('period', 'be in', 'independently model')],\n",
       "  [('pre-vaccination incidence records',\n",
       "    'be well reproduce by',\n",
       "    'simpler deterministic models with seasonal forcing'),\n",
       "   ('simpler deterministic models', 'be with', 'seasonal forcing')],\n",
       "  [('SEIR simulations', 'series ×', 'infectious period of pertussis'),\n",
       "   ('SEIR simulations', 'series on', 'N')],\n",
       "  [('p', '=', '0.2'),\n",
       "   ('pre-vaccination incidence oscillations',\n",
       "    'be with',\n",
       "    'broad 2.5 year period')],\n",
       "  [('p', '=', '0.2'),\n",
       "   ('marked contrast',\n",
       "    'be with',\n",
       "    'results for measles for same contact network'),\n",
       "   ('same contact network', 'for measles be', 'Figure 4')],\n",
       "  [('vaccination coverage', 'be with', '70 % eﬃcacy')],\n",
       "  [('agreement', 'be with', 'available data'),\n",
       "   ('increase',\n",
       "    'be in',\n",
       "    'period of recurrent epidemic peaks reported after mass vaccination')],\n",
       "  [('increased coeherence',\n",
       "    'be',\n",
       "    'In our model due to loss of stability of endemic equilibrium')],\n",
       "  [('small world networks', 'be with', 'SIR')],\n",
       "  [(\"network 's clustering coeﬃcient\", 'be in', 'when small world region'),\n",
       "   ('network', 'have', 'clustering coeﬃcient')],\n",
       "  [('interactions', 'be', 'predominantly local')],\n",
       "  [('This', 'have', 'three ma jor consequences')],\n",
       "  [('deterministic model', 'from', 'behaviour'),\n",
       "   ('spatial strucuture', 'be', 'when taken into account'),\n",
       "   ('strucuture', 'be', 'when taken into account')],\n",
       "  [('patterns', 'be with', 'slightly irregular periods'),\n",
       "   ('Third', 'series of', 'model')],\n",
       "  [('Stochastic eﬀects', 'be in', 'seasonally forced models')],\n",
       "  [('measles', 'be in', 'England')],\n",
       "  [('sublinear ﬁt', 'may', 'may result of importance of de-phasing eﬀect')],\n",
       "  [('sustained ﬂuctuations', 'be with', 'well deﬁned period')],\n",
       "  [('model', \"'s parameter be\", 'p'), ('model', 'have', 'free parameters')],\n",
       "  [('Science', 'for foundation be', 'FCT')],\n",
       "  [('JAV', 'acknowledge', 'support from FCT')],\n",
       "  [('Anderson', 'disease of', 'Humans')],\n",
       "  [('Aparicio', 'oscillation in', 'stochastic systems'),\n",
       "   ('H G Sustained oscillations', 'be in', 'stochastic systems')],\n",
       "  [('epidemic model', 'in seasonality be', '1984'),\n",
       "   ('B Seasonality', 'be in', 'epidemic model')],\n",
       "  [('Bolker', 'chaos in', 'measles models'),\n",
       "   ('B Chaos', 'be in', 'measles models')],\n",
       "  [('spreading', 'be with', 'immunization')],\n",
       "  [('complex dynamical transitions', 'be in', 'epidemics')],\n",
       "  [('measles metapopulation dynamics', 'be in', 'England'),\n",
       "   ('Grenfell',\n",
       "    'determinant of',\n",
       "    'measles metapopulation dynamics in England')],\n",
       "  [('critical behaviour', 'on p be', '1983'),\n",
       "   ('Grassberger', 'p on', 'critical behaviour of general epidemic process')],\n",
       "  [('Grenfell', 'be in', 'measles epidemics')],\n",
       "  [('Johansen', 'model of', 'recurrent epidemics')],\n",
       "  [('epidemiological model', 'in eﬀect be', '2001'),\n",
       "   ('G Small world eﬀect', 'be in', 'epidemiological model'),\n",
       "   ('Kuperman', 'eﬀect in', 'epidemiological model')],\n",
       "  [('Lloyd', 'variability in', 'models for recurrent epidemics'),\n",
       "   ('L Estimating variability', 'be in', 'models for recurrent epidemics')],\n",
       "  [('Lloyd', 'heterogeneity in', 'epidemic models'),\n",
       "   ('R M Spatial heterogeneity', 'be in', 'epidemic models')],\n",
       "  [('Moore', 'be in', 'small world networks')],\n",
       "  [('time', 'be in', 'recurrent epidemics'), ('time', 'on i be', '1999')],\n",
       "  [('A Epidemic spreading', 'be in', 'scale free networks'),\n",
       "   ('Vespigniani', 'spreading in', 'scale free networks')],\n",
       "  [('Pastor-Santorras', 'be in', 'complex networks')],\n",
       "  [('Rhodes', 'be in', 'lattice models of epidemic spread')],\n",
       "  [('1999 Opposite patterns', 'be in', 'sympatric disease metapopulations'),\n",
       "   ('synchrony', 'of pattern be', 'Science 286 968-971')],\n",
       "  [('Grenfell', 'interplay between', 'determinism'),\n",
       "   ('B interplay', 'be in', 'childhood diseaeses'),\n",
       "   ('Grenfell', 'interplay in', 'childhood diseaeses')],\n",
       "  [('It', 'be', 'small world'), ('It', 'be', 'small')],\n",
       "  [('17 FIGURES', 'figure', '1 persistence transition'),\n",
       "   ('17 FIGURES', 'figure', '1 Epidemic persistence transition')],\n",
       "  [('Infection', 'occur between', 'connected sites')],\n",
       "  [('abrupt change', 'be in', 'persistence')],\n",
       "  [('underlying small world network', 'for coeﬃcient be', 'dashed line')],\n",
       "  [('persistence transition', 'occur at', 'edge of small world regime')],\n",
       "  [('drastic reduction', 'be in', 'βef f N')],\n",
       "  [('average', 'be in', 'homogeneously mixed population'),\n",
       "   ('transmission rate β', '= at_time', '2.4 day − 1')],\n",
       "  [('pertussis', 'be with', 'p =')],\n",
       "  [('stochastic eﬀects', 'dominate', 'behaviour of time series'),\n",
       "   ('behaviour', 'be in', 'pre periods')]],\n",
       " 1: [[('This', 'be', 'complicated')],\n",
       "  [('distributions', 'means', 'their')],\n",
       "  [('extent', 'to epidemic be', 'idea about their clinical features'),\n",
       "   ('SEIR', 'epidemic', 'idea about their clinical features')],\n",
       "  [('there can large heterogeneity', 'be in', 'population')],\n",
       "  [('one', 'deﬁne', 'meaning of importance')],\n",
       "  [('θ', 'be', 'small'),\n",
       "   ('secondary cases', 'be in', 'beginning of epidemic')],\n",
       "  [('we', 'want in', 'such cases')],\n",
       "  [('seasonal inﬂuenza', 'be in', 'general population')],\n",
       "  [('infectious disease',\n",
       "    'be',\n",
       "    'In particular present in population beyond its initial phase of outbreak')],\n",
       "  [('NumericCitation',\n",
       "    'determine',\n",
       "    'upper-and lower bound for symptom-response SEIR model')],\n",
       "  [('two special classes', 'class of', 'Markov models'),\n",
       "   ('two special classes',\n",
       "    'class with',\n",
       "    'exponentially distributed latent periods'),\n",
       "   ('class', 'be with', 'exponentially distributed latent periods'),\n",
       "   ('distributed latent periods', 'be with', 'constant latent periods')],\n",
       "  [],\n",
       "  [('n', 'denote', 'number of individuals in population'),\n",
       "   ('number', 'be in', 'population')],\n",
       "  [('Individuals', 'be', 'initially susceptible')],\n",
       "  [('number', 'be in', 'population')],\n",
       "  [('Infectious contacts', 'be with', 'susceptible individuals')],\n",
       "  [('R', 'be with', 'subscript'), ('subscript', 'with r be', 't')],\n",
       "  [('epidemic starts', 'be with', 'small number of infectious individuals')],\n",
       "  [('we',\n",
       "    'moreover let',\n",
       "    'population quantities decorated with bar over it in analogous fashion')],\n",
       "  [('symptomatic cases', 'thus meet', 'fewer individuals'),\n",
       "   ('cases', 'thus meet', 'fewer individuals')],\n",
       "  [('we', 'study', 'S E IR models')],\n",
       "  [('infectious periods', 'give rise to', 'Markov models'),\n",
       "   ('periods', 'give rise to', 'Markov models')],\n",
       "  [('such models',\n",
       "    'can',\n",
       "    'In deterministic setting can described with integral equations')],\n",
       "  [('ﬁnal size', 'treat speciﬁcally in', 'NumericCitation')],\n",
       "  [('C', 'number of', 'infectious contacts'),\n",
       "   ('infected individual', 'have during', 'its infectious period')],\n",
       "  [('R0', 'number for', 'newly infected symptomatic individuals')],\n",
       "  [('we',\n",
       "    'next make',\n",
       "    'use of known results on ﬁnal size Z from NumericCitation')],\n",
       "  [('probability mass fraction', 'with distribution be', 'small outbreaks'),\n",
       "   ('two-point distribution',\n",
       "    'be with',\n",
       "    'probability mass fraction infected at 0')],\n",
       "  [('symmetry', 'be in', 'asymptomatic cases')],\n",
       "  [('We', 'turn to', 'this')],\n",
       "  [('2.2 Deterministic symptom-response epidemic models',\n",
       "    'be with',\n",
       "    'As nearly all epidemic models'),\n",
       "   ('individuals', 'be in', 'diﬀerent compartments')],\n",
       "  [('work', 'be with', 'deterministic setting')],\n",
       "  [('fractions', 'be in', 'diﬀerent disease states')],\n",
       "  [('asymptomatic individual', 'be', 'infectious at age since infection τ')],\n",
       "  [('population', 'of ea be', 't')],\n",
       "  [('its latent period', 'be with', 'probability P')],\n",
       "  [('2.2.1 Markov models', 'be in', 'Often modelling community')],\n",
       "  [('pa =', 'with k be', '7 k'), ('− αk ek i ′ k =', 'be with', 'pa =')],\n",
       "  [('consistency', 'require', '1 =')],\n",
       "  [('ιk ≡ xk', 'be with', 'ℓk'), ('ℓk', 'with xk be', 'k d')],\n",
       "  [('k', '=', 'd')],\n",
       "  [('we', 'diﬀerentiating with respect to', 't')],\n",
       "  [('it', 'be', 'straightforward')],\n",
       "  [('µn → µ', 'be with', 'µ')],\n",
       "  [('population size n', 'be', 'when large enough')],\n",
       "  [('positive fraction', 'be in', 'population at beginning of epidemic')],\n",
       "  [('fraction', 'tend to', 'zero')],\n",
       "  [('we', 'worry', 'too much'),\n",
       "   ('we', 'worry', 'too much about initial conditions')],\n",
       "  [('model framework', 'be in', 'Section 2')],\n",
       "  [('new infections', 'be cause by', 'symptomatics'),\n",
       "   ('time t', 'at s be', 't'),\n",
       "   ('new infections', 'be cause by', 'asymptomatics'),\n",
       "   ('new infections', 'be', 'caused'),\n",
       "   ('new infections', 'be', 'caused')],\n",
       "  [('this', 'characterize ρd in', 'principle')],\n",
       "  [('we', 'make', 'either of them more explicit in general setting')],\n",
       "  [('infected individual', 'become', 'symptomatic case')],\n",
       "  [('R0', 'statement regard', 'monotonicity of ρd in pd'),\n",
       "   ('monotonicity', 'be in', 'pd')],\n",
       "  [('we', 'change', 'ιd')],\n",
       "  [('we', 'change', 'also R0')],\n",
       "  [('monotonicity', 'be in', 'pd'), ('we', 'can obtain from', '10')],\n",
       "  [('extreme model d', 'd for', 'symmetry reasons')],\n",
       "  [('symptomatic individual', 'i.e. cause', 'more infections')],\n",
       "  [('would case', 'be with', 'Markov assumption')],\n",
       "  [('it', 'infect other later in', 'in epidemic')],\n",
       "  [('asymptomatics', 'also have', 'very intense infectious pressure'),\n",
       "   ('asymptomatics', 'have', 'very intense infectious pressure')],\n",
       "  [('changes', 'be in', 'latent periods'),\n",
       "   ('La', 'be', 'where long relative')],\n",
       "  [('proof', 'be in', 'NumericCitation')],\n",
       "  [('general idea', 'be in', 'proof presented')],\n",
       "  [('type',\n",
       "    'be with',\n",
       "    'edge pointing towards individual under consideration')],\n",
       "  [('We',\n",
       "    'consider',\n",
       "    'backward branching process of individual under consideration')],\n",
       "  [('9', 'be', 'distributed')],\n",
       "  [('they', 'become', 'infected themselves')],\n",
       "  [('calculations', 'be in', 'NumericCitation')],\n",
       "  [('We', 'discuss ηd in', 'more detail')],\n",
       "  [('upper-and lower bounds', 'd for', 'details'),\n",
       "   ('distributions', 'for bound be', 'd')],\n",
       "  [('3.3 epidemic curve', 'be in', 'section')],\n",
       "  [('we', 'characterize', 'curve')],\n",
       "  [('we',\n",
       "    'be in',\n",
       "    'setting of epidemic outbreak generated by symptomatic individuals only')],\n",
       "  [('we', 'condition on', 'happening')],\n",
       "  [('10 3.3.1 continuous-time Reed-Frost model',\n",
       "    'be in',\n",
       "    'continuous-time Reed-Frost model'),\n",
       "   ('periods', 'be', 'deterministic')],\n",
       "  [],\n",
       "  [('it', 'indeed start with', '1 − pd')],\n",
       "  [('k ≥ 3', 'for d be', 'ηk − ηk − 1'), ('k ≥ 3', 'for d be', '19')],\n",
       "  [('continuous-time Reed-Frost', 'be with', 'deterministic latent periods'),\n",
       "   ('asymptomatic case', 'be in', 'latent stage')],\n",
       "  [('epidemic starts', 'be with', 'symptomatic outbreak')],\n",
       "  [('qualitative two-wave process',\n",
       "    'be illustrate in',\n",
       "    'Fig. 2 of Section 4')],\n",
       "  [('numerical simulations', 'be in', 'section')],\n",
       "  [('Reed-Frost assumption', 'be in', 'temporal behaviour')],\n",
       "  [('We', 'consider', 'following two scenarios')],\n",
       "  [('scenario 1', 'for e be', 'La'),\n",
       "   ('asymptomatic cases', 'for e be', 'La'),\n",
       "   ('scenario 2', 'for e be', 'La')],\n",
       "  [('ﬁnal fraction', 'be in', 'epidemic')],\n",
       "  [('RF model', 'describe in', 'Section 3.3.1')],\n",
       "  [('inset ﬁgures', 'zoom in on', '0 25')],\n",
       "  [('beginning', 'be in', 'as ﬁnal size of epidemic')],\n",
       "  [('we', 'state in', 'Section 3.2')],\n",
       "  [('E', '=', '1.4'), ('d d E', '=', '5'), ('E', '=', '2')],\n",
       "  [('ρd', 'be', 'increasing'), ('ρd', 'be increase function of', 'pd')],\n",
       "  [('R0', 'for pd be', 'R0'), ('R0', 'for pd be', 'd =')],\n",
       "  [('E', '=', '1.4'), ('E', '=', '5'), ('E', '=', '2')],\n",
       "  [('increasing functions', 'on r0 be', 'λd'),\n",
       "   ('increasing functions', 'be in', 'example')],\n",
       "  [('pd', 'for λd be', 'R0 ='),\n",
       "   ('pd', 'for λd be', 'R0'),\n",
       "   ('pd', 'for λd be', 'd =')],\n",
       "  [('E', '=', '5')],\n",
       "  [('4.3 Illustration', 'be with', 'norovirus'),\n",
       "   ('parameter values', 'be in', 'reasonable ranges taken from literature')],\n",
       "  [('section', 'illustrate', 'role of ρd')],\n",
       "  [('we', 'incorporate', 'complexity of two infections')],\n",
       "  [('norovirus outbreak', 'be in', 'hospital setting'),\n",
       "   ('we', 'take', 'reference values'),\n",
       "   ('we', 'take', 'reference values based on NumericCitation')],\n",
       "  [('NumericCitation report', 'estimate', 'reproduction numbers')],\n",
       "  [('R0', 'hence =', '1.38'), ('R0', '=', '1.38')],\n",
       "  [('measles', 'be in', 'England'),\n",
       "   ('vaccine era', 'be with', 'subclinical infections'),\n",
       "   ('second scenario',\n",
       "    'represent',\n",
       "    'measles in England in vaccine era with subclinical infections'),\n",
       "   ('measles', 'be in', 'vaccine era with subclinical infections')],\n",
       "  [('model', 'be in', 'NumericCitation')],\n",
       "  [('curves', 'be', 'standard')],\n",
       "  [('two curves', 'be', 'Qualitatively similar')],\n",
       "  [('Parameter values', 'be', 'summarized'),\n",
       "   ('Parameter values', 'be summarize in', 'Table 2')],\n",
       "  [('we', 'occur in', 'epidemic')],\n",
       "  [('infections', 'be in', 'norovirus scenario'),\n",
       "   ('symptomatic cases', 'cause', 'most of infections in norovirus scenario')],\n",
       "  [('spread', 'be in', 'outbreak settings')],\n",
       "  [('5 Conclusion', 'be in', 'paper'),\n",
       "   ('SEIR model', 'be with', 'symptomatic cases')],\n",
       "  [('quantity ρd',\n",
       "    'give',\n",
       "    'indication of importance of symptomatic cases in transmission process in epidemic'),\n",
       "   ('importance', 'be in', 'transmission process'),\n",
       "   ('importance', 'be in', 'epidemic')],\n",
       "  [('it', 'make', 'statements about ρd')],\n",
       "  [('we', 'instead use timing issue to', 'our advantage')],\n",
       "  [('ρd', 'be in', 'Section 4')],\n",
       "  [('ρd',\n",
       "    'increase with',\n",
       "    'increasing infectious contact rate λd from symptomatic cases')],\n",
       "  [('two special classes', 'class of', 'Markov models'),\n",
       "   ('continuous-time Reed-Frost models',\n",
       "    'be with',\n",
       "    'deterministic latent periods'),\n",
       "   ('two special classes',\n",
       "    'class with',\n",
       "    'latent periods exponentially distributed'),\n",
       "   ('class', 'be with', 'latent periods exponentially distributed')],\n",
       "  [('epidemic curve', 'be with', 'ﬁxed mean'),\n",
       "   ('latent periods', 'be in', 'Sections'),\n",
       "   ('epidemic curve', 'be in', 'extreme setting')],\n",
       "  [('This', 'be', 'valid')],\n",
       "  [('our ﬁrst aim', 'illustrate', 'possible eﬀects')],\n",
       "  [('infector', 'study r0 for', 'beginning of epidemic')],\n",
       "  [('We', 'gain', 'insights')],\n",
       "  [('authors', 'be support by', 'VR')],\n",
       "  [('epidemic model', 'be with', 'several diﬀerent types of infective'),\n",
       "   ('ﬁnal outcome', 'be in', 'large population')],\n",
       "  [('epidemic model', 'be with', 'infector-dependent severity')],\n",
       "  [('tra jectory', 'be in', 'epidemic models')],\n",
       "  [('epidemic model', 'be with', 'exposure-dependent severities')],\n",
       "  [('Norovirus', 'be in', 'symptomatic individuals')],\n",
       "  [('Waning immunity', 'be in', 'England')],\n",
       "  [('NumericCitation Centers', 'be in', 'Chief')],\n",
       "  [('determinant', 'be in', 'Mycoplasma pneumoniae incidence')],\n",
       "  [('Adv.', 'be in', 'Appl')],\n",
       "  [('epidemic spread', 'be in', 'metapopulation model')]],\n",
       " 2: [[('stochastic Susceptible-Exposed-Infected-Recovered SEIR epidemiological model',\n",
       "    'be with',\n",
       "    'contact rate')],\n",
       "  [('our method',\n",
       "    'give',\n",
       "    'seasonal epidemic data consisting of number of infectious individuals')],\n",
       "  [('Seasonality',\n",
       "    'with epidemic be',\n",
       "    'Model Reduction 1 Introduction As dynamical process in epidemics'),\n",
       "   ('Keywords Epidemics', 'be with', 'Seasonality'),\n",
       "   ('Keywords Epidemics',\n",
       "    'introduction as',\n",
       "    'dynamical process in epidemics')],\n",
       "  [('childhood epidemics', 'be in', 'cities'),\n",
       "   ('childhood epidemics', 'to dynamics be', 'Citation')],\n",
       "  [('play', 'be in', 'prediction of large outbreaks'),\n",
       "   ('populations', 'be with', 'seasonal forcing')],\n",
       "  [('their', 'control', 'Citation')],\n",
       "  [('random contacts', 'be in', 'ﬁnite well-mixed populations')],\n",
       "  [('it', 'have', 'far ranging eﬀects on outbreak predictability'),\n",
       "   ('it', 'have', 'far ranging eﬀects')],\n",
       "  [('noise impacts predictability', 'be in', 'climate'),\n",
       "   ('its interaction', 'be with', 'mass action terms'),\n",
       "   ('its interaction', 'be with', 'resonant frequencies')],\n",
       "  [('chaos', 'be in', 'epidemics')],\n",
       "  [('use', 'be in', 'epidemic models')],\n",
       "  [('deterministic processes', 'dominate', 'time series')],\n",
       "  [('sensitive dependence',\n",
       "    'be in',\n",
       "    'short noisy non-chaotic time series such epidemic data')],\n",
       "  [('authors', 'get', 'measures of local reproductive rates of infection')],\n",
       "  [('predictive power', 'construct', 'model')],\n",
       "  [('Connections', 'be with', 'full models')],\n",
       "  [('dynamics', 'be in', 'higher dimensional deterministic models')],\n",
       "  [('dynamics', 'be in', 'reductions')],\n",
       "  [('stochastic model reduction', 'be', 'done correctly')],\n",
       "  [('noise', 'be', 'whereby pro')],\n",
       "  [('systems', 'be with', 'certain spectral requirements')],\n",
       "  [('data', 'ﬁt', 'predictive model of SEIR type')],\n",
       "  [('each', 'evolve in', 'time')],\n",
       "  [('Each', 'recover', 'individual')],\n",
       "  [('1b', 'be', 'stochastic'),\n",
       "   ('φ', 'be', 'white'),\n",
       "   ('1b', 'be', 'white'),\n",
       "   ('φ', 'be', 'stochastic')],\n",
       "  [('multiplicative noise', 'be in', 'infectives')],\n",
       "  [('reports', 'be with', 'monitoring agencies')],\n",
       "  [('dynamics', 'be describe suﬃciently by', 'Eqs')],\n",
       "  [('1a', 'use', 'deterministic center manifold theory')],\n",
       "  [('R0',\n",
       "    'give',\n",
       "    'number of secondary cases produced by lone infectious individual in population of susceptible individuals during one infectious period')],\n",
       "  [('system', 'have', 'linear part')],\n",
       "  [('eigenvalues', 'be with', 'negative real part'),\n",
       "   ('third matrix block', 'will possess', 'eigenvalues with zero real part'),\n",
       "   ('second matrix block',\n",
       "    'will possess',\n",
       "    'eigenvalues with negative real part'),\n",
       "   ('eigenvalues', 'be with', 'zero real part'),\n",
       "   ('eigenvalues', 'be with', 'positive real part')],\n",
       "  [('system', 'be in', 'separated block form'),\n",
       "   ('system', 'be with', 'separated stable')],\n",
       "  [('we', 'therefore transform', 'original system given by Eqs')],\n",
       "  [('new system', 'to 1a be', '1c')],\n",
       "  [('new variables', 'be substitute into', 'Eqs')],\n",
       "  [('we', 'treat µ as', 'small parameter'), ('t', '=', 'µτ')],\n",
       "  [('terms', 'be in', 'our rescaled system')],\n",
       "  [('I', 'dτ', 'dµ dτ'), ('I', '¯', 't')],\n",
       "  [('transformed system', 'give by', 'Eqs')],\n",
       "  [('8a', 'be', 'non-autonomous')],\n",
       "  [('cosine terms', 'be in', 'Eq'), ('We', 'replace', 'cosine terms in Eq')],\n",
       "  [('autonomous system', 'consist of', 'Eqs')],\n",
       "  [('it', 'transform', 'Eqs')],\n",
       "  [('block', 'to 8a be', '8c'),\n",
       "   ('diagonal form', 'be with', 'separated eigenvalue structure')],\n",
       "  [('We', 'use', 'P')],\n",
       "  [('Eqs', 'with β be', 't'),\n",
       "   ('β', 'be with', 'Eqs'),\n",
       "   ('cosine terms', 'be in', 'β with Eqs')],\n",
       "  [('zeroth-order', 'be in', 'µ')],\n",
       "  [('y =', 'matrix with', 'eigenvalues'),\n",
       "   ('nonlinear functions', 'be in', 'x'),\n",
       "   ('A', 'be', 'constant'),\n",
       "   ('constant matrix', 'be with', 'eigenvalues'),\n",
       "   ('constant matrix', 'be with', 'eigenvalues'),\n",
       "   ('B', 'be', 'constant')],\n",
       "  [('center manifold', 'be give by', '19'),\n",
       "   ('h', 'be', 'unknown'),\n",
       "   ('h', 'be', 'where unknown function')],\n",
       "  [('β µ h V', 'be in', 'general')],\n",
       "  [('Taylor series expansion', 'be in', 'V')],\n",
       "  [('unknown coeﬃcients', 'equate', 'terms of same order')],\n",
       "  [('we', 'now return to', 'stochastic SEIR system given by Eqs')],\n",
       "  [('we', 'ﬁrst', 'order in µ'),\n",
       "   ('we', 'ﬁrst', 'order'),\n",
       "   ('order', 'be in', 'µ')],\n",
       "  [('only diﬀerence', 'be eﬀect of', 'transformation on stochastic term')],\n",
       "  [('one multiplicative noise term', 'be in', 'one equation')],\n",
       "  [('additive noise terms', 'be in', 'equation'),\n",
       "   ('new additive noise term',\n",
       "    'be with',\n",
       "    'variance diﬀerent from original noise process')],\n",
       "  [('we',\n",
       "    'additionally have explicitly compute',\n",
       "    'normal form coordinate transformation')],\n",
       "  [('stochastic terms',\n",
       "    'thus justify',\n",
       "    'our use of deterministic center manifold for particular model')],\n",
       "  [('stochastic fourth-order Runge-Kutta integrator',\n",
       "    'be with',\n",
       "    'constant time step size')],\n",
       "  [('U', 'use', 'center manifold equation given by Eq')],\n",
       "  [('ﬁrst time series',\n",
       "    'use',\n",
       "    'complete stochastic system of transformed equations'),\n",
       "   ('second time series', 'use', 'reduced stochastic system of equations')],\n",
       "  [('initial disease outbreak', 'be correctly capture by', 'reduced model')],\n",
       "  [('reduced model',\n",
       "    'furthermore correctly predict outbreak for',\n",
       "    'time scale on order of decades')],\n",
       "  [('mandatory notiﬁcation', 'be in', 'UK Citation'),\n",
       "   ('6 Comparison', 'be with', 'Data Measles')],\n",
       "  [('60 cities', 'be in', 'England'), ('We', 'use', 'Citation')],\n",
       "  [('ζ2', '=', '− 0.2554'), ('ω', '=', '0.521')],\n",
       "  [('resulting time series', 'be also show in', 'Fig. 2')],\n",
       "  [('cross-correlation', 'measure', 'similarity between two time series')],\n",
       "  [('noise', 'be with', 'intensity σ')],\n",
       "  [('larger noise',\n",
       "    'lead to',\n",
       "    'signiﬁcant diﬀerences between two reduced model solutions')],\n",
       "  [('we', 'obtain', 'comprehensive idea of eﬀect of noise')],\n",
       "  [('increase', 'be in', 'speed of infection per infectious period'),\n",
       "   ('increase', 'be in', 'R0')],\n",
       "  [('increase', 'be in', 'number of exposed individuals'),\n",
       "   ('increase', 'induce', 'increase in number of exposed individuals')],\n",
       "  [('ﬁrst time series',\n",
       "    'use',\n",
       "    'complete stochastic system of transformed equations')],\n",
       "  [('one', 'model', 'disease exposure time')],\n",
       "  [('one', 'reduce', 'analytical complications introduced by time delay')],\n",
       "  [('seasonal epidemic data 13', 'be with', 'speciﬁc eigenvalue structure')],\n",
       "  [('we', 'employ', 'center manifold theory')],\n",
       "  [('we', 'reduce', 'stochastic SEIR model')],\n",
       "  [('result', 'be', 'stochastic'), ('result', 'be', 'reduced')],\n",
       "  [('we', 'ﬁt', 'measles data'), ('we', 'ﬁt', 'actual measles data')],\n",
       "  [('resulting solution', 'be with', 'data')],\n",
       "  [('good agreement', 'be with', 'data'),\n",
       "   ('we', 'provide', 'good agreement with data')],\n",
       "  [('stochastic solutions', 'be', 'In way very robust to noise')],\n",
       "  [('We',\n",
       "    'further identify noise eﬀects through',\n",
       "    'cross-correlation computations')],\n",
       "  [('epidemiological model', 'be with', 'seasonal ﬂuctuations')],\n",
       "  [('disease outbreak', 'be', 'where important')],\n",
       "  [('models', 'be with', 'high dimension')],\n",
       "  [('Stochastic ampliﬁcation', 'be in', 'epidemics')],\n",
       "  [('Ira B. Schwartz Arnold', 'l. at_time', '1998')],\n",
       "  [('Phase-space transport', 'be in', 'population dynamics of virus spread')],\n",
       "  [('unexpected dynamics', 'be in', 'epidemic outbreaks'),\n",
       "   ('Exciting chaos', 'be with', 'noise')],\n",
       "  [('Resonance eﬀects', 'be in', 'ecological time series')],\n",
       "  [('Eﬃciency', 'be in', 'England')],\n",
       "  [('Hopf bifurcation', 'be with', 'noise')],\n",
       "  [('Measles', 'be in', 'England')],\n",
       "  [('Measles', 'be in', 'England')],\n",
       "  [('Immunity', 'be in', 'Population')],\n",
       "  [('Advances', 'be in', 'Global Change Research')],\n",
       "  [('Bifurcations', 'be in', 'ﬂuctuating systems')],\n",
       "  [('delays', 'be in', 'biochemical systems')],\n",
       "  [('Epidemic outbreaks', 'be in', 'complex heterogeneous networks')],\n",
       "  [('extinction', 'be in', 'recurrent epidemics')],\n",
       "  [('Epidemic dynamics', 'be in', 'complex networks')],\n",
       "  [('ubiquitous source', 'be in', 'epidemics')],\n",
       "  [('interplay', 'be in', 'childhood diseases')],\n",
       "  [('Transient periodicity', 'be in', 'biological dynamics')],\n",
       "  [('Inﬁnite subharmonic bifurcations', 'be in', 'SEIR epidemic model')],\n",
       "  [('chaos', 'be in', 'childhood epidemics')],\n",
       "  [('small-world networks', 'be with', 'connectivity ﬂuctuations')],\n",
       "  [('F3 expressions', 'find in', 'Eqs')],\n",
       "  [('fraction', 'of 30 be', 'd')],\n",
       "  [('60 Citation -1966', 'from datum be', 'Citation')],\n",
       "  [('ζ2', '=', '− 0.2554'), ('ω', '=', '0.521')],\n",
       "  [('series', 'be', 'average')],\n",
       "  [('reduced model', 'be in', '0.810')],\n",
       "  [('ζ2', '=', '− 0.2554'), ('ω', '=', '0.521')],\n",
       "  [('data time series', 'between ln be', 'σ')],\n",
       "  [('24 Eric Forgoston', '−', 'I'), ('24 Eric Forgoston', '−', 'I')],\n",
       "  [('ζ2', '=', '− 0.2554'), ('ω', '=', '0.521')]],\n",
       " 3: [[('Citation', 'be deadliest epidemic in', 'history')],\n",
       "  [('Universit', None, 'Paul Sabatier'),\n",
       "   ('Research', 'be in', 'Mathematics'),\n",
       "   ('Mathematics', 'in research be', '3810-193 Aveiro'),\n",
       "   ('Mathematics', 'in research be', 'Portugal E-mail'),\n",
       "   ('delﬁ 2 A. Rachah', 'torres for', 'humans'),\n",
       "   ('Mathematics', 'in research be', 'Department of Mathematics'),\n",
       "   (\"l'Industrie et la Physique\", 'sabatier', 'Universit'),\n",
       "   ('Mathematics', 'in research be', 'University of Aveiro'),\n",
       "   ('Universit', 'sabatier be', 'F-31062 Toulouse Cedex 9'),\n",
       "   ('Research', 'department of', 'Mathematics'),\n",
       "   ('Research', 'university of', 'Aveiro'),\n",
       "   ('Universit', 'sabatier be', 'France E-mail')],\n",
       "  [('ﬁ Citation', 'be in', 'Congo')],\n",
       "  [('Ebola virus', 'cause', 'acute often fatal haemorrhagic illness')],\n",
       "  [('current outbreak', 'be in', 'West Africa')],\n",
       "  [('infection', 'be with', 'virus')],\n",
       "  [('symptoms', 'be', 'In ﬁnal stage followed by death NumericCitation')],\n",
       "  [('close physical contact', 'be with', 'infected bodily ﬂuids')],\n",
       "  [('key tool', 'be in', 'epidemiology'), ('models', 'be', 'key')],\n",
       "  [('we', 'refer reader to', 'NumericCitation')],\n",
       "  [('spread', 'be in', 'West Africa')],\n",
       "  [('we', 'use', 'proposed model')],\n",
       "  [('we', 'consider', 'two diﬀerent ob jective functions'),\n",
       "   ('we', 'consider', 'two diﬀerent ob functions')],\n",
       "  [('population', 'be', 'where divided into four groups'),\n",
       "   ('2 basic model equations', 'be in', 'section')],\n",
       "  [('t', 'be denote at', 'time')],\n",
       "  [('Its numerical resolution', 'be study in', 'NumericCitation')],\n",
       "  [('population', 'be', 'constant in period of time under study')],\n",
       "  [('4 A. Rachah', 'model for', 'non-constant population')],\n",
       "  [('we', 'also include', 'educational campaign with vaccination treatment')],\n",
       "  [('great importance', 'be in', 'countries')],\n",
       "  [('Strategy 3',\n",
       "    'more precisely consist in',\n",
       "    'study of educational campaign about virus coupled with vaccination treatment')],\n",
       "  [('optimal control problems', 'be with', 'help of ACADO')],\n",
       "  [('3.1 Strategy 1', 'be in', 'subsection')],\n",
       "  [('Dynamics', 'be with', 'control')],\n",
       "  [('optimal control problem', 'minimize', 'u')],\n",
       "  [('Our goal', 'be in', 'strategy')],\n",
       "  [('our optimal control problem', 'minimize', 'u')],\n",
       "  [('two ﬁrst terms', 'be in', 'functional ob jective')],\n",
       "  [('ν', 'be', 'positive')],\n",
       "  [('6 A. Rachah', 'citation by', 'Althaus'),\n",
       "   ('optimal control', 'of model be', '2')],\n",
       "  [('Wolfram', 'with', 'Mathematica language')],\n",
       "  [('infectious rate γ', '=', '0.1887'),\n",
       "   ('same recovered rate µ', '=', '0.1')],\n",
       "  [('signiﬁcant diﬀerence', 'be in', 'number of susceptible'),\n",
       "   ('individuals', 'be with', 'Strategy')],\n",
       "  [('campaign', 'be with', 'Strategy 1'),\n",
       "   ('17.3 %', 'be in', 'absence of optimal control'),\n",
       "   ('contrast', 'be with', '6.6 % at end of campaign')],\n",
       "  [('Figure 4', 'present', 'time-dependent curve of exposed individuals')],\n",
       "  [('control', 'be with', 'Strategy 2')],\n",
       "  [('control', 'be with', 'Strategy 2'), ('50 days', 'be in', 'Strategy 1')],\n",
       "  [('control', 'be with', 'Strategy 2')],\n",
       "  [('7.56 %', 'be in', 'Strategy 1')],\n",
       "  [('Strategy 1', 'in case of s be', 'days'),\n",
       "   ('3.3 Strategy 3', 'be in', 'strategy')],\n",
       "  [('Strategy 1', 'in case of e be', 'days'),\n",
       "   ('8 A. Rachah', 'e without', 'control Strategy 1 Strategy')],\n",
       "  [('vaccination treatment', 'be in', 'practical Ebola situations')],\n",
       "  [('case', 'cure of', 'Ebola')],\n",
       "  [('close physical contact', 'be with', 'infected bodily ﬂuids')],\n",
       "  [('10 A. Rachah', 'torres to', 'population'),\n",
       "   ('total number', 'cost of', 'treatment')],\n",
       "  [('ﬁrst term', 'be in', 'ob jective')],\n",
       "  [('optimal control', 'be with', 'Strategy 3')],\n",
       "  [('17.3 %', 'be in', 'absence of optimal control')],\n",
       "  [('Strategy 3', 'of j be', 'u')],\n",
       "  [('4 Discussion', 'be in', 'section'),\n",
       "   ('we', 'compare between', 'three strategies of Section 3'),\n",
       "   ('we', 'discuss', 'obtained results')],\n",
       "  [('Strategy 1', 'in case of s be', 'days')],\n",
       "  [('Strategy 1', 'in case of e be', 'days'),\n",
       "   ('12 A. Rachah', 'e without', 'control Strategy 1 Strategy')],\n",
       "  [('Strategy 1', 'in case of t be', 'days')],\n",
       "  [('great importance', 'be in', 'poor countries')],\n",
       "  [('Citation Ebola outbreak', 'be in', 'West Africa')],\n",
       "  [('we', 'precisely introduce', 't')],\n",
       "  [('population', 'into campaign be', 'Section 3.3'),\n",
       "   ('virus', 'about campaign be', 'Section 3.3')],\n",
       "  [('great importance', 'be with', 'vaccination treatment')],\n",
       "  [('we', 'plan as', 'future work')],\n",
       "  [('Strategy 1', 'in case of s be', 'days')],\n",
       "  [('Strategy 1', 'in case of t be', 'days')],\n",
       "  [('Strategy 1', 'in case of t be', 'days')],\n",
       "  [('Strategy 1', 'in case of t be', 'days')],\n",
       "  [('Science', 'for foundation be', 'FCT'),\n",
       "   ('CIDMA pro ject UID/MAT/04106', 'within foundation be', 'FCT'),\n",
       "   ('POFC-QREN', 'be with', 'COMPETE reference FCOMP-01-0124-FEDER-028894')],\n",
       "  [('reproduction number', 'be in', 'West Africa')],\n",
       "  [('ACADO toolkit user', 'have', 'manual')],\n",
       "  [('Frontiers', 'be in', 'Applied Mathematics 28')],\n",
       "  [(\"A. Tour ' e. Ebola outbreak\", 'be in', 'Conakry')],\n",
       "  [('Vaccine control', 'be in', 'poultry')],\n",
       "  [('SIR epidemic model', 'be with', 'pulse vaccination')],\n",
       "  [('automatic control', 'for framework be', 'Optimal Control Appl')],\n",
       "  [('Ebola', 'be with', 'Wolfram guage')],\n",
       "  [('Ebola virus transmission', 'be in', 'Montserrado')],\n",
       "  [('Citation Ebola outbreak', 'be in', 'West Africa')],\n",
       "  [('tuberculosis model', 'be with', 'reinfection')],\n",
       "  [('Optimal control', 'be with', 'treatment')],\n",
       "  [('outbreak', 'be in', 'Uganda')],\n",
       "  [(\"Collectif L'epid\", \"d'eveloppement en\", 'France')],\n",
       "  [('Integrated Series', 'be in', 'Information Systems')]],\n",
       " 4: [[('We',\n",
       "    'consider',\n",
       "    'stochastic Susceptible-Exposed-Infected-Recovered SEIR epidemiological model')],\n",
       "  [('excellent agreement', 'be in', 'amplitude'),\n",
       "   ('excellent agreement',\n",
       "    'be with',\n",
       "    'solution of original stochastic system for temporal scale')],\n",
       "  [('spread', 'be in', 'population')],\n",
       "  [('systems', 'be', 'important'),\n",
       "   ('1 reduction', 'be in', 'nonlinear dynamical systems'),\n",
       "   ('systems', 'be', 'high-dimensional')],\n",
       "  [('phase diﬀerences', 'be', 'where important')],\n",
       "  [('deterministic eﬀects', 'be in', 'population dynamics'),\n",
       "   ('I. INTRODUCTION', 'role in', 'modeling of infectious diseases'),\n",
       "   ('important role', 'be in', 'modeling of infectious diseases')],\n",
       "  [('models', 'arise from', 'mean ﬁeld analysis of probabilistic models')],\n",
       "  [('ecological dynamics3', 'be in', 'cities4 ,5'),\n",
       "   ('Stochastic eﬀects', 'be', 'quite prominent')],\n",
       "  [('play', 'be in', 'prediction of large outbreaks6 ,7,8'),\n",
       "   ('homogeneous populations', 'be with', 'seasonal forcing')],\n",
       "  [('geometric structure', 'be', 'When applied to population')],\n",
       "  [('nodes changes', 'be in', 'response to node information')],\n",
       "  [('delay term', 'make', 'analysis signiﬁcantly more complicated')],\n",
       "  [('system', 'be with', 'certain spectral requirements')],\n",
       "  [('we', 'derive', 'reduced stochastic center manifold equation'),\n",
       "   ('coordinate', 'transform', 'more transparent')],\n",
       "  [('Figure 1', 'schematic demonstrate', 'our approach to problem')],\n",
       "  [('SusceptibleExposed-Infected-Recovered SEIR epidemiological model',\n",
       "    'be with',\n",
       "    'stochastic forcing')],\n",
       "  [('4 simpler standard SEIR model', 'demonstrate', 'power of our method')],\n",
       "  [('Section II', 'provide', 'complete description of model')],\n",
       "  [('conclusions', 'be', 'contained respectively in Sec')],\n",
       "  [('We', 'describe', 'stochastic version of SEIR model found in Ref')],\n",
       "  [('γ', 'be rate of', 'recovery')],\n",
       "  [('Disease parameters', 'correspond to', 'typical measles')],\n",
       "  [('interpretation',\n",
       "    'be number of',\n",
       "    'secondary cases produced by single infectious individual in population of susceptibles in one infectious period')],\n",
       "  [('it', 'estimate', 'ﬂuctuating migration rates29'),\n",
       "   ('it', 'treat migration as', 'arbitrary external noise source'),\n",
       "   ('it', 'estimate', 'migration rates29')],\n",
       "  [('ﬂuctuations', 'be in', 'birth rate manifest itself as additive noise')],\n",
       "  [('extinction events', 'be in', 'article')],\n",
       "  [('rare event', 'be in', 'tail of noise distribution'),\n",
       "   ('I', 'become', 'negative')],\n",
       "  [('dynamics', 'be suﬃciently describe by', 'Eqs')],\n",
       "  [('third matrix block', 'have', 'eigenvalues with zero real part30 ,31'),\n",
       "   ('eigenvalues', 'be with', 'negative real part'),\n",
       "   ('eigenvalues', 'be with', 'zero real part30 ,31'),\n",
       "   ('second matrix block', 'have', 'eigenvalues with negative real part'),\n",
       "   ('vector ﬁeld', 'of part be', 'Jacobian'),\n",
       "   ('eigenvalues', 'be with', 'positive real part')],\n",
       "  [('new system', 'to 1a be', '1c')],\n",
       "  [('we', 'can restrict', 'transformed system')],\n",
       "  [('Our analysis',\n",
       "    'consider',\n",
       "    'governing equations for stochastic SEIR model given by Eqs')],\n",
       "  [('σiφi terms', 'be in', 'Eqs')],\n",
       "  [('deterministic system', 'have', 'two ﬁxed points')],\n",
       "  [('variables', 'of set be', 'S')],\n",
       "  [('new variables', 'be substitute into', 'Eqs')],\n",
       "  [('we', 'treat µ as', 'small parameter'), ('t', '=', 'µτ')],\n",
       "  [('terms', 'be in', 'our rescaled system')],\n",
       "  [('I', '−', '5a'),\n",
       "   ('endemic ﬁxed point', 'be', 'where now located at origin')],\n",
       "  [('Jacobian', 'ignore', 'µ components')],\n",
       "  [('it', 'transform', 'Eqs')],\n",
       "  [('diagonal form', 'be with', 'eigenvalue structure')],\n",
       "  [('We', 'use', 'P')],\n",
       "  [('zeroth-order', 'be in', 'µ'), ('zeroth-order', 'to 8a be', '8d')],\n",
       "  [('y =', 'matrix with', 'eigenvalues'),\n",
       "   ('nonlinear functions', 'be in', 'x'),\n",
       "   ('A', 'be', 'constant'),\n",
       "   ('constant matrix', 'be with', 'eigenvalues'),\n",
       "   ('constant matrix', 'be with', 'eigenvalues'),\n",
       "   ('B', 'be', 'constant')],\n",
       "  [('center manifold', 'be give by', '14'),\n",
       "   ('h', 'be', 'unknown'),\n",
       "   ('h', 'be', 'where unknown function')],\n",
       "  [('following Taylor series expansion', 'be in', 'V'),\n",
       "   ('Taylor series expansion', 'equate', 'terms of same order')],\n",
       "  [('19a', 'use', 'Eqs')],\n",
       "  [('t', '=', 'µτ time scaling')],\n",
       "  [('stochastic terms', 'be in', 'Eqs')],\n",
       "  [('argument', 'be in', 'Eqs')],\n",
       "  [('noise terms', 'be evaluate at', 'µτ')],\n",
       "  [('fraction', 'of series be', 'Eqs'), ('fraction', 'of series be', 'I')],\n",
       "  [('20a', 'be', 'exact')],\n",
       "  [('stochastic fourth-order Runge-Kutta scheme',\n",
       "    'be with',\n",
       "    'constant time step size')],\n",
       "  [('independent noise processes', 'be with', 'diﬀerent variance')],\n",
       "  [('noise', 'use', 'Eqs')],\n",
       "  [('It', 'consider', 'reduced stochastic model found by substitution of Eq')],\n",
       "  [('deterministic problem', 'for system be', 'Eqs'),\n",
       "   ('evolution equations', 'of system be', 'Eqs')],\n",
       "  [('We', 'will show', 'Eqs')],\n",
       "  [('Such errors', 'be in', 'stochastic epidemic modeling impact'),\n",
       "   ('spread', 'be in', 'population')],\n",
       "  [('fraction', 'of series be', 'I'), ('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('noise terms', 'of replacement be', 'Eqs'),\n",
       "   ('deterministic center manifold', 'be with', 'replacement of noise terms')],\n",
       "  [('SEIR model', 'of equation be', 'Eqs'),\n",
       "   ('we', 'use', 'same numerical scheme previously described')],\n",
       "  [('noise terms', 'of replacement be', 'Eqs'),\n",
       "   ('reduced system', 'along with 20a be', '20c'),\n",
       "   ('deterministic center manifold', 'be with', 'replacement of noise terms')],\n",
       "  [('U', 'use', 'center manifold equation given by Eq')],\n",
       "  [('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('time series', 'with 20a be', '20c'),\n",
       "   ('20a', 'be with', 'time series of I computed'),\n",
       "   ('noise terms', 'of replacement be', 'Eqs'),\n",
       "   ('deterministic center manifold', 'be with', 'replacement of noise terms')],\n",
       "  [('poor agreement', 'be in', 'phase')],\n",
       "  [('low value',\n",
       "    'quantitatively suggest',\n",
       "    'poor agreement between two solutions')],\n",
       "  [('we', 'use', 'same systems of transformed equations')],\n",
       "  [('data', 'ignore ﬁrst at_time', '40 years of transient solution')],\n",
       "  [('equations', 'be with', 'replacement of noise terms')],\n",
       "  [('color-bar values', 'be in', 'Figs. 4')],\n",
       "  [('two probability distributions', 'qualitatively look', 'same')],\n",
       "  [('two distributions', 'use', 'quantitative measure')],\n",
       "  [('23 dK L =', 'refer to', 'component of probability density found')],\n",
       "  [('noise terms', 'of replacement be', 'Eqs'),\n",
       "   ('deterministic center manifold', 'be with', 'replacement of noise terms')],\n",
       "  [('color-bar values', 'have', 'have normalized by 10 − 3')],\n",
       "  [('phase', 'be with', 'one another')],\n",
       "  [('we',\n",
       "    'will derive',\n",
       "    'normal form coordinate transform for complete stochastic system of transformed equations of SEIR model given by Eqs')],\n",
       "  [('we', 'construct', 'normal form coordinate')],\n",
       "  [('technique', 'apply to', 'large ﬂuctuations of multiscale problems17')],\n",
       "  [('terms', 'be in', 'evolution equations')],\n",
       "  [('fast time', 'be in', 'evolution equations')],\n",
       "  [('equations', 'be in', 'U')],\n",
       "  [('we', 'outline', 'procedure'),\n",
       "   ('simple example', 'be in', 'Appendix A. Several iterations'),\n",
       "   ('X2 dynamics', 'be in', 'new coordinate system')],\n",
       "  [('memory integrals', 'be', 'fast-time processes')],\n",
       "  [('fraction', 'of series be', 'I'), ('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('stochastic normal form coordinate', 'transform', 'Eqs')],\n",
       "  [('speciﬁc form', 'be in', 'Eqs')],\n",
       "  [('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('stochastic normal form coordinate', 'transform', 'Eqs'),\n",
       "   ('reduced system', 'along with 20a be', '20c')],\n",
       "  [('U', 'use', 'center manifold equation given by Eq')],\n",
       "  [('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('stochastic normal form coordinate', 'transform', 'Eqs'),\n",
       "   ('time series', 'with 20a be', '20c'),\n",
       "   ('20a', 'be with', 'time series of I computed')],\n",
       "  [('initial outbreak', 'be successfully capture by', 'reduced system')],\n",
       "  [('we', 'use', 'same systems of transformed equations')],\n",
       "  [('color-bar values', 'be in', 'Figs. 6')],\n",
       "  [('SEIR model', 'be with', 'mortality')],\n",
       "  [('reduced system', 'be with', 'mortality')],\n",
       "  [('color-bar values', 'have', 'have normalized by 10 − 3')],\n",
       "  [('It',\n",
       "    'consider',\n",
       "    'replacement of stochastic term by deterministic period drive of small amplitude')],\n",
       "  [('cases', 'be discuss respectively in', 'Sec')],\n",
       "  [('One', 'could rederive', 'normal form for such deterministic function')],\n",
       "  [('simple replacement', 'be with', 'deterministic realization')],\n",
       "  [('It', 'ﬁnd', 'normal form coordinate transform for periodic forcing')],\n",
       "  [('fraction', 'of series be', 'I'), ('SEIR model', 'of equation be', 'Eqs')],\n",
       "  [('normal form coordinate', 'transform', 'Eqs')],\n",
       "  [('stochastic terms', 'be in', 'systems')],\n",
       "  [('stochastic normal form coordinate', 'transform', 'Eqs'),\n",
       "   ('time series', 'with 20a be', '20c'),\n",
       "   ('20a', 'be with', 'time series of I computed')],\n",
       "  [('One', 'can see in', 'a'),\n",
       "   ('One', 'can see in', 'b'),\n",
       "   ('two solutions', 'relatively synchronize with', 'one another')],\n",
       "  [('we', 'have perform', 'Markov simulation of system')],\n",
       "  [('we', 'complete', 'Markov simulation')],\n",
       "  [('us', 'perform simulation for', '500 realizations')],\n",
       "  [('color-bar values', 'be in', 'Fig. 9'),\n",
       "   ('complete system', 'be in', 'transformed variables')],\n",
       "  [('we', 'bind dynamics to', 'ﬁrst octant'), ('s', '≥', '0')],\n",
       "  [('inequalities', 'can', 'can found in Appendix C as Eq')],\n",
       "  [('C2a', 'be in', 'Appendix C'), ('Appendix C', 'in c2a be', 'C2c')],\n",
       "  [('fraction', 'of series be', 'I')],\n",
       "  [('color-bar values', 'have', 'have normalized by 10 − 4')],\n",
       "  [('C3', 'be in', 'Appendix C')],\n",
       "  [('we', 'transform yi variable in', 'time series back to original s'),\n",
       "   ('C3', 'be with', 'population size of N =')],\n",
       "  [('fraction', 'of series be', 'I')],\n",
       "  [('color-bar values', 'have', 'have normalized by 10 − 4')],\n",
       "  [('I', 'be', 'shown'), ('I', 'be show in', 'b')],\n",
       "  [('color-bar values', 'be in', 'Fig. 10')],\n",
       "  [('stochastic dynamics',\n",
       "    'be in',\n",
       "    'complete system of transformed variables')],\n",
       "  [('speciﬁc type', 'be in', 'limit of inﬁnite population')],\n",
       "  [('stochastic terms', 'be in', 'governing equations')],\n",
       "  [('we', 'compare', 'histograms of probability density')],\n",
       "  [('phase', 'be with', 'one')],\n",
       "  [('This', 'be', 'real')],\n",
       "  [('terms', 'be in', 'normal form coordinate')],\n",
       "  [('low order', 'be in', 'normal form')],\n",
       "  [('normal form coordinate',\n",
       "    'transform method to',\n",
       "    'correctly pro ject noise onto center manifold')],\n",
       "  [('impressive improvement', 'be in', 'time series prediction')],\n",
       "  [('phase',\n",
       "    'be',\n",
       "    'many orders of magnitude longer than typical relaxation time')],\n",
       "  [('phase diﬀerences', 'be', 'where important')],\n",
       "  [('our method', 'be', 'general')],\n",
       "  [('new noise-induced phenomena', 'be in', 'generic stochastic systems'),\n",
       "   ('This',\n",
       "    'identify',\n",
       "    'new noise-induced phenomena in generic stochastic systems')]],\n",
       " 5: [[('based conservation program', 'be in', 'city of Las Vegas'),\n",
       "   ('participation', 'be in', 'incentive')],\n",
       "  [('more ﬂexibility', 'be in', 'modeling assumptions')],\n",
       "  [('We', 'use', 'SEIR'), ('16 selected neighborhoods', 'be in', 'Las Vegas')],\n",
       "  [('Energy', 'of department be', 'DOE'),\n",
       "   ('contract DE-AC05-00OR22725', 'be with', 'US Department of Energy')],\n",
       "  [('1 1 Introduction', 'be in', 'face of increasing climate variability')],\n",
       "  [('Conservation', 'target', 'landscaping'),\n",
       "   ('substantial savings', 'be', 'generated'),\n",
       "   ('Conservation', 'target', 'outdoor landscaping'),\n",
       "   ('changes', 'be in', 'customer awareness')],\n",
       "  [('Southern Nevada Water Authority',\n",
       "    'have',\n",
       "    'Water Smart Landscapes program'),\n",
       "   ('Southern Nevada Water Authority', \"'s program be\", 'WSL'),\n",
       "   ('Southern Nevada Water Authority', \"'s program be\", 'SNWA'),\n",
       "   ('xeric landscapes', 'one of', 'the longest')],\n",
       "  [('diﬀerence', 'be in', 'watering requirements of mesic landscaping')],\n",
       "  [('household participation',\n",
       "    'be in',\n",
       "    \"Las Vegas ' Water Smart Landscapes program\")],\n",
       "  [('individual', 'have on', 'their peers')],\n",
       "  [('WSL program', 'be in', 'Las Vegas')],\n",
       "  [('participating household',\n",
       "    'be in',\n",
       "    'Las Vegas Valley Water District Service area')],\n",
       "  [('reﬂection', 'be with', 'similar adoption times')],\n",
       "  [('grass', 'keep', 'alive')],\n",
       "  [('homes', 'be in', 'neighborhood'),\n",
       "   ('neighborhood', 'be in', 'context of home foreclosures')],\n",
       "  [('Gillingham', 'address', 'correlated unobservables'),\n",
       "   ('Gillingham', 'address', 'unobservables')],\n",
       "  [('linear regression', 'be with', 'ﬁxed eﬀects')],\n",
       "  [('factors', 'be', 'when zero')],\n",
       "  [('It',\n",
       "    'make',\n",
       "    'implicit network structure used in aggregate linear models completely explicit'),\n",
       "   ('complete ﬂexibility', 'be in', 'modeling transmission dynamics')],\n",
       "  [('readers', 'be in', 'ﬁeld')],\n",
       "  [('one home', 'have', 'WSL adoption behavior'),\n",
       "   (\"one home 's WSL adoption behavior\",\n",
       "    'inﬂuencing',\n",
       "    \"their neighbors ' eventual participation probabilities\"),\n",
       "   (\"one home 's WSL adoption behavior\",\n",
       "    'inﬂuencing',\n",
       "    \"their neighbors ' participation probabilities\")],\n",
       "  [('endemic eﬀect',\n",
       "    'be',\n",
       "    'thus ma jor component of individual homes activation probability')],\n",
       "  [('discrete-time epidemic spreading model SEIR', 'be with', 'autoinfection'),\n",
       "   ('participation', 'be in', 'WSL program')],\n",
       "  [('contrast', 'be with', 'standard epidemic models')],\n",
       "  [('susceptible home', 'be in', 'model')],\n",
       "  [('their changed landscape', 'so be visible to', 'their neighbors'),\n",
       "   ('homes', 'require by', 'SNWA'),\n",
       "   ('their landscape', 'so be visible to', 'their neighbors'),\n",
       "   ('Infectious homes', 'require by', 'SNWA')],\n",
       "  [('it', 'inﬂuences', 'its neighbors activation probability')],\n",
       "  [('non-participating house', 'have', 'probability of adoption')],\n",
       "  [('time', 'be with', 'ﬁnite recovery rates on order of few months to year')],\n",
       "  [('adoption environment', 'be in', 'Las Vegas')],\n",
       "  [('Section 4', 'describe', 'analytical methods used')],\n",
       "  [('90 %', 'be with', 'construction of Hoover Dam')],\n",
       "  [('Colorado River water', 'be', 'how shared')],\n",
       "  [('water', 'of m3 be', '300,000 acre-feet')],\n",
       "  [('WSL',\n",
       "    'have',\n",
       "    \"has cornerstone of Las Vegas ' outdoor water conservation strategy\"),\n",
       "   ('Las Vegas', 'reduce', 'outdoor water consumption'),\n",
       "   ('Las Vegas', 'reduce', 'water consumption')],\n",
       "  [('WSL', 'be', 'based'),\n",
       "   ('WSL', 'have', 'has always completely voluntary incentive based program'),\n",
       "   ('WSL', 'be', 'voluntary')],\n",
       "  [('it',\n",
       "    'widespread and important aspect of',\n",
       "    \"Las Vegas ' Citation -2004 drought\"),\n",
       "   ('their lawns', 'desert', 'landscaping based upon size of converted area')],\n",
       "  [('program', 'over', 'history'),\n",
       "   ('SNWA', 'of', 'Citation drought declaration')],\n",
       "  [('program', 'throughout', 'history')],\n",
       "  [('SNWA', 'have', 'eﬀorts marketing')],\n",
       "  [('you', 'push', 'lawnmower')],\n",
       "  [('messages', 'pay', 'advertising'),\n",
       "   ('messages', 'use', 'mass communication')],\n",
       "  [('SNWA', 'have', 'WSL'),\n",
       "   ('targeted mailers',\n",
       "    'be with',\n",
       "    'concurrently semi-annual citywide advertising push remind')],\n",
       "  [('Cumulative WSL conversion area', 'be in', 'acres')],\n",
       "  [('municipal governance', 'be in', 'Las Vegas metro area')],\n",
       "  [('Citation ﬁrst limits', 'be in', 'new construction'),\n",
       "   ('front yards', 'be', 'turf')],\n",
       "  [('Changes', 'apply only to', 'new construction')],\n",
       "  [('Economic Analysis', 'of bureau be', '2018'),\n",
       "   ('13 % Citation Citation', 'bureau of', 'Economic Analysis')],\n",
       "  [('Unemployment', 'now decline back to', '2018'),\n",
       "   ('Unemployment', 'now decline to', '2018'),\n",
       "   ('Labor Statistics', 'of bureau be', '2018')],\n",
       "  [('home values', 'slowly recover', '2018'),\n",
       "   ('home values', 'recover', '2018')],\n",
       "  [('vacant homes', 'of number be', 'problem')],\n",
       "  [('recession', 'inﬂuenced', 'WSL participation')],\n",
       "  [('Las Vegas', None, 'growth')],\n",
       "  [('substantial growth', 'be in', 'residential housing')],\n",
       "  [('them', 'much less attractive', 'candidates'),\n",
       "   ('25 %', 'be in', 'new construction'),\n",
       "   ('them', 'much less attractive', 'candidates for WSL participation')],\n",
       "  [('Data', 'be', 'complete'),\n",
       "   ('single family homes',\n",
       "    'be in',\n",
       "    'Las Vegas Valley Water District Service area')],\n",
       "  [('Clark County Tax Assessors records',\n",
       "    'be in',\n",
       "    'WSL conservation program records'),\n",
       "   ('463,658 single family homes',\n",
       "    'be in',\n",
       "    'Clark County Tax Assessors records')],\n",
       "  [('home', 'have', 'structural characteristics')],\n",
       "  [('we',\n",
       "    'focus on',\n",
       "    'homes with single conversions for sake of clean identiﬁcation'),\n",
       "   ('participating households', 'be in', 'study area'),\n",
       "   ('homes', 'be with', 'single conversions')],\n",
       "  [('signiﬁcant spatial clustering',\n",
       "    'be in',\n",
       "    'physical characteristics of homes')],\n",
       "  [('signiﬁcant diﬀerences', 'be in', 'characteristics of homes')],\n",
       "  [('many more pools', 'be', 'somewhat valuable'),\n",
       "   ('small increases', 'be in', 'indoor size metrics including indoor area'),\n",
       "   ('many more pools', 'be', 'somewhat more valuable')],\n",
       "  [('block groups', 'be with', 'higher homeownership rates')],\n",
       "  [('logistic regression', 'be with', 'Citation'),\n",
       "   ('standard errors', 'be in', 'parentheses')],\n",
       "  [('method', 'underestimate', 'true diﬀerence')],\n",
       "  [('participation', 'be in', 'WSL program')],\n",
       "  [('4 Methods', 'be in', 'section')],\n",
       "  [('dependent variable', 'be', 'where log of odds of WSL participation')],\n",
       "  [('homes', 'be in', 'buﬀer zone'),\n",
       "   ('homes', 'be in', 'buﬀer zone'),\n",
       "   ('boundary problems', 'be in', 'identiﬁcation of peers')],\n",
       "  [('rich spatial detail', 'be in', 'our dataset')],\n",
       "  [('pair', 'be in', 'two diﬀerent ways')],\n",
       "  [('two homes', 'be in', 'given neighborhood')],\n",
       "  [('homes', 'be in', 'WSL population'),\n",
       "   ('multiple conversions', 'remove', 'some'),\n",
       "   ('multiple approved conversions', 'remove', 'some')],\n",
       "  [('homes', 'be with', 'multiple conversions')],\n",
       "  [('0.18 0.32 0.33 0.34 0.30 0.23 0.09 0.60 0.12 0.28 0.26 0.43 0.28 0.05 0.22 0.15 0.23 0.23 Standard errors',\n",
       "    'be in',\n",
       "    'parentheses')],\n",
       "  [('discrete-time epidemic spreading model', 'be with', 'autoinfection'),\n",
       "   ('autoinfection', 'with model be', 'SEIR'),\n",
       "   ('participation', 'be in', 'WSL program')],\n",
       "  [('purple line', 'show line with', 'slope 1')],\n",
       "  [('Red', 'show', 'slope 0.7')],\n",
       "  [('Nodes', 'be in', 'network')],\n",
       "  [('individuals', 'be in', 'ﬁxed network')],\n",
       "  [('allowed transitions', 'be in', 'state')],\n",
       "  [('SEI model', 'be in', 'epidemiology')],\n",
       "  [('µ', 'be with', 'i-th entry')],\n",
       "  [('houses', 'of combination be', 'j')],\n",
       "  [('hazard model', 'be in', 'as Towe'),\n",
       "   ('its initial state x0', 'be in', 'time µt i P =')],\n",
       "  [('non-participating home', 'be infect by', 'neighbor'),\n",
       "   ('corner cases', 'of number be', '1 −')],\n",
       "  [('tra jectory', 'be in', 'time')],\n",
       "  [('it', 'show', 'evidence of peer eﬀects')],\n",
       "  [('α', '=', '0')],\n",
       "  [('peer eﬀect', 'be', 'present')],\n",
       "  [('similar problems', 'be in', 'Bollinger')],\n",
       "  [('vector', 'be in', 'similar way'),\n",
       "   ('mechanistic discrete-time SEIR model', 'be with', 'autoinfection'),\n",
       "   ('similar way', 'be in', 'as regression analysis')],\n",
       "  [('model', 'be in', 'Höhle')],\n",
       "  [('exact program', 'to', 'application date')],\n",
       "  [('exact day',\n",
       "    'might',\n",
       "    'In other words might aﬀected by error of several days')],\n",
       "  [('application', 'be', 'when ﬁled'),\n",
       "   ('delays', 'be in', 'previous two factors')],\n",
       "  [('network', 'have', 'nodes')],\n",
       "  [('percentage', 'be in', 'block')],\n",
       "  [('we', 'assess', 'presence of peer eﬀects')],\n",
       "  [('epidemic model',\n",
       "    'on',\n",
       "    'likelihood integrated with house level covariates')],\n",
       "  [('household level variations', 'thus control for', 'heterogeneity')],\n",
       "  [('estimator', 'be', 'complicated')],\n",
       "  [('price changes',\n",
       "    'be also temporally consistent with',\n",
       "    'other ma jor environmental changes in Las Vegas metro area'),\n",
       "   ('other ma jor environmental changes', 'be in', 'Las Vegas metro area')],\n",
       "  [('relation', 'be with', 'Fisher information matrix Pawitan')],\n",
       "  [('we', 'adopt', 'two diﬀerent strategies'),\n",
       "   ('epidemic contribution', 'be in', 'dynamics of adoption'),\n",
       "   ('we', 'adopt', 'two strategies')],\n",
       "  [('model', 'be with', 'lower Akaike Information Criterion')],\n",
       "  [('diﬀerent contexts', 'be in', 'Shao')],\n",
       "  [('dynamics', 'be', 'strong')],\n",
       "  [('this', 'be', 'strong')],\n",
       "  [('pool',\n",
       "    'presence of be',\n",
       "    'next most signiﬁcant factor associated with WSL participation')],\n",
       "  [('Larger', 'be in', 'lot areas')],\n",
       "  [('Citation house', 'be with', 'pool')],\n",
       "  [('Tab', 'of column be', '4')],\n",
       "  [('3', 'use', 'dummy variables'),\n",
       "   ('3', 'use dummy variable by', 'individual decades of construction')],\n",
       "  [('5.2 Inference Performance', 'be in', 'Tab')],\n",
       "  [('three models', 'one for', 'each of diﬀerent methods selecting'),\n",
       "   ('endemic model', 'be with', 'peers at all')],\n",
       "  [('further parameters', 'be', 'selected'),\n",
       "   ('Two further parameters', 'be', 'selected')],\n",
       "  [('τR =', 'be in', 'months')],\n",
       "  [('results', 'present in', 'Tab')],\n",
       "  [('5.2 reﬂect',\n",
       "    'be with',\n",
       "    'additional values of τR presented in appendix Tab')],\n",
       "  [('τR', '=', '12')],\n",
       "  [('models', 'be with', 'is')],\n",
       "  [('active individual', 'be', 'where always infectious')],\n",
       "  [('Logistic regressions', 'be in', 'WSL')],\n",
       "  [('291737 0.010 Standard errors', 'be in', 'parentheses'),\n",
       "   ('SEIR model', 'be with', 'τR ∈')],\n",
       "  [('neighborhoods', 'be in', 'model response to τd')],\n",
       "  [('signiﬁcant variance', 'be in', 'home density')],\n",
       "  [('neighborhoods', 'be with', 'small lot sizes'),\n",
       "   ('neighborhoods', 'be with', 'large lot sizes')],\n",
       "  [('models', 'be with', 'diﬀerent')],\n",
       "  [('Fig.', 'summarize', 'Tab'), ('Fig. 5', 'summarize', 'Tab')],\n",
       "  [('WSL participation rate', 'be in', 'neighborhood')],\n",
       "  [('Error bars', 'show', 'standard error')],\n",
       "  [('distance', 'for level be', 'neighborhoods n ∈ 1 6 8 9'),\n",
       "   ('distance', 'for level be', '14')],\n",
       "  [('signiﬁcant turf restrictions', 'be in', 'place')],\n",
       "  [('α =', 'be with', 'only 9 WSL'),\n",
       "   ('neighborhood 14', 'α with', 'only 9 WSL')],\n",
       "  [('home', 'be in', 'recovered status from its construction onwards')],\n",
       "  [('other two neighborhoods', 'be with', 'measurable epidemic eﬀect')],\n",
       "  [('many neighborhoods', 'be in', 'Las Vegas')],\n",
       "  [('challenges', 'be with', 'homophily')],\n",
       "  [('model', 'have', 'performance on test period'),\n",
       "   ('We', 'measure', \"model 's performance on test period\")],\n",
       "  [('model', 'be with', 'endemic components')],\n",
       "  [('we', 'should prefer model with', 'only endemic component'),\n",
       "   ('model', 'be with', 'epidemic eﬀect'),\n",
       "   ('prediction', 'have', 'one more parameter')],\n",
       "  [('data', 'of year be', 'Citation Citation'),\n",
       "   ('test period', 'as year be', 'Citation Citation')],\n",
       "  [('root',\n",
       "    'mean',\n",
       "    'square error between average participation rate over 100 Monte Carlo realizations'),\n",
       "   ('total number', 'be in', 'focal area')],\n",
       "  [('WSL rate', 'be', 'eventual WSL participation rate')],\n",
       "  [('neighborhood', 'within', 'results'), ('AIC', 'measure of', 'model ﬁt')],\n",
       "  [('τd Neighborhood Distance Metric AIC', 'α no', 'ST Eα τd WSL rate'),\n",
       "   ('τd Neighborhood Distance Metric AIC', 'α', 'ST Eα τd WSL rate')],\n",
       "  [('prediction', 'have', 'performance')],\n",
       "  [('case', 'be with', 'epidemic')],\n",
       "  [('model', 'be with', 'epidemic component'),\n",
       "   ('it', 'be insigniﬁcant for', 'neighborhoods'),\n",
       "   ('time', 'be', 'standard'),\n",
       "   ('model', 'be with', 'epidemic')],\n",
       "  [('presence', 'be in', 'adoption dynamics')],\n",
       "  [('ˆI sim T 6 Conclusions', 'be in', 'work'),\n",
       "   ('Nevada', 'have', 'water conservation program'),\n",
       "   ('participation', 'be in', 'Vegas')],\n",
       "  [('program', 'have', 'future adoption'),\n",
       "   ('epidemic model', 'predict', \"program 's future adoption\")],\n",
       "  [('recovery period', 'be in', 'inﬂuence'),\n",
       "   (\"household 's WSL conversion\", 'have on', 'neighboring homes'),\n",
       "   ('household', 'have', 'WSL conversion')],\n",
       "  [('peer eﬀect', 'fade in', 'time'), ('contagion', 'be', 'where indeﬁnite')],\n",
       "  [('variations', 'be in', 'autoinfection probabilities'),\n",
       "   ('us', 'capture', 'variations in autoinfection probabilities'),\n",
       "   ('mechanistic epidemic model', 'be with', 'autoinfection')],\n",
       "  [('changes', 'be in', 'WSL participation')],\n",
       "  [('recent methodological developments', 'be in', 'inference')],\n",
       "  [('results', 'mean', 'RMSE')],\n",
       "  [('we', 'consider', 'program adoption'),\n",
       "   ('we', 'consider program adoption as', 'main observable quantity')],\n",
       "  [('CB',\n",
       "    'also receive',\n",
       "    'partial support from ASU/SFI Center for Biosocial Complexity')],\n",
       "  [('CDB', 'be support by', 'John Templeton Foundation')],\n",
       "  [('same quantity', 'be in', 'of discrete time')],\n",
       "  [('hazard rate',\n",
       "    'be in',\n",
       "    'additive-multiplicative hazard model Höhle et al.'),\n",
       "   ('Epidemic model', 'be in', 'discrete time Kalbﬂeisch'),\n",
       "   ('1 − λi', 'be in', 'SEIR model'),\n",
       "   ('t', 'be in', 'purely multiplicative model used'),\n",
       "   ('t', 'rate in', 'additive-multiplicative hazard model Höhle et al.'),\n",
       "   ('vector', 'be in', 'similar ﬂavor as in linear regression')],\n",
       "  [('hazard rate Si', 'with s be', 't'),\n",
       "   ('event', 'of f be', 'ti'),\n",
       "   ('hazard model', 'be in', 'as Scheike'),\n",
       "   ('conditional probability', 'be in', 'discrete time')],\n",
       "  [('Multiplicative hazard model', 'be in', 'as Towe'),\n",
       "   ('diﬀerent recovery times τR', 'be in', 'months')],\n",
       "  [('neighborhood', 'have', 'id'),\n",
       "   ('α', '=', '0'),\n",
       "   ('n', 'be', \"neighborhood 's id\")],\n",
       "  [('homophily-driven diﬀusion', 'be in', 'dynamic networks')],\n",
       "  [('social contagion', 'be in', 'adoption of solar photovoltaic technology')],\n",
       "  [('Whiskey', 'be for', 'Drinking'), ('Water', 'be for', 'Fighting')],\n",
       "  [('St. Louis', 'of bank be', '2018')],\n",
       "  [('Technical Report', 'bank of', 'St. Louis')],\n",
       "  [('Spatio-temporal epidemic modelling',\n",
       "    'use',\n",
       "    'additive-multiplicative intensity models')],\n",
       "  [('Inference', 'be in', 'disease transmission experiments')],\n",
       "  [('Opinion leadership', 'be in', 'new product diﬀusion')],\n",
       "  [('Series A', 'papers of', 'mathematical character')],\n",
       "  [('Social eﬀects', 'be in', 'diﬀusion of solar photovoltaic technology'),\n",
       "   ('diﬀusion', 'be in', 'uk')],\n",
       "  [('vacant homes', 'be in', 'Las Vegas')],\n",
       "  [('Homophily',\n",
       "    'be',\n",
       "    'Generically Confounded in Observational Social Network Studies')],\n",
       "  [('com/rebates', 'access', 'Jan 3rd 2018')],\n",
       "  [('Technical Report', 'bank of', 'St. Louis')],\n",
       "  [('Unemployment Rate', 'be in', 'Las Vegas-Henderson-Paradise')],\n",
       "  [('St. Louis', 'of bank be', 'St. Louis')],\n",
       "  [('Technical Report', 'bank of', 'St. Louis')],\n",
       "  [('model selection', 'be in', 'networks')],\n",
       "  [('Innovation Diﬀusion', 'be in', 'Heterogeneous Populations')]],\n",
       " 6: [[('network', 'be', 'simpliﬁed'), ('network', 'be', 'suitable')],\n",
       "  [('patterns', 'be with', 'realistic amplitudes')],\n",
       "  [('role', 'be in', 'pathogen antigenic variation')],\n",
       "  [('local neighbourhood', 'be in', 'as lattices')],\n",
       "  [('existence',\n",
       "    'be in',\n",
       "    'many interaction networks including networks of social contacts')],\n",
       "  [('attention',\n",
       "    'have',\n",
       "    'Recently has focussed on impact of network topology on dynamics of processes running on it with emphasis on spreading of infectious diseases')],\n",
       "  [('model', '2 on', 'same network')],\n",
       "  [('We', 'determine', 'outcome of invasion in multiple strain infections'),\n",
       "   ('invasion', 'be in', 'multiple strain infections'),\n",
       "   ('proposed SIR model', 'be with', 'reinfection')],\n",
       "  [('ﬁeld model', 'be in', 'closed population of N individuals')],\n",
       "  [('square lattice', 'be with', 'N sites')],\n",
       "  [('increase', 'be in', 'spatial correlations')],\n",
       "  [('SEIR model NumericCitation', 'be with', 'epidemiological parameters'),\n",
       "   ('p', '=', '0.2')],\n",
       "  [('infective time series',\n",
       "    'exhibit incidence oscillation with',\n",
       "    'average period of two years')],\n",
       "  [('incidence oscillations',\n",
       "    'be in',\n",
       "    'real data for similar population sizes NumericCitation')],\n",
       "  [('line', 'be with', 'real data'),\n",
       "   ('peaks', 'increase signiﬁcantly in', 'line with real data')],\n",
       "  [('NumericCitation', 'eﬀects of', 'spatial correlations')],\n",
       "  [('p', '=', '0.2'), ('p', '=', '0.1')],\n",
       "  [('single viral strain',\n",
       "    'represent',\n",
       "    'cluster of co-circulating antigenically close strains'),\n",
       "   ('SIR model', 'be with', 'transmissibility β for ﬁrst infection')],\n",
       "  [('SIR model', 'be with', 'partial immunity'),\n",
       "   ('abrupt change', 'be in', 'levels of infection')],\n",
       "  [('model', 'be with', 'zero birth rate NumericCitation'),\n",
       "   ('models', 'be with', 'small birth rates')],\n",
       "  [('δ = σ × − σ',\n",
       "    'be measure of',\n",
       "    'diﬀerence between two competing strains')],\n",
       "  [('model', 'be with', 'two strains'),\n",
       "   ('reinfection threshold',\n",
       "    'be',\n",
       "    'at level of mean ﬁeld approximation boundary between two diﬀerent regimes')],\n",
       "  [('we', 'investigate', 'its eﬀect')],\n",
       "  [('system',\n",
       "    'introduce',\n",
       "    'small fraction of individuals infected by invading strain')],\n",
       "  [('p', '=', '1')],\n",
       "  [('screening', 'be in', 'our model'),\n",
       "   ('screening', 'be with', 'low rate of supply of naive susceptibles')],\n",
       "  [('network', 'of', 'detailed topology')],\n",
       "  [('reduction', 'be in', 'systems with reinfection'),\n",
       "   ('systems', 'be with', 'reinfection')],\n",
       "  [('Science', 'for foundation be', 'FCT')],\n",
       "  [('square lattice = L2', 'be with', 'periodic boundary conditions'),\n",
       "   ('We',\n",
       "    'consider',\n",
       "    'square lattice of size N = L2 with periodic boundary conditions')],\n",
       "  [('local connections', 'be with', 'k neighbouring sites'),\n",
       "   ('local connections', 'be with', 'k ='),\n",
       "   ('recovery', 'be deterministic after', 'τi time steps'),\n",
       "   ('local connections', 'be with', 'small world probability')],\n",
       "  [('recovery time', 'set', 'time scale')],\n",
       "  [('NumericCitation', 'be in', 'SEIR model deterministic recovery')],\n",
       "  [('S. Gupta', 'proceedings of', 'National Academy')],\n",
       "  [('work', 'be in', 'progress'),\n",
       "   ('NumericCitation T. Nedelea', 'work in', 'progress')],\n",
       "  [('transmission rate β', '= at_time', '2.4 day − 1')],\n",
       "  [('network', 'be with', 'p =')],\n",
       "  [('transmission rate β', '= at_time', '1.5 day − 1'),\n",
       "   ('network', 'be with', 'p ='),\n",
       "   ('network', 'be with', 'p =')],\n",
       "  [('transmissibility',\n",
       "    'preserve',\n",
       "    'average period of homogeneously mixed population')],\n",
       "  [('increase', 'be in', \"incidence peak 's amplitude\"),\n",
       "   ('incidence peak', 'in', 'amplitude')],\n",
       "  [('population birth rate', '− for', 'transmissibility'),\n",
       "   ('population birth rate', '0.27 for', 'recovery rate of diseases'),\n",
       "   ('population birth rate', '0.27 for', 'reinfection parameter')],\n",
       "  [('p', '=', '0.1')]],\n",
       " 7: [[('Citation Ebola virus outbreak', 'be in', 'West Africa')],\n",
       "  [('spread', 'be in', 'affected countries'),\n",
       "   ('it', 'better understand', 'spread of infection in affected countries')],\n",
       "  [('ted data', 'be in', 'Guinea')],\n",
       "  [('Guinea', 'for interval be', 'CI'),\n",
       "   ('Guinea', 'ci for', 'Liberia'),\n",
       "   ('Guinea', 'ci for', 'Sierra Leone')],\n",
       "  [('control effor ts', 'be in', 'Liberia')],\n",
       "  [('Citation Ebola virus outbreak', 'be in', 'West Africa'),\n",
       "   ('Social', 'of institute be', 'Switzerland Corresponding author')],\n",
       "  [('Re', 'drop below', 'unity')],\n",
       "  [('two outbreaks', 'be in', 'Congo')],\n",
       "  [('4 It', 'know', 'reproduction numbers of current EBOV outbreak'),\n",
       "   ('it', 'be', 'how affected by public health interventions'),\n",
       "   ('it', 'be', 'how affected')],\n",
       "  [('effects', 'be in', 'country')],\n",
       "  [('reported cases', 'be in', 'Guinea')],\n",
       "  [('case fatality rate', 'be give by', 'f')],\n",
       "  [('transmission rate', 'be in', 'absence of control interventions')],\n",
       "  [('t1/2 =', 'number of', 'infected cases')],\n",
       "  [('ODEs', 'use function ode from', 'package deSolve')],\n",
       "  [('reproduction number', 'be in', 'West Africa')],\n",
       "  [('Citation EBOV outbreaks', 'be in', 'Guinea')],\n",
       "  [('lines', 'represent', 'best-ﬁt model to data')],\n",
       "  [('primary case', 'be in', 'Guinea')],\n",
       "  [('ﬁt', 'treat k as', 'free parameter')],\n",
       "  [('EBOV outbreak', 'be', 'ongoing')],\n",
       "  [('I', 'hence make', 'three assumptions'),\n",
       "   ('I', 'hence make', 'three simplifying assumptions')],\n",
       "  [('outbreak', 'be in', 'Guinea'), ('Third', 'date of', 'appearance')],\n",
       "  [('reported data', 'be in', 'Guinea')],\n",
       "  [('reproduction number', 'be in', 'West Africa')],\n",
       "  [('EBOV', 'be in', 'Guinea')],\n",
       "  [('transmission rate',\n",
       "    'decay exponentially due to',\n",
       "    'introduction of control measures')],\n",
       "  [('Citation outbreak', 'be in', 'West Africa')],\n",
       "  [('Uganda', 'in virus be', 'SUDV'),\n",
       "   ('Sudan virus', 'be in', 'Uganda'),\n",
       "   ('EBOV outbreak', 'be in', 'Congo')],\n",
       "  [('2,3,4 basic reproduction number', 'be in', 'Sierra Leone')],\n",
       "  [('differences', 'be in', 'population structure')],\n",
       "  [('subsequent outbreak', 'be in', 'Sierra Leone'),\n",
       "   ('agreement',\n",
       "    'be with',\n",
       "    'time of most recent common ancestor of genome analysis of EBOV outbreak')],\n",
       "  [('Re', 'be in', 'parts such urban areas'),\n",
       "   ('varying effects', 'be in', 'different areas')],\n",
       "  [('control measures', 'be', 'sufﬁ Citation respectively'),\n",
       "   ('control measures', 'be', 'Citation respectively')]],\n",
       " 8: [[('tree network', 'be with', 'single infection source'),\n",
       "   ('source', 'be with', 'mild technical assumptions'),\n",
       "   ('likely infection path',\n",
       "    'be in',\n",
       "    'tree network with single infection source')],\n",
       "  [('SIS model', 'be with', 'homogeneous infection')],\n",
       "  [('set estimator', 'be in', 'tree network for SI model'),\n",
       "   ('this', 'be', 'optimal')],\n",
       "  [('W. Luo', 'be with', 'Micron Semiconductor Asia'),\n",
       "   ('W. Luo', 'be with', 'currently Micron Semiconductor Asia')],\n",
       "  [('rumor', 'be in', 'social network'),\n",
       "   ('contagious disease spreading', 'be in', 'community')],\n",
       "  [('Various models', 'describe', 'spreading process of infection')],\n",
       "  [],\n",
       "  [('it', 'gain immunity from', 'further infections'),\n",
       "   ('infection', 'be with', 'given probability')],\n",
       "  [('single infection source', 'be in', 'SI model')],\n",
       "  [('single source estimation',\n",
       "    'be with',\n",
       "    'priori knowledge of set of suspect nodes')],\n",
       "  [('We', 'identify', 'infection sources in NumericCitation'),\n",
       "   ('multiple infection sources', 'be in', 'NumericCitation'),\n",
       "   ('We', 'identify', 'multiple infection sources in NumericCitation'),\n",
       "   ('source', 'be with', 'limited observations of set of infected nodes')],\n",
       "  [('Jordan center1',\n",
       "    'give',\n",
       "    'optimal estimator associated with most likely infection path'),\n",
       "   ('single infection source', 'be in', 'SIR model')],\n",
       "  [('node', 'be in', 'network with smallest maximum distance'),\n",
       "   ('network', 'be with', 'smallest maximum distance'),\n",
       "   ('infected node', 'become', '1The Jordan center of infected node set')],\n",
       "  [('he', 'exhibit', 'infection symptoms')],\n",
       "  [('he', 'later become', 'reinfected with same disease')],\n",
       "  [('rumor spreading', 'be in', 'online social network')],\n",
       "  [('individual', 'be in', 'network')],\n",
       "  [('recovery rates', 'be', 'also known')],\n",
       "  [('knowledge', 'obtain in', 'practice')],\n",
       "  [('its epidemiology', 'be', 'determined')],\n",
       "  [('we', 'ﬁnd', 'optimal infection source estimator')],\n",
       "  [('similar results',\n",
       "    'be in',\n",
       "    'current literature restricted to tree networks')],\n",
       "  [('work', 'be', 'small')],\n",
       "  [('center', 'be', 'optimal'), ('infection spreading', 'for 4 be', 'i')],\n",
       "  [('optimal infection source estimator', 'be in', 'ﬁnite regular tree'),\n",
       "   ('center', 'be', 'optimal')],\n",
       "  [('1 sources', 'be in', 'estimator'),\n",
       "   ('estimator', 'be in', 'ﬁnite tree network'),\n",
       "   ('estimator', 'be in', 'accordance to SI model'),\n",
       "   ('set', 'be', 'optimal')],\n",
       "  [('multiple infection sources', 'be in', 'SI model')],\n",
       "  [('NumericCitation', 'be in', 'terms of average error distance'),\n",
       "   ('our estimator',\n",
       "    'outperform',\n",
       "    'that in NumericCitation in terms of average error distance')],\n",
       "  [('We', 'part of', 'Facebook network')],\n",
       "  [('node', 'of', 'infection')],\n",
       "  [('our result', 'be in', 'item'), ('center', 'be', 'universal')],\n",
       "  [('Jordan center',\n",
       "    'be also optimal for',\n",
       "    'SIS model in regular tree networks'),\n",
       "   ('SIS model', 'be in', 'regular tree networks')],\n",
       "  [('differences', 'be in', 'assumptions')],\n",
       "  [('better source detection rates', 'be in', 'numerical experiments')],\n",
       "  [('center', 'be', 'optimal')],\n",
       "  [('we', 'propose', 'heuristic algorithms')],\n",
       "  [('sim ulation', 'verify', 'effectiveness of proposed estimators')],\n",
       "  [('PROBLEM FORMULAT ION', 'be in', 'section')],\n",
       "  [('We', 'also describe', 'most likely infection path criterion')],\n",
       "  [('we', 'extend to', 'multiple source nodes for SI model described below')],\n",
       "  [('nodes', 'be in', 'graph G'),\n",
       "   ('Markov process', 'be with', '6 probability measure P'),\n",
       "   ('time', 'be divide into', 'discrete slots'),\n",
       "   ('time', 'be', 'divided')],\n",
       "  [('particular point', 'be in', 'time')],\n",
       "  [('SIR model', 'be in', 'SIR model')],\n",
       "  [('infected node', 'be in', 'time'),\n",
       "   ('infected node', 'be in', 'state i'),\n",
       "   ('only difference', 'be with', 'SI model')],\n",
       "  [('possible nodes states', 'be in', 'SIRI model')],\n",
       "  [('future time slot', 'be with', 'positive probability'),\n",
       "   ('recovered node', 'be in', 'state r')],\n",
       "  [('SIS model', 'be in', 'SIS model')],\n",
       "  [('they', 'satisfy', 'following Assumptions 1 4')],\n",
       "  [('proofs', 'be with', 'slight modiﬁcations')],\n",
       "  [('node', 'be in', 'network')],\n",
       "  [('estimator', 'be with', 'knowledge of infection rates')],\n",
       "  [('example', 'be in', 'Fig. 3'), ('Assumption', 'be', 'necessary')],\n",
       "  [('setup', 'be in', 'NumericCitation'), ('setup', 'be', 'special')],\n",
       "  [('α', '=', 'β')],\n",
       "  [('nodes', 'be', 'signiﬁcant'), ('we', 'observe', 'state of network')],\n",
       "  [('We', 'call', 'such networks heterogeneous')],\n",
       "  [('We', 'call', 'such networks homogeneous')],\n",
       "  [('we', 'ﬁnd', 'optimal infection sources in paper'),\n",
       "   ('B. Most Likely Infection Path Sources Estimator', 'be in', 'subsection'),\n",
       "   ('optimal infection sources', 'be in', 'paper')],\n",
       "  [('de ﬁnitions', 'be', 'repeated here'),\n",
       "   ('de ﬁnitions', 'be repeat here for', 'completeness')],\n",
       "  [('X', 'be', 'random')],\n",
       "  [('collection', 'be in', 'V from time 1 to t')],\n",
       "  [('Ts', 'be set of', 'corresponding feasible elapsed times')],\n",
       "  [('single infection source', 'be in', 'SI model')],\n",
       "  [('exponential distribution', 'be with', 'rate 1')],\n",
       "  [('time', 'be discretize into', 'unit intervals'),\n",
       "   ('NumericCitation', 'in', 'model'),\n",
       "   ('1 −', 'be in', 'time'),\n",
       "   ('time', 'be', 'where discretized into unit intervals'),\n",
       "   ('source', 'be in', \"NumericCitation 's model\")],\n",
       "  [('size', 'increase', 'exponentially fast with number of nodes')],\n",
       "  [('we', 'consider network with', 'single source in Section III'),\n",
       "   ('single source', 'be in', 'Section III')],\n",
       "  [('tree network', 'be with', 'multiple sources for SI model')],\n",
       "  [('Notations', 'be in', 'subsection')],\n",
       "  [('node', 'be with', 'incoming edge from u'),\n",
       "   ('nodes', 'be with', 'outgoing edges to u'),\n",
       "   ('child nodes', 'be in', 'A')],\n",
       "  [('states', 'be in', 'J from time slots')],\n",
       "  [('we', 'avoid', 'expressions'), ('we', 'avoid', 'cluttered expressions')],\n",
       "  [('number', 'be in', 'A'),\n",
       "   ('number', 'be in', 'A'),\n",
       "   ('shortest path', 'be in', 'graph G'),\n",
       "   ('subtree', 'be with', 'ﬁrst link of path from u v in removed d')],\n",
       "  [('conditional probability Ps', 'in time be', 'X t'),\n",
       "   ('speciﬁc time', 'be in', 'conditional probability Ps')],\n",
       "  [('s', '∈', 'V')],\n",
       "  [('its infection range', 'be', 'node in G with minimum infection range')],\n",
       "  [('Tu', 'be', 'uninfected')],\n",
       "  [('It', 'search over', 'elapsed times'),\n",
       "   ('It', 'be', 'obviously computationally inefﬁcient')],\n",
       "  [('SI model', 'be in', 'network satisfying')],\n",
       "  [('result', 'be', 'intuitive')],\n",
       "  [('value', 'be in', 'product')],\n",
       "  [('less terms', 'be in', 'product')],\n",
       "  [('problem', 'be in', '6')],\n",
       "  [('we', 'be identiﬁed after', 'most likely elapsed time'),\n",
       "   ('we', 'ﬁnd', 'source node associated with most likely infection path')],\n",
       "  [('B. Source Associated',\n",
       "    'be with',\n",
       "    'Most Likely Infection Path In subsection'),\n",
       "   ('Most Likely Infection Path', 'be in', 'subsection')],\n",
       "  [('∈ V', 'be', 'infection source')],\n",
       "  [('two infection paths', 'of pv be', 'X tv')],\n",
       "  [('we', 'establish', 'neighboring node relationship')],\n",
       "  [('Proposition 1 Proposition 2', 'be in', 'NumericCitation'),\n",
       "   ('We', 'note', 'that')],\n",
       "  [('Theorem 1', 'be in', 'NumericCitation')],\n",
       "  [('center', 'be', 'optimal')],\n",
       "  [('er', 'be', 'optimal')],\n",
       "  [('This', 'be', 'surprising')],\n",
       "  [('Assumption', 'be', 'necessary')],\n",
       "  [('n ode Uninfe', 'cted', 'n od')],\n",
       "  [('example', 'be', 'necessary')],\n",
       "  [('SI model', 'suppose', 'infection process')],\n",
       "  [('infection path X 2', 'be with', 'elapsed time 2')],\n",
       "  [('infection path Y 3', 'be with', 'elapsed time 3')],\n",
       "  [('MULT IPLE SOURCE S E ST IMAT ION', 'be in', 'section'),\n",
       "   ('FECT ION S PREAD ING', 'be in', 'TREE S')],\n",
       "  [('Jordan center', 'be in', 'Section III'),\n",
       "   ('optimal single infection source estimator',\n",
       "    'be',\n",
       "    'Jordan center of Vi in Section III')],\n",
       "  [('S', 'be in', 'de ﬁnitions')],\n",
       "  [('set', 'be in', 'G with minimum infection range'),\n",
       "   ('G', 'be with', 'minimum infection range')],\n",
       "  [('sources', 'be', '='),\n",
       "   ('G', 'be', '='),\n",
       "   ('sources', 'be', '='),\n",
       "   ('G', 'be', '='),\n",
       "   ('S =', 'be in', 'ﬁnite tree')],\n",
       "  [('time τ', 'be in', 'X t')],\n",
       "  [('u', 'belong', 'to'),\n",
       "   ('component Ai', 'in edge be', 'u'),\n",
       "   ('edge', 'be in', 'component Ai')],\n",
       "  [('infection path X t', 'give supernode be', 'S')],\n",
       "  [('> τ1', 'be in', 'X t')],\n",
       "  [('nodes', 'be in', 'B')],\n",
       "  [('A = A1 S A2', 'into supernode be', 'S')],\n",
       "  [('likely infection path', 'be in', 'G')],\n",
       "  [('Theorem 1', 'discuss in', 'Section III'),\n",
       "   ('infected node set', 'center of be', 'optimal single source estimator'),\n",
       "   ('center', 'be', 'single'),\n",
       "   ('Theorem', 'discuss in', 'Section III'),\n",
       "   ('center', 'be', 'optimal')],\n",
       "  [('k-Jordan center set', 'be', 'solution')],\n",
       "  [('Vi', 'be', 'non-empty')],\n",
       "  [('k-Jordan center set', 'be in', 'G'),\n",
       "   ('Jordan center', 'be in', 'corresponding super node graph ˜G')],\n",
       "  [('infection', 'be in', 'SI model'), ('set', 'be', 'optimal')],\n",
       "  [('infection', 'be in', 'SI model'),\n",
       "   ('Theorem 2', 'be consistent with', 'Theorem 1 for infection in SI model')],\n",
       "  [('we', 'verify', 'robustness of proposed estimators')],\n",
       "  [('underlying network G', 'be', 'where general graph'),\n",
       "   ('G', 'be', 'general'),\n",
       "   ('V. SOURCE E ST IMAT ION', 'be in', 'section')],\n",
       "  [('robustness', 'be in', 'tree networks'),\n",
       "   ('we',\n",
       "    'inspire by',\n",
       "    'robustness of Jordan center estimators in tree networks')],\n",
       "  [('source', 'be', 'general'), ('network', 'be', 'general')],\n",
       "  [('Single Jordan Center estimation algorithm',\n",
       "    '19 with',\n",
       "    'computational complexity of O')],\n",
       "  [('k', 'be', 'When greater than 1'), ('k', 'be greater than', '1')],\n",
       "  [('Multiple Jordan Center', 'set', 'MJC'),\n",
       "   ('sources', 'be', 'general'),\n",
       "   ('network', 'be', 'general')],\n",
       "  [('node', 'be in', 'ˆS l − 1')],\n",
       "  [('nearest nodes', 'be in', 'ˆS l − 1')],\n",
       "  [('nearest node', 'be in', 'ˆS l − 1')],\n",
       "  [('sl −', 'be in', 'Voronoi')],\n",
       "  [('sl − 1', 'η for', 'predetermined small positive value η')],\n",
       "  [('partition step', 'be in', 'iteration')],\n",
       "  [('re-optimization step', 'be in', 'iteration')],\n",
       "  [('infection range', 'be non-increasing over', 'iterations of MJC')],\n",
       "  [('S IMULAT ION RE SULT S', 'be in', 'section')],\n",
       "  [('SI models', 'be in', 'homogeneous')],\n",
       "  [('ﬁrst four de ﬁnitions', 'be repeat here for', 'convenience of reader')],\n",
       "  [('∈ G Xi', 'number of', 'shortest paths')],\n",
       "  [('SI model', 'be with', 'single source'),\n",
       "   ('DisC',\n",
       "    'be',\n",
       "    'ML estimator for regular trees under SI model with single source')],\n",
       "  [('node', 'be with', 'maximum eigenvector centrality'),\n",
       "   ('node', 'be in', 'H')],\n",
       "  [('d', 'be', 'damping')],\n",
       "  [('node', 'be with', 'maximum pagerank centrality'), ('node', 'be in', 'H')],\n",
       "  [('node', 'be in', 'H')],\n",
       "  [('NumericCitation', 'part of', 'Facebook network with 4039 nodes'),\n",
       "   ('Facebook network', 'be with', '4039 nodes')],\n",
       "  [('we',\n",
       "    'demonstrate',\n",
       "    'impact of spreading parameters on performance of proposed estimator')],\n",
       "  [('SI models', 'be in', 'homogeneous networks')],\n",
       "  [('infection', 'use', 'above parameters')],\n",
       "  [('infection source', 'compare result with', 'benchmarks')],\n",
       "  [('error distance', 'be show in', 'Fig. 5')],\n",
       "  [('Average error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdence int erval'),\n",
       "   ('various networks', 'be in', 'homogeneous networks')],\n",
       "  [('pi', '=', '1')],\n",
       "  [('pi', 'be randomly choose from', '0.5 1'),\n",
       "   ('SIR models', 'be in', 'homogeneous networks'),\n",
       "   ('pr', 'be', 'set'),\n",
       "   ('pi', 'be', 'randomly chosen')],\n",
       "  [('We', 'compare', 'performances in Fig. 6'),\n",
       "   ('performances', 'be in', 'Fig. 6')],\n",
       "  [('Average error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdence int erval'),\n",
       "   ('various networks', 'be in', 'homogeneous networks')],\n",
       "  [('pr', '=', '0')],\n",
       "  [('ps', 'be randomly choose from', 'pi'),\n",
       "   ('ps', 'be', 'randomly chosen'),\n",
       "   ('SIS model', 'be in', 'homogeneous networks')],\n",
       "  [('we', 'randomly choose', 'v'),\n",
       "   ('SI', 'be in', 'heterogeneous networks'),\n",
       "   ('pi', 'pr for', 'node v'),\n",
       "   ('node', 'for pr be', 'v'),\n",
       "   ('pi', 'pr from', 'NumericCitation'),\n",
       "   ('NumericCitation', 'from pr be', 'v')],\n",
       "  [('Average error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdence int erval'),\n",
       "   ('various networks', 'be in', 'homogeneous networks under SIS model')],\n",
       "  [('performances', 'be in', 'Fig. 8'),\n",
       "   ('We', 'compare', 'performances in Fig. 8')],\n",
       "  [('B. Multiple Infection Sources', 'be in', 'subsection')],\n",
       "  [('Average error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdence int erval'),\n",
       "   ('SI models', 'be in', 'heterogeneous networks')],\n",
       "  [('We', 'ﬁnding k node in', 'H with largest degree centralities'),\n",
       "   ('H', 'be with', 'largest degree centralities')],\n",
       "  [('pi', 'pr for', 'node v'),\n",
       "   ('node', 'for pr be', 'v'),\n",
       "   ('pi', 'pr from', 'NumericCitation'),\n",
       "   ('NumericCitation', 'from pr be', 'v')],\n",
       "  [('infection', 'use', 'above mentioned spreading model')],\n",
       "  [('infection sources', 'compare result with', 'benchmarks')],\n",
       "  [('Average mean error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdenc interval for various networks under different infection spreading models')],\n",
       "  [('center', 'be', 'universal')],\n",
       "  [('one infection sources', 'be in', 'tree network'),\n",
       "   ('set', 'be', 'optimal')],\n",
       "  [('Average mean error distances',\n",
       "    'be with',\n",
       "    'error bars of 95 % conﬁdenc interval for various networks under different infection spreading models')],\n",
       "  [('Pv', 'since δpv be', '8'),\n",
       "   ('Pv', 'since δpv be', '˜X t'),\n",
       "   ('it', 'show', 'claim'),\n",
       "   ('it', 'show', 'a')],\n",
       "  [('Tu', 'be', 'uninfected')],\n",
       "  [('Tu', 'be', 'uninfected')],\n",
       "  [('˜X t', '=', 'i')],\n",
       "  [('infection source', 'be with', 'infection starting at time 1')],\n",
       "  [('Tw', 'be', 'uninfected')],\n",
       "  [('Inductive step', 'hold for', '¯ d')],\n",
       "  [('infection source', 'be with', 'infection starting at n − 1')],\n",
       "  [('claim', 'be', 'now proved')],\n",
       "  [('we', 'construct', '˜X t')],\n",
       "  [('SI model', 'for claim be', 'a')],\n",
       "  [('SIRI model', 'be in', 'Lemma A. 1')],\n",
       "  [('we', 'apply', 'Lemma A. 1')],\n",
       "  [('SIRI model', 'for claim be', 'a')],\n",
       "  [('we', 'avoid', 'repetition'),\n",
       "   ('SIR model', 'for claim be', 'a'),\n",
       "   ('we', 'omit here', 'it')],\n",
       "  [('node',\n",
       "    'be',\n",
       "    'then reinfected again for multiple times by observation time')],\n",
       "  [('proof', 'be provide in', 'Appendix G.'), ('proof', 'be', 'provided')],\n",
       "  [('Vi', 'be', 'observed')],\n",
       "  [('u', 'be in', 'X t'), ('u', 'of tint be', 'u')],\n",
       "  [('ach node', 'be in', 'Hv')],\n",
       "  [('nodes', 'resist', 'infection sprea ding')],\n",
       "  [('u', 'be in', 'Y t +1')],\n",
       "  [('SIS model', 'for claim be', 'a')],\n",
       "  [('l ∈ Tv', 'for a be', 'b')],\n",
       "  [('it', 'prove', 'Proposition 2'),\n",
       "   ('it', 'construct infection path ˜x tv with', 'source node v')],\n",
       "  [('infection path Y tu', 'be with', 'source node u')],\n",
       "  [('We', 'ﬁrst', 'show')],\n",
       "  [('infection source', 'be with', 'infection starting at time 1')],\n",
       "  [('we', 'obtain', '1 − α')],\n",
       "  [('we', 'apply twice', 'Lemma A. 1')],\n",
       "  [('we', 'have', '˜X tv')],\n",
       "  [('proof', 'be in', 'SI models')],\n",
       "  [('We', 'next consider', 'SIS model')],\n",
       "  [('we', 'have y tu follow', 'Lemma A. 2')],\n",
       "  [('proof', 'be in', 'SIS model')],\n",
       "  [('ﬁrst link', 'be in', 'path')],\n",
       "  [('mathematical induction', 'by show be', '16')],\n",
       "  [('Basis step', 'hold for', 'd =')],\n",
       "  [('Basis step', 'hold for', 'd =')],\n",
       "  [('neighboring node', 'be in', 'ρ')],\n",
       "  [('pair', 'be with', 'infection starting at time 1')],\n",
       "  [('pair', 'be with', 'infection starting at time 1')],\n",
       "  [('minimum infection range', 'be in', '˜G')],\n",
       "  [('super node Supernode', 'be in', '˜G'), ('˜G', 'in supernode be', 'S ′')],\n",
       "  [('S', 'be', 'k-Jordan')],\n",
       "  [('minimum infection range', 'be in', '˜G')],\n",
       "  [('it', 'show', 'that')],\n",
       "  [('v', 'of tint be', 'v')],\n",
       "  [('we', 'follow', 'Proposition 3'), ('we', 'follow', 'b')],\n",
       "  [('nearest node', 'be in', 'ˆS l − 1'), ('j', 'be', 'nearest')],\n",
       "  [('18', 'hold for', 'three possible cases')],\n",
       "  [('Tu', 'be', 'uninfected')],\n",
       "  [('replacements', 'be', 'uninfected')],\n",
       "  [('four cases', 'for pr be', 'v'), ('four cases', 'for pr be', '1 − pi')],\n",
       "  [('t', 'consider', 'the following six possible cases for Y t +1')],\n",
       "  [('infection source', 'be with', 'infection starting at time 1')],\n",
       "  [('Tu', 'be', 'infected')],\n",
       "  [('infection source', 'be with', 'infection starting at time 1')],\n",
       "  [('Tw', 'be', 'uninfected')],\n",
       "  [('infection source', 'be with', 'infection starting at time 1')],\n",
       "  [('we', 'ﬁrst show give', 'v')],\n",
       "  [('d', 'be in', 'time')],\n",
       "  [('u', 'be in', 'X t'), ('u', 'of tint be', 'u')],\n",
       "  [('u', 'be in', '˜X t')],\n",
       "  [('It', 'show', 'that')],\n",
       "  [('Basis step', 'hold for', '¯ d')],\n",
       "  [('time slot tint', 'be in', 'X t'), ('X t', 'in tint be', 'u')],\n",
       "  [('˜X t', '=', 'i')],\n",
       "  [('one node', 'be in', 'Tu')],\n",
       "  [('˜X t', '=', 'i')],\n",
       "  [('worst case', 'be in', '24'),\n",
       "   ('X t', 's follow', 'y ∈ Tz'),\n",
       "   ('y ∈ Tz', 'follow s be', 'τ − 1')],\n",
       "  [('Inductive step', 'hold for', '¯ d')],\n",
       "  [('u', 'n. by', 'induction assumption')],\n",
       "  [('similar arguments', 'be in', 'as t')],\n",
       "  [('its', 'application', 'vol')],\n",
       "  [('Role', 'be in', 'epidemic model o f mobile individuals on networks')],\n",
       "  [('Mixed SI epidemic dynamics', 'be with', 'general degree distributions'),\n",
       "   ('Mixed SI epidemic dynamics', 'be in', 'random graph s')],\n",
       "  [('Modeling info rmation dissemination',\n",
       "    'be in',\n",
       "    'generalized social networks')],\n",
       "  [('NumericCitation N. Bailey', 'theory of', 'Infectious Diseases')],\n",
       "  [('Rumors', 'be in', 'network')],\n",
       "  [('NumericCitation Z. Wang', 'be in', 'Proc'),\n",
       "   ('Rumor source det ection', 'be with', 'multiple observations')],\n",
       "  [('source', 'be with', 'limited observations')],\n",
       "  [('sources', 'be in', 'larg e networks')],\n",
       "  [('Information source detection', 'be in', 'SIR model'),\n",
       "   ('based approach', 'be in', 'Proc')],\n",
       "  [('robust information source estimator', 'be with', 'sparse observations')],\n",
       "  [('emergence', 'be in', 'HSV-2'),\n",
       "   ('p', 'revent', 'emergence of antiviral drug resistance in HSV-2')],\n",
       "  [('Modeling relapse', 'be in', 'infec tious diseases')],\n",
       "  [('S IRI model', 'be with', 'nonlinear incidence of infection')],\n",
       "  [('Lecture Notes', 'be in', 'Biomathematics')],\n",
       "  [('NumericCitation J. McAuley', 'be in', 'Proc'),\n",
       "   ('socia l circles', 'be in', 'ego networks')]],\n",
       " 9: [[('Information Source Detection', 'be in', 'SIR Model')],\n",
       "  [('estimator', 'be in', 'general graphs')],\n",
       "  [('estimator', 'be', 'constant'),\n",
       "   ('constant distance',\n",
       "    'be with',\n",
       "    'high probability independent of number of infected nodes')],\n",
       "  [('show',\n",
       "    'be',\n",
       "    'for tree networks closer to actual source than one identiﬁed by closeness centrality heuristic')],\n",
       "  [('rumor source', 'be in', 'online social networks')],\n",
       "  [('Susceptible-Infected-Recovered model',\n",
       "    'model of',\n",
       "    'epidemics NumericCitation')],\n",
       "  [('network', 'be', 'undirected'), ('undirected graph', 'be in', 'network')],\n",
       "  [('Nodes', 'be in', 'state S')],\n",
       "  [('information', 'spread in', 'network')],\n",
       "  [('nodes', 'be', 'indistinguishable'),\n",
       "   ('recovered nodes', 'be', 'indistinguishable')],\n",
       "  [('various reasons', 'be in', 'practice'),\n",
       "   ('it', 'be', 'very important'),\n",
       "   ('it', 'take recovery into', 'consideration')],\n",
       "  [('computer', 'may recover from', 'virus attack after anti-virus software'),\n",
       "   ('user', 'may delete rumor from', 'her/his blog')],\n",
       "  [('SIR model', 'be in', 'paper'),\n",
       "   ('problem', 'signiﬁcantly more challenging than', 'that in SI model'),\n",
       "   ('information source detection problem', 'be in', 'scenarios')],\n",
       "  [('information source detection problem',\n",
       "    'can',\n",
       "    'Similar to SI model can formalized as MLE problem')],\n",
       "  [('we', 'solve', 'MLE problem'),\n",
       "   ('snapshot', 'be', 'obtained'),\n",
       "   ('we', 'consider', 'possible infection sample paths')],\n",
       "  [('t', 'be', 'known'), ('t', 'be', 'even when known')],\n",
       "  [('we', 'overcome', 'difﬁculty')],\n",
       "  [('We', 'call', 'problem optimal sample path detection problem')],\n",
       "  [('optimal sample path', 'be in', 'trees'),\n",
       "   ('We',\n",
       "    'investigate',\n",
       "    'structure properties of optimal sample path in trees')],\n",
       "  [('node', 'be with', 'minimum infection eccentricity'),\n",
       "   ('optimal sample path',\n",
       "    'node of be',\n",
       "    'node with minimum infection eccentricity'),\n",
       "   ('node',\n",
       "    'eccentricity of be',\n",
       "    'maximum distance from node to infected nodes')],\n",
       "  [('minimum eccentricity', 'be in', 'graph'),\n",
       "   ('node', 'be with', 'minimum eccentricity in graph')],\n",
       "  [('based estimator', 'be in', 'general graphs')],\n",
       "  [('algorithm can output',\n",
       "    'be with',\n",
       "    'high probability independent of number of infected nodes')],\n",
       "  [('We', 'verify', 'performance of reverse infection algorithm')],\n",
       "  [('reverse infection algorithm',\n",
       "    'outperform signiﬁcantly',\n",
       "    'random guesses')],\n",
       "  [('spread', 'be in', 'networks')],\n",
       "  [('information source detection problem', 'be study under', 'SI model')],\n",
       "  [('differences', 'be', 'summarized below')],\n",
       "  [('NumericCitation', 'be restricted to', 'set of infected nodes'),\n",
       "   ('possible sources', 'be in', 'SI model NumericCitation')],\n",
       "  [('node', 'be', 'recovered'),\n",
       "   ('possible information sources', 'be in', 'SIR model')],\n",
       "  [('nodes', 'be', 'infected'),\n",
       "   ('key observation', 'be in', 'NumericCitation')],\n",
       "  [('node', 'be with', 'maximum closeness centrality')],\n",
       "  [('infection closeness centrality',\n",
       "    'be',\n",
       "    'inverse of sum of distances to infected nodes')],\n",
       "  [('nodes', 'be with', 'maximum infection closeness')],\n",
       "  [('We', 'assume', 'time slotted system')],\n",
       "  [('its state changes', 'be with', 'probability p')],\n",
       "  [('t', 'be', 'unknown')],\n",
       "  [('Figure 1', 'be example of', 'infection process')],\n",
       "  [('left ﬁgure', 'show information propagation over', 'time')],\n",
       "  [('node 3', 'recover', '3'), ('node', 'recover', '3')],\n",
       "  [('information source', 'give x be', '0 t')],\n",
       "  [('each', 'for o be', 't2')],\n",
       "  [('nodes', 'be in', 'network'),\n",
       "   ('N', 'be', 'where number of nodes in network')],\n",
       "  [('it', 'solve', 'maximum likelihood problem')],\n",
       "  [('we', 'overcome', 'difﬁculty')],\n",
       "  [('we', 'derive', 'structure properties of optimal sample paths')],\n",
       "  [('nodes', 'be with', 'minimum infection eccentricity')],\n",
       "  [('v14', 'be', 'infected'), ('v14', 'be', 'observed')],\n",
       "  [('node', 'be with', 'minimum infection eccentricity')],\n",
       "  [('vr', 'be', 'information source'),\n",
       "   ('optimal sample path', 'be in', 't ∗')],\n",
       "  [('vr', 'equal to', 'infection eccentricity of node vr')],\n",
       "  [('optimal sample path', 'source of be', 'Jordan infection center'),\n",
       "   ('u', 'have', 'larger infection eccentricity')],\n",
       "  [('we', 'use', 'induction over k')],\n",
       "  [('X', 'slot to', 't')],\n",
       "  [('We', 'have', 'T − vr v3 v4 u u')],\n",
       "  [('infected node', 'be in', 'network')],\n",
       "  [('vr', 'infect u with', 'probability q'), ('vr', 'infect', 'u')],\n",
       "  [('u', 'be', 'susceptible'), ('u', 'be susceptible at', 'time slot t')],\n",
       "  [('node u', 'be infect at', 'time u'), ('node u', 'be', 'infected')],\n",
       "  [('u', 'recover in', 'second time slot')],\n",
       "  [('t', 'be', 'none-increasing')],\n",
       "  [('none-increasing function', 'be in', 'summary'),\n",
       "   ('Pr', 'be', 'none-increasing'),\n",
       "   ('k', '=', '0')],\n",
       "  [('k', '=', 'n +1')],\n",
       "  [('vr', 'be', 'infected'),\n",
       "   ('recovery time', 'be in', '≤ t'),\n",
       "   ('ones', 'be in', 'T i')],\n",
       "  [('node', 'be with', 'smaller infection eccentricity')],\n",
       "  [('we', 'claim', '0 t'), ('we', 'first claim', '0 t')],\n",
       "  [('tree networks', 'be with', 'inﬁnitely many levels'),\n",
       "   ('Jordan infection',\n",
       "    'center as',\n",
       "    'possible candidates of information source'),\n",
       "   ('Jordan infection center',\n",
       "    'be in',\n",
       "    'tree networks with inﬁnitely many levels'),\n",
       "   ('R EV ER S E', 'be in', 'FEC T ION A LGOR')],\n",
       "  [('information source', 'be in', 'general networks')],\n",
       "  [('its', 'identity', 'ID')],\n",
       "  [('ID', 'be in', 'message')],\n",
       "  [('message', 'be', 'received')],\n",
       "  [('information source', 'be with', 'smallest tv')],\n",
       "  [('We', 'break tie at', 'random'),\n",
       "   ('Jordan infection center', 'be with', 'largest infection closeness'),\n",
       "   ('We', 'break', 'ties')],\n",
       "  [('reverse infection algorithm',\n",
       "    'be base on',\n",
       "    'structure properties of optimal sample paths on trees')],\n",
       "  [('source', 'be', 'constant'),\n",
       "   ('constant distance',\n",
       "    'be with',\n",
       "    'high probability independent of number of infected nodes')],\n",
       "  [('regular tree', 'be with', 'inﬁnitely many levels')],\n",
       "  [('observed infection topology Y', 'contain', 'at least one infected node')],\n",
       "  [('snapshot Y', 'be', 'taken'),\n",
       "   ('d', 'time', 'taken'),\n",
       "   ('d', 'be with', 'probability')],\n",
       "  [('their parents', 'be', 'infected'),\n",
       "   ('we', 'recover node at', 'level'),\n",
       "   ('we', 'recover', 'nodes')],\n",
       "  [('|', 'be in', 'addition')],\n",
       "  [('Ties', 'be', 'broken'), ('Ties', 'be break at', 'random')],\n",
       "  [('times', 'be in', 'Z τ')],\n",
       "  [('Our proof', 'be base on', 'GW')],\n",
       "  [('process', 'be', 'stochastic'),\n",
       "   ('recurrence formula B =', 'ζi in', 'g'),\n",
       "   ('1 ζi', 'be in', 'g')],\n",
       "  [('regular tree', 'evolution of', 'Z τ'),\n",
       "   ('tree', 'be', 'branching'),\n",
       "   ('offspring distribution', 'be', 'where function of τ')],\n",
       "  [('we', 'use interchangeably', 'two notations')],\n",
       "  [('Z τ l', 'l give', 'Kl ≥ 2 for l ≤ L')],\n",
       "  [('u1 ∈ Z τ', 'be root of', 'survived B 1 branching process')],\n",
       "  [('v', 'be infected node at', 'lowest level on sub-tree'),\n",
       "   ('v', 'be', 'infected')],\n",
       "  [('τ l', 'distance between', 'v Therefore')],\n",
       "  [('geometric distribution', 'be with', 'parameter p.')],\n",
       "  [],\n",
       "  [('we', 'now give', 'L ≥ 2'), ('we', 'give', 'L ≥ 2')],\n",
       "  [('one infected node', 'be in', 'Y')],\n",
       "  [('we', 'have', 'Pr')],\n",
       "  [('Binomial Random Trees 6', 'on infection be', 'RI')],\n",
       "  [('degree', 'be', 'when large')],\n",
       "  [('Internet number', 'on infection be', 'RI')],\n",
       "  [('g', 'vary β from', '0.1 to 0.9'), ('g', 'β from', '0.1 to 0.9')],\n",
       "  [('results', 'be show in', 'Figure 8')],\n",
       "  [('duration t', 'be', 'integer uniformly chosen from NumericCitation')],\n",
       "  [('networks', 'be in', '3Available')],\n",
       "  [('0', 'from 0 be', '1')],\n",
       "  [('t', 'be', 'where uniformly chosen from NumericCitation'),\n",
       "   ('t', 'be', 'where chosen from NumericCitation')],\n",
       "  [('networks', 'be in', 'which')],\n",
       "  [('Figure 7', 'show detection rate as', 'function of g')],\n",
       "  [('average difference', 'be', '8.86 %')],\n",
       "  [('binomial distribution', 'be with', '3 Binomial random trees')],\n",
       "  [('CONCLU S ION', 'be in', 'paper')],\n",
       "  [('node', 'be with', 'minimum infection eccentricity')],\n",
       "  [('We',\n",
       "    'evaluate',\n",
       "    'performance of proposed reverse infection algorithm on several different network topologies')],\n",
       "  [('Epidemics', 'be in', 'small-world networks')],\n",
       "  [('NumericCitation A. Ganesh', 'be in', 'Proc')],\n",
       "  [('Detecting sources', 'be in', 'networks'), ('Theory', 'be in', 'Proc')],\n",
       "  [('Rumors', 'be in', 'network')],\n",
       "  [('NumericCitation', 'be in', 'Proc')],\n",
       "  [('NumericCitation N. T. J. Bailey', 'theory of', 'infectious diseases')],\n",
       "  [('Power', 'on infection be', 'RI'),\n",
       "   ('total number', 'be in', 'of range of NumericCitation')],\n",
       "  [('autonomous Figure 9',\n",
       "    'show',\n",
       "    'results on Internet autonomous systems network')],\n",
       "  [('Internet autonomous systems', 'between communication be', '31 Citation'),\n",
       "   ('Internet autonomous system network', 'be obtain base on', '31 Citation')],\n",
       "  [('Figure 9',\n",
       "    '% of',\n",
       "    'estimators identiﬁed by reverse infection algorithm')],\n",
       "  [('Epidemic spreading', 'be in', 'scale-free networks')],\n",
       "  [('M. Leng', 'region in', 'large networks'),\n",
       "   ('Identifying infection sources regions', 'be in', 'large networks')],\n",
       "  [('NumericCitation V. G. Subramanian', 'be in', 'Proc')],\n",
       "  [('NumericCitation C. Milling', 'be in', 'Proc')],\n",
       "  [('NumericCitation D. Koschutzki', 'be in', 'Network Analysis')],\n",
       "  [('Lecture Notes', 'be in', 'Computer Science')]],\n",
       " 10: [[('SI term', 'be in', 'SIR model'),\n",
       "   ('linear ODEs',\n",
       "    'be with',\n",
       "    'addition of quadratic terms for number of new infections similar to SI term in SIR model')],\n",
       "  [('plague epidemics', 'be in', 'London')],\n",
       "  [('similar processes', 'be in', 'ﬁelds such host-pathogen biology')],\n",
       "  [('fraction', 'be in', 'disease state')],\n",
       "  [('proof', 'rely on', 'operator semigroup techniques')],\n",
       "  [('proofs', 'be in', 'Kurtz')],\n",
       "  [('convergence', 'be in', 'mean-square'),\n",
       "   ('convergence', 'be in', 'probability')],\n",
       "  [('they', 'start out with', 'DTMC')],\n",
       "  [('Jensen', 'have', 'inequality')],\n",
       "  [('nodes', 'be in', 'network increases')],\n",
       "  [('corresponding product', 'from e be', 'S I'),\n",
       "   ('total stochasticity', 'be in', 'system')],\n",
       "  [('our', 'approach', 'Citation')],\n",
       "  [('infected node', 'recover at', 'rate γ')],\n",
       "  [('We', 'deﬁne', 't')],\n",
       "  [('dependence', 'be in', 'our notation')],\n",
       "  [('Sn =', 'be in', '= j − 1'),\n",
       "   ('Kolmogorov 3 equations', 'system of', 'n2 diﬀerential equations'),\n",
       "   ('Sn =', 'be in', '= j'),\n",
       "   ('Sn =', 'be in', '= j')],\n",
       "  [('sn', 'be in', 'y 2 0'),\n",
       "   ('time', 'on sn be', 't'),\n",
       "   ('y 2 0', 'in sn be', 't')],\n",
       "  [('3.1 Preliminaries', 'be in', 'subsection')],\n",
       "  [('problem', 'be with', 'diﬀerential equations')],\n",
       "  [('we', 'be', 'complete')],\n",
       "  [('We', 'describe', 'aspects of stochastic process')],\n",
       "  [('− E', '=', 'E')],\n",
       "  [('Armbruster', 'of proposition be', '2016'),\n",
       "   ('same approach', 'be in', 'as Proposition 2 of Armbruster')],\n",
       "  [('Markov process X', 'be with', 'transition rate matrix Q'),\n",
       "   ('transition rate matrix Q', 'with x be', 't')],\n",
       "  [('terms', 'be in', 'sum')],\n",
       "  [('we', 'handle', 'two types of transitions')],\n",
       "  [('Figure 1', 'illustrate for', 'numerical example')],\n",
       "  [('diﬃculty', 'be with', 'equations'),\n",
       "   ('equations', 'with diﬃculty be', '4')],\n",
       "  [('susceptibles', 'be with', 'τ'),\n",
       "   ('susceptibles', 'of fraction be', 'γ =')],\n",
       "  [('solution',\n",
       "    'be in',\n",
       "    'comparison to expected fraction calculated for increasing number of nodes')],\n",
       "  [('corresponding product', 'to e be', 'si')],\n",
       "  [('random variable Y', 'be in', 'NumericCitation')],\n",
       "  [('proof', 'be adapt from', '2014')],\n",
       "  [('we', 'apply', 'Lemma 3')],\n",
       "  [('factor', 'be in', 'Lemma 4')],\n",
       "  [],\n",
       "  [('random variables', 'for e be', 'Y 2Z'),\n",
       "   ('random variables', 'for e be', '6a'),\n",
       "   ('random variables', 'be in', 'NumericCitation')],\n",
       "  [('2', 'in sn be', 't'),\n",
       "   ('We', 'now apply', 'Lemma 5'),\n",
       "   ('sn in', 'be in', '2')],\n",
       "  [('2', 'in sn be', 't'),\n",
       "   ('sn', 'be in', '2'),\n",
       "   ('h3', 'n in', 'such 9'),\n",
       "   ('n', 'be in', 'such 9'),\n",
       "   ('E sn 2', 'be in', '='),\n",
       "   ('t', 'sn in', '='),\n",
       "   ('=', 'in sn be', 't'),\n",
       "   ('such 9', 'in n be', 't')],\n",
       "  [('h4', 'n in', 'NumericCitation such'),\n",
       "   ('n', 'be in', 'NumericCitation such'),\n",
       "   ('total variance', 'be in', 'system'),\n",
       "   ('NumericCitation such', 'in n be', 't')],\n",
       "  [('inequalities', 'be in', 'Lemma 5'),\n",
       "   ('total stochasticity', 'be in', 'system')],\n",
       "  [('proper system',\n",
       "    'be with',\n",
       "    'same variables on right hand side as on left hand side'),\n",
       "   ('right hand side', 'on variable be', 'E s'),\n",
       "   ('as left hand side', 'on variable be', 'E s')],\n",
       "  [('sn', 'be in', '0'),\n",
       "   ('initial value problem', 'of solution be', '= gn t'),\n",
       "   ('0', 'in sn be', '0')],\n",
       "  [('Our next step', 'bind', 'zn − ¯ z')],\n",
       "  [('x2', 'be with', 'solutions ϕ1')],\n",
       "  [('L. Lipschitz', 'be with', 'in')],\n",
       "  [('proof', 'be in', 'appendix')],\n",
       "  [('respect 3', 'be in', 'z')],\n",
       "  [('We', 'now apply', 'Lemma 6')],\n",
       "  [('t ≤ T', 'for z0 be', '13')],\n",
       "  [('→ 0', 'as 0 be', 'right hand side goes to zero'),\n",
       "   ('→ 0', 'as 0 be', 'n → z0')],\n",
       "  [('E', 'be in', 'y2 |')],\n",
       "  [('sn', 'be in', '− y 2 2')],\n",
       "  [('Our main tool', 'be', 'Lemma 5')],\n",
       "  [('us', 'bind in', 'right hand side of diﬀerential equations')],\n",
       "  [('linear ODEs',\n",
       "    'be with',\n",
       "    'addition of quadratic terms for number of new infections')],\n",
       "  [('We', 'use seir model as', 'example')],\n",
       "  [('existing diﬀerential equations', 'be in', 'ODE systems')],\n",
       "  [('lower red dotted curve', 'of upper be', 'green dashed-dotted curve'),\n",
       "   ('we', 'apply', 'Lemma 5')],\n",
       "  [('we', 'establish', 'ODE system equivalent to 11')],\n",
       "  [('n', 'be with', 'variance of exposed fraction on left hand side'),\n",
       "   ('h5', 'n with', 'variance of exposed fraction on left hand side')],\n",
       "  [('total variance', 'be in', 'system')],\n",
       "  [('Section 3.5', 'prove', 'convergence'),\n",
       "   ('Section', 'prove', 'convergence')],\n",
       "  [('SIR case', 'to approach be', 'Citation'),\n",
       "   ('mean-ﬁeld convergence results', 'use', 'only basic ODE theory'),\n",
       "   ('convergence results', 'use', 'only basic ODE theory'),\n",
       "   ('our', 'approach', 'Citation')],\n",
       "  [('we', 'use', 'approach'),\n",
       "   ('more researchers',\n",
       "    'rigorously analyze',\n",
       "    'accuracy of compartmental models')],\n",
       "  [('We', 'thank', 'Tom Britton')],\n",
       "  [('Lecture Notes', 'be in', 'Statistics')],\n",
       "  [('SIR process', 'be in', 'random network with heterogeneous connectivity'),\n",
       "   ('random network', 'be with', 'heterogeneous connectivity')],\n",
       "  [('Wiley series', 'be in', 'probability')],\n",
       "  [('Research', 'on symposium be', 'pages 2 15'),\n",
       "   ('Research', 'be in', 'Security')],\n",
       "  [('Limit theorems', 'approximate', 'ordinary diﬀerential processes')],\n",
       "  [('Epidemiology', 'be in', 'coevolution of parasites')],\n",
       "  [('random networks', 'be with', 'heterogeneous connectivity'),\n",
       "   ('SIR dynamics',\n",
       "    'be in',\n",
       "    'random networks with heterogeneous connectivity')],\n",
       "  [('f2 ≤', 'be with', 'constant L'),\n",
       "   ('f2 ≤', 'be with', 'solutions ϕ1'),\n",
       "   ('Lipschitz', 'be in', 'x')],\n",
       "  [('claim', 'apply', \"specialized form of Gronwall 's Inequality\"),\n",
       "   ('Gronwall', 'of', 'Inequality')],\n",
       "  [('t Gronwall', 'have', 'Inequality')],\n",
       "  [('θ', 'be', 'nonnegative'), ('θ', 'be', 'continuous')]],\n",
       " 11: [[('high-dimensional lattices', 'be with', 'random edge weights'),\n",
       "   ('Survival probabilities',\n",
       "    'be with',\n",
       "    'random edge weights Xiaofeng Xue ∗ Beijing Jiaotong University Abstract')],\n",
       "  [('1 Introduction', 'be in', 'paper')],\n",
       "  [('We', 'denote', 'origin of Zd')],\n",
       "  [('Ct t ≥ 0', 'be with', 'state space'),\n",
       "   ('stochastic SIS model', 'be with', 'edge weights')],\n",
       "  [('λ', 'be', 'positive'),\n",
       "   ('Science', 'of school be', 'X1 = at rate 1'),\n",
       "   ('rate Ct → λ 2d y', 'at ct be', 'x')],\n",
       "  [('t ≥ 0', 'be with', 'state space'),\n",
       "   ('model', 'be', 'auxiliary'),\n",
       "   ('stochastic SIR model', 'be with', 'edge weights'),\n",
       "   ('model', 'be', 'useful'),\n",
       "   ('St', 'be', 'continuous-time'),\n",
       "   ('St', 'be', '=')],\n",
       "  [('we', 'give', 'our results'), ('we', 'give', 'our main results')],\n",
       "  [('Zd', 'be with', 'infection rate λ')],\n",
       "  [('lim lim d →', 'ρ for', 'λ ≥ 1 Eρ')],\n",
       "  [('high-dimensional SIS', 'be with', 'random edge weights')],\n",
       "  [('C O λ − 1', '=', '∅'), ('classic SIS model', 'be with', 'ρ ≡ 1')],\n",
       "  [('d t 3 Similar result',\n",
       "    'be with',\n",
       "    'that in Theorem 2.1 for bond percolation model')],\n",
       "  [('O', 'belong to', 'inﬁnite open cluster'), ('2d', 'be with', 'λ >')],\n",
       "  [('infectious vertices', 'die out with', 'probability one')],\n",
       "  [('p', '=', '1 − P')],\n",
       "  [('λ', 'corollary of', 'Theorem 2.1')],\n",
       "  [('∀ t ≥', 't for', 'λ 1 Eρ')],\n",
       "  [('∀ t ≥ 0 =', 't for', 'λ')],\n",
       "  [('Section 3.1', '=', '∅'), ('I', 'o', 'NumericCitation')],\n",
       "  [('I', 'o', 't')],\n",
       "  [('> 1 x', 'be in', 'set u')],\n",
       "  [('given large inter M', 'be with', 'high probability')],\n",
       "  [('vertex', 'be with', 'l1')],\n",
       "  [('We', 'divide', 'Zd')],\n",
       "  [('d1/3 vertices', 'be in', 'Γ1')],\n",
       "  [('d1/4 vertices', 'be in', 'Γ2')],\n",
       "  [('d1/4 initial infectious vertices', 'be in', 'Γ2')],\n",
       "  [('approach', 'be in', 'step')],\n",
       "  [('survival probability',\n",
       "    'hence be least product of',\n",
       "    'probability of third event')],\n",
       "  [('we', 'make', 'above explanation rigorous')],\n",
       "  [('vertices', 'be with', 'l1 norm'),\n",
       "   ('2d', 'b as', 'set of vertices with l1 norm at most r')],\n",
       "  [('we', '≤', 'r')],\n",
       "  [('O', 'be with', 'length')],\n",
       "  [('Chebyshev', 'accord to', 'inequality')],\n",
       "  [('P', '=', 'λ'),\n",
       "   ('n ≥ 0', 'for ≥ be', 'ρ'),\n",
       "   ('biased random walk', 'as ≥ be', 'ρ')],\n",
       "  [('K', 'be', 'visited'), ('K', 'be', 'when visited')],\n",
       "  [('given M >', 'for a be', 'Ct')],\n",
       "  [('vertex', 'be with', 'l1 norm lager')],\n",
       "  [('t >', 'for ∅ be', 'τ0 < τK'), ('ω ∈ A', 'for ∅ be', 'τ0 < τK')],\n",
       "  [('Ct |', '∅ for', 't')],\n",
       "  [('we', 'have', 'Ct = ∅')],\n",
       "  [('element', 'be in', 'X3')],\n",
       "  [('exponential time', 'be with', 'rate 2d ρ for Hd'),\n",
       "   ('exponential time', 'be with', 'rate one for x'),\n",
       "   ('Y', 'be', 'exponential')],\n",
       "  [('Zd', 'be with', 'length n'), ('l', 'be', 'infectious')],\n",
       "  [('We', 'have', 'following important lemma')],\n",
       "  [('O', 'from y be', 'U'),\n",
       "   ('I', 'o', 't t'),\n",
       "   ('Pλ', 'ω for', 'ω ∈ Ωd'),\n",
       "   ('I', 'o there', 't t'),\n",
       "   ('ω ∈ Ωd', 'for ω be', '·')],\n",
       "  [('x', 'be', 'infected')],\n",
       "  [('l', 'be', 'infectious')],\n",
       "  [('identiﬁcation', 'be permit by', 'Lemma 4.1')],\n",
       "  [('we', 'introduce deﬁnitions for', 'later use')],\n",
       "  [('Lemma 4.2', 'will', 'where will utilized'),\n",
       "   ('Lemma', 'will', 'where will utilized')],\n",
       "  [('vertex', 'be in', 'B')],\n",
       "  [('D2 n', 'since y be', 'Γ2')],\n",
       "  [('subsection', 'in i.o. be', '4.4'), ('i.o.', 'be in', 'subsection')],\n",
       "  [('we', 'first introduce', 'notations')],\n",
       "  [('similar analysis', 'be with', 'that of Equation')],\n",
       "  [('we', 'deﬁne', 'φK inf n ≥ 0'),\n",
       "   ('K', 'be', 'visited'),\n",
       "   ('K', 'be', 'when visited')],\n",
       "  [('we', 'now give', 'proof of Lemma 4.2')],\n",
       "  [('I', '=', '∪ t ≥ 0'),\n",
       "   ('SIR model', 'be with', 'random edge'),\n",
       "   ('similar analysis', 'be with', 'that leads to Lemma 4.1'),\n",
       "   ('I', '=', '∪ t ≥ 0 It')],\n",
       "  [('t ≥ 0', 'be with', 'random edge B')],\n",
       "  [('ω', 'with respect to weight be', 'd x ∈')],\n",
       "  [('embedded chain', 'be with', 'random edge weights')],\n",
       "  [('d1/3 φ0', 'd1/3 in', 'sense of coupling'),\n",
       "   ('φ d1/3', 'be in', 'sense of coupling')],\n",
       "  [('ω ∈ F', 'for m2 be', 'd'), ('ω ∈ F', 'for m2 be', 'P φ d1/3')],\n",
       "  [('similar analysis', 'be with', 'that of Equation ='),\n",
       "   ('similar analysis', 'accord to 12 be', '4.10'),\n",
       "   ('similar analysis', 'accord to 12 be', '4.9')],\n",
       "  [('lim', 'be', 'arbitrary')],\n",
       "  [('Lemma 4.2', 'follow from', '4.8')],\n",
       "  [('U', 'ω for', 'ω ∈ Ω')],\n",
       "  [('ω', 'be with', 'measure νω × π'),\n",
       "   ('π',\n",
       "    'be',\n",
       "    'where probability measure of i.i.d exponential times with rate λΘ'),\n",
       "   ('probability measure', 'be with', 'rate λΘ'),\n",
       "   ('Pλ', 'ω with', 'measure νω × π')],\n",
       "  [('s', 'for q be', 'A')],\n",
       "  [('following lemma', 'be', 'crucial 4.3')],\n",
       "  [('Lemma h', '=', 'Eλ')],\n",
       "  [('⊆ Γ1', 'be with', '| A | ='), ('⊆ Γ1', 'for d be', 'λΘ')],\n",
       "  [('it', 'be easy to', 'end of λd1/3E ρ Eλ')],\n",
       "  [('d', 'note', 'Pλ')],\n",
       "  [('x ∈ B', 'for pλ be', 'd'), ('x ∈ B', 'for pλ be', 'Y < inf U')],\n",
       "  [('exponential time', 'be with', 'rate y')],\n",
       "  [('λΘ', 'be rate of', 'x')],\n",
       "  [('Y x', 'by x be', 'y ∈ Γ1')],\n",
       "  [('Chebyshev', \"'s inequality be\", '|'),\n",
       "   ('will ⊆ Γ1', 'be with', '| A | ='),\n",
       "   ('Chebyshev', 'by', 'inequality')],\n",
       "  [('4.17 Lemma 4.3', 'follow from', '4.13')],\n",
       "  [('d', 'proof of', 'Lemma 4.5')],\n",
       "  [('4.25 Lemma 4.5', 'follow from', '4.18')],\n",
       "  [('we', 'give', 'proof of Lemma 4.4'), ('4.4 Proof', 'be in', 'subsection')],\n",
       "  [('i', 'for ln be', 'x'),\n",
       "   ('= x', 'ln for', 'i'),\n",
       "   ('Zd', 'be with', 'length n.')],\n",
       "  [('σ =', 'deﬁne for', 'x')],\n",
       "  [('≥', 'be', 'self-avoiding'),\n",
       "   ('≥', 'be', 'random'),\n",
       "   ('α0 ∈ Γ3', '−', '= ej')],\n",
       "  [('it', 'be', 'then easy'), ('n ≥ 1', 'for ln be', 'α0')],\n",
       "  [('≥', 'be', 'independent')],\n",
       "  [('αn', '≥ with', 'α0 ='), ('qn n ≥ 1', 'be with', 'α0 =')],\n",
       "  [('Px', 'd for', 'x')],\n",
       "  [('similar analysis',\n",
       "    'be with',\n",
       "    'that leads to Lemma 3.4 of NumericCitation')],\n",
       "  [('large d', 'for ∈ be', '1 λEρ'), ('Equation', 'by ρ be', '4.31')],\n",
       "  [('x', 'for ρ be', 'x = y'),\n",
       "   ('x', 'for ρ be', '4.32'),\n",
       "   ('large d', 'for 1 be', 'log d')],\n",
       "  [('Lemma 4.6', 'by m4 be', 'log d')],\n",
       "  [('d', '= log d accord to', '4.27')],\n",
       "  [('k ≥ 1', 'for ϑ be', 'αk log d')],\n",
       "  [('equations', 'by d be', '4.39')],\n",
       "  [('proof', 'utilize', 'following Proposition given in NumericCitation')],\n",
       "  [('1 ≤ i ≤ n', 'for p be', 'Ai')],\n",
       "  [('detailed proof', 'be', 'given there')],\n",
       "  [('l', 'be', 'yi G y'), ('same factors', 'be in', 'denominator')],\n",
       "  [('other event', 'be in', 'factor'),\n",
       "   ('d independent exponential times', 'be with', 'rate λΘ 2d')],\n",
       "  [('Equations', 'by | be', '4.43')],\n",
       "  [('4.44 Lemma 4.6', 'follow directly from', '4.41')],\n",
       "  [('5 Proof', 'be in', 'section')],\n",
       "  [('C O', '=', '∅')],\n",
       "  [('similar analysis', 'be with', 'that leads to Equation')],\n",
       "  [('λ', '=', 'γ Eρ')],\n",
       "  [('O', 'be with', 'length n'),\n",
       "   ('l', 'be', 'infectious'),\n",
       "   ('U', 'be', 'infectious')],\n",
       "  [('O self-avoiding paths', 'be with', 'length n staring at O'),\n",
       "   ('infectious path', 'be with', 'length n')],\n",
       "  [('Borel-Cantelli', 'accord to', 'lemma')],\n",
       "  [('Lemma 4.1', 'by n be', '5.3'), ('Pλ', 'n by', 'Lemma 4.1')],\n",
       "  [('we', 'let', 'γ → 1')],\n",
       "  [('Theorem 2.2', 'follow from', '5.1')],\n",
       "  [('ﬁnancial support', 'be with', 'grant number KSRC16006536'),\n",
       "   ('ﬁnancial support', 'be with', 'grant number 11501542')],\n",
       "  [('complete convergence theorem',\n",
       "    'hold + for',\n",
       "    'contact processes on open clusters of Z d × Z')],\n",
       "  [('Oriented percolation', 'be in', 'dimensions')],\n",
       "  [('Asymptotics',\n",
       "    'be in',\n",
       "    'high dimension for Fortuin-Kasteleyn cluster model')],\n",
       "  [('contact process', 'be with', 'random vertexdependent infection rates')],\n",
       "  [('Contact processes',\n",
       "    'be with',\n",
       "    'random vertex weights on oriented lattices')],\n",
       "  [('large-dimensional contact process',\n",
       "    'be with',\n",
       "    'random recovery rates on open clusters')],\n",
       "  [('contact processes', 'be in', 'random environment on Z d × Z'),\n",
       "   ('complete convergence theorem',\n",
       "    'hold + for',\n",
       "    'contact processes in random environment on Z d × Z')]],\n",
       " 12: None,\n",
       " 13: [[('I-state', 'be with', 'evident clinical symptoms labelled as I'),\n",
       "   ('we', 'divide', 'infectious individuals')],\n",
       "  [('they', 'contact', 'I-nodes')],\n",
       "  [('diﬀerent factors', 'of inﬂuence be', 'network structure'),\n",
       "   ('diﬀerent factors', 'of inﬂuence be', 'awareness rate')],\n",
       "  [('mathematical models', 'be often establish with', 'static parameters')],\n",
       "  [('behavioural changes', 'be in', 'models of infectious disease dynamics')],\n",
       "  [('rich phase space', 'be with', 'multiple epidemic peaks'),\n",
       "   ('Perra et al.',\n",
       "    'self-initiated social distancing into',\n",
       "    'classical SIR model')],\n",
       "  [('local information based behavioral responses',\n",
       "    'reduce',\n",
       "    'prevalence of epidemic NumericCitation')],\n",
       "  [('coupled disease-behavior interaction models',\n",
       "    'be in',\n",
       "    'multiplex networks')],\n",
       "  [('epidemic dynamics', 'be in', 'adaptive networks')],\n",
       "  [('behavioral', 'respond towards', 'diseases')],\n",
       "  [('we',\n",
       "    'quantitatively analyze',\n",
       "    'eﬀects of asymptomatic infection on epidemic dynamics')],\n",
       "  [('E-state', 'be in', 'standard SEIR model')],\n",
       "  [('they', 'contact', 'symptomatic patients')],\n",
       "  [('analytical results',\n",
       "    'can quantitatively describe',\n",
       "    'impacts of diﬀerent factors on epidemic threshold')],\n",
       "  [('Our ﬁndings',\n",
       "    'may partially answer',\n",
       "    'key challenges mentioned in ﬁrst paragraph proposed in NumericCitation')],\n",
       "  [('classical SIS process', 'be in', 'complex networks')],\n",
       "  [('U-neighbor', 'be with', 'infection rate β'),\n",
       "   ('I-neighbor', 'be with', 'awareness rate βF')],\n",
       "  [('I-nodes', 's-state with', 'recovery rate µ')],\n",
       "  [('transition diagram', 'be depict in', 'upper panel of Fig. 1')],\n",
       "  [('model', 'be', 'irreversible')],\n",
       "  [('following diﬀerential equations NumericCitation',\n",
       "    'impact of',\n",
       "    'asymptomatic infection')],\n",
       "  [('U-or I-neighbor', 'be with', 'rate β'), ('S-node', 'become', 'U-node')],\n",
       "  [('A-node', 'become', 'U-node'), ('U-or I-neighbor', 'be with', 'rate γ β')],\n",
       "  [('U-nodes', 'become', 'I-nodes with rate δ'),\n",
       "   ('U-nodes', 'i-node with', 'rate δ'),\n",
       "   ('I-nodes', 'be with', 'rate δ')],\n",
       "  [('ﬁrst term', 'be in', 'right side of Eq')],\n",
       "  [('third term', 'denote', 'recovery of I-nodes of degree k')],\n",
       "  [('other terms', 'be in', 'Eqs')],\n",
       "  [('f', 'as f be', 'β')],\n",
       "  [('other hand', 'be in', 'Ref')],\n",
       "  [('it', 'be', 'when used in modiﬁed SIR model'), ('it', 'be', 'when used')],\n",
       "  [('cavity theory',\n",
       "    'obtain',\n",
       "    'epidemic threshold for SAUIR model NumericCitation')],\n",
       "  [('β', 'be in', 'our model')],\n",
       "  [('AI', 'probability of', 'AI → U I')],\n",
       "  [('β', 'number of', 'infected nodes')],\n",
       "  [('S-node', 'be', 'then infected by I-node again')],\n",
       "  [('S-node', 'become', 'A-node')],\n",
       "  [('method', 'propose by', 'Newman et al.')],\n",
       "  [('SAUIS model', 'for fig. be', 'Color online'),\n",
       "   ('NumericCitation',\n",
       "    'impact of',\n",
       "    'asymptomatic infection on coupled disease-behavior dynamics'),\n",
       "   ('β', 'be in', 'UCM networks')],\n",
       "  [('Insets', 'be in', 'bottom subﬁgures')],\n",
       "  [('U-nodes', 'enter', 'I-state'), ('U-nodes', 'quickly enter', 'I-state')],\n",
       "  [('ﬁnal epidemic size', 'reduce', 'value of γ'),\n",
       "   ('results', 'be in', 'Fig. 2')],\n",
       "  [('we', 'compare', 'Fig. 2')],\n",
       "  [('coupled disease-behavior interaction model', 'be in', 'complex networks'),\n",
       "   ('we', 'divide individual into', 'I-state')]],\n",
       " 14: [[('Infectious diseases', 'spread through', 'human networks')],\n",
       "  [('latent period', 'be take into', 'account')],\n",
       "  [('We', 'call', 'it SIIR model')],\n",
       "  [('two infectious stages',\n",
       "    'be in',\n",
       "    'our modiﬁed metapopulat ion SIR model')],\n",
       "  [('Individuals', 'be in', 'infected stage')],\n",
       "  [('1 Introduction Infectious diseases', 'spread through', 'human networks')],\n",
       "  [('It',\n",
       "    'be expected number of',\n",
       "    'infections caused by typical infectious individual in completely susceptible population NumericCitation')],\n",
       "  [('1 2 Kanako Mizuno', 'magnitude of', 'outbreak')],\n",
       "  [('movements',\n",
       "    'be',\n",
       "    'In metapopulation SIR models taken into account NumericCitation')],\n",
       "  [('Individuals', 'can move between', 'two nodes connected by link')],\n",
       "  [('R0', 'be in', 'group'),\n",
       "   ('global invasion threshold', 'be in', 'metapopulation system')],\n",
       "  [('latent period', 'be take into', 'account')],\n",
       "  [('individuals', 'be', 'latent'),\n",
       "   ('such infected individuals', 'be in', 'latent period')],\n",
       "  [('infected individuals', 'move after', 'latent period')],\n",
       "  [('Such ill individuals',\n",
       "    'infect',\n",
       "    'only susceptible ones in same population'),\n",
       "   ('susceptible ones', 'be in', 'same population')],\n",
       "  [('latent period', 'be incorporate as', 'Exposed state')],\n",
       "  [('discontinuous ﬁnal size distribution', 'be with', 'jump at R0 ='),\n",
       "   ('two infectious stages', 'be in', 'our modiﬁed metapopulation SIR model')],\n",
       "  [('modiﬁed SIR model', 'be', 'introduced in Sec. 2')],\n",
       "  [('modiﬁed model', 'be in', 'Sec. 3')],\n",
       "  [('proportion', 'be in', 'same population'),\n",
       "   ('I', 'depend on', 'proportion of I in same population'),\n",
       "   ('transmission rate', 'proportion of', 'I'),\n",
       "   ('transmission rate', 'proportion in', 'same population')],\n",
       "  [('links', 'be in', 'network')],\n",
       "  [('Spread', 'be with', 'Latent Period ¶ t Sn ='),\n",
       "   ('¶ t', 'be in', '= aSn'),\n",
       "   ('time evolution', 'be in', 'population'),\n",
       "   ('Latent Period ¶ t Sn =', 't in', '= aSn'),\n",
       "   ('Latent Period ¶ t Sn =', 't in', 'Nn − bIn'),\n",
       "   ('¶ t', 'be in', 'Nn − bIn')],\n",
       "  [('We', 'call', 'model SIIR model')],\n",
       "  [('one state', 'be in', 'model')],\n",
       "  [('Individuals', 'be in', 'population m')],\n",
       "  [('I', 'become', 'R at rate b')],\n",
       "  [('Nn − mHn', 't in', '= mHn − bIn'),\n",
       "   ('time evolution', 'be in', 'population'),\n",
       "   ('m ¶ t', 'be in', '= mHn − bIn')],\n",
       "  [('time step', 'be', 'discrete'), ('individuals', 'be in', 'state')],\n",
       "  [('P ∼ k', 'be with', 'g =')],\n",
       "  [('one infected individual', 'be', 'included'),\n",
       "   ('one individual', 'be', 'included')],\n",
       "  [('= b', 'be in', '4 o o i i t t'),\n",
       "   ('change', 'be in', 'attack ratio'),\n",
       "   ('= b', 'be in', 'high-w region')],\n",
       "  [('b =', 'be in', 'cases')],\n",
       "  [('shift', 'be in', 'low-w regions of SIR model')],\n",
       "  [('jump', 'be in', 'attack ratio')],\n",
       "  [('¶ t I', '=', '0')],\n",
       "  [('Spread', 'be with', 'Latent Period'),\n",
       "   ('SIIR model', 'be with', 'effective recovery rate b ′ ='),\n",
       "   ('SIR model', 'be with', 'given recovery rate b =')],\n",
       "  [('b ′ =', 'be in', 'case')],\n",
       "  [('SIIR model', 'be in', 'region'),\n",
       "   ('SIR model', 'be with', 'given recovery rate b =')],\n",
       "  [('result', 'imply', 'following')],\n",
       "  [('effective recovery rate', 'be in', 'SIIR model')],\n",
       "  [('we', 'travel between', 'populations'),\n",
       "   ('discontinuity', 'be in', 'ﬁnal size distribution of SIIR mode l'),\n",
       "   ('I', 'be', 'immobile')],\n",
       "  [('We', 'now modify', '2')],\n",
       "  [('simulation', 'be', 'performed in same conditions as Fig. 2')],\n",
       "  [('pro ﬁle', 'be in', 'Fig. 2'), ('curve', 'be in', 'Fig. 3')],\n",
       "  [('travel rate w', '=', '0.1')],\n",
       "  [('discontinuous ﬁnal size distribution', 'be in', 'SIIR model')],\n",
       "  [('SIR model', 'be with', 'recovery rate b b ′ in region where'),\n",
       "   ('b ′', 'be in', 'region where')],\n",
       "  [('R.M. Anderson', 'disease of', 'Humans')]],\n",
       " 15: [[('susceptibility',\n",
       "    'be in',\n",
       "    'susceptible-infectious-recovered spreading dynamics')],\n",
       "  [('people', 'on', 'susceptibility')],\n",
       "  [('epidemic outbreaks', 'accelerate', 'information dissemination'),\n",
       "   ('outbreaks', 'accelerate', 'information dissemination')],\n",
       "  [('role', 'be in', 'epidemic threshold')],\n",
       "  [('wide occurrence', 'be in', 'real world'),\n",
       "   (\"link 's appearance\", 'be in', 'infection hierarchies NumericCitation'),\n",
       "   ('link', 'of', 'appearance in infection hierarchies NumericCitation')],\n",
       "  [('individuals', 'be with', 'diﬀerent topological features')],\n",
       "  [('Previous studies',\n",
       "    'mainly focus on',\n",
       "    'spreading ability of individuals')],\n",
       "  [('inﬂuential spreaders', 'be in', 'networks'),\n",
       "   ('Many topological measures', 'be', 'employed')],\n",
       "  [('susceptible individuals', 'be in', 'contact networks')],\n",
       "  [('people', 'have', 'susceptibility')],\n",
       "  [('extensive simulations', 'be in', 'real-world social networks'),\n",
       "   ('susceptibility', 'be in', 'SIR')],\n",
       "  [('people', 'on', 'susceptibility'),\n",
       "   ('we',\n",
       "    'first explore',\n",
       "    \"impact of community structure on people 's susceptibility\")],\n",
       "  [('We', 'consider', 'them')],\n",
       "  [('we', 'quantitatively depict', 'property of people')],\n",
       "  [('we', 'examine', 'their topological traits')],\n",
       "  [('infection rates', 'be in', 'SIR')],\n",
       "  [('susceptibility',\n",
       "    'be',\n",
       "    \"For SIR spreading highly correlated to nodes ' inﬂuence\")],\n",
       "  [('epidemic ﬂows', 'be in', 'two social networks with community structure'),\n",
       "   ('two social networks', 'be with', 'community structure')],\n",
       "  [('social networks', 'have', 'have used in previous studies')],\n",
       "  [('community structure',\n",
       "    'be extract by',\n",
       "    'fast heuristic method based on modularity optimization NumericCitation')],\n",
       "  [('diﬀerent colors', 'be in', 'Fig. 1')],\n",
       "  [('we',\n",
       "    'apply',\n",
       "    'susceptible-infectious-recovered SIR NumericCitation model on above social networks')],\n",
       "  [('SIR epidemic spreading', 'be in', 'Facebook social network')],\n",
       "  [('Eight communities', 'be distinguish by', 'diﬀerent colors'),\n",
       "   ('communities', 'be distinguish by', 'diﬀerent colors')],\n",
       "  [('b d', 'distribution of', 'infected probability'),\n",
       "   ('sources', 'be in', 'diﬀerent communities')],\n",
       "  [('solid black dots', 'represent', 'outbreak origins')],\n",
       "  [('infected probability', 'average over', '105 SIR realizations')],\n",
       "  [('colors', 'indicate', 'base 10'),\n",
       "   ('infected probability', 'of value be', 'base 10')],\n",
       "  [('their', 'neighbor', 'S'),\n",
       "   ('permanent immunity', 'be with', 'probability λ')],\n",
       "  [('λ', '=', '1')],\n",
       "  [('we', 'guarantee', 'medium scale infection coverage')],\n",
       "  [('βc', 'be in', 'our simulations')],\n",
       "  [('sources', 'apply', '105 realizations'),\n",
       "   ('three nodes', 'be in', 'diﬀerent communities'),\n",
       "   ('sources', 'apply', '105 realizations of SIR model')],\n",
       "  [('epidemic', 'clearly permeate to', 'several communities')],\n",
       "  [('evolution', 'be in', 'networks with community structure')],\n",
       "  [('Community structure', 'slow down', 'spreading processes'),\n",
       "   ('facebook 0 0 50\\xa0100\\xa0150\\xa0200 0.01 0.02 0.03 0.04 0.05 0.06 0.07 0.08 0.09 0.1 Time t I n f c t e d p r o p o r t i n o ρ SIR facebook',\n",
       "    'rewire',\n",
       "    'facebook Fig. 2')],\n",
       "  [('rumor spreading model', 'be with', 'βR =')],\n",
       "  [('it', 'achieve', 'given infected proportion ρ'),\n",
       "   ('Facebook network', 'be with', 'community structure')],\n",
       "  [('community structure',\n",
       "    'therefore will slow down',\n",
       "    'speed of SIR spreading')],\n",
       "  [('our ﬁnding', 'be applicable to', 'other spreading dynamics'),\n",
       "   ('our ﬁnding', 'be', 'applicable')],\n",
       "  [('can three possible states', 'be in', 'rumor spreading model')],\n",
       "  [('Ignorant people', 'be', 'individuals unaware of rumor')],\n",
       "  [('step', 'be in', 'time'), ('spreaders', 'be with', 'probability βR')],\n",
       "  [('stiﬂers', 'be with', 'probability λ')],\n",
       "  [('λ', '=', '1')],\n",
       "  [('We', 'obtain', 'similar results')],\n",
       "  [('nodes', 'be in', 'network')],\n",
       "  [('T', '=', '104')],\n",
       "  [('deﬁnition', 'be in', 'our research')],\n",
       "  [('nodes', 'be with', 'higher susceptibility')],\n",
       "  [('susceptibility', 'be with', 'standard set of equations of SIR dynamics')],\n",
       "  [('network', 'be with', 'N nodes'), ('aij', 'be', 'connected')],\n",
       "  [('We', 'obtain', 't')],\n",
       "  [('iteration', 'converge very quickly in case of', 'sparse networks')],\n",
       "  [('susceptibility', 'can', 'Therefore can calculated by method eﬃciently')],\n",
       "  [('We', 'assign', '000 links among 104 nodes'),\n",
       "   ('We', 'randomly assign', '000 links among 104 nodes')],\n",
       "  [('discrepancy', 'can', 'can explained by invalidation of assumptions')],\n",
       "  [('node', 'have', 'neighbors')],\n",
       "  [('inset', 'present', 'their correlation coeﬃcient')],\n",
       "  [('our method', 'overestimate', \"nodes ' ceptibility\")],\n",
       "  [('theoretical predictions',\n",
       "    'be highly correlate with',\n",
       "    'simulated susceptibility'),\n",
       "   ('theoretical predictions', 'be', 'highly correlated')],\n",
       "  [('slope k', 'however vary as', 'βS increases')],\n",
       "  [('phenomenon', 'be also reﬂected in', 'd')],\n",
       "  [('we', 'display', 'residfor diﬀerent βS')],\n",
       "  [('it',\n",
       "    'discern',\n",
       "    'highly susceptible individuals with topological indicators'),\n",
       "   ('susceptibility', 'be with', 'standard set of equations spreading'),\n",
       "   ('susceptible individuals', 'be with', 'topological indicators')],\n",
       "  [('important topological measures', 'be in', 'network theory')],\n",
       "  [('degree k', 'number of', 'connections')],\n",
       "  [('degree', 'however only reﬂects', \"nodes ' number of neighbors\"),\n",
       "   ('degree', 'however only reﬂects', \"nodes ' number\")],\n",
       "  [(\"people 's global location\", 'be in', 'networks'),\n",
       "   ('people', 'have', 'global location in networks')],\n",
       "  [('1 left', 'be in', 'network'), ('nodes', 'be with', 'degree k')],\n",
       "  [('nodes', 'be with', 'large betweenness centrality')],\n",
       "  [('nodes', 'neighbor be', 'PageRank')],\n",
       "  [('PageRank', 'be in', 'network')],\n",
       "  [('individuals', 'be with', 'given combination of k')],\n",
       "  [('nodes', 'of collection be', 'number in set Υ'),\n",
       "   ('collection', 'number in', 'set Υ'),\n",
       "   ('collection', 'number of', 'nodes'),\n",
       "   ('number', 'be in', 'set Υ')],\n",
       "  [('S', 'of value be', 'base 10'),\n",
       "   ('We', 'j for', 'Enron'),\n",
       "   ('Facebook', 'rewire respectively', 'Enron social networks'),\n",
       "   ('Facebook', 'rewire', 'Enron social networks')],\n",
       "  [('nodes', 'be with', 'ﬁxed degree k')],\n",
       "  [('k shell index', 'reﬂect', \"nodes ' susceptibility than degree k\"),\n",
       "   ('k shell index', 'better reﬂect', \"nodes ' susceptibility than degree k\")],\n",
       "  [('nodes', 'be with', 'combined k-shell kS'),\n",
       "   ('nodes', 'for susceptibility be', 'base 10')],\n",
       "  [('Values', 'be', 'indicated')],\n",
       "  [('diﬀerent fraction f', 'for r be', 'f')],\n",
       "  [('infection rate βS', 'be in', 'SIR simulations')],\n",
       "  [('k shell index kS', 'predict', 'average susceptibility of individuals')],\n",
       "  [('our results', 'be in', 'original networks'),\n",
       "   ('original networks', 'be in', 'Fig. 4')],\n",
       "  [('nodes', 'be with', 'same topological measures'),\n",
       "   ('it', 'explore', 'their accuracy')],\n",
       "  [('number', 'be in', 'set')],\n",
       "  [('Facebook', 'rewire', 'Enron networks')],\n",
       "  [('shell index kS', 'be in', 'most cases')],\n",
       "  [('nodes', 'be with', 'extremely large degree')],\n",
       "  [('we', 'pick', 'high-degree nodes')],\n",
       "  [('the', 'still identify', 'more highly susceptible individuals')],\n",
       "  [('phenomenon', 'depend on', 'exact topological structure of networks')],\n",
       "  [('kS',\n",
       "    'be',\n",
       "    'So in real scenario still eﬀective predictor for highly susceptible individuals'),\n",
       "   ('kS', 'be', 'eﬀective'),\n",
       "   ('eﬀective predictor', 'be in', 'real scenario')],\n",
       "  [('relationship', 'be in', 'Fig. 6')],\n",
       "  [('Inﬂuence Mi',\n",
       "    'be deﬁned as',\n",
       "    'average size of population infected in spreading process originating at node i')],\n",
       "  [('it', 'locate', 'individuals'),\n",
       "   ('it', 'locate', 'highly susceptible individuals')],\n",
       "  [('shell', 'be in', 'fact')],\n",
       "  [('βR', 'be', 'set')],\n",
       "  [('we', 'show in', 'b')],\n",
       "  [('susceptibility', 'be in', 'rumor spreading model'),\n",
       "   ('it',\n",
       "    'ﬁnd',\n",
       "    'topological indicator of susceptibility in rumor spreading model')],\n",
       "  [('recognition rate', 'be in', 'Fig. 7')],\n",
       "  [('diﬀerent fraction f', 'for r be', 'f'),\n",
       "   ('epidemic', 'start from', 'source')],\n",
       "  [('infection rate βR', 'be in', 'rumor spreading simulations')],\n",
       "  [('it', 'spread', 'information')],\n",
       "  [('nodes', 'be with', 'high k')],\n",
       "  [('Barab', 'asi be', 'Nature 406 378')],\n",
       "  [('NumericCitation J. Leskovec',\n",
       "    'advance in',\n",
       "    'neural information processing systems 539'),\n",
       "   ('Advances', 'be in', 'neural information processing systems 539')]],\n",
       " 16: [[('we', 'use', 'heuristic arguments')],\n",
       "  [('We', 'verify', 'scaling predictions'),\n",
       "   ('We',\n",
       "    'discuss',\n",
       "    'consequences of scaling laws for near-threshold epidemic outbreaks')],\n",
       "  [('Theoretical models', 'describe', 'spread of infectious diseases')],\n",
       "  [('convenient deterministic framework', 'model', 'spread of epidemic')],\n",
       "  [('infection', 'can take oﬀ above', 'threshold')],\n",
       "  [('We', 'begin with', 'deﬁnition of infection process given in Section II')],\n",
       "  [('we', 'use', 'rate equation approach')],\n",
       "  [('we', 'first derive', 'outbreak size distribution')],\n",
       "  [('results', 'be in', 'section V')],\n",
       "  [('We',\n",
       "    'investigate',\n",
       "    'continuous-time version of SIR infection process NumericCitation')],\n",
       "  [('r', 'recover', 'individuals')],\n",
       "  [('α', 'be', 'where infection rate')],\n",
       "  [('3 behavior', 'be in', 'limit')],\n",
       "  [],\n",
       "  [('size outbreak', 'diverge in', 'vicinity of threshold')],\n",
       "  [('powers', 'in g be', 'z'),\n",
       "   ('we', 'ﬁnd', 'Gn'),\n",
       "   ('generating function G', 'be in', 'powers of z')],\n",
       "  [('exponential', 'of product be', 'Gn =')],\n",
       "  [('size distribution', 'have', '10'),\n",
       "   ('n →', 'as ≃ be', '4π'),\n",
       "   ('n →', 'as ≃ be', '10')],\n",
       "  [('α', '=', '1')],\n",
       "  [('distribution function', 'satisﬁes', 'closed evolution equation')],\n",
       "  [('N → ∞', 'population of', 'infected individuals')],\n",
       "  [('discrete diﬀusion equation',\n",
       "    'be with',\n",
       "    'diﬀusion coeﬃcient equal to size of infected population'),\n",
       "   ('equation', 'be', 'discrete')],\n",
       "  [('two ordinary diﬀerential equations', 'to equation be', '11')],\n",
       "  [('we', 'obtain', 'ansatz'), ('0', 'for t be', '12')],\n",
       "  [('size distribution', 'be', 'Thus purely exponential')],\n",
       "  [('we', 'obtain', 't')],\n",
       "  [('Pi iPi', '=', '1')],\n",
       "  [('= i/t', 'keep', 'ﬁxed')],\n",
       "  [('recovered population',\n",
       "    'be',\n",
       "    'In contrast coupled to infected population')],\n",
       "  [('second gain term', 'be in', 'rate equation')],\n",
       "  [('P =', 'be in', 'terms of original variables')],\n",
       "  [('we', 'write', '21')],\n",
       "  [('we', 'substitute', 'α')],\n",
       "  [('we', 'first analyze', 't')],\n",
       "  [('r', 'recover', 'individuals'),\n",
       "   ('r', 'recover individual at', 'time t. Of course'),\n",
       "   ('deﬁnition', 'by probability be', 'Πr =')],\n",
       "  [('We', 'have', 't'), ('t → ∞', 'as hr be', 't')],\n",
       "  [('u', '=', '1')],\n",
       "  [('limit', 'in function be', '23'),\n",
       "   ('generating function', 'be in', 'limit')],\n",
       "  [('algebraic behavior', 'be in', 'small-η limit')],\n",
       "  [('we', 'obtain', 'behavior')],\n",
       "  [('quantity Pr Πr', 'be indeed ﬁnite with', '27')],\n",
       "  [('ﬁrst term', 'be in', 'series'),\n",
       "   ('exponential behavior', 'be in', 'large-η limit')],\n",
       "  [('joint distribution function', 'adhere to', '1'),\n",
       "   ('single-population distributions', 'for behavior be', '14')],\n",
       "  [('RR dη dξ Φ', '=', '1')],\n",
       "  [('we', 'take', 'limits')],\n",
       "  [('One', 'can verify', 'that')],\n",
       "  [('Eq', 'in φ be', 'ξ'), ('tion Φ', 'be in', 'Eq')],\n",
       "  [('we', 'have', 'Z dη')],\n",
       "  [('Taylor series', 'be in', 'conjugate variables')],\n",
       "  [('particular', 'in ihηi be', '36')],\n",
       "  [('above analysis', 'be repeat for', 'inactive outbreaks')],\n",
       "  [('large-η limit', 'in exp be', '− π2 η'),\n",
       "   ('˜φ 2π2 exp', 'be in', 'large-η limit'),\n",
       "   ('coeﬃcient', 'exp in', 'large-η limit'),\n",
       "   ('leading asymptotic behavior', 'be in', 'small-η limit')],\n",
       "  [('epidemic threshold', 'below 1 be', 'α >')],\n",
       "  [('data', 'correspond to', 'average over 108 realizations')],\n",
       "  [('average duration', 'grow logarithmically with', 'Fig. 2')],\n",
       "  [('s', '− with', 'probability Pi'),\n",
       "   ('probability Pi', 'be with', 'probability Pr')],\n",
       "  [('109 independent runs', 'be in', 'populations as large as N =')],\n",
       "  [('Outbreaks', 'be in', 'vicinity of epidemic threshold')],\n",
       "  [('scaling behavior', 'apply in', 'vicinity of epidemic threshold')],\n",
       "  [('epidemic threshold',\n",
       "    'be in',\n",
       "    'numerical simulations of infection processes on complex networks')],\n",
       "  [('determination', 'be', 'challenging')]],\n",
       " 17: [[('behavior', 'be with', 'those obtained from mean-ﬁeld approximation')],\n",
       "  [('higher order moments', 'be in', 'terms of ﬁrst')],\n",
       "  [('PA', 'give', 'consistently better results than MFA')],\n",
       "  [('improvement', 'be even qualitative in', 'one dimension')],\n",
       "  [('This', 'successional dynamics in', 'ecology NumericCitation'),\n",
       "   ('successional dynamics', 'be in', 'ecology NumericCitation')],\n",
       "  [('one', 'describe', 'time evolution of diﬀerent subpopulations')],\n",
       "  [('systems', 'be in', 'study')],\n",
       "  [('present work', 'fall in', 'latter category')],\n",
       "  [('results', 'be with', 'those of stochastic SIS model'),\n",
       "   ('PA', 'give', 'quantitative improvement over MFA')],\n",
       "  [('we', 'here consider', 'general SIRS model')],\n",
       "  [('better agreement', 'be with', 'simulations')],\n",
       "  [('rate δ', 's at', 'rate γ')],\n",
       "  [('SIS model', 'also know as', 'contact process')],\n",
       "  [('δ', '=', '1')],\n",
       "  [('high accuracy', 'be in', 'd =')],\n",
       "  [('behavior', 'be in', 'higher dimension'), ('behavior', 'be in', 'one')],\n",
       "  [('simulation', 'make', 'approximations')],\n",
       "  [('single site probabilities', 'be in', 'stochastic SIRS epidemic process')],\n",
       "  [('site x', 'of neighborhood be', 'nearest neighbor sites')],\n",
       "  [('PA', 'be', 'such schemes')],\n",
       "  [('we', 'have describe in', 'Eq')],\n",
       "  [('joint probability', 'β for', 'derivation of Eq'),\n",
       "   ('joint probability', 'β at', 'site y'),\n",
       "   ('joint probability', 'β at', 'site w'),\n",
       "   ('joint probability', 'β at', 'time t.')],\n",
       "  [('approximation', 'be in', 'Eq')],\n",
       "  [('one', 'get analytically', 'steady state solutions')],\n",
       "  [('one', 'solve numerically', 'resulting diﬀerential equations')],\n",
       "  [('site x', 'be in', 'Eqs')],\n",
       "  [('site', 'be in', 'd-dimensional cubic latz −')],\n",
       "  [('we', 'always have', 'α')],\n",
       "  [('joint probability Pt', 'be in', 'Eq')],\n",
       "  [('γ → ∞', 'for dpt be', '7c'),\n",
       "   ('SIRS NumericCitation', 'dpt for', 'γ → ∞')],\n",
       "  [('t → ∞', 'as pt be', 'I')],\n",
       "  [('infection', 'be', 'endemic')],\n",
       "  [('∞', 'point of', 'PA-SIS epidemic process')],\n",
       "  [('PA',\n",
       "    'correctly represent',\n",
       "    'results of stochastic SIRS process for small γ')],\n",
       "  [('PA', 'be with', 'results from stochastic SIRS process')],\n",
       "  [('PA',\n",
       "    'clearly give',\n",
       "    'results closer to those obtained from stochastic model')],\n",
       "  [('infected sites', 'be in', 'stochastic model')],\n",
       "  [('behavior', 'be in', 'Figs. 3')],\n",
       "  [('clustering eﬀect',\n",
       "    'be also observe in',\n",
       "    'stochastic SIS process NumericCitation')],\n",
       "  [('It', 'be more pronounced in', 'one dimension'),\n",
       "   ('It', 'be pronounced in', 'one dimension')],\n",
       "  [('Routh-Hurwitz conditions',\n",
       "    'obtain',\n",
       "    'sign of real part of eigenvalues of Jacobian')],\n",
       "  [('zero steady state solution', 'be stable for', 'γ')],\n",
       "  [('non-zero imaginary parts', 'be in', 'regions of parameter space')],\n",
       "  [('PA-SIRS system', 'be in', 'Eq')],\n",
       "  [('Such oscillations', 'be', 'seen')],\n",
       "  [('evolution', 'average', 'over 103 realizations of SIRS process')],\n",
       "  [('we', 'solve', 'Eq')],\n",
       "  [('results', 'be in', 'Figs. 7')],\n",
       "  [('graph', 'of slope be', 'death rate')],\n",
       "  [('growth', 'be in', 'MFA')],\n",
       "  [('9 susceptible population', 'be', 'Once reduced')],\n",
       "  [('damped oscillatory pattern', 'be in', 'Figs. 7')],\n",
       "  [('infected population', 'grow more slowly in', 'stochastic model')],\n",
       "  [('clustering', 'be in', 'stochastic model')],\n",
       "  [('results', 'be with', 'those obtained from deterministic MFA')],\n",
       "  [('higher order moments', 'be in', 'terms of lower order ones')],\n",
       "  [('damped oscillatory behavior', 'be in', 'parameter ranges')]],\n",
       " 18: [[(\"1 Instituto de F'ısica\",\n",
       "    'point of',\n",
       "    'stochastic asynchronous Susceptible-Infected-Recovered model on square lattice')],\n",
       "  [('fundamental system', 'be in', 'epidemiological modeling'),\n",
       "   ('model', 'be', 'fundamental')],\n",
       "  [('Susceptible individuals',\n",
       "    'become',\n",
       "    'infected at given rate through contact with infected individuals'),\n",
       "   ('contact', 'be with', 'infected individuals')],\n",
       "  [('Infected individuals', 'become', 'immune'),\n",
       "   ('individuals', 'become', 'immune')],\n",
       "  [('model', 'be', 'special'), ('model', 'also consider by', 'Antal et al.')],\n",
       "  [('It', 'be', 'special')],\n",
       "  [('This', 'be', 'continuous')],\n",
       "  [('system', 'correspond to', 'epidemic threshold')],\n",
       "  [('stochastic models', 'be with', 'synchronous update NumericCitation'),\n",
       "   ('numerous studies',\n",
       "    'be in',\n",
       "    'stochastic models with synchronous update NumericCitation')],\n",
       "  [('has great deal', 'be in', 'recent years'),\n",
       "   ('interest', 'be in', 'SIR model on networks'),\n",
       "   ('there', 'be', 'great')],\n",
       "  [('spread', 'be in', 'plants')],\n",
       "  [('effect', 'be in', 'infectious period')],\n",
       "  [('correlations', 'be in', 'transmission of infection')],\n",
       "  [('we', 'precisely determine', 'critical point of model')],\n",
       "  [('we', 'give', 'our conclusions')],\n",
       "  [('stochastic lattice-gas model', 'be with', 'asynchronous dynamics')],\n",
       "  [('following rules', 'be', 'applied')],\n",
       "  [('neighboring site', 'be choose at', 'random')],\n",
       "  [('it', 'otherwise remain', 'unchanged')],\n",
       "  [('site', 'be in', 'state R')],\n",
       "  [('nearest-neighbor I', 'be with', 'probability c'), ('I', '→', 'R nI')],\n",
       "  [('η1', 'η of', 'state')],\n",
       "  [('state', 'of permutation be', '0 → 2 → 1')],\n",
       "  [('connection', 'be with', 'approach de-present veloped by Kermack')],\n",
       "  [('d dt ρ1', '=', '− β ρ1ρ2')],\n",
       "  [('disease', 'be with', 'immunization')],\n",
       "  [('one', 'analyze', 'epidemic')],\n",
       "  [('ns', 'be', 'where number of clusters of size s per site on lattice')],\n",
       "  [('∼ s', '− τ so', 'Ps ∼ s1 − τ'), ('∼ s', '− τ so that', 'Ps ∼ s1 − τ')],\n",
       "  [('NumericCitation P ≥ s', '∼', '8')],\n",
       "  [('we', 'have in', '2-d percolation'), ('σ', '=', '36')],\n",
       "  [('asynchronous SIR model', 'follow', 'procedure')],\n",
       "  [('list', 'be in', 'ﬁrst step')],\n",
       "  [('New I sites', 'be in', 'third step')],\n",
       "  [('unoccupied nearest neighbor', 'be with', 'given bond probability'),\n",
       "   ('active growth site',\n",
       "    'spread to',\n",
       "    'unoccupied nearest neighbor with given bond probability')],\n",
       "  [('spreading', 'be in', 'SIR model'), ('correlation', 'be in', 'spreading')],\n",
       "  [('it', 'know transition point to', 'high accuracy'),\n",
       "   ('properties', 'be in', 'detail')],\n",
       "  [('number', 'be in', 'parentheses')],\n",
       "  [('agreement', 'be with', 'previous central value')],\n",
       "  [('error', 'be', 'where based upon statistical error of data')],\n",
       "  [('critical behavior',\n",
       "    'ﬁnd',\n",
       "    'slope − 0.05524 consistent with percolation scaling'),\n",
       "   ('linear behavior', 'be in', 'Fig. 1 for larger s')],\n",
       "  [('I', '→ r with', '1 − c')],\n",
       "  [('it', 'use', 'list of eligible S sites')],\n",
       "  [('bounds', '≈', '0.592746')],\n",
       "  [('epidemic bond percolation growth', 'be in', 'SIR language')],\n",
       "  [('As algorithm', 'be in', 'section III')],\n",
       "  [('we', 'follow for', 'percolation')],\n",
       "  [('wetted sites', 'be with', 'correct probability'),\n",
       "   ('I site', 'have', 'four nearest-neighbor sites')],\n",
       "  [('similar algorithm', 'be give by', 'Grassberger NumericCitation')],\n",
       "  [],\n",
       "  [('we', 'have', '4')],\n",
       "  [('it', 'remain', 'infectious')],\n",
       "  [('I', 'recover on', 'average')],\n",
       "  [('it', 'recover in', 'given site')],\n",
       "  [('y', 'represent', 'ln P ≥ s'),\n",
       "   ('Plot simulated', 'be with', 'cutoﬀ of equation')],\n",
       "  [('s', '=', '2097152')],\n",
       "  [('Bethe lattice', 'be with', 'coordination number')],\n",
       "  [('z', '=', '3')],\n",
       "  [('we', 'find for', 'square lattice')],\n",
       "  [('than regular lattice', 'be with', 'same z')],\n",
       "  [('derivation', 'be give in', 'Appendix')],\n",
       "  [('28-29', 'be give in', 'Table I')],\n",
       "  [('distribution',\n",
       "    'be much diﬀerent from',\n",
       "    'binomial distribution of standard percolation Eq')],\n",
       "  [('increase', 'be with', 'increasing k for given m'),\n",
       "   ('increase', 'be in', 'probability')],\n",
       "  [('outgoing probabilities',\n",
       "    'be in',\n",
       "    'construction of Kuulasmaa NumericCitation')],\n",
       "  [('enhancement', 'be in', 'number of neighy =')],\n",
       "  [('correlations', 'be in', 'infected neighbors'),\n",
       "   ('infected neighbors', 'be in', 'SIR model')],\n",
       "  [('individuals', 'be', 'where are continuously infected')],\n",
       "  [('NumericCitation', 'be with', 'creation rate λ')],\n",
       "  [('I site', 'be in', 'SIS model'), ('particle', 'be in', 'CP model')],\n",
       "  [('us', 'consider', 'periods of time spent by individual in R state')],\n",
       "  [('individuals', 'have', 'partial immunization'),\n",
       "   ('ﬁnite', 'be in', 'SIRS model T')],\n",
       "  [('SIRS', 'be in', 'which')],\n",
       "  [('SIRS model', 'be in', 'which')],\n",
       "  [('that', 'suppress', 'a')],\n",
       "  [('can infected multiple times', 'be in', 'SIS model')],\n",
       "  [('asynchronous SIR model',\n",
       "    'be in',\n",
       "    'universality class of standard percolation through behavior of cluster size distribution')],\n",
       "  [('local correlations', 'diﬀer from', 'those of standard percolation')],\n",
       "  [('transition', 'be in', 'SIR model deﬁned on square lattice')],\n",
       "  [('square lattice', 'on process be', 'model')],\n",
       "  [('heterogeneity', 'be in', 'square-lattice SIR model')],\n",
       "  [('More details', 'will', 'will given in future publication')],\n",
       "  [('lattice-based SIR model', 'be', 'where relevant')]],\n",
       " 19: [[('adaptive networks Leah B. Shaw Department',\n",
       "    'on epidemic be',\n",
       "    'Col lege of Wil liam'),\n",
       "   ('Mary', 'model for', 'epidemics on adaptive network'),\n",
       "   ('Fluctuating epidemics', 'lege of', 'Wil liam')],\n",
       "  [('Connections', 'break', 'links from non-infected nodes to infected nodes'),\n",
       "   ('Connections', 'break', 'links from non-infected nodes'),\n",
       "   ('Connections', 'be', 'reformed')],\n",
       "  [('ﬂuctuations', 'exhibit', 'power law behavior')],\n",
       "  [('many models', 'have', 'have considered')],\n",
       "  [('mechanisms', 'be in', 'spread of infectious diseases')],\n",
       "  [('applied changes', 'be in', 'structure'),\n",
       "   ('models', 'be with', 'externally applied changes in structure')],\n",
       "  [('response', 'be in', 'turn')],\n",
       "  [('static networks', 'be in', 'large population')],\n",
       "  [('We', 'present', 'its bifurcation structure')],\n",
       "  [('dynamical properties', 'be in', 'Section V. II')],\n",
       "  [('we', 'deﬁne as', 're-susceptibility rate')],\n",
       "  [('it', 'examine', 'state solutions'),\n",
       "   ('it', 'examine', 'steady state solutions')],\n",
       "  [('Larger system sizes', 'be with', 'same node')],\n",
       "  [('Initial conditions', 'be construct in', 'one of two ways'),\n",
       "   ('Initial conditions', 'be', 'constructed')],\n",
       "  [('node', 'be in', 'state B'), ('node', 'be in', 'state A')],\n",
       "  [('K', 'be with', 'rate p along susceptible-infected link'),\n",
       "   ('2 ˙PR =', 'be in', 'ﬁrst equation')],\n",
       "  [('We', 'next write', 'system of mean ﬁeld equations for links')],\n",
       "  [('we', 'close', 'system')],\n",
       "  [('we', 'be', 'endemic')],\n",
       "  [('solutions', 'be with', 'negative values')],\n",
       "  [('We', 'generate', 'stochastic stimulations of mean ﬁeld')],\n",
       "  [('steady states', 'use', 'continuation package NumericCitation')],\n",
       "  [('rewiring', 'be', 'When introduced')],\n",
       "  [('states', 'be', 'observed')],\n",
       "  [('Bistability', 'be observe previously for', 'SIS model NumericCitation')],\n",
       "  [('Monte Carlo simulation points', 'be in', 'Fig. 1')],\n",
       "  [('we', 'locate', 'endemic state')],\n",
       "  [('steady state', 'be average over', '104 MCS')],\n",
       "  [('we', 'locate', 'disease free state')],\n",
       "  [('system', 'be simulated for', '2 × 104 MCS')],\n",
       "  [('more detail', 'be in', 'Section V')],\n",
       "  [('results', 'be in', 'Figure 1 show')],\n",
       "  [('r', '=', '0.002'), ('q', '=', '0.0016')],\n",
       "  [('we', 'use', 'mean ﬁeld model')],\n",
       "  [('They', 'will', 'will treated elsewhere'),\n",
       "   ('They', 'will', 'will treated')],\n",
       "  [('q', '=', '0.0064'), ('we', 'depict in', 'Fig. 2b')],\n",
       "  [('q', '=', '0.0064')],\n",
       "  [('saddle-node points', 'of line be', 'limit points')],\n",
       "  [('light dashed line', 'denote', 'BP')],\n",
       "  [('w', '=', '0.04')],\n",
       "  [('Dashed lines', 'be', 'branches'),\n",
       "   ('Dashed lines', 'be', 'unstable branches')],\n",
       "  [('stable', 'be for', 'low values of p disease free steady state')],\n",
       "  [('p', 'branch of', 'endemic steady states'),\n",
       "   ('endemic steady states', 'of branch be', 'subcritical')],\n",
       "  [('clear region', 'be with', 'co-existing p')],\n",
       "  [('result', 'be in', 'Fig. 2a')],\n",
       "  [('We', 'describe', 'bifurcation regions in detail for w larger than 0.1'),\n",
       "   ('bifurcation regions', 'be in', 'detail')],\n",
       "  [('we', 'cross into', 'region II for large w')],\n",
       "  [('q', 'be lower to', '0.0016')],\n",
       "  [('it', 'pass through', 'region II')],\n",
       "  [('Hopf curve', 'be actually closed isola in', 'two parameters'),\n",
       "   ('closed isola', 'be in', 'two parameters'),\n",
       "   ('curve', 'be', 'closed')],\n",
       "  [('point', 'be', 'saddle-saddle')],\n",
       "  [('bifurcation', 'be in', 'Hopf bifurcation for q =')],\n",
       "  [('we', 'observe in', 'Monte Carlo simulations of full system'),\n",
       "   ('our discussion', 'be in', 'Section V'),\n",
       "   ('our discussion', 'be in', 'endemic state')],\n",
       "  [('Rewiring', 'lead to', 'signiﬁcant alterations in network structure'),\n",
       "   ('signiﬁcant alterations', 'be in', 'network structure')],\n",
       "  [('degree distributions', 'be in', 'of Fig. 4b')],\n",
       "  [('we', 'obtain', 'better statistics'), ('we', 'obtain', 'statistics')],\n",
       "  [('we', 'use', 'mean ﬁeld ideas'), ('we', 'use', 'mean ideas')],\n",
       "  [('We', 'outline here', 'calculation brieﬂy')],\n",
       "  [('number', 'be in', 'state X'), ('they', 'become again', 'susectible')],\n",
       "  [('heavy dashed line', 'be line of', 'limit points')],\n",
       "  [('light dashed line', 'denote', 'transcritical bifurcations')],\n",
       "  [('same bifurcation diagram', 'be in', 'Fig. 1')],\n",
       "  [('10', 'can', 'can solved for degree distribution of recovereds')],\n",
       "  [('node types', 'have', 'Poissonian degree distribution')],\n",
       "  [('they', 'infect by', 'neighbor')],\n",
       "  [('n − k ′', '− for', 'n')],\n",
       "  [('actual distributions', 'be in', 'Fig. 4b')],\n",
       "  [('curves', 'be in', 'Fig. 4b'), ('they', 'be', 'indistinguishable')],\n",
       "  [('rdI', 'n for', 'n')],\n",
       "  [('infectives', 'be with', 'distribution predicted by Eq')],\n",
       "  [('mean ﬁeld approximation', 'be in', 'Eq')],\n",
       "  [('w', '=', '0.04'), ('r', '=', '0.002'), ('q', '=', '0.0016')],\n",
       "  [('distribution', 'be in', 'presence of rewiring')],\n",
       "  [('Results', 'be average over', '3 × 104 MCS')],\n",
       "  [('decay', 'be in', 'distribution of distances'),\n",
       "   ('network', 'be', 'when rewired')],\n",
       "  [('we', 'observe in', 'average of Monte Carlo simulations'),\n",
       "   ('R nodes', 'be in', 'numbers matching')],\n",
       "  [('recovereds', 'be with', 'degree 0')],\n",
       "  [('7 V. FLUCTUATIONS', 'be in', 'previous sections'),\n",
       "   ('we', 'have consider', 'long time averages')],\n",
       "  [('A. Fluctuations', 'number of', 'infectives'),\n",
       "   ('bifurcation point', 'near fluctuation be', 'number of infectives')],\n",
       "  [('Fluctuations', 'be in', 'SIRS model')],\n",
       "  [('long time series', 'be in', 'Monte Carlo')],\n",
       "  [('time series', 'be longer than', '105 MCS')],\n",
       "  [('equilibrium', 'be in', 'stochastic form'),\n",
       "   ('mean ﬁeld equations',\n",
       "    'can',\n",
       "    'For comparison can also considered near equilibrium in stochastic form')],\n",
       "  [('multiplicative noise eﬀects',\n",
       "    'generate',\n",
       "    'similar results to those reported for additive noise'),\n",
       "   ('multiplicative noise eﬀects', 'generate', 'similar results')],\n",
       "  [('mean ﬁeld system', 'be in', 'Eqs')],\n",
       "  [('noise',\n",
       "    'be in',\n",
       "    'simulations of stochastic attractors near endemic state')],\n",
       "  [('bistability regions III',\n",
       "    'be vastly diﬀerent depend',\n",
       "    'depending value of re-susceptibility rate q')],\n",
       "  [('it', 'be', 'true'), ('w', '=', '0.04'), ('it', 'be', 'saddle-node')],\n",
       "  [('saddle-node point', 'be in', 'question')],\n",
       "  [('endemic state', 'lose', 'stability')],\n",
       "  [('we', 'motivate', 'power law scaling of ﬂuctuations'),\n",
       "   ('we', 'scale near', 'generic saddle-node bifurcation')],\n",
       "  [('one', 'consider', 'generic problem of saddle-node bifurcation')],\n",
       "  [('Fluctuations', 'be in', 'infectives')],\n",
       "  [('Curves', 'guide', 'eye')],\n",
       "  [('data points', 'be with', 'best ﬁt lines')],\n",
       "  [('We',\n",
       "    'next consider',\n",
       "    'phase relationships between ﬂuctuating variables')],\n",
       "  [('number', 'be in', 'system')],\n",
       "  [('ﬂuctuations', 'be in', 'number of infective neighbors'),\n",
       "   ('ﬂuctuations', 'be in', 'number of infectives lagged behind ﬂuctuations')],\n",
       "  [('we', 'study', 'its dependence on rewiring rate')],\n",
       "  [('Monte Carlo simulations', 'discard', 'transients')],\n",
       "  [('mean ﬁeld equations', 'be with', 'noise strength ǫ =')],\n",
       "  [('rewiring', 'show in', 'Fig. 9b')],\n",
       "  [('time', 'lag for', 'mean ﬁeld model with multiplicative noise'),\n",
       "   ('mean ﬁeld model', 'be with', 'multiplicative noise'),\n",
       "   ('same trend', 'be with', 'larger rewiring'),\n",
       "   ('We',\n",
       "    'find',\n",
       "    'same trend of increasingly delayed outbreaks with larger rewiring')],\n",
       "  [('endemic steady state',\n",
       "    'lifetime of be',\n",
       "    'lifetime of endemic steady state')],\n",
       "  [('Curves', 'be', 'scaled in arbitrary units for comparison of peak times'),\n",
       "   ('Curves', 'be scale for', 'comparison of peak times')],\n",
       "  [('r', '=', '0.002'), ('w', '=', '0.09'), ('q', '=', '0.0016')],\n",
       "  [('Time', 'be in', 'MCS')],\n",
       "  [('r', '=', '0.002'),\n",
       "   ('ﬁeld noise strength ǫ', '=', '0.0001'),\n",
       "   ('noise strength ǫ', '=', '0.0001'),\n",
       "   ('q', '=', '0.0016')],\n",
       "  [('w', '=', '0.04'), ('r', '=', '0.002')],\n",
       "  [('generic saddle-node bifurcation', 'be in', 'one dimension'),\n",
       "   ('T', 'be', 'linear')],\n",
       "  [('we', 'obtain', 'good statistics near bifurcation point'),\n",
       "   ('our system', 'be with', '104 nodes')],\n",
       "  [('system', 'behave like', 'focus')],\n",
       "  [('oscillations', 'could', 'could ignored')],\n",
       "  [('scaling results', 'be show in', 'Figure 10'),\n",
       "   ('Preliminary scaling results', 'be show in', 'Figure 10')],\n",
       "  [('network', 'be with', 'adaptive rewiring')],\n",
       "  [('As SIS model', 'study previously by', 'Gross et al.')],\n",
       "  [('must appropriate type', 'be in', 'mean ﬁeld')],\n",
       "  [('we', 'manipulate', 'location of bifurcation points')],\n",
       "  [('ﬂuctuations', 'be in', 'infectives near bifurcation point')],\n",
       "  [('We', 'study', 'eﬀects of rewiring on network geometry')],\n",
       "  [('we', 'have omit here', 'discussion of analytical results'),\n",
       "   ('we', 'have omit here', 'discussion')],\n",
       "  [('Delayed outbreaks', 'be observe in', 'rewired system')],\n",
       "  [('peaks', 'be in', 'number of nodes'),\n",
       "   ('Peaks', 'be in', 'infective fraction')],\n",
       "  [('regime', 'study in', 'Monte Carlo simulations')],\n",
       "  [('it', 'lose', 'stability')],\n",
       "  [('Such extinction regimes',\n",
       "    'use',\n",
       "    'Fokker-Planck approach NumericCitation')],\n",
       "  [('agreement', 'be with', 'expected scaling')],\n",
       "  [('Real social networks', 'have', 'community structure NumericCitation'),\n",
       "   ('Real networks', 'have', 'community structure NumericCitation')],\n",
       "  [('Rewiring', 'be', 'non-local'),\n",
       "   ('new neighbors', 'introduce', 'local structure')],\n",
       "  [('impact', 'be in', 'adaptive networks')],\n",
       "  [('non-infected nodes', 'be', 'where assumed')],\n",
       "  [('organized populations', 'be with', 'local structure'),\n",
       "   ('contact tracing control',\n",
       "    'be in',\n",
       "    'organized populations with local structure')]],\n",
       " 20: [[('contact networks', 'be with', 'strongly-connected topologies')],\n",
       "  [('we', 'present alternative proof for', 'some of its properties')],\n",
       "  [('results', 'be analogous to', 'those well-known for scalar case')],\n",
       "  [('transmission', 'be in', 'communication networks'),\n",
       "   ('diﬀusion', 'be in', 'competitive economic networks'),\n",
       "   ('cascading failures', 'be in', 'power grids'),\n",
       "   ('spread', 'be in', 'contact networks'),\n",
       "   ('spreading', 'be in', 'forests')],\n",
       "  [('individuals', 'be in', 'population')],\n",
       "  [('Electrical Engineering', 'department at', 'Santa Barbara'),\n",
       "   ('Computation', 'university of', 'California'),\n",
       "   ('Department', 'be with', 'Center for Control'),\n",
       "   ('Information Engineering', 'university of', 'Padova'),\n",
       "   ('Electrical Engineering', 'department of', 'Information Engineering'),\n",
       "   ('Control', 'for center be', 'Shadi Mohagheghi')],\n",
       "  [('individuals', 'be in', 'network')],\n",
       "  [('propagation', 'be', 'stochastic'),\n",
       "   ('propagation',\n",
       "    'be',\n",
       "    'In general formulation stochastic process on complex network'),\n",
       "   ('stochastic process', 'be in', 'general formulation')],\n",
       "  [('Stochastic propagation processes',\n",
       "    'can',\n",
       "    'can modeled as Markov chains of exponential dimensions in size of network')],\n",
       "  [('parameters',\n",
       "    'estimate',\n",
       "    'states of such stochastic large-dimension phenomena')],\n",
       "  [('Three approaches',\n",
       "    'be commonly adopt in',\n",
       "    'analysis of propagation models')],\n",
       "  [('nodes', 'be with', 'same degree exhibit')],\n",
       "  [('global characteristics', 'can', 'can estimated statistically'),\n",
       "   ('global characteristics', 'can', 'can estimated')],\n",
       "  [('approach', 'result in', 'deterministic network model')],\n",
       "  [('network', 'of', 'adjacency matrix')],\n",
       "  [('well-established theorems', 'be in', 'matrix analysis')],\n",
       "  [('we', 'characterize', 'their properties'),\n",
       "   ('we', 'characterize', 'their dynamical properties')],\n",
       "  [('basic reproduction number R0',\n",
       "    'be',\n",
       "    'eﬀective reproduction number in fully-healthy susceptible population'),\n",
       "   ('eﬀective reproduction number',\n",
       "    'be in',\n",
       "    'fully-healthy susceptible population')],\n",
       "  [('we', 'focus', 'our review on deterministic network models'),\n",
       "   ('we', 'focus', 'our review')],\n",
       "  [('work', 'analyze', \"system 's asymptotic behavior\"),\n",
       "   ('system', 'have', 'asymptotic behavior')],\n",
       "  [('basic reproduction number', 'be above', 'when epidemic threshold')],\n",
       "  [('we', 'refer to', 'model proposed by La jmanovich et al.')],\n",
       "  [('claim', 'be rigorously prove by', 'Sahneh et al.')],\n",
       "  [('NumericCitation use', 'be in', 'their analysis')],\n",
       "  [('general multi-group SIR model', 'be with', 'birth')],\n",
       "  [('threshold', 'be', 'characterized')],\n",
       "  [('system', 'converge asymptotically to', 'all-healthy state')],\n",
       "  [('generalized network SIR model', 'be with', 'vital dynamics')],\n",
       "  [('epidemic threshold',\n",
       "    'be',\n",
       "    'Through simulation-based analysis given as function of spectral radius of network')],\n",
       "  [('tutorial comprehensive treatment',\n",
       "    'be with',\n",
       "    'comprehensive statements for network SI')],\n",
       "  [('convergence', 'positivity of', 'infection probabilities'),\n",
       "   ('its', 'convergence', 'positivity of infection probabilities')],\n",
       "  [('results', 'be in', 'NumericCitation'),\n",
       "   ('We', 'present alternative proof for', 'them'),\n",
       "   ('We', 'present proof for', 'them')],\n",
       "  [('expansions', 'be in', 'limit of high infection rates')],\n",
       "  [],\n",
       "  [('distribution', 'be in', 'system')],\n",
       "  [('uniqueness', 'be', 'For iterative algorithm rigorously proved')],\n",
       "  [('comparison', 'be with', 'NumericCitation'),\n",
       "   ('our treatment',\n",
       "    'establish',\n",
       "    'numerous properties including novel characterization of epidemic threshold')],\n",
       "  [('our deterministic network models',\n",
       "    'be',\n",
       "    'derived from Markov-chain models through mean-ﬁeld approximation')],\n",
       "  [('Section 6', 'be', 'conclusion')],\n",
       "  [('individuals', 'be in', 'population')],\n",
       "  [('E', 'be set of', 'directed links')],\n",
       "  [('single individuals', 'be in', 'contact network')],\n",
       "  [('G', 'be', 'connected'), ('G', 'be', 'strongly connected')],\n",
       "  [('nodes', 'be', 'single individuals')],\n",
       "  [('nodes', 'be', 'populations')],\n",
       "  [('individuals', 'be in', 'population')],\n",
       "  [],\n",
       "  [('symbol φ', 'denote', 'empty set')],\n",
       "  [('entries', 'be', 'x1'), ('diagonal entries', 'be', 'x1')],\n",
       "  [('we', 'moreover let', 'A')],\n",
       "  [],\n",
       "  [('1 n umax', '=', '1')],\n",
       "  [('3 Susceptible-Infected Model', 'be in', 'section')],\n",
       "  [('its dynamical behavior', 'be', 'given by lemma below')],\n",
       "  [('β 0', 'with model be', '1'), ('scalar SI model', 'be with', 'β > 0')],\n",
       "  [('monotonical ly', 'be in', 'solution x'),\n",
       "   ('diﬀerent initial conditions', 'with equation be', '1'),\n",
       "   ('equation', 'be with', 'diﬀerent initial conditions')],\n",
       "  [('network SI model', 'be in', '− diag x'),\n",
       "   ('weighted digraph', 'be with', 'adjacency matrix A =')],\n",
       "  [('0 > 0n', '0n for', 'al l t'),\n",
       "   ('al l t >', 'for 0n be', 'then x 0n >'),\n",
       "   ('connected graph', 'be with', 'adjacency matrix A')],\n",
       "  [('equilibrium point 0n', 'about model be', '4')],\n",
       "  [('it', 'be', 'exponential')],\n",
       "  [('x', 'n for', 't')],\n",
       "  [('Nagumo', 'of', 'Theorem')],\n",
       "  [('t ∈', 'for x be', 't')],\n",
       "  [('¯ x', '=', '1n'), ('x', '=', '1n')],\n",
       "  [('¯ x =', 'be', 'equilibrium points')],\n",
       "  [('this', 'be', 'smooth')],\n",
       "  [('initial-times linear evolution satisﬁes', 'be in', 'Rn')],\n",
       "  [('exponential growth', 'be with', 'rate βλmax')],\n",
       "  [('ﬁnal infection speed', 'be proportional to', 'degree centrality')],\n",
       "  [('we', 'review', 'Susceptible-Infected-Susceptible SIS epidemic model'),\n",
       "   ('4 Susceptible-Infected-Susceptible model', 'be in', 'section')],\n",
       "  [('infection process', 'be with', 'rate β')],\n",
       "  [('infected x', 'be with', 'x'),\n",
       "   ('4.1 Scalar SIS model', 'be in', 'scalar SIS model'),\n",
       "   ('x', 'with x be', 't')],\n",
       "  [('SIS model', 'be with', 'β >'), ('β >', 'with model be', '7')],\n",
       "  [('β > γ',\n",
       "    'trajectory from',\n",
       "    'initial condition x > 0 converges to exponential ly stable equilibrium x ∗ =')],\n",
       "  [('basic reproduction number', 'be in', 'deterministic scalar SIS model')],\n",
       "  [('intertwined SIS model', 'be in', 'NumericCitation'),\n",
       "   ('4.2 Network SIS Model', 'be in', 'section')],\n",
       "  [('network SIS model', 'be with', 'infection rate β'),\n",
       "   ('˙x = β', 'be in', '− diag')],\n",
       "  [('Evolution',\n",
       "    'be with',\n",
       "    'varying initial fraction of infected individuals')],\n",
       "  [('Behavior', 'be in', 'subsection')],\n",
       "  [('connected digraph', 'be with', 'adjacency matrix A.')],\n",
       "  [('0 > 0n', '0n for', 'ii')],\n",
       "  [('− γ', 'be in', 'x')],\n",
       "  [('it', 'attribute theorem to', 'NumericCitation')],\n",
       "  [('Nagumo', 'of', 'Theorem')],\n",
       "  [('γ Ax ∗', '=', 'x ∗')],\n",
       "  [('x ∗', 'be', 'γ Ax'), ('∗', 'be', 'γ')],\n",
       "  [('it', 'be', 'ﬁxed')],\n",
       "  [('vmax ≤ vmax', 'be in', '− diag'),\n",
       "   ('vmax ≤ vmax', 'be in', 'real part'),\n",
       "   ('other eigenvalue', 'than maxx be', 't')],\n",
       "  [('ˆA', 'be stable', 'Schur')],\n",
       "  [('connected digraph', 'be with', 'adjacency matrix A.')],\n",
       "  [('d', '=', 'A1n')],\n",
       "  [('0 > 0n', '0n for', 'ii')],\n",
       "  [('β γ y', 'by rn be', 'k')],\n",
       "  [('0', 'be', 'scalar')],\n",
       "  [('Historically', 'existence of', 'unique endemic state')],\n",
       "  [('Taylor expansions', 'be in', 'parts'), ('iiib', 'be in', 'part')],\n",
       "  [('selected statements', 'be in', 'Theorem 5')],\n",
       "  [('one positive entry', 'be in', 'row')],\n",
       "  [('βλmax', 'suppose', '0')],\n",
       "  [('We', 'have γ', 'ˆA βA/γ'), ('We', 'have γ', 'ˆA = βA/γ')],\n",
       "  [('y', 'be', 'scalar'), ('line', 'be in', 'previous paragraph')],\n",
       "  [('equilibrium point', 'be with', 'entry equal to 0')],\n",
       "  [('y ∗', 'assume', 'y'), ('y ∗ j', '=', '0')],\n",
       "  [('G', 'be', 'connected')],\n",
       "  [('we', 'second assume by', 'contradiction')],\n",
       "  [('y ∗', 'be', 'equilibrium')],\n",
       "  [('x ∗', 'be inﬁnitesimal as', 'function of δ')],\n",
       "  [('12 5 Network Susceptible-Infected-Recovered Model', 'be in', 'section'),\n",
       "   ('we', 'review', 'Susceptible-Infected-Susceptible SIR epidemic model')],\n",
       "  [('5.1 Scalar SIR model', 'be in', 'model individuals')],\n",
       "  [('its dynamical behavior', 'be in', 'lemma below')],\n",
       "  [('al l t ≥ 0', 'for s be', 't')],\n",
       "  [('epidemic outbreak', 'growth of', 't → x'),\n",
       "   ('we', 'describe', 'behavior in statement'),\n",
       "   ('epidemic outbreak', 'growth for', 'small times'),\n",
       "   ('1 − log β', '+ s be', '0'),\n",
       "   ('behavior', 'be in', 'statement')],\n",
       "  [('eﬀective reproduction number',\n",
       "    'be in',\n",
       "    'deterministic scalar SIR model')],\n",
       "  [('graph', 'be with', 'adjacency matrix A'), ('iv', 'be in', 'Lemma 6')],\n",
       "  [('two curves', 'be in', 'equation'), ('intersection', 'be with', 's =')],\n",
       "  [('third equation', 'be redundant because of', 't')],\n",
       "  [('dynamical system', 'be in', 'vector form')],\n",
       "  [('We', 'state', 'our main novel results of section below')],\n",
       "  [('about s', '∗', '0n'), ('t ≥ τ', 'for x be', 't')],\n",
       "  [('βλmax', 't with', 'rate βλmax'),\n",
       "   ('weighted average t', 'be with', 'rate βλmax'),\n",
       "   ('smal l time', 'for βλmax be', 'weighted average t fast with rate βλmax')],\n",
       "  [('eﬀective reproduction number',\n",
       "    'be in',\n",
       "    'deterministic network SIR model')],\n",
       "  [('α', '=', 't2'), ('t1', 'be with', 'decreasing')],\n",
       "  [('we', 'obtain', '0n')],\n",
       "  [('sides', 'on vmax be', '0')],\n",
       "  [('t ∈', 'for 0 be', '0 τ')],\n",
       "  [('x', 'be', 'diﬀerentiable')],\n",
       "  [('s', 'for x be', 'tra jectory s'), ('s', 'for x be', '0')],\n",
       "  [('equilibria', 'be with', 'form')],\n",
       "  [('t ≥ 0', 'for δs be', 't')],\n",
       "  [('us', 'suppose', 'βλ ∗ − γ > 0')],\n",
       "  [('t →', 'as 0n be', '0n')],\n",
       "  [('of function', 'as r be', 't')],\n",
       "  [('tra jectories', 'be on', 'level sets')],\n",
       "  [('model', 'be with', 'positive rates'),\n",
       "   ('positive rates', 'with model be', '14'),\n",
       "   ('model', 'be with', 'initial condition s'),\n",
       "   ('initial condition s', 'with model be', '14')],\n",
       "  [('map H', 'be in', 'set s ∈ Rn | 0n ≤ s')],\n",
       "  [('A', 'be', 'non-negative')],\n",
       "  [],\n",
       "  [('≥', 'r', '0'), ('k ∈ N', 'for p be', '0')],\n",
       "  [('limit', 'be', 'ﬁxed')],\n",
       "  [('q ∗', '=', 'k'), ('q', 'be', 'ﬁxed')],\n",
       "  [('q', 'since si be', '0')],\n",
       "  [('Ni', 'be in', 'rest of section')],\n",
       "  [('adjacency matrix A', 'be', 'binary')],\n",
       "  [('γ', '=', '0.4')],\n",
       "  [('zero', 'correspond to', '0'), ('dark-gray node', 'be in', 'Figure 5')],\n",
       "  [('parameters', 'lead to', '0')],\n",
       "  [('undirected graph', 'be in', 'Figure 5')],\n",
       "  [('interval NumericCitation', 'be in', 'time')],\n",
       "  [('behavior',\n",
       "    'be in',\n",
       "    'populations starting from small initial infection fraction')],\n",
       "  [('We',\n",
       "    'study',\n",
       "    'transient propagation of epidemic starting from small initial fractions of infected nodes')],\n",
       "  [('infection',\n",
       "    'monotonically vanish for',\n",
       "    'arbitrary ﬁxing topology graphs')],\n",
       "  [('We', 'analyze', 'its behavior')],\n",
       "  [('Network SIS model sections', 'include', 't')],\n",
       "  [('threshold condition', 'be', 'proposed')],\n",
       "  [('Set-Theoretic Methods', 'be in', 'Control')],\n",
       "  [('Epidemics', 'be in', 'Complex Networks')],\n",
       "  [('deterministic model', 'be in', 'nonhomogeneous population')],\n",
       "  [('Graph Theoretic Methods', 'be in', 'Multiagent Networks')],\n",
       "  [('Virus spread', 'be in', 'networks')],\n",
       "  [('Epidemic spreading', 'be in', 'real networks')],\n",
       "  [('SIR epidemics', 'be in', 'contact networks')]],\n",
       " 21: [[('experimental analysis',\n",
       "    'be make on',\n",
       "    'ﬁ ve diﬀerent empirical complex networks'),\n",
       "   ('analysis', 'be make on', 'ﬁ ve diﬀerent empirical complex networks')],\n",
       "  [('it', 'be', 'Therefore reasonable')],\n",
       "  [('Singapore 138671', 'c', 'Mile ˇSiki'),\n",
       "   ('subdivision', 'be with', 'common characteristic features called')],\n",
       "  [('model', 'be', 'good'), ('individual', 'be in', 'population')],\n",
       "  [('Diﬀerent mathematical models', 'study', 'epidemic spreading'),\n",
       "   ('Diﬀerent mathematical models', 'have', 'have used'),\n",
       "   ('Diﬀerent models', 'study', 'epidemic spreading')],\n",
       "  [('epidemic dynamics', 'be', 'neglected'),\n",
       "   ('dynamics', 'be', 'neglected'),\n",
       "   ('Contact network epidemiology', 'model', 'epidemic spreading')],\n",
       "  [('epidemic outbreak', 'be in', 'SIS model NumericCitation'),\n",
       "   ('SIS model NumericCitation', 'in outbreak be', 'endemic state')],\n",
       "  [('important application', 'be in', 'epidemic predictions'),\n",
       "   ('Realistic epidemic simulations',\n",
       "    'have become',\n",
       "    'very important application of high-performance computing in epidemic predictions')],\n",
       "  [('These', 'be', 'few')],\n",
       "  [('cities', 'be', 'connected'),\n",
       "   ('cities', 'be connect through', 'airline transportation network')],\n",
       "  [('epidemic spreading', 'be with', 'SIR model on complex networks')],\n",
       "  [('number', 'be in', 'epidemic spreading on arbitrary network'),\n",
       "   ('we',\n",
       "    'predict',\n",
       "    'expected value of number of infected nodes in epidemic spreading on arbitrary network'),\n",
       "   ('nonzero value', 'be in', 'bimodal part of epidemic phase diagram')],\n",
       "  [('contribution', 'be', 'novel')],\n",
       "  [('N set', 'l-set of', 'links')],\n",
       "  [('we', 'use', 'standard stochastic SIR model'),\n",
       "   ('we', 'simulate', 'epidemic propagation')],\n",
       "  [('discrete time model', 'be', 'used')],\n",
       "  [('epidemic', 'spread', 'de ﬁnes one epidemic simulation')],\n",
       "  [('we', 'denote with', 'letter λ')],\n",
       "  [('infected node', 'recover in', 'one discrete time step')],\n",
       "  [('simulation', 'be', 'random')],\n",
       "  [('Epidemic simulations', 'be', 'independent'),\n",
       "   ('Epidemic simulations', 'be', 'mutually independent')],\n",
       "  [('X', 'be', 'random')],\n",
       "  [('epidemic simulation problem',\n",
       "    'compute',\n",
       "    'ﬁrst n moments of random variable X')],\n",
       "  [('following analytical form NumericCitation', 'be in', 'SIR model')],\n",
       "  [('infected node', 'infect sequentially', 'its neighbors')],\n",
       "  [('node', 'be', 'susceptible')],\n",
       "  [('network', 'use', 'adjacency list')],\n",
       "  [('infected nodes', 'of queue be', 'v')],\n",
       "  [('us', 'start with', 'case of one infected node with k neighbors'),\n",
       "   ('one infected node', 'be with', 'k neighbors')],\n",
       "  [('it', 'transmit', 'infection'),\n",
       "   ('it', 'transmit infection to', 'each of its neighbors'),\n",
       "   ('it', 'try in', 'one cycle')],\n",
       "  [('node', 'be in', 'infected state'),\n",
       "   ('geometric distribution', 'be with', 'expectation 1/q'),\n",
       "   ('cycles',\n",
       "    'number of be',\n",
       "    'sample from geometric distribution with expectation 1/q')],\n",
       "  [('infected nodes', 'be in', 'network')],\n",
       "  [],\n",
       "  [('network', 'be with', 'cycles')],\n",
       "  [('Xn', 'variable by', 'infected node of degree n NumericCitation')],\n",
       "  [('n', 'be', 'random')],\n",
       "  [('Naive SIR algorithm', 'of e be', 'T n')],\n",
       "  [('total number', 'be in', 'epidemic')],\n",
       "  [('we', 'look at', 'complexity of Algorithm 1')],\n",
       "  [('idea', 'choose number from', 'distribution')],\n",
       "  [('k Xl', '=', '0 k l')],\n",
       "  [('Distributions', 'be', 'used'),\n",
       "   ('Distributions', 'be use for', 'several networks')],\n",
       "  [('simulation', 'use', 'diﬀerent parameters NumericCitation'),\n",
       "   ('it', 'be', 'necessary'),\n",
       "   ('it', 'run', 'simulation'),\n",
       "   ('simulation', 'use', 'diﬀerent starting parameters NumericCitation')],\n",
       "  [('infected nodes', 'of queue be', 'v')],\n",
       "  [('simulations', 'be in', 'Naive SIR algorithm')],\n",
       "  [('we', 'call', 'generation index'),\n",
       "   ('parameter', 'be in', 'case of FastSIR')],\n",
       "  [('simulation', 'proceed to', 'next step')],\n",
       "  [('we', 'have choose', 'nodes'),\n",
       "   ('we', 'have choose', 'initially infected nodes')],\n",
       "  [('we', 'prove in', 'following proposition')],\n",
       "  [('Y', 'be', 'random'), ('Z', 'be', 'random')],\n",
       "  [('k', 'be infect by', 'node')],\n",
       "  [('remaining i − k', 'be in', 'set of n − m')],\n",
       "  [('That', 'can', 'can done in n − m k diﬀerent ways')],\n",
       "  [('k infected nodes', 'be in', 'set of m susceptible nodes')],\n",
       "  [('k Xl', '=', '0 k l')],\n",
       "  [('instances', 'be in', 'equations')],\n",
       "  [('its', 'k', 'degree')],\n",
       "  [('data', 'be with', 'O'), ('it', 'be', 'possible')],\n",
       "  [('unifo rmly', 'choose', 'value for cumulative distribution')],\n",
       "  [('we', 'can ﬁnd k1 in', 'k'), ('we', 'can ﬁnd', 'k1')],\n",
       "  [('infection',\n",
       "    'should',\n",
       "    'In last step should transmitted to k1 neighboring nodes')],\n",
       "  [('it', 'be', 'equal')],\n",
       "  [],\n",
       "  [],\n",
       "  [('q', 'equal', 'one')],\n",
       "  [('k', 'be', 'big enough')],\n",
       "  [('q', 'equal', 'one')],\n",
       "  [('sum', 'be in', 'equation')],\n",
       "  [('running time', 'be', 'T 1 c'), ('time', 'be', 'T 1 c')],\n",
       "  [('second part', 'be from', 'random selection of k1 neighbors')],\n",
       "  [('r1', 'be greater if', 'c2 > c3'),\n",
       "   ('r1', 'be', 'greater than 1 if c2 > c3')],\n",
       "  [('network structure', 'have', 'an in ﬂuence on ratio too')],\n",
       "  [('it', 'emphasise', 'c2')],\n",
       "  [('networks', 'be with', 'more cycles')],\n",
       "  [('S', 'be with', 'respect to number of links L')],\n",
       "  [('calculation cost', 'be', 'proportional'),\n",
       "   ('calculation cost', 'be proportional to', 'kma x 4')],\n",
       "  [('we', 'obtain', 'calculation costs'),\n",
       "   ('we', 'obtain', 'lower calculation costs')],\n",
       "  [('k Xl', '=', '0 k l')],\n",
       "  [('k Xl', '=', '0 k − 1 l − 1')],\n",
       "  [('k Xl', '=', '0 k − 1 l − 1')],\n",
       "  [('k Xl', '=', '0 k − 1 l')],\n",
       "  [('minimum precision', 'be', '64 bits')],\n",
       "  [('50000', 'be', 'high enough value of degree for majority of networks'),\n",
       "   ('50000', 'be', 'high value of degree for majority of networks')],\n",
       "  [('Similar recursive formula', 'can', 'can used')],\n",
       "  [('Parallelization', 'be in', 'Like similar algorithms NumericCitation')],\n",
       "  [('Parallelization', 'be', 'challenging'),\n",
       "   ('it', 'will', 'will scope of one of our next investigations')],\n",
       "  [('Experimental results', 'be in', 'section')],\n",
       "  [('Algorithms', 'be implement in', 'C')],\n",
       "  [('Astrophysics E-Print Archive', 'snapshot of', 'structure'),\n",
       "   ('US Western States Power Grid',\n",
       "    'network of',\n",
       "    'coauthorships between scientists posting on Astrophysics E-Print Archive between Jan 1 1995 reconstructed from BGP tables posted by University'),\n",
       "   ('Jan 1 1995',\n",
       "    'between archive be',\n",
       "    'symmetrized snapshot of structure of Internet at level of autonomous systems'),\n",
       "   ('Astrophysics E-Print Archive',\n",
       "    'snapshot at',\n",
       "    'level of autonomous systems')],\n",
       "  [('Table 1', 'show', 'basic information for above mentioned networks')],\n",
       "  [('we', 'also measure', 'execution time for distribution precalculation'),\n",
       "   ('we', 'also measure', 'execution time')],\n",
       "  [('Upper', 'bind memory consumption for', 'experiments')],\n",
       "  [('addition', 'be in', 'Table 3')],\n",
       "  [('Running time', 'be in', 'seconds for Live Journal network')],\n",
       "  [('results', 'be in accordance with', 'above analysis')],\n",
       "  [('p', 'be', 'When small')],\n",
       "  [('0.1 1 Online version', 'be in', 'colour')],\n",
       "  [('0.1 1 Online version', 'be in', 'colour')],\n",
       "  [('0.1 1 Online version', 'be in', 'colour')],\n",
       "  [('Online version', 'be in', 'colour')],\n",
       "  [('running time ratio', 'be', 'where strictly less'),\n",
       "   ('Online version', 'be in', 'colour')],\n",
       "  [('running time ratio', 'be', 'where strictly less than one'),\n",
       "   ('entire parametric space', 'on ratio be', 'Citation simulations'),\n",
       "   ('running time ratio', 'be', 'where strictly less')],\n",
       "  [('speed-up', 'be dependent on', 'network structure')],\n",
       "  [('analysis', 'be in', 'Section 4')],\n",
       "  [('networks', 'be with', 'more cycles')],\n",
       "  [('that', 'result for', 'Live Journal network of 5 mil')],\n",
       "  [('links', 'be', 'fast'), ('links', 'be', 'very fast')],\n",
       "  [('results', 'be achieve without', 'parallelization'),\n",
       "   ('results', 'be', 'achieved')],\n",
       "  [('version', 'be with', 'interventions')],\n",
       "  [('running time ratio', 'be', 'where strictly less'),\n",
       "   ('Online version', 'be in', 'colour')],\n",
       "  [('running time ratio', 'be', 'where strictly less'),\n",
       "   ('Online version', 'be in', 'colour')],\n",
       "  [('application', 'be in', 'virus propagation'),\n",
       "   ('knowledge spreading', 'be with', 'SIR model')],\n",
       "  [('arbitrary network structure', 'be with', 'reduced running time'),\n",
       "   ('Discussion', 'be in', 'paper')],\n",
       "  [('parallel version', 'be with', 'interventions')],\n",
       "  [],\n",
       "  [('depending position', 'be in', 'phase diagram')],\n",
       "  [('Experimental analysis',\n",
       "    'be',\n",
       "    'made on ﬁve di ﬀerent empirical complex network on single core processor')],\n",
       "  [('excellent agreement', 'be with', 'our expectations')],\n",
       "  [('STAR', 'republic of', 'Singapore')],\n",
       "  [('Barab', 'asi be', 'Rev. Mod')],\n",
       "  [('Barab', 'asi be', 'Science 286 508 1999')],\n",
       "  [('H. ˇStefan ˇci', 'a be', '390'),\n",
       "   ('complex networks', 'be in', 'preparation'),\n",
       "   ('complex networks', 'in c be', \"H. ˇStefan ˇci ' c Physica A\"),\n",
       "   ('H. ˇStefan ˇci', 'a be', '2010'),\n",
       "   ('c', 'a', 'H. ˇStefan ˇci'),\n",
       "   ('Epidemic centrality NumericCitation A.Lan ˇci', 'c be', 'c Physica A'),\n",
       "   ('c', 'be in', 'complex networks')]],\n",
       " 22: [[('nodes', 'be in', 'network')],\n",
       "  [('model', 'be', 'popular')],\n",
       "  [('greatest portion', 'be in', 'expectation')],\n",
       "  [('nodes', 'be often refer to', 'to spreaders')],\n",
       "  [('related problem', 'be in', 'Chen et al.')],\n",
       "  [('today', 'of', 'computer systems')],\n",
       "  [('Imprecision', 'be with', 'β')],\n",
       "  [('k-shell', 'outperform', 'degree')],\n",
       "  [('β', 'threshold of', 'network')],\n",
       "  [('comparison', 'be with', 'eigenvector centrality'),\n",
       "   ('k-shell', 'measure of', 'Kitsak et')],\n",
       "  [('This',\n",
       "    'be follow by',\n",
       "    'empirStudySIRpreprint 22 Citation 18:33 Network Science 3 Fig. 2')],\n",
       "  [('p', 'be with', 'β =')],\n",
       "  [('degree', 'outperform', 'k-shell')],\n",
       "  [('experimental results', 'be in', 'Section 5')],\n",
       "  [('2 SIR Model', 'be in', 'As Kitsak et al.')],\n",
       "  [('time', 'at step be', 'node infected')],\n",
       "  [('individuals', 'be in', 'population infected under IC model')],\n",
       "  [('eigenvector centrality', 'outperform', 'k-shell'),\n",
       "   ('eigenvector centrality', 'typically outperform', 'k-shell')],\n",
       "  [('their proof',\n",
       "    'identify',\n",
       "    'inﬂuence spread of single node under SIR model')],\n",
       "  [('Theorem 2.1',\n",
       "    'calculate',\n",
       "    'inﬂuence spread of single node under SIR models')],\n",
       "  [('We', 'show', 'reduction from known # P-complete problem s')],\n",
       "  [('G', 'be', 'directed'), ('E', 'denote', 'set of edges')],\n",
       "  [('s', 'be', 'where connected to t'), ('s', 'be', 'where connected')],\n",
       "  [('Ms', 'number of', 'infectees given initially infected node s'),\n",
       "   ('Ms', 'number with', 'β ='),\n",
       "   ('expected number', 'be with', 'β =')],\n",
       "  [('we', 'consider', 'graph G')],\n",
       "  [('s', 'be in', 'G')],\n",
       "  [('today', 'with', 'computer systems')],\n",
       "  [('We', 'describe centrality measure in', 'Section 3')],\n",
       "  [('model', 'be in', 'our experiments')],\n",
       "  [('literature', 'have', 'k')],\n",
       "  [('epidemic threshold calculation', 'be in', 'Madar et')],\n",
       "  [('SIR model', 'use', 'method')],\n",
       "  [('k', 'be', 'average degree'),\n",
       "   ('n', 'assume', 'randomly connected network')],\n",
       "  [('spreading', 'be in', 'Kitsak et'), ('topic', 'be in', 'Castellano')],\n",
       "  [('betweenness', 'be in', 'Kitsak et')],\n",
       "  [('we', 'examine in', 'our experiments')],\n",
       "  [('major centrality measures', 'be in', 'literature')],\n",
       "  [('several other well-known radial measures',\n",
       "    'citation include',\n",
       "    'closeness')],\n",
       "  [('it', 'note throughout', 'literature')],\n",
       "  [('voter model', 'be in', 'Antal et')],\n",
       "  [('degree', 'be', 'radial'), ('degree', 'point out in', '2006')],\n",
       "  [('Degree', 'be one of', 'three measures')],\n",
       "  [('High shell-number nodes', 'be in', 'network')],\n",
       "  [('k-shell number', 'be', 'generally outperformed'),\n",
       "   ('k-shell number', 'be generally outperform by', 'eigenvector centrality')],\n",
       "  [('Place', 'remove', 'nodes'), ('Place', 'remove node into', 'shell VS')],\n",
       "  [('1 Technically', 'work of', 'Antal et')],\n",
       "  [('number', 'be in', 'thee network')],\n",
       "  [('elimination', 'be with', 'degree 1')],\n",
       "  [('F', 'ﬁnalizes decomposition with', 'third shell')],\n",
       "  [('many paths', 'be in', 'network pass')],\n",
       "  [('betweenness', 'be', 'medial'),\n",
       "   ('betweenness', 'be', 'Hence medial centrality measure')],\n",
       "  [('its', 'cc', 'i')],\n",
       "  [('centrality measure', 'be in', 'Bonacich')],\n",
       "  [('network V =', 'centrality of', 'node i satisﬁes xi')],\n",
       "  [('x', \"be vector 's\", \"xi 's\")],\n",
       "  [('convergence', 'be', 'reached')],\n",
       "  [('number', 'be in', 'network')],\n",
       "  [('i.', 'of distance be', 'metric counts'),\n",
       "   ('q =', 'for distance be', 'metric counts'),\n",
       "   ('i', 'have', 'neighbors')],\n",
       "  [('it', 'identify', 'inﬂuential spreaders'),\n",
       "   ('it', 'identify', 'spreaders')],\n",
       "  [('imprecision', 'function from', '2010')],\n",
       "  [('p', 'be', 'real'), ('N', 'denote', 'number of nodes')],\n",
       "  [('100', 'predict', 'most efﬁcient spreaders')],\n",
       "  [('R R Development Core Team', 'of version be', '2011')],\n",
       "  [('3.0 GHz', 'be with', '4 GB of RAM')],\n",
       "  [('R', 'in package be', '2006'), ('package', 'be in', 'R')],\n",
       "  [('We', 'obtain', 'our datasets')],\n",
       "  [('authors', 'newman be', '2011'), ('Citation', 'be', 'academic')],\n",
       "  [('ca-GrQc-GCC', 'be', 'academic')],\n",
       "  [('urv-email', 'be', 'e-mail')],\n",
       "  [('users', 'establish', '2009')],\n",
       "  [('sample', 'be extract in', 'Dec. 2008')],\n",
       "  [('Links', 'represent', 'two individuals')],\n",
       "  [('sexual experience', 'be with', 'other individual'),\n",
       "   ('std-GCC', 'be', 'online'),\n",
       "   ('online sex community', 'be in', 'Brazil')],\n",
       "  [('Citation', 'be', 'day'), ('Citation 0102', 'construct from', '2012')],\n",
       "  [('It', 'be extract on', 'Jan 2 Citation'), ('It', 'be', 'extracted')],\n",
       "  [('Mark Newman', 'have', 'data repository at University of Michigan')],\n",
       "  [('many mechanisms', 'generate', 'scale free networks')],\n",
       "  [('they', 'use', 'preferential attachment model')],\n",
       "  [('shell number', 'predict', 'spreaders')],\n",
       "  [('λ', 'be exponent of', 'power law of degree distribution'),\n",
       "   ('KS', 'be', 'maximum shell present in network')],\n",
       "  [('epidemic threshold', 'be in', 'question')],\n",
       "  [('number changes', 'be with', 'β')],\n",
       "  [('number', 'be', 'better')],\n",
       "  [('we', 'depict', 'dependence on β')],\n",
       "  [('p', 'plot', 'imprecision function of degree')],\n",
       "  [('degree', 'outperform', 'shell number'),\n",
       "   ('degree', 'outperform', 'number')],\n",
       "  [('these', 'be generate by', 'preferential attachment models')],\n",
       "  [('log scales', 'be use on', 'axes')],\n",
       "  [('closeness', 'like measure be', 'outperform degree')],\n",
       "  [('eigenvector centrality', 'perform best for', 'network')],\n",
       "  [('imprecision',\n",
       "    'function over',\n",
       "    'all of our networks including ca-HepTh network')],\n",
       "  [('Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvalue centrality', 'perform best for', 'network')],\n",
       "  [('centrality measure', 'perform on', 'average')],\n",
       "  [('many neighbors', 'be with', 'high degree'),\n",
       "   ('node', 'be with', 'high eigenvector centrality')],\n",
       "  [('5.3 Large values', 'be in', 'Kitsak et')],\n",
       "  [('difference',\n",
       "    'be in',\n",
       "    'ability of starting node spread even at seven times')],\n",
       "  [('We', 'display', 'our imprecision functions'),\n",
       "   ('larger values', 'be in', 'Figure 16')],\n",
       "  [('Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvalue centrality', 'perform best for', 'network')],\n",
       "  [('Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvalue centrality', 'perform best for', 'network')],\n",
       "  [('Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvalue centrality', 'perform best for', 'network')],\n",
       "  [('Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvalue centrality', 'perform best for', 'network')],\n",
       "  [('Average Imprecision', 'be with', 'β =')],\n",
       "  [('Average Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvector', 'perform on', 'average'),\n",
       "   ('eigenvector', 'perform best on', 'average')],\n",
       "  [('Average Imprecision', 'be with', 'β =')],\n",
       "  [('eigenvector', 'perform best on', 'average')],\n",
       "  [('p', '=', '5')],\n",
       "  [('eigenvector', 'perform best on', 'average')],\n",
       "  [('We', 'study', 'multiple values of infection probability β')],\n",
       "  [('eigenvector centrality',\n",
       "    'consistently outperform',\n",
       "    'other centrality measures usually independent of β')],\n",
       "  [('spreaders',\n",
       "    'be in',\n",
       "    'non-monotonic models of diffusion processes such voter model')]],\n",
       " 23: [[('SIR model', 'be with', 'equal death'),\n",
       "   ('e Astrof ısica da Universidade de Lisboa', '‡ in', 'paper'),\n",
       "   ('P. R. China M. K. Mak ‡', 'be in', 'paper'),\n",
       "   ('e Astrof ısica da Universidade de Lisboa', 'department of', 'Computing'),\n",
       "   ('e Astrof ısica da Universidade de Lisboa',\n",
       "    'institute of',\n",
       "    'Vocational Education'),\n",
       "   ('e Astrof ısica da Universidade de Lisboa', 'c8', 'Ediﬁc')],\n",
       "  [('we', 'use', 'exact solution'),\n",
       "   ('numerical solution', 'reproduce', 'exactly analytical solution')],\n",
       "  [('reduction', 'be with', 'vital dynamics to Abel type equation')],\n",
       "  [('SIR model', 'be with', 'vital dynamics')],\n",
       "  [('scientist', 'be in', 'fact')],\n",
       "  [('he', 'argue', 'advantages of variolation NumericCitation')],\n",
       "  [('ordinary diﬀerential equations NumericCitation',\n",
       "    'of system be',\n",
       "    'dy dt =')],\n",
       "  [('one recovery', 'be in', 'D units of time')],\n",
       "  [('constants Ni', 'with x be', '0'),\n",
       "   ('constants Ni', 'with x be', 'y ='),\n",
       "   ('initial conditions x =', 'be with', 'constants Ni')],\n",
       "  [('non-linear diﬀerential equations systems', 'give by', 'Eqs')],\n",
       "  [('stochastic epidemic-type model', 'be with', 'enhanced connectivity')],\n",
       "  [('master equation',\n",
       "    'be transform via',\n",
       "    'quantum spin operator formulation')],\n",
       "  [('age-structured SIR epidemic model',\n",
       "    'be with',\n",
       "    'time periodic coeﬃcients')],\n",
       "  [('epidemic model', 'be with', 'stage structure')],\n",
       "  [('variety', 'be in', 'epidemiology')],\n",
       "  [('traveling wave ordinary diﬀerential equations',\n",
       "    'be in',\n",
       "    'usual spatial')],\n",
       "  [('chaotic regimes', 'be', 'characterized respectively')],\n",
       "  [('solution', 'be obtain in', 'exact parametric form')],\n",
       "  [('SIR model', 'be with', 'vital dynamics')],\n",
       "  [('SIR model', 'be with', 'deaths'),\n",
       "   ('exact parametric form', 'be in', 'Section III')],\n",
       "  [('We', 'conclude', 'our results in Section IV'),\n",
       "   ('our results', 'be in', 'Section IV')],\n",
       "  [('N3', 'be', 'arbitrary'),\n",
       "   ('N1', 'be', 'positive'),\n",
       "   ('N2', 'be', 'arbitrary'),\n",
       "   ('N2', 'be', 'positive'),\n",
       "   ('N3', 'be', 'positive'),\n",
       "   ('N1', 'be', 'arbitrary')],\n",
       "  [('ﬁxed population N', 'be with', 'only three compartments')],\n",
       "  [('next step', 'be in', 'our study')],\n",
       "  [('latter', 'eliminate y from', 'Eqs')],\n",
       "  [('dz dt', '=', '′ γ x')],\n",
       "  [('constant', 'be', 'positive')],\n",
       "  [('latter basic diﬀerential equation',\n",
       "    'describe spread of non-fatal disease in',\n",
       "    'given population Eq')],\n",
       "  [('19', 'be equivalent to', 'system of diﬀerential equations Eqs')],\n",
       "  [('u du dt', '=', '0')],\n",
       "  [('Bernoulli type diﬀerential equation',\n",
       "    'be with',\n",
       "    'general solution given by φ ='),\n",
       "   ('C1', 'be', 'arbitrary'),\n",
       "   ('general solution', 'with equation be', '24')],\n",
       "  [('time', 'be in', 'as view of Eqs')],\n",
       "  [('t0', 'be', 'where arbitrary integration constant'),\n",
       "   ('t0', '=', '0'),\n",
       "   ('t0', 'be', 'arbitrary')],\n",
       "  [('we', 'hence have obtain', 'complete exact solution of system of Eqs')],\n",
       "  [('we', 'compare', 'Eq')],\n",
       "  [('C1', 'be', 'negative')],\n",
       "  [('two constants', 'be in', 'solution')],\n",
       "  [('change', 'be in', 'values')],\n",
       "  [('N2', '=', '15'), ('N3', '=', '10')],\n",
       "  [('γ', '=', '0.02')],\n",
       "  [('complete agreement',\n",
       "    'be with',\n",
       "    'numerical results obtained in NumericCitation')],\n",
       "  [('diﬀerent initial conditions x', 'for z be', 't')],\n",
       "  [('brief discussion', 'be with', 'births'),\n",
       "   ('time evolution', 'be with', 'equal birth')],\n",
       "  [('SIR MODEL', 'be with', 'EQUAL DEATH')],\n",
       "  [('x', 'of variation be', 'dotted curve')],\n",
       "  [('β =', 'for z be', 't')],\n",
       "  [('SIR model', 'be with', 'vital dynamics')],\n",
       "  [('x', 'of variation be', 'dotted curve')],\n",
       "  [('SIR model', 'be with', 'equal death')],\n",
       "  [('SIR model', 'be with', 'vital dynamics'),\n",
       "   ('SIR model', 'be with', 'vital dynamics'),\n",
       "   ('two dimensional dynamical systems', 'be in', 'x = N invariant plane')],\n",
       "  [('tra jectories', 'be with', 'initial conditions out invariant plane')],\n",
       "  [('∗', 'be', 'global')],\n",
       "  [('SIR model', 'be with', 'deaths')],\n",
       "  [('initial conditions', 'be with', 'x')],\n",
       "  [('We', 'use', 'Lyapounov direct method')],\n",
       "  [('we', 'scale variable by', 'population size')],\n",
       "  [('basic SIR model', 'be with', 'vital dynamics')],\n",
       "  [('evolution equation', 'be with', 'vital dynamics')],\n",
       "  [('We', 'diﬀerentiate', 'Eq')],\n",
       "  [('diﬀerential equation y ′', '=', '1 γ')],\n",
       "  [('initial conditions', 'be with', 'equal rates of deaths')],\n",
       "  [('SIR model', 'be with', 'vital dynamics to Abel type equation')],\n",
       "  [('58', 'be equivalent to', 'system of diﬀerential equations Eqs')],\n",
       "  [('order', 'be', 'arbitrary')],\n",
       "  [('Ψ', 'be with', 'Ψ0')],\n",
       "  [('SIR model', 'be with', 'deaths')],\n",
       "  [('x', 'for 42 be', '71')],\n",
       "  [('form y ′', '=', '1 γ ψ ′'), ('Y0', 'be', 'arbitrary')],\n",
       "  [('= N3', 'be with', 'initial condition')],\n",
       "  [('SIR model', 'be with', 'vital dynamics')],\n",
       "  [('y', 'be with', 'diﬀerent order approximations obtained'),\n",
       "   ('diﬀerent order approximations', 'with y be', 't')],\n",
       "  [('n', '=', '20')],\n",
       "  [('we', 'have consider', 'two versions of SIR model'),\n",
       "   ('CONCLUSIONS', 'be in', 'present paper'),\n",
       "   ('spread', 'be in', 'given population')],\n",
       "  [('it', 'reproduce', 'exactly numerical solution of model equations')],\n",
       "  [('SIR model', 'be with', 'births')],\n",
       "  [('signiﬁcant simpliﬁcation', 'be in', 'study of model'),\n",
       "   ('Abel equation',\n",
       "    'thus lead to',\n",
       "    'signiﬁcant simpliﬁcation in study of model')],\n",
       "  [('SIR epidemic model', 'be with', 'deaths')]],\n",
       " 24: [[('epidemic model', 'be in', 'two-layer network')],\n",
       "  [('isolation period', 'be', 'When maximum')],\n",
       "  [('excellent agreement', 'be with', 'theoretical results'),\n",
       "   ('We', 'ﬁnd', 'excellent agreement with theoretical results')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('Epidemic spreading models', 'explain', 'propagation of diseases')],\n",
       "  [('contact', 'be with', 'infected node'), ('it', 'become', 'immune')],\n",
       "  [('tree-like structure', 'be with', 'branches of infection')],\n",
       "  [('we', 'describe', 'it')],\n",
       "  [('probability', 'be in', 'framework')],\n",
       "  [('node', 'be with', 'k connections')],\n",
       "  [('cluster', 'be in', 'link percolation theory'),\n",
       "   ('epidemic-free phase', 'be with', 'only small outbreaks')],\n",
       "  [('multilayer networks', 'network of', 'networks')],\n",
       "  [('diﬀerent layers', 'be with', 'diﬀerent contacts in layer'),\n",
       "   ('diﬀerent contacts', 'be in', 'layer'),\n",
       "   ('can actors', 'be in', 'multilayer networks'),\n",
       "   ('individuals', 'can', 'In multilayer networks can actors')],\n",
       "  [('SIR model', 'be in', 'two networks')],\n",
       "  [('nodes', 'be in', 'diﬀerent layers'),\n",
       "   ('infection', 'spread between', 'nodes in diﬀerent layers')],\n",
       "  [('layer', 'be', 'epidemic-free'),\n",
       "   ('can epidemic-free phase', 'be in', 'epidemic phase')],\n",
       "  [('actors', 'be in', 'layers')],\n",
       "  [('dynamic', 'have', 'has modiﬁed by presence of other disease'),\n",
       "   ('SIR', 'be in', 'ﬁnal state')],\n",
       "  [('SIR model', 'be with', 'β'),\n",
       "   ('SIR model', 'be in', 'system composed of two overlapping layers')],\n",
       "  [('transmissibility', 'decrease', 'overlap')],\n",
       "  [('3', 'act in', 'many layers'), ('3', 'act in', 'layers')],\n",
       "  [('individual', 'have', 'work environment'),\n",
       "   ('equal size N', 'of network be', 'A')],\n",
       "  [('degree distribution', 'be in', 'layer')],\n",
       "  [('individuals', 'be in', 'layers')],\n",
       "  [('We', 'randomly infect', 'individual in layer A')],\n",
       "  [('infected individual', 'transmit disease to', 'their neighbors')],\n",
       "  [('4 infected', 'be give by', '1 − β')],\n",
       "  [('we', 'mapping process onto', 'link percolation in two layers'),\n",
       "   ('probability', 'for a be', 'B'),\n",
       "   ('link percolation', 'be in', 'two layers'),\n",
       "   ('there will node', 'be in', 'randomly chosen edge traversed by disease')],\n",
       "  [('node', 'be in', 'layer A/B of connectivity k')],\n",
       "  [('R', 'be', 'where given')],\n",
       "  [('recovered nodes', 'be in', 'layers')],\n",
       "  [('isolation parameter', 'time of', 'isolation')],\n",
       "  [('β', 'invert', 'Eq'), ('β', 'be', 'where obtained')],\n",
       "  [('red arrows', 'indicate', 'direction of branches of infection')],\n",
       "  [('gray node', 'be in', 'b')],\n",
       "  [('it', 'be average time of', 'duration of disease')],\n",
       "  [('Pi ∼ k', 'be with', 'λA =')],\n",
       "  [],\n",
       "  [('one-to-one nodes', 'be in', 'network A')],\n",
       "  [('Simulations results', 'be with', 'tr ='),\n",
       "   ('Simulations results', 'be in', 'ﬁnal state of process')],\n",
       "  [('full lines', 'correspond to', 'theoretical evaluation of Eq')],\n",
       "  [('w =', 'be in', 'blue')],\n",
       "  [('two layers', 'each of', 'size N')],\n",
       "  [('kmin =', 'with network be', 'λB ='),\n",
       "   ('λA', 'with network be', 'λB ='),\n",
       "   ('two scale free networks =', 'be with', 'kmin ='),\n",
       "   ('two ER layers', 'be with', 'hkA i ≃ hkB i ≃ 2.31')],\n",
       "  [('critical threshold βc', 'be at', 'intersection of two Eqs')],\n",
       "  [('coupled equation', 'of jacobian be', 'k'), ('Jacobian', 'be with', 'Ji')],\n",
       "  [('phase diagram', 'be in', 'plane β − w'),\n",
       "   ('plot', 'be with', 'λA ='),\n",
       "   ('two power law networks', 'be with', 'exponential cutoﬀ c ='),\n",
       "   ('two ER multilayer networks NumericCitation',\n",
       "    'be with',\n",
       "    'average degree hkAi ≃ hkB i ≃ 2.31')],\n",
       "  [('Phase diagram', 'be in', 'plane β − w')],\n",
       "  [('two ER networks', 'be with', 'hkA i ≃ hkB i ≃ 2.31'),\n",
       "   ('two ER networks', 'be with', 'kmin =')],\n",
       "  [('λB', 'be with', 'kmin =')],\n",
       "  [('multiplex networks', 'be with', 'βc ≈ 0.043')],\n",
       "  [('w', 'be with', 'wc =')],\n",
       "  [('kmin =', 'be in', 'Fig. 3'), ('kmin =', 'be in', 'Fig. 3')],\n",
       "  [('infected individuals', 'be in', 'layers')],\n",
       "  [('two ER networks', 'be with', 'hkA i ≃ hkB i ≃ 2.31'),\n",
       "   ('λB =', 'be with', 'kmin ='),\n",
       "   ('tw =', 'be with', 'λA ='),\n",
       "   ('two ER networks', 'be with', 'kmin ='),\n",
       "   ('function', 'be with', 'exponential cutoﬀ c =')],\n",
       "  [('contact', 'be with', 'healthy individuals'),\n",
       "   ('infected patient',\n",
       "    'on',\n",
       "    'knowledge of consequences in contact with healthy individuals')],\n",
       "  [('isolation', 'be in', 'only one layer')],\n",
       "  [('tw', '=', '0')],\n",
       "  [('underestimation', 'can strongly aﬀect', 'spreading dynamics')],\n",
       "  [('plots', 'be with', 'tw 0 for two ER networks')],\n",
       "  [('critical threshold',\n",
       "    'be',\n",
       "    'how much more underestimated in SIR model of two-layer SF networks than in SIIR model')],\n",
       "  [('SIR model', 'be in', 'multilayer networks NumericCitation')],\n",
       "  [('panic', 'negatively eﬀect', 'economy of region'),\n",
       "   ('declaration', 'be', 'non-trivial')],\n",
       "  [('recent Ebola outbreak', 'be in', 'West Africa')],\n",
       "  [('This', 'be', 'important')],\n",
       "  [('layers', 'be with', 'probability w'),\n",
       "   ('DISCUSSION', 'be in', 'summary'),\n",
       "   ('SIIR epidemic model', 'be in', 'two-layer network'),\n",
       "   ('we', 'discussion in', 'summary')],\n",
       "  [('Supplementary information',\n",
       "    'be available in',\n",
       "    'online version of paper')],\n",
       "  [('synchronization problems', 'be in', 'complex netwrks')],\n",
       "  [('synchronization mechanism', 'as a. be', 'Macri P. A.')],\n",
       "  [('reaction-diﬀusion processes', 'be in', 'scale-free networks'),\n",
       "   ('NumericCitation Gallos L. K.', 'be in', 'reaction-diﬀusion processes')],\n",
       "  [('NumericCitation Cohen R.', 'strategy for', 'Computer Networks')],\n",
       "  [('NumericCitation Newman M. E. J.',\n",
       "    'be with',\n",
       "    'arbitrary degree distributions')],\n",
       "  [('NumericCitation Pastor-Satorras R.', 'be in', 'Scale-Free Networks')],\n",
       "  [('C.', 'a. in', 'populations with heterogeneous infection rates'),\n",
       "   ('Macri P. A.', 'be in', 'populations with heterogeneous infection rates'),\n",
       "   ('populations', 'be with', 'heterogeneous infection rates')],\n",
       "  [('NumericCitation Pastor-Satorras R.', 'be in', 'complex networks')],\n",
       "  [('NumericCitation Granell C.', 'be in', 'Multiplex Networks')],\n",
       "  [('NumericCitation Cozzo E.', 'be in', 'Multiplex Networks')],\n",
       "  [('NumericCitation Anderson', 'disease of', 'humans')],\n",
       "  [('epidemic pathways', 'be in', 'global outbreaks of infectious diseases')],\n",
       "  [('graphs', 'of handbook be', 'Citation'),\n",
       "   ('NumericCitation Cohen R.', 'handbook of', 'graphs')],\n",
       "  [('Optimal path', 'be in', 'random weighted networks')],\n",
       "  [('failures', 'be in', 'interdependent networks'),\n",
       "   ('NumericCitation Buldyrev S. V.',\n",
       "    'cascade of',\n",
       "    'failures in interdependent networks')],\n",
       "  [('NumericCitation Valdez L. D.',\n",
       "    'be in',\n",
       "    'correlated interdependent networks')],\n",
       "  [('NumericCitation Brummitt C. D.', 'be in', 'interdependent networks')],\n",
       "  [('Multiplexity-facilitated cascades', 'be in', 'networks')],\n",
       "  [('Coevolution Multiplexity', 'be in', 'Multiplex Networks')],\n",
       "  [('NumericCitation Cozzo E.', 'a. of', 'Boolean multilevel networks')],\n",
       "  [('Emergence', 'feature from', 'Multiplexity')],\n",
       "  [('Information Diﬀusion', 'be in', 'Overlaying Social-Physical Networks')],\n",
       "  [('L. A. Epidemics', 'be in', 'partially overlapped multiplex networks')],\n",
       "  [('N.', 'm. of', 'Small-World Networks')],\n",
       "  [('random graphs', 'be with', 'given degree sequence')],\n",
       "  [('N', 'a. towards', 'Characterization of BehaviorDisease Models')],\n",
       "  [('isolation', 'occur in', 'only one layer'),\n",
       "   ('14 Supplementary Information',\n",
       "    'be with',\n",
       "    'critical values obtained by model')],\n",
       "  [('scenario', 'be in', 'other layer'), ('scenario', 'be in', 'one layer')],\n",
       "  [('N individuals', 'be in', 'layers'), ('it', 'spread there', 'disease')],\n",
       "  [('isolated infected nodes', 'be in', 'layer A')],\n",
       "  [('transmissibility', 'be in', 'A'),\n",
       "   ('isolated individuals', 'be in', 'layer A')],\n",
       "  [('A', 'be in', 'layer A')],\n",
       "  [('probability', 'for equation be', 'fA =')],\n",
       "  [('6 critical threshold βc', 'be at', 'intersection of two Eqs')],\n",
       "  [('Jacobian', 'be with', '15'), ('Phase diagram', 'be in', 'plane β − w')],\n",
       "  [('two ER networks', 'be with', 'hkA i ≃ hkB i ≃ 2.31'),\n",
       "   ('two ER networks', 'be with', 'kmin =')],\n",
       "  [('λB', 'be with', 'kmin =')],\n",
       "  [('multiplex networks', 'be with', 'βc ≈ 0.043')],\n",
       "  [('phase diagram', 'be in', 'plane β − w'),\n",
       "   ('two ER multilayer networks NumericCitation',\n",
       "    'be with',\n",
       "    'average degree hkA i ≃ hkB i ≃ 2.31'),\n",
       "   ('plot', 'be with', 'λA ='),\n",
       "   ('two power-law networks', 'be with', 'exponential cutoﬀ c =')],\n",
       "  [('curves', 'be in', 'Fig. 5')],\n",
       "  [('phase diagram', 'be in', 'plane β − w'),\n",
       "   ('16 0.2 0.15 β', 'be in', 'plane β − w')],\n",
       "  [('isolation', 'be in', 'layers')],\n",
       "  [('two ER networks', 'be with', 'hkA i ≃ hkB i ≃ 2.31')],\n",
       "  [('λB', 'be with', 'kmin =')],\n",
       "  [('tw =', 'be with', 'tr =')],\n",
       "  [('isolation', 'take place in', 'one network')],\n",
       "  [('w 0.8', 'be in', 'scenario presented')],\n",
       "  [('isolation', 'be in', 'layers'),\n",
       "   ('earlier scenario', 'be with', 'isolation in layers')],\n",
       "  [('w', 'be in', 'scenario'),\n",
       "   ('isolation', 'be in', 'one layer'),\n",
       "   ('isolation', 'be in', 'layers')],\n",
       "  [('NumericCitation Newman M. E. J.',\n",
       "    'be with',\n",
       "    'arbitrary degree distributions')]],\n",
       " 25: [[('B', 'be', 'life-threatening')],\n",
       "  [('mathematical model',\n",
       "    'be with',\n",
       "    'considerations of diﬀerent classes of individuals')],\n",
       "  [('model', 'be in', 'study')],\n",
       "  [('its stability', 'use', 'Routh-Hurwitz theorem')],\n",
       "  [('B', 'be', 'life-threatening')],\n",
       "  [('HBV', 'be', 'DNA virus classiﬁed in virus family of Hepadnaviridae')],\n",
       "  [('fulminant hepatitis', 'require', 'liver transplantation')],\n",
       "  [('most', 'result in', '000 deaths NumericCitation')],\n",
       "  [('increase', 'be in', 'transmission')],\n",
       "  [('albeit', 'can vary from', '30-180 days')],\n",
       "  [('instance people', 'be with', 'AIDS')],\n",
       "  [('infection', 'become', 'chronic'), ('person', 'become', 'infected')],\n",
       "  [('dynamical systems', 'be in', 'spatial eﬀects NumericCitation')],\n",
       "  [('mathematical model',\n",
       "    'describe',\n",
       "    'characteristics of HBV disease transmission')],\n",
       "  [('we', 'motivate by', 'work mentioned above')],\n",
       "  [('population size', 'be in', 'compartment'),\n",
       "   ('epidemic process', 'be', 'deterministic')],\n",
       "  [('population size', 'be in', 'compartment'),\n",
       "   ('epidemic process', 'be', 'deterministic')],\n",
       "  [('changes', 'be in', 'population of compartment')],\n",
       "  [('people', 'be in', 'compartment')],\n",
       "  [('L', 'be number of', 'latently infected/exposed individuals'),\n",
       "   ('S', 'denote', 'number of susceptible individuals')],\n",
       "  [('incoming individuals', 'of proportion be', 'cP')],\n",
       "  [('infection', 'be in', 'susceptible class')],\n",
       "  [('equilibrium state', 'at equation be', '1'),\n",
       "   ('dR dt', '=', '0'),\n",
       "   ('dM dt', 'be', '=')],\n",
       "  [('Equation', 'from equation be', '8')],\n",
       "  [('disease', 'be totally eradicate from', 'population')],\n",
       "  [('Jacobian determinant', 'be with', 'Eigen values λi'),\n",
       "   ('system', 'be in', 'model at equilibrium state')],\n",
       "  [('we', 'only use', 'theorem')],\n",
       "  [('Presented', 'be', 'mathematical'),\n",
       "   ('mathematical model', 'be in', 'population'),\n",
       "   ('5 Discussion Presented', 'be in', 'paper')],\n",
       "  [('new babies', 'be', 'when born'), ('babies', 'be', 'when born')],\n",
       "  [('other words', 'in kµ be', 'rate'),\n",
       "   ('other words',\n",
       "    'in kµ be',\n",
       "    'sum of φ P β of latently infected individuals'),\n",
       "   ('kµ', 'be in', 'other words for disease-free equilibrium state <'),\n",
       "   ('kµ', 'sum of', 'φ P β')],\n",
       "  [('hepatitis B virus', 'be in', 'China')],\n",
       "  [('19822154 NumericCitation Anderson R', 'disease of', 'humans')],\n",
       "  [('hepatitis B virus transmission model', 'be with', 'vaccination'),\n",
       "   ('J',\n",
       "    'behavior of',\n",
       "    'hepatitis B virus transmission model with vaccination')],\n",
       "  [('20553944 NumericCitation Khan M', 'model of', 'hepatitis b virus'),\n",
       "   ('20553944 NumericCitation Khan M', 'model with', 'migration eﬀect'),\n",
       "   ('Haq Z. Transmission model', 'be with', 'migration eﬀect')]],\n",
       " 26: [[('epidemic threshold', 'of ﬁcation be', 'Chengdu'),\n",
       "   ('ﬁnite-si ze networks', 'on ﬁcation be', 'Chengdu')],\n",
       "  [('lot', 'be in', 'classic Susceptible-Infected-Recovered')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('epidemic outbreak distribution',\n",
       "    'be in',\n",
       "    'SIS models Effectiveness of simulated identi ﬁcation measure unde r cutoff hypothesis')],\n",
       "  [('only healthy nodes', 'be with', 'stationary density of infected nodes')],\n",
       "  [('infected nodes',\n",
       "    'ﬁnal fraction of be',\n",
       "    'above which ﬁnite NumericCitation')],\n",
       "  [('traditional theoretical study', 'be base on', 'HMF')],\n",
       "  [('networks', 'be with', 'power-law scaling P'),\n",
       "   ('vanishing threshold', 'be in', 'thermodynamic limit for γ ≤ 3'),\n",
       "   ('γ', 'be', 'where degree exponent')],\n",
       "  [('scaling', 'be with', 'maximum degree')],\n",
       "  [('network', 'be with', 'large size N')],\n",
       "  [('SIR epidemic threshold',\n",
       "    'be inversely proportional to',\n",
       "    'average connectivity hki NumericCitation')],\n",
       "  [('NumericCitation', 'threshold of', 'SIR model')],\n",
       "  [('Three conventional methods', 'be', 'ﬁ nite-size scaling analysis')],\n",
       "  [('networks', 'be with', 'strong structural heterogeneity NumericCitation'),\n",
       "   ('precise numerical determination',\n",
       "    'be in',\n",
       "    'absorbing-state phase transitions')],\n",
       "  [('ﬁnitesimal f raction', 'be in', 'SIR model NumericCitation')],\n",
       "  [('epidemic threshold', 'be in', 'con ﬁguration model NumericCitation'),\n",
       "   ('he', 'accurate determination of', 'SIR epidemic threshold')],\n",
       "  [('networks', 'be with', 'ﬁnite size')],\n",
       "  [('we', 'determine', 'epidemic threshold'),\n",
       "   ('we', 'determine', 'threshold')],\n",
       "  [('we', 'discuss', 'validity of simulated methods')],\n",
       "  [('Sec.', 'give', 'conclusions'), ('Sec. 5', 'give', 'conclusions')],\n",
       "  [('effective simulated identi ﬁcation measure', 'be in', 'simulations')],\n",
       "  [('seeds', 'be in', 'SIS model')],\n",
       "  [('initial time', 'be in', 'SIR model')],\n",
       "  [('simulations', 'use', 'synchronous updating scheme')],\n",
       "  [('it contacts', 'be with', 'one')],\n",
       "  [('recovered nodes', 'acquire', 'permanent immunity in SIR model'),\n",
       "   ('rate µ', 'be in', 'SIS model'),\n",
       "   ('permanent immunity', 'be in', 'SIR model')],\n",
       "  [('µ', '=', '1')],\n",
       "  [('RRN', 'be with', 'constant degree k')],\n",
       "  [('recovered nodes', 'be in', 'SIR model'),\n",
       "   ('infected nodes', 'be in', 'SIS model')],\n",
       "  [('susceptibility χ', 'estimate', 'epidemic threshold of SIR model')],\n",
       "  [('critical point', 'be in', 'equilibrium phase on magnetic system')],\n",
       "  [('Comparison', 'be with', 'simulated thresholds on RRNs')],\n",
       "  [('k', 'be', 'where sub ﬁgure'),\n",
       "   ('N', 'be', 'where set to 104'),\n",
       "   ('squares', 'k for', 'SIS'),\n",
       "   ('N', 'be set to', '104'),\n",
       "   ('SIS', 'for λc be', 'squares'),\n",
       "   ('k', 'be', 'where ﬁgure')],\n",
       "  [('The', 'set to', '10')],\n",
       "  [('triangleups', 'be in', 'circles')],\n",
       "  [('results', 'be average', 'over 104 independent realizations on network')],\n",
       "  [('λ ∆ p', 'be with', 'prediction from HMF theory')],\n",
       "  [('large k', 'for ≃ be', 'k −')],\n",
       "  [('Ref', 'in k be', '1'),\n",
       "   ('k −', 'be in', 'Ref'),\n",
       "   ('difference', 'be in', 'work')],\n",
       "  [('c network size', 'be in', 'Figs. 1'),\n",
       "   ('further comparison', 'be with', 'susceptibility measure'),\n",
       "   ('we', 'make', 'further comparison with susceptibility measure')],\n",
       "  [('λ ∆ p', 'λs', 'I R c'), ('λ ∆ p', 'be', 'closer')],\n",
       "  [('distribution', 'be in', 'SIS model')],\n",
       "  [('RRN', 'be with', 'k =')],\n",
       "  [('SIS model', 'be in', 'Fig. 2'),\n",
       "   ('simulated threshold λc', '=', '1/hki ≃ 0.1')],\n",
       "  [('prevalence', 'be', 'still very small'),\n",
       "   ('prevalence', 'be', 'exponential distribution'),\n",
       "   ('prevalence', 'be', 'exponential')],\n",
       "  [('ρR ﬂuctuation', 'be in', 'SIR model'),\n",
       "   ('infected nodes',\n",
       "    'hρi',\n",
       "    'i. order of one-thousandth of ρR ﬂuctuation in SIR model')],\n",
       "  [('Fig. 2', 'in hki be', 'b'),\n",
       "   ('ρI', 'be in', 'SIS model'),\n",
       "   ('hki −', 'be in', 'Fig. 2')],\n",
       "  [('power-law distribution R', 'be with', 'cutoff at value')],\n",
       "  [('λ > λc', 'distribution of', 'outbreak sizes')],\n",
       "  [('critical point', 'be in', 'Fig. 2')],\n",
       "  [('good agreement',\n",
       "    'be with',\n",
       "    'results obtained by numerical simulations in Fig. 2')],\n",
       "  [('outbreak sizes',\n",
       "    'indeed obey',\n",
       "    'power-law distribution with exponent -1.5'),\n",
       "   ('power-law distribution', 'be with', 'exponent -1.5')],\n",
       "  [('non-ignorable lump',\n",
       "    'be in',\n",
       "    'ﬂuential in simulated determination of SIR epi demic threshold')],\n",
       "  [('b Simulated distribution', 'be in', 'SIS model ρR'),\n",
       "   ('SIS model ρR', 'be in', 'SIR model for λ ='),\n",
       "   ('outbreak sizes', 'of distribution be', 'λ ='),\n",
       "   ('SIS model ρR', 'in distribution be', 'λ ='),\n",
       "   ('λ =', 'for distribution be', 'λ =')],\n",
       "  [('k', '=', '10')],\n",
       "  [('results', 'be average', 'over 106 independent realizations on network')],\n",
       "  [('complete lump', 'be in', 'distribution')],\n",
       "  [('susceptibility', 'be in', 'Fig. 3'),\n",
       "   ('outbreak size', 'be with', 'ρR ≤ rc')],\n",
       "  [('cutoff value rc', 'correspond to', 'ﬁrst peak of ∆')],\n",
       "  [('Dirac delta function', 'be in', 'theory'),\n",
       "   ('lump', 'distribution of be', 'Dirac delta function in theory')],\n",
       "  [('rc', 'be with', 'P')],\n",
       "  [('different values', 'be in', 'Fig. 3')],\n",
       "  [('variability ∆', 'determine', 'epidemic threshold of SIR model')],\n",
       "  [('Applications', 'be in', 'section')],\n",
       "  [('SFNs', 'be with', 'structural cutoff in Figs. 4'),\n",
       "   ('structural cutoff', 'be in', 'Figs. 4')],\n",
       "  [('largest degree kmax', 'variation with', 'Ref'),\n",
       "   ('whose variation', 'be with', 'Ref')],\n",
       "  [('Color online Susceptibility χ', 'be with', 'cutoff')],\n",
       "  [('small outbreak sizes', 'be with', 'ρR ≤ rc')],\n",
       "  [('k', '=', '10')],\n",
       "  [('CONTENTS 9 N', 'depend strongly on', 'γ')],\n",
       "  [('changes', 'be with', 'slowly N')],\n",
       "  [('SFNs', 'be with', 'structural cutoff'),\n",
       "   ('variations', 'be with', 'γ'),\n",
       "   ('SFNs', 'be with', 'natural cutoff')],\n",
       "  [('γ', '=', '2.25'), ('SFNs', 'be with', 'natural cutoff c Fig. 4')],\n",
       "  [('It',\n",
       "    'can',\n",
       "    'can seen from above prediction in most cases on SFNs NumericCitation'),\n",
       "   ('above prediction', 'be in', 'most cases on SFNs NumericCitation')],\n",
       "  [('Comparison', 'be with', 'simulated thresholds on SFNs')],\n",
       "  [('SFNs', 'be with', 'structural cutoff'), ('N', 'be set', '104')],\n",
       "  [('SFNs', 'be with', 'structural cutoff')],\n",
       "  [('results',\n",
       "    'be average over',\n",
       "    '10 × 104 independent realizations on different 10 networks')],\n",
       "  [('results', 'be in', 'Sec. 2'), ('larger λ', 'be in', 'SIR model')],\n",
       "  [('we', 'calculate', 'Λ'),\n",
       "   ('real networks NumericCitation', 'of ipr be', 'Λ')],\n",
       "  [('result', 'agree with', 'conclusion of Ref')],\n",
       "  [('HMF', 'result', 'c of adjacent matrix'),\n",
       "   ('kmax', 'be', 'maximum degree')],\n",
       "  [('Conclusions', 'be in', 'summary'),\n",
       "   ('we', 'conclusion in', 'summary'),\n",
       "   ('complex networks', 'be with', 'ﬁnite size')],\n",
       "  [('threshold', 'be with', 'λ')],\n",
       "  [('epidemic',\n",
       "    'quickly approximately exponential distribution',\n",
       "    'when λ < λc')],\n",
       "  [('power-law distribution', 'be with', 'exponent -1.5')],\n",
       "  [('λ', 'distribution of', 'outbreak sizes')],\n",
       "  [('lump', 'be in', 'simulated scattergram'),\n",
       "   ('simulated outbreak probability', 'be in', 'theory')],\n",
       "  [('second lump', 'be', 'when ignored')],\n",
       "  [('real networks', 'be with', 'assortative mixing')],\n",
       "  [('effectiveness', 'analyze', 'critical distribution')],\n",
       "  [('cases', 'be in', 'asynchronous updating scheme')],\n",
       "  [('work', 'verify', 'theoretical analysis of critical point')],\n",
       "  [('Foundation',\n",
       "    'program of',\n",
       "    'Outstanding Ph. D. Candidate in Academic Research'),\n",
       "   ('Outstanding Ph. D. Candidate', 'be in', 'Academic Research'),\n",
       "   ('China', 'of foundation be', 'Program by UESTC'),\n",
       "   ('Foundation', 'program by', 'UESTC')],\n",
       "  [('Korea', 'of foundation be', 'NRF'),\n",
       "   ('Education', 'of ministry be', 'Citation R1A1A2010067')],\n",
       "  [('Appendix', 'be with', 'steady-state')],\n",
       "  [('recovery rate µ', '=', '1'),\n",
       "   ('distribution', 'be in', 'SIR model'),\n",
       "   ('distribution', 'be in', 'bond percolation')],\n",
       "  [('general network', 'be with', 'arbitrary degree distribution pk')],\n",
       "  [('ds − 2 dz s', '−', '7')],\n",
       "  [('g0 =', 'be in', 'addition')],\n",
       "  [('References NumericCitation Barrat A', 'm on', 'Complex Networks')],\n",
       "  [('NumericCitation Aderson R M', 'be in', 'Humans')],\n",
       "  [('NumericCitation G', 's be', 'Borge-Holthoefer J'),\n",
       "   ('NumericCitation G', 's be', 'Arenas A'),\n",
       "   ('NumericCitation G', 's be', 'Meloni S')],\n",
       "  [('NumericCitation Barth', 'm be', 'Barrat A'),\n",
       "   ('NumericCitation Barth', 'm be', 'Pastor-Satorras R')],\n",
       "  [('NumericCitation Marro J', 'be in', 'Lattice Models')],\n",
       "  [('2010 Monte Carlo Simulation', 'be in', 'Statistical Physics')],\n",
       "  [('Data', 'from discovery be', 'ACM TKDD')],\n",
       "  [('2004 Phys', 'as m be', 'Pastor-Satorras R'),\n",
       "   ('2004 Phys', 'as m be', 'Diaz-Guilera A')]],\n",
       " 27: [[('uncorrelated random walk',\n",
       "    'however be typically assume as',\n",
       "    'type of movement')],\n",
       "  [('we',\n",
       "    'moreover mainly focus on',\n",
       "    'eﬀect of reduced mobility of infected agents')],\n",
       "  [('people', 'be', 'when infected'), ('people', 'be', 'infected')],\n",
       "  [('we', 'conduct', 'extensive simulations'),\n",
       "   ('we', 'conduct', 'simulations')],\n",
       "  [('state', 'be in', 'which')],\n",
       "  [('populations', 'be with', 'spatial structures in humans'),\n",
       "   ('spatial structures', 'be in', 'humans'),\n",
       "   ('contrast', 'be with', 'traditional models')],\n",
       "  [('epidemic models', 'be with', 'explicit individual movement')],\n",
       "  [('epidemic models', 'be with', 'uncorrelated random walk')],\n",
       "  [('models', 'be often describe by', 'PDE')],\n",
       "  [('directional walking pattern',\n",
       "    'can',\n",
       "    'can described by biased random walk')],\n",
       "  [('explicit movement', 'with model be', 'ABM'),\n",
       "   ('agent-based models', 'be with', 'explicit movement')],\n",
       "  [('we', 'use', 'ABM')],\n",
       "  [('directional movement', 'be in', 'one-dimensional space')],\n",
       "  [('space', 'be', 'dimensional')],\n",
       "  [('we', 'here extend model to', 'two-dimensional space')],\n",
       "  [('we',\n",
       "    'more importantly consider',\n",
       "    'eﬀect of reduced mobility of infected agents')],\n",
       "  [('people', 'be', 'When infected'), ('people', 'be', 'infected')],\n",
       "  [('size', 'be consider', 'particle')],\n",
       "  [('movement phase', 'first take', 'place')],\n",
       "  [('agents', 'reveal', 'eﬀect of biased movements')],\n",
       "  [('I', 'have', 'activity'), (\"I 's activity\", 'decrease', 'due to illness')],\n",
       "  [('time', 'be in', 'state η')],\n",
       "  [('normal distribution', 'be with', 'mean 0 deviation ση')],\n",
       "  [('epidemic phase', 'second take', 'place')],\n",
       "  [('infection', 'always take place within', 'radius')],\n",
       "  [('infection rate', 'be 1 in', 'other words'),\n",
       "   ('infection rate', 'be 1 in', 'words')],\n",
       "  [('I', 'become', 'R with recovery rate γ'),\n",
       "   ('R', 'be with', 'recovery rate γ')],\n",
       "  [('we', 'will mainly focus on', 'on results')],\n",
       "  [('r', '=', '2')],\n",
       "  [('This', 'be', 'relatively unrealistic')],\n",
       "  [('We', 'call hereafter', 'high movement case')],\n",
       "  [('structure', 'be in', 'our model')],\n",
       "  [('corresponding simulation', 'result respectively in', 'one run')],\n",
       "  [('movies', 'be in', 'supplemental materials')],\n",
       "  [('susceptible agents', 'pass through', 'infectious agents')],\n",
       "  [('susceptible agents', 'get', 'When infected'),\n",
       "   ('susceptible agents', 'get', 'infected')],\n",
       "  [('ﬁnal epidemic size', 'fraction of', 'recovered agents'),\n",
       "   ('orange line', 'be in', 'Fig. 1g')],\n",
       "  [('Figure 2', 'show', 'result of high movement case')],\n",
       "  [('γ', '=', '0.01')],\n",
       "  [('ﬁnal epidemic size', 'therefore be much larger than', 'Fig. 1')],\n",
       "  [('half', 'be in', 'high movement case')],\n",
       "  [('We', 'conduct', 'sensitivity analysis')],\n",
       "  [('σI', 'be', 'when changed')],\n",
       "  [('two hundred simulation runs', 'be average for', 'data point')],\n",
       "  [('γ', 'be', 'when high'), ('γ', 'be', 'high')],\n",
       "  [('γ', 'be', 'when high'), ('γ', 'be', 'high')],\n",
       "  [('we', 'next focus on', 'eﬀect of σI')],\n",
       "  [('results', 'be in', 'Figs.')],\n",
       "  [('σI', 'mean', 'diﬀusion rate of infected agents')],\n",
       "  [('it', 'therefore raise', 'ﬁnal epidemic size')],\n",
       "  [('σI', 'be', 'when signiﬁcant')],\n",
       "  [('contact opportunities', 'be with', 'infected agents')],\n",
       "  [('We', 'call', 'minimum contact case')],\n",
       "  [('I', 'be', 'agent')],\n",
       "  [('contact', 'be with', 'infected agents')],\n",
       "  [('other settings', 'be same as', 'Fig. 1')],\n",
       "  [('disease', 'completely dissapear from', 'environment')],\n",
       "  [('they', 'result in', 'longer time of extinction')],\n",
       "  [('Figure 5', 'show', 'results')],\n",
       "  [('contact', 'be with', 'infected agents')],\n",
       "  [('N', 'be', 'when large'), ('N', 'be', 'large')],\n",
       "  [('N', 'be', 'when varied')],\n",
       "  [('extinction time', 'see in', 'Fig. 3'),\n",
       "   ('ﬁnal epidemic size', 'be', 'when small')],\n",
       "  [('further infection', 'result in', 'shortest extinction time')],\n",
       "  [('N', 'be', 'when varied'),\n",
       "   ('N', 'be', 'varied'),\n",
       "   ('peaks', 'be in', 'middle')],\n",
       "  [('disease', 'once spread to', 'entire population')],\n",
       "  [('total tendency', 'be same with', 'other results')],\n",
       "  [('r', 'be', 'even when low'), ('r', 'be', 'even when relatively low')],\n",
       "  [('contact opportunities', 'be with', 'infected agents')],\n",
       "  [('other settings', 'be same as', 'Fig. 1')],\n",
       "  [('peaks', 'correspond to', 'raised points for ﬁnal epidemic size')],\n",
       "  [('disease', 'once spread to', 'entire population')],\n",
       "  [('peaks', 'thus exist at', 'medium r. IV')],\n",
       "  [('SUMMARY', 'be in', 'paper')],\n",
       "  [('traditional epidemic models', 'be with', 'random diﬀusion')],\n",
       "  [('they', 'make', 'policies')]],\n",
       " 28: [[('model', 'divide to', 'individuals of population in three categories'),\n",
       "   ('population', 'be in', 'three categories')],\n",
       "  [('other hand', 'on similarity be', 'computer science propagation of virus'),\n",
       "   ('evolution', 'in similarity be', 'computer science propagation of virus'),\n",
       "   ('has similarity', 'be in', 'evolution of epidemics infect'),\n",
       "   ('has similarity', 'propagation of', 'virus')],\n",
       "  [('well-known article', 'be in', 'Nature magazine4')],\n",
       "  [('many diseases', 'be in', 'virus transmission'),\n",
       "   ('virus transmission', 'be in', 'networks of computers'),\n",
       "   ('This',\n",
       "    'can even give',\n",
       "    'relatively ﬁt results for many diseases in virus transmission in networks of computers')],\n",
       "  [('it', 'be consider to', 'users like susceptible individuals to disease')],\n",
       "  [('air', 'be in', 'case of inﬂuenza'), ('contacts', 'be with', 'all')],\n",
       "  [('Bn', 'be', 'binary')],\n",
       "  [('convention', 'be use in', 'work')],\n",
       "  [('k → ∞', 'type of', 'network'), ('P ≈ k − 2 − γ', 'be with', 'γ >')],\n",
       "  [('investigations', 'be in', 'closed community')],\n",
       "  [('news', 'be in', 'Internet')],\n",
       "  [('actor interchanges information', 'be with', 'its pairs')],\n",
       "  [],\n",
       "  [('It', 'be', 'assigned'),\n",
       "   ('assigned value', 'be in', 'rank C ∈ NumericCitation')],\n",
       "  [('Bernoulli raﬄe', 'be with', 'probability given by value C')],\n",
       "  [('< Bernoulli raffle', 'be with', 'probability C')],\n",
       "  [('its', 'state', 'H')],\n",
       "  [('survivors', 'of variance be', 'Table II'),\n",
       "   ('survivors', 'of variance be', 'V')],\n",
       "  [('comparative results', 'be in', 'following tables'),\n",
       "   ('comparative results', 'be in', 'particular situation')],\n",
       "  [('sum', 'be in', 'row')],\n",
       "  [('CONCLUSIONS', 'be in', 'prospective analysis of data'),\n",
       "   ('probability', 'be in', 'one given network X')],\n",
       "  [('random varibles', 'have', 'statistical independence')],\n",
       "  [('it', 'be', 'When compared'), ('it', 'be', 'compared')],\n",
       "  [('networks', 'be', 'equivalent')],\n",
       "  [('it', 'be', 'When compared'), ('it', 'be', 'compared')],\n",
       "  [('networks', 'be', 'equivalent')],\n",
       "  [('LM', 'be with', 'SM'),\n",
       "   ('it', 'be', 'When compared'),\n",
       "   ('it', 'be', 'compared')],\n",
       "  [('they', 'have', 'probability of life'),\n",
       "   ('they', 'have', 'more probability of life'),\n",
       "   ('behavior', 'survive', 'population of found one in toy networks')],\n",
       "  [('nonreliability', 'be in', 'theoretical')],\n",
       "  [('Epidemic variability', 'be in', 'complex networks arXiv'),\n",
       "   (\"1 Pascal Cr ' epey\", 'variability in', 'complex networks arXiv'),\n",
       "   ('Pascal Cr', 'epey be', 'Epidemic variability in complex networks arXiv')],\n",
       "  [('9 8 M.F. Laguna G. Abramson',\n",
       "    'be in',\n",
       "    'model of social inﬂuence Physica A')]],\n",
       " 29: [[('arbitrary networks', 'over spread be', 'namely SIS'),\n",
       "   ('discrete-time epidemics', 'of spread be', 'namely SIS')],\n",
       "  [('their', 'o', 'n')],\n",
       "  [('We',\n",
       "    'provide',\n",
       "    'complete global analysis of epidemic dynamics of nonlinear mean-ﬁeld approximation')],\n",
       "  [('similar result', 'have', 'has also shown in continuous-time case')],\n",
       "  [('disease-free ﬁxed point',\n",
       "    'be',\n",
       "    'when globally stable for mean-ﬁeld model')],\n",
       "  [('ﬁxed-points', 'cost of', 'epidemic NumericCitation')],\n",
       "  [('most well-known propagation models', 'be in', 'literature')],\n",
       "  [('Flu', 'be example of', 'model')],\n",
       "  [('Computing', 'institute of', 'Technology')],\n",
       "  [('Electrical Engineering', 'institute of', 'Technology')],\n",
       "  [('SIS case', 'be in', 'its entirety'),\n",
       "   ('network', 'be with', 'n nodes'),\n",
       "   ('Markov chain', 'be with', '2n states')],\n",
       "  [('This', 'be', 'discrete-space')],\n",
       "  [('non-linear approximations', 'have', 'have proposed')],\n",
       "  [('We',\n",
       "    'provide',\n",
       "    'complete global analysis of dynamics of nonlinear model')],\n",
       "  [('this', 'use', 'underlying Markov chain model')],\n",
       "  [('linear model',\n",
       "    'provide only',\n",
       "    'upper bound on marginal probabilities of marginals')],\n",
       "  [('nodes', 'be', 'infected')],\n",
       "  [('NumericCitation', 'be in', 'continuous-time'),\n",
       "   ('NumericCitation', 'numericcitation in', 'discrete-time case'),\n",
       "   ('NumericCitation', 'be in', 'discrete-time case')],\n",
       "  [('application', 'be in', 'hand')],\n",
       "  [('SIS model', 'be in', 'continuous-time case')],\n",
       "  [('4', 'be', 'devoted respectively')],\n",
       "  [('we', 'start from', 'SIS epidemics')],\n",
       "  [('we', 'return back to', 'exact Markov chain model')],\n",
       "  [('us', 'establish', 'relation')],\n",
       "  [('results', 'prove throughout', 'paper')],\n",
       "  [('We', 'conclude in', 'Section 6'), ('We', 'compare', 'them')],\n",
       "  [('models', 'be in', 'section'), ('confusion', 'facilitate', 'reading')],\n",
       "  [('proofs', 'be postpone to', 'appendix')],\n",
       "  [('ﬁrst', 'appear in', 'NumericCitation')],\n",
       "  [('2 S', 'be', 'EP IDEM ICS Fig. 1'), ('S', 'be', 'EP IDEM ICS Fig. 1')],\n",
       "  [('single node', 'be in', 'SIS model')],\n",
       "  [('Wavy arrow', 'represent', 'transition')],\n",
       "  [('I', 'be', 'infected')],\n",
       "  [('ξ1', '· for', 'given connected network G with n nodes'),\n",
       "   ('ξ1', 'state of', 'health'),\n",
       "   ('given connected network G', 'be with', 'n nodes')],\n",
       "  [('it', 'be', 'healthy'), ('n', 'be', 'binary')],\n",
       "  [('we', 'understand', 'behavior of model')],\n",
       "  [('origin', 'be', 'ﬁxed'),\n",
       "   ('t', 'be in', '+ βA P'),\n",
       "   ('+ βA P', 'in t be', '2.12'),\n",
       "   ('origin', 'be', 'unique')],\n",
       "  [('+ βA', 'in λmax be', '1 − δ'), ('λmax', 'be in', '+ βA')],\n",
       "  [('Nontrivial Fixed Point', 'of δ be', 'trivial ﬁxed point'),\n",
       "   ('A δ', 'point of', 'system equation')],\n",
       "  [('nontrivial ﬁxed point', 'be', 'unique')],\n",
       "  [('NumericCitation n', 'j at', 'origin')],\n",
       "  [('Ξi ∂ xj', 'be in', 'G'), ('Nj', 'be in', 'G.')],\n",
       "  [('s', 'be', 'decreasing')],\n",
       "  [('→ Rn', 'be with', 'Ξ')],\n",
       "  [('+ βA', 'in λmax be', '1 − δ'),\n",
       "   ('2.2.2.2 Global Stability', 'point of', 'system'),\n",
       "   ('λmax', 'be in', '+ βA')],\n",
       "  [('it', 'be simply marginal probability of', 't')],\n",
       "  [('One', 'can interpret', 't')],\n",
       "  [('p0', 'datum as', 'hidden complete data at time t')],\n",
       "  [('fi ∈ Rn +1', 'be', 'i-th unit column vector')],\n",
       "  [('It', 'can', 'can formally expressed')],\n",
       "  [('βA', 'in t be', '2.18'),\n",
       "   ('p', 'be in', 'βA p t'),\n",
       "   ('βA p t', 'in p be', '1 − δ'),\n",
       "   ('βA p t', 'be in', 'βA'),\n",
       "   ('βA p t', 'in p be', 't')],\n",
       "  [('one', 'obtain', 'bounds'), ('one', 'obtain', 'tighter bounds')],\n",
       "  [('+ βA', 'in time be', '1 − δ'), ('λmax', 'for time be', '1 − δ')],\n",
       "  [('5 2.3.2 Par tial Ordering µX ≥', 'be in', 'section')],\n",
       "  [('It', 'be', 'trivial'), ('trivial', 'be', 'well-deﬁned partial order')],\n",
       "  [('nodes', 'be in', 'network')],\n",
       "  [('u', 'by u be', 'r'), ('r1', 'u by', 'u')],\n",
       "  [('it', 'follow', 'that')],\n",
       "  [('equation', 'in n be', 'Y = P ξi ='),\n",
       "   ('0,1 n', 'be in', 'equation above')],\n",
       "  [('Lemma 2.7', 'be also true for', 'M'), ('Lemma 2.7', 'be true for', 'M')],\n",
       "  [('2.21', 'by i be', '1n')],\n",
       "  [('transition matrix', 'be', 'S')],\n",
       "  [('2.4 Generalized Contact Model 2.5 Immune-Admitting Model',\n",
       "    'be in',\n",
       "    'section')],\n",
       "  [('contact model', 'be in', 'other words M')],\n",
       "  [('node i', 'of rate be', 'probability')],\n",
       "  [('δ', 'be in', 'model'), ('node', 'become', 'healthy from infected')],\n",
       "  [('immune-admitting model', 'go to', 'steady state')],\n",
       "  [('similar properties', 'be with', 'epidemic maps')],\n",
       "  [('here', 'of φ be', '·')],\n",
       "  [('origin', 'point of', 'system')],\n",
       "  [('+ βA', 'in λmax be', '1 − δ'), ('λmax', 'be in', '+ βA')],\n",
       "  [('largest absolute value', 'be in', 'above Jacobian matrix'),\n",
       "   ('eigenvalue',\n",
       "    'be with',\n",
       "    'largest absolute value in above Jacobian matrix')],\n",
       "  [('2.6', 'in i be', 'j ∈ 1'),\n",
       "   ('2.33', 'in j be', '· · ·'),\n",
       "   ('≥ 0', 'j in', '2.33'),\n",
       "   ('j ∈ 1', 'be in', '2.33'),\n",
       "   ('2.33', 'in j be', 'n')],\n",
       "  [('we', 'study', 'stability of nontrivial ﬁxed point with high probability'),\n",
       "   ('stability', 'be with', 'high probability')],\n",
       "  [('We', 'ﬁx', 'x')],\n",
       "  [('i ∈ Nj', 'be in', 'G')],\n",
       "  [('G', 'be', 'random')],\n",
       "  [('c > 1', 'be', 'where constant')],\n",
       "  [('nontrivial ﬁxed point', 'be with', 'high probability for ﬁxed β')],\n",
       "  [('G', 'be with', 'p = For proof see'), ('proof', 'for p be', 'n')],\n",
       "  [('MC', 'stand for', 'Markov chain model')],\n",
       "  [('nontrivial stable ﬁxed point', 'be with', 'high probability')],\n",
       "  [('expected degree', 'be with', 'high probability'),\n",
       "   ('such graphs', 'study in', 'NumericCitation'),\n",
       "   ('such random graphs', 'study in', 'NumericCitation')],\n",
       "  [('epoch', 'node in', 'susceptible state'),\n",
       "   ('independent events', 'be with', 'probability β'),\n",
       "   ('nodes', 'be in', 'susceptible state')],\n",
       "  [('nodes', 'be in', 'recovered state'),\n",
       "   ('nodes can transition', 'be with', 'probability γ')],\n",
       "  [('We', 'start again with', 'exact Markov chain model')],\n",
       "  [('γ', 'be', 'immunization loss probability')],\n",
       "  [('single node', 'be in', 'SIRS model')],\n",
       "  [('Wavy arrow', 'represent', 'transition')],\n",
       "  [('S', 'get', 'infected')],\n",
       "  [('independence', 'due to yi be', '3.2')],\n",
       "  [('probability', 'for i be', 't'), ('pR', 'i for', 'probability')],\n",
       "  [('state R', 'be in', 'state')],\n",
       "  [('nonlinear mapping', 'be with', '2n states rather 3n states')],\n",
       "  [('1 − δ', 'be in', 'βA')],\n",
       "  [('1 − δ', 'be in', 'βA')],\n",
       "  [('origin', 'be', 'ﬁxed'), ('origin', 'be', 'stable')],\n",
       "  [('Nontrivial Fixed Point', 'of δ be', 'origin')],\n",
       "  [('i', 'be', 'increasing')],\n",
       "  [('pR', 'relationship between', 'probabilities')],\n",
       "  [('model', 'can represent', 'temporary')],\n",
       "  [('γ', '=', '0')],\n",
       "  [('single node', 'be in', 'SIRS-with-Vaccination model')],\n",
       "  [('Wavy arrow', 'represent', 'transition')],\n",
       "  [('θ', 'represent', 'probability of direct immunization')],\n",
       "  [('4.1 Infection-Dominant Model', 'be in', 'case')],\n",
       "  [('presence', 'in i be', 't'),\n",
       "   ('pR', 'be in', 'other words'),\n",
       "   ('other words', 'in pr be', 'pR')],\n",
       "  [('nodes', 'be', 'where all decoupled')],\n",
       "  [('Reduced Markov chain', 'be in', 'steady state')],\n",
       "  [('M =', 'be in', 'P ∗ S βA'),\n",
       "   ('M =', 'be in', '− θP ∗ S βA'),\n",
       "   ('t', 'be in', '0n 0n × n P ∗ R 1n')],\n",
       "  [('4.2 Vaccination-Dominant Model', 'be in', 'variation of model')],\n",
       "  [('enyi graph', 'be with', 'n')],\n",
       "  [('blue curves', 'show', 'fast extinction of epidemic')],\n",
       "  [('1 − δ', 'be in', 'non-negative')],\n",
       "  [('2 u', 'be in', '+ βA'), ('1 − δ', 'be in', 'βA')],\n",
       "  [('vaccination', 'be', 'dominant'), ('vaccination', 'be', 'when dominant')],\n",
       "  [('nontrivial ﬁxed point', 'be also stable in', 'most cases')],\n",
       "  [('observed threshold',\n",
       "    'be',\n",
       "    'However for certain networks different from threshold for stability of nonlinear model')],\n",
       "  [('Infected Nodes', 'of stepnumber be', '1 − θ')],\n",
       "  [('it', 'complete', 'proof of claim')],\n",
       "  [('i.', '0 since', 'x ∗ ∈ U')],\n",
       "  [('i ∈', 'for ψi be', 'x ∗')],\n",
       "  [('x ∗', 'be unique nontrivial zero of', 'Ψ')],\n",
       "  [('K', 'be', 'separation of vertex set of system')],\n",
       "  [('node', 'be in', 'K0'), ('G', 'be', 'connected')],\n",
       "  [('K0', 'be', 'empty set')],\n",
       "  [('It', 'be', 'trivial'), ('It', 'check', 'that')],\n",
       "  [('Φ', 'be', 'increasing')],\n",
       "  [('y ∗', '=', 'x ∗')],\n",
       "  [('A. 12', 'by ξi be', 'x ∗'), ('holds', 'above ξi be', 'x ∗')],\n",
       "  [('t', '∗ go to', 'inﬁnity')],\n",
       "  [('P', 'converge to', 'x ∗')],\n",
       "  [('Φk', '=', 'x ∗')],\n",
       "  [('It', 'follow', 'that')],\n",
       "  [('Ni', 'j for', 'j 1')],\n",
       "  [('| Ni', 'x', 'small > 0 such')],\n",
       "  [('We', 'claim', 'that')],\n",
       "  [('ﬁrst inequality', 'hold by', 'b')],\n",
       "  [('K0', '=', '∅')],\n",
       "  [('Property', 'by hold be', 'A. 87')],\n",
       "  [('x ∗', 'hence unique nontrivial ﬁxed point of', 'system')],\n",
       "  [('fi ∈ R2n +1', 'represent', 'ith unit column vector')],\n",
       "  [('us', 'drop', 't')],\n",
       "  [('Deﬁne ˆi ∈', 'be', 'susceptible')],\n",
       "  [('a', 'be', 'true'), ('ﬁxed point', 'statement', 'true')],\n",
       "  [('same way', 'be in', 'as Lemma 2.2'),\n",
       "   ('v', 'in λmax be', '1 − δ'),\n",
       "   ('λmax', 'be in', '∗'),\n",
       "   ('∗', 'in λmax be', '1 − δ'),\n",
       "   ('λmax', 'be in', 'v')],\n",
       "  [('1 − P ∗ R u', 'be in', 'n × 2n')],\n",
       "  [('0n', 'be with', 'main difference')],\n",
       "  [('γ', 'proof of', 'Theorem 4.3'), ('Theorem 4.3', 'of proof be', 'A. 120')],\n",
       "  [('pi pI', 'be in', '+ βA pI'),\n",
       "   ('It', 'follow', 'that'),\n",
       "   ('+ βA pI', 'in pi be', 't'),\n",
       "   ('+ βA pI', 'in pi be', '1 − δ')],\n",
       "  [('1 − δ', 'be in', '+ P ∗ S βA')],\n",
       "  [('statement b', 'also follow from', 'upper bounding Eq')],\n",
       "  [('2n T', 'be with', 'following values')]],\n",
       " 30: [[('Small-World Networks',\n",
       "    'on model be',\n",
       "    'Egypt 2Mathematics Department'),\n",
       "   ('Small-World Networks', 'on model be', 'A. S. Hegazi1'),\n",
       "   ('Small-World Networks', 'on model be', 'Al-Ain P. O. Box 17551'),\n",
       "   ('Small-World Networks', 'on model be', 'Faculty of Science'),\n",
       "   ('Small-World Networks', 'on model be', 'Faculty of Science'),\n",
       "   ('Small-World Networks', 'on model be', 'Mansoura 35516'),\n",
       "   ('Epidemic Model', 'faculty of', 'Science'),\n",
       "   ('Epidemic Model', 'faculty of', 'Science')],\n",
       "  [('ring vaccination programme', 'be', 'included')],\n",
       "  [('model',\n",
       "    'can',\n",
       "    'can theoretical explanation for nonlocal interactions in epidemic spreading'),\n",
       "   ('nonlocal interactions', 'be in', 'epidemic spreading')],\n",
       "  [('Ring vaccination', 'eradicate', 'FMD')],\n",
       "  [('outbreaks', 'be in', 'developing countries')],\n",
       "  [('sheep', 'be', 'essential source'),\n",
       "   ('sheep', 'be essential source for', 'human food')],\n",
       "  [('disease', 'once appear in', 'it'),\n",
       "   ('high loss', 'be in', 'national income')],\n",
       "  [('contacts', 'be with', 'infected animals'),\n",
       "   ('FMD', 'can', 'Also can transmitted')],\n",
       "  [('infected animal', 'be', 'So once discovered'),\n",
       "   ('whole farm', 'be', 'infected')],\n",
       "  [('This', 'be call', 'ring vaccination NumericCitation')],\n",
       "  [('Ring vaccination', 'always cost', 'less than mass vaccination'),\n",
       "   ('Ring vaccination', 'always cost', 'less')],\n",
       "  [('ring vaccination',\n",
       "    'be',\n",
       "    'more suitable for FMD than mass vaccination NumericCitation'),\n",
       "   ('ring vaccination', 'be more suitable for', 'FMD')],\n",
       "  [('herds', 'form', 'clusters')],\n",
       "  [('farm', 'be', 'Once infected')],\n",
       "  [('mass vaccination programme', 'have', 'had applied')],\n",
       "  [('nonlocal interaction', 'be in', 'spreading of outbreak')],\n",
       "  [('Regular graphs', 'can display only', 'clustering property')],\n",
       "  [('results', 'use', '6-8')],\n",
       "  [('dynamical system', 'be with', 'discrete time'),\n",
       "   ('lattice', 'be', 'dynamical')],\n",
       "  [('systems', 'often display', 'spatiotemporal chaotic behaviour')],\n",
       "  [('aspects', 'be in', 'many systems'), ('many systems', 'be in', 'ﬁelds')],\n",
       "  [('epidemic spreading', 'be in', 'population')],\n",
       "  [('It', 'be', 'connected'),\n",
       "   ('It',\n",
       "    'be',\n",
       "    'connected 1-dimensional lattice of size L with periodic boundary conditions'),\n",
       "   ('It', 'be', '1-dimensional'),\n",
       "   ('connected 1-dimensional lattice',\n",
       "    'be with',\n",
       "    'periodic boundary conditions')],\n",
       "  [('shortcuts', '=', '1'), ('shortcuts', 'be with', 'length k')],\n",
       "  [('average number', 'be in', 'lattice'),\n",
       "   ('φ', 'be', 'average number of shortcuts per bond in lattice')],\n",
       "  [('two random vertices', 'be connect by', 'shortcut')],\n",
       "  [('this', 'be derive in', 'NumericCitation')],\n",
       "  [('population', 'be with', 'random susceptibility'),\n",
       "   ('spread', 'be in', 'population with random susceptibility')],\n",
       "  [('number', 'be in', 'model')],\n",
       "  [('sphere', 'be with', 'surface Γd td − 1')],\n",
       "  [('one', 'have ev in', 'one dimension')],\n",
       "  [('important eﬀect', 'be in', 'vaccination policies'),\n",
       "   ('result', 'have', 'important eﬀect in vaccination policies')],\n",
       "  [('individuals', 'be with', 'shortcuts')],\n",
       "  [('susceptible-infected-recovered-susceptible SIRS model NumericCitation',\n",
       "    'be more closer to',\n",
       "    'FMD')],\n",
       "  [('step', 'be with', 'probability q')],\n",
       "  [('I-individuals', 'be recover by', 'rate q3')],\n",
       "  [('I', '−', 'q3 I')],\n",
       "  [('I', '−', 'q2R')],\n",
       "  [('we', 'combine', 'interactions')],\n",
       "  [('1-dimensional lattice', 'be with', 'periodic boundary conditions')],\n",
       "  [('local interaction', 'be with', 'nearest neighbours')],\n",
       "  [('ring vaccination programme', 'control', 'disease propagation')],\n",
       "  [('SWN', 'be with', 'L =')],\n",
       "  [('shortcuting neighbours', 'be', 'ﬁxed beforehand')],\n",
       "  [('results', 'give in', 'Fig')],\n",
       "  [('SWN', 'describe', 'interactions')],\n",
       "  [('Ring vaccination',\n",
       "    'eradicate fmd for',\n",
       "    'even high infection probability'),\n",
       "   ('Ring vaccination', 'eradicate', 'FMD'),\n",
       "   ('Ring vaccination', 'be', 'capable')],\n",
       "  [('model', 'have', 'has used before in diﬀerent contexts NumericCitation')],\n",
       "  [('it', 'now be generalize to', 'inhomogeneous case')],\n",
       "  [('infected individuals', 'be in', 'ith patch')],\n",
       "  [('infection', 'be with', 'rate λi')],\n",
       "  [('recovery rate', 'be represent by', 'γi')],\n",
       "  [('second term', 'be in', 'Eq')],\n",
       "  [('it', 'be', 'Then applied to Eq'), ('it', 'be apply to', 'Eq')],\n",
       "  [('aii | ≤ Pj 6 =', 'result on', 'eigenvalues of system')],\n",
       "  [('system', 'for solution be', 'A8')],\n",
       "  [('we', 'generalize to', '2-species CML')],\n",
       "  [('Σ', '=', '0 0')],\n",
       "  [('CML', 'be with', 'delay NumericCitation')],\n",
       "  [('stability conditions', 'd for', 'A16'),\n",
       "   ('2-species system', 'of solution be', 'A4')],\n",
       "  [('above results', 'from d be', '1 − D')],\n",
       "  [('it', 'be', 'stable')],\n",
       "  [('A24', 'be in', 'case')],\n",
       "  [('long range edges', 'be in', 'case of SWN')],\n",
       "  [('patches', 'be', 'where diﬀerent')],\n",
       "  [('second order terms', 'be in', 'µ')],\n",
       "  [('a1', '=', '− Xi αi')],\n",
       "  [('Mathematical Models', 'be in', 'Biology')],\n",
       "  [('Coupled Systems', 'in zanette be', 'cond-mat/0003174'),\n",
       "   ('Coupled Systems', 'be with', 'Time Delayes'),\n",
       "   ('D. H. Zanette', 'be in', 'Globally Coupled Systems with Time Delayes')]],\n",
       " 31: [[('RANDOM GRAPH', 'be with', 'GIVEN DEGREES SVANTE JANSON')],\n",
       "  [('process', 'be', 'simple')],\n",
       "  [('Individuals',\n",
       "    'be represent by',\n",
       "    'vertices in graph G with edges corresponding to potentially infectious contacts'),\n",
       "   ('vertices',\n",
       "    'be in',\n",
       "    'graph G with edges corresponding to potentially infectious contacts'),\n",
       "   ('graph G',\n",
       "    'be with',\n",
       "    'edges corresponding to potentially infectious contacts')],\n",
       "  [('G', 'be', 'how chosen')],\n",
       "  [('G', 'be', 'random')],\n",
       "  [(\"R'enyi G graphs NumericCitation\",\n",
       "    'those with',\n",
       "    'local household structure NumericCitation')],\n",
       "  [('SIR epidemics', 'be with', 'given degree sequence')],\n",
       "  [('Random graphs', 'be with', 'given degree sequence')],\n",
       "  [('SIR epidemic process', 'graph with', 'given degree sequence'),\n",
       "   ('random graph', 'be with', 'given degree sequence')],\n",
       "  [('k core', 'of size be', 'diameter NumericCitation')],\n",
       "  [('studies', 'be with', 'given degree sequence')],\n",
       "  [('Volz equations', 'may', 'may derived as corollary')],\n",
       "  [('results', 'be in', 'NumericCitation')],\n",
       "  [('conﬁguration model', 'be with', 'bounded vertex degrees')],\n",
       "  [('Wormald', 'have', 'diﬀerential equations method')],\n",
       "  [],\n",
       "  [('graphs', 'be with', 'given degree sequence'),\n",
       "   ('given degree sequence', 'be with', 'bounded vertex degrees')],\n",
       "  [('random graph', 'be with', 'given degrees')],\n",
       "  [(\"large numbers ' result\", 'be in', 'spirit'), ('epidemic', 'at', 'epoch')],\n",
       "  [('Our approach', 'lead to', 'fairly simple proofs')],\n",
       "  [('multigraphs',\n",
       "    'be with',\n",
       "    'given degree sequence deﬁned by conﬁguration model')],\n",
       "  [('Section 8',\n",
       "    'contain',\n",
       "    'few remarks on random 2 time shift used in our proof')],\n",
       "  [('time change', 'be in', 'Section 3'),\n",
       "   ('Appendix A', 'contain', 'technical lemma on time change in Section 3')],\n",
       "  [('Appendix B', 'be summary of', 'main notation used in paper')],\n",
       "  [('results', 'be', 'simple'),\n",
       "   ('notation', 'be', 'simple'),\n",
       "   ('Model', 'be', 'simple'),\n",
       "   ('assumptions', 'be', 'simple'),\n",
       "   ('from graphs', 'be with', 'degree sequence')],\n",
       "  [('i', 'must at', 'must even least')],\n",
       "  [('nR', 'k of', 'vertices'),\n",
       "   ('k', 'be with', 'degree k'),\n",
       "   ('nR', 'k with', 'degree k')],\n",
       "  [('E ′ ⊆ E', 'on y be', 't')],\n",
       "  [('∞', 'be with', 'high probability')],\n",
       "  [('We',\n",
       "    'assume',\n",
       "    'following regularity conditions for degree sequence asymptotics')],\n",
       "  [('event En', 'hold', 'D1')],\n",
       "  [('assumptions k', 'further =', 'k n → αSλ'),\n",
       "   ('assumptions k', 'further =', '0 knS')],\n",
       "  [('susceptibles', 'be with', 'non-zero degree')],\n",
       "  [('P ∞ k', '=', 'k')],\n",
       "  [('k', '=', 'k')],\n",
       "  [('uniform summability', 'be in', 'Remark 2.2')],\n",
       "  [('random multigraph', 'be with', 'given degree sequence'),\n",
       "   ('graph', 'be with', 'degree sequence'),\n",
       "   ('e.g. multigraph', 'be', 'simple'),\n",
       "   ('we', 'obtain', 'di')],\n",
       "  [('conﬁguration model',\n",
       "    'have',\n",
       "    'has used in study of epidemics NumericCitation')],\n",
       "  [('results', 'also hold for', 'SIR epidemic on G')],\n",
       "  [('G ∗', 'be', 'simple'), ('G ∗', 'be bound as', 'n → ∞')],\n",
       "  [('results', 'hold without', 'even it')],\n",
       "  [('error probabilities', 'be in', 'their case'),\n",
       "   ('size', 'be in', 'G'),\n",
       "   ('multigraph', 'be', 'simple')],\n",
       "  [('We', 'do here', 'anything similar')],\n",
       "  [('it', 'belong', 'to')],\n",
       "  [('pair', 'be remove from', 'pool of free half-edges')],\n",
       "  [('Infective vertices', 'also recover at', 'rate ρ'),\n",
       "   ('Infective vertices', 'recover at', 'rate ρ')],\n",
       "  [('remaining edges', 'be in', 'G ∗')],\n",
       "  [('interest', 'number of', 'susceptible')],\n",
       "  [('random graph G', 'be with', 'degree sequence')],\n",
       "  [('k', '=', 'k')],\n",
       "  [('limiting fraction', 'so θk be', '2.11'),\n",
       "   ('limiting fraction', 'so θk be', 'θ ∈ NumericCitation'),\n",
       "   ('time', 'at vs be', 'θt'),\n",
       "   ('we', 'deﬁne', 'θ'),\n",
       "   ('susceptible half-edges', 'of vs be', 'θt')],\n",
       "  [('we', 'let', 'θ')],\n",
       "  [('given half-edge', 'be still free at', 'time t')],\n",
       "  [('justiﬁcation', 'be in', 'proof below')],\n",
       "  [('hI =', 'with ∈ be', '0 1'), ('unique θ ∞ ∈', 'be with', 'hI =')],\n",
       "  [('deterministic limits', 'be in', 'Theorems')],\n",
       "  [('hI =', 'with ∈ be', '0 1'), ('unique θ ∞ ∈', 'be with', 'hI =')],\n",
       "  [('fractions', 'be in', 'case')],\n",
       "  [('T0', '=', '∞')],\n",
       "  [('time', 'be in', 'random process')],\n",
       "  [('St', '= s0 similarly for', 'other processes')],\n",
       "  [('G1', 'be', 'satisﬁed')],\n",
       "  [('hI =', 'with ∈ be', '0 1'), ('unique θ ∞ ∈', 'be with', 'hI =')],\n",
       "  [('T0', 'be', 'deﬁned')],\n",
       "  [('T0', 'on xs be', '∞ =')],\n",
       "  [('R →', 'be with', 'θt = for t 6 ˆA0'), ('θt =', 'with → be', '< 0')],\n",
       "  [('probability', 'be in', 'Theorem 5.3')],\n",
       "  [('small outbreak', 'be', 'really small')],\n",
       "  [('threshold', 'be in', 'R0')],\n",
       "  [('we', 'do', 'that')],\n",
       "  [('connection', 'be with', 'NumericCitation')],\n",
       "  [('multinomial distribution', 'be with', 'parameters pI')],\n",
       "  [('giant component', 'be in', 'G')],\n",
       "  [('fraction', 'be in', 'giant component NumericCitation')],\n",
       "  [('R0', '=', '1')],\n",
       "  [('p2', 'can', 'can non-zero')],\n",
       "  [('three diﬀerent types', 'be in', 'G')],\n",
       "  [('example', 'be', 'random'),\n",
       "   ('components', 'be', 'cycles'),\n",
       "   ('example', 'be', '2-regular')],\n",
       "  [('> V2', '> denote', 'ordered component sizes')],\n",
       "  [('size', 'be in', 'case')],\n",
       "  [('random 2-regular graph', 'be with', 'n − n1/2 vertices')],\n",
       "  [('largest component', 'be in', 'resulting graph'),\n",
       "   ('we', 'ni for', 'our third example')],\n",
       "  [('Analogously', 'be', 'bounded')],\n",
       "  [('unique giant component', 'be with', 'n − op')],\n",
       "  [('n − op susceptibles', 'be in', 'giant component')],\n",
       "  [('recovered vertices', 'be in', 'SIR dynamics')],\n",
       "  [('their eﬃcacy', 'be in', 'related epidemic model')],\n",
       "  [('neither', 'can', 'can studied with present argument')],\n",
       "  [('We', 'add tilde to', 'our notation for post-vaccinated epidemic')],\n",
       "  [('Xk =', 'k of', 'degree k vertices')],\n",
       "  [('0 ∞ Xk =', 'k of', 'infective vertices')],\n",
       "  [('It', 'follow', 'that')],\n",
       "  [('k', 'be vaccinate with', '0 1'), ('k', 'be', 'vaccinated')],\n",
       "  [('SIR', 'epidemic on', 'random multigraph G ∗')],\n",
       "  [('key step', 'be in', 'proof')],\n",
       "  [('number', 'end of', 'epidemic')],\n",
       "  [('time', 'be in', 'modiﬁed process')],\n",
       "  [('P ∞', 'mean', 'τ ∧ τ ∗')],\n",
       "  [('only jumps', 'be in', 'S ′ τ')],\n",
       "  [('k', '=', '0 S')],\n",
       "  [('τ', 'be with', 'modiﬁed transition rates')],\n",
       "  [('13 diﬀerential notation', 'be in', '3.7')],\n",
       "  [('positive bounded functions', 'for inequality be', '3.9'),\n",
       "   ('Gronwall', 'have', 'S ′')],\n",
       "  [('us', 'τ', '6τ1 p ﬁnite variation')],\n",
       "  [('jumps', 'be in', 'MS'), ('jumps', 'be in', 'S ′ τ')],\n",
       "  [('jump', 'be in', 'S ′ τ')],\n",
       "  [('Doob', 'have', 'L2')],\n",
       "  [('ﬁnite sum', 'be in', 'last line of 3.13')],\n",
       "  [('Z τ Gronwall', 'have', 'inequality yields')],\n",
       "  [('jumps', 'be in', 'X ′ τ'), ('jumps', 'be in', 'MX')],\n",
       "  [('one jump', 'be in', 'X ′ X ′')],\n",
       "  [('infective vertex', 'be with', 'k free half-edges')],\n",
       "  [('τ', 'be', '=')],\n",
       "  [('expression', 'be in', '2.14')],\n",
       "  [('Gronwall', 'have', 'inequality')],\n",
       "  [('0 knR', 'µr by', 'D4')],\n",
       "  [('integrand', 'be in', 'second term'),\n",
       "   ('X ′ σ', 'of convergence be', '3.4')],\n",
       "  [('3.16 nβ Eτ', 'be in', 'MR')],\n",
       "  [('D5', 'by mr be', 'τ | =')],\n",
       "  [('X ′ I', 'of convergence be', 'n − X ′ S'),\n",
       "   ('X ′ I', 'of convergence be', '3.6')],\n",
       "  [('us', 'take', 'τ1 = ∞')],\n",
       "  [('τ ∗', 'be bound', 'w.h.p.'),\n",
       "   ('strengthening', 'show in', 'Section 4 below')],\n",
       "  [('it', 'obtain', 'quantitative statements of convergence')],\n",
       "  [('we', 'close', 'section')],\n",
       "  [('bracketed term', 'be in', 'integrand')],\n",
       "  [('its', 'inverse', 'so Aτ = t for t >'),\n",
       "   ('its increasing continuous inverse', 't for', 't >'),\n",
       "   ('We', 'let', 't')],\n",
       "  [('0 16 processes', 'be in', 'their original time scaling')],\n",
       "  [('we', 'run out of', 'free infective half-edges')],\n",
       "  [('We', 'study', 'SIR epidemic on random multigraph G ∗')],\n",
       "  [('Theorem 2.9', 'of part be', 'i')],\n",
       "  [('it', 'prove for', 'modiﬁed epidemic studied in Section 3')],\n",
       "  [('It', 'be', 'enough'), ('It', 'consider', 'θ')],\n",
       "  [('h', '=', '0')],\n",
       "  [('We', 'apply', 'Lemma 3.1')],\n",
       "  [('− → 0', 'of sup be', 'number')],\n",
       "  [('it', 'see', 'τ ∗'), ('oﬀ', 'be with', 'other')],\n",
       "  [('beta distribution', 'be with', 'density 1 2 x − 1/2 on NumericCitation')],\n",
       "  [('example', 'be consider at', 'end of Section 2.1')],\n",
       "  [('h', 'be', 'concave'), ('h', 'be concave on', '1')],\n",
       "  [('pI', 'be', 'Hence continuous on θ ∞ 1'),\n",
       "   ('pI', 'hence be continuous on', 'θ ∞ 1')],\n",
       "  [('solution θt', 'of existence be', 'constant function θ ∞')],\n",
       "  [('limit', 'equal', 'θ ∞')],\n",
       "  [('satisﬁes', 'ˆτ', 't')],\n",
       "  [('inverse', 'follow', 'ˆA')],\n",
       "  [('τ ∗', 'side of', '4.8')],\n",
       "  [('its', 'τ', 't')],\n",
       "  [('integrand', 'be in', '3.17')],\n",
       "  [('We', 'assume', 'ε')],\n",
       "  [('t', 'on → be', '4.11')],\n",
       "  [('convergence statements', 'be in', '2.22')],\n",
       "  [('fraction It/n infective vertices', 'be in', '2.21')],\n",
       "  [('c adl ag martingale', 'be with', 'MI')],\n",
       "  [('Doob', 'have', 'inequality')],\n",
       "  [('distribution', 'be in', 'C'),\n",
       "   ('n', 'be in', 'D'),\n",
       "   ('D', 'in n be', '0 ∞')],\n",
       "  [('ˆI0', 'clearly =', 'αI'), ('ˆI0', '=', 'αI')],\n",
       "  [('ˆIt', 'be', 'deterministic')],\n",
       "  [('con-n → ∞ ˆIt =', 'to mi be', '4.15'),\n",
       "   ('con-n → ∞ ˆIt =', 'to mi be', 't ='),\n",
       "   ('con-n → ∞ ˆIt =', 'to mi be', 'dominated convergence theorem'),\n",
       "   ('Xt/n', 'of convergence be', '2.22')],\n",
       "  [('unique solution ˆIt', 'with equation be', '2.20'),\n",
       "   ('ˆIt equation', 'be with', 'unique solution ˆIt')],\n",
       "  [('1 − ˆIt1 − vS', 'w.h.p. for', 't')],\n",
       "  [('ˆIt', 'be', 'non-decreasing'), ('case ρ', 'in 20 be', 'recoveries')],\n",
       "  [('> t1', 'be in', 'summary')],\n",
       "  [('statement', 'be', 'immediate')],\n",
       "  [('susceptible vertices', 'of op be', 'n'),\n",
       "   ('only root', 'be in', 'NumericCitation')],\n",
       "  [('isolated vertices', 'from p be', '4.3')],\n",
       "  [('unique root', 'be in', '0 1'), ('h', 'deﬁned in', '4.1')],\n",
       "  [('h one root', 'be in', '0 1')],\n",
       "  [('b',\n",
       "    'be in',\n",
       "    'As proof of Theorem 2.6 of solution follow from Lipschitz continuity of pI')],\n",
       "  [('0 τ ˆτ ∞', 'of → be', '0 ˆτ ∞')],\n",
       "  [('process', 'of τ be', 't'), ('we', 'study', 't')],\n",
       "  [('convention', 'for part be', 'c'), ('we', 'let', 't')],\n",
       "  [('convergence', 'be in', 'probability')],\n",
       "  [('t1', 'be', 'given')],\n",
       "  [('right-hand side', 'diverge as', 'ε → 0')],\n",
       "  [('t ∈', 'over ˆτ be', 't'), ('t ∈', 'over ˆτ be', '4.25')],\n",
       "  [('convergence p p ST0', 'be in', 'just as proof of Theorem 2.6'),\n",
       "   ('convergence p p ST0', 'be in', '2.27')],\n",
       "  [('distribution', 'be in', 'D')],\n",
       "  [('IT0', 'be in', 'distribution along original sequence')],\n",
       "  [('t1', 'on it0 be', 'a.s. for t1 > 0'), ('ˆIt IT0', 'a.s. for', 't1 > 0')],\n",
       "  [('4.27', 'by ∞ be', 'IT0 < αS − vS')],\n",
       "  [('This', 'be', 'ds'), ('This', 'be', 'immediate')],\n",
       "  [('s2 ∈', 'be with', 's1 s2')],\n",
       "  [('P', 'clearly =', '0')],\n",
       "  [('t', 'for n be', 'vS')],\n",
       "  [('XS ,0 − XS', '∞ of', 'half-edges'), ('T0', 'on op be', 'n')],\n",
       "  [('T0', 'on 0 be', 'X0 − Xt')],\n",
       "  [('we', 'study', 'beginning phase of epidemic'),\n",
       "   ('number XI', 't of', 'free infective half-edges')],\n",
       "  [('infection time', 'be with', 'distribution Exp'),\n",
       "   ('random recovery time', 'be with', 'distribution Exp')],\n",
       "  [('we', 'can ignore', 'them'), ('black half-edges', 'pair', 'oﬀ')],\n",
       "  [('Zt', 'be number of', 'of proof')],\n",
       "  [('nS', 'vertex of', 'degree k'),\n",
       "   ('k', 'for ns be', 'k − εn remaining susceptible vertices of degree k')],\n",
       "  [('ﬁnal expresssion', 'be', 'nonnegative')],\n",
       "  [('net change', 'be in', 'Zt')],\n",
       "  [('half-edge', 'be', 'When paired oﬀ')],\n",
       "  [('∆ Zti >', '0 with', 'probability 2ε/µ otherwise'),\n",
       "   ('mixture distribution Yk − 1', 'be with', 'probability k'),\n",
       "   ('k >', 'for zti be', '− 1 0 with probability 2ε/µ otherwise'),\n",
       "   ('infective half-edge', 'be', 'paired oﬀ')],\n",
       "  [('ε', 'be', 'chosen')],\n",
       "  [('Ui', '−', '1 such')],\n",
       "  [('pXI ,0 infective vertices', 'be with', 'at least one half-edge')],\n",
       "  [('we', 'have', 'P')],\n",
       "  [('probability', 'be in', 'section'),\n",
       "   ('expression', 'be in', 'Theorem 2.9')],\n",
       "  [('2F1', 'deﬁned in', 'e.g. NumericCitation')],\n",
       "  [('infective vertex', 'be with', 'ℓ >')],\n",
       "  [('Z 1 0 ℓ dx β β ℓ j Γ', 'ℓ in', '5.1'),\n",
       "   ('5.1', 'in ℓ be', 'j'),\n",
       "   ('ℓ', 'be in', '5.1')],\n",
       "  [('m', 'be', 'white'), ('normalization urn', 'be with', 'N balls')],\n",
       "  [('probability µR/µ', 'with − be', '5.3'),\n",
       "   ('Yk − 1 0', 'be with', 'probability αSkpk'),\n",
       "   ('probability αSkpk', 'with − be', '5.3'),\n",
       "   ('k', 'for − be', '5.3'),\n",
       "   ('Yk − 1 0', 'be with', 'probability µR/µ')],\n",
       "  [('µR', '=', '0')],\n",
       "  [('Watson process', 'be with', 'Yk individuals instead one')],\n",
       "  [('extinction probability', 'be with', 'numbers independent'),\n",
       "   ('product', 'be in', '5.5')],\n",
       "  [('number', 'be in', 'Section 4.3.1')],\n",
       "  [('we', 'prove', 'Theorem 5.3')],\n",
       "  [('their degrees', 'be', 'bounded'),\n",
       "   ('infected half-edges', 'be in', 'also sub critical case')],\n",
       "  [('precise size', 'be in', 'subcritical case')],\n",
       "  [('Proof', 'be in', 'multigraph case')],\n",
       "  [('random walk', 'be', 'thus absorbed there')],\n",
       "  [('i → ∞', 'as p be', '0 ˆZi < N')],\n",
       "  [('us', 'now return to', 'epidemic')],\n",
       "  [('us', 'restart there', 'epidemic')],\n",
       "  [('we', 'combine', 'facts')],\n",
       "  [('N', 'infective', '5.5'),\n",
       "   ('product', 'be in', '5.5'),\n",
       "   ('ε', 'be', 'arbitrary')],\n",
       "  [('theorems', 'be in', 'supercritical case')],\n",
       "  [('comparison', 'be with', 'random walk as in supercritical case'),\n",
       "   ('details', 'be', 'omitted'),\n",
       "   ('random walk', 'be in', 'as supercritical case')],\n",
       "  [('unique root', 'be in', '0 1 of ∞ Xk =')],\n",
       "  [('Watson process', 'k for', 'of simplicity')],\n",
       "  [('multi-type Galton', 'be with', 'types'),\n",
       "   ('nI', 'individual for', 'k >'),\n",
       "   ('nI', 'individual of', 'type k')],\n",
       "  [('total number', 'be in', 'generation')],\n",
       "  [('product', 'be in', '5.5')],\n",
       "  [('we', 'consider', 'simple random graph G'),\n",
       "   ('proofs', 'be in', 'section')],\n",
       "  [('G. Again', 'for theorem be', 'case'), ('case', 'number of', 'edges')],\n",
       "  [('edges', 'be in', 'graph')],\n",
       "  [('half-assume nI ,0', '=', '0'),\n",
       "   ('nI ,0', '=', '0'),\n",
       "   ('it', 'consider', 'subsequences')],\n",
       "  [('nI/n → αI', '=', '0')],\n",
       "  [('bounded number', 'be with', 'uniformly bounded degrees')],\n",
       "  [('nI', 'k of', 'initially infected vertices 29'),\n",
       "   ('we', 'consider', 'subsequences'),\n",
       "   ('we', 'again consider', 'subsequences')],\n",
       "  [('1 gk nI', 'k in', 'Theorem 5.3'), ('k', 'be in', 'Theorem 5.3')],\n",
       "  [('G ∗', 'be', 'thus simple')],\n",
       "  [('L', 'be event', 'at most log n vertices will infected')],\n",
       "  [],\n",
       "  [('similar result', 'be in', 'NumericCitation'),\n",
       "   ('same way', 'be in', 'as proof of similar result'),\n",
       "   ('conditioning', 'use', 'method of moments')],\n",
       "  [('we', 'give here', 'self-contained proof')],\n",
       "  [('W d − → cW', 'be with', 'convergence')],\n",
       "  [('right-hand side', 'on o be', '1')],\n",
       "  [('We', 'next consider', 'W1 | L')],\n",
       "  [('We', 'show', 'W1 | L'), ('method', 'by that be', 'W1 | L')],\n",
       "  [('E W m', 'be', 'd')],\n",
       "  [('our', 'assumption', 'D1')],\n",
       "  [('probability', 'αm before', 'log n')],\n",
       "  [('P P L', 'be in', '¯ G ∗'),\n",
       "   ('Theorem 5.3', 'consequently apply to', '¯ G ∗')],\n",
       "  [('bounded second moment', 'on assumption be', 'G1')],\n",
       "  [('uniform summability assumption', 'seem', 'indispensible')],\n",
       "  [('diﬀerential equation', 'to ˜θt be', 't'),\n",
       "   ('we', 'may extend ˜θt to', 'solution ˜θt'),\n",
       "   ('some', 'for → be', 'θ ∞')],\n",
       "  [('same value b', 'be with', 'b ∈')],\n",
       "  [('we', 'furthermore deﬁne ˆa as', 'before')],\n",
       "  [('ˆA0', '=', '− ∞')],\n",
       "  [('rest', 'be same as', 'before')],\n",
       "  [('small positive x0', 'for ∞ be', '7.2')],\n",
       "  [('pk ∼ k − α − 1', 'be with', 'α 6 2')],\n",
       "  [('≍ xα − 2', 's', '1 − x'),\n",
       "   ('7.3', 'in hs be', '1'),\n",
       "   ('x → 0', 'as s be', '1 − x'),\n",
       "   ('hS', 'be in', '7.3')],\n",
       "  [('α', '=', '2'), ('ˆA0', '=', '− ∞')],\n",
       "  [('R0', 'be', 'ﬁnite')],\n",
       "  [('random time shift', 'be in', 'Theorems 2.9')],\n",
       "  [('1 knI', 'w.h.p. by', 'Theorem 2.9'), ('G.', 'for theorem be', 'ii')],\n",
       "  [('we', 'let', 'δ → 0')],\n",
       "  [('T0', 'be in', 'Example 7.1')],\n",
       "  [('following lemma', 'justiﬁes', 'time change in Section 3'),\n",
       "   ('time change', 'be in', 'Section 3')],\n",
       "  [('ﬁnite state space E', 'with chain be', 'j ∈ E'),\n",
       "   ('τ', 'be', 'continuous'),\n",
       "   ('τ continuous time Markov chain', 'be with', 'ﬁnite state space E')],\n",
       "  [('its', 'τ', 't')],\n",
       "  [('Markov chain', 'be with', 'transition rates')],\n",
       "  [('St', 'number of', 'vertices'),\n",
       "   ('St', 'number at', 'time'),\n",
       "   ('St', 'number of', 'given type')],\n",
       "  [('following quantities', 'be also important in', 'our analysis')],\n",
       "  [('Network epidemic models', 'be with', 'two levels of mixing')],\n",
       "  [('Sipser', 'be with', 'ﬁxed degree sequence')],\n",
       "  [('random graphs', 'be with', 'ﬁxed degree sequence')],\n",
       "  [('random graphs', 'be with', 'tunable clustering')],\n",
       "  [('Graphs', 'be with', 'speciﬁed degree distributions')],\n",
       "  [('Contact processes', 'have', 'critical value 0'),\n",
       "   ('random graphs', 'be with', 'power law degree distributions')],\n",
       "  [('SIR process', 'be in', 'random network with heterogeneous connectivity'),\n",
       "   ('random network', 'be with', 'heterogeneous connectivity')],\n",
       "  [('chromatic number', 'be with', 'ﬁxed degree sequence')],\n",
       "  [('random multigraph', 'be', 'simple')],\n",
       "  [('random graph', 'be with', 'given degrees')],\n",
       "  [('core', 'be in', 'random graphs')],\n",
       "  [('critical random graph', 'be with', 'given degree sequence')],\n",
       "  [('random graphs', 'be with', 'given degree sequence')],\n",
       "  [('SIR dynamics', 'be in', 'random networks')],\n",
       "  [('random graphs', 'be with', 'given degree sequence')],\n",
       "  [('size', 'be with', 'given degree sequence')],\n",
       "  [('Random graphs', 'be with', 'arbitrary degree distributions')],\n",
       "  [('Stochastic integration', 'volume of', 'Stochastic Model ling')],\n",
       "  [('Continuous martingales',\n",
       "    'volume of',\n",
       "    'Grund lehren der Mathematischen Wissenschaften')],\n",
       "  [('random networks', 'be with', 'heterogeneous connectivity'),\n",
       "   ('SIR dynamics',\n",
       "    'be in',\n",
       "    'random networks with heterogeneous connectivity')],\n",
       "  [('Mathematics', 'of department be', 'Sweden E-mail address'),\n",
       "   ('∼ svante', 'university of', 'London'),\n",
       "   ('London', 'of university be', 'UK')]],\n",
       " 32: [[('epidemics', 'be with', 'multiple stages'),\n",
       "   ('Outbreak size distributions', 'be in', 'epidemics with multiple stages')],\n",
       "  [('We', 'mostly focus on', 'SIR')],\n",
       "  [('results',\n",
       "    'apply to',\n",
       "    'more general multiple-type critical branching processes'),\n",
       "   ('asymptotic results',\n",
       "    'apply to',\n",
       "    'more general multiple-type critical branching processes')],\n",
       "  [('multistage SIR model', 'be in', 'ﬁnite population'),\n",
       "   ('we',\n",
       "    'establish',\n",
       "    'scaling laws for multistage SIR model in ﬁnite population')],\n",
       "  [('CBP', 'be', 'branching'), ('CBP', 'be', 'peculiar')],\n",
       "  [('simplest branching process', 'hence call', 'birth-death process')],\n",
       "  [('they', 'equivalently be remove from', 'system')],\n",
       "  [('simplest model', 'be with', 'duplication corresponding to speciations')],\n",
       "  [('average', 'be in', 'stationary state')],\n",
       "  [('Branching processes', 'can', 'can studied in discrete time')],\n",
       "  [('death probabilities', 'be', 'For critical process equal')],\n",
       "  [('results', 'be', 'yet still cumbersome')],\n",
       "  [('we', 'derive outbreak size distribution in', 'inﬁnite population limit')],\n",
       "  [('we', 'give', 'short discussion')],\n",
       "  [('CRITICAL BIRTH-DEATH PROCESS', 'be in', 'section')],\n",
       "  [('B', 'b into', 'C')],\n",
       "  [('We', 'determine', 'outbreak size distribution')],\n",
       "  [('A1', '=', '1 2')],\n",
       "  [('Γ n', 'be in', 'powers of z')],\n",
       "  [('we', 'get', 'second term 1 2 on right-hand side of Eq')],\n",
       "  [('type B', 'be just tackle', 'Eq')],\n",
       "  [('which', 'from 1/2p be', '11'), ('we', 'introduce', 'z')],\n",
       "  [('joint outbreak size distribution ﬁrst',\n",
       "    'become',\n",
       "    'relevant for two-type CBP')],\n",
       "  [('unity', 'to x be', '3')],\n",
       "  [('k − 1', 'be with', 's1 ,1 ='), ('s1 ,1 =', 'with − be', '17')],\n",
       "  [('above formulas probabilities Pm', 'n for', 'm'),\n",
       "   ('m',\n",
       "    'lead follow',\n",
       "    'to 1 4 1 16 1 32 1 32 1 64 1 128 5 256 5 512 3 512 5 2048 7 512 7 1024 9 2048 5 2048 7 8192 21 2048 21 4096 7 2048 35 16384 35 32768 21 65536 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 One'),\n",
       "   ('example', 'for 945 be', '18')],\n",
       "  [('One', 'can also identify', '2n − 2')],\n",
       "  [('we', 'denote infection rate by', 'α')],\n",
       "  [('We',\n",
       "    'can still calculate outbreak size distribution by',\n",
       "    'conditioning on ﬁnite outbreaks')],\n",
       "  [('single individual results', 'be in', 'ﬁnite outbreak')],\n",
       "  [('course', 'of | be', '28'),\n",
       "   ('A', 'be in', 'powers of z'),\n",
       "   ('∼ 1 4α π √', 'be with', 'exponential cutoﬀ around ξ =')],\n",
       "  [('individual', 'be', 'where replaced by k individuals at rate αk')],\n",
       "  [('k individuals', 'be with', 'probability'), ('process', 'be', 'simple')],\n",
       "  [('critical process', 'be with', 'three oﬀspring')],\n",
       "  [('A5', '=', '32 729'),\n",
       "   ('A7', '=', '512 19683'),\n",
       "   ('A9', '=', '28160 1594323')],\n",
       "  [('one', 'have', 'k oﬀspring')],\n",
       "  [('1', 'for c be', '1 − z'),\n",
       "   ('branching process', 'be with', 'single cal ly 1 −')],\n",
       "  [('∼ c 4π n', 'be with', 'c')],\n",
       "  [('We', 'write', 'z')],\n",
       "  [('An', 'for √ be', '35')],\n",
       "  [('Epidemic outbreaks', 'obviously involve', 'ﬁnite populations'),\n",
       "   ('Epidemic outbreaks', 'involve', 'ﬁnite populations')],\n",
       "  [('ﬁniteness', 'also play', 'crucial role')],\n",
       "  [('can diﬀerent stages', 'be in', 'multistage model infected individuals')],\n",
       "  [('recovery rates', 'be', 'equal to one as in inﬁnite-population limit')],\n",
       "  [('probability distribution', 'of tail be', '4')],\n",
       "  [('upper limit', 'be in', 'sum')],\n",
       "  [('outbreak', 'be in', 'N/M')],\n",
       "  [('two-type CBP', 'be in', 'context of epidemic processes')],\n",
       "  [('it', 'can cause', 'death')],\n",
       "  [('We', 'again obtain', 'estimate √ MA ∝ N/MB')],\n",
       "  [('second relation', 'be in', '43'), ('we', 'scale', 'laws')],\n",
       "  [('N', 'give n/mb for', 'average 6 FIG. 1'),\n",
       "   ('two-type CBP', 'be in', 'ﬁnite populations')],\n",
       "  [('three-type CBP', 'be in', 'ﬁnite populations')],\n",
       "  [('we', 'get ma n/mb for', 'average size of outbreak'),\n",
       "   ('size', 'be in', 'so case')],\n",
       "  [('three-type CBP', 'for n be', '45'),\n",
       "   ('average outbreak sizes', 'be in', 'two-and three-type CBPs'),\n",
       "   ('nA ∝ N 3 11', 'n for', 'three-type CBP'),\n",
       "   ('good agreement', 'be with', 'simulations')],\n",
       "  [('distributions', 'have', 'universal algebraic tails')],\n",
       "  [('we', 'scale', 'laws for ﬁnite populations'), ('we', 'scale', 'laws')],\n",
       "  [('We', 'find', 'excellent agreement')],\n",
       "  [('scaled sizes', 'be in', 'Eq')],\n",
       "  [('maximal outbreak sizes', 'for law be', '44')],\n",
       "  [('It', 'determine', 'duration'),\n",
       "   ('100101102103104105101102103104105106107108N <nA> <nB> 100101102103104105106101102103104105106107108N <nA> <nB> <nC> 7',\n",
       "    'average of',\n",
       "    'outbreaks'),\n",
       "   ('problem',\n",
       "    'have',\n",
       "    'for classical critical SIR process has investigated in NumericCitation')],\n",
       "  [('We', 'be grateful to', 'Burak Buke')],\n",
       "  [('PLK', 'acknowledge', 'support from NSF grant CCF-0829541')],\n",
       "  [('we', 'hence should differentiate', '∂ m')],\n",
       "  [('l =', 'with l be', 'm − 1'),\n",
       "   ('m terms', 'be proportional to', 'm − 1'),\n",
       "   ('l', 'be with', 'l =')],\n",
       "  [('sn', 'for formula be', '17')],\n",
       "  [('Setting y =', 'be in', 'sum on smallest k term')],\n",
       "  [('NumericCitation W. Feller', 'introduction to', 'Probability Theory'),\n",
       "   ('Its', 'application', 'Vol')],\n",
       "  [('NumericCitation T. E. Harris', 'theory of', 'Branching Processes')],\n",
       "  [('Branching Processes', 'be in', 'Biology'),\n",
       "   ('NumericCitation M. Kimmel', 'processes in', 'Biology')],\n",
       "  [('NumericCitation N. T. J. Bailey', 'theory of', 'Infectious Diseases')],\n",
       "  [('NumericCitation G. Odor',\n",
       "    'universality in',\n",
       "    'Nonequilibrium Lattice Systems'),\n",
       "   ('Universality', 'be in', 'Nonequilibrium Lattice Systems')]],\n",
       " 33: [[('maximal extinction time', 'be in', 'SIR simulations on networks')],\n",
       "  [('average distances', 'be in', 'isolated components'),\n",
       "   ('distances', 'be', 'good')],\n",
       "  [('contact network NumericCitation',\n",
       "    'in clustering be',\n",
       "    'many short cycles'),\n",
       "   ('broad distribution degree', 'be in', 'contact network NumericCitation'),\n",
       "   ('clustering', 'be in', 'contact network NumericCitation')],\n",
       "  [('it', 'burn out in', 'population')],\n",
       "  [('competing effects',\n",
       "    'handle analytically with',\n",
       "    'even approximate approach')],\n",
       "  [('Results', 'be in', 'section')],\n",
       "  [('Networks',\n",
       "    'be with',\n",
       "    'power-law degree distribution generated by configuration model')],\n",
       "  [('vertex', 'be with', 'degree k proportional to k')],\n",
       "  [('γ', '=', '3')],\n",
       "  [('we', 'add', 'triangles')],\n",
       "  [('We',\n",
       "    'repeatedly connect',\n",
       "    'two distinct random neighbors of random vertex')],\n",
       "  [('networks', 'be with', 'power-law degree distributions')],\n",
       "  [('power-law degree distribution', 'be with', 'exponent 2')],\n",
       "  [('Ω λ', 'be', 'indicator function for epidemic threshold')],\n",
       "  [('infection seed', 'be', 'where located')],\n",
       "  [('we', 'get', 'estimated extinction time')],\n",
       "  [('best', 'predict', 'maximal time to extinction')],\n",
       "  [('it', 'just lift', 'point cloud bit')],\n",
       "  [('distance', 'be', 'better')],\n",
       "  [('we', 'summarize', 'type of scatterplots')],\n",
       "  [('explanatory power', 'be in', 'cluster-size vs. distance metric')],\n",
       "  [('all networks', 'be with', 'power-law degree distribution SWAE')],\n",
       "  [('LCAD', 'have', 'higher R2-values than SWD'),\n",
       "   ('γ =', 'be with', 'added triangles')],\n",
       "  [('curves', 'be in', 'Fig. 3C')],\n",
       "  [('degree distribution', 'do apparently influence considerably', 'outcome')],\n",
       "  [('largest component',\n",
       "    'apparently be still more influential than',\n",
       "    'others')],\n",
       "  [('networks', 'be with', 'more heterogeneous degree distributions')],\n",
       "  [('network', 'be with', 'converged component-size distribution'),\n",
       "   ('We', 'can assume', '𝜏'),\n",
       "   ('distances',\n",
       "    'be in',\n",
       "    'components of Erdős-Rényi model scales like log N')],\n",
       "  [('weak trends', 'be in', 'directions')],\n",
       "  [('epidemic threshold', 'be in', 'case')],\n",
       "  [('We',\n",
       "    'have investigate',\n",
       "    'characteristic time of SIR disease-spreading model')],\n",
       "  [('time', 'be', 'interesting')],\n",
       "  [('This', 'be', 'surprising')],\n",
       "  [('extinction time',\n",
       "    'would',\n",
       "    'Indeed in limit of high transmission probabilities would exactly proportional to average eccentricity weighted by size of components')],\n",
       "  [('they', 'use', 'only largest component')],\n",
       "  [('networks', 'be with', 'heterogeneous degree distributions')],\n",
       "  [('average degrees', 'be in', 'non-largest components'),\n",
       "   ('they', 'be under', 'epidemic threshold')],\n",
       "  [('it', 'come to', 'real epidemics'),\n",
       "   ('their network structure',\n",
       "    'be with',\n",
       "    'heterogeneous degree distributions')],\n",
       "  [('more research', 'be in', 'long-time behavior of epidemic models'),\n",
       "   ('work',\n",
       "    'inspire',\n",
       "    'more research in long-time behavior of epidemic models')],\n",
       "  [('We', 'use', 'individual-based SIR model')],\n",
       "  [('vertices', 'of pair be', 'edges')],\n",
       "  [('Time', 'proceed in', 'discrete time steps')],\n",
       "  [('i', 'be', 'Susceptible')],\n",
       "  [('next time step', 'be with', 'probability λ')],\n",
       "  [('Recovered vertices', 'stay', 'Recovered for rest of simulation'),\n",
       "   ('Recovered vertices', 'recover for', 'rest of simulation')],\n",
       "  [('we', 'measure from', 'run')],\n",
       "  [('value', 'be', 'unimodal')],\n",
       "  [('we', 'measure', 'maximal time to extinction maximum')],\n",
       "  [('0.1 %', 'be in', 'fewest possible measure'),\n",
       "   ('Brent', 'have', 'method locate')],\n",
       "  [('we', 'calculate', '𝜏')],\n",
       "  [('1 ≤ 𝑘 ≤ 𝑁 crete distribution', 'be with', 'probability mass function')],\n",
       "  [('This', 'be', 'emergent')],\n",
       "  [('much memory', 'be in', 'simulations')],\n",
       "  [('results', 'be in', 'large-N limit')],\n",
       "  [('generated graphs', 'be multigraph', 'where allowed')],\n",
       "  [('added edges', 'be in', 'configuration model')],\n",
       "  [('We', 'add', 'such to configuration model networks')],\n",
       "  [('ηM triangles', 'be', 'added')],\n",
       "  [('η', '=', '0.6')],\n",
       "  [('we', 'furthermore use', 'Erdős-Rényi model NumericCitation')],\n",
       "  [('We', 'still have', 'large components')],\n",
       "  [('distance measures', 'be in', 'graphs')],\n",
       "  [('number', 'be in', 'shortest path')],\n",
       "  [('number', 'be in', 'G')],\n",
       "  [('distant vertex', 'be in', 'same compo')],\n",
       "  [('SWAE', 'be expect eccentricity of', 'random vertex')],\n",
       "  [('two vertices', 'be in', 'component of randomly chosen vertex')],\n",
       "  [('name efficiency', 'be in', 'Ref')],\n",
       "  [('Its re-LCD 𝐺', '=', 'max')],\n",
       "  [('G', 'be', 'where largest component of G')],\n",
       "  [('Real-world networks', 'be', 'connected')],\n",
       "  [('same component', 'in tex be', '7'), ('tex', 'be in', 'same component')],\n",
       "  [('Szendroi B', 'be in', 'contact networks')],\n",
       "  [('Barthélemy M',\n",
       "    'velocity of',\n",
       "    'epidemic outbreaks in scale-free networks'),\n",
       "   ('epidemic outbreaks', 'be in', 'scale-free networks')],\n",
       "  [('Jansson S',\n",
       "    'law of',\n",
       "    'large numbers for SIR epidemic on random graph with given degrees'),\n",
       "   ('random graph', 'be with', 'given degrees')],\n",
       "  [('Vespignani A Epidemic spreading', 'be in', 'scale-free networks'),\n",
       "   ('Pastor-Santorras R', 'spreading in', 'scale-free networks')],\n",
       "  [('May RM', 'dynamics on', 'scale-free networks')],\n",
       "  [('average distances', 'be in', 'random graphs'),\n",
       "   ('average distances', 'be with', 'given expected degrees')],\n",
       "  [('2004 Sexual networks', 'be in', 'contemporary Western societies')],\n",
       "  [('infectious disease epidemics', 'use', 'networks'),\n",
       "   ('infectious disease epidemics', 'use', 'temporal networks')],\n",
       "  [('Erdős P', 'a on', 'Random Graphs'), ('Random Graphs', 'on a be', '1959')],\n",
       "  [('Buckley F', 'distance in', 'graphs'),\n",
       "   ('Harary F Distance', 'be in', 'graphs')],\n",
       "  [('P', 'vulnerability of', 'complex networks')],\n",
       "  [('Latora V', 'behavior of', 'small-world networks')],\n",
       "  [('power-law degree distribution', 'be with', 'parameter values')],\n",
       "  [('We', 'average', 'over 106 runs of disease simulation')],\n",
       "  [('B', 'display', 'result for configuration model with of points'),\n",
       "   ('configuration model', 'be with', 'of points')],\n",
       "  [('Error bars', 'be in', '𝜏'),\n",
       "   ('C', 'show', 'outcome for Erdős-Rényi model')],\n",
       "  [('configuration model', 'be with', 'exponent 2')],\n",
       "  [('abbreviations', 'be explain in', 'Methods section')],\n",
       "  [('Error bars', 'would', 'would have smaller than symbol size')],\n",
       "  [('B', 'be for', 'configuration model with added triangles'),\n",
       "   ('C', 'be for', 'Erdős-Rényi model'),\n",
       "   ('configuration model', 'be with', 'added triangles')]],\n",
       " 34: [[('2CompleX Lab', 'university of', 'Electronic Science'),\n",
       "   ('3 Big Data Research Center', 'university of', 'Electronic Science')],\n",
       "  [('locatin g', 'be with', 'great theoretical'),\n",
       "   ('locatin g', 'be in', 'ﬂuential nodes of complex networks'),\n",
       "   ('g', 'be', 'promising')],\n",
       "  [('DS centrality', 'locate in', 'ﬂuential spreaders')],\n",
       "  [('processes', 'be in', 'nature')],\n",
       "  [('epidemics', 'accelerate', 'innovation 8'),\n",
       "   ('essential proteins', 'be in', 'protein interaction networks 13'),\n",
       "   ('similar methods', 'identify', 'inﬂuential spreader in social networks 9'),\n",
       "   ('epidemics',\n",
       "    'predict',\n",
       "    'essential proteins in protein interaction networks 13'),\n",
       "   ('inﬂuential spreader', 'be in', 'social networks 9')],\n",
       "  [('inﬂuential spreaders', 'be in', 'networks')],\n",
       "  [('node', 'have', 'inﬂuence only accounting for 6 local surroundings')],\n",
       "  [('introduced dynamical processes', '0 such as', 'random walk 22 23')],\n",
       "  [('infectivity', 'be', 'when very high'),\n",
       "   ('spreading rate', 'be', 'when very small')],\n",
       "  [(\"nodes ' dynamical inﬂuences\",\n",
       "    'be in',\n",
       "    'susceptible-infected-r ecovered spreading model 32')],\n",
       "  [('their suggested index',\n",
       "    'thus only work well in',\n",
       "    'limited range of parameter space')],\n",
       "  [('A', 'be', 'symmetric')],\n",
       "  [('rate β', 'recover with', 'recovering rate µ')],\n",
       "  [('derivation', 'be in', 'Materials'),\n",
       "   ('I', 'be', 'where n × n unit matrix')],\n",
       "  [('n × 1 vector', 'be with', 'only ith element 1')],\n",
       "  [('elements', 'be in', 'columns')],\n",
       "  [('A', 'inﬂuence of', 'nodes')],\n",
       "  [('t', '=', '1')],\n",
       "  [],\n",
       "  [('node', 'have', 'inﬂuence')],\n",
       "  [('we', 'will show', 'lat er')],\n",
       "  [('2 Results', 'be with', 'varying spreading rate β')],\n",
       "  [('data description', 'be in', 'Materials')],\n",
       "  [('Kendall', 'have', 'Tau τ 36')],\n",
       "  [('Kendall', 'have', 'Tau τ for DS centrality')],\n",
       "  [('Kendall', 'by', 'Tau'), ('SIR model', 'be in', 'four real networks')],\n",
       "  [('similar result', 'show in', 'Fig. 2')],\n",
       "  [('A', 'be', 'symmetric'), ('A', 'be', 'real-valued')],\n",
       "  [('increase', 'with 8 be', '9')],\n",
       "  [('relationship', 'be in accordance with', 'results presented in Fig')],\n",
       "  [('we', 'calculate', 'DS centrality')],\n",
       "  [('inﬂuential ones', 'be in', 'average')],\n",
       "  [('Kendall', 'by', 'Tau')],\n",
       "  [('could principle deal', 'be with', 'networked Markov processes')],\n",
       "  [('contact', 'be with', 'its neighbors'),\n",
       "   ('contact', 'be infect with', 'probability β')],\n",
       "  [('we', 'set without', 'loss of generality'), ('µ', '=', '1')],\n",
       "  [('case', 'be with', 'µ =')],\n",
       "  [('Erd ¨ os', 'of feature be', 'degree hki')],\n",
       "  [('spreading rate', 'be', 'when small'),\n",
       "   ('its neighbors', 'of number be', '12'),\n",
       "   ('aij', 'be', 'where element of matrix A. Degree')],\n",
       "  [('index', 'be', 'better')],\n",
       "  [('k-shell', 'can', 'can obtained by so-called k-core decomposition 50')],\n",
       "  [('nodes', 'be with', 'degree k ≤ 2')],\n",
       "  [('Kendall', 'have', 'Tau')],\n",
       "  [('Kendall', 'by', 'Tau 36'), ('sgn', 'be', 'piecewise')],\n",
       "  [('larger τ', 'correspond to', 'better performance')],\n",
       "  [('data set', 'can', 'can freely downloaded from web site')],\n",
       "  [('Spain', 'of virgili be', 'URV')],\n",
       "  [('Protein 53', 'version of', 'proteome-scale map')],\n",
       "  [('Zhou', 'dynamics on', 'complex networks')],\n",
       "  [('S. Percolation transition', 'be in', 'dynamical trafﬁc net work'),\n",
       "   ('Li', 'transition in', 'dynamical trafﬁc net work')],\n",
       "  [('Colizza', 'be in', 'heterogeneous networks')],\n",
       "  [('P. Modeling Infectious Diseases', 'be in', 'Humans'),\n",
       "   ('Keeling', 'disease in', 'Humans')],\n",
       "  [('defense', 'be in', 'complex networks')],\n",
       "  [('Pei', 'dynamics in', 'complex networks'),\n",
       "   ('H. A. Spreading dynamics', 'be in', 'complex networks')],\n",
       "  [('Identiﬁcation', 'be in', 'complex networks')],\n",
       "  [('R. Competing activation mechanisms', 'be in', 'epidemics on networks'),\n",
       "   ('Castellano', 'mechanism in', 'epidemics on networks')],\n",
       "  [('spreading inﬂue nce', 'be in', 'complex networks')],\n",
       "  [('node spreading inﬂuence', 'be with', 'largest 19')],\n",
       "  [('Chen', 'node in', 'complex networks'),\n",
       "   ('T. Identifying inﬂuential nodes', 'be in', 'complex networks')],\n",
       "  [('Chen', 'node in', 'large-scale directed networks'),\n",
       "   ('T. Identifying inﬂuen tial nodes',\n",
       "    'be in',\n",
       "    'large-scale directed networks')],\n",
       "  [('Pei',\n",
       "    'searching for',\n",
       "    'superspreaders of information in real-world social media'),\n",
       "   ('superspreaders', 'be in', 'real-world social media')],\n",
       "  [('L.', 'leaders in', 'social networks'),\n",
       "   ('T. Leaders', 'be in', 'social networks')],\n",
       "  [('Li', 'spreader by', 'weighted LeaderRank')],\n",
       "  [('Ren', 'allocation for', 'ranking spreaders in complex networks'),\n",
       "   ('ranking spreaders', 'be in', 'complex networks')],\n",
       "  [('Borge-Holthoefer', 'spreader on', 'online social network')],\n",
       "  [('Borge-Holthoefer',\n",
       "    'absence of',\n",
       "    'inﬂuential spreaders in rumor dynamics'),\n",
       "   ('inﬂuential spreaders', 'be in', 'rumor dynamics')],\n",
       "  [('individual role', 'be in', 'collective dynamics')],\n",
       "  [('Bauer', 'be in', 'epidemic models')],\n",
       "  [(\"Barth ' elemy\", 'be in', 'scale-free networks')],\n",
       "  [('Zhou', 'behaviors of', 'susceptible-infected epidemics'),\n",
       "   ('scale-free networks',\n",
       "    'on zhou be',\n",
       "    'B. H. Behaviors of susceptible-infected epidemics'),\n",
       "   ('scale-free networks', 'be with', 'identical infectivity')],\n",
       "  [('Aral', 'member of', 'social networks')],\n",
       "  [('Ide', 'centrality in', 'Interconnected Networks'),\n",
       "   ('A. Diffusion Centrality', 'be in', 'Interconnected Networks')],\n",
       "  [('percolation', 'be in', 'networks')],\n",
       "  [('Ghanbarnejad',\n",
       "    'impact of',\n",
       "    'individual nodes in Boolean network dynamics'),\n",
       "   ('individual nodes', 'be in', 'Boolean network dynamics')],\n",
       "  [('Central Nodes', 'be in', 'Networks'),\n",
       "   ('Klemm', 'searchability of', 'Central Nodes in Networks')],\n",
       "  [('J. F. Critical phenomena', 'be in', 'complex networks'),\n",
       "   ('Dorogovtsev', 'phenomenon in', 'complex networks')],\n",
       "  [('Castellano', 'physics of', 'social dynamics')],\n",
       "  [('C. Synchronization', 'be in', 'complex networks'),\n",
       "   ('Arenas', 'synchronization in', 'complex networks')],\n",
       "  [('Guimera', 'structure in', 'network of human interactions'),\n",
       "   ('A. Self-similar community structure',\n",
       "    'be in',\n",
       "    'network of human interactions')],\n",
       "  [('Artiﬁci al Intelligence', 'on conference be', '2015')],\n",
       "  [('discussion', 'be with', 'Wen-Xu Wang'),\n",
       "   ('Erd ¨ os Number Project', 'be in', 'Oakland University')],\n",
       "  [('China', 'of foundation be', 'Shanghai'),\n",
       "   ('China', 'of project be', 'No')],\n",
       "  [('Special Appointment', 'of professor be', 'Eastern Scholar')]],\n",
       " 35: [[('Robust Information Source Estimator',\n",
       "    'be with',\n",
       "    'Sparse Observations Kai Zhu')],\n",
       "  [('piece', 'be in', 'network')],\n",
       "  [('TRODUC T ION', 'be in', 'paper')],\n",
       "  [('epidemics', 'news/rumors in', 'social networks'),\n",
       "   ('news/rumors', 'be in', 'social networks')],\n",
       "  [('NumericCitation',\n",
       "    'numericcitation under',\n",
       "    'homogeneous susceptible-infection')],\n",
       "  [('information source detection', 'be in', 'large-scale networks'),\n",
       "   ('information source detection', 'snapshot of', 'real world network'),\n",
       "   ('large-scale networks',\n",
       "    'in detection be',\n",
       "    'complete snapshot of real world network')],\n",
       "  [('most networks', 'be heterogeneous in', 'reality')],\n",
       "  [('regions', 'be with', 'poor medical care systems')],\n",
       "  [('it', 'be', 'Therefore important')],\n",
       "  [('based approach',\n",
       "    'locate information source with',\n",
       "    'sparse observations')],\n",
       "  [('Jordan infection center', 'be with', 'partial observation'),\n",
       "   ('Jordan infection center', 'be in', 'inﬁnite trees')],\n",
       "  [('value independent', 'be with', 'high probability')],\n",
       "  [('it', 'very appeal in', 'practice'), ('it', 'appeal in', 'practice')],\n",
       "  [('several related works', 'be in', 'area')],\n",
       "  [('general SIR model', 'be with', 'complete observations')],\n",
       "  [('all nodes', 'be in', 'network')],\n",
       "  [('our algorithm', 'signiﬁcantly outperform', 'DMP')],\n",
       "  [('we', 'introduce', 'heterogeneous SIR model for information propagation')],\n",
       "  [('Time', 'be', 'slotted')],\n",
       "  [('v', 'succeed with', 'probability quv')],\n",
       "  [('susceptible node',\n",
       "    'be successfully contact by',\n",
       "    'one of its infected neighbors')],\n",
       "  [('infected state', 'be in', 'Figure 1')],\n",
       "  [('details', 'will', 'will given in next section')],\n",
       "  [('We', 'further deﬁne', '·')],\n",
       "  [('homogeneous SIR model', 'be with', 'complete snapshot')],\n",
       "  [('heterogeneous SIR model', 'be with', 'sparse observations')],\n",
       "  [('MA', 'be in', 'R E SU LT S'), ('MA', 'be in', 'section')],\n",
       "  [('sample path based estimators', 'be in', 'section')],\n",
       "  [('tree networks', 'be with', 'inﬁnitely many levels'),\n",
       "   ('notation', 'be in', 'NumericCitation'),\n",
       "   ('we', 'adopt', 'notation in NumericCitation')],\n",
       "  [('nodes', 'be with', 'minimum observed infection eccentricity')],\n",
       "  [('it', 'use', 'Lemma 4'), ('it', 'repeatedly use', 'Lemma 4')],\n",
       "  [('partial snapshot Y', 'contain', 'at least one infected node')],\n",
       "  [('estimator', 'be', 'constant')],\n",
       "  [('gmin', 'qmin', 'lower bound on q')],\n",
       "  [('observed infection topology Y',\n",
       "    'be generate by',\n",
       "    'unbiased sampling algorithm')],\n",
       "  [('we', 'assume', 'node'), ('we', 'i.e. assume', 'node')],\n",
       "  [('result', 'will', 'will proved in Lemma 3 in Section VI'),\n",
       "   ('Lemma 3', 'be in', 'Section VI')],\n",
       "  [('we',\n",
       "    'have',\n",
       "    'smaller observed infection eccentricity than node u. Based on Lemma 3')],\n",
       "  [('it', 'occur with', 'higher probability')],\n",
       "  [('result', 'will', 'will proved in Lemma 4 in Section VI')],\n",
       "  [('example', 'be in', 'Figure 2')],\n",
       "  [('Jordan infection center',\n",
       "    'be with',\n",
       "    'observed infection eccentricity 1')],\n",
       "  [('d', 'be', 'where independent of size of infected subnetwork'),\n",
       "   ('d', 'be with', 'probability')],\n",
       "  [('O', 'be with', 'high probability'),\n",
       "   ('high probability', 'with o be', '1'),\n",
       "   ('O', 'be in', 'other words'),\n",
       "   ('other words', 'in o be', '1')],\n",
       "  [],\n",
       "  [('c', 'be', 'two survived nodes'),\n",
       "   ('information source', 'be in', 'Figure 3')],\n",
       "  [('two sampled nodes', 'be in', 'Figure 3')],\n",
       "  [('we', 'consider', 'Figure 3')],\n",
       "  [('its', 'identity', 'ID')],\n",
       "  [('This', 'be', 'surprising')],\n",
       "  [('path', 'be', 'robust')],\n",
       "  [('weighted graph',\n",
       "    'be with',\n",
       "    'link weights chosen proportionally according SIR parameters')],\n",
       "  [('V. S IMU LAT ION S', 'be in', 'section')],\n",
       "  [('we', 'use in', 'simulation')],\n",
       "  [('information source', 'be', 'randomly selected')],\n",
       "  [('total number', 'be in', 'infection')],\n",
       "  [('infection graph reports', 'be with', 'probability σ')],\n",
       "  [],\n",
       "  [('We', 'evaluate', 'performance on different networks')],\n",
       "  [],\n",
       "  [('sample probability', 'be', 'When larger than 6 %')],\n",
       "  [('similar performance', 'be with', 'RI')],\n",
       "  [('we', 'again vary σ from', '0.01 to 0.1'),\n",
       "   ('we', 'vary σ from', '0.01 to 0.1')],\n",
       "  [('results', 'be show in', 'Figure 4b')],\n",
       "  [('difference', 'be in', 'terms of average number of hops')],\n",
       "  [('.020.040.060.080.166.577.588.59 Sample Probability σAverage Distance',\n",
       "    'be in',\n",
       "    'section')],\n",
       "  [('network', 'be', 'sparse')],\n",
       "  [('simulation results', 'be show in', 'Figure 4c')],\n",
       "  [('nodes', 'be in', 'network')],\n",
       "  [('simulation results', 'be show in', 'Figure 4d')],\n",
       "  [('sample probability', 'be', 'when large')],\n",
       "  [('performance', 'be with', 'sparse observations')],\n",
       "  [('machine', 'be with', '1.8 GB memory')],\n",
       "  [('algorithms', 'be implement in', 'Python 2.7')],\n",
       "  [('it', 'take', 'DMP 229.12 seconds'),\n",
       "   ('it', 'locate estimator for', 'one snapshot')],\n",
       "  [('RI', 'so be much faster than', 'DMP'),\n",
       "   ('RI', 'be much faster than', 'DMP')],\n",
       "  [('RI', 'therefore outperform', 'DMP')],\n",
       "  [('optimal sample path', 'be with', 'node')],\n",
       "  [('Time Inequality', 'consider', 'inﬁnite tree rooted at vr')],\n",
       "  [('observed snapshot Y', 'contain', 'at least one infected node')],\n",
       "  [('We', 'adopt below', 'set of children of v')],\n",
       "  [('tree', 'be root in', 'v')],\n",
       "  [('we', 'assign', '0 t')],\n",
       "  [('Step 2', 'consider', 'i.e. Y ∈ Y n +1'),\n",
       "   ('inequality', 'hold for', 'k ≤ n')],\n",
       "  [('following notations', 'will', 'will used in later analysis')],\n",
       "  [('Pictorial Description', 'be in', 'probability')],\n",
       "  [('we', 'consider give', 'Given following two events')],\n",
       "  [('l', 'node at', 'bottom survived'), ('φ u', 'be in', 'Event 2')],\n",
       "  [('network', 'be at', 'most L − 1')],\n",
       "  [('we', 'consider', 'event 2'),\n",
       "   ('information', 'propagate for', 'time slots')],\n",
       "  [('T', 'be', 'infected'), ('T', 'be', 'observed')],\n",
       "  [],\n",
       "  [('Two Auxiliary Processes', 'be in', 'Lemma')]],\n",
       " 36: None,\n",
       " 37: [[('epidemic models', 'be in', 'dimensions')],\n",
       "  [('they', 'be immune to', 'further infection'),\n",
       "   ('they', 'be', 'after which immune to further infection')],\n",
       "  [('Dawson-Watanabe process', 'be with', 'location-dependent killing')],\n",
       "  [('Adler', 'of', '1993 conjecture')],\n",
       "  [('analogous results', 'for aldous be', '1997'),\n",
       "   ('analogous results', 'be in', 'simpler setting of mean-ﬁeld models')],\n",
       "  [('spatial SIR epidemics', 'be in', 'dimensions')],\n",
       "  [('integer lattice Zd', 'be in', 'd dimensions')],\n",
       "  [('they', 'be', 'after which immune'),\n",
       "   ('they', 'be', 'after which immune to further infection')],\n",
       "  [('scale', 'be with', 'village size N'),\n",
       "   ('scale', 'be in', 'way'),\n",
       "   ('expected number', 'be in', 'otherwise healthy population')],\n",
       "  [('Our', 'result below', 'Theorem 2')],\n",
       "  [('Dawson-Watanabe measure-valued diffusion', 'be in', 'd =')],\n",
       "  [('Dawson-Watanabe process', 'be with', 'killing')],\n",
       "  [('branching random walk',\n",
       "    'be',\n",
       "    'often referred to as branching envelope of epidemic')],\n",
       "  [('infection attempts', 'be in', 'coupled epidemic'),\n",
       "   ('some', 'be realize in', 'epidemic'),\n",
       "   ('targets', 'be target of', 'other simultaneous infection attempts')],\n",
       "  [('i.i.d.', 'be with', 'Binomial')],\n",
       "  [('Poisson distribution', 'be with', 'mean 1')],\n",
       "  [('its branching envelope', 'be in', 'way'),\n",
       "   ('number', 'be in', 'branching envelope')],\n",
       "  [('particles', 'be', 'Initially red')],\n",
       "  [('particles', 'neighbor site accord to', 'law RN described above')],\n",
       "  [('epidemic will', 'diverge signiﬁcantly from', 'branching envelope')],\n",
       "  [('Our', 'result below', 'Theorem 2')],\n",
       "  [('its branching envelope', 'be in', 'second')],\n",
       "  [('sum', 'behavior of', 'occupation time statistics branching'),\n",
       "   ('number', 'of sum be', 'asymptotic behavior')],\n",
       "  [('Watanabe', 'have', 'Theorem')],\n",
       "  [('1/k', 'by mass be', 'that')],\n",
       "  [('Watanabe', 'have', 'Theorem')],\n",
       "  [('k', 'be', 'branching'),\n",
       "   ('k', 'be', 'random'),\n",
       "   ('branching random walk', 'be with', 'offspring distribution RN')],\n",
       "  [('X k', 'be', 'corresponding counting measure'),\n",
       "   ('site Zd', 'be in', 'generation')],\n",
       "  [('standard Dawson-Watanabe process Xt', 'be with', 'variance parameter σ2'),\n",
       "   ('variance parameter σ2', 'with xt be', 'super-Brownian motion')],\n",
       "  [('Watanabe', 'of', 'Theorem')],\n",
       "  [('Watanabe', 'from', 'theorem'),\n",
       "   ('random walks', 'converge to', 'those of super-Brownian motion')],\n",
       "  [('D', 'in path be', '0 ∞'),\n",
       "   ('Xt', 'have', '0 ∞'),\n",
       "   ('Proposition 2.15', 'be in', 'Etheridge'),\n",
       "   ('continuous sample paths', 'be in', 'D')],\n",
       "  [('Watanabe', 'from', 'theorem')],\n",
       "  [('sample paths', 'be in', 'space of absolutely continuous measures')],\n",
       "  [('t', 'x for', 't >')],\n",
       "  [('assumptions', 'be in', \"Watanabe 's Theorem\"),\n",
       "   ('Watanabe', 'in', 'Theorem')],\n",
       "  [('Xt', 'be', 'singular')],\n",
       "  [('Sugitani', 'of', 'theorem')],\n",
       "  [('Sugitani', 'have', 'Theorem'), ('reader', 'for', 'convenience')],\n",
       "  [('d/2 Sugitani', 'have', 'Theorem')],\n",
       "  [('= X0', 'be in', 't ≥ 0')],\n",
       "  [('density process', 'of lt be', 'x')],\n",
       "  [],\n",
       "  [('Sugitani', 'have', 'theorems')],\n",
       "  [('k', 'be', 'branching'),\n",
       "   ('offspring distribution', 'be', 'Poisson'),\n",
       "   ('k', 'be', 'random'),\n",
       "   ('Poisson', 'be with', 'mean 1'),\n",
       "   ('offspring distribution', 'be poisson with', 'mean 1')],\n",
       "  [('initial conﬁgurations', 'satisfy', '4'),\n",
       "   ('Sugitani', 'of', 'theorem'),\n",
       "   (\"Sugitani 's theorem\", 'of hypothesis be', '6'),\n",
       "   (\"Watanabe 's theorem\", 'of hypothesis be', '4'),\n",
       "   ('Watanabe', 'of', 'theorem')],\n",
       "  [('weak convergence', 'be in', 'topology of D')],\n",
       "  [('super-Brownian motion', 'be with', 'variance parameter σ2')],\n",
       "  [('Theorem 1', 'will', 'will proved'),\n",
       "   ('Theorem 1', 'will', 'will proved in § 2')],\n",
       "  [('Poisson', 'be with', 'mean 1')],\n",
       "  [('offspring distribution', 'have', 'exponentially decaying tail')],\n",
       "  [('Sufﬁcient condition s', 'be give below in', 'Proposition 1')],\n",
       "  [('Xt', 'be super-brownian motion in', 'dimension d')],\n",
       "  [('Proposition 35', 'be in', 'Lalley')],\n",
       "  [('Dawson-Watanabe processes', 'be with', 'variable-rate killing'),\n",
       "   ('we', 'state', 'our result')],\n",
       "  [('Dawson-Watanabe process Xt', 'be with', 'killing rate θ'),\n",
       "   ('martingale',\n",
       "    'be with',\n",
       "    'same quadratic variation as super-Brownian motion'),\n",
       "   ('super-Brownian motion', 'be with', 'variance parameter σ2')],\n",
       "  [],\n",
       "  [('Existence', 'be in', 'general'),\n",
       "   ('distributional', 'prove in', 'various cases')],\n",
       "  [('standard Dawson-Watanabe process', 'be with', 'same variance parameter')],\n",
       "  [('Dawson-Watanabe process', 'be with', 'killing')],\n",
       "  [('Z', 'be with', 'village size N'),\n",
       "   ('number', 'be', 'critical'),\n",
       "   ('It', 'be show in', '2007'),\n",
       "   ('Dawson-Watanabe process', 'be with', 'location-dependent killing')],\n",
       "  [('SIR-d model', 'be with', 'village size N')],\n",
       "  [('initial FN α µN', 'be with', 'compact support')],\n",
       "  [('Dawson-Watanabe process',\n",
       "    'be with',\n",
       "    'initial conﬁguration variance parameter σ2'),\n",
       "   ('limit process Xt',\n",
       "    'be',\n",
       "    'where Dawson-Watanabe process with initial conﬁguration variance parameter σ2')],\n",
       "  [('Theorem 2', 'will', 'will proved'),\n",
       "   ('Theorem 2', 'will', 'will proved in § 3')],\n",
       "  [('epidemic',\n",
       "    'now look like',\n",
       "    'branching random walk with location-dependent killing'),\n",
       "   ('branching random walk', 'be with', 'location-dependent killing')],\n",
       "  [('α =', 'be in', 'four dimensional case'),\n",
       "   ('α =', 'be in', 'dimensions d 5')],\n",
       "  [('probability', 'in yn/f be', 'n'), ('Yn/f → 0', 'be in', 'probability')],\n",
       "  [('LOCA L T IME', 'be in', 'd = 3 2.1')],\n",
       "  [('will crucial importance', 'be in', 'proof of Theorem 1')],\n",
       "  [('We', 'collect', 'several useful estimates'),\n",
       "   ('We', 'collect here', 'several useful estimates')],\n",
       "  [('Gauss kernel', 'be in', 'Rd')],\n",
       "  [('γ ≤ 1', 'for 1 be', 'Pn x −'),\n",
       "   ('γ ≤ 1', 'for 1 be', '21'),\n",
       "   ('γ ≤ 1', 'for 1 be', '20')],\n",
       "  [('Our arguments',\n",
       "    'will also require',\n",
       "    'following estimates on discretized Green kernel')],\n",
       "  [('β parameter', 'β', '2')],\n",
       "  [('Sugitani', \"'s theorem be\", 'Z t 0 Xs ds = Zx Lt ψ dx'),\n",
       "   ('Sugitani', 'have', 'Theorem')],\n",
       "  [('sequence', 'be in', 'in view of hypothesis')],\n",
       "  [('Lemma 6', 'lemma in', 'Sugitani'), ('Lemma 3.1', 'be in', 'Sugitani')],\n",
       "  [('X', 'be', 'random'), ('random variable', 'be with', 'moment generating')],\n",
       "  [('we', 'obtain', 'n'),\n",
       "   ('monotone', 'be in', 'ψ'),\n",
       "   ('we', 'obtain n in', 'general linear')],\n",
       "  [('interest', 'of case be', 'Poisson distribution with mean 1'),\n",
       "   ('case', 'distribution with', 'mean 1'),\n",
       "   ('origin', 'be in', 'Zd'),\n",
       "   ('origin', 'be include in', 'collection'),\n",
       "   ('Poisson distribution', 'be with', 'mean 1'),\n",
       "   ('offspring distribution', 'be in', 'case of interest')],\n",
       "  [('µ', 'r. by', 'arguments of preceding paragraph')],\n",
       "  [('β >', 'for β be', 'y ∈ Zd as 1 Φl'), ('β', 'zd as', '1 Φl')],\n",
       "  [('h ≥ 2', 'for 35 be', 'n ∈'),\n",
       "   ('h ≥ 2', 'for 35 be', 'n |'),\n",
       "   ('h ≥ 2', 'for 35 be', 'κh')],\n",
       "  [('Lemma 3', 'of assertion be', '21'), ('κ1', 'be in', 'fact')],\n",
       "  [('n', 'for κ1 be', '|')],\n",
       "  [('of < h', 'for deﬁnition be', '33')],\n",
       "  [('h Xm', '=', 'h')],\n",
       "  [('1 √ ka − 1 √ kb', 'estimate', '|')],\n",
       "  [('µk', 'satisfy', 'hypotheses of Theorem 1')],\n",
       "  [('k', 'for b be', '√ k'), ('assumption', 'by b be', '0 r')],\n",
       "  [('k large', 'for hµk be', 'Xl <'), ('k large', 'for hµk be', 'Xl < kt')],\n",
       "  [('Lemma 6', 'by t be', 'a')],\n",
       "  [('positive integers', 'of tuple be', 'h1')],\n",
       "  [('claim', 'be in', 'Part A')],\n",
       "  [('We', 'be', 'ready')],\n",
       "  [('we', 'set', 'ψ'), ('1 √ ka', 'use', '38')],\n",
       "  [('n +1', 'be in', 'terms of ψ'),\n",
       "   ('terms', 'in +1 be', 'x'),\n",
       "   ('κh', '+1 in', 'terms of ψ')],\n",
       "  [('one',\n",
       "    'similarly can verify kolmogorov-centsov criterion for',\n",
       "    'time variable')],\n",
       "  [('d', '=', '3')],\n",
       "  [('particles', 'be in', 'ball of radius 3C1 k1/6')],\n",
       "  [('µk', 'be', 'decreasing'), ('decreasing function', 'be in', '| y |')],\n",
       "  [('proposition', 'be', 'natural'),\n",
       "   ('proposition', 'be natural analogue of', '1989'),\n",
       "   ('Sugitani', 'in proposition be', '1989'),\n",
       "   ('Proposition 1', 'be in', 'Sugitani')],\n",
       "  [('we', 'prove', 'Proposition 1')],\n",
       "  [('x ∈ Zd', 'for y be', 'Ψk t')],\n",
       "  [('convergence', 'be', 'locally uniform')],\n",
       "  [('it', 'show', 'that')],\n",
       "  [('we', 'use ascoli-arzela criterion for', 'this')],\n",
       "  [('Ψk t − Ψk s 2ε', 'for ε be', 't ≤ δ')],\n",
       "  [('t ≥ δ =', 'for t be', 'x')],\n",
       "  [('Ψk', 'be in', 'local uniform topology')],\n",
       "  [('other hand', 'on compact be', 'µkGkt'), ('C', 'in compact be', 'µkGkt')],\n",
       "  [('qσ2 t', 'be in', 'C'), ('C', 'in t be', 'x')],\n",
       "  [('k2 − d/2', 'verify', 'Ascoli-Arzela criterion')],\n",
       "  [('Spitzer', 'proposition on', 'p72'), ('t ≥ δ', 'for time be', 'µkGkt')],\n",
       "  [('they', 'be', 'equi-continuous')],\n",
       "  [('y ∈ Zd', 'be with', '| y'), ('k', 'for y be', 'C1 k1/6')],\n",
       "  [('we', 'term', 'I')],\n",
       "  [('µk', 'be', 'decreasing')],\n",
       "  [('µk', 'be', 'decreasing'), ('decreasing function', 'be in', '| y |')],\n",
       "  [('O Z o 0 contradiction', 'be with', '51')],\n",
       "  [('f', 'have', 'support'), ('f', 'have', 'compact support')],\n",
       "  [('decreasing functions', 'be in', '| compact support'),\n",
       "   ('f', 'be', 'decreasing')],\n",
       "  [('sup i1', 'be in', 'n Xk =')],\n",
       "  [('n Xk', '=', '1 3')],\n",
       "  [('appropriate Dawson-Watanabe process', 'of ratio be', '12'),\n",
       "   ('likel ihood ratios', 'converge weakly to', '12'),\n",
       "   ('law', 'be with', 'village size N')],\n",
       "  [('it', 'work with', 'likelihood ratio for modiﬁcation of SIRd epidemic')],\n",
       "  [('blue particles', 'of offspring be', 'A')],\n",
       "  [('Theorem 2', 'of hypothesis be', '14'),\n",
       "   ('absolute difference', 'be in', 'number of infected individuals'),\n",
       "   ('common probability space', 'be in', 'way')],\n",
       "  [('modiﬁed SIR-d epidemic',\n",
       "    'be',\n",
       "    'after Feller-rescaling indistinguishable')],\n",
       "  [('numbers', 'be in', 'SIR-d epidemic'),\n",
       "   ('Proposition 3', 'be in', 'Lalley'),\n",
       "   ('Lalley', 'in proposition be', '2007')],\n",
       "  [('These', 'be give below in', 'Lemma 9')],\n",
       "  [('Proposition 28', 'be in', 'Lalley')],\n",
       "  [('0 Xz Pi', 'proof of', 'Lemma 9')],\n",
       "  [('site x', 'at n be', 'x'),\n",
       "   ('susceptible individual η', 'for n be', 'x'),\n",
       "   ('pair ξ', 'ζ of', 'infected individuals'),\n",
       "   ('pair ξ', 'ζ at', 'neighboring sites'),\n",
       "   ('collisions', 'of n be', 'x')],\n",
       "  [('dominance',\n",
       "    'by xx be',\n",
       "    'position of initial particles of our epidemic model'),\n",
       "   ('58 Xn ≤ N α t Xx', 'position of', 'initial particles')],\n",
       "  [('α ≤ 2/3', 'since o be', 'N × N α')],\n",
       "  [('α ≤ 1/2', 'since o be', 'N × N α'),\n",
       "   ('dimension 2', 'in pn be', 'x'),\n",
       "   ('log n Pn', 'be in', 'therefore dimension 2'),\n",
       "   ('Lemma 10', 'by pn be', 'x')],\n",
       "  [('errant infection attempts', 'of number be', 'n x')],\n",
       "  [('same argument', 'by op be', 'N α')],\n",
       "  [('pairs ̺', 'of n be', 'x'), ('This', 'be bound by', 'x')],\n",
       "  [('dimension 2', 'be in', 'dimension 3')],\n",
       "  [('we', 'enlarge', 'C')],\n",
       "  [('it',\n",
       "    'prove',\n",
       "    'corresponding result for modiﬁed SIR epidemic deﬁned in § 3.2')],\n",
       "  [('we', 'shall analyze', 'likelihood ratios'),\n",
       "   ('we', 'shall analyze likelihood ratio for', 'this')],\n",
       "  [('there can most one errant infection attempt', 'be in', 'modiﬁed process'),\n",
       "   ('offspring', 'be in', 'branching envelope'),\n",
       "   ('collisions', 't. give', 'evolution of process'),\n",
       "   ('up time', 'to collision be', 't. Given evolution of process')],\n",
       "  [('same calculation', 'be in', 'as Lalley')],\n",
       "  [('ﬁrst sum', 'be in', 'exponential'), ('increments', 'be in', 't')],\n",
       "  [('N', 'be in', 'equation'),\n",
       "   ('equation', 'in n be', 'x'),\n",
       "   ('quantities', 'furthermore ∆', 'x')],\n",
       "  [('analogous representation', 'be in', 'one-dimensional case'),\n",
       "   ('stochastic integration', 'on walsh be', '1986'),\n",
       "   ('background', 'for walsh be', '1986')],\n",
       "  [('M N', 'be', 'deﬁned')],\n",
       "  [('ﬁnite Borel measure µ', 'against hµ be', 'ψi for integral of ψ'),\n",
       "   ('hµ', 'ψi for', 'integral of ψ')],\n",
       "  [('ﬁrst sum', 'be in', 'exponential of equation')],\n",
       "  [('Dawson-Watanabe process', 'be with', 'initial conﬁguration µ')],\n",
       "  [('same argument', 'be in', 'as Proposition 4 of Lalley'),\n",
       "   ('Lalley', 'of proposition be', '2007')],\n",
       "  [('θN', 'follow from', 'Theorem 1')],\n",
       "  [('N → ∞', 'as d/2 be', '64 dQN dP N − → 1 in probability'),\n",
       "   ('64 dQN dP N − → 1', 'be in', 'probability'),\n",
       "   ('3 − d/2', '→ in', 'probability'),\n",
       "   ('P N', 'under d/2 be', '64 dQN dP N − → 1 in probability')],\n",
       "  [('same argument', 'be in', 'as one-dimensional case'),\n",
       "   ('second sum', 'zero in', 'probability'),\n",
       "   ('equation', 'be in', 'Lalley'),\n",
       "   ('Lalley', 'in equation be', '60'),\n",
       "   ('second sum', 'be in', 'likelihood ratio')],\n",
       "  [('it', 'complete', 'proof of Theorem 2')],\n",
       "  [('two sums', 'be in', 'exponential of equation'),\n",
       "   ('corresponding integrals', 'be in', 'exponential of equation')],\n",
       "  [('Lalley', 'of corollary be', '2007'),\n",
       "   ('same argument', 'be in', 'as proof of Corollary 4 of Lalley'),\n",
       "   ('Lalley', 'of corollary be', 'Xn Xx ∆ N n ̺N n = Z Z θN')],\n",
       "  [('proof', 'be in', 'Lalley')],\n",
       "  [('∆ N n', 'be in', 'sum theorem'),\n",
       "   ('Watanabe', 'have', 'λN n'),\n",
       "   ('Watanabe', \"'s n be\", 'x'),\n",
       "   ('∆ N n', 'be in', 'particular'),\n",
       "   ('sum', 'in n be', 'x'),\n",
       "   ('particular', 'in n be', 'x')],\n",
       "  [('this', 'reduce to', 'x')],\n",
       "  [('dimension 2', 'be in', 'dimension 3')],\n",
       "  [('we', 'enlarge', 'C')],\n",
       "  [('We', 'begin with', 'unbounded region')],\n",
       "  [('points', 'be in', 'ball of radius 1/2 centered at origin')],\n",
       "  [('This', 'be', 'decreasing')],\n",
       "  [('Lawler', 'be in', 'case')],\n",
       "  [('it', 'prove', '18')],\n",
       "  [('y', 'be with', 'length 1'),\n",
       "   ('Qy', 'be', 'cube centered at y with length 1')],\n",
       "  [('Λj', 'be with', 'Λi')],\n",
       "  [('h1', 'for φl be', 'u')],\n",
       "  [('Boston', 'in adler be', 'Superprocess local l ocal times'),\n",
       "   ('Boston', 'in adler be', 'R. J.'),\n",
       "   ('Birkhäuser Adler', 'be in', 'Boston')],\n",
       "  [('Dawson', 'be in', 'dence')],\n",
       "  [('Etheridge', 'introduction to', 'superprocesses')],\n",
       "  [('Fleischmann', 'behavior of', 'measure')],\n",
       "  [('Springer-Verlag 2nd ed.', 'text in', 'Mathematics'),\n",
       "   ('graduate Texts', 'be in', 'Mathematics'),\n",
       "   ('Spitzer F.', 'principle of', 'random'),\n",
       "   ('Martin-L öf',\n",
       "    'size of',\n",
       "    'nearly critical ep Wiener process to parabolic barrier')],\n",
       "  [('measure-valu', 'branch', 'diffusion processes')],\n",
       "  [('Berlin', 'in walsh be', 'XIV'), ('Walsh', 'be in', 'Berlin')],\n",
       "  [('Lecture Notes', 'be in', 'Math.')],\n",
       "  [('Mathematics', 'of department be', 'Canada V6T 1Z2 E-mail address')]],\n",
       " 38: [[('Systems', '2department of', 'Mathematics'),\n",
       "   ('SIR', 'be in', 'Temporal Networks Kaiyuan Sun ,1 Andrea Baronchelli ,2')],\n",
       "  [('we', 'study', 'the')],\n",
       "  [('spreading process', 'be in', 'SIR models'),\n",
       "   ('epidemic threshold', 'reduce', 'ﬁnal fraction of recovered nodes')],\n",
       "  [('disease', 'be in', 'endemic state')],\n",
       "  [('heterogeneity', 'be in', 'tie strengths')],\n",
       "  [('We', 'study', 'processes on two real temporal networks')],\n",
       "  [('individuals', 'exchange', 'information')],\n",
       "  [('Recent advances', 'be in', 'technology')],\n",
       "  [('network', 'have', 'temporal properties')],\n",
       "  [('heterogeneity', 'be in', \"nodes ' activities\")],\n",
       "  [('strong ties', 'be with', 'their peers')],\n",
       "  [('its understanding', 'be in', 'context of time-varying graphs')],\n",
       "  [('studies', 'nonetheless have reveal', 'rich phenomenology')],\n",
       "  [('individual', 'of', 'contacts')],\n",
       "  [('fraction', 'be in', 'endemic state'),\n",
       "   ('eﬀect', 'be in', 'SIR processes'),\n",
       "   ('threshold', 'be in', 'SIS dynamics')],\n",
       "  [('epidemic threshold', 'be shift to', 'larger values')],\n",
       "  [('disease', 'of', 'parameters')],\n",
       "  [('SIS-like diseases', 'be in', 'tightly connected clusters')],\n",
       "  [('role', 'be in', 'case'),\n",
       "   ('we', 'isolate', 'role of non-Markovian dynamics in case')],\n",
       "  [('driven models', 'be', 'In their basic formulation activity memoryless')],\n",
       "  [('Non-active nodes',\n",
       "    'can still receive',\n",
       "    'connections from other active vertices')],\n",
       "  [('t', 'edge in', 'network Gt')],\n",
       "  [('Gt', 'be', 'random'),\n",
       "   ('Gt', 'be', 'simple'),\n",
       "   ('simple random graph', 'be with', 'low average connectivity'),\n",
       "   ('network Gt', 'be simple random graph with', 'low average connectivity')],\n",
       "  [('that', 'so numericcitation be', 'FIG. 1')],\n",
       "  [('same activity distribution', 'be in', 'cases'),\n",
       "   ('simulation scenario', 'be in', 'which for models')],\n",
       "  [('nodes', 'be with', 'high activity rate')],\n",
       "  [('links', 'be in', 'real-world networks')],\n",
       "  [('ck', 'be', 'constant'), ('ck', 'be', 'dependent'), ('ck', 'be', '=')],\n",
       "  [('it', 'set for', 'k'),\n",
       "   ('it', 'generalize', \"individuals ' memory NumericCitation\")],\n",
       "  [('random', 'at probability be', 'ni')],\n",
       "  [],\n",
       "  [('LCC', 'interestingly appear earlier in', 'ML networks')],\n",
       "  [('individuals', 'activate', 'previous connections'),\n",
       "   ('individuals', 'activate', 'connections')],\n",
       "  [('I. SIR', 'be in', 'ACTIVITY DRIVEN NETWORKS')],\n",
       "  [('can susceptible compartments', 'be in', 'SIR model nodes')],\n",
       "  [('C Emergence', 'be in', 'ML')],\n",
       "  [('function', 'as lcc be', 'LCC/N')],\n",
       "  [('we', 'consider', '102 independent realizations')],\n",
       "  [('Infected nodes', 'spread', 'it'), ('nodes', 'spread', 'it')],\n",
       "  [('Susceptible nodes', 'infect with', 'rate β'),\n",
       "   ('Susceptible nodes', 'be in', 'contact with infected individuals'),\n",
       "   ('Susceptible nodes', 'become', 'infected with rate β'),\n",
       "   ('contact', 'be with', 'infected individuals')],\n",
       "  [('second transition', 'describe', 'recovery process')],\n",
       "  [('Infected individuals', 'recover permanently with', 'rate µ')],\n",
       "  [('quantity ξS IR', 'deﬁnes', 'epidemic threshold of process')],\n",
       "  [('β µ ≥ ξS', 'be in', 'ML networks')],\n",
       "  [('threshold', 'be', 'Interestingly same i.e. ξS I S = ξS IR')],\n",
       "  [('second order phase transition', 'be with', 'control parameter β')],\n",
       "  [('SIR spreading', 'be in', 'ML'),\n",
       "   ('ML', 'in spreading be', 'blue circles')],\n",
       "  [('µ', '=', '5 × 10 − 3 β'), ('R ∞', 'be in', 'ML networks')],\n",
       "  [('epidemic threshold', 'be in', 'WM networks'),\n",
       "   ('epidemic threshold', 'be clearly larger than', 'than ML systems')],\n",
       "  [('systems', 'less vulnerable to', 'disease spreading')],\n",
       "  [('infected nodes', 'be in', 'WM networks')],\n",
       "  [('numerical estimation', 'be in', 'of infected compartment')],\n",
       "  [('inﬁnite', 'be in', 'SIS processes for values of β'),\n",
       "   ('SIS processes', 'in inﬁnite be', 'endemic state'),\n",
       "   ('coverage', 'reach', '1')],\n",
       "  [('susceptibility χ', 'be in', 'standard percolation theory')],\n",
       "  [('coverage', 'remain below', 'threshold C')],\n",
       "  [('system', 'more fragile to', 'disease spreading'),\n",
       "   ('repeated connections', 'be in', 'ego-centered networks of node')],\n",
       "  [('SIS spreading', 'be in', 'ML'),\n",
       "   ('ML', 'in spreading be', 'blue circles')],\n",
       "  [('last point', 'be in', 'plot')],\n",
       "  [('SIS', 'be in', 'REAL TIME VARYING NETWORKS')],\n",
       "  [('We', 'adopt', 'time resolution of one year')],\n",
       "  [('step', 'be', 'at time preserved In Figure 5-A')],\n",
       "  [('it', 'of version be', 'blue circles')],\n",
       "  [('its randomized version', 'in time be', 'L'),\n",
       "   ('SIS process', 'be in', 'original Twitter network'),\n",
       "   ('SIS process', 'of time be', 'L'),\n",
       "   ('time', 'be in', 'its randomized version considering')],\n",
       "  [('threshold', 'be in', 'original network')],\n",
       "  [('dynamical properties', 'be in', 'activity')],\n",
       "  [('epidemic threshold', 'be in', 'basic activity')],\n",
       "  [('system', 'more resilient to', 'disease spreading')],\n",
       "  [('systems', 'more prone to', 'disease invasion'),\n",
       "   ('disease', 'of', 'parameters')],\n",
       "  [('SIS-like diseases',\n",
       "    'be in',\n",
       "    'local groups of tightly connected individuals'),\n",
       "   ('heterogeneity', 'be in', \"ties ' strength\")],\n",
       "  [('they', 'disappearance of', 'old ones')],\n",
       "  [('introduction', 'be in', 'modeling framework'),\n",
       "   ('10 − 2', 'network in', 'Twitter'),\n",
       "   ('network', 'be in', 'Twitter'),\n",
       "   ('10 − 2', 'network of', 'communications')],\n",
       "  [('memory',\n",
       "    'corroborate',\n",
       "    'important role played by non-Markovian dynamics on dynamical processes unfolding on temporal networks NumericCitation')]],\n",
       " 39: None,\n",
       " 40: [[('common approach', 'be in', 'epidemiology'),\n",
       "   ('transmission', 'be in', 'population')],\n",
       "  [('ma jor breakthrough', 'be in', 'theoretical computer science')],\n",
       "  [('members', 'fall into', 'one of three classes')],\n",
       "  [('much recent work',\n",
       "    'thus have focus on',\n",
       "    'role of network of disease-transmitting contacts')],\n",
       "  [('McKendrick', 'have', 'ODE model')],\n",
       "  [('real-world social contact networks',\n",
       "    'however exhibit',\n",
       "    'complex patterns of interconnection between individuals'),\n",
       "   ('real-world social contact networks',\n",
       "    'exhibit',\n",
       "    'complex patterns of interconnection between individuals')],\n",
       "  [('ﬁrst author', 'acknowledge', 'support from NIH Grant K25 AI079404-02')],\n",
       "  [('ﬁxed graph G', 'be with', 'labelling Pr'),\n",
       "   ('ﬁxed graph G', 'be in', 'formalism'),\n",
       "   ('G', 'be', 'ﬁxed'),\n",
       "   ('G', 'be', 'In formalism ﬁxed graph G with labelling Pr')],\n",
       "  [('enormous amount', 'be in', 'theoretical epidemiology')],\n",
       "  [('mathematical formalism', 'describe', 'spread of infectious diseases')],\n",
       "  [('Such calculations', 'be', 'based')],\n",
       "  [('its creation', 'be in', 'early nineties')],\n",
       "  [('network', 'have', 'operation')],\n",
       "  [('same formalism', 'explain above for', 'social networks')],\n",
       "  [('computers', 'be in', 'P'),\n",
       "   ('communications network G', 'set of', 'computers')],\n",
       "  [('we',\n",
       "    'state above in',\n",
       "    'context of epidemics on social contact networks')],\n",
       "  [('problem', 'be', 'computationally intractable')],\n",
       "  [('standard benchmark', 'be class of', 'NP-complete problems')],\n",
       "  [('unknown PROBABILITY', 'be in', 'SIR NETWORK'),\n",
       "   ('algorithm', 'be with', 'polynomial running time')],\n",
       "  [('running time', 'be bound by', 'polynomial'),\n",
       "   ('running time', 'be bound by', 'polynomial')],\n",
       "  [('Such problems', 'be term', 'NP-hard'), ('Such problems', 'be', 'termed')],\n",
       "  [('ﬁrst problem', 'be in', 'network epidemiology known')],\n",
       "  [('relevant concepts', 'be in', 'computational complexity')],\n",
       "  [('connection', 'be with', 'percolation theory')],\n",
       "  [('their structural isomorphism', 'be with', 'certain problems concerning')],\n",
       "  [('Section 5',\n",
       "    'study',\n",
       "    'computational complexity of extended/generalized epidemiological problems')],\n",
       "  [('We', 'ﬁnish in', 'Section 6')],\n",
       "  [('Computational complexity', 'be in', 'section'),\n",
       "   ('we', 'give', 'brief account of class NP-complete')],\n",
       "  [('class', 'be', 'common')],\n",
       "  [('we', 'recommend', 'NumericCitation')],\n",
       "  [('it', 'describe', 'class P'), ('it', 'describe', 'class NP-complete')],\n",
       "  [('set', 'be in', 'example')],\n",
       "  [('number', 'be in', 'n'),\n",
       "   ('number', 'be in', 'example'),\n",
       "   ('instance n', 'of ℓ be', 'n')],\n",
       "  [('problem', 'be in', 'class P')],\n",
       "  [('total number', 'be in', 'b')],\n",
       "  [('class NP', 'consist of', 'non-deterministic polynomial time problems')],\n",
       "  [('c', 'be', 'composite'), ('c', 'be', 'correct')],\n",
       "  [('One',\n",
       "    'can deﬁne class in',\n",
       "    'terms of operation of non-deterministic Turing machines')],\n",
       "  [('many problems', 'be in', 'NP')],\n",
       "  [('hardest problems', 'be in', 'NP'),\n",
       "   ('class NP-complete', 'consist of', 'hardest problems in NP')],\n",
       "  [('problems', 'be in', 'NP-complete')],\n",
       "  [('P', '=', 'NP')],\n",
       "  [('Γ', 'can', 'Thus can used'), ('Γ', 'solve', 'Π')],\n",
       "  [('problems', 'be call', 'NP-hard')],\n",
       "  [('reduction', 'be in', 'SIR NETWORK')],\n",
       "  [('NP-hard problems', 'be', 'found')],\n",
       "  [('each', 'be possible successor of', 'previous state')],\n",
       "  [('epidemic', 'probability', 'it arrives at particular state'),\n",
       "   ('it', 'arrive at', 'particular state')],\n",
       "  [('us', 'formalize', 'this')],\n",
       "  [('graph', 'be with', 'vertex set V'), ('G edge', 'set', 'E')],\n",
       "  [('form', 'be with', 'u'), ('u', 'with form be', 'u')],\n",
       "  [('R. Infected individuals', 'recover in', 'one step')],\n",
       "  [('edge', 'be with', 'ϕ1 =')],\n",
       "  [('Infected individuals', 'be infect by', 'neighbour')],\n",
       "  [('individuals', 'recover in', 'in exactly time-step'),\n",
       "   ('requirement', 'be', 'drastic')],\n",
       "  [('formalism', 'extend period of', 'infectivity'),\n",
       "   ('formalism', 'accommodate', 'patterns of latency')],\n",
       "  [('state ϕ1', 'be follow by', 'particular successor state ϕ2')],\n",
       "  [('we', 'abuse', 'notation')],\n",
       "  [('these', 'be', 'Accordingly disjoint cases')],\n",
       "  [('we', 'can identify ϕ1 with', 'set of tibles')],\n",
       "  [('NP-HARD 7 path τ', 'be in', 'G'),\n",
       "   ('E → NumericCitation PROBABILITY', 'be in', 'SIR NETWORK')],\n",
       "  [('ﬁnite sum', 'be in', 'particular')],\n",
       "  [('integer coeﬃcients', 'with pr be', 'e'),\n",
       "   ('it', 'accordingly be polynomial in', 'e'),\n",
       "   ('values Pr', 'be with', 'integer coeﬃcients')],\n",
       "  [('each', 'have', 'known likelihood of failure')],\n",
       "  [('NumericCitation', 'introduction to', 'ﬁeld')],\n",
       "  [('k chosen terminals', 'be', 'mutually wise connected'),\n",
       "   ('k chosen terminals', 'be', 'mutually connected')],\n",
       "  [('other reasonable problems', 'arise from', 'practical applications'),\n",
       "   ('many other reasonable problems', 'arise from', 'practical applications')],\n",
       "  [('states', 'of op be', 'G ′ ='), ('states', 'of op be', 'G')],\n",
       "  [('whole stochastic binary system', 'specify', 'pathsets for G')],\n",
       "  [('edges', 'be in', 'pathset')],\n",
       "  [('links', 'be in', 'network')],\n",
       "  [('Theorem 1', 'be in', 'NumericCitation')],\n",
       "  [('problems', 'belong to', 'class # P-complete NumericCitation')],\n",
       "  [('# P',\n",
       "    'be set of',\n",
       "    'counting problems associated with decision problems in set NP'),\n",
       "   ('decision problems', 'be in', 'set NP')],\n",
       "  [('corresponding decision problem', 'solve', 'enumeration problem')],\n",
       "  [('Πepidemic', 'be', 'NP-hard')],\n",
       "  [('well-know procedures', 'be in', 'algorithmic graph theory')],\n",
       "  [('P', 'consist of', 'single vertex'),\n",
       "   ('PROBABILITY', 'be in', 'SIR NETWORK')],\n",
       "  [('extended problems',\n",
       "    'be in',\n",
       "    'epidemiology Epidemic on networks with time-varying transmission probabilities'),\n",
       "   ('networks', 'be with', 'time-varying transmission probabilities')],\n",
       "  [('individual', 'have', 'chances of infection')],\n",
       "  [('set', 'be', 'single'),\n",
       "   ('initial infectives', 'set of be', 'where single individual')],\n",
       "  [('transmission probabilities', 'vary over', 'time'),\n",
       "   ('We', 'can generalize', 'Πepidemic'),\n",
       "   ('We', 'allow', 'transmission probabilities')],\n",
       "  [('time-varying transmission probabilities', 'can', 'can encoded as Pr'),\n",
       "   ('longest self-avoiding path', 'be in', 'G.')],\n",
       "  [('problem', 'time-varying version of be', 'Thus NP-hard')],\n",
       "  [('networks', 'be with', 'disease latency')],\n",
       "  [('One', 'extend period of', 'infectivity4'),\n",
       "   ('One', 'period of', 'infectivity4')],\n",
       "  [('distinct states', 'of sequence be', 'IN'),\n",
       "   ('distinct states', 'of sequence be', 'I1')],\n",
       "  [('We', 'take', 'ρN')],\n",
       "  [('ϕ =', 'transition to', 'state R'),\n",
       "   ('transitions', 'be with', 'probability ρi'),\n",
       "   ('ϕ =', 'transition to', 'state Ii +1'),\n",
       "   ('ϕ =', 'transition with', 'probability ρi'),\n",
       "   ('probability ρi', 'be with', 'probability 1 − ρi')],\n",
       "  [('I', 'be', 'non-trivial')],\n",
       "  [('instance', 'be instance of', 'Π epidemic')],\n",
       "  [('We', 'take', 'µ')],\n",
       "  [('u', 'remain', 'infected for i steps')],\n",
       "  [('v', 'remain', 'uninfected')],\n",
       "  [('Πrel poly', 'be', 'polynomially reducible'),\n",
       "   ('Πrel poly', 'be polynomially reducible to', 'ΠI')],\n",
       "  [('I', 'be non-trivial pattern of', 'infectivity'),\n",
       "   ('I', 'be', 'non-trivial')],\n",
       "  [('We', 'choose', 'pN +1')],\n",
       "  [('unknowns', 'be coeﬃcients of', 'reliability polynomial')],\n",
       "  [('individual', 'have', 'probability of infection')],\n",
       "  [('us', 'formalize', 'this')],\n",
       "  [('E → NumericCitation PROBABILITY', 'be in', 'SIR NETWORK')],\n",
       "  [('Πrel poly', 'be', 'NP-hard')],\n",
       "  [('results', 'be in', 'communications network reliability')],\n",
       "  [('their solutions', 'work for', 'epidemiologists')],\n",
       "  [('We', 'give pointer into', 'their literature NumericCitation'),\n",
       "   ('We', 'give', 'pointers')],\n",
       "  [('estimates', 'converge to', 'actual value'),\n",
       "   ('estimates', 'increase', 'number of simulated samples')],\n",
       "  [('homogeneous models', 'be in', 'epidemiology')],\n",
       "  [('NumericCitation Eubank S. Network',\n",
       "    'base model of',\n",
       "    'infectious disease spread'),\n",
       "   ('NumericCitation Eubank S. Network',\n",
       "    'model of',\n",
       "    'infectious disease spread')],\n",
       "  [('Jpn J', 'infect', 'Dis')],\n",
       "  [('Pourbohloul B. Heterogeneous bond percolation',\n",
       "    'be with',\n",
       "    'application to epidemic dynamics')],\n",
       "  [('Percolation', 'be in', 'random clustered networks')],\n",
       "  [('Epidemic outbreaks', 'be in', 'complex heterogeneous networks')],\n",
       "  [('LA', 'epidemic on', 'directed contact networks')],\n",
       "  [('random networks', 'be with', 'heterogeneous connectivity'),\n",
       "   ('NumericCitation Volz E. SIR dynamics',\n",
       "    'be in',\n",
       "    'random networks with heterogeneous connectivity')],\n",
       "  [('Susceptibleˆae PROBABILITY', 'be in', 'SIR NETWORK')],\n",
       "  [('NumericCitation Laumanns M', 'complexity of', 'impact size estimation')],\n",
       "  [('you', 'get', 'vaccinated')],\n",
       "  [('LM', 'percolation on', 'heterogeneous networks'),\n",
       "   ('model', 'as lm be', 'Koopman J. Percolation on heterogeneous networks')],\n",
       "  [('Graph', 'be', 'Disconnected')],\n",
       "  [('Industrial Engineering', 'university of', 'Toronto')],\n",
       "  [('NumericCitation', 'harm', 'Devitt JS')],\n",
       "  [('Experiments', 'be with', 'Symbolic Algebra Environment')],\n",
       "  [('NumericCitation Barlow RE', 'theory of', 'reliability')],\n",
       "  [('Probability models', 'series in', 'Decision Processes'),\n",
       "   ('International Series', 'be in', 'Decision Processes'),\n",
       "   ('Decision Processes',\n",
       "    'series in',\n",
       "    'Quantitative Methods for Decision Making'),\n",
       "   ('Series', 'be in', 'Quantitative Methods for Decision Making')],\n",
       "  [('NumericCitation Gertsbakh IB', 'model of', 'Network Reliability')],\n",
       "  [('NumericCitation Lomonosov M',\n",
       "    'combinatorics of',\n",
       "    'reliability Monte Random Structures Algorithms')],\n",
       "  [('J', 'intervention during', 'emerging epidemic')],\n",
       "  [('Technology Zurich', 'of institute be', 'ETH Z ¨ urich'),\n",
       "   ('Swiss Federal Institute', 'institute of', 'Bioinformatics'),\n",
       "   ('Technology Zurich', 'of institute be', 'Basel Institute')]],\n",
       " 41: [[('Mathematical Sciences', 'of department be', 'University'),\n",
       "   ('Department', 'university of', 'Liverpool'),\n",
       "   ('Mathematical Sciences', 'of department be', 'Liverpool L69 7ZL')],\n",
       "  [('Mathematical Sciences', 'of school be', 'UK')],\n",
       "  [('Mathematical Sciences', 'of department be', 'University'),\n",
       "   ('Department', 'university of', 'Liverpool'),\n",
       "   ('Mathematical Sciences', 'of department be', 'Liverpool L69 7ZL')],\n",
       "  [('it', 'become', 'permanently recovered'),\n",
       "   ('it', 'become', 'recovered'),\n",
       "   ('it', 'make', 'contacts')],\n",
       "  [('its origin', 'be in', 'McKendrick')],\n",
       "  [('transition rates', 'with chain be', 'X t'),\n",
       "   ('transition rates', 'be in', 'as Table 1 with constraint'),\n",
       "   ('as Table 1', 'be with', 'constraint'),\n",
       "   ('dynamics', 'be in', 'general stochastic SIR epidemic x'),\n",
       "   ('continuous-time Markov chain',\n",
       "    'be with',\n",
       "    'transition rates as in Table 1'),\n",
       "   ('transition rates', 'with chain be', 'Y')],\n",
       "  [('number', 'recover at', 'time t')],\n",
       "  [('SIR', 'be in', 'case'), ('Poisson infection', 'be', 'where assumed')],\n",
       "  [('we', 'denote', 'time derivatives')],\n",
       "  [('its origin', 'be in', 'McKendrick')],\n",
       "  [('expected number', 'be in', 'stochastic model')],\n",
       "  [('comparison', 'hold also for', 'general stochastic SIR epidemic')],\n",
       "  [('number', 'be in', 'general stochastic epidemic')],\n",
       "  [('t', 'for e be', 'X Smes > S t')],\n",
       "  [],\n",
       "  [('its deterministic counterpart', 'of s be', '0')],\n",
       "  [('general stochastic SIR epidemic',\n",
       "    'be equivalent to',\n",
       "    'Markovian SIR epidemic on complete graph')],\n",
       "  [('S0', 'be with', 'i 6 = j')],\n",
       "  [('us', 'simplify', '5')],\n",
       "  [('Imes', 'with rme be', '10'),\n",
       "   ('Imes', 'with rme be', '˙Rmes ='),\n",
       "   ('Rmes', 'be with', 'Imes'),\n",
       "   ('Imes', 'with rme be', 't')],\n",
       "  [('al l t >', 'for sme be', 't')],\n",
       "  [('form', 'in 13 be', 'i'),\n",
       "   ('form', 'in 13 be', '14'),\n",
       "   ('initially-infected population', 'recover at', 'rate γ')],\n",
       "  [('al l t >', 'for sme be', 't')],\n",
       "  [('analogy', 'be with', '7')],\n",
       "  [('t >', 'for t be', 'then Lemma 2')],\n",
       "  [('x ∈', 'for x be', '0 1'),\n",
       "   ('t >', 'for ˙f2 be', 't'),\n",
       "   ('t >', 'for ˙f2 be', 'whence F2 > S2 t')],\n",
       "  [('all', 'for numericcitation be', '25'),\n",
       "   ('S0 +1 S2 I0 i. Substituting u = t − τ', 'be in', 'integral')],\n",
       "  [('t', 'for ˙f1 be', 'whence F1 > S1 t'),\n",
       "   ('t 0', 'for x be', '24'),\n",
       "   ('t', 'for ˙f1 be', 't'),\n",
       "   ('t 0', 'for x be', 'F2')],\n",
       "  [('size', 'so sme be', '∞')],\n",
       "  [('number', 'number of', 'carriers')],\n",
       "  [('We', 'state result in', 'following theorem')],\n",
       "  [('It', 'be', 'also straightforward')],\n",
       "  [('expected number susceptible', 'be underestimate', 't')],\n",
       "  [('KermackMcKendrick SIR model', 'be with', 'Poisson transmission')],\n",
       "  [('mean number', 'be in', 'general stochastic SIR epidemic'),\n",
       "   ('number', 'be in', 'corresponding deterministic epidemic')],\n",
       "  [('βN N', 'tend to', 'strictly positive ﬁnite limit as N → ∞'),\n",
       "   ('infection rate β', 'depend on', 'population size N')],\n",
       "  [('threshold', 'be in', 'limit as N → ∞')],\n",
       "  [('circumstances',\n",
       "    'in threshold be',\n",
       "    'deterministic epidemic overestimates'),\n",
       "   ('expected fraction', 'be in', 'even limit as N → ∞'),\n",
       "   ('threshold', 'be in', 'circumstances')],\n",
       "  [('we', 'set for', 'i ∈ V'), ('zi', '=', '0'), ('zi', '=', '1')],\n",
       "  [('they', 'overestimate', 'expected spread of infection')],\n",
       "  [('model', 'be', 'special')],\n",
       "  [('model', 'be with', 'nonMarkovian disease dynamics')],\n",
       "  [('Message passing', 'simplify', 'calculations')],\n",
       "  [('it', 'only aﬀects', 'fates of others')],\n",
       "  [('it', 'be in', 'when cavity state')],\n",
       "  [('time', 'become', 'infected')],\n",
       "  [('it', 'satisﬁes', 'equality')],\n",
       "  [('Acknowledgements R.R.W.', 'acknowledge', 'support')],\n",
       "  [('Lecture Notes', 'be in', 'Mathematics')],\n",
       "  [('Springer Lecture Notes', 'be in', 'Statistics')],\n",
       "  [('Downton', 'at', 'carrier-borne epidemic model')],\n",
       "  [('Lecture Notes', 'be in', 'Biomathematics')],\n",
       "  [('application', 'be in', 'compartmental models'),\n",
       "   ('death processes',\n",
       "    'be with',\n",
       "    'application to variability in compartmental models'),\n",
       "   ('Interparticle correlation',\n",
       "    'be in',\n",
       "    'death processes with application to variability')],\n",
       "  [('special reference',\n",
       "    'with mckendrick be',\n",
       "    'A.G. Studies on theory of continuous probabilities'),\n",
       "   ('McKendrick',\n",
       "    'be with',\n",
       "    'special reference to its bearing on natural phenomena of progressive nature'),\n",
       "   ('McKendrick', 'study on', 'theory of continuous probabilities')],\n",
       "  [('McKendrick', 'application to', 'medical problems'),\n",
       "   ('McKendrick', 'application of', 'mathematics')]],\n",
       " 42: [[('incubation', 'be', 'included'),\n",
       "   ('incubation', 'be', 'included separately')],\n",
       "  [('Phase transitions', 'be in', 'models')],\n",
       "  [('models', 'be', 'Then introduced'),\n",
       "   ('inhomogeneous models', 'be', 'Then introduced')],\n",
       "  [('They', 'have study phase transition in', 'models')],\n",
       "  [('SWN', 'model', 'several real systems'), ('SWN', 'model', '5-8')],\n",
       "  [('our aim', 'study', 'phase transitions'),\n",
       "   ('phase transitions',\n",
       "    'be in',\n",
       "    'modiﬁed versions of SIRS model deﬁned on SWN')],\n",
       "  [('Phase transitions', 'be in', 'SIRS versions'),\n",
       "   ('Phase transitions', 'be study in', 'section 3')],\n",
       "  [('conclusions', 'be summarize in', 'Section 5')],\n",
       "  [('its clustering coeﬃcient', 'be', 'high')],\n",
       "  [('N 1/d', 'be in', 'd dimensions')],\n",
       "  [('distance l', 'number of', 'vertices N'),\n",
       "   ('random graph NumericCitation', 'be with', 'coordination number'),\n",
       "   ('random graph NumericCitation', 'for l be', 'total number of vertices N')],\n",
       "  [('logarithmic increase', 'be with', 'N')],\n",
       "  [('quantity', 'zero for', 'large N')],\n",
       "  [('regular one-dimensional chain',\n",
       "    'be with',\n",
       "    'periodic boundary conditions')],\n",
       "  [('N', 'become enough', 'large'), ('it', 'ﬁnd', 'shortcut')],\n",
       "  [('transition', 'occur at', 'certain system size ζ')],\n",
       "  [('f', 'be', 'universal')],\n",
       "  [('1-d', 'for l be', '5')],\n",
       "  [('SWN', 'be', 'Then shown')],\n",
       "  [('population', 'be classiﬁed into', 'three classes'),\n",
       "   ('3 Phase transitions', 'be in', 'epidemic models')],\n",
       "  [('s =', '1 for', 'S − individuals'),\n",
       "   ('R − individuals', 'for s be', '0 1')],\n",
       "  [('− 1', 'be with', 'probability p1')],\n",
       "  [('9', 'be', 'mean'), ('9', 'be', 'ﬁeld')],\n",
       "  [('It', 'assume', 'global interacting system')],\n",
       "  [('phase transition', 'be in', 'model')],\n",
       "  [('it', 'be close to', 'percolation NumericCitation'),\n",
       "   ('it', 'be close to', 'ordinary percolation NumericCitation')],\n",
       "  [('case', 'be well describe by', 'directed percolation NumericCitation')],\n",
       "  [('we', 'here will generalize', 'their work')],\n",
       "  [('1-d chain', 'be with', 'periodic boundary conditions')],\n",
       "  [('Shortcuts', 'be', 'ﬁxed beforehand')],\n",
       "  [('models', 'evolve for', '10000 time steps')],\n",
       "  [('We', 'will study separately', 'four diﬀerent versions of SIRS model'),\n",
       "   ('We', 'will study separately', 'four versions of SIRS model')],\n",
       "  [('ﬁrst', 'be original sirs', 'itself')],\n",
       "  [('− 1', 'be with', 'probability p1')],\n",
       "  [('phase diagram', 'be show in', 'Fig. 1')],\n",
       "  [('value',\n",
       "    'be slightly less than',\n",
       "    'that observed from 1-d original model')],\n",
       "  [('s =', '0 for', 'S − individuals'), ('S − individuals', 'for 0 be', '1')],\n",
       "  [('model', 'be deﬁned on', 'SWN')],\n",
       "  [('lower susceptibility', 'consider', '30 % of population')],\n",
       "  [('probability 0.3', 'with i be', 't')],\n",
       "  [('phase diagram', 'be give in', 'Fig. 2')],\n",
       "  [('case', 'be call', 'incubation-latent model')],\n",
       "  [('0 1 2', 'represent', 'susceptibility'),\n",
       "   ('state function', 'represent', 'latency')],\n",
       "  [('case', 'ﬁrst', 'case')],\n",
       "  [('− 2', 'be with', 'probability p1')],\n",
       "  [('interaction', 'be with', 'distant neighbors')],\n",
       "  [('results', 'be summarize in', 'table 1')],\n",
       "  [('epidemic phase', 'increase signiﬁcantly in', 'four cases')],\n",
       "  [('susceptibility', 'thus also diﬀers from', 'one to another')],\n",
       "  [('unit', 'per m be', '15')],\n",
       "  [('main aspects', 'be in', 'reality')],\n",
       "  [('Inhomogeneous models',\n",
       "    'be construct for',\n",
       "    'four cases studied in previous section')],\n",
       "  [('same conditions', 'be', 'applied')],\n",
       "  [('models', 'be in', 'SWN')],\n",
       "  [('SWN', 'be with', 'k ='),\n",
       "   ('regular lattice', 'swn with', 'k ='),\n",
       "   ('regular lattice', 'swn with', 'k')],\n",
       "  [('5 Conclusions Phase transitions',\n",
       "    'be in',\n",
       "    'modiﬁed versions of SIRS model for epidemics'),\n",
       "   ('SWN', 'be with', 'k =')],\n",
       "  [('inhomogeneous models', 'be', 'Also introduced'),\n",
       "   ('models', 'be', 'Also introduced')],\n",
       "  [('We', 'thank e. ahmed for', 'helpful discussions'),\n",
       "   ('We', 'thank', 'E. Ahmed')],\n",
       "  [('Mathematical Models', 'be in', 'Biology')],\n",
       "  [('SWN', 'be with', 'N =')],\n",
       "  [('Tow phases', 'appear', 'I'), ('p1c', '=', '0.61')],\n",
       "  [('SWN', 'be with', 'N =')],\n",
       "  [('Lower susceptibility', 'be', 'introduced'),\n",
       "   ('susceptibility', 'be', 'introduced')],\n",
       "  [('SWN', 'be with', 'N =')],\n",
       "  [('SWN', 'be with', 'N =')]],\n",
       " 43: [[('distribution', 'be in', 'number of recovered individuals')],\n",
       "  [('SIR model', 'be in', 'also percolation class')],\n",
       "  [('models', 'of type be', 'deterministic NumericCitation')],\n",
       "  [('space structure', 'be explicitly take into', 'account NumericCitation')],\n",
       "  [('site', 'be', 'in which occupied by just one individual')],\n",
       "  [('I', '→', 'R')],\n",
       "  [('immunization', 'be', 'permanent')],\n",
       "  [('one', 'increase', 'infection rate'),\n",
       "   ('infection', 'spread over', 'whole lattice')],\n",
       "  [('cluster',\n",
       "    'eventually become',\n",
       "    'composed of recovered individuals only due to spontaneous immunization')],\n",
       "  [('< cc', 'cluster of', 'recovered individuals')],\n",
       "  [('cluster probability distribution',\n",
       "    'apply to',\n",
       "    'standard percolation models')],\n",
       "  [('parameter P', 'number of', 'recovered sites')],\n",
       "  [('section 2', 'be with', 'numerical simulations')],\n",
       "  [('model', 'be', 'how related to percolation around criticality')],\n",
       "  [('ﬁnite-size analysis', 'be introduce in', 'section 4')],\n",
       "  [('conclusion', 'be draw in', 'last section')],\n",
       "  [('nearest-neighbor sites', 'be in', 'state I')],\n",
       "  [('R', '=', '1 − b'), ('R', 'be with', 'probability c')],\n",
       "  [('it', 'be in', 'state R')],\n",
       "  [('we', 'have measure in', 'Monte Carlo simulation'),\n",
       "   ('mean number', 'value of', 'square'),\n",
       "   ('recovered individuals', 'of number be', 'mean value')],\n",
       "  [('second moment', 'between u be', 'that')],\n",
       "  [('We',\n",
       "    'begin with',\n",
       "    'infected individual placed at center of lattice full of susceptible individuals')],\n",
       "  [('we', 'keep', 'list of I sites')],\n",
       "  [('we', 'choose', 'one its four nearest-neighbor sites'),\n",
       "   ('it', 'otherwise remain', 'unchanged'),\n",
       "   ('we', 'choose', 'one of its four nearest-neighbor sites')],\n",
       "  [('rules',\n",
       "    'be equivalent to',\n",
       "    'deﬁnition of model given in beginning of section')],\n",
       "  [('state', 'be', 'absorbing')],\n",
       "  [('M/S 2', 'ﬁnd', 'order parameter P')],\n",
       "  [('linear system size L. parameter',\n",
       "    'be in',\n",
       "    'standard percolation theory')],\n",
       "  [('one cluster', 'be in', 'epidemic sample')],\n",
       "  [('Ps = sns', 'be in', 'standard percolation theory'),\n",
       "   ('site', 'belong to', 'cluster of size s')],\n",
       "  [('site', 'be', 'occupied')],\n",
       "  [('site', 'belong to', 'inﬁnite percolating cluster')],\n",
       "  [('probability', 'be in', 'site')],\n",
       "  [('mean cluster size', 'be in', 'percolation theory')],\n",
       "  [('SIR model', 'be with', 'quantities deﬁned in section')],\n",
       "  [('standard ﬁnite-size scaling analysis',\n",
       "    'obtain',\n",
       "    'critical behavior of model')],\n",
       "  [('phase transition', 'be in', 'SIR model')],\n",
       "  [('ˆA', 'be', 'universal')],\n",
       "  [('behavior', 'be in', 'limit of inﬁnite system')],\n",
       "  [('we', 'use', 'ﬁnite-size scaling')],\n",
       "  [('function', 'be in', 'ﬁgures 4')],\n",
       "  [('22', 'be', 'independent')],\n",
       "  [('critical point', 'at value be', '1'),\n",
       "   ('U P', 'for value be', '1'),\n",
       "   ('agreement', 'be with', 'result')],\n",
       "  [('c', '=', '0.1765')],\n",
       "  [('we', 'here do', 'at least 108 samples for size')],\n",
       "  [('we', 'see in', 'ﬁgure')],\n",
       "  [('results', 'be in', 'ﬁgure 10'),\n",
       "   ('We', 'also show', 'results in ﬁgure 10')],\n",
       "  [('error bars', 'be smaller than', 'size of symbols')],\n",
       "  [('critical point', 'order of', 'size')],\n",
       "  [('U P', 'as be expect by', 'arguments'),\n",
       "   ('U P', 'be', 'seen'),\n",
       "   ('U P', 'be see in', 'ﬁgure 9')],\n",
       "  [('We',\n",
       "    'investigate',\n",
       "    'critical behavior of stochastic SIR model on ﬁnite square lattices')],\n",
       "  [('order parameter P', 'number of', 'recovered individuals')],\n",
       "  [('quantities', 'obey', 'same scaling laws used in percolation theory')],\n",
       "  [('we', 'study', 'lattices of diﬀerent sizes'), ('we', 'study', 'lattices')],\n",
       "  [('agreement', 'be with', 'those of isotropic percolation')],\n",
       "  [('excellent agreement', 'be with', 'previous calculations'),\n",
       "   ('U P', 'be independent of', 'system size')],\n",
       "  [('We', 'acknowledge', 'ﬁnancial support of Brazilian agency CNPq')],\n",
       "  [('Epidemic', 'of theory be', 'Citation'),\n",
       "   ('NumericCitation N. T. J. Bailey', 'theory of', 'Epidemic')],\n",
       "  [('Biological Populations', 'be in', 'Space')],\n",
       "  [('T. Tom', None, 'Phys')],\n",
       "  [('T. Tom', 'e be', 'J. Math')],\n",
       "  [('G.', 'odor be', 'Phys')],\n",
       "  [('T. Tom', 'e be', 'J. Phys')],\n",
       "  [('T. Tom', 'e be', 'Physica A 389 1142')],\n",
       "  [('NumericCitation V. Privman', 'be in', 'Phase Transitions')]],\n",
       " 44: [[('total size', 'be', 'allowed')],\n",
       "  [('single equations', 'be with', 'same content')],\n",
       "  [('multivariate process', 'be with', 'joint probability function given'),\n",
       "   ('representation', 'be in', 'terms of second-quantized bosonic operators'),\n",
       "   ('configurations', 'be give entirely in', 'terms of occupation numbers')],\n",
       "  [('recovered state R. infected state C', 'be in', 't t t t0')],\n",
       "  [('our model',\n",
       "    'be in',\n",
       "    'hamiltonian Transition Description Contribution to birth of susceptible individual'),\n",
       "   ('infected individual', 'of individual be', 'rate'),\n",
       "   ('recovered individual birth', 'of individual be', 'rate')],\n",
       "  [('master equation', 'into state be', '2')],\n",
       "  [('kind o f equation',\n",
       "    'be also call',\n",
       "    'forward Kolmogorov differential equation')],\n",
       "  [('algebra', 'after 4 be', 'following differential equations for')]],\n",
       " 45: [[('School', 'university of', 'London'),\n",
       "   ('Epidemic spreading', 'be in', 'School')],\n",
       "  [('multilayer networks',\n",
       "    'be with',\n",
       "    'arbitrary number of links across layers')],\n",
       "  [('analytical expression', 'be in', 'arbitrary number of layers')],\n",
       "  [('epidemic threshold',\n",
       "    'strongly depend on',\n",
       "    'degree correlations of multilayer structure')],\n",
       "  [('studied processes', 'be in', 'networks having')],\n",
       "  [('NumericCitation', 'be attract', 'interest'),\n",
       "   ('NumericCitation', 'be attract', 'increasing interest')],\n",
       "  [('cascading failures', 'can occur', 'NumericCitation')],\n",
       "  [('They', 'can', 'can classiﬁed')],\n",
       "  [('spreading', 'be in', 'multilayer networks 2')],\n",
       "  [('social online networks',\n",
       "    'can have in',\n",
       "    'actual spreading of viral epidemic')],\n",
       "  [('awarenessunawareness-awareness spreading',\n",
       "    'be in',\n",
       "    'social online network'),\n",
       "   ('susceptible-infected-susceptible spreading',\n",
       "    'be in',\n",
       "    'social contact network')],\n",
       "  [('multilayer network framework', 'characterize', 'SIS dynamics'),\n",
       "   ('It',\n",
       "    'predict',\n",
       "    'epidemic threshold on temporal networks NumericCitation')],\n",
       "  [('epidemic threshold',\n",
       "    'eﬀect of',\n",
       "    'diﬀerent immunization strategies investigated')],\n",
       "  [('We',\n",
       "    'provide',\n",
       "    'solution of bond percolation problem valid for locally tree-like multilayer networks')],\n",
       "  [('dynamical properties',\n",
       "    'be in',\n",
       "    'context of single networks NumericCitation')],\n",
       "  [('epidemics', 'be achieve with', 'message passing techniques')],\n",
       "  [('direction', 'be in', 'future publications')],\n",
       "  [('Epidemic spreading', 'be in', 'multilayer networks 3 2')],\n",
       "  [('nodes', 'can in', 'can three possible states')],\n",
       "  [('SIR model',\n",
       "    'be characterize by',\n",
       "    'non-equilibrium phase transition occurring as function of infection rate')],\n",
       "  [('theoretical solution',\n",
       "    'thanks to',\n",
       "    'mapping of model to bond percolation NumericCitation')],\n",
       "  [('multilayer network M', 'be with', 'M layers α =')],\n",
       "  [('number', 'be in', 'layer β'),\n",
       "   ('N M × N M supra-adjacency matrix Aiα', 'β of', 'elements')],\n",
       "  [('node', 'be in', 'layer α'), ('node', 'be in', 'layer β')],\n",
       "  [('node', 'be in', 'layer α'), ('node', 'be in', 'layer β')],\n",
       "  [('layer α', 'neighbour of', 'susceptible node in layer β'),\n",
       "   ('infected node', 'be in', 'layer α'),\n",
       "   ('susceptible node', 'be in', 'layer β')],\n",
       "  [('infected node', 'be with', 'lifetime τ')],\n",
       "  [('infected individual', 'be in', 'layer α')],\n",
       "  [('node', 'be in', 'layer α'), ('node', 'be in', 'layer β')],\n",
       "  [('epidemic threshold',\n",
       "    'consequently correspond to',\n",
       "    'epidemic threshold of bond percolation problem')],\n",
       "  [('percolation', 'be', 'interesting'), ('percolation', 'be', 'critical')],\n",
       "  [('bond percolation', 'be in', 'multilayer networks'),\n",
       "   ('bond percolation', 'be with', 'arbitrary number of links across layers')],\n",
       "  [('solution',\n",
       "    'will reveal',\n",
       "    'important eﬀects of correlations in properties of epidemic spreading in multilayer networks'),\n",
       "   ('epidemic spreading', 'be in', 'multilayer networks'),\n",
       "   ('important eﬀects', 'be in', 'properties of epidemic spreading')],\n",
       "  [('SIR model', 'be in', 'single multilayer network 3.1')],\n",
       "  [('spreading', 'be in', 'multilayer networks'),\n",
       "   ('Bond percolation',\n",
       "    'be in',\n",
       "    'generic multilayer network Given locally tree-like multilayer network of M layers')],\n",
       "  [('approach', 'be', 'straightforward')],\n",
       "  [('nodes', 'be in', 'diﬀerent layers')],\n",
       "  [('values', 'be determine by', 'self-consistent set of equations')],\n",
       "  [],\n",
       "  [('σiα → j β', '=', '1 − pγασℓγ → iα')],\n",
       "  [('nodes', 'be in', 'giant component')],\n",
       "  [('message', 'pass', 'Eqs')],\n",
       "  [('nodes', 'be in', 'giant component S'),\n",
       "   ('they',\n",
       "    'develop',\n",
       "    'non-trivial solution consistent with non-vanishing fraction of nodes in giant component S')],\n",
       "  [('σiα j β', '=', '0'), ('σiα → j β', '=', '0')],\n",
       "  [('12 Epidemic spreading', 'be in', 'multilayer networks 6')],\n",
       "  [('L × L matrix links', 'be in', 'multilayer network')],\n",
       "  [('σiα j β', '=', '0'), ('σiα → j β', '=', '0')],\n",
       "  [('it', 'use', 'Eq')],\n",
       "  [('ﬁnite networks', 'be with', 'loops')],\n",
       "  [('SIR model', 'be in', 'esemble of multilayer networks 4.1')],\n",
       "  [('Multilayer networks', 'be with', 'number of layers M')],\n",
       "  [('epidemic spreading', 'be in', 'multilayer network ensembles')],\n",
       "  [('multilayer ensemble', 'be with', 'given multilayer degree sequence')],\n",
       "  [('Z', 'be', 'normalization factor')],\n",
       "  [('15 Epidemic spreading', 'be in', 'multilayer networks node'),\n",
       "   ('iα k', 'spreading in', 'multilayer networks node'),\n",
       "   ('probability πiα', 'β of', 'link between P')],\n",
       "  [('expression', 'be in', 'ensured by degree cutoﬀs K')],\n",
       "  [('multilayer degrees', 'necessarily satisfy', '16')],\n",
       "  [('fact', 'in pα be', 'k'), ('Pα =', 'be in', 'fact')],\n",
       "  [('xαβ', '=', '0')],\n",
       "  [('Eq', 'in probability be', '21'), ('probability', 'be in', 'Eq')],\n",
       "  [('generic node', 'be in', 'of terms of average messages'),\n",
       "   ('Sα', '=', 'k'),\n",
       "   ('nodes', 'be in', 'giant component')],\n",
       "  [('1 M M Xα', '=', '1 Sα'), ('M M Xα', '=', '1 Sα')],\n",
       "  [('single networks', 'be in', 'locally tree-like approximation')],\n",
       "  [('us', 'consider', 'case of duplex network')],\n",
       "  [('E hk', 'be in', 'multilayer networks')],\n",
       "  [('Poisson degree distribution k NumericCitation',\n",
       "    'with layer be',\n",
       "    'layer 2'),\n",
       "   ('one', 'still have', 'giant component'),\n",
       "   ('one layer',\n",
       "    'be with',\n",
       "    'scale-free distribution of intralayer degrees k NumericCitation'),\n",
       "   ('one layer', 'be with', 'Poisson degree distribution k NumericCitation'),\n",
       "   ('scale-free distribution', 'with layer be', 'layer 1'),\n",
       "   ('Poisson degree distribution k NumericCitation',\n",
       "    'be with',\n",
       "    'average degree smaller than 1')],\n",
       "  [('it', 'use', 'Eq')],\n",
       "  [('38 Epidemic spreading', 'be in', 'multilayer networks 10')],\n",
       "  [('epidemic threshold', 'give by', 'Eq')],\n",
       "  [('multilayer network', 'be with', 'M =')],\n",
       "  [('percolation threshold', 'be', 'smaller'),\n",
       "   ('network', 'be', 'more robust')],\n",
       "  [('multilayer network', 'be', 'more fragile'),\n",
       "   ('percolation threshold', 'be', 'larger')],\n",
       "  [('network',\n",
       "    'be with',\n",
       "    'identical interlayer degree sequences maximally anti-correlated')],\n",
       "  [('multilayer network', 'be with', 'same interlayer network structure'),\n",
       "   ('intralinks', 'be', 'placed')],\n",
       "  [('other sequence', 'be in', 'decresing order'),\n",
       "   ('one sequence', 'be in', 'increasing order'),\n",
       "   ('previous case', 'be with', 'exception')],\n",
       "  [('interdegree',\n",
       "    'perform',\n",
       "    'random permutation of corresponding interdegree sequence')],\n",
       "  [('ﬁgure 1', 'of panel be', 'a')],\n",
       "  [('ﬁgure 1', 'of panel be', 'b')],\n",
       "  [('correlated interdegree', 'for threshold be', 'panel')],\n",
       "  [('Poisson distribution', 'be with', 'average degree c =')],\n",
       "  [('annealed network Epidemic spreading', 'be in', 'multilayer networks')],\n",
       "  [('annealed approximation', 'be in', 'Ref'),\n",
       "   ('results exact', 'be with', 'results obtained for SIS dynamics using')],\n",
       "  [('multilayer network', 'be with', 'two layers'),\n",
       "   ('we', 'sum up', 'results of previous section')],\n",
       "  [('equations', 'can', 'In fact can directly obtained by Eqs')],\n",
       "  [('expression', 'be in', 'terms of infectivities λ given by Eq')],\n",
       "  [('0 = h1 − λ NumericCitation', 'for transition be', '46')],\n",
       "  [('multilayer networks', 'be with', 'two layers'),\n",
       "   ('Numerical results', 'be in', 'single multilayer networks')],\n",
       "  [('evidence', 'be in', 'than general multilayer networks'),\n",
       "   ('we', 'observe', 'same qualitative phenomenon observed for SIS dynamics')],\n",
       "  [('Poisson', 'be with', 'average degree one'),\n",
       "   ('Poisson', 'be with', 'average degree two')],\n",
       "  [('we', 'observe', 'outbreaks'), ('we', 'observe', 'epidemic outbreaks')],\n",
       "  [('power-law distribution', 'be with', 'exponent γ ='),\n",
       "   ('Poisson interdegree distribution', 'be with', 'average smaller than one'),\n",
       "   ('other layer',\n",
       "    'have',\n",
       "    'Poisson interdegree distribution with average smaller than one')],\n",
       "  [('it', 'observe', 'epidemic spreading')],\n",
       "  [('Poisson', 'be with', 'average degree smaller than one')],\n",
       "  [('corresponding bond percolation problem',\n",
       "    'use',\n",
       "    'message passing technique')],\n",
       "  [('Conclusions', 'be in', 'conclusion'),\n",
       "   ('general multilayer network',\n",
       "    'be with',\n",
       "    'arbitrary number of links across layers'),\n",
       "   ('conclusion', 'be in', 'paper')],\n",
       "  [('bond percolation', 'be in', 'multilayer networks')],\n",
       "  [('characterization', 'be in', 'ensemble of multilayer networks')],\n",
       "  [('mapping',\n",
       "    'be with',\n",
       "    'annealed approximation predictions of epidemic spreading of SIS model on multilayer networks')],\n",
       "  [('SIR dynamics', 'be in', 'multilayer networks')]],\n",
       " 46: [[('Complex Systems', 'of institute be', 'UBICS'),\n",
       "   ('University', 'franqu`es', 'Mart'),\n",
       "   ('Mechanical', 'of department be', 'USA Dipartimento di Elettronica'),\n",
       "   ('Epidemic spreading', 'be in', 'Department'),\n",
       "   ('Telecomunicazioni', 'laboratory for', 'Modeling of Biological Systems'),\n",
       "   ('University', 'institute for', 'Scientiﬁc Interchange')],\n",
       "  [('model', 'be with', 'tunable modularity')],\n",
       "  [('such attributes', 'have', 'critical eﬀects')],\n",
       "  [('heterogeneity', 'be in', 'connectivity patterns')],\n",
       "  [('temporal changes', 'be in', \"networks ' structures\")],\n",
       "  [('their interplay', 'introduce', 'non-trivial effects')],\n",
       "  [('interactions network', 'be in', 'school')],\n",
       "  [('presence', 'be in', 'network')],\n",
       "  [('lower epidemic threshold', 'be in', 'latter case')],\n",
       "  [('similar results', 'have', 'Interestingly have recently obtained'),\n",
       "   ('similar results',\n",
       "    'have',\n",
       "    'Interestingly have recently obtained in models of time-varying networks characterised by correlated topological features induced by reinforcement of speciﬁc ties NumericCitation')],\n",
       "  [('Our results', 'characterize', 'mechanisms')],\n",
       "  [('a. Modular activity', 'drive', 'networks')],\n",
       "  [('social interaction', 'be with', 'others')],\n",
       "  [('∆ t', 'be with', 'N')],\n",
       "  [('probability 1 − µ', 'be with', 'nodes in other groups'),\n",
       "   ('nodes', 'be in', 'other groups'),\n",
       "   ('node', 'within', 'community')],\n",
       "  [('few nodes', 'be', 'active'), ('nodes', 'be', 'active')],\n",
       "  [('complete set', 'for ρ be', 'k')],\n",
       "  [('we', 'show', 'active nodes')],\n",
       "  [('average degree', 'have', 'dynamics')],\n",
       "  [('s', 'be', \"where node 's community size\")],\n",
       "  [('activity class', 'be in', 'communities of size s')],\n",
       "  [('nodes', 'of kc be', 't'), ('s t ∼', 'value of', 'in-community degree')],\n",
       "  [('con-The time', 'be t', 'plotted'),\n",
       "   ('con-The rescaled time', 'be t', 'plotted')],\n",
       "  [('ν', '=', '2.1'), ('m', '=', '1'), ('ω', '=', '2.1'), ('smin', '=', '10')],\n",
       "  [],\n",
       "  [('we', 'plot', 'three degree distributions')],\n",
       "  [('exponent − ν', 'with ρ be', 'ko'),\n",
       "   ('out-community ρ', 'be with', 'exponent − ν')],\n",
       "  [(\"modules ' size\", 'be far smaller than', 'net-N N')],\n",
       "  [('contrary', 'on hand be', 'results presented in FIG. 3'),\n",
       "   ('contrary', 'on hand be', 'work size √')],\n",
       "  [('m', '=', '1'), ('ν', '=', '2.1'), ('ω', '=', '2.1'), ('smin', '=', '10')],\n",
       "  [('this', 'be', 'true')],\n",
       "  [('we', 'will consider in', 'following')],\n",
       "  [('SIR processes', 'be', 'Indeed always characterized')],\n",
       "  [('pear', 'existence of', 'endemic state')],\n",
       "  [('quantity', 'deﬁnes', 'conditions necessary for spreading of illness')],\n",
       "  [('contagion process', 'be in', 'modular activity')],\n",
       "  [('block', 'be in', 'class of activity')],\n",
       "  [('ﬁrst term',\n",
       "    'be in',\n",
       "    'r.h.s accounts for recovery of infected individuals')],\n",
       "  [('Susceptible node', 'be in', 'community of size s')],\n",
       "  [('we', 'consider least initially at', 'ity')],\n",
       "  [('ﬁrst order terms', 'be in', 'Ia s Ra we obtain'),\n",
       "   ('activity distribution', 'be in', 'community of s')],\n",
       "  [('all', 'of first be', 'terms associated to Ra')],\n",
       "  [('activity distribution', 'be with', 'corresponding quantities'),\n",
       "   ('ﬂuctuations', 'diﬀerentiating between', 'values'),\n",
       "   ('terms', 'be in', 'µ'),\n",
       "   ('activity distribution', 'be in', 'network')],\n",
       "  [('diﬀerent approaches', 'be in', 'Refs'),\n",
       "   ('threshold', 'rive with', 'diﬀerent approaches Refs')],\n",
       "  [('time-scales', 'be in', 'opposite limit µ → 1 networks')],\n",
       "  [('Fluctuations', 'become', 'important of network')],\n",
       "  [('single infected node', 'be in', 'community of size s')],\n",
       "  [('connections', 'large jority of be', 'vertices in same group'),\n",
       "   ('vertices', 'be in', 'same group')],\n",
       "  [('Such connections', 'hamper', 'spreading of disease')],\n",
       "  [('SIR processes', 'be inhibit by', 'highly modular connectivity patterns'),\n",
       "   ('SIR processes', 'be', 'inhibited')],\n",
       "  [('I', '∞', 'NumericCitation')],\n",
       "  [('we', 'follow ref for', 'reasons')],\n",
       "  [('susceptibility', 'be in', 'phase transitions NumericCitation')],\n",
       "  [('populations', 'of fraction be', 'Y')],\n",
       "  [('we', 'set', '1 % of randomly selected nodes')],\n",
       "  [('N', '=', '105'), ('Other parameters', 'be', 'set')],\n",
       "  [('Vertical black line', 'derive in', 'max value of R ∞')],\n",
       "  [('Panel B', 'function of', 'µ')],\n",
       "  [('we', 'set in', 'red curves'), ('smin', '=', '100')],\n",
       "  [('95 % conﬁdence interval', 'be in', 'gray')],\n",
       "  [('network', 'be with', 'bigger communities')],\n",
       "  [('we', 'interestingly observe', 'weak dependence of threshold on µ')],\n",
       "  [('second observation', 'study', 'diﬀerent community structures'),\n",
       "   ('we', 'be', 'same'),\n",
       "   ('µ', 'be in', 'our set-In')],\n",
       "  [('µ → 1', 'spread of', 'disease')],\n",
       "  [('contacts', 'be with', 'nodes')],\n",
       "  [('main observations', 'be', 'Overall four')],\n",
       "  [('epidemic size', 'increase', 'modularity')],\n",
       "  [('larger modules', 'yield', 'higher epidemic size')],\n",
       "  [('smin', '=', '10')],\n",
       "  [('Vertical lines', 'be', 'epidemic threshold')],\n",
       "  [('β γ', 'be in', 'correspondence of Lmax')],\n",
       "  [('blue curves smin', '=', '10'),\n",
       "   ('we', 'set in', 'red curves'),\n",
       "   ('smin', '=', '100'),\n",
       "   ('curves smin', '=', '10')],\n",
       "  [('average', 'be', 'independent')],\n",
       "  [('we', 'simulate', 'µ'),\n",
       "   ('network', 'be with', 'many connected components')],\n",
       "  [('average community size', 'be', 'same')],\n",
       "  [],\n",
       "  [('curves', 'converge to', 'same epidemic threshold')],\n",
       "  [('network', 'be with', 'larger modules')],\n",
       "  [('communities', 'be with', 'smaller average sizes')],\n",
       "  [('dashed blue line', 'be in', 'Fig. 5-B')],\n",
       "  [('contagion process', 'be in', 'SIS models'),\n",
       "   ('µ → 1', 'repetition of', 'same contacts within communities')],\n",
       "  [('synthetic models', 'be with', 'real world system'),\n",
       "   ('It',\n",
       "    'ground',\n",
       "    'picture emerging from synthetic models with real world system')],\n",
       "  [('temporal network', 'be in', 'American Physical Society')],\n",
       "  [('data', 'of year be', 'Citation')],\n",
       "  [('we',\n",
       "    'single out',\n",
       "    'eﬀects introduced by communities on contagion processes')],\n",
       "  [('we', 'ﬁnd', 'communities'),\n",
       "   ('randomization process', 'remove', 'topological structures')],\n",
       "  [('we', 'use', 'two networks')],\n",
       "  [('we', 'present result in', 'Fig. 6A-B')],\n",
       "  [('model', 'be with', 'tunable modularity')],\n",
       "  [('We',\n",
       "    'have provide',\n",
       "    'analytical characterization of time-aggregated properties of such networks')],\n",
       "  [('modularity', 'slow down', 'spreading process')],\n",
       "  [('We', 'ﬁx', 'γ')],\n",
       "  [('system', 'more prone to', 'disease spreading')],\n",
       "  [('pathogen', 'reach more easily', 'endemic state')],\n",
       "  [('spreading processes', 'be', 'how aﬀected by features of real networks')],\n",
       "  [('SIR models nodes', 'in susceptible be', 'S')],\n",
       "  [('Susceptible nodes', 'describe', 'healthy individuals'),\n",
       "   ('Susceptible nodes', 'describe', 'individuals')],\n",
       "  [('contact', 'be with', 'infected individuals'),\n",
       "   ('R. Susceptible nodes', 'be in', 'contact with infected individuals')],\n",
       "  [('transition', 'describe', 'recovery process')],\n",
       "  [('recovery', 'describe by', 'γ − → S')],\n",
       "  [('Barab', 'asi be', 'Nature Physics 8')],\n",
       "  [('A. Vespignani', 'process on', 'complex networks')],\n",
       "  [(\"Barab ' asi\", 'proceedings of', 'National Academy'),\n",
       "   ('Barab',\n",
       "    'asi be',\n",
       "    'Proceedings of National Academy of Sciences 104 7332 2007')],\n",
       "  [('Pan', 'esz', 'J. Kert')],\n",
       "  [('A. Baronchelli', 'proceedings of', 'National Academy')],\n",
       "  [('M. Rosvall', 'journal of', 'Complex Networks 3 177')],\n",
       "  [('NumericCitation M. Keeling', 'disease in', 'Humans'),\n",
       "   ('Modeling Infectious Disease', 'be in', 'Humans')],\n",
       "  [('Scientiﬁc', 'report', '2014')],\n",
       "  [('C. Faloutsos', 'be in', 'Databases Lecture Notes'),\n",
       "   ('Databases Lecture Notes', 'be in', 'Computer Science 6323 99'),\n",
       "   ('Databases Lecture Notes', 'in faloutsos be', '2010')],\n",
       "  [('NumericCitation J. Tang',\n",
       "    'be in',\n",
       "    'Proceedings of IEEE 12th International Symposium on World of Wireless')],\n",
       "  [('NumericCitation A. Aharony', 'introduction to', 'percolation theory')],\n",
       "  [('Barab', 'asi be', 'EPL')],\n",
       "  [('Scientiﬁc', 'report', '2016')],\n",
       "  [('ACKNOWLEDGEMENTS M.S.',\n",
       "    'acknowledge',\n",
       "    'ﬁnancial support from James S. McDonnell Foundation')],\n",
       "  [('W911NF-15-1-0267', 'be with', 'Drs. A. Garcia')],\n",
       "  [('A.R.', 'acknowledge', 'ﬁnancial support')],\n",
       "  [('E.U.', 'review', 'manuscript')],\n",
       "  [('total number N', 'be in', 'network'),\n",
       "   ('network', 'be deﬁned by means of', 'following parameters')],\n",
       "  [('nodes',\n",
       "    'be in',\n",
       "    'communities of size s drawn from size-distribution P s')],\n",
       "  [('its own community', 'be with', 'probability µ')],\n",
       "  [('i', \"'s community be\", 'in-community degree')],\n",
       "  [('integrated network', 'in growth be', 'node'),\n",
       "   ('NETWORK GROWTH', 'be in', 'integrated network')],\n",
       "  [('out-community degree ko', 'be treat separately in', 'fact')],\n",
       "  [('community', 'be large ing from', 'rest of network')],\n",
       "  [('in-degree time dependence',\n",
       "    'be easily approximate with',\n",
       "    'probabilistic consideration')],\n",
       "  [('step', 'be with', 'of probability µ')],\n",
       "  [('confusions', 'be with', 'community size distribution P s')],\n",
       "  [('We', 'note', 'Eq')],\n",
       "  [('we', 'indeed can rewrite', 'Eq'), ('we', 'can rewrite', 'Eq')],\n",
       "  [('1 c s', '−', '1 − 1'), ('1 − c s', '−', '1 − 1')],\n",
       "  [('typical time', 'be in', 'single evolution step')],\n",
       "  [('numto', 'be in', 'other words')],\n",
       "  [('possible edges', 'be in', 'network'),\n",
       "   ('edge eij', 'be in', 'community of size s ber of edges activated')],\n",
       "  [('degree', 'be in', 'degree'),\n",
       "   ('time', 'be in', 'general'),\n",
       "   ('we', 'assume', '∆ t'),\n",
       "   ('this', 'be', 'simple')],\n",
       "  [('we', 'get t. me for', 'in-degree kc distribution')],\n",
       "  [('nodes', 'be in', 'network ﬁre')],\n",
       "  [('node', 'be in', 'instantaneous network')],\n",
       "  [('we can neglect', 'be in', 'regime t τ s')],\n",
       "  [('two nodes', 'become', 'active'),\n",
       "   ('two nodes', 'become together', 'active')],\n",
       "  [('we', 'follow', 'same procedure')],\n",
       "  [('t', 'n for', 'time'),\n",
       "   ('N smax', 'since t be', 'out-degree ko N for time'),\n",
       "   ('N smax', 'since t be', '17')],\n",
       "  [('us', 'now resume', 'results found in section'),\n",
       "   ('t τc s', '− for', 's t τc s'),\n",
       "   ('t τc s', 'for − be', '18a')],\n",
       "  [('18a', 'be in', 'Eq')],\n",
       "  [('terms', 'give', 'k')],\n",
       "  [('computation', 'be in', 'large time limit')],\n",
       "  [('18b', 'put', 'everything'), ('18b', 'put', 'everything equal to zero')],\n",
       "  [('k < s', 'for 1 be', '26a')],\n",
       "  [('s', '−', '1')],\n",
       "  [('equation 24', 'can', 'Finally can written')],\n",
       "  [('nodes', 'for k be', 's t')],\n",
       "  [('regimes', 'follow by', 'initial growth')],\n",
       "  [('equal increment', 'in ρ be', 'kth'),\n",
       "   ('equal increment', 'be in', 'activity'),\n",
       "   ('degree values', 'in ρ be', 'kth'),\n",
       "   ('ρ', 'be in', 'equal increment'),\n",
       "   ('ρ', 'be in', 'degree values')],\n",
       "  [('COMPARISON', 'be with', 'NUMERICAL SIMULATIONS'),\n",
       "   ('we', 'perform', 'numerical simulation'),\n",
       "   ('we', 'perform', 'simulation')],\n",
       "  [('− ν', 'be with', 'ν ='), ('ω', 'be with', 'ω =')],\n",
       "  [('it', 'write', 'degree distribution'),\n",
       "   ('we', 'now have', 'expression of average degree')],\n",
       "  [('We', 'start', 'our simulations'),\n",
       "   ('We', 'set', '1 % of randomly selected nodes')],\n",
       "  [('disease outbreak', 'be in', 'SIS models'),\n",
       "   ('it', 'favor', 'disease outbreak in SIS models')],\n",
       "  [('higher epidemic size', 'be in', 'SIS models')],\n",
       "  [('phenomenon', 'be particularly visible in', 'solid blue curve')],\n",
       "  [('γ', 'be in', 'correspondence of Lmax')],\n",
       "  [('we', 'set in', 'red curves'), ('smin', '=', '100')],\n",
       "  [('95 % conﬁdence interval', 'be in', 'gray'),\n",
       "   ('% conﬁdence interval', 'be in', 'gray')],\n",
       "  [('Dashed curves',\n",
       "    'have',\n",
       "    'constant community size equal to average value of power law')],\n",
       "  [('Functions', 'be in', 'panels A-B')],\n",
       "  [('we', 'plot for', 'nodes tion given in Eq'),\n",
       "   ('kc → k c', '= kc kc in', 'panel D'),\n",
       "   ('kc → k c', '=', 'kc kc')],\n",
       "  [('diﬀerent activity potential', 'belong to', 'com'),\n",
       "   ('activity potential', 'belong to', 'com')],\n",
       "  [('time t →', '=', 't/τ')],\n",
       "  [('size s', '=', 's')],\n",
       "  [('m', '=', '1'),\n",
       "   ('ν', '=', '2.1'),\n",
       "   ('ω', '=', '2.1'),\n",
       "   ('t τc s', 'for po be', 'k'),\n",
       "   ('smin', '=', '10'),\n",
       "   ('µ', '=', '0.9')],\n",
       "  [('great agreement', 'be with', 'analytical predictions')],\n",
       "  [('we', 'complete', 'discussion'),\n",
       "   ('it', 'obtain', 'exact for expectation')],\n",
       "  [('in-community probability', 'deviate from', 'Gaussian distribution')],\n",
       "  [('disease dynamics', 'evolve without', 'late-time constrains')],\n",
       "  [('multi-edge', 'will', 'will created'), ('we', 'select', 'two pairs')],\n",
       "  [('number', 'be in', 'instantaneous network')],\n",
       "  [('we', 'detect', 'community structure')],\n",
       "  [('network', 'have', 'community structure')],\n",
       "  [('OSLOM ﬁnds communities', 'be in', 'non-deterministic way'),\n",
       "   ('last option', 'get', 'rid of stochastic ﬂuctuations'),\n",
       "   ('last option', 'have', 'more reliable community structure')],\n",
       "  [('we', 'finally evaluate quantitatively', 'modularity of two networks')],\n",
       "  [('Average degree', 'be in', 'selected subset of APS dataset'),\n",
       "   ('ten years', 'of average be', 'solid black line')],\n",
       "  [('integrated APS network', 'be', 'modular')],\n",
       "  [('network', 'have', 'community structure'),\n",
       "   ('we', 'destroy', \"network 's community structure\")],\n",
       "  [('Panel A Community sizes probability density function',\n",
       "    'be in',\n",
       "    'original network')]],\n",
       " 47: [[('both', 'be in', 'contact')],\n",
       "  [('quantum formulation', 'be in', 'terms of spin operators')],\n",
       "  [('model', 'describe', 'well-localized disease outburst')],\n",
       "  [('system', 'be', 'nonequilibrium')],\n",
       "  [('Our eﬀort', 'understand', 'evolution of cooperative behavior')],\n",
       "  [('SIR model',\n",
       "    'oﬀers',\n",
       "    'simple approach by set of evolution equations NumericCitation')],\n",
       "  [('S', 'contact between', 'individual')],\n",
       "  [('contact', 'be with', 'other')],\n",
       "  [('quantitative comparison', 'be with', 'mean-ﬁeld solution')],\n",
       "  [('we', 'account for', 'ﬂuctuations')],\n",
       "  [('we', 'ﬁnish in', 'Sec. 6')],\n",
       "  [('spontaneous recovery', 'be with', 'rate γ')],\n",
       "  [('last equation',\n",
       "    'follow from',\n",
       "    'simply conservation of total number of individuals')],\n",
       "  [('it', 'introduce', 't')],\n",
       "  [('randomness', 'be in', 'infection')],\n",
       "  [('individuals', 'be represent by', 'nodes of graph')],\n",
       "  [('distributed random time', 'be with', 'rate β I'),\n",
       "   ('contact', 'be with', 'i')],\n",
       "  [('quantity', 'be', 'random')],\n",
       "  [('distributed random time', 'be with', 'ﬁxed rate γ')],\n",
       "  [('stochastic process',\n",
       "    'be in',\n",
       "    'Thus double contrast to evolution studied in mean ﬁeld approach')],\n",
       "  [('infection rate',\n",
       "    'be proportional to',\n",
       "    'size of ful l population of infectious individuals')],\n",
       "  [('contact', 'be with', 'other')],\n",
       "  [('ﬂuctuations', 'disappear by', 'law of large numbers')],\n",
       "  [('complete graph',\n",
       "    'be in',\n",
       "    'thermodynamic limit of inﬁnite population size')],\n",
       "  [('expected state h nX', 'be in', 'our stochastic model')],\n",
       "  [('We', 'shall focus on', 'uncorrelated random initial distributions')],\n",
       "  [('we', 'write', 'it')],\n",
       "  [('single susceptible S',\n",
       "    'catch disease in',\n",
       "    'when contact with infectious individuals situated on neighboring nodes'),\n",
       "   ('it',\n",
       "    'be in',\n",
       "    'when contact with infectious individuals situated on neighboring nodes'),\n",
       "   ('contact',\n",
       "    'be with',\n",
       "    'infectious individuals situated on neighboring nodes')],\n",
       "  [('last process', 'be in', 'Eq')],\n",
       "  [('infectious individual', 'become', 'immunized')],\n",
       "  [('APPROACH', 'be', 'important'),\n",
       "   ('important issue',\n",
       "    'be in',\n",
       "    'theoretical understanding of biological processes'),\n",
       "   ('QUANTUM APPROACH',\n",
       "    'be in',\n",
       "    'quantum Hamiltonian representation Since combined inﬂuence of noise of freedom')],\n",
       "  [('we',\n",
       "    'here use',\n",
       "    'transformation of master equation into quantum language'),\n",
       "   ('transparent method', 'transformation of', 'master equation'),\n",
       "   ('transparent method', 'transformation into', 'quantum language')],\n",
       "  [('us', 'get', 'exact solution for process deﬁned above')],\n",
       "  [('occupation numbers', 'as nx be', 'i'), ('particles', 'by nx be', 'i')],\n",
       "  [('probability distribution',\n",
       "    'be in',\n",
       "    'terms of initial distribution at time t =')],\n",
       "  [('result', 'as i. be', '11')],\n",
       "  [('exclusion principle', 'eigenvalue of', 'particle operators')],\n",
       "  [('Similar relations', 'hold for', 'Bi')],\n",
       "  [('B. Cluster', 'use', 'Eq')],\n",
       "  [('cluster equations',\n",
       "    'simplify under',\n",
       "    'natural assumption of translation invariant initial distribution')],\n",
       "  [('one straightforward calculation',\n",
       "    'only take algebraic property eq into',\n",
       "    'account')],\n",
       "  [('recovery', 'be with', 'rate γ')],\n",
       "  [('G', 'be', 'arbitrary'), ('G', 'be', 'initial')],\n",
       "  [('solution', 'be in', 'ﬁrst equation')],\n",
       "  [('relaxation time',\n",
       "    'depend on',\n",
       "    'initial conditions through initial density of susceptibles')],\n",
       "  [('comparison',\n",
       "    'be with',\n",
       "    'predictions of original SIR model deﬁned by set of diﬀerential equations')],\n",
       "  [('ﬁrst equation', 'be in', 'Eq')],\n",
       "  [('S', 'be', 'decreasing')],\n",
       "  [('maximum', 'be in', 'number of infectious individual'),\n",
       "   ('situation',\n",
       "    'be more subtle in',\n",
       "    'condition for existence of maximum in number of infectious individual')],\n",
       "  [('second equation', 'be in', 'Eq')],\n",
       "  [('same relation', 'be in', 'case of random initial conditions')],\n",
       "  [('our model',\n",
       "    'hence reproduce',\n",
       "    'several key features of original SIR model')],\n",
       "  [('average behavior',\n",
       "    'be with',\n",
       "    'deterministic behavior of mean ﬁeld model')],\n",
       "  [('maximum', 'be in', 'I')],\n",
       "  [('maximum', 'be in', 'I')],\n",
       "  [('mean ﬁeld model', 'capture', 'eﬀects of low connectivity')],\n",
       "  [('ﬁrst equation', 'be in', 'Eq')],\n",
       "  [('g', 'be with', 'Eq'), ('Eq', 'with g be', 't')],\n",
       "  [('overdamped motion', 'be in', 'potential dR dt =')],\n",
       "  [('Eq', 'be in', 'limit t')],\n",
       "  [('result', 'be strongly diﬀerent from', '31')],\n",
       "  [('deterministic SIR dynamics',\n",
       "    'be with',\n",
       "    'renormalized infection rate βmf')],\n",
       "  [('Our exact results',\n",
       "    'be obtain for',\n",
       "    'thermodynamic limit of inﬁnite population size')],\n",
       "  [('one', 'expect in', 'single outbreak of disease')],\n",
       "  [('infectious individual', 'be with', 'probability nI =')],\n",
       "  [('arbitrary lattice site j', 'be', 'chosen randomly')],\n",
       "  [('R', 'be with', 'probability γ')],\n",
       "  [('procedure', 'be', 'repeated')],\n",
       "  [('N such update steps', 'deﬁne', 'one Monte-Carlo time step')],\n",
       "  [('maximum', 'be in', 'number of infectious particles'),\n",
       "   ('two diﬀerent runs', 'be with', 'population size 1000')],\n",
       "  [('1/N', 'be of', 'order 1')],\n",
       "  [('single run', 'be with', 'computed mean')],\n",
       "  [('deviations', 'be at', 'most in range of few percent')],\n",
       "  [('Fig. 2', 'demonstrate', 'the')],\n",
       "  [('dashed line', 'represent', 'exact solution')],\n",
       "  [('N', '=', '103 0 500000 1000000 1500000 0')],\n",
       "  [('dashed line', 'be', 'exact solution')],\n",
       "  [('CONCLUSIONS', 'be in', 'work'),\n",
       "   ('population',\n",
       "    'for model be',\n",
       "    'infectious I evolving under stochastic dynamics')],\n",
       "  [('they', 'anticommute at', 'same site'),\n",
       "   ('Pauli operators', 'commute at', 'diﬀerent lattice sites')],\n",
       "  [('us',\n",
       "    'ﬁnd',\n",
       "    'coupled set of evolution equations forcertain cluster functions')],\n",
       "  [('clusters',\n",
       "    'allow for',\n",
       "    'exact analytical treatment of whole hierarchy of evolution equations')],\n",
       "  [('disease', 'be in', 'framework of SIR mechanism')],\n",
       "  [('our model', 'of equation be', '28')],\n",
       "  [('mean-ﬁeld model', 'be with', 'small renormalized infection rate βmf')],\n",
       "  [('eﬀect', 'be in', 'initial distribution')]],\n",
       " 48: [[('seasonal cholera outbreaks', 'be in', 'Bangladesh')],\n",
       "  [('cholera outbreaks', 'be in', 'Bangladesh'),\n",
       "   ('We',\n",
       "    'develop',\n",
       "    'predictive model for cholera outbreaks in Bangladesh based on environmental predictors')],\n",
       "  [('we', 'do', 'this'),\n",
       "   ('contribution', 'be in', 'context of disease transmission model')],\n",
       "  [('observed states', 'be', 'numbers of cholera cases reported')],\n",
       "  [('We', 'ﬁt', 'hidden SIRS model')],\n",
       "  [('infected individuals',\n",
       "    'be in',\n",
       "    'population weeks observed number of cholera cases increases'),\n",
       "   ('increase', 'be in', 'number of infected individuals')],\n",
       "  [('1 Introduction', 'be in', 'Bangladesh'), ('r', 'be', 'endemic')],\n",
       "  [('several weeks', 'be in', 'advance')],\n",
       "  [('other resources', 'curb', 'impact of disease')],\n",
       "  [('disease dynamics', 'be', 'how related to environmental covariates')],\n",
       "  [('propagation', 'be in', 'environment'),\n",
       "   ('proliferation', 'be in', 'environment')],\n",
       "  [('administrative subdistricts', 'be with', 'police station'),\n",
       "   ('multiple thanas', 'be in', 'rural Bangladesh')],\n",
       "  [('current collection phase', 'be', 'ongoing')],\n",
       "  [('association', 'be in', 'thana'),\n",
       "   ('NumericCitation',\n",
       "    'study',\n",
       "    'association between lagged predictors from particular water body to cholera cases in thana')],\n",
       "  [('non-linear dynamics', 'be in', 'speciﬁcally tractability of likelihood'),\n",
       "   ('diﬃculties', 'be with', 'statistical inference'),\n",
       "   ('tractability', 'in dynamics be', 'Citation')],\n",
       "  [('Grenfell NumericCitation', 'overcome', 'intractability')],\n",
       "  [('large scale climate ﬂuctuations inﬂuence cholera transmission',\n",
       "    'be in',\n",
       "    'Bangladesh')],\n",
       "  [('NumericCitation', 'study', 'malaria transmission'),\n",
       "   ('NumericCitation', 'study', 'malaria transmission in India'),\n",
       "   ('malaria transmission', 'be in', 'India')],\n",
       "  [('seasonal cholera epidemics', 'be in', 'Bangladesh'),\n",
       "   ('we',\n",
       "    'similarly here develop',\n",
       "    'model-based predictive framework for seasonal cholera epidemics in Bangladesh')],\n",
       "  [('we', 'incorporate', 'environmental covariates'),\n",
       "   ('we', 'incorporate', 'covariates'),\n",
       "   ('cholera transmission', 'be in', 'Bangladesh')],\n",
       "  [('We', 'describe by', 'Andrieu et al.')],\n",
       "  [('we',\n",
       "    'further predict',\n",
       "    'future behavior of epidemic within our Bayesian framework')],\n",
       "  [('Cholera transmission dynamics', 'be in', 'our model')],\n",
       "  [('model', 'be with', 'time-varying environmental covariate'),\n",
       "   ('our Bayesian inference procedure',\n",
       "    'use',\n",
       "    'simulated cholera data generated from model with time-varying environmental covariate')],\n",
       "  [('diﬀerent time intervals', 'be in', 'data observation period'),\n",
       "   ('changes', 'be in', 'reported number of cases')],\n",
       "  [('predictive distribution', 'number of', 'infected individuals'),\n",
       "   ('cholera surveillance', 'be in', 'Bangladesh'),\n",
       "   ('two three weeks', 'be in', 'advance'),\n",
       "   ('our methodology',\n",
       "    'potentially useful during',\n",
       "    'cholera surveillance in Bangladesh'),\n",
       "   ('our methodology', 'useful during', 'cholera surveillance in Bangladesh')],\n",
       "  [('2 SIRS model', 'be with', 'environmental predictors'),\n",
       "   ('We', 'consider', 'compartmental model of disease transmission')],\n",
       "  [('We', 'model', 'continuous process observed at discrete time points')],\n",
       "  [('amount', 'be in', 'environment'),\n",
       "   ('susceptible individual', 'have', 'rate of infection')],\n",
       "  [('1/γ', 'be', 'where average length of infectious period')],\n",
       "  [('infected individual', 'once have recover from', 'infection')],\n",
       "  [('1/µ', 'be', 'where average length of immunity')],\n",
       "  [('inﬁnitesimal rates', 'with process be', 'Citation'),\n",
       "   ('inhomogeneous Markov process', 'be with', 'inﬁnitesimal rates')],\n",
       "  [('immunity', 'be', 'lost')],\n",
       "  [('I', 'µr', '0'), ('S', '=', 'S − 1'), ('S', '=', 'S'), ('I', '=', 'I')],\n",
       "  [('Rt', '=', 'N − St −')],\n",
       "  [('other cholera models', 'be in', 'literature')],\n",
       "  [('model', 'only consider', 'direct transmission')],\n",
       "  [('Code ¸ co NumericCitation', 'use sir model with', 'R +1 S-1')],\n",
       "  [('t', 'be', 'random')],\n",
       "  [('success probability Iti ρ',\n",
       "    'probability of',\n",
       "    'infected individuals seeking'),\n",
       "   ('ti distribution', 'be with', 'size Iti')],\n",
       "  [('inﬁnitesimal rates', 'be in', 'Equation')],\n",
       "  [('k', 'force of', 'infection')],\n",
       "  [('We', 'assume', 't')],\n",
       "  [('independent Poisson initial distributions', 'be with', 'means φS')],\n",
       "  [('logit transformation', 'be use for', 'probability ρ')],\n",
       "  [('NumericCitation', 'require', 'only unbiased estimate of likelihood')],\n",
       "  [('we', 'construct', 'likelihood estimate')],\n",
       "  [('Poisson distributions', 'be with', 'means φS')],\n",
       "  [('We', 'simulate', 'tra jectories of unobserved states')],\n",
       "  [('estimated likelihood', 'be in', 'acceptance ratio')],\n",
       "  [('hidden state processes',\n",
       "    'to x be',\n",
       "    'X ∗ particle weights of SMC algorithm'),\n",
       "   ('proposed X ∗', 'weight of', 'SMC algorithm')],\n",
       "  [('cholera transmission model',\n",
       "    'be with',\n",
       "    'time-varying environmental force of infection'),\n",
       "   ('we',\n",
       "    'simulate from',\n",
       "    'cholera transmission model with time-varying environmental force of infection')],\n",
       "  [('This', 'be', 'common'),\n",
       "   ('common problem', 'be in', 'chemical kinetics literature')],\n",
       "  [('of time', 'in rate be', 'k2 ∼ Poisson of infections'),\n",
       "   ('inﬁnitesimal rates', 'be in', 'of time'),\n",
       "   ('infection', 'be in', 'time'),\n",
       "   ('inﬁnitesimal rates', 'poisson of', 'infections')],\n",
       "  [('We', 'deﬁne', 'daily time intervals Ai')],\n",
       "  [('our rates', 'use', 'τ')],\n",
       "  [('it',\n",
       "    'construct',\n",
       "    'covariance of multivariate normal random walk proposal distribution in ﬁnal run of PMMH algorithm'),\n",
       "   ('multivariate normal random walk proposal distribution',\n",
       "    'be in',\n",
       "    'ﬁnal run of PMMH algorithm')],\n",
       "  [('we', 'assess', 'predictive ability of our model')],\n",
       "  [('weights', 'of set be', 'Citation')],\n",
       "  [('we',\n",
       "    'compare',\n",
       "    'posterior predictive distribution of observed cholera cases to test data')],\n",
       "  [('hidden SIRS model',\n",
       "    'be with',\n",
       "    'time-varying environmental force of infection'),\n",
       "   ('5 Simulation',\n",
       "    'test pmmh algorithm on',\n",
       "    'simulated infectious disease data'),\n",
       "   ('we',\n",
       "    'generate',\n",
       "    'data from hidden SIRS model with time-varying environmental force of infection')],\n",
       "  [('many people', 'have', 'where have previously infected'),\n",
       "   ('we', 'assume', 'independent Poisson initial distributions'),\n",
       "   ('many people', 'have', 'have previously infected')],\n",
       "  [('we', 'use', 'modiﬁed Gillespie algorithm described in Appendix A')],\n",
       "  [('t. Simulation', 'be with', 'seasonally varying α'),\n",
       "   ('simulated hidden states',\n",
       "    'of plot be',\n",
       "    't. Simulation with seasonally varying α'),\n",
       "   ('data', 'be with', 'seasonal epidemic peaks'),\n",
       "   ('enough information', 'be in', 'data'),\n",
       "   ('Plots', 'simulation with', 'seasonally varying α')],\n",
       "  [('we', 'predict datum after', 'line'), ('line', 'estimate', 'parameters')],\n",
       "  [('models', 'be with', 'parameter assumptions')],\n",
       "  [('10000 iterations', 'run of', '10000 iterations'),\n",
       "   ('PMMH algorithm', 'use', 'data')],\n",
       "  [('models', 'be with', 'diﬀerent assumptions on values of φS')],\n",
       "  [('algorithm', 'provide', 'good estimates of true parameter values')],\n",
       "  [('Future cases', 'be', 'then predicted until next cut oﬀ')],\n",
       "  [('we', 'separate', 'our simulated data')],\n",
       "  [('simulated observed data', 'be show in', 'right plot of Figure 2')],\n",
       "  [('Case counts', 'be observe at_time', 'once 14 days')],\n",
       "  [('case', 'count in', 'test data')],\n",
       "  [('plot', 'be in', 'bottom row of Figure 3'),\n",
       "   ('our model',\n",
       "    'see in',\n",
       "    'tra jectory of predicted fraction of infected individuals')],\n",
       "  [('plot',\n",
       "    'illustrate',\n",
       "    'interplay of hidden states of underlying compartmental model')],\n",
       "  [('susceptible individuals', 'be', 'born')],\n",
       "  [('we', 'test', 'sensitivity to assumptions')],\n",
       "  [('water bodies', 'in temperature be', 'WT'),\n",
       "   ('signiﬁcant lagged relationship', 'be with', 'cholera incidence'),\n",
       "   ('temperature', 'be in', 'water bodies')],\n",
       "  [('Environmental data',\n",
       "    'be',\n",
       "    'also collected approximately two weeks from six water bodies')],\n",
       "  [('t0', 't0 of', 'lag')],\n",
       "  [('sample standard intervals', 'be', 'eﬀect')],\n",
       "  [('κ', '=', '21')],\n",
       "  [('Figure 4', 'count in', 'Mathbaria')],\n",
       "  [('covariates', 'be show with', 'lag of three weeks')],\n",
       "  [('normal prior distributions', 'be with', 'standard deviations of 5')],\n",
       "  [('normal prior distribution', 'be with', 'mean log'),\n",
       "   ('12 Jan 04Apr 04Jul 04Oct 04Jan 05Apr 05Jul 05Oct 05Jan 06Apr 06Jul 06Oct 06Jan 07Apr 07Jul 07Oct 07Jan 08Apr 08Jul 08Oct 08Jan 09Apr 09Jul 09Oct 09Jan 10Apr 10Jul 10Oct 10Jan 11Apr 11Jul 11Oct 11Jan 12Apr 12Jul 12Oct 12Jan 13Apr 13Jul 130246810Number Cholera Cases',\n",
       "    'be in',\n",
       "    'Mathbaria − 2 −')],\n",
       "  [('normal prior distribution', 'be with', 'mean logit')],\n",
       "  [('changes', 'be in', 'population size N')],\n",
       "  [('infection', 'of force be', 'Citation'),\n",
       "   ('higher concentration', 'be in', 'environment')],\n",
       "  [('infected individuals', 'be in', 'population'),\n",
       "   ('γ', 'part of', 'reproductive number')],\n",
       "  [('epidemic peaks', 'be in', 'our model')],\n",
       "  [('full cholera data', 'be with', 'smoothed covariates')],\n",
       "  [('parameter values', 'simulate', 'possible realizations of test data')],\n",
       "  [('case counts', 'of peak be', 'Citation Citation')],\n",
       "  [('plot', 'be in', 'bottom row of Figure 5')],\n",
       "  [('prediction', 'in uncertainty be', 'Citation Citation'),\n",
       "   ('more uncertainty', 'be in', 'prediction for fraction of susceptibles')],\n",
       "  [('predicted fraction', 'be in', 'population')],\n",
       "  [('We',\n",
       "    'also use',\n",
       "    'quasi-Poisson regression model similar to one used by Huq et al.')],\n",
       "  [('it', 'overestimate', 'duration of outbreaks')],\n",
       "  [('hidden variable', 'be in', 'SIRS model')],\n",
       "  [('cholera transmission', 'be in', 'Bangladesh')],\n",
       "  [('we', 'recover well', 'true parameter values'),\n",
       "   ('hidden SIRS model',\n",
       "    'be with',\n",
       "    'time-varying environmental force of infection')],\n",
       "  [('we', 'test', 'predictive ability of our model')],\n",
       "  [('prediction', 'overall result', 'look promising')],\n",
       "  [('noticeable increase', 'be in', 'fraction of infected individuals'),\n",
       "   ('predicted hidden states', 'base on', 'data collected')],\n",
       "  [('fraction', 'be in', 'population')],\n",
       "  [('eﬃciency', 'be', 'important'), ('approach', 'be in', 'ﬁeld')],\n",
       "  [('we', 'also assume', 'ρ')],\n",
       "  [('we', 'address model reﬁnements in', 'future analyses')],\n",
       "  [('we', 'allow for', 'variable selection over large number of covariates')],\n",
       "  [('sparsity', 'induce', '16 priors')],\n",
       "  [('NumericCitation', 'would allow for', 'such extensions')],\n",
       "  [('further model development', 'for option be', 'Citation')],\n",
       "  [('information', 'accounting for', 'correlations between thanas')],\n",
       "  [('concentration', 'be in', 'environment'), ('we', 'accomplish', 'this')]],\n",
       " 49: [[('immunization strategies', 'have', 'have developed')],\n",
       "  [('we', 'consider', 'limited vaccines'),\n",
       "   ('we', 'consider', 'vaccines'),\n",
       "   ('Susceptible-Infected-Recovered model',\n",
       "    'be with',\n",
       "    'local dynamic vaccination')],\n",
       "  [('susceptibles', 'be in', 'contact with infected individual'),\n",
       "   ('contact', 'be with', 'infected individual')],\n",
       "  [('disease', 'be with', 'ﬁxed β')],\n",
       "  [('spreading', 'be in', 'social systems')],\n",
       "  [('vaccination programs NumericCitation', 'provide', 'immunity'),\n",
       "   ('vaccination programs NumericCitation',\n",
       "    'provide immunity to',\n",
       "    'individuals')],\n",
       "  [('non-pharmaceutical interventions', 'protect', 'society')],\n",
       "  [('it', 'ﬁnd', 'balance between two strategies')],\n",
       "  [('individual', 'be in', 'state S'),\n",
       "   ('individual', 'be in', 'contact with I'),\n",
       "   ('contact', 'be with', 'I')],\n",
       "  [('system', 'be', 'epidemic-free'), ('it', 'be', 'epidemic')],\n",
       "  [('NumericCitation',\n",
       "    'use',\n",
       "    'eﬀect of intermittent social distancing strategy on propagation of epidemics in adaptive complex networks'),\n",
       "   ('propagation', 'be in', 'adaptive complex networks'),\n",
       "   ('SIR model',\n",
       "    'eﬀect of',\n",
       "    'intermittent social distancing strategy on propagation in adaptive complex networks')],\n",
       "  [('contact', 'be with', 'infected individual')],\n",
       "  [('Ebola', 'of', 'Citation')],\n",
       "  [('high impact', 'be in', 'mainly Guinea')],\n",
       "  [('Citation', 'be in', 'capital city')],\n",
       "  [('vaccine trial', 'test', 'eﬃcacy of experimental vaccine against Ebola')],\n",
       "  [('models', 'be with', 'scenario of limited vaccines availability')],\n",
       "  [('we', 'motivate by', 'this'), ('vaccine trial', 'be in', 'Guinea')],\n",
       "  [('limited amount', 'be with', 'aim reducing')],\n",
       "  [('contact', 'be with', 'infected one'),\n",
       "   ('susceptible individual', 'be in', 'contact with infected one')],\n",
       "  [('R', 'approach to', 'zero')],\n",
       "  [('authors', 'use', 'susceptible-infected-susceptible')],\n",
       "  [('vaccine', 'be with', 'probability ω')],\n",
       "  [('We',\n",
       "    'deﬁne vl as',\n",
       "    'fraction of available vaccines over entire population')],\n",
       "  [('may contact', 'be with', 'infected individual'),\n",
       "   ('may contact', 'be in', 'random strategy')],\n",
       "  [('dynamical vaccination',\n",
       "    'thus make',\n",
       "    'more eﬀective usage of available vaccines')],\n",
       "  [('rules', 'be with', 'limited vaccines for small network of size N ='),\n",
       "   ('rules', 'of demonstration be', 'recovery time tr =')],\n",
       "  [('its neighbors', 'be with', 'probability ω')],\n",
       "  [('tr', '=', '1')],\n",
       "  [('node', 'be in', 'target of arrow')],\n",
       "  [('individuals', 'be in', 'susceptible state')],\n",
       "  [('we', 'compute', 'θt')],\n",
       "  [('it', 'be', 'infected'), ('it', 'induced vaccination to', 'root node')],\n",
       "  [('probability', 'be denote by', 'ΦR')],\n",
       "  [('it', 'do induce', 'vaccination of root node Recovered')],\n",
       "  [('probabilities', 'be in', 'Table I'),\n",
       "   ('We', 'summarize', 'probabilities in Table I')],\n",
       "  [('neighbor', 'be with', 'degree k')],\n",
       "  [('base node', 'be susceptible at', 'time t')],\n",
       "  [('∆ θt +1', '=', '1 − ω')],\n",
       "  [('Diagram',\n",
       "    'show',\n",
       "    'relations between variables used in compartmental model')],\n",
       "  [('third equation', 'be', 'bit more complicated')],\n",
       "  [('immunization', 'cause', 'its infection'),\n",
       "   ('probability ΦI', 'neighbor at', 'time t'),\n",
       "   ('its immunization', 'cause', 'its infection')],\n",
       "  [('infected base node', 'infect', 'it')],\n",
       "  [('will state I', 'be in', 'next time step'),\n",
       "   ('we', 'be in', 'state I in next time step')],\n",
       "  [('nodes', 'be in', 'state S')],\n",
       "  [('last term', 'be in', '3rd equation')],\n",
       "  [('additional probabilities', 'be in', 'Table II'),\n",
       "   ('We', 'explain', 'additional probabilities in Table II')],\n",
       "  [('ﬁnite diﬀerence', 'be in', 'ﬁrst equation'),\n",
       "   ('change', 'be in', 'fraction of susceptible individuals')],\n",
       "  [('change', 'be in', 'susceptible individuals')],\n",
       "  [('change', 'be in', 'fraction of infected individuals')],\n",
       "  [('fraction',\n",
       "    'hence contribute negatively to',\n",
       "    'fraction of infected individuals')],\n",
       "  [('temporal evolution', 'be with', 'unlimited vaccines')],\n",
       "  [('we',\n",
       "    'have',\n",
       "    'limited amount of vaccination units lower than number of individuals in system'),\n",
       "   ('number', 'be in', 'system')],\n",
       "  [('we',\n",
       "    'thus impose limit vl as',\n",
       "    'maximal fraction of vaccinated individuals')],\n",
       "  [('V', 'match exactly with', 'VL')],\n",
       "  [('immunization process', 'be', 'stopped')],\n",
       "  [('problem', 'reproduce', 'exactly results from computational simulations')],\n",
       "  [('ER network', 'be with', 'hki ='),\n",
       "   ('SIR model', 'of evolution be', 'tr ='),\n",
       "   ('local vaccination', 'with evolution be', 'VL ='),\n",
       "   ('SIR model', 'of evolution be', 'VL ='),\n",
       "   ('ω =', 'for evolution be', 'tr ='),\n",
       "   ('ω =', 'for evolution be', 'VL ='),\n",
       "   ('Temporal evolution', 'be with', 'local vaccination for ER network'),\n",
       "   ('local vaccination', 'with evolution be', 'tr =')],\n",
       "  [('vaccines', 'be', 'depleted')],\n",
       "  [(\"Erd˝os R'enyi\", 'be with', 'average degree hki ='),\n",
       "   ('average degree hki =', \"with r'enyi be\", 'ER')],\n",
       "  [('networks', 'use', 'Molloy-Reed algorithm NumericCitation')],\n",
       "  [('13 A. Temporal Evolution', 'be in', 'Fig. 3')],\n",
       "  [('vaccines', 'be', 'when suddenly exhausted')],\n",
       "  [('disease', 'fade away as', 'fraction of infected individuals')],\n",
       "  [('κ = hk2i/hki', 'be branch factor of', 'degree distribution of network'),\n",
       "   ('tr', 'eﬀective probability of', 'contagion')],\n",
       "  [('dramatic changes', 'be in', 'behavior of magnitudes at steady state')],\n",
       "  [('we', 'call', 'β ∗'), ('we', 'call', '∗')],\n",
       "  [('other magnitudes', 'also show', 'standard behavior')],\n",
       "  [('further increase', 'be in', 'fraction of recovered nodes'),\n",
       "   ('further decrease', 'be in', 'number of susceptible individuals')],\n",
       "  [('abrupt transition', 'be with', 'β')],\n",
       "  [('single peak', 'be in', 'as Fig. 4')],\n",
       "  [('ER network', 'be with', 'hki ='),\n",
       "   ('model', 'of evolution be', 'VL ='),\n",
       "   ('ω =', 'for evolution be', 'VL =')],\n",
       "  [('insets',\n",
       "    'show',\n",
       "    'magniﬁed image of temporal fraction of infected individuals')],\n",
       "  [('ER network', 'be with', 'hki ='),\n",
       "   ('results', 'be', 'how aﬀected by infection probability β'),\n",
       "   ('fraction', 'be in', 'steady state')],\n",
       "  [('green solid lines', 'represent', 'case of unlimited vaccines')],\n",
       "  [('VL', '=', '1'), ('VL', 'i.e. =', '1')],\n",
       "  [('decrease', 'be in', 'fraction of vaccinated individuals')],\n",
       "  [('networks', 'be with', 'heterogeneous power law degree distribution')],\n",
       "  [('β', 'of range be', 'VL'), ('V', 'reach', 'its limit value')],\n",
       "  [('abrupt transition', 'be in', 'Fig. 5')],\n",
       "  [('VL =', 'supply of', 'immunization units')],\n",
       "  [('curve', 'be with', 'blue circles')],\n",
       "  [('it', 'reach', 'steady state'), ('we', 'see in', 'Fig. 4')],\n",
       "  [('ER', 'be with', 'hki =')],\n",
       "  [('VL', '=', '0.45'), ('VL', '=', '0.4'), ('VL', '=', '0.5')],\n",
       "  [('it', 'reach', 'steady state')],\n",
       "  [('we', 'take into', 'consideration')],\n",
       "  [('process', 'reach', 'steady state')],\n",
       "  [('vaccination units', 'stop', 'it'),\n",
       "   ('immunization strategy', 'stop', 'spreading of disease')],\n",
       "  [('Fig. 9', 'be in', 'appendix E')],\n",
       "  [('steady state change', 'be with', 'vaccination probability ω')],\n",
       "  [('VL =', 'be in', 'Figs. 6'), ('VL =', 'be in', 'Figs. 6')],\n",
       "  [('it', 'start', 'decreasing')],\n",
       "  [('vaccines', 'be', 'depleted')],\n",
       "  [('vaccination', 'be with', 'probability ω')],\n",
       "  [('small increase', 'be in', 'fraction of immunized individuals'),\n",
       "   ('dramatic drop', 'be in', 'number of recovered individuals'),\n",
       "   ('= ω ∗', 'increase in', 'fraction of immunized individuals')],\n",
       "  [('one', 'β', '†'), ('one', 'can easily relate', 'ω †')],\n",
       "  [('number', 'be in', 'system'), ('disease', 'be with', 'β')],\n",
       "  [('generating functions formalism', 'branch', 'theory NumericCitation')],\n",
       "  [('solid curve', 'be in', 'VL − β plane'), ('ω', '=', '0.45')],\n",
       "  [('vaccines-depletion region', 'be in', 'purple')],\n",
       "  [('suﬃcient vaccines', 'be in', 'yellow'),\n",
       "   ('little vaccines', 'stop', 'disease')],\n",
       "  [('we', 'show', 'phase diagram in plane VL − ω'),\n",
       "   ('phase diagram', 'be in', 'plane VL − ω')],\n",
       "  [('immunization strategy', 'get', 'more eﬀective against disease')],\n",
       "  [('available immunization resources', 'be in', 'medical institutes')],\n",
       "  [('β =', 'be in', 'infection probability')],\n",
       "  [('V. DISCUSSION', 'be in', 'manuscript'),\n",
       "   ('vaccines', 'be in', 'S I R model with local vaccination'),\n",
       "   ('S I R model', 'be with', 'local vaccination')],\n",
       "  [('medical institutions', 'run out of', 'immunization units')],\n",
       "  [('Phase diagram VL', 'be with', 'ω ='), ('VL =', 'be in', 'plane')],\n",
       "  [('†', 'be in', 'b')],\n",
       "  [('ωc', 'be in', 'b')],\n",
       "  [('height', 'depend on', 'VL')],\n",
       "  [('discontinuous transition', 'be', 'observed')],\n",
       "  [('its existence', 'depend on', 'topology of network')],\n",
       "  [('vaccination probability',\n",
       "    'control',\n",
       "    'epidemic with available immunization resources'),\n",
       "   ('discontinuous transition', 'be in', 'fraction of recovered individuals')],\n",
       "  [('model', 'use', 'EBCM'),\n",
       "   ('model', 'ﬁnding', 'excellent agreement with stochastic simulations'),\n",
       "   ('excellent agreement', 'be with', 'stochastic simulations')],\n",
       "  [('Israel Science Foundation', 'ministry of', 'Science'),\n",
       "   ('Israel Ministry', 'be with', 'Italy Ministry of Foreign Aﬀairs'),\n",
       "   ('Israel Science Foundation',\n",
       "    'ministry with',\n",
       "    'Italy Ministry of Foreign Aﬀairs'),\n",
       "   ('BIU Center', 'be in', 'Applied Cryptography')],\n",
       "  [('appendix', 'in model be', 'EBCM'),\n",
       "   ('Edge-based compartmental model', 'be in', 'appendix')],\n",
       "  [('recovery time tr', 'also be replace by', 'recovery rate γ')],\n",
       "  [('we', 'can adapt', 'them')],\n",
       "  [('we', 'rewrite', 'Eq')],\n",
       "  [('ΦR', 'be', 'negligible at beginning of process')],\n",
       "  [('equation', 'be with', 'S')],\n",
       "  [('we', 'alternatively can derive', 'Eq'), ('we', 'can derive', 'Eq')],\n",
       "  [('equations', 'of set be', '4')],\n",
       "  [('we', 'avoid', 'ﬂuctuations')],\n",
       "  [('ω0', 'be', 'where vaccination probability at beginning of process')],\n",
       "  [('deal', 'be with', 'limit of vaccine units'),\n",
       "   ('we', 'iterate', 'equations')],\n",
       "  [('node', 'be in', 'state')],\n",
       "  [('node', 'be infect', 'probability of vaccination changes')],\n",
       "  [('ω', 'be prove in', 'detail'),\n",
       "   ('ω', 'be', 'proved in detail in Appendix C')],\n",
       "  [('ω', 'depend on', 'time Ω')],\n",
       "  [('we', 'include', 'immunization probability ω')],\n",
       "  [('ω', '=', '0')],\n",
       "  [('we', 'can write', 'Eq'), ('we', 'furthermore can write', 'Eq')],\n",
       "  [('C2', 'be in', 'closed form Ω1 =')],\n",
       "  [('Yj', '=', '1 27 Eq')],\n",
       "  [('limit', 'be in', 'number of vaccines'),\n",
       "   ('Computation', 'fraction of', 'vaccinated individuals'),\n",
       "   ('steady state', 'in computation be', 'fraction of vaccinated individuals'),\n",
       "   ('Computation', 'be in', 'steady state of our model'),\n",
       "   ('β ∗', 'of computation be', 'fraction')],\n",
       "  [('Ω1', 'be in', 'Eq'), ('expression', 'be same as', 'Ω1 in Eq')],\n",
       "  [('we', 'start', 'increasing β'), ('we', 'start', 'β')],\n",
       "  [('vaccinated individuals', 'be in', 'steady state')],\n",
       "  [('unlimited-vaccines curve',\n",
       "    'equal vaccination limit vl for',\n",
       "    'values of β')],\n",
       "  [('ω ∗', '=', 'ω †')],\n",
       "  [('solutions',\n",
       "    'be give by',\n",
       "    'intersection of r.h.s. of equations with identity'),\n",
       "   ('intersection', 'be with', 'identity')],\n",
       "  [('Supplementary ﬁgures', 'be in', 'Fig. 9')],\n",
       "  [('ﬁgure', 'fig.', '5')],\n",
       "  [('network', 'be with', 'power law degree distribution'),\n",
       "   ('other hand', 'be in', 'Fig. 10'),\n",
       "   ('fracion', 'be in', 'steady state')],\n",
       "  [('tr', '=', '1')],\n",
       "  [('β †', 'explain in', 'main text')],\n",
       "  [('recovered individuals', 'which as', 'expected decrease')],\n",
       "  [('recovery time tr', '=', '3'), ('ER', 'be with', 'hki =')],\n",
       "  [('it', 'take to', 'process')],\n",
       "  [('recovery time tr', '=', '3')],\n",
       "  [('E. Profumo', 'proceedings of', 'National Academy')],\n",
       "  [('NumericCitation N. T. J. Bailey', 'theory of', 'Infectious Diseases')],\n",
       "  [('Reports', 'be in', 'Physics 80 036603'),\n",
       "   ('L. A. Braunstein', 'reports on', 'Progress'),\n",
       "   ('L. A. Braunstein', 'reports in', 'Physics 80 036603')],\n",
       "  [('NumericCitation E. Volz', 'journal of', 'mathematical biology 56 293')],\n",
       "  [('mathematical biology', 'of journal be', '2011'),\n",
       "   ('NumericCitation J. C. Miller', 'journal of', 'mathematical biology')],\n",
       "  [('R. C. Brunham', 'journal of', 'theoretical biology 232 71')],\n",
       "  [('L. A. Meyers', 'proceedings of', 'Royal Society')],\n",
       "  [('NumericCitation L. G. Alvarez-Zuzek',\n",
       "    'be in',\n",
       "    'Journal of Physics Conference Series')],\n",
       "  [('its', 'eradication', 'Citation')],\n",
       "  [('E. M. Volz', 'journal of', 'Royal Society Interface 9 890')],\n",
       "  [('L. A. Braunstein', 'journal of', 'Physics 20 013007')],\n",
       "  [('NumericCitation F. Vel', 'jas be', 'Physical Review E 95 052315 2017')],\n",
       "  [('Scientiﬁc', 'report', '19767 2016')]],\n",
       " 50: [[('Severe population collapses',\n",
       "    'be in',\n",
       "    'multi-host epidemic dynamics')],\n",
       "  [('severity', 'be in', 'populations of two host species')],\n",
       "  [('large extinction-level population collapse', 'be in', 'one of species')],\n",
       "  [('Such collapse-driven extinctions',\n",
       "    'depend on',\n",
       "    'factors diﬀerent than those in endemic steady state of disease')],\n",
       "  [('pathogens', 'typically infect', 'species'),\n",
       "   ('pathogens', 'typically infect', 'multiple species')],\n",
       "  [('cholera commensal',\n",
       "    'be in',\n",
       "    'number zooplankton species NumericCitation'),\n",
       "   ('diseases', 'be with', 'multiple host species')],\n",
       "  [('disease', 'of', 'endemic state')],\n",
       "  [('we', 'use', 'only simpler two-species model')],\n",
       "  [('their', 'death', 'D')],\n",
       "  [('death', 'result in', 'population'),\n",
       "   ('former results', 'be in', 'population')],\n",
       "  [('its ultimate steady state population',\n",
       "    'be in',\n",
       "    'endemic state of diseases')],\n",
       "  [('population collapse', 'be', 'When especially severe')],\n",
       "  [('single key parameter', 'r0 to', 'βS'),\n",
       "   ('γ', 'be in', 'notation used above')],\n",
       "  [('population size', 'remain', 'virtually unchanged')],\n",
       "  [('epidemic spreads', 'number of', 'susceptible targets declines')],\n",
       "  [('non-linear term', 'be in', 'SIR model'),\n",
       "   ('exponential decline', 'be in', 'the'),\n",
       "   ('time', 'be in', 'units of number of deaths')],\n",
       "  [('its', 's', 'collapse')],\n",
       "  [('its steady state value', 'be in', 'endemic state of disease'),\n",
       "   ('collapse',\n",
       "    'be with',\n",
       "    'its steady state value in endemic state of disease')],\n",
       "  [('host', 'have', 'population')],\n",
       "  [('0 exp', 'drop below', 'one individual')],\n",
       "  [('that',\n",
       "    'say',\n",
       "    'diseases transiently or permanently transverse species boundaries')],\n",
       "  [('reduction', 'be in', 'gorilla populations NumericCitation')],\n",
       "  [('its', 'host', 'fruit eating bats')],\n",
       "  [('non-sexually diseases', 'be in', 'well-mixed populations'),\n",
       "   ('disease propagation', 'be in', 'species')],\n",
       "  [('we', 'ignore', 'births of new susceptible individuals'),\n",
       "   ('we', 'ignore', 'births')],\n",
       "  [('other timescales', 'be in', 'system')],\n",
       "  [('small numbers', 'be in', 'one'),\n",
       "   ('time t =', 'be with', 'very small numbers of infected hosts in one')],\n",
       "  [('1', 'see', 'Fig. 1')],\n",
       "  [('small birth term', 'be with', 'saturation')],\n",
       "  [('individuals', 'be', 'infertile'),\n",
       "   ('infected individuals', 'be', 'infertile')],\n",
       "  [('single pathogen', 'be with', 'infection rates')],\n",
       "  [('I2', '=', '10 − 6')],\n",
       "  [('we', 'describe in', 'text')],\n",
       "  [('second population S2', 'of state be', 'steady state')],\n",
       "  [('transmission', 'be', 'when unidirectional 2 → 1')],\n",
       "  [('equations 1', 'be with', 'β11 =')],\n",
       "  [('red dashed line', 'be in', 'Fig. 1')],\n",
       "  [('logarithm', 'be in', 'each of Eqs')],\n",
       "  [('other words', 'fraction of', 'survivors exp'),\n",
       "   ('survival probabilities',\n",
       "    'be in',\n",
       "    'infections transmitted by members of its own species')],\n",
       "  [('S1 collapse', 'be in', 'case of uni-directional transmission')],\n",
       "  [('Collapse ratio', 'be with', 'ﬁxed intra-species collapse factor C11 ='),\n",
       "   ('Collapse ratio', 'be in', 'population of species 1')],\n",
       "  [('overall reproduction number', 'fall below', '1')],\n",
       "  [('population', 'be with', 'ﬁxed value of C22 =')],\n",
       "  [('corporate recovery', 'be with', 'immunity')],\n",
       "  [('they', 'acthe', 'properties of system')],\n",
       "  [('Eq', 'be in', 'case')],\n",
       "  [('population collapse', 'be in', 'species 1')],\n",
       "  [('limit', 'correspond to', 'saturation of reservoir of species 2')],\n",
       "  [('species survival', 'be in', 'endemic state of disease'),\n",
       "   ('than initial epidemics', 'in parameter be', 'relative population sizes'),\n",
       "   ('diﬀerent parameters', 'be in', 'than initial epidemics'),\n",
       "   ('than initial epidemics', 'in parameter be', 'growth rates')],\n",
       "  [('ND', 'origins of', 'ma jor human infectious diseases')],\n",
       "  [('occurrence', 'be in', 'Chesapeake Bay')],\n",
       "  [('their role', 'be in', 'cholera')],\n",
       "  [('inﬂuence', 'be in', 'England 1500-1667')],\n",
       "  [('pathogenic H5N1 inﬂuenza virus infection', 'be in', 'migratory birds')],\n",
       "  [('pathogens', 'be with', 'multiple host species')],\n",
       "  [('infectious diseases', 'be in', 'heterogeneous populations'),\n",
       "   ('basic reproduction ratio R0', 'be in', 'models for infectious diseases')],\n",
       "  [('model', 'be with', 'bacteria')],\n",
       "  [('Ebola outbreak', 'kill', '5000 gorillas')],\n",
       "  [('EM', 'bat as', 'reservoirs of Ebola virus')],\n",
       "  [('NumericCitation Lederberg S. Suppression',\n",
       "    'be in',\n",
       "    'lysogenic bacteria')],\n",
       "  [('NumericCitation Maslov S',\n",
       "    'be in',\n",
       "    'dynamic Kill-the-Winner model of microbial ecosystems')],\n",
       "  [('Scientiﬁc Reports', 'be in', 'press')]],\n",
       " 51: [[('giant susceptible component', 'be', 'destroyed')],\n",
       "  [('Our theoretical results',\n",
       "    'be conﬁrmed by',\n",
       "    'extensive simulations of SIR process')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('mathematical tools', 'apply in', 'particular'),\n",
       "   ('mathematical tools', 'model', 'epidemics'),\n",
       "   ('tools', 'model', 'epidemics')],\n",
       "  [('infected individuals', 'recover at', 'ﬁxed time tr'),\n",
       "   ('step', 'be in', 'its discrete formulation')],\n",
       "  [('order', 'be in', 'SIR model'),\n",
       "   ('second order phase transition',\n",
       "    'be with',\n",
       "    'critical threshold transmissibility Tc')],\n",
       "  [('P', 'with network be', 'SF'),\n",
       "   ('pure Scale-Free network', 'be with', 'P')],\n",
       "  [('networks', 'be with', 'λ')],\n",
       "  [('real networks', 'however have', 'ﬁnite critical transmisibilities')],\n",
       "  [('Newman NumericCitation', 'use', 'generating function formalism')],\n",
       "  [('∗', 'be', 'important')],\n",
       "  [('void node', 'be in', 'link percolation'),\n",
       "   ('they', 'use', 'percolation framework'),\n",
       "   ('susceptible individual',\n",
       "    'correspond to',\n",
       "    'void node in link percolation')],\n",
       "  [('percolation theory',\n",
       "    'describe',\n",
       "    'steady state of SIR model on complex networks')],\n",
       "  [('system', 'be with', 'O')],\n",
       "  [('SIR process', 'be with', 'rates')],\n",
       "  [('network', 'economy of', 'region')],\n",
       "  [('dynamic void node percolation process',\n",
       "    'be with',\n",
       "    'instantaneous void control parameter')],\n",
       "  [('temporal second order phase transition',\n",
       "    'be with',\n",
       "    'mean ﬁeld exponents'),\n",
       "   ('giant susceptible component',\n",
       "    'overcome',\n",
       "    'temporal second order phase transition with mean ﬁeld exponents')],\n",
       "  [('we', 'finally present', 'our conclusions'),\n",
       "   ('we', 'present', 'our conclusions')],\n",
       "  [('he', 'be', 'infected')],\n",
       "  [('SIR', 'be with', 'ﬁxed recovery time')],\n",
       "  [('individuals', 'be in', 'SIR')],\n",
       "  [('ﬁrst one', 'be generate function of', 'k')],\n",
       "  [('node', 'be in', 'target of arrow'),\n",
       "   ('root', 'be in', 'which'),\n",
       "   ('base', 'be', 'its neighbor')],\n",
       "  [('root node', 'be with', 'connectivity k')],\n",
       "  [('increase', 'be in', 'Eq')],\n",
       "  [('susceptible individuals', 'be in', 'SIR model')],\n",
       "  [('A. Temporal percolation', 'be in', 'Ref')],\n",
       "  [('NumericCitation', 'be with', 'link occupancy probability T')],\n",
       "  [('cluster', 'reach nth generation follow', 'link')],\n",
       "  [('void cluster', 'lose', 'node')],\n",
       "  [('We', 'call', 'type of percolation node void percolation')],\n",
       "  [('void node', 'be remove due to', 'occupancy of link')],\n",
       "  [('susceptible individuals', 'be in', 'steady state')],\n",
       "  [('void network', 'undergo', 'second order phase transition')],\n",
       "  [('percolation', 'be', 'useful')],\n",
       "  [('12', 'be', 'time-dependent'), ('13', 'be', 'time-dependent')],\n",
       "  [('ER', 'be with', 'same hki for'), ('function', 'as s1 be', 't')],\n",
       "  [('function', 'as s1 be', 't')],\n",
       "  [('connectivity 4.07', 'be in', 'giant component'),\n",
       "   ('B SF network', 'be with', 'λ')],\n",
       "  [('tr =', 'for 13 be', 'solid line')],\n",
       "  [('function', 'as s1 be', 't')],\n",
       "  [('dynamic percolation',\n",
       "    'be with',\n",
       "    'instantaneous void transmissibility V s ≡')],\n",
       "  [('two points', 'be with', 'same image of function')],\n",
       "  [('G1', 'be', 'concave'),\n",
       "   ('two solutions', 'solve equation for', 'variable ωt above T ∗')],\n",
       "  [('S1', '=', '0')],\n",
       "  [('ΦS', '=', '1/hki')],\n",
       "  [('giant susceptible component', 'be', 'destroyed')],\n",
       "  [('dashed lines', 'show', 'possible solutions of Eq')],\n",
       "  [('connectivity 4.07', 'be in', 'giant component')],\n",
       "  [('solid lines', 'correspond to', 't'),\n",
       "   ('Eqs', 'of s1 be', 't'),\n",
       "   ('Eqs', 'of s1 be', 'blue solid line')],\n",
       "  [('V s c', 'be obtain from', 'Eq')],\n",
       "  [('1000 network realizations', 'be with', 'N =')],\n",
       "  [('susceptible individuals', 'be in', 'SIR process')],\n",
       "  [('giant susceptible cluster', 'exist at', 'given time')],\n",
       "  [('second order phase transition', 'be in', 'susceptible network'),\n",
       "   ('second order phase transition', 'be with', 'mean ﬁeld exponents at tc'),\n",
       "   ('discussion', 'be in', 'Appendix Sec.II')],\n",
       "  [('Conclusions', 'be in', 'paper')],\n",
       "  [('second order transition', 'be with', 'mean ﬁeld exponents'),\n",
       "   ('susceptible network',\n",
       "    'overcome',\n",
       "    'second order transition with mean ﬁeld exponents')],\n",
       "  [('excellent agreement', 'be with', 'simulations')],\n",
       "  [('authorities', 'preserve', 'topological features of SF networks')],\n",
       "  [('work', 'be part of', 'Pict 0293/2008'),\n",
       "   ('UNMdP', 'of ject be', 'Pict 0293/2008')],\n",
       "  [('neighbor', 'transmit disease to', 'root node')],\n",
       "  [('probability', 'be in', 'EBCM approach')],\n",
       "  [('SIR model', 'be with', 'infection')],\n",
       "  [('increase', 'be in', 'Eq'),\n",
       "   ('term', 'contribute to', 'increase of ΦI in Eq')],\n",
       "  [('second term', 'correspond to', 'term of Eq')],\n",
       "  [('we', 'obtain', 'evolution of I t')],\n",
       "  [('S5', 'connectivity of', 'susceptible individuals'),\n",
       "   ('dt =', 'evolution of', 'fraction')],\n",
       "  [('second term',\n",
       "    'correspond to',\n",
       "    'recovery of infected individuals at rate γ')],\n",
       "  [('they', 'be', 'infected')],\n",
       "  [('epidemics', 'be with', 'tr ='),\n",
       "   ('mean connectivity 4.07', 'be in', 'giant component'),\n",
       "   ('SF', 'be with', 'λ ='),\n",
       "   ('networks', 'be with', 'mean connectivity 4.07 in giant component')],\n",
       "  [('solid black curve', 'be numerical solution of', 'Eq')],\n",
       "  [('temporal shift transformation', 'without s9 be', 'solid black line')],\n",
       "  [('node void percolation', 'in percolation be', 'void node'),\n",
       "   ('time domain', 'in percolation be', 'void node'),\n",
       "   ('Node void percolation', 'be in', 'node void percolation'),\n",
       "   ('Node void percolation', 'be in', 'time domain')],\n",
       "  [('their higher degree nodes', 'ﬁrst as', 'as intentional attack')],\n",
       "  [('resulting susceptible network',\n",
       "    'be',\n",
       "    'As consequence more homogeneous than original')],\n",
       "  [('Simulation results', 'kmin at', 'diﬀerent times'),\n",
       "   ('degree distribution', 'of result be', 'kmin = at diﬀerent times'),\n",
       "   ('beginning', 'at result be', 'kmin ='),\n",
       "   ('SF network', 'be with', 'λ ='),\n",
       "   ('spreading', 'of beginning be', 'blue solid line')],\n",
       "  [('we',\n",
       "    'plot',\n",
       "    'simulations results of ﬁnite size distribution of susceptible nodes')],\n",
       "  [('tc', 'be in', 'ER network with hki = for tc = for tc =')],\n",
       "  [('10000 network realizations', 'be with', 'N ='),\n",
       "   ('Our simulations', 'be average', 'over 10000 network realizations')],\n",
       "  [('ER network', 'be with', 'hki ='),\n",
       "   ('ER network', 'in approach be', 'solid line'),\n",
       "   ('analytical approach', 'be in', 'ER network with hki =')],\n",
       "  [('Our simulations', 'be average', 'over 1000 network realizations'),\n",
       "   ('1000 network realizations', 'be with', 'N =')],\n",
       "  [('power law', 'be with', 'exponent τ ≈ 2.5')],\n",
       "  [('exponent one', 'be with', 'distance to criticality Φs'),\n",
       "   ('power law', 'be with', 'exponent one')],\n",
       "  [('two critical exponents', 'determine', 'universality class')],\n",
       "  [('other second order phase transitions',\n",
       "    'be in',\n",
       "    'physics like relaxation time near Curie temperature')],\n",
       "  [('ER network', 'be with', 'hki ='),\n",
       "   ('connectivity 4.07', 'be in', 'giant component'),\n",
       "   ('connectivity 4.07', 'be in', 'ER network with hki ='),\n",
       "   ('function', 'as tc be', 'tr ='),\n",
       "   ('function', 'as tc be', 'tr =')],\n",
       "  [('dashed line', 'represent', 'value of T ∗')],\n",
       "  [('dotted lines', 'be use as', 'guide to eyes')],\n",
       "  [('NumericCitation R. M. Anderson', 'disease of', 'Humans')],\n",
       "  [('E. M. Volz', 'journal of', 'Royal Society Interface 9 890')],\n",
       "  [('NumericCitation', 'be with', '2 < λ')],\n",
       "  [('stubs', 'be in', 'random edge')],\n",
       "  [('NumericCitation D. Stauﬀer', 'introduction to', 'percolation theory')]],\n",
       " 52: [[('∗ Ming Tang ,1 3', 'university of', 'Technology'),\n",
       "   ('∗ Ming Tang ,1 3', 'school of', 'Sciences'),\n",
       "   ('∗ Ming Tang ,1 3', 'school of', 'Physics'),\n",
       "   ('∗ Ming Tang ,1 3', 'university of', 'Electronic Science'),\n",
       "   ('Physics', 'ﬁcation of', 'effective epidemic threshold'),\n",
       "   ('Physics', 'university of', 'Fribourg'),\n",
       "   ('Optoelectronic Engineering', '5department of', 'Physics'),\n",
       "   ('Physics', 'ee', 'Chemin du Mus')],\n",
       "  [('extensive numerical experiments',\n",
       "    'validate',\n",
       "    'proposed theoretical predictions')],\n",
       "  [('Bu t', 'be with', 'synchronous updating'),\n",
       "   ('ﬁnal outbreak size', 'increase with', 'recovery rate')],\n",
       "  [('phase transition', 'be with', 'arbitrary recovery rate'),\n",
       "   ('us', 'understand', 'phase transition with arbitrary recovery rate')],\n",
       "  [('effective epidemic threshold', 'attract', 'increasing attentions')],\n",
       "  [('it',\n",
       "    'predict',\n",
       "    'effective epidemic thresholds with different recovery rates'),\n",
       "   ('effective epidemic thresholds', 'be with', 'different recovery rates')],\n",
       "  [('we', 'verify', 'accuracy of theoretical predictions')],\n",
       "  [('we', 'ﬁnd', 'same variation trend of effective epidemic threshold'),\n",
       "   ('epidemic outbreaks',\n",
       "    'be with',\n",
       "    'synchronous updating on scale-free networks')],\n",
       "  [('recovered nodes',\n",
       "    'ﬁnal fraction of be',\n",
       "    'above which ﬁnite NumericCitation')],\n",
       "  [('networks', 'be with', 'power-law scaling P')],\n",
       "  [('It', 'be', 'worth')],\n",
       "  [('diseases', 'be with', 'different infection durations')],\n",
       "  [('difference', 'be in', 'effective epidemic threshold'),\n",
       "   ('cases', 'be with', 'µ =')],\n",
       "  [('synchronous', 'be', 'two famous numerical methods'),\n",
       "   ('synchronous',\n",
       "    'be',\n",
       "    'two famous numerical methods for dynamics NumericCitation')],\n",
       "  [('SIR model', 'be with', 'arbitrary recovery rate')],\n",
       "  [('predictions', 'be', 'good'),\n",
       "   ('good agreement',\n",
       "    'be with',\n",
       "    'numerical effective epidemic thresholds identiﬁe by variability measure'),\n",
       "   ('good agreement', 'be in', 'most cases')],\n",
       "  [('networks', 'be with', 'disassortative mixing'),\n",
       "   ('theoretical predictions',\n",
       "    'be in',\n",
       "    'networks with disassortative mixing')],\n",
       "  [('node', 'be with', 'degree k')],\n",
       "  [('we', 'solve', 't'), ('we', 'solve', 'S'), ('we', 'know', 't')],\n",
       "  [('According de ﬁnition', 'above ξr be', 't')],\n",
       "  [('t', 'be with', 'probability θ')],\n",
       "  [('uncorrelated networks', 'for p be', 'k')],\n",
       "  [('we', 'sum over', 'possible k ′')],\n",
       "  [('we', 'take into', 'consideration')],\n",
       "  [('De', 'ﬁne', 'right hand of Eq')],\n",
       "  [('15 It', 'can', 'can seen'), ('15 It', 'can', 'can seen from Eq')],\n",
       "  [('effective epidemic threshold',\n",
       "    'also decrease with',\n",
       "    'recovery rate µ in synchronous updating process')],\n",
       "  [('µ', '=', '1'), ('it', 'approach', 'Eq')],\n",
       "  [('case', 'for 9 be', 'combining Eqs')],\n",
       "  [('we', 'can rewrite', 'Eq'), ('expression', 'be in', 'terms of θ')],\n",
       "  [('Eq', 'of ∗ be', '∞')],\n",
       "  [('MAIN RESULTS', 'be in', 'SIR model')],\n",
       "  [('other nodes', 'be', 'susceptible')],\n",
       "  [('ni', 'be', 'where number of its infected neighbors')],\n",
       "  [('recovered nodes', 'acquire', 'permanent immunity')],\n",
       "  [('we',\n",
       "    'numerically identify',\n",
       "    'effective epidemic threshold of SIR model')],\n",
       "  [('we',\n",
       "    'estimate',\n",
       "    'numerical effective epidemic threshold λnum from position of peak of variability')],\n",
       "  [('Fig. 1', 'show with', 'asynchronous updating')],\n",
       "  [('ﬁnal outbreak size', 'be', 'n ot affected by recovery rate')],\n",
       "  [('developed theory', 'can describe very well', 'phenomena')],\n",
       "  [('Overview', 'be with', 'different recovery rates')],\n",
       "  [('µ = 0.6 1.0', 'be with', 'asynchronous updating'),\n",
       "   ('µ =',\n",
       "    'for r be',\n",
       "    'µ = 0.6 1.0 down triangles with asynchronous updating'),\n",
       "   ('Final outbreak size R', 'µ with', 'asynchronous updating'),\n",
       "   ('λ', 'vs. r be', 'µ = 0.6 1.0 down triangles with asynchronous updating'),\n",
       "   ('Final outbreak size R', 'µ down', 'triangles')],\n",
       "  [('synchronous updating', 'with 0.6 be', 'circles')],\n",
       "  [('vertical lines',\n",
       "    'point out',\n",
       "    'positions of numerical effective epidemic thresholds')],\n",
       "  [('results',\n",
       "    'be average',\n",
       "    'over 102 × 104 independent realizations on 102 different networks')],\n",
       "  [('k', '=', '6')],\n",
       "  [('SIR epidemic dynamics', 'be with', 'synchronous updating'),\n",
       "   ('recovery rate',\n",
       "    'have',\n",
       "    'remarkable effect on SIR epidemic dynamics with synchronous updating')],\n",
       "  [('synchronous updating method', 'be', 'when used')],\n",
       "  [('Color online', 'be in', 'spreading dynamics with synchronous updating'),\n",
       "   ('spreading dynamics', 'be with', 'synchronous updating')],\n",
       "  [('one edge', 'through t be', 'λ =')],\n",
       "  [('k', '=', '6')],\n",
       "  [('results',\n",
       "    'be average',\n",
       "    'over 102 × 104 independent realizations on 102 different networks')],\n",
       "  [('synchronous updating method', 'be in', 'Fig. 2')],\n",
       "  [('µ', 'be', 'small'), ('µ', 'be', 'when small')],\n",
       "  [('T increases', 'be with', 'recovery rate µ')],\n",
       "  [('effect',\n",
       "    'lead to',\n",
       "    'decrease of effective epidemic threshold with recovery probability µ'),\n",
       "   ('decrease', 'be with', 'recovery probability µ')],\n",
       "  [('epidemic', 'outbreak for', 'just large λ'),\n",
       "   ('recovery probability', 'be', 'when large')],\n",
       "  [('scale-free networks', 'be with', 'power-law degree distribution P'),\n",
       "   ('SIR epidemic dynamics',\n",
       "    'be with',\n",
       "    'arbitrary recovery rate on scale-free networks')],\n",
       "  [('We', 'build', 'SFNs')],\n",
       "  [('SFNs', 'be with', 'γ =')],\n",
       "  [('µ', 'be in', 'such networks'),\n",
       "   ('weak degree heterogeneity', 'be with', 'γ =')],\n",
       "  [('such networks', 'be with', 'ﬁnite size NumericCitation'),\n",
       "   ('SFNs', 'be with', 'γ ='),\n",
       "   ('SFNs', 'be with', 'γ =')],\n",
       "  [('cases', 'be with', 'synchronous updating')],\n",
       "  [('Color online', 'be in', 'spreading dynamics with synchronous updating'),\n",
       "   ('SFNs', 'be with', 'degree exponents γ ='),\n",
       "   ('spreading dynamics', 'be with', 'synchronous updating')],\n",
       "  [('We', 'perform', '102 × 104 independent realizations')],\n",
       "  [('r', 'be', 'Pearson correlation coefﬁcient'),\n",
       "   ('c', 'be', 'clustering coefﬁcient'),\n",
       "   ('hki', 'be', 'average degree'),\n",
       "   ('kmax', 'be', 'maximum degree')],\n",
       "  [('difference',\n",
       "    'be in',\n",
       "    'variation of effective epidemic threshold for different networks')],\n",
       "  [('CONCLUSION', 'be in', 'paper'),\n",
       "   ('SIR model', 'be with', 'arbitrary recovery rate')],\n",
       "  [('Two basic updating methods', 'be', 'considered')],\n",
       "  [('Color online',\n",
       "    'λc as',\n",
       "    'function of recovery rate µ on real-world networks'),\n",
       "   ('spreading dynamics',\n",
       "    'in online be',\n",
       "    'effective epidemic threshold λc as function of recovery rate µ on real-world networks'),\n",
       "   ('Color online', 'be in', 'spreading dynamics with synchronous updating'),\n",
       "   ('spreading dynamics', 'be with', 'synchronous updating')],\n",
       "  [('value', 'be show by', 'right scale mark'),\n",
       "   ('value', 'be show by', 'left scale mark')],\n",
       "  [('RRNs', 'be with', 'constant degree'),\n",
       "   ('we', 'verify', 'theoretical predictions')],\n",
       "  [('obvious difference', 'be in', 'ﬁnal outbreak size')],\n",
       "  [('we', 'explore', 'university of conclusions')],\n",
       "  [('networks', 'be with', 'disassortative mixing patterns')],\n",
       "  [('effective SIR epidemic threshold', 'be with', 'constant value µ =')],\n",
       "  [('A. Vespignani', 'processes on', 'Complex Networks')],\n",
       "  [('Infections Diseases', 'be in', 'Humans'),\n",
       "   ('Humans', 'in diseases be', 'Citation'),\n",
       "   ('NumericCitation R. M. Anderson', 'diseases in', 'Humans')],\n",
       "  [('NumericCitation M. Barth', 'elemy be', 'Phys')],\n",
       "  [('Barab', 'asi be', 'Rev. Mod')],\n",
       "  [('7 updating', 'for clive be', 'Primary HIV/AIDS practical guide care')],\n",
       "  [('SIR dynamics', 'be with', 'arbitrary recovery rate'),\n",
       "   ('SIR dynamics',\n",
       "    'for threshold be',\n",
       "    'message-passing approach NumericCitation')]],\n",
       " 53: [[('outbreak', 'be in', 'animal epidemics Eric Dumonteil ∗')],\n",
       "  [('complex network interactions', 'can', 'when can safely neglected')],\n",
       "  [('We', 'compare', 'them')],\n",
       "  [('populations', 'of class be', 'the infected')],\n",
       "  [('it', 'transmit disease to', 'susceptible NumericCitation')],\n",
       "  [('one', 'can just monitor', 'evolution of infected population alone')],\n",
       "  [('spatial eﬀects', 'must', 'must necessarily considered')],\n",
       "  [('propagation', 'be in', 'animals')],\n",
       "  [('population size', 'ﬂuctuates with', 'time')],\n",
       "  [('One ma jor challenge',\n",
       "    'be in',\n",
       "    'very practical ﬁeld of disease control')],\n",
       "  [('issue',\n",
       "    'have',\n",
       "    'For animal epidemics has investigated experimentally for instance in case of equine inﬂuenza NumericCitation'),\n",
       "   ('instance', 'be in', 'case of equine inﬂuenza NumericCitation')],\n",
       "  [('infections', 'have spread up to', 'time t')],\n",
       "  [('properties', 'be', 'formidable'),\n",
       "   ('hull', 'be', 'complex'),\n",
       "   ('statistical properties', 'be', 'formidable problem')],\n",
       "  [('particle', 'be with', 'rate γ')],\n",
       "  [('local contact', 'individual from', 'background with probability b dt'),\n",
       "   ('background', 'be with', 'probability b dt')],\n",
       "  [('rate βN', 'be in', 'SIR')],\n",
       "  [('diﬀuses', 'be with', 'diﬀusion constant D'),\n",
       "   ('diﬀuses', 'be with', 'probability 1 of γ')],\n",
       "  [('zero mean', 'with ηx be', 't'),\n",
       "   ('independent Gaussian white noises ηx =', 'be with', 'zero mean')],\n",
       "  [('only dimensionless parameter', 'be in', 'model')],\n",
       "  [('ﬁnal shape', 'be', 'precisely convex hull associated to set')],\n",
       "  [('convex hull', 'of a be', 't')],\n",
       "  [('2 prediction', 'seem', 'rather paradoxical at ﬁrst glance')],\n",
       "  [('its area', 'be', 'divergent')],\n",
       "  [('results', 'be in', 'Eqs')],\n",
       "  [('A', 'divergent as', 't → ∞')],\n",
       "  [('R0', '=', '1')],\n",
       "  [('epidemic', 'ex-in', 'tinction')],\n",
       "  [('formidable task', 'be in', 'even absence branching'),\n",
       "   ('statistics', 'be', 'formidable'),\n",
       "   ('diﬀusion process', 'be in', 'two dimensions')],\n",
       "  [('Cauchy', 'of', 'integral geometric formulae')],\n",
       "  [('Cauchy', 'have', 'formulae')],\n",
       "  [('xm dimensional stochastic process', 'be in', 'x direction'),\n",
       "   ('isotropy', 'be in', 'space')],\n",
       "  [('path', 'be', 'marked'), ('path', 'be mark in', 'red')],\n",
       "  [('tm', 'give component be', 'clearly y2 = 2D tm'),\n",
       "   ('tra jectory', 'of component be', 'clearly y2 = 2D tm')],\n",
       "  [('red path', 'be', 'Therefore regular one dimensional Brownian motion'),\n",
       "   ('path', 'be', 'Brownian'),\n",
       "   ('path', 'be', 'regular'),\n",
       "   ('path', 'be', 'dimensional')],\n",
       "  [('area',\n",
       "    'be',\n",
       "    'related to extreme statistics of one dimensional branching Brownian motion with death'),\n",
       "   ('one dimensional branching Brownian motion', 'be with', 'death')],\n",
       "  [('this', 'be', 'performed exactly')],\n",
       "  [('its', 'x-displacement', 'xm ≥ 0')],\n",
       "  [('ηx dt', 'be with', 'probability 1 − γ'),\n",
       "   ('subsequent interval', 'during walker be', '0 dt'),\n",
       "   ('three exclusive stochastic moves', 'be in', '∞ 0 nonnegative')],\n",
       "  [('∆ x', 'be in', 'last case')],\n",
       "  [('third term', 'correspond to', 'diﬀusion')],\n",
       "  [('Θ', 'be', 'where Heaviside step function'),\n",
       "   ('ﬁrst order', 'be in', 'dt'),\n",
       "   ('dt', 'be in', '∆ x')],\n",
       "  [('average area', 'be in', 'Eq')],\n",
       "  [('we', 'evaluate', 't')],\n",
       "  [('we', 'determine', 'tm'),\n",
       "   ('random variable tm', 'of pt be', 'tm'),\n",
       "   ('we', 'also compute', 'tm')],\n",
       "  [('interval', 'be with', 'probability qt')],\n",
       "  [('initial condition T0 =', 'with r0 be', '14'),\n",
       "   ('R0', 'be with', 'initial condition T0 =')],\n",
       "  [('stationary solution',\n",
       "    'be',\n",
       "    'In fact already known in context of genetic propagation of mutant allele NumericCitation')],\n",
       "  [('average area', 'be in', 'Eq')],\n",
       "  [('power-law tail', 'due to m be', 't')],\n",
       "  [('decaying function', 'be with', 'f'), ('f', 'be', 'decaying')],\n",
       "  [('we', 'get to', 'leading order for large t')],\n",
       "  [('characterization', 'be in', 'Eq')],\n",
       "  [('tm', 'determine in', 'Supplementary Material')],\n",
       "  [('we', 'determine', '19'),\n",
       "   ('logarithmic divergence', 'be in', 'time'),\n",
       "   ('large t', 'for t be', '19')],\n",
       "  [('NumericCitation', 'be in', 'Eq')],\n",
       "  [('NumericCitation', 'give', 'result announced in Eq')],\n",
       "  [('Cauchy', 'of', 'formula')],\n",
       "  [('stationary distribution', 'be with', 'power-law tail'),\n",
       "   ('A', '− for', 'm')],\n",
       "  [('Our scaling predictions', 'be in', 'agreement'),\n",
       "   ('agreement', 'be with', 'our Monte Carlo simulations')],\n",
       "  [('tW =', 'be in', 'Eq')],\n",
       "  [('function', 'as t be', 'W')],\n",
       "  [('front advances', 'be with', 'constant velocity')],\n",
       "  [('large t.',\n",
       "    'at o be',\n",
       "    't one can approximate W Θ due sharpness of front to leading order for large near front'),\n",
       "   ('large t.', 'at o be', '1')],\n",
       "  [('former', 'give from', 'Eq')],\n",
       "  [('result', 'announce in', 'Eq')],\n",
       "  [('Conclusions', 'be in', 'paper')],\n",
       "  [('one',\n",
       "    'describe',\n",
       "    'spatial extent of epidemic outbreak in relatively realistic situations'),\n",
       "   ('epidemic outbreak', 'be in', 'relatively realistic situations')],\n",
       "  [('many assumptions', 'be in', 'epidemic model')],\n",
       "  [('one', 'resort to', 'several distinct convex hulls'),\n",
       "   ('one', 'conveniently resort to', 'several distinct convex hulls')],\n",
       "  [('realistic models', 'resort to', 'cumbersome Monte Carlo simulations'),\n",
       "   ('realistic models', 'could', 'More generally could taken into account'),\n",
       "   ('models', 'resort to', 'cumbersome Monte Carlo simulations')],\n",
       "  [('underlying process', 'be', 'rotationally invariant')],\n",
       "  [('We', 'conclude with', 'additional remark')],\n",
       "  [('Supplementary Materials Cauchy', 'have', 'formula')],\n",
       "  [('convex hull', 'be with', '0 ≤ τ ≤ t')],\n",
       "  [('origin', 'take', 'given direction θ')],\n",
       "  [('support function', 'be in', 'direction θ'), ('stick', 'of m be', 'θ')],\n",
       "  [('one', 'come from', 'inﬁnity')],\n",
       "  [('C', 'be', 'When convex hull of at time t'), ('its', 'm', 'θ')],\n",
       "  [('time', 'at x be', 'tm'), ('its', 'x', 'tm')],\n",
       "  [('Cauchy', 'over', 'formulas'), ('average', 'use', 'isotropy')],\n",
       "  [('branching Brownian motion', 'be with', 'death')],\n",
       "  [('Time', 'set', 't')],\n",
       "  [('We', 'thus have', 'i')],\n",
       "  [('We', 'have veriﬁed', 'that')],\n",
       "  [('walker', 'be', 'removed')],\n",
       "  [('corresponding convex hull',\n",
       "    'resort to',\n",
       "    'algorithm proposed in NumericCitation')],\n",
       "  [('Pt t =', 'be in', 'Eq')],\n",
       "  [('latter condition', 'be obtain by', 'Laplace transxm > 0')],\n",
       "  [('∼ ez', 'iﬁed', 'Bessel functions'), ('ez', 'iﬁed', 'Bessel functions')],\n",
       "  [('we', 'hence be leave with', 'z')],\n",
       "  [('we', 'determine', 'Laplace transe −'),\n",
       "   ('we', 'revert to', 'variable xm')],\n",
       "  [('it', 'obtain', 'p ∞'),\n",
       "   ('we', 'once know', '˜p ∞ s'),\n",
       "   ('it', 'obtain', 'tm')],\n",
       "  [('xm', 'over behavior be', '31'), ('s', '→', '0 of ˜p ∞ s')],\n",
       "  [('its', 'application', 'Griﬃn')],\n",
       "  [('Bartlett M S Stochastic Population Models', 'be in', 'Ecology')],\n",
       "  [('their Statistical Analysis Lecture Notes', 'be in', 'Statistics')],\n",
       "  [('epidemics', 'be with', 'multiple stages'),\n",
       "   ('2012 Outbreak size distributions',\n",
       "    'be in',\n",
       "    'epidemics with multiple stages')],\n",
       "  [('role', 'be in', 'emergence of infectious diseases')],\n",
       "  [('Kendall D G Deterministic epidemics', 'be in', 'closed populations')],\n",
       "  [('4:149-165', 'consider as', 'subtype of Brownian branching random ﬁeld')],\n",
       "  [('J', 'hull of', 'N planar Brownian 12')],\n",
       "  [('Riley S', 'model of', 'Infectious Disease')],\n",
       "  [('Fraser C', 'potential of', 'Strain')],\n",
       "  [('role', 'be in', 'prediction of global epidemics'),\n",
       "   ('Colizza V', 'm', 'Barth')],\n",
       "  [('L', 'theory of', 'home range estimation'),\n",
       "   ('L', 'theory from', 'displacement measurements of animal populations')],\n",
       "  [('Brunet E', 'statistics at', 'tip of branching random walk')],\n",
       "  [('t', '=', '0')],\n",
       "  [('t', 'be', 'displayed'), ('t', 'be display as', 'red dots')],\n",
       "  [('function', 'as a be', 't'), ('convex hull', 'of a be', 't')],\n",
       "  [('We', 'consider', 'ﬁve diﬀerent values of R0')],\n",
       "  [('results', 'be display as', 'solid lines')],\n",
       "  [('death', 'be with', 'same parameters'),\n",
       "   ('Monte Carlo simulations', 'be with', 'death')],\n",
       "  [('Monte Carlo', 'be display as', 'symbols'),\n",
       "   ('Monte Carlo', 'be', 'displayed')],\n",
       "  [('dashed lines', 'give in', 'Eq')],\n",
       "  [('Distribution', 'be with', 'γ ='),\n",
       "   ('Monte Carlo simulations', 'be with', '2 · 106 realizations')],\n",
       "  [('dashed line', 'predict by', 'Eq NumericCitation')],\n",
       "  [('time behavior',\n",
       "    'be in',\n",
       "    'supercritical regime for diﬀerent values of R0 >')],\n",
       "  [('lines', 'represent', 'limit'), ('Dashed lines', 'represent', 'limit')],\n",
       "  [('asymptotic scaling', 'be in', 'as Eq')],\n",
       "  [('1 − R −', 'be in', 'as Eq')],\n",
       "  [('branching random walk', 'compose of', 'ﬁve individuals')],\n",
       "  [('t', 'start', 'blue line'), ('t', 'start', 'diﬀusing')],\n",
       "  [('t', 'be', 'at given time displayed as red dots')],\n",
       "  [('component', 'be', 'Brownian'), ('component', 'be', 'regular')],\n",
       "  [('Cauchy', 'have', 'construction of M representing')],\n",
       "  [('6', 'of l be', 't')],\n",
       "  [('We', 'consider', 'ﬁve diﬀerent values of R0')],\n",
       "  [('NumericCitation', 'be with', 'choices dt =')],\n",
       "  [('results', 'be display as', 'solid lines')],\n",
       "  [('Monte Carlo simulations =', 'be with', 'results')],\n",
       "  [('Monte Brownian motion', 'be with', 'death'),\n",
       "   ('death', 'be with', 'same parameters'),\n",
       "   ('Monte Brownian motion',\n",
       "    'be with',\n",
       "    'choice of convex hull as function of chosen D =')],\n",
       "  [('dashed lines', 'give in', 'Eq')],\n",
       "  [('realizations', 'number of be', '2 · 106')],\n",
       "  [('time behavior',\n",
       "    'be in',\n",
       "    'supercritical regime for diﬀerent values of R0 >')],\n",
       "  [('red curve', 'correspond to', 'critical regime')]],\n",
       " 54: [[('heterogeneous networks', 'be with', 'identical infectivity Rui Yang'),\n",
       "   ('paper', 'in center be', 'University of Science'),\n",
       "   ('Nonlinear Science Center', 'university of', 'Science'),\n",
       "   ('paper', 'in center be', 'PR China'),\n",
       "   ('Nonlinear Science Center', 'be in', 'paper'),\n",
       "   ('paper', 'in center be', 'Hefei 230026')],\n",
       "  [('recovered individuals', 'be in', 'present model')],\n",
       "  [('threshold value', 'be independent of', 'topology of underlying network')],\n",
       "  [('simulations', 'agree well with', 'analytic results')],\n",
       "  [('hierarchical dynamics', 'be with', 'three plateaus')],\n",
       "  [('present results',\n",
       "    'be of',\n",
       "    'could practical importance in setup of dynamic control strategies'),\n",
       "   ('could practical importance',\n",
       "    'be in',\n",
       "    'setup of dynamic control strategies')],\n",
       "  [('Xt I. INTRODUCTION Many real-world systems',\n",
       "    'range from',\n",
       "    'nature to society'),\n",
       "   ('Xt I. INTRODUCTION Many real-world systems',\n",
       "    'can',\n",
       "    'can described by complex networks')],\n",
       "  [('power-law degree distributions',\n",
       "    'be',\n",
       "    'Recently have observed in various networks NumericCitation'),\n",
       "   ('power-law degree distributions',\n",
       "    'be',\n",
       "    'have observed in various networks NumericCitation')],\n",
       "  [('node',\n",
       "    'have',\n",
       "    'potential infection-activity measured by possibly maximal contribution to propagation process within one time step')],\n",
       "  [('nodes', 'be with', 'large degree')],\n",
       "  [('cases', 'be in', 'real world')],\n",
       "  [],\n",
       "  [('their clients', 'invite', 'others')],\n",
       "  [('Similar phenomena', 'be', 'common'),\n",
       "   ('Similar phenomena', 'be common in', 'our daily lives')],\n",
       "  [('direction', 'be in', 'real lives')],\n",
       "  [('γ', '=', '1'), ('we', 'lose', 'generality')],\n",
       "  [('function', 'as r be', '∞'),\n",
       "   ('function', 'be with', 'hki ='),\n",
       "   ('function', 'as r be', 'N =')],\n",
       "  [('analytic results', 'agree well with', 'simulations')],\n",
       "  [('x', 'of', 'infected neighbors')],\n",
       "  [('dynamical process', 'randomly select', 'one infected node')],\n",
       "  [('approximation', 'be in', 'case')],\n",
       "  [('we', 'obtain from', 'Eq')],\n",
       "  [('epidemic spreading', 'be in', 'SF networks')],\n",
       "  [('simulation', 'be in', 'Fig. 1')],\n",
       "  [('identical infectivity A',\n",
       "    'of larger be',\n",
       "    'higher for same λ from Fig. 1')],\n",
       "  [('we', 'further demonstrate', 'proposition'),\n",
       "   ('simulation', 'result on', 'diﬀerent networks')],\n",
       "  [('BA networks', 'be with', 'diﬀerent sizes'),\n",
       "   ('BA networks', 'be with', 'diﬀerent average degrees')],\n",
       "  [('function', 'as r be', '∞'),\n",
       "   ('BA networks', 'be with', 'diﬀerent size N'),\n",
       "   ('function', 'as r be', 'b'),\n",
       "   ('function', 'as r be', 'BA networks for diﬀerent attachment number m'),\n",
       "   ('color online R', 'network for', 'diﬀerent attachment number m'),\n",
       "   ('function', 'be with', 'hki =')],\n",
       "  [('recovered nodes', 'be in', 'BA networks of size N'),\n",
       "   ('BA networks', 'in node be', 'a')],\n",
       "  [('convenient quantities', 'be in', 'numerical spreading experiments'),\n",
       "   ('numerical spreading experiments', 'be in', 'BA networks')],\n",
       "  [('hierarchical dynamics', 'be with', 'three plateaus')],\n",
       "  [('infection', 'take ﬁrstly', 'control of large degree vertices')],\n",
       "  [('we', 'indicate', 'detailed information on infection propagation')],\n",
       "  [('infection', 'also infer from', 'Fig. 3')],\n",
       "  [('node', 'on', 'degree')],\n",
       "  [('HIV', 'be in', 'Africa NumericCitation such druggers'),\n",
       "   ('druggers', 'such as numericcitation be', 'high-risk population')]],\n",
       " 55: [[('Information spreading', 'be in', 'population')],\n",
       "  [('Pontryagin', 'have', 'Maximum Principle')],\n",
       "  [('changing interest level', 'be in', 'campaign'),\n",
       "   ('time varying information spreading rate', 'be in', 'our formulation')],\n",
       "  [('Information Epidemics', \"principle 's\", 'Pontryagin'),\n",
       "   ('Pontryagin', 'have', 'Maximum principle')],\n",
       "  [('opportunity', 'be in', 'to ﬂuence')],\n",
       "  [('phenomenon', 'be call', 'infor mation epidemics'),\n",
       "   ('pathogens', 'be in', 'human network')],\n",
       "  [('we', 'address', 'problems described above')],\n",
       "  [('We', 'assume', 'homogeneously mixed population')],\n",
       "  [('Individuals', 'exchange', 'messages'),\n",
       "   ('Individuals', 'give', 'rise'),\n",
       "   ('Individuals', 'give rise to', 'information epidemics')],\n",
       "  [('similarities', 'be in', 'ways disease spreads'),\n",
       "   ('ways disease spreads', 'be in', 'biological network')],\n",
       "  [('way', 'be in', 'which'),\n",
       "   ('communicable disease spreads', 'be in', 'population')],\n",
       "  [('spread', 'be in', 'human networks')],\n",
       "  [('diﬀerences', 'be in', 'epidemic models')],\n",
       "  [('time-varying state variable', 'be in', 'cost function')],\n",
       "  [('work', 'be in', 'NumericCitation')],\n",
       "  [('themselves', 'increase', 'removal rate of infected individuals')],\n",
       "  [('their', 'neighbor', 'Pull')],\n",
       "  [('authors', 'be in', 'NumericCitation aim compute')],\n",
       "  [('information', 'be in', 'population'),\n",
       "   ('study', 'be in', 'NumericCitation de')],\n",
       "  [('spread', 'be in', 'Daley-Kendall models')],\n",
       "  [('system', 'be control over', 'whole campaign period')],\n",
       "  [('Authors', 'be in', 'NumericCitation')],\n",
       "  [('case', 'be in', 'paper')],\n",
       "  [('fraction', 'be in', 'addition')],\n",
       "  [('our model', 'assume', 'quadratic cost')],\n",
       "  [('objective functionals', 'be in', 'our models')],\n",
       "  [('Previous studies',\n",
       "    'have ignore',\n",
       "    'characteristic of information epidemics')],\n",
       "  [('information spread', 'be in', 'two diﬀerent models')],\n",
       "  [('spreads', 'be in', 'n ext')],\n",
       "  [('control signal', 'be in', 'SIS model')],\n",
       "  [('solution', 'be in', 'problems')],\n",
       "  [('campaign deadline', 'be', 'when suﬃciently small')],\n",
       "  [('optimal control strategy', 'be with', 'constant control strategy')],\n",
       "  [],\n",
       "  [('contact', 'be with', 'infected individual')],\n",
       "  [('buzz', 'be in', 'population')],\n",
       "  [('it', 'receive', 'diﬀerent message about same topic')],\n",
       "  [('t.', 'let', 't')],\n",
       "  [('contact', 'be with', 'single infected node')],\n",
       "  [('contact', 'be with', 'average of km others')],\n",
       "  [('population', 'of s be', 't'),\n",
       "   ('rate', 'be in', 'population due to susceptible-infected contact')],\n",
       "  [('evolution', 'be in', 'uncontrolled SIS process')],\n",
       "  [('objective functional', 'be choose', 'T')],\n",
       "  [('tment control', 'be with', '0 De ﬁnition 1')],\n",
       "  [('nodes', 'act as', 'spreaders')],\n",
       "  [('enthusiastic individuals', 'promote', 'product')],\n",
       "  [('he SIR process', 'be in', 'limit of large N')],\n",
       "  [('individuals', 'act as', 'spreaders via advertisements in mass media'),\n",
       "   ('advertisements', 'be in', 'mass media')],\n",
       "  [('u', 'be', 'Lebesgue integrable')],\n",
       "  [('control signal', 'take value in', '0 uma x')],\n",
       "  [('cost', 'be in', 'cost functional')],\n",
       "  [('We', 'solve', '4'),\n",
       "   ('Pontryagin', 'have', 'Maximum principle NumericCitation')],\n",
       "  [('λ ∗', '=', '1')],\n",
       "  [('shooting method NumericCitation', 'be in', 'MATLAB'),\n",
       "   ('We', 'solve', 'boundary value problem')],\n",
       "  [('7', 'use', \"MATLAB 's value problem\"),\n",
       "   ('7', 'use', \"MATLAB 's initial value problem\"),\n",
       "   ('MATLAB', 'have', 'initial value problem')],\n",
       "  [('T', 'control to', 'problem')],\n",
       "  [('details', 'be in', 'Appendix A. 6 5')],\n",
       "  [('problem', 'in u1 be', 'u2'), ('u1', 'be in', 'problem')],\n",
       "  [('details', 'be', 'omitted')],\n",
       "  [('Pontryagin', 'have', 'Maximum Principle NumericCitation'),\n",
       "   ('we', 'get', 'following equations')],\n",
       "  [('Hamiltonian maximizing condition', 'u', 't'),\n",
       "   ('maximization 1', 'to problem be', 'u ∗ r ∗ H =')],\n",
       "  [('16', 'from ∗ be', '14'),\n",
       "   ('16', 'from ∗ be', '12'),\n",
       "   ('16', 'from ∗ be', 't')],\n",
       "  [('shapes', 'be in', 'Sections')],\n",
       "  [('various parameters', 'be in', 'SIS models'),\n",
       "   ('plots', 'be in', 'Section 6.3 show variation of cost functional J')],\n",
       "  [('Uniqueness Solution', 'be with', 'Direct Recruitment')],\n",
       "  [('T', 'solution to', 'optimal control problem')],\n",
       "  [('details', 'be in', 'Appendix B. 6')],\n",
       "  [('We', 'divide section into', 'three parts')],\n",
       "  [('tree', 'be in', 'Fig. 1'),\n",
       "   ('results', 'be', 'how organized in two subsections')],\n",
       "  [('o', 'be with', 'control R 0 ='), ('control R 0 =', 'with i be', 't')],\n",
       "  [('control s', 'with i be', 't'),\n",
       "   ('t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s',\n",
       "    'be with',\n",
       "    'control i'),\n",
       "   ('control i', 'with s be', 't')],\n",
       "  [('Optimal control', 'evolution without', 'control for SIS epidemic'),\n",
       "   ('SIR epidemic model', 'be in', 'Section 3'),\n",
       "   ('Optimal control', 'evolution with', 'control'),\n",
       "   ('model', 'be in', 'Section 2'),\n",
       "   ('state evolutions', 'be with', 'control')],\n",
       "  [('eﬀective spreading rate', 'be', 'when constant over time')],\n",
       "  [('Optimal control', 'evolution without', 'control for SIS epidemic'),\n",
       "   ('SIR epidemic model', 'be in', 'Section 3'),\n",
       "   ('Optimal control', 'evolution with', 'control'),\n",
       "   ('control R 0 =', 'with i be', 't'),\n",
       "   ('model', 'be in', 'Section 2'),\n",
       "   ('epidemic', 'for number be', 'R0'),\n",
       "   ('state evolution', 'be with', 'control')],\n",
       "  [('infected node', 'be in', 'early stages of epidemic outbreak')],\n",
       "  [('epidemic', 'die out with', 'probability 1 NumericCitation')],\n",
       "  [('population', 'be in', 'infected state')],\n",
       "  [('word-of-mouth control', 'be in', 'Fig. 2b')],\n",
       "  [('additional observation', 'be with', 'reference to control signals'),\n",
       "   ('R0 =', 'be in', 'Fig. 2a')],\n",
       "  [('newbie', 'be in', 'market')],\n",
       "  [('equations', 'in ﬁned be', '19')],\n",
       "  [('control i', 'with i be', 't')],\n",
       "  [('control s', 'with i be', 't'),\n",
       "   ('t i n o 1 4 5 0.05 0.01 0.02 0.03 0.04 c n o t r o l s',\n",
       "    'be with',\n",
       "    'control i'),\n",
       "   ('control i', 'with s be', 't')],\n",
       "  [('Optimal control', 'evolution without', 'control for SIS epidemic'),\n",
       "   ('time-varying spreading rate β1',\n",
       "    'for control be',\n",
       "    'state evolution with control without control'),\n",
       "   ('Optimal control', 'evolution with', 'control'),\n",
       "   ('state evolution', 'be with', 'control')],\n",
       "  [('changes', 'be in', 'demand of product/service')],\n",
       "  [('control i', 'with i be', 't')],\n",
       "  [('control s', 'with i be', 't'),\n",
       "   ('t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s',\n",
       "    'be with',\n",
       "    'control i'),\n",
       "   ('control i', 'with s be', 't')],\n",
       "  [('Optimal control',\n",
       "    'evolution without',\n",
       "    'control for SIS epidemics for time-varying spreading rate β2'),\n",
       "   ('Optimal control', 'evolution with', 'control'),\n",
       "   ('state evolution', 'be with', 'control')],\n",
       "  [('state variables', 'be plot with', 'respect control signals'),\n",
       "   ('state variables', 'be', 'plotted')],\n",
       "  [('control i', 'with i be', 't')],\n",
       "  [('control s', 'with i be', 't'),\n",
       "   ('t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s',\n",
       "    'be with',\n",
       "    'control i'),\n",
       "   ('control i', 'with s be', 't')],\n",
       "  [('Optimal control',\n",
       "    'evolution without',\n",
       "    'control for SIS epidemics for time-varying spreading rate β3'),\n",
       "   ('Optimal control', 'evolution with', 'control'),\n",
       "   ('state evolution', 'be with', 'control')],\n",
       "  [('state variables', 'be plot with', 'respect control signals'),\n",
       "   ('state variables', 'be', 'plotted')],\n",
       "  [('Equations', 'in ﬁned be', '19')],\n",
       "  [('eﬀective spreading rate β', 'be', 'when constant over time')],\n",
       "  [('number', 'be in', 'SIS model'),\n",
       "   ('recovered nodes', 'be in', 'SIR model')],\n",
       "  [('bene ﬁcial', 'be in', 'real world scenarios')],\n",
       "  [('dependent quantity', 'be in', 'real world applications')],\n",
       "  [('diﬀerent strategies', 'be with', 'varying parameter values')],\n",
       "  [('uma x', '=', '0.06'), ('T', '=', '5'), ('b', '=', '15')],\n",
       "  [('it', 'be', 'applied')],\n",
       "  [('fraction', 'be in', 'control scenario')],\n",
       "  [('individuals', 'convince', 'susceptibles'),\n",
       "   ('infected individuals', 'convince', 'susceptibles')],\n",
       "  [('model', 'be in', 'Section 3')],\n",
       "  [('SIR model', 'in x be', 'word-of-mouth'),\n",
       "   ('1 2 u2ma x', 'be in', 'SIR model')],\n",
       "  [('total control eﬀorts', 'be', 'however considerably diﬀerent')],\n",
       "  [('SIS', 'be in', 'Figs. 9 12')],\n",
       "  [('plots', 'be in', 'section')],\n",
       "  [('more viral campaigns', 'run than', 'less viral campaigns'),\n",
       "   ('viral campaigns', 'run than', 'less viral campaigns'),\n",
       "   ('more viral campaigns', 'be less costly in', 'other words')],\n",
       "  [('We', 'incur', 'costs as application of control'),\n",
       "   ('We', 'incur', 'more costs as application of control')],\n",
       "  [('relative increase', 'be in', 'J')],\n",
       "  [('changes', 'be in', 'b')],\n",
       "  [('Conclusion', 'be in', 'paper'),\n",
       "   ('information spreading rate', 'be', 'when function of time')],\n",
       "  [('change', 'be in', 'spreading rate'),\n",
       "   ('change', 'be in', 'interest level of population'),\n",
       "   ('population', 'be in', 'subject of campaign')],\n",
       "  [('direct recruitment', 'be in', 'SIR epidemic process')],\n",
       "  [('We',\n",
       "    'uniqueness of',\n",
       "    'solutions for suﬃciently small campaign deadline')],\n",
       "  [('interesting future research direction',\n",
       "    'study',\n",
       "    'sensitivity of control strategies to estimation errors')],\n",
       "  [('Theorem 4.2', 'uniqueness of', 'Solution to SIS Model')],\n",
       "  [('variable t', 'be drop for', 'notational brevity'),\n",
       "   ('variable t', 'be', 'dropped')],\n",
       "  [('a', 'be', 'eat'),\n",
       "   ('a', 'be', 'real'),\n",
       "   ('a', 'be', 'positive'),\n",
       "   ('a', 'be', '=')],\n",
       "  [('variable t', 'be drop for', 'notational brevity'),\n",
       "   ('t', 'be drop for', 'notational brevity')],\n",
       "  [('solution', 'thus be unique for', 'T')],\n",
       "  [('Asano', 'control of', 'Vaccine Distribution'),\n",
       "   ('Optimal Control', 'be in', 'Rabies Metapopulation Model'),\n",
       "   ('Asano', 'control in', 'Rabies Metapopulation Model')],\n",
       "  [('NumericCitation H. Behncke', 'control of', 'Deterministic Epidemics')],\n",
       "  [('NumericCitation H. Gaﬀ', 'applied for', 'Various Epidemiological Models'),\n",
       "   ('NumericCitation H. Gaﬀ', 'applied to', 'Vaccination')],\n",
       "  [('NumericCitation A. A. Lashari',\n",
       "    'control of',\n",
       "    'Vector Borne Disease with Horizontal Transmission'),\n",
       "   ('Vector Borne Disease', 'be with', 'Horizontal Transmission')],\n",
       "  [('General SIR Model', 'be with', 'Vaccination'),\n",
       "   ('Optimal Singular Controls', 'on ledzewicz be', 'H. Sch ¨ attler')],\n",
       "  [('Optimal Control', 'on morton be', 'K. H. Wickwire'),\n",
       "   ('Advances', 'be in', 'Applied Probability 622 635')],\n",
       "  [('NumericCitation S. P. Sethi',\n",
       "    'control of',\n",
       "    'Simple Deterministic Epidemic Models')],\n",
       "  [('Zhu', 'control under', 'Delayed Model'),\n",
       "   ('Zhu', 'control of', 'Computer Virus')],\n",
       "  [('Educational Campaigns', 'journal of', 'Diﬀerential Equations')],\n",
       "  [('NumericCitation A. Karnik',\n",
       "    'proceedings of',\n",
       "    'IEEE Communication Systems'),\n",
       "   ('NumericCitation A. Karnik', 'control of', 'Information Epidemics')],\n",
       "  [('Chierichetti', 'spreading in', 'Social Networks'),\n",
       "   ('Rumor Spreading', 'be in', 'Social Networks')],\n",
       "  [('Design', 'conference on', 'Computer Communications')],\n",
       "  [('Rumours', 'university of', 'Adelaide'),\n",
       "   ('S. Belen', 'behaviour of', 'Rumours')],\n",
       "  [('New-Product Adoption Model', 'journal of', 'Optimization Theory'),\n",
       "   ('S. P. Sethi', 'advertising in', 'New-Product Adoption Model'),\n",
       "   ('Optimal Advertising', 'be in', 'New-Product Adoption Model')],\n",
       "  [('Durable-Good Duopoly', 'journal of', 'Operations Research'),\n",
       "   ('Krishnamoorthy', 'pricing in', 'Durable-Good Duopoly'),\n",
       "   ('Optimal Pricing', 'be in', 'Durable-Good Duopoly')],\n",
       "  [('HIV Model', 'journal of', 'Diﬀerential Equations'),\n",
       "   ('Optimizing Chemotherapy', 'be in', 'HIV Model'),\n",
       "   ('Fister', 'chemotherapy in', 'HIV Model')],\n",
       "  [('urse Notes', 'department of', 'Applied Mathematics'),\n",
       "   ('Applied Mathematics', 'university of', 'Washington'),\n",
       "   ('urse Notes', 'be in', 'Economics 18'),\n",
       "   ('Economics 18', 'in note be', 'Department of Applied Mathematics')]],\n",
       " 56: [[('Susceptible-Infective-Recovered-Susceptible model',\n",
       "    'for equation be',\n",
       "    'PA')],\n",
       "  [('same degree', 'of graph be', 'RRG')],\n",
       "  [('parameter values', 'be in', 'oscillatory phase')],\n",
       "  [('common paradigm', 'emerge from', 'simple deterministic framework')],\n",
       "  [('NumericCitation', 'be in', 'real populations')],\n",
       "  [('performance', 'be in', 'description of steady states')],\n",
       "  [('homogeneous degree k', 'graph of', 'degree k')],\n",
       "  [('recovered individuals', 'lose immunity at', 'rate γ'),\n",
       "   ('recovered individuals', 'lose', 'immunity')],\n",
       "  [('inﬁnite population limit',\n",
       "    'be with',\n",
       "    'assumptions of spatial homogeneity')],\n",
       "  [('variables s', 'stand for', 'probability')],\n",
       "  [('active phase', 'be with', 'oscillatory behavior')],\n",
       "  [('sustained oscillations', 'be in', 'thermodynamic limit')],\n",
       "  [('Phase diagram', 'be in', 'plane for PA SIRS model')],\n",
       "  [('region', 'be with', 'limit cycle solutions')],\n",
       "  [('δ', 'for model be', 'k ='), ('δ', 'for model be', 'k =')],\n",
       "  [('several values', 'be in', 'range k')],\n",
       "  [('dotted lines',\n",
       "    'be',\n",
       "    'Within active phase numerical plots of Hopf bifurcation curves')],\n",
       "  [('SIR dynamics',\n",
       "    'be with',\n",
       "    'diﬀerent mechanisms of replenishment of susceptibles'),\n",
       "   ('similar phase diagram',\n",
       "    'be with',\n",
       "    'Hopf bifurcation critical line bounding')],\n",
       "  [('small γ', 'be in', 'present case'),\n",
       "   ('oscillatory phase',\n",
       "    'be in',\n",
       "    'regime of slow driving through introduction of new susceptible individuals')],\n",
       "  [('recurrent epidemics', 'be in', 'infectious diseases')],\n",
       "  [('k =',\n",
       "    'be with',\n",
       "    'results of stochastic simulations on RRG-4 for several system sizes')],\n",
       "  [('Gillespie', 'on', 'method NumericCitation'),\n",
       "   ('random initial condition', 'be with', 'given node'),\n",
       "   ('stochastic processes',\n",
       "    'be in',\n",
       "    \"spatially structured systems based on Gillespie 's method NumericCitation\")],\n",
       "  [('comparison',\n",
       "    'be with',\n",
       "    'results of stochastic simulations on RRG-4 = for parameter values in region II'),\n",
       "   ('parameter values', 'be in', 'region II')],\n",
       "  [('λ', '=', '2.5')],\n",
       "  [('densities', 'be', 'shown'),\n",
       "   ('densities', 'be show in', 'Fig. 2 for two sets of parameter values')],\n",
       "  [('numerical solutions', 'result of', 'simulations in solid lines'),\n",
       "   ('simulations', 'be in', 'solid lines')],\n",
       "  [('parameter values', 'be in', 'as Fig. 2a')],\n",
       "  [('solution', 'together with one be', 'black solid line'),\n",
       "   ('same parameter values', 'for equation be', 'blue solid line'),\n",
       "   ('initial conditions', 'be in', 'plot denoted by A')],\n",
       "  [('absorbing phase', 'be in', 'stochastic simulations'),\n",
       "   ('oscillatory phase',\n",
       "    'be',\n",
       "    'why seen as absorbing phase in stochastic simulations')],\n",
       "  [('10 −', 'be in', 'transient regime')],\n",
       "  [('comparison',\n",
       "    'be with',\n",
       "    'results of stochastic simulations on RRG-4 plane'),\n",
       "   ('parameter values', 'be in', 'region III'),\n",
       "   ('RRG-4', 'be with', 'N'),\n",
       "   ('red dots', 'be in', 's')],\n",
       "  [('bistability regime', 'find in', 'region III')],\n",
       "  [('simulations', 'be in', 'region II'),\n",
       "   ('suppression', 'be in', 'region'),\n",
       "   ('results boundary', 'be with', 'region III')],\n",
       "  [('quantitative diﬀerences', 'be in', 'Monte Carlo simulations')],\n",
       "  [('Science', 'for foundation be', 'FCT')],\n",
       "  [('Spatial Interactions', 'be in', 'Simple Ecological Models')],\n",
       "  [('PhD dissertation', 'university of', 'Warwick')],\n",
       "  [('Szab', 'o be', 'G.')]],\n",
       " 57: [[('Euclidean network Abdul Khaleque1',\n",
       "    'on model be',\n",
       "    'University of Calcutta'),\n",
       "   ('Euclidean network Abdul Khaleque1', 'on model be', '92 APC Road'),\n",
       "   ('Euclidean network Abdul Khaleque1', 'on model be', 'Kolkata 700009'),\n",
       "   ('Susceptible-Infected-Recovered model', '1department of', 'Physics'),\n",
       "   ('Euclidean network Abdul Khaleque1',\n",
       "    'on model be',\n",
       "    '1Department of Physics'),\n",
       "   ('Susceptible-Infected-Recovered SIR epidemic model',\n",
       "    'be in',\n",
       "    'one dimension'),\n",
       "   ('Euclidean network Abdul Khaleque1', 'on model be', 'India'),\n",
       "   ('Susceptible-Infected-Recovered model', 'university of', 'Calcutta')],\n",
       "  [('R', 'fraction of', 'population'), ('R', 'duration of', 'epidemic')],\n",
       "  [('saturation value Rsat', 'attain', 'ﬁnite value')],\n",
       "  [('τ', 'scale behavior in', 'ﬁnite system of size N')],\n",
       "  [('qc', 'increase for', '1 < δ')],\n",
       "  [('logarithmic increase', 'be with', 'system size NumericCitation'),\n",
       "   ('transition temperature',\n",
       "    'show',\n",
       "    'logarithmic increase with system size NumericCitation')],\n",
       "  [('opinion dynamics models NumericCitation propagation',\n",
       "    'synchronization of',\n",
       "    'NumericCitation')],\n",
       "  [('one dimensional network', 'be with', 'additional random links')],\n",
       "  [('its dynamics',\n",
       "    'can',\n",
       "    'can conceived in diﬀerent manners on networks leading to conservation of diﬀerent physical quantities')],\n",
       "  [('diﬀerent dynamical rules',\n",
       "    'however be equivalent numericcitation on',\n",
       "    'regular lattices')],\n",
       "  [('individuals', 'can in', 'can three possible states'),\n",
       "   ('individuals', 'can in', 'where can three possible states')],\n",
       "  [('it', 'be', 'dynamical'),\n",
       "   ('its critical behavior',\n",
       "    'coincide with',\n",
       "    'that of static isotropic percolation on regular lattices NumericCitation')],\n",
       "  [('those', 'consider in', 'WS network')],\n",
       "  [('Nearest neighbor links', 'be always present in', 'model')],\n",
       "  [('similar model', 'be in', 'two dimensions')],\n",
       "  [('network', 'be', 'still small world here'), ('network', 'be', 'small')],\n",
       "  [('searching', 'have', 'have also studied on network NumericCitation')],\n",
       "  [('nature', 'be in', 'controversial region')],\n",
       "  [('results', 'be in', 'section III')],\n",
       "  [('results earlier studies', 'make on', 'same network')],\n",
       "  [('concluding remarks', 'be make in', 'last section')],\n",
       "  [('network', 'be generate in', 'following way')],\n",
       "  [('distance', 'be with', 'probability P')],\n",
       "  [('three densities', 'be relate through', '1')],\n",
       "  [('infected nodes', 'recover at', 'rate µ')],\n",
       "  [('k', 'be', 'number of contacts per unit time')],\n",
       "  [('systems', 'be with', 'size N')],\n",
       "  [('infected people',\n",
       "    'infect',\n",
       "    'susceptible individuals with probability q connected directly to them within same time scale'),\n",
       "   ('susceptible individuals',\n",
       "    'be with',\n",
       "    'probability q connected directly to them within same time scale')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('quantities', 'be', 'averaged')],\n",
       "  [('secondary averaging', 'consider', '100 diﬀerent network conﬁgurations')],\n",
       "  [('Periodic boundary condition', 'have', 'has used in simulation')],\n",
       "  [('recovered nodes', 'of r be', 't')],\n",
       "  [('We', 'update', 'state of nodes')],\n",
       "  [('continuous phase transition', 'take here', 'place'),\n",
       "   ('SIR', 'show', 'same criticality as isotropic 3 percolation problem')],\n",
       "  [('it', 'be', 'ﬁnite')],\n",
       "  [('Rsat', 'have', 'scaling form similar to magnetization')],\n",
       "  [('g1', 'be', 'where scaling function')],\n",
       "  [('We', 'obtain', 'data collapse')],\n",
       "  [('1', 'indeed match with', 'mean ﬁeld value for percolation')],\n",
       "  [('qc', '=', '1')],\n",
       "  [('variation', 'be with', 'value of exponent ˜ν')],\n",
       "  [('its peak value', 'increase with', 'Fig. 3')],\n",
       "  [('functions', 'be in', 'Fig. 5')],\n",
       "  [('variation', 'be with', 'δ')],\n",
       "  [('qc', 'reach', '1 slightly above δ')],\n",
       "  [('variation', 'be with', 'N')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('discussion', 'be in', 'preceding subsection'),\n",
       "   ('Distribution', 'be in', 'discussion')],\n",
       "  [('it', 'find in', 'NumericCitation'),\n",
       "   ('Rsat', 'of φ be', 'Rsat'),\n",
       "   ('We', 'have also study', 'Rsat')],\n",
       "  [('boundary condition', 'assume in', 'ﬁtting')],\n",
       "  [('F', '= dr dt from', 'then eq')],\n",
       "  [('F', 'be', 'where maximum')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('mean ﬁeld type recursion relation',\n",
       "    'be with',\n",
       "    'discrete time steps for SIR model'),\n",
       "   ('average degree', 'be hki from', 'eq')],\n",
       "  [('τ', 'use', 'hki')],\n",
       "  [('Variation', 'be with', 'q')],\n",
       "  [('scales', 'be with', 'N')],\n",
       "  [('comparison', 'be with', 'earlier works')],\n",
       "  [('It', 'have', 'has mentioned earlier'), ('It', 'have', 'has mentioned')],\n",
       "  [('it', 'be eﬀect of', 'ﬁnite sizes NumericCitation')],\n",
       "  [('δ', '=', '2.0 NumericCitation')],\n",
       "  [('contradictory results', 'again be obtain in', 'two diﬀerent studies'),\n",
       "   ('contradictory results', 'be', 'again obtained')],\n",
       "  [('study', 'be with', 'larger network NumericCitation')],\n",
       "  [('SIR', 'be relate to', 'percolation phenomena'),\n",
       "   ('SIR', 'be', 'expected')],\n",
       "  [('detailed study', 'be make in', 'two dimensions')],\n",
       "  [('exponents', 'be dependent', 'δ')],\n",
       "  [('exponent values', 'be in', 'diﬀerent dimensions')],\n",
       "  [('stronger dependence', 'be in', 'region 1')],\n",
       "  [('present results',\n",
       "    'be with',\n",
       "    'those of static percolation on same network')],\n",
       "  [('we', 'conclude remark from', 'discussion of last section')],\n",
       "  [('we', 'talk about', 'dimensionality'), ('we', 'add here', 'few words')],\n",
       "  [('ﬁnite size scaling', 'be in', 'types of systems NumericCitation')],\n",
       "  [('dimensionality', 'be in', 'between 4')],\n",
       "  [('agreement', 'be with', 'NumericCitation')],\n",
       "  [('variation', 'be with', 'δ'),\n",
       "   ('variation', 'be in', 'intermediate region')],\n",
       "  [('appreciable change',\n",
       "    'be in',\n",
       "    'scaling with system size at criticality here')],\n",
       "  [('it', 'have', 'small world behavior')],\n",
       "  [('AK', 'acknowledge', 'ﬁnancial support')],\n",
       "  [('2008', 'press at_time', 'Cambridge'),\n",
       "   ('NumericCitation A. Barrat', 'process on', 'complex networks')],\n",
       "  [('NumericCitation P. Sen', 'be in', 'Recent developments'),\n",
       "   ('NumericCitation P. Sen', 'be in', 'Handbook eds'),\n",
       "   ('Recent developments', 'be in', 'Theoretical Physics')],\n",
       "  [('sitions', 'be in', 'Lattice Models')],\n",
       "  [('NumericCitation D. Stauﬀer',\n",
       "    'introduction to',\n",
       "    'Percolation NumericCitation D. J. Watts')]],\n",
       " 58: [[('We',\n",
       "    'study',\n",
       "    'critical eﬀect of quarantine on propagation of epidemics on adaptive network of social contacts')],\n",
       "  [('we', 'reconnect', 'them')],\n",
       "  [('we', 'start from', 'single infected individual')],\n",
       "  [('rewiring probability', 'be proportional to', 'degree of infected nodes')],\n",
       "  [('many problems', 'be in', 'epidemiology NumericCitation')],\n",
       "  [('they', 'avoid', 'contacts with infected people'),\n",
       "   ('they', 'avoid', 'contacts'),\n",
       "   ('contacts', 'be with', 'infected people')],\n",
       "  [('eﬃcient isolation', 'be with', 'high degree')],\n",
       "  [('case', 'be with', 'w constant'),\n",
       "   ('preferential rewiring', 'be more eﬃcient than', 'case with w constant')],\n",
       "  [('susceptible state s', 'be with', 'one node chosen at'),\n",
       "   ('infected state', 'in random be', 'seed')],\n",
       "  [('chosen susceptible node', 'diﬀerent from', 'its present neighbors'),\n",
       "   ('it', 'reconnect', 'it')],\n",
       "  [('susceptible nodes', 'how fast react to', 'quarantine probability')],\n",
       "  [('tR', 'be', 'β')],\n",
       "  [('s', 'become', 'infected')],\n",
       "  [('corresponding expression', 'be in', 'standard SIR model Pn =')],\n",
       "  [('w', '=', '0')],\n",
       "  [('critical point', 'be give by', '1 − wc')],\n",
       "  [('individual', 'be', 'sick')],\n",
       "  [('T B β', 'w for', 'identical values of β')],\n",
       "  [('diﬀerence', 'be in', 'knowledge of states of nodes')],\n",
       "  [('phase diagrams', 'be in', 'w − β plane')],\n",
       "  [('network', 'be with', 'κ =')],\n",
       "  [('values', 'be in', 'region above curve β')],\n",
       "  [('strategy A', 'b. with', 'largest rewiring probability w'),\n",
       "   ('region', 'be', 'epidemic')],\n",
       "  [('β', 'be', 'when higher than pc'), ('β', 'be higher than', 'pc')],\n",
       "  [('epidemic', 'can', 'can always stopped even for β → 1')],\n",
       "  [('adaptive networks', 'be in', 'SIR models'),\n",
       "   ('Our theoretical predictions',\n",
       "    'illustrate',\n",
       "    'novel feature about dynamics of adaptive networks in SIR models')],\n",
       "  [('Our approach', 'predict through', 'Eq'),\n",
       "   ('Our analytical approach', 'predict through', 'Eq')],\n",
       "  [('K', 'be', 'where degree cutoﬀ')],\n",
       "  [('networks', 'be with', 'ﬁnite threshold pc')],\n",
       "  [('NGC', 'be', 'where size of giant component')],\n",
       "  [('ER network', 'for plot be', 'N ='),\n",
       "   ('ER network', 'be with', 'hkiGC ='),\n",
       "   ('function', 'as plot be', 'N ='),\n",
       "   ('nI', 'of plot be', 'N =')],\n",
       "  [('nI', 'approach', 'zero')],\n",
       "  [('wc increase', 'be with', 'β')],\n",
       "  [('β', '=', '0.1')],\n",
       "  [('Inset Data',\n",
       "    'be with',\n",
       "    'w rescaled by appropriate wc calculated from Eq')],\n",
       "  [('averages', 'be over', '104 realizations'), ('N', '=', '104')],\n",
       "  [('We', 'compute', '0')],\n",
       "  [('6', 'be', 'valid'),\n",
       "   ('6',\n",
       "    'be valid expression of',\n",
       "    'transition point for adaptive SIR model on ER networks')],\n",
       "  [('than homogeneous networks', 'be with', 'same mean degree hki')],\n",
       "  [('w =', 'be with', 'three diﬀerent hki')],\n",
       "  [('averages', 'be over', '104 realizations'), ('N', '=', '104')],\n",
       "  [('transition',\n",
       "    'will eventually disappear for',\n",
       "    'large heterogeneous networks')],\n",
       "  [('good agreement', 'be with', 'our predictions from Eq')],\n",
       "  [('κ increases', 'be with', 'K')],\n",
       "  [('propagation processes', 'be very diﬃcult due to', 'high degree nodes'),\n",
       "   ('propagation processes', 'stop', 'NumericCitation')],\n",
       "  [('NumericCitation', 'be', 'eﬃcient')],\n",
       "  [('we', 'test', 'prediction')],\n",
       "  [('tR =', 'be with', 'K ='), ('λ', '=', '2.1')],\n",
       "  [('kmax', 'be', 'where largest degree of network'),\n",
       "   ('degree k', 'be with', 'wk given by general form wk =')],\n",
       "  [('rewiring', 'increase with', 'degree k of infected node')],\n",
       "  [('SF networks', 'be with', 'K =')],\n",
       "  [('quarantine', 'isolate', 'superspreaders')],\n",
       "  [('K', '=', '20')],\n",
       "  [('other susceptible peers', 'be with', 'probability w')],\n",
       "  [('propagation', 'use', 'high enough values of w')],\n",
       "  [('quarantine', 'be', 'eﬀective'), ('eﬀective way', 'be in', 'other words')],\n",
       "  [('quarantine model', 'allow for', 'chance of infection')],\n",
       "  [('transition', 'disappear for', 'only large very heterogeneous networks')],\n",
       "  [('transition', 'eventually disappear for', 'large enough systems')],\n",
       "  [('individuals', 'be with', 'very large connectivity')],\n",
       "  [('we', 'conﬁrm', 'this')],\n",
       "  [('long time', 'be in', 'epidemic phase'),\n",
       "   ('adaptive models', 'be with', 'SIS')],\n",
       "  [('strong correlation', 'be with', 'topology')],\n",
       "  [('NumericCitation A. Vespignani', 'structure of', 'Complex Networks')],\n",
       "  [('Infectious Disease', 'be in', 'Humans'),\n",
       "   ('NumericCitation R.M. Anderson', 'disease in', 'Humans')],\n",
       "  [('aproximation pc ∼ =', 'be in', 'only networks larger than 109 nodes')],\n",
       "  [('Barab', 'asi be', 'Nature 401 130')]],\n",
       " 59: [[('Cornel l University', 'laboratory of', 'Atomic State Physics'),\n",
       "   ('quasi-neutral competition',\n",
       "    'be in',\n",
       "    'broad class of stochastic competition models'),\n",
       "   ('analysis', 'be in', 'two epidemic models'),\n",
       "   ('Plant Pathology', 'of department be', 'MY 14853'),\n",
       "   ('Two-strain competition',\n",
       "    'be in',\n",
       "    'quasi-neutral stochastic disease dynamics')],\n",
       "  [('model', 'be', 'two-strain')],\n",
       "  [('straight line', 'be in', 'phase space of sub-population sizes'),\n",
       "   ('system', 'be with', 'inﬁnite population size')],\n",
       "  [('Our perturbation method',\n",
       "    'explicitly track',\n",
       "    'dynamics of probability distribution of sub-populations in vicinity of CL'),\n",
       "   ('sub-populations', 'be in', 'vicinity of CL')],\n",
       "  [('central paradigm', 'be in', 'ecology'), ('Competition', 'be', 'central')],\n",
       "  [('example', 'be in', 'context of mode competition'),\n",
       "   ('mode competition', 'be in', 'lasers NumericCitation'),\n",
       "   ('minority phase', 'be in', 'process of Ostwald ripening NumericCitation')],\n",
       "  [('mutualism', 'promote', 'pathogen evolution')],\n",
       "  [('spread', 'be in', 'populations')],\n",
       "  [('contact', 'be with', 'infected')],\n",
       "  [('SIR model', 'be with', 'population turnover')],\n",
       "  [('averhost', 'be in', 'fully susceptible population')],\n",
       "  [('two simple models',\n",
       "    'have',\n",
       "    'have extended in diﬀerent directions NumericCitation')],\n",
       "  [('SIR model', 'be with', 'population turnover'),\n",
       "   ('two strains', 'be in', 'context of SIS model')],\n",
       "  [('same fraction', 'be in', 'case')],\n",
       "  [('strains', 'be term', 'quasi-neutral'),\n",
       "   ('situation', 'be with', 'diﬀerent rates')],\n",
       "  [('We', 'refer reader to', 'Ref')],\n",
       "  [('than cases', 'be with', 'diﬀerent R0')],\n",
       "  [('each', 'be', 'In deterministic limit line of ﬁxed points'),\n",
       "   ('line', 'be in', 'deterministic limit')],\n",
       "  [('ﬁxation scales',\n",
       "    'be with',\n",
       "    'characteristic population size N NumericCitation')],\n",
       "  [('method', 'be base on', 'time scale separation')],\n",
       "  [('ﬁxation', 'be in', 'two-strain extensions of SIS models'),\n",
       "   ('SI1 I2R model', 'be with', 'population turnover')],\n",
       "  [('method', 'be in', 'context of population genetics NumericCitation')],\n",
       "  [('total population size', 'be in', 'SI1 I2S model'),\n",
       "   ('model', 'be ﬁxed as', 'total population size in SI1 I2S model')],\n",
       "  [('sub-populations', 'be in', 'vicinity of CL')],\n",
       "  [('models', 'be with', 'ﬁxed total population size NumericCitation'),\n",
       "   ('slow strain',\n",
       "    'observe earlier in',\n",
       "    'models with ﬁxed total population size NumericCitation'),\n",
       "   ('slow strain', 'reach', 'ﬁxation')],\n",
       "  [('competition', 'be in', 'important situation')],\n",
       "  [('we',\n",
       "    'discuss',\n",
       "    'nature of their deterministic solutions focusing on quasi-neutral case')],\n",
       "  [('method', 'be in', 'Section III C'),\n",
       "   ('involved SI1 I2R model', 'be with', 'population turnover')],\n",
       "  [('probabilities', 'be in', 'Section IV')],\n",
       "  [('same susceptible population',\n",
       "    'be in',\n",
       "    'framework of SIS model NumericCitation')],\n",
       "  [('contact', 'be with', 'I1')],\n",
       "  [('N', 'be', 'ﬁxed'),\n",
       "   ('N', 'be', 'total'),\n",
       "   ('N 1', 'be', 'where ﬁxed total population size')],\n",
       "  [('I1', 'become again', 'susceptible')],\n",
       "  [('I1 κ1 I1 Recovery', 'of strain be', 'β2'),\n",
       "   ('strain 1',\n",
       "    'be with',\n",
       "    'strain 2 of I1 κ1 I1 Recovery of I2 κ2 I2 TABLE I. Transition rates'),\n",
       "   ('stochastic S I1 I2S model', 'for s be', 'I2 → I2'),\n",
       "   ('stochastic S I1 I2S model',\n",
       "    'for s be',\n",
       "    'S → S with strain 1 with strain 2 of I1 κ1 I1 Recovery of I2 κ2 I2 TABLE I. Transition rates'),\n",
       "   ('Event S', 'i1 of', 'transition S → S − 1'),\n",
       "   ('Event S', 's with', 'strain 1'),\n",
       "   ('stochastic S I1 I2S model',\n",
       "    'for s be',\n",
       "    'I1 → I1 of transition S → S − 1'),\n",
       "   ('strain 2', 'with strain be', 'β1'),\n",
       "   ('S → S', 'be with', 'strain 1'),\n",
       "   ('stochastic S I1 I2S model', 'for s be', 'S → S')],\n",
       "  [('dynamical system', 'be in', 'addition'),\n",
       "   ('non-negative population sizes', 'with ﬁxed be', 'FPs')],\n",
       "  [('other', 'be', 'attractor')],\n",
       "  [('R1 < R2', 'character of', 'ﬁxed points')],\n",
       "  [('two strains', 'be possible of', 'attracting'),\n",
       "   ('state', 'be with', 'larger Ri')],\n",
       "  [('N 1 −', 'form', '3'),\n",
       "   ('I1 I2 endemic plane', 'on line be', '3'),\n",
       "   ('I1 I2 endemic plane', 'on line be', 'CL')],\n",
       "  [('CL', 'of n be', '0 1 − 1/R'),\n",
       "   ('continuum family', 'be with', 'arbitrary proportion of strains')],\n",
       "  [('we', 'lose', 'generality')],\n",
       "  [('quasi-neutral SI1 I2S model', 'be in', 'rescaled variables')],\n",
       "  [('Figure 1', 'show', 'phase plane of system')],\n",
       "  [('Our second model deals',\n",
       "    'be with',\n",
       "    'two infectious strains competing for same susceptible population in framework of SIR model'),\n",
       "   ('B. SI1 I2R model', 'be with', 'population turnover'),\n",
       "   ('same susceptible population', 'be in', 'framework of SIR model')],\n",
       "  [('SIR model', 'include', 'following processes')],\n",
       "  [('strain 1', 'be with', 'rate')],\n",
       "  [('I1', 'recover with', 'immunity')],\n",
       "  [('stochastic SI1 I2R model', 'be with', 'population turnover'),\n",
       "   ('three ﬁxed points', 'be with', 'non-negative pop')],\n",
       "  [('saddle', 'be with', 'only one attracting direction')],\n",
       "  [('other', 'be', 'attractor')],\n",
       "  [('saddle', 'be with', 'two attracting directions in I2S plane'),\n",
       "   ('two attracting directions', 'be in', 'I2S plane')],\n",
       "  [('in SI1 I2S model', 'be with', 'R1')],\n",
       "  [('vicinity', 'of character be', 'relaxation time scale')],\n",
       "  [('three-dimensional phase space I1 I2S', 'in line be', 'CL'),\n",
       "   ('deterministic coexistence line',\n",
       "    'be in',\n",
       "    'three-dimensional phase space I1 I2S'),\n",
       "   ('R1', '=', 'R2 ≡ R')],\n",
       "  [('SI1 I2R model', 'be with', 'population turnover'),\n",
       "   ('CL', 'be', 'stable'),\n",
       "   ('SI1 I2R model', 'for equation be', '7'),\n",
       "   ('stable spiral', 'be in', 'direction transverse to CL')],\n",
       "  [('quasi-neutral SI1 I2R model', 'be in', 'rescaled variables')],\n",
       "  [('property', 'lead to', 'Eq')],\n",
       "  [('SI1 I2R model', 'be in', 'rescaled variables x')],\n",
       "  [('we', 'follow in', 'remainder of paper')],\n",
       "  [('eﬀect', 'show in', 'Fig. 3')],\n",
       "  [('example', 'for pm be', 'n'),\n",
       "   ('we', 'will use for', 'explanations in subsection'),\n",
       "   ('SI1 I2R model', 'for pm be', 'n'),\n",
       "   ('explanations', 'be in', 'subsection')],\n",
       "  [('we', 'employ', 'van Kampen system size ex-small parameter 1/N 1')],\n",
       "  [('we',\n",
       "    'derive',\n",
       "    'eﬀective one-dimensional Fokker-Planck equation for slowly-evolving probability distribution of system along CL')],\n",
       "  [('probability distribunumbers', 'be in', 'of tion P')],\n",
       "  [('sharp peak', 'evolve into', 'CL')],\n",
       "  [('It', 'be', 'probability distribution spread along FIG. 3')],\n",
       "  [('population turnover', 'with realization be', 'upper panel'),\n",
       "   ('its', 'jection', 'I2 plane'),\n",
       "   ('stochastic realization',\n",
       "    'be with',\n",
       "    'population turnover in phase space of I1'),\n",
       "   ('SI1 I2R model', 'of realization be', 'upper panel'),\n",
       "   ('population turnover', 'be in', 'phase space of I1')],\n",
       "  [('Extinction', 'can', 'can seen')],\n",
       "  [('SIS model', 'be with', 'population turnover NumericCitation'),\n",
       "   ('SIR model', 'be with', 'population turnover NumericCitation')],\n",
       "  [('Markov stochastic dynamics',\n",
       "    'be in',\n",
       "    'discrete state space of sub-population sizes')],\n",
       "  [('˙Pm', 'be with', 'time rescaled by 1/µ'),\n",
       "   ('˙Pm', 'be in', 'as deterministic equations')],\n",
       "  [('new variables', 'in ρ be', 'X t'), ('∂ t ρ =', 'be in', 'new variables')],\n",
       "  [('Y ∼ 1', 'be in', 'region of ridge')],\n",
       "  [('we', 'ing', 'right hand side of Eq')],\n",
       "  [('O', 'of scale be', 'ε − 2')],\n",
       "  [('0 ρ =', 'be in', 'second order of ε'),\n",
       "   ('second order', 'in ρ be', '2'),\n",
       "   ('ﬁrst order', 'in ρ be', '0'),\n",
       "   ('ﬁrst order', 'in ρ be', '19'),\n",
       "   ('1 ρ', 'be in', 'ﬁrst order of ε')],\n",
       "  [('f', 'be', 'arbitrary')],\n",
       "  [('unknown f', 'with ρ be', '0'),\n",
       "   ('unknown f', 'with ρ be', 'X τ'),\n",
       "   ('ρ', 'be with', 'yet unknown f')],\n",
       "  [('20', 'be', 'full')],\n",
       "  [('it', 'vanish upon', 'integration')],\n",
       "  [('ﬁrst order', 'be in', 'ε')],\n",
       "  [('bypass',\n",
       "    'fortunately be possible for',\n",
       "    'whole class of quasineutral competition models')],\n",
       "  [('25', 'be', 'given'), ('25', 'be give in', 'Eqs')],\n",
       "  [('It', 'involve', 'diﬀusion')],\n",
       "  [('bias', 'be in', 'favor of slow strain for which')],\n",
       "  [('Moran model', 'model of', 'random genetic variations'),\n",
       "   ('Moran model', 'model in', 'haploid population'),\n",
       "   ('minimalist model', 'be in', 'haploid population')],\n",
       "  [('it', 'have', 'proper variance ∼ 1/N')],\n",
       "  [('their results', 'be', 'identical'),\n",
       "   ('their results', 'be identical to', 'those following from our Eqs')],\n",
       "  [('geometric ratchet mechanism',\n",
       "    'importantly be independent of',\n",
       "    'speciﬁc competition model')],\n",
       "  [('we', 'now return to', 'Eq')],\n",
       "  [('ξ', '=', '1'), ('ξ', '=', '− 1'), ('I2', '=', '0')],\n",
       "  [('ξ0', 'be', 'where determined')],\n",
       "  [('our method',\n",
       "    'give',\n",
       "    'explicit description of dynamics of probability distribution of sub-populations in vicinity of CL'),\n",
       "   ('sub-populations', 'be in', 'vicinity of CL')],\n",
       "  [('description', 'be', 'In leading order in 1 √ N given by Eq'),\n",
       "   ('leading order', 'be in', '1 √ N')],\n",
       "  [('SI1 I2R model', 'be with', 'population turnover involved'),\n",
       "   ('C. SI1 I2R model', 'be with', 'population turnover')],\n",
       "  [('application',\n",
       "    'be',\n",
       "    'more of our perturbation method here leads again to eﬀective one-dimensional Fokker-Planck equation along CL')],\n",
       "  [('Time t', 'have', 'has rescaled'),\n",
       "   ('Time t', 'have', 'has rescaled by 1/κ1')],\n",
       "  [('before', 'as equation be', '20'), ('before', 'as equation be', '18')],\n",
       "  [('normalization factor', 'be in', 'contrast to two-dimensional case')],\n",
       "  [('we', 'use', '36')],\n",
       "  [('⇠', '=', '1'), ('I2', '=', '0')],\n",
       "  [('quadratic form', 'be in', 'Y'),\n",
       "   ('quadratic form', 'be in', 'exponential'),\n",
       "   ('exponential', 'be in', 'Eq')],\n",
       "  [('following expression', 'be in', 'terms of principal')],\n",
       "  [('1 Λ', 'be in', 'terms of Λ ±')],\n",
       "  [('full analogy', 'be with', 'Eq')],\n",
       "  [('47 Calculation', 'be in', 'Eq')],\n",
       "  [('what general procedure', 'outline in', 'Appendix C')],\n",
       "  [('ξ', '=', '1'), ('ξ', '=', '− 1')],\n",
       "  [('N r', 'be in', 'standard Fokker-Planck form'),\n",
       "   ('equation', 'be in', 'variable ξ')],\n",
       "  [('ξ0', 'be', 'where fully determined')],\n",
       "  [('one-dimensional problem', 'be', 'once solved'),\n",
       "   ('sub-populations', 'be in', 'vicinity of CL')],\n",
       "  [('leading order', 'be in', '1')],\n",
       "  [('we', 'study', 'quasi-neutral competition')],\n",
       "  [('SI1 I2R model', 'for v be', 'ξ')],\n",
       "  [('large diﬀerence', 'be in', 'terms of rates')],\n",
       "  [('SI1 I2S model', 'for v be', 'ξ')],\n",
       "  [('→ 0', 'as v be', 'ξ')],\n",
       "  [('SI1 I2S model', 'for − be', 'ξ0'),\n",
       "   ('agreement', 'be with', 'Refs'),\n",
       "   ('ξ0', 'be in', 'agreement with Refs'),\n",
       "   ('agreement', 'in ξ0 be', '59')],\n",
       "  [('set', 'for ea be', '60')],\n",
       "  [('CL', 'be in', 'cases')],\n",
       "  [('Eq', 'from v be', 'ξ')],\n",
       "  [('→ 0', 'as = be', 'dotted line'),\n",
       "   ('SI1 I2R model', 'be with', '= 0.02 solid line'),\n",
       "   ('SI1 I2R model', 'for 54 be', '= 0.2 dashed line')],\n",
       "  [('models', 'for 1 be', 'ξ0'),\n",
       "   ('models', 'for 1 be', 'π +1 ='),\n",
       "   ('models', 'for 1 be', '62')],\n",
       "  [('boundary conditions T', 'with d be', 'ξ0 T ξ0'),\n",
       "   ('1 2 D =', 'be with', 'boundary conditions T')],\n",
       "  [('deﬁnition', 'be in', 'two models')],\n",
       "  [('integrals', 'be in', 'Eq')],\n",
       "  [('initial condition ξ0', 'vs. online be', 'rN')],\n",
       "  [('dots', 'numerically solve', 'Eq')],\n",
       "  [('eﬀect', 'be especially pronounced in', 'SI1 I2R model')],\n",
       "  [('SI1 I2R model', 'behave diﬀerently at', '→ 0')],\n",
       "  [('approaches 1', 'as ξ0 be', '3/4')],\n",
       "  [('solid curves', 'be calculate from', 'Eq')],\n",
       "  [('dots', 'numerically solve', 'Eq')],\n",
       "  [('analytic results',\n",
       "    'be with',\n",
       "    'numerical results obtained by numerical solution of master equation for SI1 I2S model'),\n",
       "   ('SI1 I2R', 'be in', 'cases'),\n",
       "   ('mean time', 'be in', 'Appendix D. Figure 7')],\n",
       "  [('origin', 'be in', 'Fig. 1')],\n",
       "  [('we', 'show in', 'Fig. 10')],\n",
       "  [('F', '=', '1/2')],\n",
       "  [('Color online Competitive advantage', 'be in', 'case')],\n",
       "  [('we', 'obtain f for', 'larger L')],\n",
       "  [('we', 'can use eqs for', 'SI1 I2S model')],\n",
       "  [('we', 'ﬁnally obtain', '1 − a')],\n",
       "  [('n', 'be', 'where number of individuals with new strain'),\n",
       "   ('individuals', 'be with', 'new strain')],\n",
       "  [('changes', 'be in', \"system 's behavior\"), ('system', 'in', 'behavior')],\n",
       "  [('it', 'be', 'When taken into account'), ('it', 'be', 'When taken')],\n",
       "  [('slow strain',\n",
       "    'have',\n",
       "    'competitive advantage for uniformly distributed initial conditions')],\n",
       "  [('quasineutral competition', 'be in', 'epidemiology')],\n",
       "  [('quasineutral model', 'of line be', 'CL')],\n",
       "  [('whole multi-dimensional probability distribution',\n",
       "    'be in',\n",
       "    'vicinity of CL'),\n",
       "   ('method',\n",
       "    'also describe',\n",
       "    'whole multi-dimensional probability distribution of system in vicinity of CL')],\n",
       "  [('future work', 'envision in', 'Ref'),\n",
       "   ('quasi-neutral competition', 'be in', 'spatial systems')],\n",
       "  [('our interest', 'be in', 'quasi-neutral competition'),\n",
       "   ('discussion', 'be in', 'noisy limit cycles'),\n",
       "   ('We', 'attract', 'our interest'),\n",
       "   ('We', 'attract', 'our interest in quasi-neutral competition')],\n",
       "  [('pro ject', 'be', 'where started')],\n",
       "  [('Linear stability', 'be in', 'variables')],\n",
       "  [('±', 'be unrelated to', 'inverse widths of bi-Gaussian Λ ± given by Eq')],\n",
       "  [('orX Y Z', 'to 15 be', 'A4')],\n",
       "  [('independent variables', 'be in', 'operator'),\n",
       "   ('independent variables', 'be in', 'case one'),\n",
       "   ('X', 'be', 'frozen')],\n",
       "  [('drift terms', 'be in', 'ˆL'),\n",
       "   ('full deterministic equations', 'be in', 'variables')],\n",
       "  [('subset', 'be in', 'variables')],\n",
       "  [('integration', 'be in', 'Eq'), ('integration', 'be in', 'Y variables')],\n",
       "  [('ﬁrst term', 'be in', 'Eq')],\n",
       "  [('ﬁrst two terms', 'be in', 'Eq')],\n",
       "  [('equation', 'generalize', 'Eq'),\n",
       "   ('Our starting point', 'be in', 'Appendix')],\n",
       "  [('O', 'of scale be', 'ε − 2')],\n",
       "  [('0 ρ =', 'be in', 'second order of ε'), ('second order', 'in ρ be', '2')],\n",
       "  [('balance', 'be in', 'derivation of Eq'), ('ˆL', 'be', 'general')],\n",
       "  [('f', 'be', 'arbitrary')],\n",
       "  [('function ρ', 'be with', 'yet unknown f'),\n",
       "   ('ρ', 'be', 'sharp'),\n",
       "   ('unknown f', 'with ρ be', '0'),\n",
       "   ('unknown f', 'with ρ be', 'X τ')],\n",
       "  [('Eq', 'from cij be', 'X')],\n",
       "  [('we', 'obtain', '∂ τ ρ')],\n",
       "  [('this', 'do in', 'multi-dimensional problems')],\n",
       "  [('SI1 I2R model', 'be with', 'population turnover')],\n",
       "  [('diﬀusion term', 'be in', 'full Fokker-Planck equation')],\n",
       "  [('total derivatives', 'be with', 'Ξi')],\n",
       "  [('ﬁrst term', 'be in', 'Eq')],\n",
       "  [('it', 'be linear in', 'Y'), ('it', 'be', 'linear')],\n",
       "  [('ˆL †', 'be in', 'Eq')],\n",
       "  [('Y Y Y diﬀerential equation', 'be with', 'deﬁnitions')],\n",
       "  [('double-starred equality', 'follow from', '18 in Eq')],\n",
       "  [('it', 'indeed follow from', 'Eqs')],\n",
       "  [('h X Y', '2 for', 'SI1 I2R model Eq')],\n",
       "  [('last line', 'be in', 'Eq')],\n",
       "  [],\n",
       "  [('− 3 ln', 'ξ0', '1 − ξ0'),\n",
       "   ('we', 'obtain for', 'SI1 I2S model'),\n",
       "   ('rN', 'rescaling in', 'everywhere Appendix')],\n",
       "  [('models', 'for ln be', 'ξ0')],\n",
       "  [('It', 'also grow as', 'decreases below 1')],\n",
       "  [('expression', 'be very cumbersome for', 'SI1 I2R model')],\n",
       "  [('ln ln 1', 'to simpliﬁes be', 'D4')],\n",
       "  [('SI1 I2R model', 'for point be', 'T ξ0'),\n",
       "   ('D7 20', 'be in', 'region of 1 − ξ0 2a'),\n",
       "   ('we', 'obtain', '→ 0')],\n",
       "  [('ﬁrst term', 'be in', 'numerator of Eq')],\n",
       "  [('NumericCitation E. Tielkes', 'competition in', 'Changing World'),\n",
       "   ('NumericCitation E. Tielkes', 'competition for', 'Resources'),\n",
       "   ('Competition', 'be in', 'Changing World'),\n",
       "   ('Rural Development', 'for drive be', 'Citation')],\n",
       "  [('Ostwald Ripening', 'be in', 'Material Processing')],\n",
       "  [('Modeling Infectious Diseases', 'be in', 'Humans')],\n",
       "  [('NumericCitation M.S. Bartlett', 'model in', 'Ecology'),\n",
       "   ('Stochastic Population Models', 'be in', 'Ecology'),\n",
       "   ('R.M. May', 'disease of', 'Humans')],\n",
       "  [('NumericCitation C. W. Gardiner', 'handbook of', 'Stochastic Methods'),\n",
       "   ('Stochastic Methods', 'of handbook be', 'Citation')],\n",
       "  [('M. Lax', 'be in', 'Statistical Physics')]],\n",
       " 60: [[('vaccines',\n",
       "    'use of be',\n",
       "    'Through years has always controversial issue'),\n",
       "   ('use', 'be', 'controversial')],\n",
       "  [('People', 'be in', 'society')],\n",
       "  [('attitude', 'be in', 'face of vaccines'),\n",
       "   ('their transformation', 'be in', 'epidemics'),\n",
       "   ('consequences', 'be in', 'spread of diseases')],\n",
       "  [('disease spreading', 'be', 'where described by SIR-Model')],\n",
       "  [('his opinion', 'be in', 'other layer'),\n",
       "   ('contact', 'be with', 'infected neighbor')],\n",
       "  [('contact', 'be with', 'infected neighbor')],\n",
       "  [('diﬀerent behaviors', 'be in', 'spread of epidemic'),\n",
       "   ('depending trend', 'be in', 'opinion of society')],\n",
       "  [('it', 'vary with', 'opinion parameter r')],\n",
       "  [('2 Introduction', 'be in', '1796 Edward Jenner')],\n",
       "  [('pioneering essay diﬀerent vaccines',\n",
       "    'prevent',\n",
       "    'long list of infectious diseases')],\n",
       "  [('associated vaccination', 'be with', 'autism NumericCitation')],\n",
       "  [('debate', 'be', 'actual')],\n",
       "  [('spreads', 'be in', 'diﬀerent ways')],\n",
       "  [('has sub ject', 'be in', 'recent years')],\n",
       "  [('NoN',\n",
       "    'be',\n",
       "    'system formed by single networks interacting through external connections between them')],\n",
       "  [],\n",
       "  [('contact', 'be with', 'infected node'),\n",
       "   ('it', 'become', 'immune'),\n",
       "   ('we', 'call', 'virulence of disease')],\n",
       "  [('infection', 'of period be', 'ﬂu'), ('example', 'for period be', 'ﬂu')],\n",
       "  [('study', 'be in', 'real networks NumericCitation')],\n",
       "  [('studies', 'design', 'vaccination campaigns')],\n",
       "  [('eﬀective NumericCitation', 'be in', 'general')],\n",
       "  [('individuals', 'be with', 'infectious symptoms')],\n",
       "  [('co-evolves', 'be with', 'social context')],\n",
       "  [('people', 'of', 'opinions'),\n",
       "   ('importance', 'be in', \"formation of people 's opinions\")],\n",
       "  [('opinion', 'of', 'formation')],\n",
       "  [('two social dynamics', 'one for', 'opinion formation')],\n",
       "  [('one model', 'dominate', 'behavior of system')],\n",
       "  [('propagation', 'be in', 'population')],\n",
       "  [('he', 'be', 'vaccinated'),\n",
       "   ('he', 'become completely against', 'vaccination')],\n",
       "  [('SIR model', 'be in', 'multiplex system composed by two networks'),\n",
       "   ('SIR model', 'be with', 'vaccination')],\n",
       "  [('they', 'get', 'infected')],\n",
       "  [('Section 4', 'be', 'devoted')],\n",
       "  [('formation', 'be in', 'favor')],\n",
       "  [('ideas', 'be with', 'other 5 individuals')],\n",
       "  [('he', 'get', 'vaccinated')],\n",
       "  [('opinions', 'can', 'can transmitted via other media')],\n",
       "  [('spread', 'be in', 'layer A')],\n",
       "  [('opinion M', '=', '+2')],\n",
       "  [('contact', 'be with', 'infected individual'),\n",
       "   ('individual S', 'be in', 'contact with infected individual')],\n",
       "  [('he', 'acquire', 'immunity')],\n",
       "  [('This', 'be', 'extreme')],\n",
       "  [('we',\n",
       "    'use m model numericcitation for',\n",
       "    'process of opinion formation in network B'),\n",
       "   ('opinion formation', 'be in', 'network B')],\n",
       "  [('phenomena', 'be in', 'population of interacting individuals')],\n",
       "  [('interaction', 'be with', 'neighbor 6 j')],\n",
       "  [('± 1 → xi', '=', '± 2'), ('± xi', '=', '± 2')],\n",
       "  [('it', 'remain', 'extremist')],\n",
       "  [('i', 'be', 'vaccinated')],\n",
       "  [('more extremist', 'be with', 'probability p')],\n",
       "  [('ratio r', '=', 'p/q')],\n",
       "  [('his opinion', 'be with', 'probability p'),\n",
       "   ('opinion',\n",
       "    'be with',\n",
       "    'diﬀerent orientation approaches to opinion of neighbor'),\n",
       "   ('opinion', 'be with', 'probability q'),\n",
       "   ('neighbor', 'be with', 'state xj')],\n",
       "  [('time', 'after i be', 'red'),\n",
       "   ('tr', 'go to', 'blue'),\n",
       "   ('probability β', 'with i be', 'red')],\n",
       "  [('contact', 'be with', 'infected individual')],\n",
       "  [('its opinion', 'be with', 'probability p')],\n",
       "  [('S', 'vaccinate', 'gray')],\n",
       "  [('extensive Monte Carlo simulations', 'be with', 'synchronous update')],\n",
       "  [('Nodes', 'be in', 'layer'), ('pair', 'each from', 'diﬀerent layer')],\n",
       "  [('Erdős-Rényi NumericCitation degree distribution', 'be with', 'hki =')],\n",
       "  [('recovery time', 'be in', 'tr')],\n",
       "  [('rest', 'be', 'susceptible'),\n",
       "   ('whose opinion',\n",
       "    'be choose at',\n",
       "    'random between four possible opinion states')],\n",
       "  [('vaccinated neighbors', 'be with', 'probability'),\n",
       "   ('their susceptible neighbors', 'be with', 'probability β')],\n",
       "  [('recovery nodes', 'fraction of', 'vaccinated nodes')],\n",
       "  [('nodes', 'be with', 'positive opinion state')],\n",
       "  [('recovered individuals', 'be in', 'ﬁnal state')],\n",
       "  [('r', '=', '0.1'), ('population', 'have', 'opinion')],\n",
       "  [('R', 'fraction of', 'vaccinated individuals')],\n",
       "  [('R', 'of derivative be', 'dashed line'),\n",
       "   ('function =', 'as τ be', 'solid line')],\n",
       "  [('inﬂection points', 'be in', 'number of recovered agents')],\n",
       "  [('r', '=', '0.1')],\n",
       "  [('it', 'prevent', 'epidemic')],\n",
       "  [('recovery individuals', 'be in', 'steady state')],\n",
       "  [('moderate opinions', 'be in', 'favor')],\n",
       "  [('agent', 'be', 'when vaccinated'),\n",
       "   ('he', 'will only change', 'his opinion'),\n",
       "   ('his', 'opinion', '2')],\n",
       "  [('opinion', 'be', 'polarized')],\n",
       "  [('vaccinated agents', 'pin in', 'opinion +2'),\n",
       "   ('vaccinated agents', 'stay', 'pinned in opinion +2')],\n",
       "  [('consensus', 'be in', 'favor of vaccine')],\n",
       "  [('it', 'spread', 'disease')],\n",
       "  [('agents', 'be in', 'system')],\n",
       "  [('Fig. 2', 'inset in', 'ones')],\n",
       "  [('epidemics', 'duration of be', 'when maximum'),\n",
       "   ('maximum', 'be in', 'derivatives')],\n",
       "  [('Fig. 3', 'show r as', 'function ω')],\n",
       "  [('consensus', 'be in', 'favor of vaccine')],\n",
       "  [('Agents', 'be with', 'negative opinion'),\n",
       "   ('Agents', 'change', 'their opinion')],\n",
       "  [('M ag', 'be in', 'opinion dynamic')],\n",
       "  [('opinions', 'model of be', 'where consensus')],\n",
       "  [('R', 'fraction of', 'vaccinated individuals')],\n",
       "  [('R', 'of derivative be', 'dashed line'),\n",
       "   ('function =', 'as τ be', 'solid line')],\n",
       "  [('r', '=', '1')],\n",
       "  [('agents', 'be with', 'extremist opinion'),\n",
       "   ('agents', 'convince agent with', 'moderate opinion')],\n",
       "  [('extremist', 'be in', 'opinions')],\n",
       "  [('q', '=', '0.09')],\n",
       "  [('vaccinated agents', 'be in', 'opinion state')],\n",
       "  [('vaccinated agent', 'be', 'When infected')],\n",
       "  [('epidemic dynamics', 'be faster than', 'dynamics of opinions')],\n",
       "  [('R', 'fraction of', 'vaccinated individuals')],\n",
       "  [('R', 'of derivative be', 'dashed line'),\n",
       "   ('function =', 'as τ be', 'solid line')],\n",
       "  [('r', '=', '10')],\n",
       "  [('system', 'always stay in', 'epidemic regime')],\n",
       "  [('disease', 'have', 'fewer initial vaccinated nodes')],\n",
       "  [('neutral society', 'prevent', 'epidemic')],\n",
       "  [('agents', 'have', 'moderate opinion')],\n",
       "  [('This', 'become', 'extremists in favor of vaccination'),\n",
       "   ('extremists', 'be in', 'favor of vaccination')],\n",
       "  [('virulence', 'be', 'even when small')],\n",
       "  [('agent', 'be with', 'positive moderate opinion become'),\n",
       "   ('extremist', 'be in', 'favor of vaccine')],\n",
       "  [('Threshold epidemic value β ∗', 'be with', 'tr =')],\n",
       "  [('Discussion', 'be in', 'paper')],\n",
       "  [('recovery individuals', 'be in', 'steady state')],\n",
       "  [('they', 'become', 'extremist in favor of vaccination'),\n",
       "   ('they', 'extremist in', 'favor of vaccination'),\n",
       "   ('extremist', 'be in', 'favor of vaccination')],\n",
       "  [('virulence', 'be', 'even when small')],\n",
       "  [('negative opinion', 'be in', 'favor on vaccination')],\n",
       "  [('it', 'be', 'easier'),\n",
       "   ('it', 'convince', 'people'),\n",
       "   ('people', 'get', 'vaccinated'),\n",
       "   ('compromise', 'be in', 'same proportion')]],\n",
       " 61: [[('conjunction', 'be with', 'model of susceptible')],\n",
       "  [('heterogeneity', 'be in', 'transmission'),\n",
       "   ('transmission',\n",
       "    'in heterogeneity be',\n",
       "    'value quite similar to that computed for strains of inﬂuenza NumericCitation')],\n",
       "  [('syndrome', 'be', 'respiratory'),\n",
       "   ('China', 'in', 'southern province of Guangdong'),\n",
       "   ('syndrome', 'be', 'new')],\n",
       "  [('substantial negative economic impact', 'be in', 'parts of East Asia'),\n",
       "   ('it',\n",
       "    'have',\n",
       "    'substantial negative economic impact in parts of East Asia')],\n",
       "  [('One man', 'be with', 'SARS made')],\n",
       "  [('severity', 'be in', 'diﬀerent people')],\n",
       "  [('adverse economic impact', 'be in', 'parts of East Asia'),\n",
       "   ('disruption', 'be in', 'part of world NumericCitation')],\n",
       "  [('NumericCitation', 'understanding of', 'causative virus'),\n",
       "   ('NumericCitation', 'be', 'early')],\n",
       "  [('virus', 'be', 'how transmitted')],\n",
       "  [('SARS', 'be', 'public'),\n",
       "   ('SARS', 'be', 'public health crisis on scale rarely seen')],\n",
       "  [('ﬁrst identiﬁ Citation',\n",
       "    'be in',\n",
       "    'Guongdong Province of China NumericCitation'),\n",
       "   ('obvious question', 'be in', 'such crisis 2 SARS'),\n",
       "   ('SARS',\n",
       "    'be',\n",
       "    'ﬁrst identiﬁ Citation in Guongdong Province of China NumericCitation')],\n",
       "  [('2003 oﬃcials', 'be in', 'Hong Kong')],\n",
       "  [('U. S. health oﬃcials', 'be currently investigate', '199 cases'),\n",
       "   ('199 cases', 'be in', '34 states')],\n",
       "  [('NumericCitation', 'be with', '3 − 5 days most common NumericCitation')],\n",
       "  [('fever', 'shortness of', 'breath')],\n",
       "  [('people', 'be with', 'SARS')],\n",
       "  [('389 cases', 'be in', '26 countries')],\n",
       "  [('victims', 'die', 'NumericCitation'),\n",
       "   ('165 victims', 'die', 'NumericCitation')],\n",
       "  [('some', 'produce', 'common colds'),\n",
       "   ('Erasmus Medical Center', 'be in', 'Rotterdam'),\n",
       "   ('SARS', 'mode of', 'transmission'),\n",
       "   ('researchers', 'be in', 'Erasmus Medical Center'),\n",
       "   ('some', 'produce', 'colds')],\n",
       "  [('conjunction', 'be with', 'global SARS data'),\n",
       "   ('model', 'estimate', 'initial growth rate of SARS epidemic')],\n",
       "  [('rates', 'estimate', 'R0'),\n",
       "   ('SARS', 'number be', 'R0'),\n",
       "   ('SARS',\n",
       "    'number be',\n",
       "    'classical epidemiological measure associated with reproductive power of disease')],\n",
       "  [('average number', 'be in', 'population of susceptibles NumericCitation'),\n",
       "   ('it', 'estimate', 'initial growth of SARS outbreak')],\n",
       "  [('may variations', 'be in', 'contact rates'),\n",
       "   ('ﬁrst cases', 'be in', 'Toronto'),\n",
       "   ('SARS', 'be in', 'Hong Kong')],\n",
       "  [('1/γ1', 'have', 'has reported')],\n",
       "  [('SARS outbreaks', 'be in', 'diﬀerent parts of world')],\n",
       "  [('we', 'can reach', 'two conclusions'),\n",
       "   ('we', 'can reach', 'two important conclusions')],\n",
       "  [('except transient response', 'be in', 'diagnostic rate α'),\n",
       "   ('fundamental growth rate β',\n",
       "    'be essentially constant from',\n",
       "    'one region of world to another')],\n",
       "  [('rapid roll-over', 'be in', 'growth rate of recognized cases')],\n",
       "  [('SEIJR model', 'be with', 'parameters α')],\n",
       "  [('We', 'examine', 'non-exponential dynamics of Toronto outbreak')],\n",
       "  [('Toronto data',\n",
       "    'of feature be',\n",
       "    'steep increase in number after March 31st in growth'),\n",
       "   ('March 31st', 'be in', 'growth of new recognized cases'),\n",
       "   ('Two features',\n",
       "    'increase after',\n",
       "    'March 31st in growth of new recognized cases'),\n",
       "   ('Two features', 'increase in', 'number of recognized cases')],\n",
       "  [('rate', 'be in', 'δ')],\n",
       "  [('4 %', 'rather than mortality be', 'δ')],\n",
       "  [('612 diagnosed cases', 'be with', 'l =')],\n",
       "  [('our model', 'overestimate', 'size of outbreak')],\n",
       "  [('situation', 'be in', 'Toronto'), ('outbreak', 'be', 'contained')],\n",
       "  [('R. M. May', 'disease of', 'Humans')],\n",
       "  [('age structure', 'with model be', 'J. Math'),\n",
       "   ('Epidemiological models', 'be with', 'age structure')],\n",
       "  [('NumericCitation CDC', 'centers for', 'Disease Control')],\n",
       "  [('p', 'be measure of', 'reduced susceptibility to SARS in class S2'),\n",
       "   ('reduced susceptibility', 'be in', 'class S2')],\n",
       "  [('I', 'denote', 'individuals')],\n",
       "  [('rate δ', 'be', \"SARS ' disease-induced mortality\")],\n",
       "  [('comparison', 'be with', 'epidemiological statistics'),\n",
       "   ('it', 'track', 'total number of diagnosed individuals')],\n",
       "  [('March 31', 'from number be', 'Hong Kong'),\n",
       "   ('March 31', 'from number be', 'Ontario'),\n",
       "   ('World', 'for number be', 'Ontario'),\n",
       "   ('cumulative number', 'all of', 'Canada'),\n",
       "   ('World', 'for number be', 'all of Canada'),\n",
       "   ('March 31', 'from number be', 'top row'),\n",
       "   ('World', 'for number be', 'top row'),\n",
       "   ('SARS cases', 'of number be', 'Ontario'),\n",
       "   ('SARS cases', 'of number be', 'all'),\n",
       "   ('March 31', 'from number be', 'Canada'),\n",
       "   ('Canada', 'of all be', 'third row'),\n",
       "   ('SARS cases', 'of number be', 'Canada'),\n",
       "   ('SARS cases', 'of number be', 'Hong Kong'),\n",
       "   ('World', 'for number be', 'Canada'),\n",
       "   ('March 31', 'from number be', 'all of Canada'),\n",
       "   ('World', 'for number be', 'Hong Kong'),\n",
       "   ('SARS cases', 'of number be', 'top row')],\n",
       "  [('inclusion', 'explain', 'jump in data for Ontario on March 31st'),\n",
       "   ('jump', 'be in', 'data for Ontario')],\n",
       "  [('cumulative number', 'be in', 'Ontario'),\n",
       "   ('March 31st', 'day of', 'jump')],\n",
       "  [('l =', 'with time be', 'SEIJR model'),\n",
       "   ('Cumulative number', 'be in', 'Hong Kong'),\n",
       "   ('time', 'be with', 'l =')],\n",
       "  [('Singapore', 'have', 'Table 1')],\n",
       "  [('data', 'start at_time', 'March 31')]],\n",
       " 62: None,\n",
       " 63: [[('6 School', 'academy of', 'Arts'),\n",
       "   ('susceptible-infected-recovered epidemic model M. Saeedian ,1 M. Khalighi ,1 N. Azimi-Tafreshi ,2 G. R. Jafari ,1',\n",
       "    'institute for',\n",
       "    'Advanced Studies 3School of Biological Sciences'),\n",
       "   ('M. Ausloos5 6', '1department of', 'Physics'),\n",
       "   ('Network Science', 'for ,1 be', '45195-1159 Zanjan'),\n",
       "   ('Management', 'of school be', 'LE1 7RH'),\n",
       "   ('Management', 'of school be', 'Royal Netherlands Academy'),\n",
       "   ('Network Science', 'for ,1 be', 'Institute'),\n",
       "   ('4Center Institute', 'be in', 'Fundamental Sciences'),\n",
       "   ('Management', 'of school be', 'University Road'),\n",
       "   ('Biological Sciences', 'institute for', 'Research'),\n",
       "   ('Fundamental Sciences', 'be in', 'Basic Sciences'),\n",
       "   ('Management', 'of school be', 'United Kingdom 7 eHumanities group'),\n",
       "   ('Biological Sciences', 'institute in', 'Fundamental Sciences'),\n",
       "   ('Management', 'of school be', 'Leicester'),\n",
       "   ('6 School', 'university of', 'Leicester'),\n",
       "   ('Management', 'of school be', 'Joan Muyskenweg 25')],\n",
       "  [('memory eﬀects',\n",
       "    'be in',\n",
       "    'susceptible-infected-recovered SIR epidemic model')],\n",
       "  [('memory prone SIR model dynamics', 'use', 'fractional derivatives')],\n",
       "  [('fractional derivatives',\n",
       "    'be in',\n",
       "    'corresponding nonlinear fractional diﬀerential evolutio 7 1 0 2')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('2 exact epidemic thresholds', 'be in', 'Markovian epidemic models')],\n",
       "  [('their model',\n",
       "    'fall into',\n",
       "    'one of three universal classes due to behavior of ﬁxed point curves')],\n",
       "  [('infection', 'be in', 'which')],\n",
       "  [('epidemic threshold', 'above c be', 'individuals')],\n",
       "  [('size', 'be in', 'fact')],\n",
       "  [('classical Markov process', 'in δ be', 't − t ′'),\n",
       "   ('delta function δ', 'be in', 'classical Markov process')],\n",
       "  [('us', 'rewrite', 'Eqs')],\n",
       "  [('fractional diﬀerential equations', 'be with', 'Caputo-type derivative')],\n",
       "  [('sides', 'on − be', 'α − 1')],\n",
       "  [('following fractional differential equations',\n",
       "    'can',\n",
       "    'can obtained for SIR model'),\n",
       "   ('memory eﬀects', 'be in', 'dynamical systems')],\n",
       "  [('system', 'tend toward', 'memoryless system')],\n",
       "  [('we', 'note', 'that'), ('we', 'note', 'that for simplicity')],\n",
       "  [('taking', 'take into', 'account diﬀerent time scales')],\n",
       "  [('growth rate', 'be in', 'Eq')],\n",
       "  [('6', 'be', 'positive')],\n",
       "  [('it', 'be reemphasize', 'Eqs')],\n",
       "  [('we', 'solve', 'equations')],\n",
       "  [('y', 'of each be', '0 T'),\n",
       "   ('interval', 'on each be', 'i'),\n",
       "   ('interval', 'on each be', '0 T'),\n",
       "   ('y', 'of each be', 'i')],\n",
       "  [('n', '=', '0')],\n",
       "  [('coeﬃcients', 'thereby solve', 'Eq')],\n",
       "  [('us', 'consider successively', 'two pertinent aspects'),\n",
       "   ('us', 'consider', 'two pertinent aspects')],\n",
       "  [('memory eﬀects', 'be with', 'memoryless case'),\n",
       "   ('us',\n",
       "    'compare',\n",
       "    'evolution of system including memory eﬀects with memoryless case')],\n",
       "  [('t', '=', '20 200 2000')],\n",
       "  [('higher values', 'be with', 'decreasing α')],\n",
       "  [('numbers', 'be present as', '10-base exponential notation')],\n",
       "  [('epidemic size', 'get', 'non-zero value')],\n",
       "  [('system', 'decrease', 'value of α including memory')],\n",
       "  [('it', 'can', 'can seen in Fig. 1')],\n",
       "  [('model', 'be with', 'memory'),\n",
       "   ('memory eﬀects',\n",
       "    'be of',\n",
       "    'model with memory for diﬀerent values of ﬁnite time t')],\n",
       "  [('memory eﬀects decay', 'be in', 'time like power-law function')],\n",
       "  [('variation', 'be with', 'increasing ﬁnite time')],\n",
       "  [('whence more individuals', 'become', 'infected')],\n",
       "  [('numbers', 'be present as', 'base 10 exponential notation')],\n",
       "  [('period', 'be in', 'past')],\n",
       "  [('individuals', 'have about', 'disease')],\n",
       "  [('network', 'be with', 'degree exponent λ =')],\n",
       "  [('scale-free network', 'be with', 'degree exponent λ'),\n",
       "   ('SIR model =', 'for size be', '1 − St')],\n",
       "  [('network', 'be with', 'degree exponent λ =')],\n",
       "  [('memory eﬀects', 'be with', 'diﬀerent values of α')],\n",
       "  [('stationary time', 'decrease', 'value of α')],\n",
       "  [('Markov epidemic spreading', 'be with', 'λ =')],\n",
       "  [('epidemic decreases', 'be with', 'decreasing α')],\n",
       "  [('networks', 'be with', '2 λ')],\n",
       "  [('we', 'use', 'fractional calculus technique')],\n",
       "  [('we', 'compare', 'Eq')],\n",
       "  [('A2', 'be with', 'its counterpart xα t'),\n",
       "   ('interesting references', 'be in', 'NumericCitation')],\n",
       "  [('time scales', 'be in', 'fractional-and integer-order dynamics'),\n",
       "   ('we',\n",
       "    'here compare',\n",
       "    'time scales in fractional-and integer-order dynamics')],\n",
       "  [('ch .4', 'be in', 'Climate change'),\n",
       "   ('changing climate', 'under disease be', 'Ed'),\n",
       "   ('2008 NumericCitation A. Legr', 'disease under', 'changing climate')],\n",
       "  [('M. Bogu˜n', None, 'Phys')],\n",
       "  [('M.', 'serrano be', 'Phys')],\n",
       "  [('Physicists', 'for introduction be', 'Citation')],\n",
       "  [('B. West', 'application of', 'fractional calculus in physics'),\n",
       "   ('fractional calculus', 'be in', 'physics')],\n",
       "  [('J. J. Trujillo', 'theory of', 'fractional diﬀerential equations')],\n",
       "  [('G. R. Jafari', 'journal of', 'Artiﬁcial Society')],\n",
       "  [('NumericCitation K. Diethelm',\n",
       "    'subroutine for',\n",
       "    'numerical solution of diﬀ Citation S. Heinzel')],\n",
       "  [('Epidemics', 'be in', 'Ch2olera'),\n",
       "   ('Hamilton', 'have', 'Forgotten Epidemics')]],\n",
       " 64: [[('model',\n",
       "    'reproduce',\n",
       "    'qualitative features of random disease propagation observed in true data')],\n",
       "  [('scale-free phenomena',\n",
       "    'be',\n",
       "    'Unlike traditional deterministic models observed in epidemic network')],\n",
       "  [('299', 'die', 'NumericCitation')],\n",
       "  [('its mortality rate', 'be', 'fortunately NumericCitation')],\n",
       "  [('current epidemiological models', 'be of', 'Generally two types'),\n",
       "   ('current models', 'be of', 'Generally two types')],\n",
       "  [('SW model', 'arouse', 'new wave of epidemiological research')],\n",
       "  [('Our work', 'model', 'SARS data'),\n",
       "   ('Our work', 'model', 'SARS data for HK')],\n",
       "  [('SIR model', 'be with', 'ﬁxed population')],\n",
       "  [('Quick action', 'be in', 'early stage')],\n",
       "  [('SIR model', 'in model be', 'ﬁxed population N'),\n",
       "   ('I. SUSCEPTIBLE-INFECTED-RECOVERED MODEL', 'be in', 'SIR model')],\n",
       "  [('following ﬂow chart',\n",
       "    'show',\n",
       "    'basic procession of SIR model NumericCitation')],\n",
       "  [('outbreak', 'be in', 'England')],\n",
       "  [('Population N', '=', '7.3 × 106'), ('Si', '=', 'N')],\n",
       "  [('value', 'be', 'used for following prediction')],\n",
       "  [('thick red curve', 'show clearly', 'bottom of sharp valley')],\n",
       "  [('stage', 'of day be', 'March 20 2003')],\n",
       "  [('prediction', 'plot in', 'Fig. 2')],\n",
       "  [('best ﬁt', 'be also process weekly for', 'detail discussion later')],\n",
       "  [('3 stages', 'be in', 'Fig. 2')],\n",
       "  [('infectious SARS coronavirus spread',\n",
       "    'be in',\n",
       "    'quickly public with few protections')],\n",
       "  [('new transmission mode explosive growth',\n",
       "    'be in',\n",
       "    'contrast to transmission'),\n",
       "   ('transmission',\n",
       "    'be in',\n",
       "    '-20 0 20 40 60 80 100 0 20 40 60\\xa080\\xa0100\\xa0120 SARS data for HK revised data')],\n",
       "  [('stage', 'have', '15 days')],\n",
       "  [('black dots', 'curve of be', 'revised data')],\n",
       "  [('we', 'refer as', 'burning growth')],\n",
       "  [('high-density housing estate', 'be in', 'Hong Kong'),\n",
       "   ('infections', 'be in', 'full epidemic')],\n",
       "  [('intimate contacts', 'be with', 'SARS patient'),\n",
       "   ('intimate contacts', 'lead to', 'infection NumericCitation')],\n",
       "  [('control', 'be in', 'second stage')],\n",
       "  [('increasing trend', 'be in', 'ﬁrst stage')],\n",
       "  [('it', 'be', '15.9 %')],\n",
       "  [('270 fatalities', 'be in', 'total'),\n",
       "   ('we', 'therefore predict', 'approximately 270 fatalities'),\n",
       "   ('we', 'therefore predict', 'approximately 270 fatalities in total')],\n",
       "  [('triangle curve', 'be in', 'Fig. 2 reﬁnes')],\n",
       "  [('numbers', 'be close to', 'very true data NumericCitation')],\n",
       "  [('SARS', 'of spread be', 'acquisition')],\n",
       "  [('signiﬁcant statistical ﬂuctuations', 'be in', 'data')],\n",
       "  [('diseases', 'be in', 'epidemiology research')],\n",
       "  [('removal rate', 'exceed', 'infection rate r')],\n",
       "  [('squares', 'be in', 'also Fig. 3'),\n",
       "   ('evaluations',\n",
       "    'be with',\n",
       "    'dash line estimated weekly as squares also in Fig. 3')],\n",
       "  [('Two distinct methods',\n",
       "    'be describe for',\n",
       "    'disease propagation dynamics')],\n",
       "  [('explosive mode',\n",
       "    'particularly be much more hazardous than',\n",
       "    'burning mode')],\n",
       "  [('method', 'be widely apply to', 'various epidemiological data')],\n",
       "  [('circles', 'be for', '3-stage analysis mentioned in Fig. 2'),\n",
       "   ('3-stage analysis', 'mention in', 'Fig. 2')],\n",
       "  [('epidemic chain', 'model of', 'social contacts')],\n",
       "  [('m', '=', '2700')],\n",
       "  [('two long range contacts', 'be with', '2 other nodes')],\n",
       "  [('its', 'contact', 'value of nodes linked'),\n",
       "   ('its short range contacts', 'value of', 'nodes linked')],\n",
       "  [('change', 'be', 'irreversible')],\n",
       "  [('SARS patient', 'be in', 'hospital')],\n",
       "  [('infected indviduals', 'be highly infectious during', 'time')],\n",
       "  [('true variation', 'be in', 'control strategy')],\n",
       "  [('geographical map', 'be with', '100 × 100 nodes =')],\n",
       "  [('clusteringly scatter', 'be in', 'map')],\n",
       "  [('clusters', 'show', 'eﬀect from nearest neighbour connections')],\n",
       "  [('it', 'compare with', 'complete infection epidemic')],\n",
       "  [('small system', 'be with', '100 × 100 nodes')],\n",
       "  [('clusters', 'be with', 'infected nodes')],\n",
       "  [('It', 'contain', 'infection facts in ﬁrst 45 days')],\n",
       "  [('true SARS infection distribution', 'be in', 'HK')],\n",
       "  [('epidemic chain', 'record', 'infection fact from seed to last patient')],\n",
       "  [('bad node', 'infect', 'it')],\n",
       "  [('rich-get-richer',\n",
       "    'widely accept preferential attachment of be',\n",
       "    'good answer NumericCitation'),\n",
       "   ('attachment', 'be', 'good')],\n",
       "  [('linear preference function', 'be', 'applied here')],\n",
       "  [('chain', 'be', 'scale-invariant'),\n",
       "   ('chain', 'be', 'scale-invariant SW network')],\n",
       "  [('distribution', 'be in', 'log-log'), ('distribution', 'be in', 'Fig. 5')],\n",
       "  [('system', 'be with', '100 × 100 nodes'),\n",
       "   ('hollow circle', 'be for', 'system with 100 × 100 nodes')],\n",
       "  [('piecewise linear', 'be in', 'b'), ('power law', 'be in', 'a')],\n",
       "  [('we', 'conﬁrm', 'this'),\n",
       "   ('same ﬁxed p1 ,2 rest curves', 'be in', 'Figs. 5')],\n",
       "  [('solid square curve', 'be distribuin', 'tion of whole epidemic network')],\n",
       "  [('correlation coeﬃcient R', 'be in', 'Fig. 5')],\n",
       "  [('piecewise linear ﬁts', 'have r in', 'b')],\n",
       "  [('hollow triangle', 'be in', 'Figs. 5')],\n",
       "  [('1 %', 'stage of', 'full infection epidemic'),\n",
       "   ('linear ﬁt', 'be in', 'a')],\n",
       "  [('problem', 'answer correctly in', 'simulations')],\n",
       "  [('simulations',\n",
       "    'most closely match',\n",
       "    'qualitative features of observed data')],\n",
       "  [('above full infection', 'be with', 'ﬁxed parameters')],\n",
       "  [('geographical map', 'be with', '100 × 100 nodes'),\n",
       "   ('circle', 'be', 'epidemic'),\n",
       "   ('circle', 'be', 'full'),\n",
       "   ('circle',\n",
       "    'be for',\n",
       "    'full infection epidemic network simulated in geographical map with 100 × 100 nodes')],\n",
       "  [('curves', 'be for', 'one with 1000 × 1000 nodes')],\n",
       "  [('two dash lines',\n",
       "    'be piecewise linear ﬁts for',\n",
       "    'full infection process with 1000 × 1000 nodes'),\n",
       "   ('full infection process', 'be with', '1000 × 1000 nodes')],\n",
       "  [('two piecewise linear parts', 'be in', 'log-linear graph')],\n",
       "  [('infection', 'be in', 'early stage')],\n",
       "  [('moment', 'be in', 'other words')],\n",
       "  [('long range infections', 'be in', 'model')],\n",
       "  [('SARS cases', 'be in', 'HK')],\n",
       "  [('curve', 'be in', 'log-log diagram'),\n",
       "   ('linear ﬁt correlation coeﬃcient R', 'give =', '− 0.989')],\n",
       "  [('model', 'be with', '2700 × 2700 nodes'),\n",
       "   ('b o i l i t y p 1', 'number from', 'March 12 2003'),\n",
       "   ('b o i l i t y p 1', 'number of', 'days')],\n",
       "  [('model', 'in curve be', 'black dots'),\n",
       "   ('simulated epidemic curve', 'be in', 'model'),\n",
       "   ('HK', 'for datum be', 'grey column')],\n",
       "  [('distribution', 'be in', 'simulations'),\n",
       "   ('simulations', 'be in', 'two dimensional map')],\n",
       "  [('early spreading', 'be in', 'Singapore NumericCitation'),\n",
       "   ('deﬁnite SW structure', 'be with', 'small number of nodes'),\n",
       "   ('nodes', 'be with', 'large number of links')],\n",
       "  [('character', 'be also veriﬁed in', 'our model')],\n",
       "  [('index node', 'clearly have', 'many long range linkages')],\n",
       "  [('scaling behaviour', 'be in', 'log-log coordinates'),\n",
       "   ('simulated epidemic', 'for behaviour be', 'the scaling')],\n",
       "  [('duration', 'be with', 'high probability p1 ,2')],\n",
       "  [('points', 'be in', 'curves'),\n",
       "   ('curves', 'be in', 'log-linear diagrams of Figs. 8')],\n",
       "  [('All', 'be', 'located'),\n",
       "   ('All', 'be located near', 'linkages number of 6-7')],\n",
       "  [('general problem', 'even embed', 'geographical map NumericCitation')],\n",
       "  [('More work', 'address', 'issue')],\n",
       "  [('simulations', 'be with', 'full infection data')]],\n",
       " 65: [[('individual', 'have', 'infectiousness')],\n",
       "  [('we',\n",
       "    'finally analyze',\n",
       "    'transition from stochastic to deterministic phase')],\n",
       "  [('1 Introduction Infectious diseases', 'impact', 'public health')],\n",
       "  [('SARS', 'emergence of', 'H1N1 swine inﬂuenza')],\n",
       "  [('changes', 'be in', 'S/N'),\n",
       "   ('changes', 'be in', 'N/S'),\n",
       "   ('changes', 'be in', 'N/S'),\n",
       "   ('we will couch', 'be in', 'terms of changes')],\n",
       "  [('cumulative distribution', 'be in', 'ﬁgure 1')],\n",
       "  [('distributions',\n",
       "    'converge to',\n",
       "    'steady-state distribution as t increases')],\n",
       "  [('several stages', 'be in', 'paper')],\n",
       "  [('epidemics', 'be', 'possible')],\n",
       "  [('We', 'consider', 'only supercritical outbreaks')],\n",
       "  [('CrumpModeJagers', 'process', 'NumericCitation')],\n",
       "  [('C', 'be', 'random')],\n",
       "  [('process', 'be', 'memoryless')],\n",
       "  [('individual', 'of', 'infection')],\n",
       "  [('Ignoring age-of-infection eﬀects', 'lose', 'important details'),\n",
       "   ('Ignoring age-of-infection eﬀects', 'lose', 'details')],\n",
       "  [('infections', 'be bias towards', 'young infection ages')],\n",
       "  [('Similar observations', 'hold during', 'decay')],\n",
       "  [('Several approaches', 'have', 'have developed'),\n",
       "   ('Several approaches', 'study', 'age-of-infection models'),\n",
       "   ('Several approaches', 'study', 'models')],\n",
       "  [('Some', 'explicitly track', 'history of epidemic NumericCitation')],\n",
       "  [('epidemic', 'have', 'growth'),\n",
       "   ('equations', 'govern', \"epidemic 's growth\")],\n",
       "  [('infectiousness', 'be independent of', 'infection duration'),\n",
       "   ('infectiousness', 'be', 'independent')],\n",
       "  [('P', '=', '0'), ('no-one', 'remain', 'infectious forever')],\n",
       "  [('P', 'be', 'diﬀerentiable')],\n",
       "  [('We', 'have', 'full derivations of equations in appendix A'),\n",
       "   ('equations', 'be in', 'appendix A')],\n",
       "  [('k individuals', 'be infect at', 'time t'),\n",
       "   ('k individuals', 'be', 'infected')],\n",
       "  [('We', 'describe in', 'appendix B')],\n",
       "  [('50 000 simulations', 'be in', 'ﬁgure 2'),\n",
       "   ('We', 'compare solution with', '50 000 simulations in ﬁgure 2')],\n",
       "  [('infection duration', 'of function be', 'pdf')],\n",
       "  [('close match', 'be with', 'analytic results'),\n",
       "   ('considerable noise', 'be in', 'simulations')],\n",
       "  [('I', 'even become', 'much larger than 1')],\n",
       "  [('we', 'derive in', '§ 3'),\n",
       "   ('N/S', 'remain', 'ﬁxed'),\n",
       "   ('C', 'be', 'random')],\n",
       "  [('solution φ', 'know as', 'Malthusian parameter')],\n",
       "  [('we', 'require for', 'convergence')],\n",
       "  [('growth', 'be', 'deterministic')],\n",
       "  [('outbreak', 'be', 'Once suﬃciently large')],\n",
       "  [('population', 'be with', 'constant infectiousness')],\n",
       "  [('we', 'underpredict', 'current size')],\n",
       "  [('outbreaks', 'be with', 'only a few infections')],\n",
       "  [('eﬀects', 'be', 'once unimportant'),\n",
       "   ('3 Deterministic Phase τ', 'be in', 'section'),\n",
       "   ('stochastic eﬀects', 'be', 'once unimportant')],\n",
       "  [('We',\n",
       "    'discuss',\n",
       "    'transition from stochastic phase to deterministic phase in § 4'),\n",
       "   ('transition', 'be in', '§ 4')],\n",
       "  [('infection', 'after β be', 'τ'), ('average rate', 'as β be', 'τ')],\n",
       "  [('N/S', 'be', 'constant')],\n",
       "  [('exponential growth', 'be in', 'time φ')],\n",
       "  [('our approach', 'be with', 'changing N/S')],\n",
       "  [('I', '=', 'F G'), ('coeﬃcients change', 'be in', 'time')],\n",
       "  [('similarity', 'be in', 'notation')],\n",
       "  [('change', 'be in', 'S')],\n",
       "  [('changes', 'be in', 'N/S'),\n",
       "   ('equilibrium', 'be with', 'current value of N/S')],\n",
       "  [('us', 'create', 'equations')],\n",
       "  [('equations',\n",
       "    'system of be',\n",
       "    'Computationally only mildly more diﬃcult than standard SIR equations')],\n",
       "  [('that', 'be', 'Once done')],\n",
       "  [('simulations', 'be with', 'exact age-of-infection model')],\n",
       "  [('diﬀerence', 'be in', 'initial condition'),\n",
       "   ('temporal shift', 'be result of', 'diﬀerence in initial condition')],\n",
       "  [('This', 'hold for', '10'), ('our', 'system', '10')],\n",
       "  [('We', 'take', 'τ')],\n",
       "  [('they', 'become once more', 'infectious')],\n",
       "  [('individuals', 'possibly have', 'symptoms'),\n",
       "   ('individuals', 'have', 'symptoms')],\n",
       "  [('stochastic shifts', 'be in', 'time')],\n",
       "  [('diﬀerence', 'be in', 'timing β')],\n",
       "  [('gamma distributed recovery time', 'be with', 'constant infectiousness'),\n",
       "   ('exact system diﬀers', 'be in', 'only time shifts')],\n",
       "  [('diﬀerences', 'be in', 'initial conditions')],\n",
       "  [('change', 'be in', 'N/S'), ('model', 'be', 'good')],\n",
       "  [('epidemic course', 'use', '1')],\n",
       "  [('R0', 'duration of', 'infection'), ('We', 'can identify', 'constraints')],\n",
       "  [('gamma-distributed infectious period',\n",
       "    'be with',\n",
       "    'constant infectiousness'),\n",
       "   ('They',\n",
       "    'consider',\n",
       "    'gamma-distributed infectious period with constant infectiousness')],\n",
       "  [('n', '=', '100')],\n",
       "  [('us', 'deﬁne coeﬃcients for', 'our approximation')],\n",
       "  [('epidemic', 'timing of', 'epidemic')],\n",
       "  [('epidemic', 'be', 'once suﬃciently large')],\n",
       "  [('simplicity', 'be in', 'our analysis')],\n",
       "  [('us', 'assume', 'that')],\n",
       "  [('recoveries', 'happen in', 'interval')],\n",
       "  [('diﬀerence', 'be', 'Skellam'), ('rates', 'remain', 'constant')],\n",
       "  [('11 error', 'be in', 'number of new infections')],\n",
       "  [('we', 'use again', '17')],\n",
       "  [('system', 'be', 'deterministic')],\n",
       "  [('we', 'occur at', 'slowly changing rate')],\n",
       "  [('extinct', 'be strongly inﬂuenced by', 'stochastic eﬀects'),\n",
       "   ('extinct', 'be inﬂuenced by', 'stochastic eﬀects')],\n",
       "  [('outbreak', 'once become', 'large')],\n",
       "  [('many events', 'happen in', 'interval')],\n",
       "  [('outbreak', 'be', 'Once deterministic')],\n",
       "  [('they', 'save', 'history of epidemic'),\n",
       "   ('it', 'use', 'models'),\n",
       "   ('it', 'use', 'approximate models')],\n",
       "  [('changes', 'be in', 'susceptible fraction'),\n",
       "   ('system', 'respond adiabatically to', 'changes in susceptible fraction')],\n",
       "  [('It', 'deviate close to', 'peak')],\n",
       "  [('it',\n",
       "    'look at',\n",
       "    'how quickly susceptible fraction changes during typical infectious period')],\n",
       "  [('generation interval distribution', 'be close to', 'when discrete'),\n",
       "   ('it', 'reach', 'equilibrium')],\n",
       "  [('SIR', 'be with', 'birth')]],\n",
       " 66: [[('Stochastic eﬀects',\n",
       "    'be in',\n",
       "    'seasonally forced epidemic model G. Rozhnova1 2')],\n",
       "  [('model', 'be in', 'thermodynamic limit')],\n",
       "  [('Size eﬀects', 'be study through', 'additional simulations'),\n",
       "   ('Size eﬀects', 'be', 'studied')],\n",
       "  [('Gg', 'be', 'complex'),\n",
       "   ('Gg', 'spread in', 'human populations'),\n",
       "   ('05.45.-a I. INTRODUCTION Epidemic spread', 'be in', 'human populations')],\n",
       "  [('structure', 'interplay between', 'immune response')],\n",
       "  [('data records', 'be', 'available'),\n",
       "   ('similarities', 'be in', 'contagion mechanisms'),\n",
       "   ('similarities', 'be in', 'infectious waning NumericCitation')],\n",
       "  [('SEIR compartmental model', 'be with', 'periodic forcing'),\n",
       "   ('approach', 'be', 'compartmental')],\n",
       "  [('system',\n",
       "    'successfully reproduce',\n",
       "    'main features of observed time series for measles incidence')],\n",
       "  [('noisy oscillations',\n",
       "    'be with',\n",
       "    'same frequency as damped oscillations of deterministic system')],\n",
       "  [('several recent analytical studies', 'deal with', 'unforced systems'),\n",
       "   ('all deal', 'be with', 'unforced systems'),\n",
       "   ('stochastic eﬀects', 'play', 'more fundamental role')],\n",
       "  [('van Kampen', 'have', 'system size expansion')],\n",
       "  [('correlations', 'be in', 'un-forced epidemic models')],\n",
       "  [('system', 'have', 'nonlinearities'),\n",
       "   ('stochastic effects', 'be in', 'childhood diseases epidemiology')],\n",
       "  [('method', 'develop in', 'Ref')],\n",
       "  [('forced SEIR model',\n",
       "    'be in',\n",
       "    'realistic parameter region for childhood infectious diseases')],\n",
       "  [('free parameter', 'be in', 'our study')],\n",
       "  [('SEASONALLY FORCED DETERMINISTIC SEIR EPIDEMIC MODEL',\n",
       "    'be in',\n",
       "    'section')],\n",
       "  [('mathematical literature',\n",
       "    'be in',\n",
       "    \"connection with acute infectious diseases ' modeling\"),\n",
       "   ('connection', 'be with', \"acute infectious diseases ' modeling\")],\n",
       "  [('higher order nonlinearities', 'be in', 'infection term'),\n",
       "   ('infection term', 'in nonlinearity be', 'immigration of infectives'),\n",
       "   ('higher order nonlinearities', 'immigration of', 'infectives')],\n",
       "  [('contact rate', 'incorporate seasonality in', 'transmission of infection'),\n",
       "   ('rate', 'be', 'periodic'),\n",
       "   ('rate', 'be', 'sinusoidal'),\n",
       "   ('periodic sinusoidal function', 'be with', 'period =')],\n",
       "  [('periodic change', 'be in', 'disease transmission')],\n",
       "  [('1/χ', 'hence be average latent period of', 'disease')],\n",
       "  [('average lifespan', 'be give by', '1/µ')],\n",
       "  [('susceptible population', 'be in', 'one infectious period')],\n",
       "  [('stability analysis', 'show', 'Eqs')],\n",
       "  [('we', 'use', 'parameter values given in Refs')],\n",
       "  [('ﬁxed set', 'for model be', '4'), ('ﬁxed set', 'for model be', '2')],\n",
       "  [('we', 'recommend ref for', 'interested reader')],\n",
       "  [('Table I', 'summary of', 'bifurcation diagram')],\n",
       "  [('it', 'be', 'hard')],\n",
       "  [('three main stable attractors', 'cycle of', 'period 1 2')],\n",
       "  [('solid curves', 'be compute by', 'numerical integration of Eqs')],\n",
       "  [('initial conditions', 'so avoid', 'transients'),\n",
       "   ('conditions', 'so avoid', 'transients')],\n",
       "  [('unforced SEIR model', 'of equilibrium be', '9')],\n",
       "  [('β1 =', 'be in', 'Fig. 1')],\n",
       "  [('13 recovered individuals', 'be', 'permanently immune')],\n",
       "  [('corresponding transition rates', 'be', 'diﬀerent')],\n",
       "  [('system', 'be in', 'state')],\n",
       "  [('term', 'be with', 'm equal to m ′')],\n",
       "  [('we',\n",
       "    'get',\n",
       "    'insight into dynamics of seasonally forced stochastic SEIR model for large system sizes')],\n",
       "  [('we', 'will expand', 'Eq')],\n",
       "  [('diﬀerent orders', 'be in', 'N')],\n",
       "  [('terms', 'yield', 'Eqs')],\n",
       "  [('periodic functions', 'be with', 'same period T of limit cycle')],\n",
       "  [('Gaussian noise terms', 'be with', 'zero mean hLj i')],\n",
       "  [('one', 'ﬁnd', 'solution of Eq')],\n",
       "  [('eﬀects',\n",
       "    'be in',\n",
       "    'nonlinear oscillations close bifurcations NumericCitation')],\n",
       "  [('general solution', 'be with', 'matrix function A')],\n",
       "  [('A periodic', 'with system be', '40'),\n",
       "   ('homogeneous system', 'be with', 'A periodic')],\n",
       "  [('X', 'be', 'fundamental')],\n",
       "  [('stochastic ﬂuctuations', 'of spectrum be', 'power spectral density')],\n",
       "  [('we',\n",
       "    'will compare',\n",
       "    'power spectra calculated analytically through method described above with those measured from simulations of stochastic process deﬁned by Eqs')],\n",
       "  [('explicit time dependence', 'be in', 'transition rates')],\n",
       "  [('other parameters', 'ﬁxed as', 'as Table I.')],\n",
       "  [('comparison', 'be with', 'prevaccination era')],\n",
       "  [('ﬁnal results', 'be', 'obtained numerically')],\n",
       "  [('unforced SEIR deterministic model', 'of jacobian be', '5')],\n",
       "  [('NumericCitation', 'be in', 'same parameter region'),\n",
       "   ('This', 'be reﬂected in', '2')],\n",
       "  [('complex conjugate', 'be in', 'Table')],\n",
       "  [('working precision', 'typically set up in', 'integrations')],\n",
       "  [('immediate comparison', 'be with', 'power spectra of simulations')],\n",
       "  [('deterministic perturbations decay', 'be in', 'damped oscillatory way')],\n",
       "  [('resonant ampliﬁcation', 'be observe', 'NumericCitation'),\n",
       "   ('unforced systems', 'be with', 'stable focus')],\n",
       "  [('we', 'validate', 'theory developed in study')],\n",
       "  [('excellent agreement', 'be in', 'cases'), ('we', 'be', 'lead')],\n",
       "  [('it', 'have', 'has noticed')],\n",
       "  [('lower panels', 'show', 'corresponding power spectra')],\n",
       "  [('vertical helper lines', 'mark', 'frequencies predicted by Eq')],\n",
       "  [('plus', 'be in', 'equation'), ('equation', 'in plus be', 'minus')],\n",
       "  [('T', 'be period of', 'limit cycle')],\n",
       "  [('1/2 − Im ρ1 ,2', 'at peak be', '2π')],\n",
       "  [('plots',\n",
       "    'be',\n",
       "    'done in lin-log scale for better observation of structure of power spectra')],\n",
       "  [('We',\n",
       "    'consider',\n",
       "    'case of annual limit cycle induced by low forcing amplitude')],\n",
       "  [('realizations', 'obtain', 'curve')],\n",
       "  [('β1', '=', '0.02'),\n",
       "   ('forcing amplitude', 'correspond to', 'annual limit cycle')],\n",
       "  [('vertical helper lines', 'mark', 'frequencies predicted by Eq')],\n",
       "  [('second', 'be situate at', '2π')],\n",
       "  [('eﬀect',\n",
       "    'have',\n",
       "    'has already observed before in spectra of ﬂuctuations around stable node in unforced systems NumericCitation')],\n",
       "  [('spectra',\n",
       "    'be',\n",
       "    'fully described by theory only for very large populations')],\n",
       "  [('change', 'be in', 'relative amplitude of main stochastic peaks')],\n",
       "  [('main stochastic peak', 'be', 'where signiﬁcantly namics'),\n",
       "   ('noisier dyfor N =', 'be in', 'Fig. 3')],\n",
       "  [('vertical helper lines', 'mark', 'frequencies predicted by Eq')],\n",
       "  [('changes', 'be in', 'spectra for diﬀerent population sizes')],\n",
       "  [('stochastic peaks',\n",
       "    'be',\n",
       "    'For largest system size predicted correctly by theory')],\n",
       "  [('ma jor stochastic peak', 'be now located at', '2π')],\n",
       "  [('vertical helper lines', 'mark', 'frequencies predicted by Eq')],\n",
       "  [('we', 'will go back to', 'it'), ('we', 'will go back later to', 'it')],\n",
       "  [('two close values', 'be in', 'vicinity of deterministic period'),\n",
       "   ('β1 =', 'correspond to', 'stable annual limit cycle')],\n",
       "  [('transition', 'show up later in', 'stochastic system')],\n",
       "  [('stable annual limit cycles', 'be in', 'deterministic model'),\n",
       "   ('β1', '=', '0.1')],\n",
       "  [('bifurcation point', 'be in', 'simulations')],\n",
       "  [('for literature NumericCitation', 'be in', 'particular')],\n",
       "  [('indication', 'be in', 'stochastic system')],\n",
       "  [('other set', 'start from', 'random initial condition'),\n",
       "   ('it', 'will become later', 'clear')],\n",
       "  [('left panel', 'be in', 'Fig. 7'),\n",
       "   ('helper lines', 'mark', 'frequencies for triennial limit cycle')],\n",
       "  [('density',\n",
       "    'be in',\n",
       "    'typical single realization of stochastic model starting from triennial limit cycle')],\n",
       "  [('Their characteristic features',\n",
       "    'very low number of',\n",
       "    'infectives between recurrent epidemics'),\n",
       "   ('features', 'be', 'high'),\n",
       "   ('Their features',\n",
       "    'very low number of',\n",
       "    'infectives between recurrent epidemics')],\n",
       "  [('large population sizes',\n",
       "    'be with',\n",
       "    'ever increasing role of stochastic ﬂuctuations as N decreases')],\n",
       "  [('DISCUSSION', 'be in', 'conclusion')],\n",
       "  [('van Kampen',\n",
       "    'have',\n",
       "    'expansion of master equation of corresponding stochastic model')],\n",
       "  [('analytical results',\n",
       "    'be with',\n",
       "    'those obtained from direct simulations of stochastic model'),\n",
       "   ('We',\n",
       "    'have compare',\n",
       "    'analytical results with those obtained from direct simulations of stochastic model')],\n",
       "  [('broader peaks',\n",
       "    'be due to',\n",
       "    'resonant ampliﬁcation of stochastic ﬂuctuations around limit cycle')],\n",
       "  [('deterministic peaks',\n",
       "    'be in',\n",
       "    'power spectra obtained from real data records of childhood diseases')],\n",
       "  [('role', 'be in', 'seasonally forced stochastic SEIR models')],\n",
       "  [('irregular dynamics',\n",
       "    'be',\n",
       "    'why observed in corresponding stochastic system')],\n",
       "  [('it', 'be', 'systematic')],\n",
       "  [('3 such switching', 'be absent in', 'stochastic system')],\n",
       "  [('most initial conditions', 'be with', 'very low number of infectives')]],\n",
       " 67: [[('We',\n",
       "    'present',\n",
       "    'mathematical analysis of early detection of Ebola virus')],\n",
       "  [('SIR model', 'be with', 'vital dynamics')],\n",
       "  [('numerical simulations', 'conﬁrm', 'theoretical analysis')],\n",
       "  [('Citation detection',\n",
       "    'parameter of',\n",
       "    'model identiﬁed from World Health Organization data'),\n",
       "   ('Ebola virus', 'be in', 'Guinea'),\n",
       "   ('Ebola virus', 'of detection be', 'parameters')],\n",
       "  [('Introduction Ebola virus',\n",
       "    'be currently aﬀecting',\n",
       "    'several African countries')],\n",
       "  [('ﬁ Citation', 'be in', 'Democratic Republic of Congo near Ebola River')],\n",
       "  [('Ebola outbreaks', 'have appear sporadically in', 'Africa')],\n",
       "  [('symptoms', 'include', 'diarrhoea')],\n",
       "  [('Ebola virus',\n",
       "    'be',\n",
       "    \"transmitted to initial human by contact with infected animal 's body ﬂuid\"),\n",
       "   ('contact', 'be with', \"infected animal 's body ﬂuid\"),\n",
       "   ('infected animal', 'with', 'body ﬂuid')],\n",
       "  [('eyes', 'be with', 'blood of person')],\n",
       "  [('models', 'be', 'powerful')],\n",
       "  [('simplest model', 'be in', 'epidemiology')],\n",
       "  [('available outbreak data',\n",
       "    'obtain',\n",
       "    'estimates for parameters of model NumericCitation')],\n",
       "  [('we', 'describe', 'behaviour of virus')],\n",
       "  [('we',\n",
       "    'here focus on',\n",
       "    'mathematical analysis of early detection of Ebola virus')],\n",
       "  [('we',\n",
       "    'present in',\n",
       "    'our previous study of description of behaviour of Ebola virus NumericCitation')],\n",
       "  [('model', 'be with', 'vital dynamics')],\n",
       "  [('We', 'also consider', 'induced death rate for infected individuals')],\n",
       "  [('number', 'cost of', 'vaccination')],\n",
       "  [('We', 'end with', 'Section 5 of conclusions')],\n",
       "  [('Mathematical formulation', 'be in', 'section'),\n",
       "   ('early detection', 'be in', 'West Africa')],\n",
       "  [('Infectious', 'for i be', 't'),\n",
       "   ('three categories', '• for', 'susceptible'),\n",
       "   ('contact', 'be with', 'other individuals'),\n",
       "   ('contact', 'be with', 'susceptible category')],\n",
       "  [('initial conditions', 'be give by', '0')],\n",
       "  [('induced death rate', 'be in', 'basic SIR model'),\n",
       "   ('model', 'be with', 'vital dynamics')],\n",
       "  [('we', 'consider', 'vital dynamics'), ('we', 'consider', 'dynamics')],\n",
       "  [('we',\n",
       "    'consider',\n",
       "    'induced death rate associated to infected individuals')],\n",
       "  [('Such model', 'be new in', 'Ebola context NumericCitation')],\n",
       "  [('we', 'include', 'demographic eﬀects')],\n",
       "  [('SIR model', 'be with', 'vital dynamics'),\n",
       "   ('vital dynamics', 'with model be', '3.1')],\n",
       "  [('Compartment diagram', 'be with', 'vital dynamics')],\n",
       "  [('propagation dynamics', 'be with', 'death'),\n",
       "   ('model', 'of equation be', '3.1'),\n",
       "   ('we', 'analyse', '3.1')],\n",
       "  [('I', '−', 'γS'),\n",
       "   ('deﬁnition', 'by e be', '3.3'),\n",
       "   ('µI γR', '=', '0'),\n",
       "   ('µI − γR', '=', '0')],\n",
       "  [('equilibrium point', 'be call', 'disease-free equilibrium')],\n",
       "  [('we', 'add', '3.4')],\n",
       "  [('S', '=', 'ψN γ')],\n",
       "  [('R0', 'be in', 'agreement with expressions')],\n",
       "  [('Simulation', 'be with', 'demographic eﬀects')],\n",
       "  [('we', 'estimate', 'parameters of model'),\n",
       "   ('we', 'ﬁtting', 'real data of conﬁrmed cases of infectious in Guinea')],\n",
       "  [('µ', '=', '0.0447')],\n",
       "  [('γI', '=', '3.735 γ')],\n",
       "  [('agreement', 'be with', 'data shown in Figure 2 of WHO data')],\n",
       "  [('Guinea', 'of', 'population')],\n",
       "  [('same parameters', 'with model be', '3.1'),\n",
       "   ('SIR model', 'be with', 'same parameters')],\n",
       "  [('oscillations', 'be in', 'numbers of three compartments')],\n",
       "  [('equilibrium S ∗', 'be compute theoretically by', '3.10')],\n",
       "  [('value MODELING', 'control of', 'EBOLA VIRUS SPREAD 7')],\n",
       "  [('detection', 'be in', 'Guinea'),\n",
       "   ('validation', 'be with', 'vital dynamics')],\n",
       "  [('One', 'seem', 'highly eﬀective NumericCitation')],\n",
       "  [('we', 'introduce into', '3.1')],\n",
       "  [('mathematical model', 'be with', 'control')],\n",
       "  [('S', 'be with', 'S ='),\n",
       "   ('I =', 'with infected be', 'b I t'),\n",
       "   ('c R', 'be with', 'R ='),\n",
       "   ('R =', 'with r be', 't'),\n",
       "   ('S', 'with s be', 't')],\n",
       "  [('Evolution', 'be with', 'vital dynamics'),\n",
       "   ('individuals', 'be in', 'compartments of SIR model')],\n",
       "  [('τ', 'be', 'positive')],\n",
       "  [('Recovered X', 'r', 't')],\n",
       "  [('Evolution', 'be with', 'vital dynamics'),\n",
       "   ('individuals', 'be in', 'compartments of SIR model')],\n",
       "  [('functional', 'be', 'nonlinear')],\n",
       "  [('team MODELING', 's without', 'control S with control X'),\n",
       "   ('control S', 'be with', 'control X'),\n",
       "   ('control R', 'be with', 'control X'),\n",
       "   ('C/C', 'be in', 'framework of Inria'),\n",
       "   ('control S', 'without s be', 't'),\n",
       "   ('control', 'without i be', 't')],\n",
       "  [('model', 'study', 'information provided by model'),\n",
       "   ('early detection', 'be in', 'West Africa')],\n",
       "  [('model', 'be in', 'case of basic reproduction number R0 1'),\n",
       "   ('current situation', 'be in', 'Guinea')],\n",
       "  [('We', 'study', 'equilibria')],\n",
       "  [('numerical simulations',\n",
       "    'conﬁrmed',\n",
       "    'theoretical analysis of equilibria for model')],\n",
       "  [('Science', 'for foundation be', 'FCT')],\n",
       "  [('problem', 'for u be', 't ∈'),\n",
       "   ('τ', '=', '0.02'),\n",
       "   ('initial conditions', 'with u be', 't'),\n",
       "   ('initial conditions', 'with u be', 't ∈'),\n",
       "   ('Optimal control function u', 'be with', 'initial conditions'),\n",
       "   ('problem', 'for u be', 't')],\n",
       "  [('control N', 'be with', 'control')],\n",
       "  [('Center', 'be in', 'Mathematics')],\n",
       "  [('comments', 'signiﬁcantly contribute to', 'quality of paper')],\n",
       "  [('fractional order', 'on model be', 'Adv. Diﬀerence Equ')],\n",
       "  [('ACADO Toolkit User', 'have', 'Manual'),\n",
       "   ('Engineering Center', 'in citation be', 'OPTEC'),\n",
       "   ('M. Vukov', 'citation in', 'Engineering Center')],\n",
       "  [('NumericCitation M. Barry', 'outbreak in', 'Conakry'),\n",
       "   ('Ebola outbreak', 'be in', 'Conakry')],\n",
       "  [('K. J. Pitt',\n",
       "    'algorithm for',\n",
       "    'direct solution of optimal control problems')],\n",
       "  [('NumericCitation L. Borio', 'virus as', 'biological weapons'),\n",
       "   ('NumericCitation L. Borio', 'group on', 'Civilian Biodefense')],\n",
       "  [('C. J. Peters', 'transmission of', 'Ebola hemorrhagic fever')],\n",
       "  [('NumericCitation H. W. Hethcote',\n",
       "    'mathematics of',\n",
       "    'infectious diseases')],\n",
       "  [('A. P. Galvani', 'dynamics in', 'Montserrado'),\n",
       "   ('A. P. Galvani', 'dynamics of', 'Ebola virus transmission'),\n",
       "   ('Dynamics', 'be in', 'Montserrado')],\n",
       "  [('NumericCitation C. J. Peters', 'introduction to', 'Ebola')],\n",
       "  [('NumericCitation A. Rachah', 'be in', 'West Africa')],\n",
       "  [('spread', 'be in', 'West Africa')],\n",
       "  [('D. F. M. Torres', 'dynamics of', 'dengue epidemics')],\n",
       "  [('Health', 'of ministry be', 'outbreak in Uganda'),\n",
       "   ('NumericCitation Uganda Ministry', 'outbreak of', 'Ebola'),\n",
       "   ('outbreak', 'be in', 'Uganda'),\n",
       "   ('NumericCitation Uganda Ministry', 'outbreak in', 'Uganda')],\n",
       "  [('Wang', 'outbreak in', 'West Africa'),\n",
       "   ('Ebola outbreak', 'be in', 'West Africa')],\n",
       "  [('Research', 'for center be', 'D. F. M. Torres'),\n",
       "   ('Research', 'for center be', 'Department of Mathematics'),\n",
       "   ('Research', 'for center be', '3810-193 Aveiro'),\n",
       "   ('Center', 'be in', 'Mathematics'),\n",
       "   ('Center', 'department of', 'Mathematics'),\n",
       "   ('Center', 'university of', 'Aveiro'),\n",
       "   ('Mathematics', 'in center be', '3810-193 Aveiro'),\n",
       "   ('Mathematics', 'in center be', 'D. F. M. Torres'),\n",
       "   ('Mathematics', 'in center be', 'Department of Mathematics'),\n",
       "   ('Research', 'for center be', 'University of Aveiro'),\n",
       "   ('Mathematics', 'in center be', 'University of Aveiro')]],\n",
       " 68: [[('model', 'determine by', 'dynamics of underlying primary infection'),\n",
       "   ('branching process spreading',\n",
       "    'be in',\n",
       "    'random time-sensitive environment')],\n",
       "  [('Superinfections', 'be', 'major')],\n",
       "  [('outbreaks', 'be', 'possible')],\n",
       "  [('one infection', 'have', 'strict obligate relationship with another'),\n",
       "   ('strict obligate relationship', 'be with', 'another')],\n",
       "  [('Blythe', 'spread as', 'contact process on lattice'),\n",
       "   ('Blythe parasites',\n",
       "    'spread as',\n",
       "    'contact process restricted sites currently occupied hosts')],\n",
       "  [('related model', 'be in', 'context of epidemic dynamics'),\n",
       "   ('epidemic dynamics', 'be in', 'social contact networks')],\n",
       "  [('SusceptibleInfectiveRecovered epidemics',\n",
       "    'be with',\n",
       "    'strictly obligate relationship such')],\n",
       "  [('Possible events', 'be in', 'network superinfection model')],\n",
       "  [('nodes', 'be in', 'network')],\n",
       "  [('population', 'be in', 'NumericCitation')],\n",
       "  [('random graph', 'be with', 'mean degree c')],\n",
       "  [('disease', 'be with', 'rate ρ1'),\n",
       "   ('on their neighbours', 'be with', 'rate β1'),\n",
       "   ('population',\n",
       "    'be with',\n",
       "    'infective individuals passing on to their neighbours')],\n",
       "  [('our secondary infection',\n",
       "    'spread on',\n",
       "    'subgraph of hosts infected with primary')],\n",
       "  [('us', 'see for', 'instance in NumericCitation'),\n",
       "   ('instance', 'be in', 'NumericCitation')],\n",
       "  [('we', 'characterise', 'phase diagram of system')],\n",
       "  [('network', 'of connectivity be', 'mean node degree')],\n",
       "  [],\n",
       "  [('α', 'be', 'unchanged')],\n",
       "  [('we', 'inspire by', 'results of NumericCitation')],\n",
       "  [('β2/ρ2', '=', 'α'), ('β2/ρ2', '= also', 'α')],\n",
       "  [('boundary', 'be', 'boundary')],\n",
       "  [('it', 'survive with', 'positive probability')],\n",
       "  [('k k', 'be', 'hypergeometric function'), ('k', 'be', 'hypergeometric')],\n",
       "  [('c =', 'c for', 'ϕ → 0')],\n",
       "  [('we', 'be for', 'discussion including illustrative numerical results'),\n",
       "   ('we', 'be for', 'discussion')],\n",
       "  [('us', 'consider', 'infection spread as generational'),\n",
       "   ('infection', 'spread as', 'generational')],\n",
       "  [('Zn = m', 'be in', 'branching model')],\n",
       "  [('children', 'be in', 'subsequent generation of branching process')],\n",
       "  [('random graph',\n",
       "    'may',\n",
       "    'may coupled to branching model with high probability')],\n",
       "  [('type t', '=', 't2 − t1')],\n",
       "  [('primary infection', 'be in', 'new host')],\n",
       "  [('multi-type branching process', 'be with', 'type space T =')],\n",
       "  [('space', 'over cb be', '¯ T')],\n",
       "  [('multi-type branching process =',\n",
       "    'be with',\n",
       "    'production operator M arising as above from kernel µ')],\n",
       "  [('4 only nonzero eigenvalue', 'be with', 'non-negative eigenfunction'),\n",
       "   ('radius', 'be', 'positive')],\n",
       "  [('ψ', 'for n be', 'ψ')],\n",
       "  [('ﬁnite time', 'be with', 'probability one')],\n",
       "  [('branching processes', 'be with', 'uniformly positive kernel')],\n",
       "  [('Choosing T large enough forces',\n",
       "    'close agreement in',\n",
       "    'maximum eigenvalues of corresponding production operators'),\n",
       "   ('Choosing T large enough forces close agreement',\n",
       "    'be in',\n",
       "    'maximum eigenvalues of corresponding production operators')],\n",
       "  [('M T', 'of λ be', 'T'), ('kernel', 'be', 'so strictly positive')],\n",
       "  [('˜M T', 'of ˜λ be', 'T'), ('Cb', 'to m be', 'T')],\n",
       "  [('λ |', 'zero with', 'T')],\n",
       "  [('untrimmed process satisand',\n",
       "    'hence also survive with',\n",
       "    'positive probability')],\n",
       "  [('we', 'must explicitly identify', 'operator M')],\n",
       "  [('infections', 'be in', 'parent'), ('primary infection', 'be in', 'child')],\n",
       "  [('primary infection', 'be in', 'parent')],\n",
       "  [('we', 'put together', 'contributions')],\n",
       "  [],\n",
       "  [('infections', 'be in', 'parent')],\n",
       "  [('We', 'state', 'our result about kernel'), ('We', 'be', '8 now ready')],\n",
       "  [('β1', 'be', 'hypergeometric'),\n",
       "   ('0F1', 'be', 'hypergeometric'),\n",
       "   ('ρ2', 'be', 'hypergeometric'),\n",
       "   ('β2', 'be', 'hypergeometric'),\n",
       "   ('ρ1', 'be', 'hypergeometric')],\n",
       "  [('λ', 'be top eigenvalue of', 'M'), ('λψ', '=', 'M ψ')],\n",
       "  [('functions', 'of family be', 'Ψ')],\n",
       "  [('Equating coefﬁcients', 'be in', 'determines')],\n",
       "  [('λ', 'for equation be', '9')],\n",
       "  [('discussion', 'be in', 'Section 2.2')],\n",
       "  [('their', 'formula', '1.2')],\n",
       "  [('Rényi random graphs', 'be in', 'limit'),\n",
       "   ('Rényi random graphs', 'be with', 'mean degree c'),\n",
       "   ('multi-type branching process',\n",
       "    'be with',\n",
       "    'production kernel given by equation'),\n",
       "   ('production kernel', 'with process be', 'Zn')],\n",
       "  [('ratio', 'be with', 'c')],\n",
       "  [('functions', 'be with', 'appropriate ﬁnite limits'),\n",
       "   ('bounds', 'be in', 'Lemma 3')],\n",
       "  [('we', 'check', 'upper bound')],\n",
       "  [('Lemma 3', 'of equation be', '19')],\n",
       "  [('upper', 'bind on', 'Φ given in Lemma 3'), ('bound', 'for u be', 'ϕ')],\n",
       "  [('Lemma 3', 'of equation be', '18')],\n",
       "  [('desired scaling', 'be in', 'large ϕ')],\n",
       "  [('We', 'begin with', 'ϕ')],\n",
       "  [('limits', 'of l be', 'ϕ')],\n",
       "  [('Figure 4', 'illustrate', 'phenomenon')],\n",
       "  [('agreement', 'be with', 'branching process for small values of ϕ')],\n",
       "  [('unexpected tail', 'be comprise of', 'few very large outbreaks'),\n",
       "   ('they', 'reach', 'most of graph')],\n",
       "  [('we', 'encounter', 'ﬁnite size effects'),\n",
       "   ('infection', 'be in', 'closed connected community'),\n",
       "   ('we', 'encounter', 'size effects')],\n",
       "  [('Similar ﬁnite size scaling effects',\n",
       "    'have',\n",
       "    'have observed in other coevolving infection models')],\n",
       "  [('interesting choice', 'be in', 'strategy of secondary infection')],\n",
       "  [('1000 simulations', 'be with', 'mean degree c ='),\n",
       "   ('secondary infection', 'of 100 be', 'p blue curve')],\n",
       "  [('branching process', 'of region be', 'pale green box')],\n",
       "  [('URL factsheets/hepatitis-d/en', 'be with', 'formulas')],\n",
       "  [('87 1974 integral operators', 'be with', 'nonnegative kernels')],\n",
       "  [('pp. 259',\n",
       "    'be in',\n",
       "    'Proceedings of Fifth Berkeley symposium on Mathematical Statistics')],\n",
       "  [('Coupled ﬂuctuations', 'be in', 'stacked contact processes')],\n",
       "  [('Phase transitions', 'be in', 'cooperative coinfections')],\n",
       "  [('Mathematical Analysis',\n",
       "    '214 on',\n",
       "    'variation with parameter of zeros of Bessel')],\n",
       "  [('Deterministic epidemics', 'be in', 'closed populations')],\n",
       "  [('Assessing node risk', 'be in', 'epidemics on networks')],\n",
       "  [('Branching processes', 'be in', 'disease epidemics')]],\n",
       " 69: [[('changes', 'be in', 'environment')],\n",
       "  [('SIR model', 'be with', 'variable transmission rate')],\n",
       "  [('algorithm', 'be with', 'historic UK measles data')],\n",
       "  [('we', 'provide', 'explicit formula for solution')],\n",
       "  [('Our approach', 'also yield', 'explicit formula for transmission rate')],\n",
       "  [('extensions', 'be in', 'Section IV B'),\n",
       "   ('SEIR epidemic model', 'be with', 'historical Citation -1966')],\n",
       "  [('its logarithmic derivative', 'be', 'always bounded below'),\n",
       "   ('its logarithmic derivative', 'be always bound below by', '− ν')],\n",
       "  [('ν', 'be', 'necessary')],\n",
       "  [('condition', 'be', 'also suﬃcient')],\n",
       "  [('inverse problem', 'be under-determined in', 'sense')],\n",
       "  [('algorithm', 'require', 'two conditions')],\n",
       "  [('function', 'be in', 'p t')],\n",
       "  [('Condition 1', 'prevent', 'zero denominaStep 3')],\n",
       "  [('T', 'be', 'where time length of infection data')],\n",
       "  [('This', 'be', 'mild')],\n",
       "  [('scaling trick', 'be in', 'Section III')],\n",
       "  [('SEIR model', 'be with', 'variable vital rates')],\n",
       "  [('Our algorithm',\n",
       "    'can',\n",
       "    'can extended to virtually such compartment model')],\n",
       "  [('recovery algorithm', 'use', 'two simulated data sets')],\n",
       "  [('small black squares', 'be in', 'Figure 1'), ('sampling', 'by f be', 't')],\n",
       "  [('notiﬁcation data', 'be in', 'NumericCitation'),\n",
       "   ('D.', 'use', 'notiﬁcation data in NumericCitation')],\n",
       "  [('SEIR model', 'be with', 'variable vital rates'),\n",
       "   ('we', 'use', 'same data set')],\n",
       "  [('we', 'examine', 'robustness of our new results')],\n",
       "  [('Bolker', 'have', 'measles data archive NumericCitation')],\n",
       "  [('births', 'show', 'large annual variations')],\n",
       "  [('actual births', 'be in', 'our model'),\n",
       "   ('we', 'include', 'actual births in our model'),\n",
       "   ('we', 'include', 'actual births')],\n",
       "  [('infections', 'be', 'reported')],\n",
       "  [('our results', 'be with', 'those of pervious authors')],\n",
       "  [('We', 'clarify issue for', 'us')],\n",
       "  [('weekly infection number', 'be', 'two parts')],\n",
       "  [('four days', 'be in', 'June'),\n",
       "   ('three days', 'be in', 'May'),\n",
       "   ('we', 'incorporate', 'it')],\n",
       "  [('recovered β', 'be in', 'panels'),\n",
       "   ('increasing peaks', 'be in', 'panel'),\n",
       "   ('panels', 'in β be', 't'),\n",
       "   ('increasing peaks', 'be in', 'panel c')],\n",
       "  [('moduli', 'transform of', 't')],\n",
       "  [('underreporting bias', 'be in', 'UK measles data'),\n",
       "   ('92.3 %', 'be compute from', 'NumericCitation'),\n",
       "   ('underreporting bias', 'be with', 'estimated mean reporting rate'),\n",
       "   ('we', 'test', 'robustness of our spectral peaks'),\n",
       "   ('92.3 %', 'be', 'computed')],\n",
       "  [('stage', 'be in', 'SEIR model'),\n",
       "   ('92.3 % correction factor', 'essentially double', 'number of cases')],\n",
       "  [('corrected weekly notiﬁcation data',\n",
       "    'hence should provide',\n",
       "    'good approximation for total weekly infections')],\n",
       "  [('precise methodology', 'be use in', 'NumericCitation')],\n",
       "  [('our observation', 'be with', 'frequencies')],\n",
       "  [('Little empirical evidence', 'support', 'their assumed functions'),\n",
       "   ('Little empirical evidence',\n",
       "    'currently support',\n",
       "    'their assumed functions')],\n",
       "  [],\n",
       "  [('We', 'estimate', '0')],\n",
       "  [('Hadeler', 'manipulate', 'our derivation')],\n",
       "  [('inverse method', 'be in', 'paper')],\n",
       "  [('Fourier', 'transform of', 'our recovered transmission rate function')],\n",
       "  [('notiﬁcations', 'of proportion be', 'number')],\n",
       "  [('one', 'nearly always construct', 't')],\n",
       "  [('epidemic model', 'be with', 'time-dependent transmission rate')],\n",
       "  [('NumericCitation', 'use', 'stochastic transmission models')],\n",
       "  [('Our algorithm', 'can', 'can also extended')],\n",
       "  [('SEIR model', 'be with', 'vital rates')],\n",
       "  [('SIR model', 'be with', 'vital rates')],\n",
       "  [('1/m', 'be', 'where memory period of immunity'),\n",
       "   ('SIR model', 'be with', 'waning immunity S ′')],\n",
       "  [('SIR model', 'be with', 'time-dependent indirect transmission rate'),\n",
       "   ('time-dependent indirect transmission rate', 'with model be', 'Joh et')],\n",
       "  [('αE − ν I', '=', 'R ′')],\n",
       "  [('SEIR model', 'be with', 'vital rates')],\n",
       "  [('case', 'in 29 be', '30')],\n",
       "  [('modiﬁed recovery algorithm',\n",
       "    'have',\n",
       "    'ﬁve steps together with three conditions'),\n",
       "   ('ﬁve steps', 'be with', 'three conditions')],\n",
       "  [('34 Recovery algorithm 12', 'be in', 'case')]],\n",
       " 70: [[('Patrick Harrington †', 'university of', 'Michigan')],\n",
       "  [('We',\n",
       "    'evaluate',\n",
       "    'solution on simulated spread of infectious over complex network')],\n",
       "  [('convex optimization solvers', 'can', 'can applied')],\n",
       "  [('nodes', 'be in', 'large networks'), ('nodes', 'be in', 'bioinformatics')],\n",
       "  [('they', 'be', 'diﬃcult to scale to larger graphs'),\n",
       "   ('they', 'be', 'However diﬃcult to scale to larger graphs')],\n",
       "  [('S I R processes', 'model', 'transmission events')],\n",
       "  [('Examples', 'include', 'GMMs')],\n",
       "  [('S I R model', 'use throughout', 'paper')],\n",
       "  [('model', 'be', 'diﬀerent'),\n",
       "   ('model', 'be', 'generative'),\n",
       "   ('S I R model', 'be diﬀerent generative model than', 'MRF')],\n",
       "  [('Research', 'be in', 'MRFs'),\n",
       "   ('we', 'will adopt', 'approach for S I R model')],\n",
       "  [('graphs', 'occur in', 'propagation of infectious disease'),\n",
       "   ('proposed sparse structure learning method',\n",
       "    'be design for',\n",
       "    'such occurs in propagation of infectious disease')],\n",
       "  [('approximate H1N1 ﬂu', 'inspire', '32')],\n",
       "  [('evolution', 'be in', 'network')],\n",
       "  [('have SPATIO-TEMPORAL GRAPHICAL MODEL SELECTION 3',\n",
       "    'be in',\n",
       "    'limit of large populations with equal mixing rates'),\n",
       "   ('large populations', 'be with', 'equal mixing rates')],\n",
       "  [('individual nodes', 'be in', 'network')],\n",
       "  [('nodes', 'be in', 'network'),\n",
       "   ('discrete time', 'model for', 'states of nodes in network'),\n",
       "   ('model', 'be', 'discrete')],\n",
       "  [('they', 'be in', 'when infected state')],\n",
       "  [('k', 'denote', 'time')],\n",
       "  [('model',\n",
       "    'be speciﬁed by',\n",
       "    'k | k − 1 state transition probabilities given in 3x3 stochastic matrix 0 Pi')],\n",
       "  [('S I RS', 'refer to', 'three state stochastic process')],\n",
       "  [('non-zero entries', 'be in', '2.2')],\n",
       "  [('observed tra jectory', 'of distribution be', 'l P =')],\n",
       "  [('k P', 'be with', 'indicator variable z i'),\n",
       "   ('conditional transition distribution', 'be in', '2.3')],\n",
       "  [('proposed graphical model selection method', 'be in', 'paper')],\n",
       "  [('probability', 'be in', '2.5'),\n",
       "   ('log log', 'be in', '3.2'),\n",
       "   ('3.2', 'in log be', '1 − ω')],\n",
       "  [('we', 'arrive at', 'following j')],\n",
       "  [('k − 1', 'be with', 'z'), ('j =', '− with', 'z')],\n",
       "  [('θi',\n",
       "    'j to',\n",
       "    'continuous valued variable lying between its discrete values log'),\n",
       "   ('its', 'log', '1 − ω')],\n",
       "  [('λ θ 1 log θ 0 s.t.', 'be with', 'λ >')],\n",
       "  [('symmetry', 'be in', 'interactions j =')],\n",
       "  [('symmetry', 'be in', 'θi')],\n",
       "  [('their corresponding constraints', 'be', 'summarized in Table 1')],\n",
       "  [('Table 1 Prior Knowledge Form',\n",
       "    'size of',\n",
       "    'Neighborhood Max Possible Size'),\n",
       "   ('Constraint Symmetry θi',\n",
       "    'of form be',\n",
       "    'j ∈ ηi Known Non-Interactions Min Possible Size'),\n",
       "   ('Constraint Symmetry θi', 'of form be', 'i Known Interactions θi'),\n",
       "   ('Constraint Symmetry θi', 'of form be', 'j =')],\n",
       "  [('b · log < s', 'constraint of', 'θ 1 ≤ s')],\n",
       "  [('proposed penalized likelihood problem', 'be in', '3.9')],\n",
       "  [('proposed numerical solution', 'be in', 'paper')],\n",
       "  [('nodes', 'method such as', 'one given')],\n",
       "  [('m', 'with g be', 'm'),\n",
       "   ('m', 'with g be', 'm'),\n",
       "   ('g', 'be with', 'm'),\n",
       "   ('i ∈ ˆηj SPATIO-TEMPORAL GRAPHICAL MODEL SELECTION 9',\n",
       "    'be with',\n",
       "    'step size')],\n",
       "  [('norm', 'be with', 'linear inequality constraints')],\n",
       "  [('H', 'be with', 'α'), ('α', 'with h be', 'm')],\n",
       "  [('12 positive entries', 'h on', 'entries in H i'),\n",
       "   ('entries', 'be in', 'H i'),\n",
       "   ('entries', 'on h be', 'm')],\n",
       "  [('dimensional quadratic program', 'be in', '3.12')],\n",
       "  [('10 HARRINGTON', 'be with', 'u')],\n",
       "  [('ˆθ', 'be', 'initial'), ('∈', 'be', 'initial')],\n",
       "  [('j =', 'be with', 'surrogate 2')],\n",
       "  [('convergence criteria', 'repeat', 'step 1')],\n",
       "  [('Symmetry', 'be', 'imposed')],\n",
       "  [('Such active set updates', 'have', 'have used successfully'),\n",
       "   ('Such active set updates', 'estimate', 'sparse partial correlations'),\n",
       "   ('Such active set updates', 'estimate', '24')],\n",
       "  [('They', 'block', 'co-ordinate descent'),\n",
       "   ('co-ordinate descent', 'be in', 'group lasso logistic regression')],\n",
       "  [('Algorithm 1', 'require', 'speciﬁcation of tuning parameter λ')],\n",
       "  [('attack rate ω', 'be', 'known')],\n",
       "  [('non-zero entries', 'be in', 'estimator')],\n",
       "  [('terms', 'be in', 'ith log-likelihood')],\n",
       "  [('there will multiple regularization parameters',\n",
       "    'one for',\n",
       "    'neighborhood')],\n",
       "  [('environment', 'be in', 'MATLAB')],\n",
       "  [('LR', 'selection in', 'paper'),\n",
       "   ('proposed graphical model selection', 'be in', 'paper')],\n",
       "  [('estimates', 'be with', 'non-zero value')],\n",
       "  [('5 % gain', 'be in', 'power')],\n",
       "  [('increased frequency', 'be in', 'tight clusters of small-world network')],\n",
       "  [('proposed method', 'be with', 'global speciﬁc penalties')],\n",
       "  [('images', 'be in', 'Figures')],\n",
       "  [('SIR', 'be with', 'neighborhood speciﬁc 1 respectively'),\n",
       "   ('SIR', 'be with', 'single penalty')],\n",
       "  [('mean', 'be with', 'standard deviation shown in parentheses')],\n",
       "  [('% zeros', 'be in', 'reconstruction of edges'),\n",
       "   ('reconstruction', 'be in', '200 node synthetic scale free network')],\n",
       "  [('% zeros', 'be in', 'reconstruction of edges'),\n",
       "   ('reconstruction', 'be in', '200 node synthetic scale free network')],\n",
       "  [('time horizon', 'be with', 'trajectories resampled')],\n",
       "  [('Initial condition', 'be', '40 randomly selected nodes')],\n",
       "  [('similar results', 'be in', 'detection performance')],\n",
       "  [('interactions', 'be in', 'spatio-temporal graphical model'),\n",
       "   ('We',\n",
       "    'have present',\n",
       "    'estimator of topology of interactions in spatio-temporal graphical model')],\n",
       "  [('proposed method',\n",
       "    'accurately reconstruct',\n",
       "    'topology of 05100.50.60.70.80.91 node degreea')],\n",
       "  [('% zeros',\n",
       "    'be in',\n",
       "    'reconstruction of edges 200 node synthetic smal l world network')],\n",
       "  [('% zeros',\n",
       "    'be in',\n",
       "    'reconstruction of edges 200 node synthetic smal l world network')],\n",
       "  [('time horizon', 'be with', 'trajectories resampled')],\n",
       "  [('node degree =', 'be with', 'trajectories resampled'),\n",
       "   ('200 node smal l-world network', 'be with', 'T')],\n",
       "  [('discussions', 'be with', 'Ami Wiesel')],\n",
       "  [('S. J. Clark', 'annals of', 'APPLIED STATISTICS'),\n",
       "   ('S. J. Clark', 'projection of', 'hiv prevalence')],\n",
       "  [('Epidemic thresholds', 'be in', 'real networks'),\n",
       "   ('C. Faloutsos', 'threshold in', 'real networks')],\n",
       "  [('D. ben Avraham', 'strategy for', 'computer networks')],\n",
       "  [('NumericCitation G. F. Cooper',\n",
       "    'method for',\n",
       "    'induction of probabilistic networks from data')],\n",
       "  [('NumericCitation G. Csardi', 'package for', 'research')],\n",
       "  [(\"L. Massouli'e\", 'threshold for', 'virus spread on networks')],\n",
       "  [('NumericCitation S. Eubank',\n",
       "    'outbreak in',\n",
       "    'realistic urban social networks'),\n",
       "   ('Model ling disease outbreaks',\n",
       "    'be in',\n",
       "    'realistic urban social networks')],\n",
       "  [('R. Tibshirani', 'numericcitation with', 'graphical lasso'),\n",
       "   ('Sparse inverse covariance estimation NumericCitation',\n",
       "    'be with',\n",
       "    'graphical lasso')],\n",
       "  [('K. Eames', 'journal of', 'Royal Society Interface')],\n",
       "  [('S. Boyd', 'method for', 'large-scale l1regularized logistic regression')],\n",
       "  [('I. Yang', 'journal of', 'Computational')],\n",
       "  [('Advances', 'be in', 'Neural Information Processing Systems 2007'),\n",
       "   ('D. Koller', 'learning of', 'Markov networks'),\n",
       "   ('Lee', 'be in', 'Advances')],\n",
       "  [('P. B ¨ uhlmann', 'lasso for', 'logistic regression')],\n",
       "  [('variable selection', 'be with', 'lasso'),\n",
       "   ('NumericCitation N. Meinshausen', 'annals of', 'Statistics')],\n",
       "  [('20 HARRINGTON', 'spread on', 'networks'),\n",
       "   ('20 HARRINGTON', 'spread of', 'epidemic disease')],\n",
       "  [('NumericCitation M. E. J. Newman', 'structure of', 'complex networks')],\n",
       "  [('J. Zhu', 'journal of', 'American Statistical Association'),\n",
       "   ('J. Zhu',\n",
       "    'estimation by',\n",
       "    'joint NumericCitation NumericCitation sparse regression models')],\n",
       "  [('NumericCitation A. Raftery', 'trend in', 'HIV/AIDS'),\n",
       "   ('Estimating trends', 'be in', 'HIV/AIDS')],\n",
       "  [('Model selection', 'be in', 'gaussian graphical models'),\n",
       "   ('B. Yu', 'selection in', 'gaussian graphical models'),\n",
       "   ('High-dimensional consistency', 'be in', 'NIPS')],\n",
       "  [('lasso',\n",
       "    'journal of',\n",
       "    'Royal Statistical Society Series B-Methodological'),\n",
       "   ('NumericCitation R. Tibshirani', 'shrinkage via', 'lasso')],\n",
       "  [('NumericCitation P. Tseng',\n",
       "    'method for',\n",
       "    'nonsmooth separable minimization')],\n",
       "  [('I. M. Longini', 'transmissibility of', 'pandemic inﬂuenza')],\n",
       "  [('NumericCitation M. Yuan', 'be in', 'gaussian graphical model')]],\n",
       " 71: [[('framework',\n",
       "    'involve',\n",
       "    'large scale simulation of contagion spreading processes from set of potential source locations')],\n",
       "  [('Diﬀerent frameworks', 'study', 'epidemic spreading'),\n",
       "   ('Diﬀerent mathematical frameworks', 'study', 'epidemic spreading')],\n",
       "  [('This', 'be', 'mathematical'),\n",
       "   ('diﬀerential equations', 'model', 'epidemic dynamics'),\n",
       "   ('This', 'be', 'traditional'),\n",
       "   ('equations', 'model', 'epidemic dynamics')],\n",
       "  [('we', 'address in', 'work')],\n",
       "  [('Our statistical inference framework', 'be', 'applicable')],\n",
       "  [('NumericCitation', 'be', 'adequate')],\n",
       "  [('partial epidemic realization',\n",
       "    'be ﬁned by',\n",
       "    'complete knowledge of infected'),\n",
       "   ('partial epidemic realization',\n",
       "    'be de ﬁned by',\n",
       "    'complete knowledge of infected')],\n",
       "  [('impact', 'be in', 'case')],\n",
       "  [('we', 'solve', 'source detection problem')],\n",
       "  [('signiﬁcant contribution', 'be in', 'problems of source de tection'),\n",
       "   ('we',\n",
       "    'however have make',\n",
       "    'signiﬁcant contribution in problems of source de tection for more general spreading processes on arbitrary network structures')],\n",
       "  [('we', 'describe sir compartment model in', 'section 1')],\n",
       "  [('network', 'be with', 'set of nodes V'),\n",
       "   ('set', 'with network be', 'example'),\n",
       "   ('node i probability',\n",
       "    'be in',\n",
       "    'one discrete time step number of simulations for speciﬁc SIR process')],\n",
       "  [('contact-network', 'be', 'static')],\n",
       "  [('spreading process', 'use', 'discrete time step model')],\n",
       "  [('process', 'be', 'stochastic')],\n",
       "  [('infected node u',\n",
       "    'infect',\n",
       "    'adjacent susceptible node v in one discrete time step')],\n",
       "  [('infected node', 'recover in', 'one discrete time step')],\n",
       "  [('parameter',\n",
       "    'be in',\n",
       "    'inference procedure for source location estimation')],\n",
       "  [('source localization', 'be in', 'network')],\n",
       "  [('R', 'be', 'random')],\n",
       "  [('observed infected set', 'be in', 'network')],\n",
       "  [('we', 'apply', 'Bayes theorem')],\n",
       "  [('source location estimation problem',\n",
       "    'core of be',\n",
       "    'Thus determination of likelihood of observed epidemic realization initiated at source location Θ')],\n",
       "  [('We',\n",
       "    'determine',\n",
       "    'maximum likelihood for observed epidemic realization')],\n",
       "  [('source', 'can', 'where can node from set S')],\n",
       "  [('it', 'be', 'major'),\n",
       "   ('it',\n",
       "    'be also major determinant of',\n",
       "    'running time of estimation procedure')],\n",
       "  [('θk', 'be with', 'maximum likelihood ˆP')],\n",
       "  [('S IR process', 'be with', 'source θ'), ('We', 'ﬁrst', '~ rθ')],\n",
       "  [('xi', 'be', 'characteristic')],\n",
       "  [('n', 'be', 'where number of random simulations')],\n",
       "  [('ﬁrst', 'be', 'binary')],\n",
       "  [('measure', 'be', 'known'),\n",
       "   ('number', 'be in', 'r1'),\n",
       "   ('nodes', 'be in', '~ r1'),\n",
       "   ('number', 'be in', '~ r2'),\n",
       "   ('nodes', 'be in', '~ r2')],\n",
       "  [('bit count', 'be with', 'Biran-Kernignan method')],\n",
       "  [('1 n', 'number of', 'simulations'),\n",
       "   ('Run SIR simulation', 'be with', 'Θ =')],\n",
       "  [('we', 'estimate', 'source likelihood ˆP')]],\n",
       " 72: [[('Biotechnology', 'of institute be', 'University')],\n",
       "  [('model', 'be with', 'SIR structure')],\n",
       "  [('statistics', 'be in', 'system')],\n",
       "  [('1',\n",
       "    'have',\n",
       "    'has most widely studied epidemic model in literature NumericCitation'),\n",
       "   ('has studied epidemic model', 'be in', 'literature NumericCitation')],\n",
       "  [('outbreak', 'be in', 'susceptible pool')],\n",
       "  [('1', 'be', 'rescaled by rate of recovery without loss of generality')],\n",
       "  [('α =', 'be in', 'our case')],\n",
       "  [('changes', 'be in', 'NumericCitation')],\n",
       "  [('scaling', 'in p be', 'gree of variability'),\n",
       "   ('p', 'be in', 'scaling depending on number introducing'),\n",
       "   ('scaling exponents', 'be in', 'multi-stage SIR'),\n",
       "   ('p', 'gree of', 'variability')],\n",
       "  [('system', 'in s be', 'R'),\n",
       "   ('system', 'in s be', 'susceptible experiences'),\n",
       "   ('system', 'in s be', 'I'),\n",
       "   ('S', 'be in', 'system')],\n",
       "  [('statistics', 'be important for', 'several reasons')],\n",
       "  [('outbreak statistics',\n",
       "    'be',\n",
       "    'qualitatively diﬀerent for externally driven system than for simple SIR')],\n",
       "  [('distribution', 'be in', 'limit of N → ∞')],\n",
       "  [('endemic equilibrium', 'be in', 'reservoir'),\n",
       "   ('infection', 'be', 'endemic'),\n",
       "   ('infection', 'be at', 'where endemic equilibrium in reservoir')],\n",
       "  [('INFINITE POPULATION', 'be in', 'limit of inﬁnite system size'),\n",
       "   ('analysis',\n",
       "    'have provide',\n",
       "    'crucial insights in to full nonlinear process')],\n",
       "  [('BDI process', 'be with', 'ν =')],\n",
       "  [('time-dependent distribution', 'be in', 'strictly sub-critical case'),\n",
       "   ('sub-critical case', 'in distribution be', '1'),\n",
       "   ('number', 'of distribution be', '1')],\n",
       "  [('driven SIR process', 'be with', '3 outbreaks')],\n",
       "  [('micro-outbreak',\n",
       "    'be',\n",
       "    'where analogous to customer be-associated with single imported infection')],\n",
       "  [('ing', 'service at', 'station')],\n",
       "  [('outbreak', 'be in', 'BDI process')],\n",
       "  [('we', 'obtain', 'distribution of outbreak sizes')],\n",
       "  [('notomers', 'enter immediately into', 'service')],\n",
       "  [('Markovian arrival process',\n",
       "    'stand for',\n",
       "    'general service time distribution')],\n",
       "  [('arrival', 'be in', 'inﬁnite server queue')],\n",
       "  [('micro-outbreak', 'be in', 'BDI process')],\n",
       "  [('ν =', 'be with', 'single infectious host at beginning')],\n",
       "  [('closed form solution', 'be', 'available NumericCitation'),\n",
       "   ('closed form solution', 'be', 'NumericCitation'),\n",
       "   ('distribution', 'be in', 'BD process'),\n",
       "   ('single server', 'be in', 'queue system')],\n",
       "  [('statistics', 'use', 'established methods'),\n",
       "   ('statistics', 'can', 'can calculated')],\n",
       "  [('we', 'embark on', 'new calculations')],\n",
       "  [('we', 'queue', 'theory NumericCitation')],\n",
       "  [('Arrivals', 'be in', 'analogous M'),\n",
       "   ('Poisson process', 'be with', 'rate ν α')],\n",
       "  [('cycle', 'be in', 'when idle state'),\n",
       "   ('cycle', 'be', 'thinned'),\n",
       "   ('arrival', 'occur in', 'when idle state')],\n",
       "  [('outbreak', 'be in', 'BDI process')],\n",
       "  [('distribution', 'be with', 'infection at time 0')],\n",
       "  [('process', 'be in', 'idle state decays with time')],\n",
       "  [('10', 'diverge at', 'critical threshold')],\n",
       "  [('we', 'shall adopt for', 'our purpose')],\n",
       "  [('outbreak sizes', 'be', 'ﬁnite')],\n",
       "  [('ﬁrst ingredient', 'be in', 'calculation')],\n",
       "  [('BD process', 'be with', 'birth rate α set to 1')],\n",
       "  [('T P T ≤ t', 'yn of', 'outbreak')],\n",
       "  [('correlation', 'integrate', 'joint distribution')],\n",
       "  [('we', 'simplify', 'integration')],\n",
       "  [('terms', 'be in', 'PGF')],\n",
       "  [('17a', 'can', 'can solved explicitly')],\n",
       "  [('y =', 'around h be', 'y')],\n",
       "  [('simpliﬁcation', 'be in', 'last step')],\n",
       "  [('more ﬂat', 'be with', 'increasing ν')],\n",
       "  [('pronounced bump', 'be in', 'simulations'), ('bump', 'be', 'ﬁnite')],\n",
       "  [('eq', 'in h be', 'y'), ('PGF H', 'be in', 'eq')],\n",
       "  [('comparison', 'be with', 'stochastic simulations')],\n",
       "  [('Inset', 'scale by', 'N 2/3')],\n",
       "  [('ν', '=', '3')],\n",
       "  [('it', 'have accumulate', 'outbreaks of two distinct scales')],\n",
       "  [('presence', 'be in', 'bump')],\n",
       "  [('series expansion', 'be in', 'frac')],\n",
       "  [('tional powers', 'be in', 'expansion')],\n",
       "  [('ν', 'apply', 'property given by eq')],\n",
       "  [('17a', 'simpliﬁes pgf to', 'y'),\n",
       "   ('BD process', 'be with', 'prefactor of 1/2')],\n",
       "  [('analysis', 'scale', 'ex-FIG'),\n",
       "   ('Asymptotic analysis', 'scale', 'ex-FIG')],\n",
       "  [('ν', '=', '1')],\n",
       "  [('outbreak size distribution',\n",
       "    'fall oﬀ more steeply with',\n",
       "    'increasing ν')],\n",
       "  [('remaining probability mass 1 − ν − 1', 'be with', 'inﬁnite')],\n",
       "  [('we', 'more formally have', 'log n')],\n",
       "  [('minishes', 'be with', 'n')],\n",
       "  [('statistics', 'thus be conditional on', 'ﬁnite sized outbreak')],\n",
       "  [('one contiguous outbreak', 'be with', 'probability 1 − ν − 1')],\n",
       "  [('idle period', 'be thus transient in', 'case')],\n",
       "  [('analysis', 'be in', 'NumericCitation')],\n",
       "  [('algebraic distribution', 'be in', 'n · P')],\n",
       "  [('2 ∼ M', 'be in', 'ﬁnite-sized system')],\n",
       "  [('n', 'be in', 'ﬁgure 7')],\n",
       "  [('expression', 'be in', 'eq')],\n",
       "  [('solution', 'lead to', 'N log N')],\n",
       "  [('outbreak size distribution', 'of scaling be', 'ν +2'),\n",
       "   ('ν =', 'for scaling be', 'ν +2'),\n",
       "   ('M ∼', 'by scaling be', 'ν +2')],\n",
       "  [('y-axis', 'be scale by', 'theoretical scaling law of eq')],\n",
       "  [('exponent', 'be in', 'eq')],\n",
       "  [('40', 'be', 'increasing')],\n",
       "  [('n pl', 'be', 'average outbreak size in power-law regime'),\n",
       "   ('average outbreak size', 'be in', 'power-law regime')],\n",
       "  [('scaling', 'be in', 'power-law regime')],\n",
       "  [('other', 'on top of collapse be', 'exponential cutoﬀs collapse at 1/3'),\n",
       "   ('power-laws collapse', 'collapse at', '1/3'),\n",
       "   ('N 2/3', 'at collapse be', 'exponential cutoﬀs collapse')],\n",
       "  [('log N 1/3', 'be in', 'M')],\n",
       "  [('inﬁnite', 'be with', 'probability 1 − ν − 1'),\n",
       "   ('inﬁnite', 'be in', 'size')],\n",
       "  [('average outbreak size', 'be in', 'power-law regime')],\n",
       "  [('n ∞', 'be', 'average outbreak size condieq')],\n",
       "  [('outbreak', 'be in', 'inﬁnite regime')],\n",
       "  [('M', 'be in', 'scale')],\n",
       "  [('power-laws', 'be in', 'P n')],\n",
       "  [('derivation', 'be in', 'eq')],\n",
       "  [('power-law regime', 'thus only depend on', 'N')],\n",
       "  [('process', 'how fast or slow approach', 'scale')],\n",
       "  [('abbreviation', 'be with', 'for probability'),\n",
       "   ('w.p.', 'be', 'where abbreviation for with probability')],\n",
       "  [('agreement', 'be with', 'stochastic simulations')],\n",
       "  [('power law', 'be with', 'continuously varying exponent'),\n",
       "   ('power law', 'be with', 'ﬁxed exponent of 2/3')],\n",
       "  [('would O outbreak', 'be with', 'ν')],\n",
       "  [('we', 'use', 'above results')],\n",
       "  [('B. Outbreak duration', 'be with', 'eﬀective transmission rate α')],\n",
       "  [('we', 'use', 'eq')],\n",
       "  [('scale', 'be in', 'inﬁnite regime from eq')],\n",
       "  [('same scaling relationship',\n",
       "    'be',\n",
       "    'noted in NumericCitation for simple SIR')],\n",
       "  [('agreement', 'be with', 'stochastic simulations')],\n",
       "  [('plots', 'be split into', 'two ﬁgures for clarity')],\n",
       "  [('Dashed lines', 'scale', 'laws')],\n",
       "  [('inﬁnite regime', 'in 10 be', 'n ∞ t ∞ ∼'),\n",
       "   ('inﬁnite regime', '10 for', 'only ν')],\n",
       "  [('universality', 'be in', 'power-law characteristics')],\n",
       "  [('parameter ν', 'should away from', 'should far critical points')],\n",
       "  [('ν', '=', '1 ±')],\n",
       "  [('scaling laws', 'be in', 'inﬁnite regime')],\n",
       "  [('scaling power laws',\n",
       "    'will',\n",
       "    'will muddied via interference from logarithmic factors'),\n",
       "   ('scaling power laws', 'will', 'will muddied')],\n",
       "  [('window', 'be', 'decreasing')],\n",
       "  [('ν', 'be away from', 'far its critical values')],\n",
       "  [('DISCUSSION', 'be in', 'work'), ('we', 'discussion in', 'work')],\n",
       "  [('α', '=', '1')],\n",
       "  [('second critical point', 'be in', 'process separating')],\n",
       "  [('tunability', 'be', 'non-trivial')],\n",
       "  [('simple SIR process', 'describe', 'data'),\n",
       "   ('ﬁne scale data', 'be', 'available')],\n",
       "  [('such ﬁne scale data', 'collect on', 'timescales of outbreaks')]],\n",
       " 73: [[('seasonality', 'be', 'well-known'),\n",
       "   ('well-known phenomenon', 'study from', 'ancient times')],\n",
       "  [('development', 'be', 'unsolved'), ('development', 'be', 'actual')],\n",
       "  [('second-order ODE', 'be with', 'outer excitation')],\n",
       "  [('complicomparison', 'be with', 'real epidemiological data')],\n",
       "  [('kind', 'resetting of', 'variables'),\n",
       "   ('seasonality', 'of kind be', 'resetting')],\n",
       "  [('kinetic scheme S', 'within susceptible be', 'S')],\n",
       "  [('large oscillations', 'be in', 'system')],\n",
       "  [('k', 'of variation be', 'satisfactory reproduces')],\n",
       "  [('real seasonal variations', 'be in', 'disease activity of ILI')],\n",
       "  [('it',\n",
       "    'be',\n",
       "    'also conﬁrmed by study of individual outbreaks by stochastic SIRS model')],\n",
       "  [('ﬂu activity', 'be in', 'Berlin')],\n",
       "  [('European regions', 'be with', 'continental climate')],\n",
       "  [('example', 'be', 'common')],\n",
       "  [('their geographical large-scale spread', 'way of be', 'goals of work')],\n",
       "  [('it', 'only consider ode equation for', 'modeling')],\n",
       "  [('Berlin', 'for light-gray be', 'red online lines')],\n",
       "  [('Results',\n",
       "    'be with',\n",
       "    'substituted temperature-dependent reaction rate mentioned above'),\n",
       "   ('comparison', 'be with', 'data series from Google Flu Trends')],\n",
       "  [('sample', 'between', 'nodes')],\n",
       "  [('Calculations', 'use matlab routine for', 'Fast Fourier Transform')],\n",
       "  [('used temperature variations', 'enforce', 'desired oscillations')],\n",
       "  [('period', 'of', 'magnitude')],\n",
       "  [('many lower parts', 'be in', 'Fig. 1')],\n",
       "  [('correlation coeﬃcient',\n",
       "    'be',\n",
       "    'larger within ma jority of subintervals than for distributions taken for whole interval')],\n",
       "  [('their shape', 'be correlate for', 'even such simple model'),\n",
       "   ('their shape', 'be suﬃciently correlate for', 'even such simple model')],\n",
       "  [('temperature dependence',\n",
       "    'could',\n",
       "    'Correspondingly could considered as leading factor')],\n",
       "  [('factors', 'play ma jor role at', 'highest levels of ﬂu activity')],\n",
       "  [('all', 'of first be', 'system')],\n",
       "  [('that', 'be', 'stable')],\n",
       "  [('all', 'of first be', 'ﬁrst term in left-hand side'),\n",
       "   ('First', 'term in', 'left-hand side of Eq'),\n",
       "   ('ﬁrst term', 'be in', 'left-hand side of Eq')],\n",
       "  [('second line', 'be in', 'Eq')],\n",
       "  [('non-linear third line', 'be in', 'Eq')],\n",
       "  [('9 terms', 'include', 't')],\n",
       "  [('non-linear terms', 'be in', 'Eq')],\n",
       "  [('its variation', 'be in', 'second line of Eq')],\n",
       "  [('endogenous diseases', 'be with', 'vital dynamics')],\n",
       "  [('third line', 'be in', 'Eq')],\n",
       "  [('second line', 'be in', 'Eq')],\n",
       "  [('expression', 'be', 'harmonic'),\n",
       "   ('expression', 'be', 'simple'),\n",
       "   ('expression', 'be', 'non-homogeneous')],\n",
       "  [('one', 'consider', 'Fig. 2A')],\n",
       "  [('Color online A', 'value for', 'spectra of daily mean temperature')],\n",
       "  [('light-gray common ﬂu activity', 'be in', 'Berlin')],\n",
       "  [('harmonic ﬁlter', 'be in', 'Eq')],\n",
       "  [('black line', 'be in', 'Fig. 2')],\n",
       "  [('their high similarity', 'be in', 'shape')],\n",
       "  [('Summary', 'be in', 'work')],\n",
       "  [('inﬂuenza epidemics', 'use', 'search engine query data')]],\n",
       " 74: None,\n",
       " 75: [[('RANDOM GRAPH', 'be with', 'GIVEN DEGREES SVANTE JANSON')],\n",
       "  [('Emergence', 'be', 'key'), ('Emergence', 'be', 'public')],\n",
       "  [('near-critical behaviour',\n",
       "    'be in',\n",
       "    'context of susceptibleinfectiverecovered epidemic on random graph on n vertices with given degree sequence')],\n",
       "  [('1 n threshold',\n",
       "    'be with',\n",
       "    'ω tending to inﬁnity slowly as population size')],\n",
       "  [('We', 'size of', 'large epidemic')],\n",
       "  [('standard binomial random graph', 'be in', 'sparse regime')],\n",
       "  [('random graph', 'be with', 'given degrees')],\n",
       "  [('contacts', 'of network be', 'graph'),\n",
       "   ('population', 'be represent by', 'graph')],\n",
       "  [('Diﬀerent individuals', 'lead to', 'diﬀerent numbers of contacts')],\n",
       "  [('graph', 'be with', 'given degrees'),\n",
       "   ('given degrees', 'with graph be', 'critical window')],\n",
       "  [('ML', 'support by', 'EPSRC Leadership Fellowship EP/J004022/2')],\n",
       "  [('TH', 'support by', 'EPSRC')],\n",
       "  [('degree', 'be number of', 'contacts of corresponding individual')],\n",
       "  [('infectious individuals', 'recover at', 'rate ρ > 0')],\n",
       "  [('disease', 'be in', 'surviving population'),\n",
       "   ('ma jor outbreak', 'be in', 'supercritical case')],\n",
       "  [('practical interest',\n",
       "    'be in',\n",
       "    'theoretical understanding of behaviour of near-critical epidemics')],\n",
       "  [('populations', 'be with', 'complete mixing')],\n",
       "  [('near-criticality', 'use', 'arguments'),\n",
       "   ('near-criticality', 'be discuss in', 'NumericCitation'),\n",
       "   ('near-criticality', 'use', 'non-rigorous arguments')],\n",
       "  [('NumericCitation studies',\n",
       "    'of ¨ be',\n",
       "    'generalized Reed-Frost epidemic model')],\n",
       "  [('infectives', 'observe', 'bimodality'),\n",
       "   ('limit distribution', 'be with', 'edge probability p.')],\n",
       "  [('populations', 'be with', 'homogeneous mixing'),\n",
       "   ('diﬀusion limit', 'be derive for', 'ﬁnal size of near-critical epidemic')],\n",
       "  [('random graph', 'be with', 'given vertex degrees'),\n",
       "   ('graph', 'be', 'random'),\n",
       "   ('epidemic spreading', 'be in', 'population of large size n')],\n",
       "  [('from graphs', 'be with', 'speciﬁed sequence of contact numbers')],\n",
       "  [('greater inhomogeneity',\n",
       "    'be with',\n",
       "    'rather arbitrary distribution of number of contacts for diﬀerent persons'),\n",
       "   ('random graph model',\n",
       "    'allow for',\n",
       "    'greater inhomogeneity with rather arbitrary distribution of number of contacts for diﬀerent persons')],\n",
       "  [('1 n 1/3', 'be with', 'ω growing')],\n",
       "  [('total number',\n",
       "    'be with',\n",
       "    'initial infectious degree XI ,0 of potential infectious contacts at beginning of epidemic number of acquaintances of initially infectious individuals')],\n",
       "  [('we', 'furthermore determine', 'likely size of large epidemic')],\n",
       "  [('ﬁrst', 'be in', 'paragraph above')],\n",
       "  [('maximum susceptible degree', 'be in', 'population of size n')],\n",
       "  [('us',\n",
       "    'consider',\n",
       "    'multiple initial conditions within same realisation of process'),\n",
       "   ('we', 'demonstrate behaviour for', 'particular example')],\n",
       "  [('R0', '=', '1.01'),\n",
       "   ('Poisson degree distribution', 'be with', 'mean λ =')],\n",
       "  [(\"R'enyi random graph\", 'be with', 'edge probability λ/n')],\n",
       "  [('size O', 'epidemic of', 'size')],\n",
       "  [('graphs', 'be with', 'given degrees')],\n",
       "  [('Convergence', 'be in', 'case')],\n",
       "  [('results', 'be in', 'case of bounded vertex degrees')],\n",
       "  [('random graph', 'be with', 'given degrees')],\n",
       "  [('such results', 'be in', 'literature')],\n",
       "  [('size', 'be with', 'given vertex degrees under weakened assumptions'),\n",
       "   ('conjecture', 'be in', 'NumericCitation'),\n",
       "   ('size', 'be in', 'barely supercritical random graph')],\n",
       "  [('our proofs',\n",
       "    'also involve',\n",
       "    'couplings of evolution of total infective degree with suitable Brownian motions'),\n",
       "   ('couplings', 'be with', 'suitable Brownian motions')],\n",
       "  [('size', 'be in', 'supercritical random graph')],\n",
       "  [('simple graph', 'be with', 'n vertices'), ('n', 'be', 'simple')],\n",
       "  [('number', 'be with', 'degree k >')],\n",
       "  [('Our assumptions', 'be', 'follows')],\n",
       "  [('n', 'let', 'DS')],\n",
       "  [('∞ k =', 'be with', 'ﬁnite mean λ')],\n",
       "  [('0 knI', 'k of', '∞ Xk'), ('nSα3 k', '=', 'ν')],\n",
       "  [('∞ E DS', '0 by', 'D6'), ('D3 S', 'of integrability be', 'D2')],\n",
       "  [('random multigraph', 'be with', 'given degree sequence'),\n",
       "   ('graph', 'be with', 'degree sequence')],\n",
       "  [('results', 'also hold for', 'SIR epidemic on G')],\n",
       "  [('nS P ∞', 'be in', 'cases')],\n",
       "  [('Theorem 2.3', 'of case be', 'i')],\n",
       "  [('P ∞ k', '=', '0 knR'), ('P ∞ k', '=', 'knR')],\n",
       "  [('condition', 'be in', 'Theorem 2.4'), ('2.6', 'by k be', 'k → ∞')],\n",
       "  [('we', 'combine', 'two parts of argument'), ('our', '∞', '2.22')],\n",
       "  [('We', 'see', 'D1')],\n",
       "  [('βn', 'assume → λ as', 'n → ∞')],\n",
       "  [('SIR', 'epidemic on', 'random graph'),\n",
       "   ('ηn → 0', 'consider', 'SIR epidemic on random graph')],\n",
       "  [('other vertices', 'be', 'susceptible')],\n",
       "  [('G', 'be with', 'm = nλ')],\n",
       "  [('we', 'repeat', 'Skorohod trick')],\n",
       "  [('that', 'be', 'assumed'),\n",
       "   ('we', 'hold with', 'ν = µ/λ3'),\n",
       "   ('it', 'be', 'also easy')],\n",
       "  [('nI ,0', 'delete', 'initially infective vertices of degree 0')],\n",
       "  [('0',\n",
       "    'imply',\n",
       "    'corresponding results for general nR by following argument'),\n",
       "   ('our results', 'be in', 'case nR')],\n",
       "  [('similar arguments', 'be in', 'related situation')],\n",
       "  [('other variants', 'be in', '1')],\n",
       "  [('it', 'be', 'attached'), ('it', 'be', 'attached to')],\n",
       "  [('Unpaired half-edges', 'be', 'said')],\n",
       "  [('pair', 'be remove from', 'set of free half-edges')],\n",
       "  [('Infective vertices', 'recover at', 'rate ρn > 0')],\n",
       "  [('remaining edges', 'be in', 'G ∗')],\n",
       "  [('step', 'however be irrelevant for', 'course of epidemic')],\n",
       "  [('time change', 'be in', 'process')],\n",
       "  [('Rt', 'be increase moreover in', 't.'), ('Rt', 'be increase in', 't.')],\n",
       "  [('our simplifying assumptions', 'be in', 'Section 3.1')],\n",
       "  [('O', 'w.h.p.', 'Remark 3.6'), ('duration', 'be in', 'Section 3.4')],\n",
       "  [('exponential distribution', 'be with', 'rate dS')],\n",
       "  [('Bernstein',\n",
       "    'by',\n",
       "    'inequality for sums of bounded independent centred random variables')],\n",
       "  [('ωn', 'be', 'such'), ('ωn', 'be', 'valued')],\n",
       "  [('l', 'for ωn be', 'ωn − 1')],\n",
       "  [('2 free half-edges', 'be in', 'total'), ('state', 'be with', 'xI >')],\n",
       "  [('˜Xt', 'be', 'linear')],\n",
       "  [('k', '=', '0 knk − 1')],\n",
       "  [('˜St', 'be', 'linear')],\n",
       "  [('we', 'have', 'k'),\n",
       "   ('k 6', 'with ˜st be', 'k'),\n",
       "   ('˜St', 'be with', 'k 6')],\n",
       "  [('Lemma 6.1', 'be with', 'd ='), ('k 6 ˜α − 1 n', 'for ns be', '3.28')],\n",
       "  [('Jensen', 'have', 'Cauchy')],\n",
       "  [('Xk', '=', '1 k3nS')],\n",
       "  [('free infective half-edge pairs', 'be with', 'free recovered half-edge')],\n",
       "  [('βnxI', 'be in', 'state with xI infective vertices recover at rate ρn'),\n",
       "   ('βnxI', 'be in', 'time-changed process'),\n",
       "   ('state', 'be with', 'xI infective vertices recover at rate ρn')],\n",
       "  [('Gronwall', 'have', 'inequality yields')],\n",
       "  [('3.23', 'by op be', 'n ˜α2 n')],\n",
       "  [('same', 'be true of', 'martingale term')],\n",
       "  [('infective vertex', 'be with', 'k free half-edges')],\n",
       "  [('unique positive root', 'be in', 'cases'), ('we', 'have', '0')],\n",
       "  [('˜αn αn → ∞', 'be in', 'case ν =')],\n",
       "  [('cases', 'in hold be', '3.8')],\n",
       "  [('hI', 'n around', 'zero'), ('zero', 'around n be', 't')],\n",
       "  [('t > =', 'for = be', 'k'), ('t > =', 'for = be', 'k − knk')],\n",
       "  [('0 ˜αn t', 'be in', 'uniformly t')],\n",
       "  [('w.h.p. T ∗ 6 2κ ˜αn', 'be in', 'particular')],\n",
       "  [('we', 'conclude −', '→ 0')],\n",
       "  [('t ∈', 'for f be', 't')],\n",
       "  [('t =', 'for f be', 't')],\n",
       "  [('ν', '=', '0')],\n",
       "  [('= 2κ', 'be in', 'Theorem 3.1'),\n",
       "   ('we', '= 2κ in', 'Theorem 3.1'),\n",
       "   ('we', '2κ in', 'Theorem 3.1')],\n",
       "  [('y >', 'for ns be', 'k'), ('− kpk t', 'be in', 't')],\n",
       "  [('deﬁnitions', 'be in', '3.37')],\n",
       "  [('εn', '=', 'ε > 0'), ('ν', '=', '0')],\n",
       "  [('ν', '=', '0')],\n",
       "  [('we', 'otherwise colour', 'it black'), ('we', 'colour', 'it black')],\n",
       "  [('vertex', 'become', 'infective'),\n",
       "   ('its remaining free half-edges', 'be', 'given')],\n",
       "  [('our ¯ Zm', 'be', 'right continuous'),\n",
       "   ('our ¯ Zm', 'be', 'continuous'),\n",
       "   ('we', 'have colour', 'new infective free half-edges')],\n",
       "  [('Zm', '=', '0')],\n",
       "  [('susceptible vertices', 'be with', 'k free half-edges')],\n",
       "  [('0 knI', 'fraction of', 'More general ly')],\n",
       "  [('P Z0 δXI ,0', '=', '0'), ('P Z0 6 δXI ,0', '=', '0')],\n",
       "  [('i', 'be', 'number of red free half-edges at vertex i')],\n",
       "  [('Chebyshev', 'have', 'yields')],\n",
       "  [('We', 'assume', 'ﬁrst')],\n",
       "  [('R ⋆', 'be', 'its recovery time')],\n",
       "  [('We', 'have', 'Then Z0')],\n",
       "  [('binomial distribution', 'be with', 'parameters dI')],\n",
       "  [('δ', 'be', 'small')],\n",
       "  [('τ 6 ∞', 'be', 'stopping time'), ('∞', 'be', 'stopping')],\n",
       "  [('martingale', 'be with', 'respect')],\n",
       "  [('Doob', 'by', 'inequality')],\n",
       "  [('powers', 'over 6m6 be', '2N'), ('we', 'obtain', 'Wm')],\n",
       "  [('δ', 'be', 'positive'), ('δ', 'be', 'small')],\n",
       "  [('2.8', 'by kns be', '4.14'),\n",
       "   ('2.8', 'by kns be', 'k − δnαn'),\n",
       "   ('time', 'at kstm be', 'k'),\n",
       "   ('> knS', 'nsα2 since', 'o nαn'),\n",
       "   ('2.8', 'by kns be', 'k = o nSα2 since o nαn')],\n",
       "  [('red free half-edge pairs', 'be with', 'free susceptible half-edge')],\n",
       "  [('R0', 'of deﬁnition be', '2.1')],\n",
       "  [('free susceptible half-edge',\n",
       "    'with pair be',\n",
       "    'expected square of jump satisﬁes'),\n",
       "   ('red free half-edge pairs', 'square of', 'jump satisﬁes'),\n",
       "   ('m ∗ ∗', 'k by', 'assumption'),\n",
       "   ('red free half-edge pairs',\n",
       "    'be with',\n",
       "    'free susceptible half-edge at vertex of degree k'),\n",
       "   ('m', 'for pair be', 'expected square of jump satisﬁes'),\n",
       "   ('assumption', 'by k be', 'k P')],\n",
       "  [('Lemma 4.2', 'be with', 'b')],\n",
       "  [('w.h.p. ∞ Xk', '=', '> δnαn')],\n",
       "  [('ε >', 'for yield be', 'k2'),\n",
       "   ('Lemma 3.2', 'be with', '˜αn = yield for ε >'),\n",
       "   ('4.19 sup', 'lemma with', '˜αn = yield for ε >')],\n",
       "  [('degree k', 'be', 'infected')],\n",
       "  [('beta-binomial distribution', 'be with', 'parameters')],\n",
       "  [('D', 'be', 'positive'), ('D', 'be', 'random')],\n",
       "  [('We', 'recall', '4.1')],\n",
       "  [('susceptible vertex', 'be with', 'ℓ half-edges')],\n",
       "  [('knk', 'be', 'random'), ('random variable', 'be with', 'distribution P')],\n",
       "  [('ζm', '=', '∆ ¯ Zm')],\n",
       "  [('nα3 n →', 'since i be', '∞')],\n",
       "  [('P ∞ k', '=', '0 k3pk k')],\n",
       "  [('i.i.d.', 'be with', 'mean 0')],\n",
       "  [('Bt', 'be', 'Brownian'), ('Bt', 'be', 'standard')],\n",
       "  [('We', 'deﬁne', 'tα − 2 n t t t ⌊ t ⌋')],\n",
       "  [('distribution', 'be in', 'Skorohod space D'),\n",
       "   ('non-integer t', 'be in', 'by Here distribution')],\n",
       "  [('Theorem 2.4', 'of z be', 'i')],\n",
       "  [('t ∈', 'for 0 be', '0 δ')],\n",
       "  [('w.h.p. m', 'stop with', 'Zm'), ('epidemic stops', 'be with', 'Zm')],\n",
       "  [('δ', 'be', 'arbitrary')],\n",
       "  [('random half-edges', 'sequence of', 'constructed')],\n",
       "  [('we', 'pair', 'infective red half-edge ym'), ('ym', 'be with', 'xm')],\n",
       "  [('= ζm', 'number of', 'new red free half-edges'),\n",
       "   ('we', 'pair with', 'xm'),\n",
       "   ('xm', 'be', 'initially 28 infective')],\n",
       "  [('m', 'be', 'i.i.d.'), ('i.i.d. sequence', 'be with', 'distribution Y')],\n",
       "  [('xm', 'be susceptible at', 'susceptible')],\n",
       "  [('m 6 M', 'for k be', 'c3n')],\n",
       "  [('αn ¯ Z', 'be in', 'distribution')],\n",
       "  [('¯ Ztα − 2', 'be in', 'D')],\n",
       "  [('inﬁmum', 'be', 'continuous')],\n",
       "  [('4.44 Brownian motion Bt', 'be in', '4.42')],\n",
       "  [('We', 'have deﬁned', 'm ∗')],\n",
       "  [('particular', 'in o be', '1'), ('o', 'be in', 'particular')],\n",
       "  [('we', 'obtain', 'P Yn 6 − αnZ0')],\n",
       "  [('n', 'be in', 'Theorem 2.3'), ('Theorem 2.3', 'in n be', '4.51')],\n",
       "  [('t6A σBt', 'be in', 'other words')],\n",
       "  [('o nαn', 'be in', 'proof of case'), ('case', 'of proof be', 'i')],\n",
       "  [('Theorem 2.3', 'in w.h.p. be', 'b'), ('w.h.p.', 'be in', 'Theorem 2.3')],\n",
       "  [('for n', 'on m be', '4.52')],\n",
       "  [('exponential distribution', 'be with', 'parameter 2λ − 1')],\n",
       "  [('Z0', 'since independent be', 'P')],\n",
       "  [('expectation', 'be in', '4.57')],\n",
       "  [('deﬁnition', 'be in', '4.34')],\n",
       "  [('∞ σBt', 'be in', 'simple graph case')],\n",
       "  [('corresponding result', 'be in', '21'),\n",
       "   ('argument', 'be with', 'minor modiﬁcations')],\n",
       "  [('We', 'work with', 'random multigraph G ∗')],\n",
       "  [('our trick', 'be in', 'Section 3.1 n n')],\n",
       "  [('Chebyshev', 'have', 'inequality')],\n",
       "  [('parallel edges', 'be in', 'G ∗'),\n",
       "   ('cW', 'be in', 'conditional probability P')],\n",
       "  [('W variable cW', 'be with', 'convergence of moments')],\n",
       "  [('vertex', 'be in', 'as NumericCitation'), ('vertex', 'be with', 'degree')],\n",
       "  [('we', 'refer for', 'details')],\n",
       "  [('asymptotic probability', 'for formula be', '2.18')],\n",
       "  [('notation', 'be in', 'Lemma 4.3')],\n",
       "  [('fraction', 'let', 'χ')],\n",
       "  [('extra term', 'be in', '4.63')],\n",
       "  [('Jensen', 'by', 'inequality')],\n",
       "  [('latter', 'be in', 'its most general sense')],\n",
       "  [('they', 'draw', 'further numbers'),\n",
       "   ('they', 'draw', 'further random numbers')],\n",
       "  [('network', 'be with', 'elements'),\n",
       "   ('network', 'be with', 'adjacency matrix')],\n",
       "  [('individual i', 'pick', '1')],\n",
       "  [('individual', 'have', 'resistance to infection')],\n",
       "  [('We', 'continue', 'iterative procedure'),\n",
       "   ('size', 'be in', 'G'),\n",
       "   ('criticality', 'near n be', 'di n'),\n",
       "   ('methods', 'be in', 'paper')],\n",
       "  [('Luczak', 'be in', 'preparation')],\n",
       "  [('number', 'be in', 'C1'), ('vertices', 'be in', 'C1')],\n",
       "  [('> 0', 'be', 'equivalent')],\n",
       "  [('our proof', 'be in', 'Section 3')],\n",
       "  [('i.e. ρn', '=', '0')],\n",
       "  [('ρn', '=', '0')],\n",
       "  [('concentration', 'be in', 'as Theorem 3.1')],\n",
       "  [('analogous result', 'be in', 'NumericCitation')],\n",
       "  [('Lemma 6.3', 'be with', 'Theorem 3.1')],\n",
       "  [('Taylor expansion remainder term', 'be in', 'analogue of')],\n",
       "  [('χπ − 1', 'note', 'that')],\n",
       "  [('ρn', '=', '0')],\n",
       "  [('We', 'obtain', '4.63')],\n",
       "  [('role', 'be in', 'emergence of infectious diseases')],\n",
       "  [('Network epidemic models', 'be with', 'two levels of mixing')],\n",
       "  [('random graphs', 'be with', 'ﬁxed degree sequence')],\n",
       "  [('Graphs', 'be with', 'speciﬁed degree distributions')],\n",
       "  [('SIR process', 'be in', 'random network with heterogeneous connectivity'),\n",
       "   ('random network', 'be with', 'heterogeneous connectivity')],\n",
       "  [('near-critical SIR', 'be with', 'vaccination')],\n",
       "  [('Brownian motion', 'be with', 'parabolic drift')],\n",
       "  [('Adv.', 'be in', 'Appl')],\n",
       "  [('percolation', 'be in', 'random graphs with given vertex degrees'),\n",
       "   ('random graphs', 'be with', 'given vertex degrees')],\n",
       "  [('random multigraph', 'be', 'simple')],\n",
       "  [('random graph', 'be with', 'given degrees')],\n",
       "  [('SIR dynamics', 'be in', 'random networks')],\n",
       "  [('networks', 'be with', 'large initial conditions')],\n",
       "  [('random networks', 'be with', 'heterogeneous connectivity'),\n",
       "   ('SIR dynamics',\n",
       "    'be in',\n",
       "    'random networks with heterogeneous connectivity')],\n",
       "  [('Mathematics', 'of department be', 'Sweden E-mail address'),\n",
       "   ('Mathematical Sciences', 'of school be', 'UK')],\n",
       "  [('Mathematics', 'of school be', 'UK E-mail address'),\n",
       "   ('Mathematics', 'of school be', 'M13 9PL')],\n",
       "  [('n =', 'for relationship be', 'n ='),\n",
       "   ('epidemic ﬁnal size', 'between relationship be', 'n ='),\n",
       "   ('Poisson degree distribution', 'be with', 'mean λ =')]],\n",
       " 76: [[('Biocomputation', 'for institute be', 'BIFI'),\n",
       "   ('Biocomputation', 'for institute be', 'University of Zaragoza'),\n",
       "   ('Biocomputation', 'for institute be', '50018 Zaragoza'),\n",
       "   ('Mathematics', '2department of', 'Theoretical Physics'),\n",
       "   ('Mathematics', 'university of', 'Strathclyde'),\n",
       "   ('3 Institute', 'university of', 'Zaragoza'),\n",
       "   ('Mathematics', 'university of', 'Zaragoza'),\n",
       "   ('50018 Zaragoza', 'institute for', 'Biocomputation of Complex Systems')],\n",
       "  [('certain real-world scenarios',\n",
       "    'be in',\n",
       "    'like analysis of disease propagating through plants')],\n",
       "  [('nodes', 'be', 'distributed')],\n",
       "  [('propagation', 'be in', 'plots/ﬁelds of same area')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('ma jor development',\n",
       "    'be in',\n",
       "    'application of network theory to real-world problems')],\n",
       "  [('use', 'be in', 'epidemiological models')],\n",
       "  [('application', 'be in', 'ecology')],\n",
       "  [('more theoretical developments',\n",
       "    'have',\n",
       "    'Since then have presented in literature')],\n",
       "  [('stochasticity', 'be in', 'transmission of infection')],\n",
       "  [('spatial eﬀects', 'be in', 'transmission of diseases')],\n",
       "  [('persistence', 'be in', 'natural landscapes')],\n",
       "  [('propagation', 'be in', 'plants')],\n",
       "  [('plants', 'be', 'distributed')],\n",
       "  [('shape', 'be in', 'diﬀerent scenarios')],\n",
       "  [('we', 'first should mention', 'experimental plots for diﬀerent crops')],\n",
       "  [('crops', 'be', 'when cultivated in country ﬁelds'),\n",
       "   ('crops', 'be cultivate in', 'country ﬁelds')],\n",
       "  [('they', 'be', 'elongated')],\n",
       "  [('these', 'be', 'third'),\n",
       "   ('plants', 'be grow naturally in', 'given environment')],\n",
       "  [('distribution', 'acquire', 'many diﬀerent shapes')],\n",
       "  [],\n",
       "  [('elongated shapes',\n",
       "    'decrease',\n",
       "    'impact of epidemics on plant populations')],\n",
       "  [('nodes',\n",
       "    'be',\n",
       "    'In case uniformly and independently distributed on unit rectangle of given side lengths')],\n",
       "  [('we',\n",
       "    'can analyze',\n",
       "    'eﬀect of elongation of epidemic spreading on plants')],\n",
       "  [('nodes', 'be', 'where distributed in RGG')],\n",
       "  [('we',\n",
       "    'describe',\n",
       "    'propagation of disease on plant population on ﬁeld of varied rectangular shapes')],\n",
       "  [('right', 'be with', 'elongation of rectangle')],\n",
       "  [('interaction', 'be in', 'population')],\n",
       "  [('entries Aij', 'have =', '1 0')],\n",
       "  [('edges incident', 'of number be', 'equivalently ki = Pj Aij')],\n",
       "  [('G', 'be', 'connected'), ('G', 'be', 'simple')],\n",
       "  [('eigenvalue λ1',\n",
       "    'be',\n",
       "    'known as principal eigenvalue of adjacency matrix also as Perron-Frobenius eigenvalue')],\n",
       "  [('λ1', 'determine', 'conditions of invasion')],\n",
       "  [('given ﬁxed number', 'know as', 'connection radius')],\n",
       "  [('disease',\n",
       "    'can',\n",
       "    'can transmitted from one plant to its nearest neighbors')],\n",
       "  [('n points', 'be in', 'unit rectangle')],\n",
       "  [('two RRGs', 'be with', 'diﬀerent values of rectangle side length')],\n",
       "  [('RGG', 'be with', 'n nodes connected'),\n",
       "   ('connection radius',\n",
       "    'value of be',\n",
       "    'for which RGG with n nodes connected with high probability')],\n",
       "  [('average degree', 'be for', '1')],\n",
       "  [('2 bottom', 'with = be', 'top')],\n",
       "  [('they', 'be', 'smaller'), ('they', 'be', 'Euclidean')],\n",
       "  [('expected average degree', 'be in', 'RRG')],\n",
       "  [('variation',\n",
       "    'be with',\n",
       "    'change of connection radius for diﬀerent values of rectangle elongation obtained by computational realizations of RRGs')],\n",
       "  [('increase', 'be in', 'as case of RGGs'),\n",
       "   ('sigmoid function',\n",
       "    'be with',\n",
       "    'increase of connection radius in similar way as in case of RGGs'),\n",
       "   ('increase', 'be in', 'similar way'),\n",
       "   ('RRG',\n",
       "    'be connect',\n",
       "    'changes as sigmoid function with increase of connection radius in similar way as in case of RGGs')],\n",
       "  [('we', 'determine', 'critical radii'), ('Fig. 2', 'of panel be', 'b')],\n",
       "  [],\n",
       "  [],\n",
       "  [('It', 'can', 'can read in two diﬀerent ways'), ('It', 'can', 'can read')],\n",
       "  [('infectious individuals', 'for infected be', 'I')],\n",
       "  [('she', 'enter', 'recovered compartment'), ('she', 'acquire', 'immunity')],\n",
       "  [('node i', 'be', 'susceptible respectively')],\n",
       "  [('Change',\n",
       "    'be with',\n",
       "    'change of connection radius for diﬀerent values of rectangle elongation')],\n",
       "  [('infected 7', 'recover from', 'disease'),\n",
       "   ('contact', 'be with', 'infected one through link of network')],\n",
       "  [('ratio β/µ', 'drive', 'spreading of disease')],\n",
       "  [('non-equilibrium second order phase transition',\n",
       "    'be in',\n",
       "    'statistical physics NumericCitation')],\n",
       "  [('threshold', 'be in', 'sense')],\n",
       "  [('τ', '=', '1')],\n",
       "  [],\n",
       "  [('RGG', 'be in', 'd-dimensional cube'),\n",
       "   ('RGG', 'be in', 'd-dimensional cube'),\n",
       "   ('cd', 'be', 'where constant characteristic of RGG in d-dimensional cube'),\n",
       "   ('cd', 'be', 'constant')],\n",
       "  [('spectral radius', 'be bound by', 'average degree of RGG'),\n",
       "   ('nr2', 'be in', 'graphs')],\n",
       "  [('dependance', 'be with', 'topological parameters of RRGs'),\n",
       "   ('we',\n",
       "    'derive',\n",
       "    'analytical results about dependance of epidemic threshold with topological parameters of RRGs'),\n",
       "   ('EPIDEMIC THRESHOLD', 'be in', 'RRGS')],\n",
       "  [('square', 'be in', 'RGG')],\n",
       "  [('we', 'disconnect', 'RRG')],\n",
       "  [('average degree', 'decrease with', 'elongation of rectangle')],\n",
       "  [('process', 'be in', 'work')],\n",
       "  [('RRGs', 'be with', '1 left'),\n",
       "   ('diﬀerent values', 'for 30 be', 'right'),\n",
       "   ('spectral radius', 'of plot be', '30 centre')],\n",
       "  [('spectral radius', 'be in', 'general')],\n",
       "  [('rectangle', 'be', 'when elongated')],\n",
       "  [('RRGs', 'be with', '1 left'),\n",
       "   ('diﬀerent values', 'for 30 be', 'right'),\n",
       "   ('RRGs', 'for degree be', '30 centre')],\n",
       "  [('they', 'be', 'indeed correlated'),\n",
       "   ('they', 'be', 'indeed very highly linearly correlated')],\n",
       "  [('rectangle', 'be', 'when elongated')],\n",
       "  [('We', 'discuss in', 'more detail in net section'),\n",
       "   ('more detail', 'be in', 'net section')],\n",
       "  [('SIMULATIONS', 'be in', 'section'),\n",
       "   ('radius r', 'be with', 'goal checking')],\n",
       "  [('rectangle', 'be in', 'RRG model')],\n",
       "  [('fraction', 'be in', 'stationary state')],\n",
       "  [('same trend', 'be with', 'elongation of rectangle')],\n",
       "  [('30 fall', 'be in', 'second case')],\n",
       "  [('results',\n",
       "    'agree with',\n",
       "    'large accumulation of empirical evidence about role of plots/ﬁelds elongation on propagation of diseases on plants')],\n",
       "  [('networks', 'be with', 'inﬂuence produced by shape variation of plots')],\n",
       "  [('grant', 'through aragón be', 'Spain')],\n",
       "  [('SM', 'be also support by', 'MINECO through Juan de la Cierva Program')],\n",
       "  [('MS', 'be', 'supported'),\n",
       "   ('Weir Advanced Research Centre', 'university of', 'Strathclyde')],\n",
       "  [('scale-free networks', 'be with', 'degree correlations'),\n",
       "   ('Absence', 'be in', 'scale-free networks with degree correlations')],\n",
       "  [('epidemic spreading', 'be in', 'networks')],\n",
       "  [('spread', 'be in', 'networks')],\n",
       "  [('Epidemics',\n",
       "    'be in',\n",
       "    'networks of spatially correlated three-dimensional root-branching structures')],\n",
       "  [('epidemic threshold', 'obtain via', 'numerical simulations')],\n",
       "  [('Line', 'represent', 'theoretical prediction of Eq')],\n",
       "  [('inset', 'show', 'zoom for ﬁrst case of Eq')],\n",
       "  [('− 1 ≤ r', '≤', 'dashed line')],\n",
       "  [('≤ r ≤', 'be in', 'simulations')],\n",
       "  [('Other parameters', 'be', '=')],\n",
       "  [('experimental evidence', 'be in', 'model system'),\n",
       "   ('invasion', 'be in', 'spatial epidemics')],\n",
       "  [('Heterogeneity', 'be in', 'susceptible')],\n",
       "  [('populations', 'less susceptible to', 'epidemic outbreaks'),\n",
       "   ('Complexity', 'be in', 'host morphology')],\n",
       "  [('individuals', 'be with', 'non-uniform population density')],\n",
       "  [('Synchronization', 'be in', 'random geometric graphs')],\n",
       "  [('Worm epidemics', 'be in', 'wireless ad hoc networks')],\n",
       "  [],\n",
       "  [('ﬁeld experiments', 'be with', 'aerially disseminated pathogens')],\n",
       "  [('severity', 'be in', 'wheat cultivar mixtures')],\n",
       "  [('Connectivity', 'be in', 'dense networks conﬁned within right prisms')],\n",
       "  [('Wireless Networks', 'symposium on', 'pp. 628-635'),\n",
       "   ('Modeling', 'be in', 'Mobile')],\n",
       "  [('Connectivity', 'be in', 'fractal regions')],\n",
       "  [('Nonequilibrium phase transitions', 'be in', 'lattice models')],\n",
       "  [('Epidemic thresholds', 'be in', 'real networks')],\n",
       "  [('Discrete-time Markov chain approach', 'be in', 'complex networks')],\n",
       "  [('Virus spread', 'be in', 'networks')],\n",
       "  [('Spectral analysis', 'be in', 'random geometric networks')]],\n",
       " 77: [[('massive employment', 'be in', 'network epidemiology')],\n",
       "  [('Propagation', 'be', 'reliable')],\n",
       "  [('same method',\n",
       "    'predict',\n",
       "    'future evolution of epidemic outbreak from partial observations at early stage of dynamics')],\n",
       "  [('Monte Carlo direct sampling',\n",
       "    'be in',\n",
       "    'case of SIR model on random graphs for diﬀerent observation methods')],\n",
       "  [('disease transmission', 'be in', 'contact networks')],\n",
       "  [('Holme NumericCitation', 'use', 'simple SIR model')],\n",
       "  [('epidemics',\n",
       "    'be in',\n",
       "    'individual-based models from partial observations')],\n",
       "  [('missing information', 'apply to', 'problem of epidemic forecasting')],\n",
       "  [('we',\n",
       "    'predict',\n",
       "    'evolution of SIR model on given network from partial observation of states of nodes in early stage of dynamics'),\n",
       "   ('partial observation', 'be in', 'early stage of dynamics')],\n",
       "  [('we', 'introduce', 'metrics used for validation of results')],\n",
       "  [('partial observations', 'be in', 'SIR model')],\n",
       "  [('x0 i i ∈ V', 'number of', 'diﬀerent realizations')],\n",
       "  [('form', 'be in', 'particular')],\n",
       "  [('information available', 'involve', 'only fraction of overall population'),\n",
       "   ('state xTobs', 'be in', 'Vobs')],\n",
       "  [('future contacts', 'be in', 'time-varying networks')],\n",
       "  [('Vobs', '=', 'V servation'),\n",
       "   ('its relative frequency', 'be in', 'experiment')],\n",
       "  [('realization', 'be in', 'particular'),\n",
       "   ('time t', 'be in', 'realization of stochastic')],\n",
       "  [('Direct sampling', 'predict', 'future states'),\n",
       "   ('Direct sampling', 'be', 'then employed'),\n",
       "   ('sampling', 'predict', 'future states')],\n",
       "  [('observed nodes', 'be with', 'same attributes'),\n",
       "   ('state',\n",
       "    'be with',\n",
       "    'probability computed from knowledge of states of observed nodes')],\n",
       "  [('posterior probability', 't give', 'observation xTobs at time Tobs'),\n",
       "   ('conﬁguration xt',\n",
       "    'of probability be',\n",
       "    'xTobs time t given observation xTobs at time Tobs')],\n",
       "  [('normalization constant probability distribution P xt',\n",
       "    'x0 in',\n",
       "    'of our analysis'),\n",
       "   ('observed state', 'prediction of', 'epidemic evolution'),\n",
       "   ('xTobs | x0', 'be in', 'of our analysis')],\n",
       "  [('states', 'be in', 'at principle')],\n",
       "  [('Conﬁgurations', 'be in', 'leftmost column'),\n",
       "   ('epidemic process', 'be in', 'speciﬁc realization'),\n",
       "   ('other columns',\n",
       "    'represent',\n",
       "    'time evolution of epidemic process in speciﬁc realization')],\n",
       "  [('P xt', 'x0 by', 'approximate sampling method')],\n",
       "  [('unobserved nodes', 'be with', 'at least one susceptible neighbour')],\n",
       "  [('We', 'know', 'initial time of epidemics within ∆ T of time steps')],\n",
       "  [('realization', 'be with', 'origin')],\n",
       "  [('ROC curve', 'start from', 'origin of axes')],\n",
       "  [('ranking', 'be', 'When equal to real one')],\n",
       "  [('size', 'number of', 'nodes reached by infection')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('direct ∆ T0', '=', '1')],\n",
       "  [('realizations Ms', '=', '8 · 105')],\n",
       "  [('Belief Propagation technique', 'be with', 'partial observation')],\n",
       "  [('scheme', 'disregard', 'existing correlations between 90 % of nodes')],\n",
       "  [('heatplots', 'be in', 'Fig. 3')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('actual number', 'be in', 'observation'),\n",
       "   ('density sampling', 'be in', 'regimes')],\n",
       "  [('BP', 'overestimate', 'epidemic size')],\n",
       "  [('direct sampling', 'monotonically decrease', 'AUC values'),\n",
       "   ('direct sampling',\n",
       "    'monotonically decrease auc value for',\n",
       "    'increasing times')],\n",
       "  [('AUC', 'remain below', 'considerably one')],\n",
       "  [('heatplots', 'be in', 'Fig. 5a')],\n",
       "  [('degree-based observation',\n",
       "    'be particularly convenient for',\n",
       "    'heterogenous networks')],\n",
       "  [('2 %', 'be in', 'particular'),\n",
       "   ('direct sampling', 'be with', 'complete observation'),\n",
       "   ('ROC area increase',\n",
       "    'be in',\n",
       "    'ﬁrst stage of epidemics for prediction methods')],\n",
       "  [('number', 'be in', 'observation')],\n",
       "  [('initial evolution', 'be', 'more predictable'),\n",
       "   ('evolution', 'be', 'more predictable')],\n",
       "  [('similarity sampling', 'underestimate at', 'late times')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('partial observation', 'consider', 'connected subgraph of 30 % of nodes')],\n",
       "  [('topological information', 'be in', 'observation')],\n",
       "  [('observations', 'be with', 'few infected/recovered nodes')],\n",
       "  [('this', 'show in', 'Fig. 6c'),\n",
       "   ('Belief Propagation',\n",
       "    'perform very well in',\n",
       "    'case of degree-based observation')],\n",
       "  [('nodes', 'be in', 'network'),\n",
       "   ('direct sampling', 'be with', 'complete observation')],\n",
       "  [('better results',\n",
       "    'introduce degree-dependence in',\n",
       "    'infection probability inferred from observation')],\n",
       "  [('extinction time distribution',\n",
       "    'be',\n",
       "    'for extinction time distribution global feature of epidemic process'),\n",
       "   ('distribution', 'be', 'global')],\n",
       "  [('Pext =', 'non-trivial at_time', 'peculiar topological structure')],\n",
       "  [('Fig. 8 shows',\n",
       "    'be in',\n",
       "    'Even case of complete obthe regular trees ing on observation'),\n",
       "   ('Fig. 8 shows', 'be in', 'particular'),\n",
       "   ('networks',\n",
       "    'be with',\n",
       "    'Fig. 8 shows in particular Even in case of complete obthe regular trees ing on observation')],\n",
       "  [('snapshots', 'be in', 'panels')],\n",
       "  [('Tpeak', '=', '21'), ('snapshots', 'be in', 'panels')],\n",
       "  [('distribution', 'reach maximum at', 'larger times')],\n",
       "  [('graphs', 'be with', 'small-world properties'),\n",
       "   ('graphs', 'be in', 'general')],\n",
       "  [('we',\n",
       "    'motivate by',\n",
       "    'observed strong variability of extinction time distribution')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('prediction', 'be with', 'complete observation')],\n",
       "  [('heat-plots', 'be in', 'Fig. 10b')],\n",
       "  [('number', 'be', 'when increased'),\n",
       "   ('observations', 'be with', 'few infected')],\n",
       "  [('fraction',\n",
       "    'be with',\n",
       "    'large similarity index to observed incomplete snapshot')],\n",
       "  [('remarkable quantitative diﬀerences',\n",
       "    'be with',\n",
       "    'results from direct sampling'),\n",
       "   ('direct sampling', 'be with', 'complete observation')],\n",
       "  [('best performances', 'be in', 'particular'),\n",
       "   ('observations', 'be with', 'large number of infected')],\n",
       "  [('eﬀect', 'lead', 'overestimation of probability of long extinction times'),\n",
       "   ('non-convergence', 'be in', 'instances of epidemic process')],\n",
       "  [('similarity sampling', 'give', 'largest average diﬀerence')],\n",
       "  [('low number', 'be', 'when observed')],\n",
       "  [('We',\n",
       "    'consider',\n",
       "    'real network dataset of sexual encounters of internet-mediated prostitution NumericCitation')],\n",
       "  [('12 50,185 contacts', 'be', 'recorded'),\n",
       "   ('comment', 'be in', 'thread about B. Citation Citation')],\n",
       "  [('Nevertheless BP', 'be', 'better'),\n",
       "   ('Nevertheless BP', 'be better than', 'other methods')],\n",
       "  [('network',\n",
       "    'be characterize by',\n",
       "    'well connected core surrounded by many low degree nodes')],\n",
       "  [('few infected nodes', 'be', 'When observed'),\n",
       "   ('infected nodes', 'be', 'When observed'),\n",
       "   ('epidemic process', 'spread slowly at', 'early time')],\n",
       "  [('jectories', 'be', 'small')],\n",
       "  [('observation', 'occur at', 'later times')],\n",
       "  [('results',\n",
       "    'can',\n",
       "    'However can likely improved by higher computational power')],\n",
       "  [('Belief Propagation', 'recover only', 'nodes'),\n",
       "   ('Belief Propagation', 'recover', 'nodes')],\n",
       "  [('short loops', 'be in', 'network')],\n",
       "  [('node i', 'infect', 'node j')],\n",
       "  [('direct sampling', 'be with', 'complete observation'),\n",
       "   ('AUC',\n",
       "    'even decrease with',\n",
       "    'time much faster compared to direct sampling with complete observation')],\n",
       "  [('marginal probability', 'be in', 'network'),\n",
       "   ('Conclusions', 'be in', 'present work')],\n",
       "  [('direct sampling', 'be with', 'complete information')],\n",
       "  [('We',\n",
       "    'also analyze',\n",
       "    'real-world contact network obtained from Brazilian database of sexual encounters')],\n",
       "  [('actual number', 'be in', 'observation')],\n",
       "  [('signiﬁcant distortion', 'be in', 'particular')],\n",
       "  [('BP',\n",
       "    'be',\n",
       "    'In general more accurate in classiﬁcation of individual marginals than in estimate of average epidemic size')],\n",
       "  [('presence', 'be in', 'network')],\n",
       "  [('ROC classiﬁcation', 'be', 'based')],\n",
       "  [('BP',\n",
       "    'finally usually approximate',\n",
       "    'extinction time probability distribution better than other methods')],\n",
       "  [('BP', 'give', 'better results for observations')],\n",
       "  [('combined eﬀect', 'be in', 'observation of many short loops'),\n",
       "   ('many short loops', 'be in', 'network')],\n",
       "  [('observations', 'be with', 'suﬃciently large number of infected')],\n",
       "  [('t', 'be', 'at time represented by variable xt process')],\n",
       "  [('one-to-one correspondence',\n",
       "    'be with',\n",
       "    'static conﬁguration of individual infection times')],\n",
       "  [('xt', 'be with', 'ζ t i i ='),\n",
       "   ('P xt | t', 'xt at', 'time t'),\n",
       "   ('14 ζ t ti', 'xt with', 'ζ t i i =')],\n",
       "  [('represented', 'be represent by', 'variable node')],\n",
       "  [('underlying graph', 'be', 'when tree')],\n",
       "  [],\n",
       "  [('they',\n",
       "    'take value in',\n",
       "    'space of single-variable probability distributions')],\n",
       "  [('x0 terms', 'be in', 'latter expression')],\n",
       "  [('that', 'describe', 'susceptible nodes')],\n",
       "  [('35 obs obs obs', 'be in', 'factor graph representation')],\n",
       "  [('patient-zero problem', 'be with', 'noisy observations')],\n",
       "  [('NumericCitation N. Antulov-Fantulin', 'c', 'H. ˇStefanˇci'),\n",
       "   ('patient zero', 'be in', 'static networks')],\n",
       "  [('Inference', 'be in', 'epidemics on temporal contact networks')],\n",
       "  [('Predictability', 'be in', 'global outbreaks of infectious diseases')],\n",
       "  [('epidemic outbreaks', 'be in', 'networks')],\n",
       "  [('Contact patterns', 'be in', 'high school')],\n",
       "  [('Epidemic processes', 'be in', 'complex networks')],\n",
       "  [('Information dynamics',\n",
       "    'shape',\n",
       "    'sexual networks of internet-mediated prostitution')],\n",
       "  [('Simulated epidemics',\n",
       "    'be in',\n",
       "    'empirical spatiotemporal network of 50,185 sexual contacts')],\n",
       "  [('NumericCitation J. Stehl', 'e be', 'N. Voirin')],\n",
       "  [('High-resolution measurements', 'be in', 'primary school')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('heatplots', 'compute by', 'belief propagation')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('sampling', 'be with', 'homogeneous parameters')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('sampling', 'be with', 'homogeneous parameters')],\n",
       "  [('direct sampling', 'be with', 'complete observation')],\n",
       "  [('trees', 'be with', 'branching ratio k =')],\n",
       "  [('diﬀerent predicted extinction time distributions',\n",
       "    'be with',\n",
       "    'maximum value')],\n",
       "  [('direct sampling', 'be with', 'complete information')],\n",
       "  [('trees', 'be with', 'uniform epidemic parameters'),\n",
       "   ('trees', 'be with', 'branching ratio 3')],\n",
       "  [('random regular graphs =', 'on b be', 'µ ='),\n",
       "   ('random regular graphs =', 'be with', 'uniform epidemic parameters')],\n",
       "  [('direct sampling', 'be with', 'complete information')],\n",
       "  [('trees', 'be with', 'uniform epidemic parameters'),\n",
       "   ('trees', 'be with', 'branching ratio 3')],\n",
       "  [('b', 'be with', 'uniform epidemic parameters'),\n",
       "   ('N', 'of graph be', 'µ =')],\n",
       "  [('function', 'as area be', 'c'), ('ROC curve', 'under area be', 'c')],\n",
       "  [('Tobs =', 'at observation be', 'd'),\n",
       "   ('30 %', 'of observation be', 'd'),\n",
       "   ('SIR dynamics', 'for sampling be', 'green')],\n",
       "  [('function', 'as area be', 'c'), ('ROC curve', 'under area be', 'c')],\n",
       "  [('Tobs =', 'at observation be', 'd'),\n",
       "   ('30 %', 'of observation be', 'd'),\n",
       "   ('SIR dynamics', 'for sampling be', 'green')],\n",
       "  [('instances', 'be with', 'observed infected')]],\n",
       " 78: [[('Anhui University', 'spreading of', 'infectious disease'),\n",
       "   ('Anhui University', 'university of', 'Electronic Science'),\n",
       "   ('Anhui University', 'xi', 'Shan'),\n",
       "   ('epidemic spreading', 'be in', 'complex networks')],\n",
       "  [('responses', 'class of', 'individuals')],\n",
       "  [('response rate', 'be in', 'study')],\n",
       "  [('Via percolation method', 'formula for', 'epidemic threshold')],\n",
       "  [('analytical results', 'be also veriﬁed by', 'numerical simulations')],\n",
       "  [('reality', 'study', 'corresponding controlling measures')],\n",
       "  [('many eﬀorts', 'study', 'ﬁeld1 ,2')],\n",
       "  [],\n",
       "  [('they', 'can also trigger', 'ness of susceptible individual19 ,22')],\n",
       "  [('reduction', 'be in', 'transmissibility of infectious disease'),\n",
       "   ('classical S I R model', 'into class be', 'S F')],\n",
       "  [('new phenomena', 'may', 'may observed'),\n",
       "   ('phenomena', 'may', 'may observed')],\n",
       "  [('we', 'investigate', 'its spreading dynamics')],\n",
       "  [],\n",
       "  [('reasons', 'be present in', 'Sec')],\n",
       "  [('infected node', 'can enter r state with', 'recovery rate µ')],\n",
       "  [],\n",
       "  [('0 ≤ γ < 1', 'be', 'where nodes discount factor')],\n",
       "  [('γ', '=', '0')],\n",
       "  [('ANALYSIS', 'probability of', 'S'),\n",
       "   ('ANALYSIS', 'be in', 'our model'),\n",
       "   ('our model', 'in analysis be', 'probabilities of S'),\n",
       "   ('suﬃciently small time', 'during analysis be', 'probabilities of S')],\n",
       "  [('S F', 'of rate be', 'probability'),\n",
       "   ('transition rate', 'probability of', 'S F')],\n",
       "  [('we', 'analyze', 'our proposed model'), ('we', 'analyze', 'our model')],\n",
       "  [('i', 'of', 'infected neighbors'),\n",
       "   ('they', 'play role on', 'node i one'),\n",
       "   ('we', 'facilitate', 'analysis')],\n",
       "  [('result', 'as 9 be', 'probability of node i ﬁrst becoming')],\n",
       "  [('trivial solution θ =', 'be in', 'self-consistent equation')],\n",
       "  [('following condition', 'have', 'non-trivial solution')],\n",
       "  [('X k', '=', '0 P xk')],\n",
       "  [('SIMULATION RESULTS', 'be in', 'section'),\n",
       "   ('theoretical predictions', 'be in', 'Section III')],\n",
       "  [('labeled ER network 26', 'be with', 'network size N =')],\n",
       "  [('owing', 'to', 'the')],\n",
       "  [('critical point', 'be in', 'equilibrium phase on magnetic system31')],\n",
       "  [('we', 'predict', 'epidemic threshold')],\n",
       "  [('recovery rate µ', '=', '1.0')],\n",
       "  [('one', 'show in', 'Fig. 1')],\n",
       "  [('βF', 'be', 'increased'), ('βF', 'be', 'when increased')],\n",
       "  [('value', 'may', 'may larger than one'), ('value', 'may', 'may larger')],\n",
       "  [('γ', '=', '0.3')],\n",
       "  [('ER network', 'be in', 'Fig. 2'),\n",
       "   ('theoretical results',\n",
       "    'be with',\n",
       "    'Monte Carlo simulations on ER network in Fig. 2')],\n",
       "  [('Ineq', 'accord to ehki be', '20'), ('Ineq', 'accord to ehki be', 'x −')],\n",
       "  [('simulations', 'between r be', '∞')],\n",
       "  [('remarkable agreement', 'be with', 'numerical simulations')],\n",
       "  [('result', 'be in', 'Fig. 3')],\n",
       "  [('response rate βF', 'be', 'when larger')],\n",
       "  [('ER networks', 'on r be', '∞')],\n",
       "  [('denoted', 'be denote by', 'corresponding lines')],\n",
       "  [('theoretical results', 'substitute', 'Eq')],\n",
       "  [('values', 'be', 'where maximal')],\n",
       "  [('good agreement', 'be with', 'numerics'),\n",
       "   ('analytical results', 'be in', 'good agreement with numerics')],\n",
       "  [('UCM networks', 'on r be', '∞')],\n",
       "  [('denoted', 'be denote by', 'corresponding lines')],\n",
       "  [('values', 'be', 'where maximal')],\n",
       "  [('simulation', 'result base on', 'Eqs')],\n",
       "  [('βF', 'be', 'when reduced')],\n",
       "  [('infected nodes', 'of o be', 'ρI')],\n",
       "  [('O', 'to proportional be', 'ρ2 I')],\n",
       "  [('individuals', 'take', 'measures'),\n",
       "   ('their neighbors', 'be', 'infected'),\n",
       "   ('S S F I R epidemiological model', 'be in', 'complex networks'),\n",
       "   ('individuals', 'take', 'protective measures')],\n",
       "  [('discount factor γ', 'be', 'reduced')],\n",
       "  [('mean ﬁeld based analysis', 'provide', 'wrong result')],\n",
       "  [('second order inﬁnitesimal',\n",
       "    'be with',\n",
       "    'since development of technology')],\n",
       "  [('theory', 'be', 'ideal'),\n",
       "   ('may ideal framework', 'be in', 'view of this')],\n",
       "  [('challenge', 'be in', 'our further work')],\n",
       "  [('D. J. Earn', 'proceedings of', 'National Academy')],\n",
       "  [('D. J. Earn', 'proceedings of', 'National Academy')],\n",
       "  [('V. A. Jansen', 'journal of', 'Royal Society Interface 7 1247')],\n",
       "  [('M. Barth', 'elemy be', 'Physical Review E 73 046131')]],\n",
       " 79: [[('zombies', 'be in', 'popular culture ﬁlms')],\n",
       "  [('parameters', 'be modiﬁed to', 'diﬀerent infectious disease dynamics'),\n",
       "   ('parameters', 'with recover be', 'R')],\n",
       "  [('Shaun', 'for model be', 'E')],\n",
       "  [('2', 'be', 'new'),\n",
       "   ('parameter estimation', 'apply to', 'dynamical models'),\n",
       "   ('Bayesian parameter estimation', 'apply to', 'dynamical models')],\n",
       "  [('statistical model', 'use', 'methods')],\n",
       "  [('their', 'uncertainty', 'Citation')],\n",
       "  [('entertaining example',\n",
       "    'that of',\n",
       "    'impending so-called zombie apocalypse')],\n",
       "  [('methods', 'directly applicable to', 'real-world disease dynamics')],\n",
       "  [('Haitian Creole term zonbi', 'be in', 'Caribbean'),\n",
       "   ('North Mbundu term nzumbe', 'be in', 'Africa')],\n",
       "  [('depictions', 'be in', 'ﬁlms')],\n",
       "  [('ﬁrst', 'present in', 'Munz et')],\n",
       "  [('dynamics', 'exhibit in', 'their paper')],\n",
       "  [('Figure 1C', 'show', 'basic model proposed by Munz et')],\n",
       "  [('death', 'be in', 'zombie models')],\n",
       "  [('interaction', 'be with', 'susceptible')],\n",
       "  [('ﬁlms', 'destroy', 'zombie')],\n",
       "  [('we', 'refer to', 'to Munz et')],\n",
       "  [('removed population', 'inevitably recycle into', 'zombie subpopulation')],\n",
       "  [('Romeros', 'ﬁlms be', 'Citation'),\n",
       "   ('ﬁrst depiction', 'be in', 'popular culture')],\n",
       "  [('contact', 'be with', 'one')],\n",
       "  [('contact', 'be with', 'zombie')],\n",
       "  [('zombie', 'can', 'can permanently removed'), ('brain', 'burn', 'body')],\n",
       "  [('interaction', 'be with', 'zombie Z')],\n",
       "  [('term', 'be', 'removed'), ('removed subpopulation', 'be in', 'models')],\n",
       "  [('same function', 'be in', 'mathematically models')],\n",
       "  [('ﬁxed rate ζ', 'be in', 'relation to size of exposed subpopulation size')],\n",
       "  [('zombie apocalypse', 'take', 'days')],\n",
       "  [('rarity', 'be in', 'typical human population'),\n",
       "   ('rarity', 'be in', 'Romero ﬁlms'),\n",
       "   ('Romero ﬁlms', 'percent of', 'susceptibles')],\n",
       "  [('Dead', 'shaun of', 'the Dead'), ('ﬁlm', 'have', 'title')],\n",
       "  [('zombies', 'see in', 'previous model proposed'),\n",
       "   ('interaction', 'be with', 'susceptibles'),\n",
       "   ('zombie', 'have', 'brain'),\n",
       "   ('susceptible', 'destroy', \"zombie 's brain\")],\n",
       "  [('suicide', 'be', 'commonplace'), ('suicide', 'be', 'where commonplace')],\n",
       "  [('exposed people', 'zombie within', 'minutes'),\n",
       "   ('connection', 'be with', 'standard SIR model'),\n",
       "   ('exposed people', 'become', 'zombies within minutes')],\n",
       "  [('meter', 'square', 'area'), ('scenes', 'be with', 'ﬁeld of view')],\n",
       "  [('Dead Time', 'of shaun be', 'hours')],\n",
       "  [('uncertainty', 'be in', 'values'), ('parameters', 'be in', 'models')],\n",
       "  [('our models', 'be', 'included in supplemental materials')],\n",
       "  [('dynamical models',\n",
       "    'use',\n",
       "    'freely available wrapper around odeint function in Scipy'),\n",
       "   ('odeint function', 'be in', 'Scipy')],\n",
       "  [('complex dynamical system', 'be with', 'a minimal of technical syntax')],\n",
       "  [('stability', 'use in', '2009')],\n",
       "  [('we', 'do in', 'our models'), ('we', 'set', 'ζ')],\n",
       "  [('al.', 'be', 'direct')],\n",
       "  [('3.2 Parameter Distributions Shown', 'be in', 'Figures')],\n",
       "  [('summary statistics', 'be show in', 'Table 2')],\n",
       "  [('we', 'observe', 'joint distribution between two variables')],\n",
       "  [('diﬀer', 'be in', 'their values for rate of')],\n",
       "  [('Citation -2012', 'name', '95 % interval')],\n",
       "  [('models', 'be', 'signiﬁcant'), ('models', 'be in', 'Munz et')],\n",
       "  [('terms', 'in al. be', '2009')],\n",
       "  [('we', 'quantiﬁed', 'their uncertainty'),\n",
       "   ('we', 'use', 'Bayesian parameter estimation'),\n",
       "   ('we', 'use', 'parameter estimation')],\n",
       "  [('parameters', 'for simulation be', 'left'),\n",
       "   ('Night', 'for simulation be', 'left'),\n",
       "   ('we', 'use', 'exactly same techniques')],\n",
       "  [('Shaun', 'for simulation be', 'left'),\n",
       "   ('parameters', 'for distribution be', 'right')],\n",
       "  [('we', 'explore', 'this'),\n",
       "   ('zombie population', 'restore', 'civilization')],\n",
       "  [('Other hypothetical scenarios',\n",
       "    'can',\n",
       "    'can also explored in straightforward way')],\n",
       "  [('reasonable models', 'apply to', 'data on initial spread of disease')],\n",
       "  [('Seasonality', 'be in', 'epidemic model')],\n",
       "  [('parameter estimation', 'be in', 'dynamical models')],\n",
       "  [('AMC Studios', 'circle of', 'Confusion')],\n",
       "  [('β', 'for parameter be', 'data obtained from Google Trends')],\n",
       "  [('Error bars', 'show', '95 % credible regions for parameters')],\n",
       "  [('several seasons', 'through series be', 'bottom')],\n",
       "  [('Global stability analysis',\n",
       "    'be with',\n",
       "    'discretization approach for age-structured multigroup SIR epidemic model')],\n",
       "  [('SIR epidemic model', 'be with', 'nonlinear incidence')],\n",
       "  [],\n",
       "  [('Bayesian stochastic modelling', 'be in', 'python')],\n",
       "  [('Milho Film', 'news on', 'Request')]],\n",
       " 80: [[('us', 'more accurately describe', \"network 's topology\"),\n",
       "   ('network', 'have', 'topology')],\n",
       "  [('diﬀerences',\n",
       "    'be in',\n",
       "    'higher-order structure impact on epidemic threshold'),\n",
       "   ('we', 'generate', 'homogeneous networks')],\n",
       "  [('measuring',\n",
       "    'advance',\n",
       "    'our understanding of impact of network topology on dynamics unfolding on networks')],\n",
       "  [('modelling', 'be', 'essential')],\n",
       "  [('network', 'have', 'structure')],\n",
       "  [('such structures', 'eﬀect of be', 'signiﬁcant NumericCitation')],\n",
       "  [('excellent agreement',\n",
       "    'be with',\n",
       "    'values obtained from stochastic simulations')],\n",
       "  [('networks',\n",
       "    'be',\n",
       "    'By construction loop-less in limit of large network size')],\n",
       "  [('networks', 'be with', 'tunable clustering')],\n",
       "  [('Big-V', 'be', 'iterative'),\n",
       "   ('clustering', 'be commonly use by', 'network scientists NumericCitation'),\n",
       "   ('clustering', 'be', 'commonly used')],\n",
       "  [('x-y', 'be', 'created')],\n",
       "  [('clustering', 'be in', 'chain')],\n",
       "  [('2.1.2', 'be', 'iterative')],\n",
       "  [('global edge', 'cut', 'them')],\n",
       "  [('complete motif', 'be with', '4 members'),\n",
       "   ('process', 'be perform on', 'complete motif with 4 members')],\n",
       "  [],\n",
       "  [('corners', 'of conﬁguration be', '3')],\n",
       "  [('Fig. 3', 'illustrate', 'corner allocation for example node')],\n",
       "  [('quintuple', 'be arrange into', 'one complete square corner')],\n",
       "  [('φ', '=', '0.4')],\n",
       "  [('Table 1', 'show', 'expected motif allocation per node')],\n",
       "  [('degree/motif distribution',\n",
       "    'of pgf be',\n",
       "    'Probability Generating Function')],\n",
       "  [('initialise node', 'be with', 'ﬁve stubs')],\n",
       "  [('ﬁve stubs', 'form', 'lines')],\n",
       "  [('Pi pi', '=', '1')],\n",
       "  [('n', 'be', 'where corner-stub cardinality')],\n",
       "  [('number', 'be in', 'network NumericCitation')],\n",
       "  [('number', 'be in', 'network')],\n",
       "  [('clustering', 'be in', 'global sense')],\n",
       "  [('adjacency matrix', 'with share be', 'connection')],\n",
       "  [('Clustering', 'deﬁned in', 'two ways')],\n",
       "  [('8 Global clustering',\n",
       "    'be deﬁned as',\n",
       "    'ratio of total number of closed triples to total number of connected structures with 3 nodes'),\n",
       "   ('connected structures', 'be with', '3 nodes')],\n",
       "  [('network NumericCitation', 'of set be', 'geodesics'),\n",
       "   ('shortest paths', 'of set be', 'geodesics'),\n",
       "   ('node', 'how often appear in', 'geodesics')],\n",
       "  [('Nodes', 'be with', 'high betweenness centrality')],\n",
       "  [('that', 'contain', 'node n')],\n",
       "  [('removal', 'be with', 'high betweenness centrality')],\n",
       "  [('clustered networks', 'of gcc be', 'Giant Connected Component')],\n",
       "  [('we', 'remove', 'it'),\n",
       "   ('we', 'plot', 'cumulative distribution of component size'),\n",
       "   ('network', 'cycle through', 'edge in list')],\n",
       "  [('it', 'be', 'coarse'), ('it', 'be', 'very coarse measurement')],\n",
       "  [('It', 'deﬁne', 'new clustering type metrics'),\n",
       "   ('new clustering type metrics', 'use', 'structures'),\n",
       "   ('It', 'be', 'possible'),\n",
       "   ('new clustering type metrics', 'use', 'structures larger than triangles')],\n",
       "  [('connected structures', 'to square be', 'φ2 4'),\n",
       "   ('it', 'deﬁne', 'four new structural measurements'),\n",
       "   ('it', 'deﬁne', 'four new measurements')],\n",
       "  [('our results', 'be in', 'two formats')],\n",
       "  [('number', 'be in', 'square')],\n",
       "  [('networks', 'be with', 'same degree distribution')],\n",
       "  [('Previous work NumericCitation',\n",
       "    'predict',\n",
       "    'eﬀect of order-four structure on epidemic dynamics'),\n",
       "   ('Kirkwood', 'have', 'superposition approximation')],\n",
       "  [('networks',\n",
       "    'be with',\n",
       "    'markedly diﬀerent order-four motif distributions')],\n",
       "  [('networks', 'be with', 'same degree distribution')],\n",
       "  [('Gephi NumericCitation',\n",
       "    'visualize',\n",
       "    'sample networks generated by proposed algorithms')],\n",
       "  [('rewiring algorithms construct',\n",
       "    'be with',\n",
       "    'more heterogeneity in clustering'),\n",
       "   ('more heterogeneity', 'be in', 'clustering')],\n",
       "  [('diﬀerence',\n",
       "    'translate into',\n",
       "    'more modular structure for rewired networks')],\n",
       "  [('stubs', 'be with', 'probability p1')],\n",
       "  [('clustering', 'be in', 'range 0.3')],\n",
       "  [('largest spread', 'be with', 'half of nodes having'),\n",
       "   ('clustering', 'be in', 'range 0.2 ≤ φ ≤ 0.6')],\n",
       "  [('local clustering', 'of box-plot be', 'Fig. 5')],\n",
       "  [('complete motifs',\n",
       "    'must',\n",
       "    'must compensated with other parts of network decomposed into much more random graph-type structure')],\n",
       "  [('betweenness centrality plot',\n",
       "    'show',\n",
       "    'slightly higher spread for MD networks')],\n",
       "  [('nodes', 'be with', 'high betweenness centrality rank')],\n",
       "  [('connected network', 'be with', 'low clustering')],\n",
       "  [('above table', 'use', 'unique counts')],\n",
       "  [('Networks', 'be with', 'low clustering')],\n",
       "  [('Big-V algorithm',\n",
       "    'still exhibit',\n",
       "    'mild sensitivity to removal of edges')],\n",
       "  [('diﬀerence', 'be in', 'φ1 4')],\n",
       "  [('we', 'read', 'column-wise'),\n",
       "   ('squares', 'be with', 'diagonals'),\n",
       "   ('we', 'read', 'column-wise down Fig. 8')],\n",
       "  [('greatest heterogeneity', 'be in', 'distribution of diagonal squares')],\n",
       "  [('φ', '=', '0.8')],\n",
       "  [('they', 'be', 'when members of complete')],\n",
       "  [('networks', 'be with', 'φ =')],\n",
       "  [('triangle corner', 'be with', 'p =')],\n",
       "  [('diﬀerence', 'be with', 'less dramatic eﬀect for S I S epidemics')],\n",
       "  [('Nodes', 'be with', 'high betweenness centrality')],\n",
       "  [('nodes', 'be with', 'high betweenness centrality')],\n",
       "  [('MD networks', 'be particularly dependent on', 'isolated motifs'),\n",
       "   ('this', 'be reﬂected in', 'consistently smaller ﬁnal epidemic size')],\n",
       "  [('heterogeneity', 'be in', 'contact')],\n",
       "  [('model development', 'of stage be', 'propensity')],\n",
       "  [('Many network models', 'use', 'synthetic networks')],\n",
       "  [('properties', 'have', 'eﬀect on higher-order structure'),\n",
       "   ('this', 'be', 'can signiﬁcant')],\n",
       "  [('network', 'be', 'coherent')],\n",
       "  [],\n",
       "  [('CCM', 'be long way from', 'randomly introduced clustering')],\n",
       "  [('networks', 'be with', 'same degree distribution of clustering')],\n",
       "  [('diﬀerences', 'be in', 'higher-order structures')],\n",
       "  [('use', 'be in', 'low-dimensional ODE approximations such pairwise model')],\n",
       "  [('Global clustering', 'use', 'purely unique counts'),\n",
       "   ('unique counts', 'be', 'when used'),\n",
       "   ('clustering', 'use', 'purely unique counts')],\n",
       "  [('use', 'be in', 'clustering-type ratios')],\n",
       "  [('square', 'be with', 'diagonal')],\n",
       "  [('hyper-node', 'be with', 'n edges'), ('dt d Q4', '=', '6λQ5 − 5λQ4')],\n",
       "  [('hyper-graph counting', 'quickly become', 'too tedious'),\n",
       "   ('Such hyper-graph counting', 'quickly become', 'too tedious')],\n",
       "  [('quantities', 'be in', 'Table 1')],\n",
       "  [('3 3 4 2 0 Ti 0 6 0 12 24 Table 1', 'number of', 'local links li'),\n",
       "   ('3 3 4 2 0 Ti 0 6 0 12 24 Table 1', 'number of', 'global stubs σi')],\n",
       "  [('P', 'be', 'ordered')],\n",
       "  [],\n",
       "  [('P', 'be in', 'P Ll')],\n",
       "  [],\n",
       "  [('undirected network', 'be with', 'self loops')],\n",
       "  [('real edges', 'be in', 'network')],\n",
       "  [('It', 'count', 'more complex paths'), ('k', '=', '1')],\n",
       "  [('It', 'count', 'six diﬀerent')],\n",
       "  [('it', 'count', 'structures'), ('it', 'count', 'order-four structures')],\n",
       "  [('5 nodes', 'be with', '4 edges')],\n",
       "  [('homogeneous graph', 'be with', 'k')],\n",
       "  [('node', 'be initially allocate', 'quintuple of stubs')],\n",
       "  [('will part', 'be with', 'probability p1')],\n",
       "  [('φ', '=', '0.4'), ('Homogeneous networks', 'be with', 'parameters')],\n",
       "  [('nodes', 'have', 'low local clustering')],\n",
       "  [('Local clustering',\n",
       "    'be',\n",
       "    'measure of interconnectivity between neighbours of given node')],\n",
       "  [('MD', 'provide', 'largest variance in local clustering'),\n",
       "   ('largest variance', 'be in', 'local clustering')],\n",
       "  [('Betweenness centrality', 'rank', 'nodes')],\n",
       "  [('clustering', 'be', 'tuned higher')],\n",
       "  [('network', 'be with', 'probability pr')],\n",
       "  [('homogeneous networks', 'be with', 'hki ='), ('Results', 'be', 'taken')],\n",
       "  [('pr increases', 'be in', 'clockwise direction')],\n",
       "  [('motifs', 'be', 'how counted'), ('motifs', 'be', 'counted')],\n",
       "  [('hki', '=', '5'), ('φ', '=', '0.0018')],\n",
       "  [('hki', '=', '5')],\n",
       "  [('epidemics', 'be seed with', '5 infectious nodes')],\n",
       "  [('ten Gillespie simulations', 'be perform for', 'value of τ'),\n",
       "   ('Twenty networks', 'be', 'generated')]],\n",
       " 81: [[('Models', 'be', 'traditional'),\n",
       "   ('traditional topic', 'be in', 'mathematical epidemiology')],\n",
       "  [('disease', 'lose', 'immunity against disease-causing pathogen'),\n",
       "   ('Bolyai Institute', 'university of', 'Szeged'),\n",
       "   ('Bolyai Institute', 'university of', 'Szeged')],\n",
       "  [('contact', 'be with', 'infective one')],\n",
       "  [('he', 'afterwards possibly become again', 'susceptible'),\n",
       "   ('he', 'afterwards possibly become', 'susceptible')],\n",
       "  [('newborn', 'be immune for', 'several months after birth NumericCitation'),\n",
       "   ('one', 'be', 'transient')],\n",
       "  [('immunization', 'be achieve by', 'vaccine administration NumericCitation'),\n",
       "   ('Active immunization',\n",
       "    'be achieve by',\n",
       "    'vaccine administration NumericCitation')],\n",
       "  [('us', 'consider', 'case of natural infection')],\n",
       "  [('These', 'be activate in case of', 'secondary infection')],\n",
       "  [('individuals', 'be with', 'diﬀerent levels of immunity')],\n",
       "  [('newborns', 'be with', 'passive immunity')],\n",
       "  [('level', 'by immunity be', 'W'),\n",
       "   ('SIRWS model', 'be in', 'NumericCitation')],\n",
       "  [('authors', 'describe', 'sole waning immunity process')],\n",
       "  [('delay', 'represent', 'average duration of disease-induced immunity')],\n",
       "  [('immune population', 'be structure by', 'level of immunity')],\n",
       "  [('physiological structure', 'be in', 'immune population')],\n",
       "  [('it', 'be', 'done'), ('hosts', 'be with', 'waning immunity')],\n",
       "  [('abstract Cauchy problem', 'be in', 'state-space X')],\n",
       "  [('we', 'use', 'appropriate comparison system'),\n",
       "   ('we', 'determine', 'invariant subset in X'),\n",
       "   ('invariant subset', 'be in', 'X')],\n",
       "  [('connection', 'be with', 'theory of Volterra integral equations')],\n",
       "  [('those', 'such as model be', 'M1'),\n",
       "   ('systems', 'be with', 'constant delay')],\n",
       "  [('new class', 'be with', 'delay')],\n",
       "  [('we', 'derive', 'model equations for SIRS dynamics'),\n",
       "   ('2 new class', 'be in', 'section')],\n",
       "  [('we', 'obtain from', 'discrete approach'),\n",
       "   ('infectives', 'can', 'can easily introduced'),\n",
       "   ('infectives', 'can', 'can introduced')],\n",
       "  [('We', 'conclude session with', 'remarks on total immune population')],\n",
       "  [('Memory cells', 'be', 'activated'),\n",
       "   ('Memory cells', 'be activate in case of', 'secondary infection')],\n",
       "  [('N 7 → b', 'be with', '0 < b + < ∞'),\n",
       "   ('b', 'be', 'nonnegative'),\n",
       "   ('0 < b', 'with b be', 'N'),\n",
       "   ('0 nonnegative C 1', 'be with', 'b =')],\n",
       "  [('Contact', 'be with', 'infectives'),\n",
       "   ('susceptible hosts', 'become', 'infective themselves')],\n",
       "  [('1/γ', 'be', 'average infection duration')],\n",
       "  [('individuals', 'once recover from', 'infection')],\n",
       "  [('immunity loss', 'become again', 'susceptible'),\n",
       "   ('immunity loss', 'become', 'susceptible')],\n",
       "  [('t', 'be with', 'immunity level')],\n",
       "  [('zmin', 'correspond to', 'low level of immunity')],\n",
       "  [('contact', 'be with', 'infectives')],\n",
       "  [('special case', 'be in', 'Sect. 3.1')],\n",
       "  [('immunity levels', 'be in', 'interval')],\n",
       "  [('NumericCitation', 'be', 'diﬀerentiable'),\n",
       "   ('host', 'be with', 'immunity level y')],\n",
       "  [('NumericCitation', 'be', 'diﬀerentiable'),\n",
       "   ('infectives', 'boost to', 'other level'),\n",
       "   ('host', 'be with', 'immunity level y')],\n",
       "  [('immunity level', 'be in', 'at time'),\n",
       "   ('time', 'at | be', 'z'),\n",
       "   ('total number', 'be with', 'immunity level in at time')],\n",
       "  [('we', 'let', '∆ t → 0')],\n",
       "  [('immune hosts', 'of g be', 'zmin')],\n",
       "  [('time t recruitment rate natural death rate disease-induced death rate transmission rate recovery rate g',\n",
       "    'at n be',\n",
       "    't'),\n",
       "   ('zmax zmin ψ maximal level', 'be in', 'immune hosts'),\n",
       "   ('t I N', '2 with', 'initial values S'),\n",
       "   ('immune individuals', 'be with', 'immunity level'),\n",
       "   ('minimal level', 'be in', 'immune hosts')],\n",
       "  [('quantities', 'be', 'nonnegative')],\n",
       "  [('ﬁrst glance equation', 'at r be', '4')],\n",
       "  [('immune population', 'for pde be', '3')],\n",
       "  [('model analysis',\n",
       "    'result',\n",
       "    'much more challenging than one of classical size-structured models')],\n",
       "  [('immune individuals', 'of r be', 't')],\n",
       "  [('contact', 'be with', 'infectives')],\n",
       "  [('terms', 'be in', 'square bracket cancel')],\n",
       "  [],\n",
       "  [('Y → Y', 'φ by', 'F'), ('β', 'be with', 'π')],\n",
       "  [('r', 'for condition be', '7')],\n",
       "  [('A', 'of d be', 'A')],\n",
       "  [('k · kOP', 'be', 'where operator norm')],\n",
       "  [('long computation', 'include in', 'Appendix')],\n",
       "  [('it', 'prove', 'Lipschitz continuity of Q'), ('it', 'be', 'suﬃcient')],\n",
       "  [('We', 'proceed in', 'two steps')],\n",
       "  [('boosting', 'restore', 'maximal immune status'),\n",
       "   ('we', 'first introduce', 'M2')],\n",
       "  [('comparison system', 'be in', 'proof of nonnegativity'),\n",
       "   ('PDE', 'be in', 'model')],\n",
       "  [('contact', 'be with', 'pathogen')],\n",
       "  [('model M1', 'in equation be', '3'), ('equation', 'be in', 'model M1')],\n",
       "  [('We', 'shall refer to', '10'),\n",
       "   ('model', 'as to system be', '10'),\n",
       "   ('model', 'as to system be', '9'),\n",
       "   ('model', 'as to system be', '2')],\n",
       "  [('0 T ×', 'on problem be', '9'),\n",
       "   ('nonnegative function r', 'be with', 'r integrable')],\n",
       "  [('cone D', 'be positively invariant for', 'model M1')],\n",
       "  [('We', 'start with', 'infective population')],\n",
       "  [('0 ≥ 0', 'be', 'given')],\n",
       "  [('PDE', 'of r be', 't')],\n",
       "  [('we', 'show', 'that'),\n",
       "   ('we', 'conclude', 'proof'),\n",
       "   ('t ≥ 0', 'for r be', 't')],\n",
       "  [('ﬁrst time', 'be in', 'other words')],\n",
       "  [('model M1', 'in pde be', '3'), ('PDE', 'be in', 'model M1')],\n",
       "  [('we', 'complete', 'proof')],\n",
       "  [('we', 'finally let', 'ǫ → 0')],\n",
       "  [('ψ ≥ 0', 'for r be', 't')],\n",
       "  [('problem 0', '=', 'N ∗')],\n",
       "  [('solution', 'be', 'triple')],\n",
       "  [('I', '∗', '> 0')],\n",
       "  [('we', 'present', 'further results'), ('we', 'present', 'results')],\n",
       "  [('reference parameter', 'be in', 'mathematical epidemiology used'),\n",
       "   ('disease', 'will spread among', 'population')],\n",
       "  [('Proposition 1', 'existence of', 'DFE')],\n",
       "  [('R ∗', '=', 'N ∗')],\n",
       "  [('N ∗', '=', 'N ∗')],\n",
       "  [('We', 'prove', 'stability from ﬁrst principle')],\n",
       "  [('us', 'deﬁne', 'M')],\n",
       "  [('we', 'now consider', 'susceptible population')],\n",
       "  [('It', 'show', 'that')],\n",
       "  [('solution', 'be with', '˘ν =')],\n",
       "  [('disease free equiIn particular', 'librium in', 'X'),\n",
       "   ('3 librium', 'be in', 'X')],\n",
       "  [('t S', 'n for', 't >'), ('t >', 'for n be', 't')],\n",
       "  [('we', 'now show', 'N ∞ ≥ η')],\n",
       "  [('ODE x ′ C t', 'be with', 'A'), ('A', 'with t be', '17')],\n",
       "  [('t t ∗', 'be with', 'x =')],\n",
       "  [('We', 'conclude', 'N ∞ ≥ η')],\n",
       "  [('t =', 'with equation be', '16'), ('equation', 'be with', 't =')],\n",
       "  [('proof', 'be', 'complete')],\n",
       "  [('18 Proposition 2', 'instability of', 'DFE')],\n",
       "  [('DFE', 'about model be', 'M1')],\n",
       "  [('5 Connection', 'be in', 'Sect. 1'),\n",
       "   ('5 Connection', 'be with', 'ODE models')],\n",
       "  [('we', 'can obtain', 'such ODE systems'), ('our', 'model', 'M1')],\n",
       "  [('zj j ∈ N', 'be with', 'hj'),\n",
       "   ('demonstration', 'as simple as', 'possible')],\n",
       "  [('r immune hosts', 'be with', 'high level of immunity')],\n",
       "  [('r immune hosts', 'be with', 'intermediate level of immunity')],\n",
       "  [('immune system', 'move to', 'RF')],\n",
       "  [('r immune hosts', 'be with', 'critically low level of immunity'),\n",
       "   ('RF', 'to probability be', '1 − θ'),\n",
       "   ('t.', 'be with', 'probability θ boosting to RW')],\n",
       "  [('dr', '∂ with', 'boundary condition Rzmax'),\n",
       "   ('r', 'be in', 'model M1'),\n",
       "   ('∂', 'be with', 'boundary condition Rzmax')],\n",
       "  [('we', 'analogously ﬁnd', 'equations')],\n",
       "  [('individuals', 'be protect for', 'while')],\n",
       "  [('integral term', 'be in', 'model')],\n",
       "  [('boosting', 'always restore', 'maximal level of immunity')],\n",
       "  [('θ', '=', '0')],\n",
       "  [('21 6 Connection', 'be with', 'DDE models Delay models'),\n",
       "   ('21 6 Connection', 'be with', 'constant delay')],\n",
       "  [('new SIS model', 'be with', 'delay'), ('model', 'be with', 'delay')],\n",
       "  [('6.1 SIRS', 'be with', 'constant delay')],\n",
       "  [('we', 'have', '6')],\n",
       "  [('model assumptions', 'time after', 'recovery immune hosts')],\n",
       "  [('he exits', 'be in', 'other words'), ('he', 'enter', 'S')],\n",
       "  [('delay term', 'be in', 'equation'),\n",
       "   ('classical SIRS model', 'be with', 'constant delay S ′')],\n",
       "  [('22 6.2 new class', 'be with', 'constant delay'),\n",
       "   ('boosting', 'restore', 'maximal level of immunity')],\n",
       "  [('contact', 'be with', 'infectives')],\n",
       "  [('immune hosts equation', 'be in', 'model M1'),\n",
       "   ('model M1', 'in equation be', '3')],\n",
       "  [('models', 'be with', 'constant delay'), ('that', 'be', 'general')],\n",
       "  [('immune population', 'be in', 'model M1')],\n",
       "  [('method',\n",
       "    'however provide',\n",
       "    'only rough approximation of structured population')],\n",
       "  [('Models', 'be with', 'constant delay such those in NumericCitation'),\n",
       "   ('Models', 'be obtain from', 'M1')],\n",
       "  [('new class', 'be with', 'constant delay')],\n",
       "  [('state-dependent type', 'be in', 'case'),\n",
       "   ('obtained delay', 'be of', 'In case indeed state-dependent type')],\n",
       "  [('connection', 'be with', 'state-dependent delay')],\n",
       "  [('measles', 'be in', 'humans'),\n",
       "   ('example NumericCitation', 'be in', 'mice')],\n",
       "  [('more individuals', 'be with', 'low level of immunity'),\n",
       "   ('individuals', 'be with', 'vaccine-induced immunity')],\n",
       "  [('e. g. measles', 'be in', 'highly vaccinated populations')],\n",
       "  [('vaccine-induced immunity', 'be in', 'our framework')],\n",
       "  [('various models', 'be in', 'literature'),\n",
       "   ('example', 'be in', 'NumericCitation')],\n",
       "  [('A/2 -11-1-2012-0001 Naframework ropean Social Fund',\n",
       "    'be in',\n",
       "    'tional Excel lence Program')],\n",
       "  [('GR', 'be', 'supported')],\n",
       "  [('25 Proof', 'be in', 'following')],\n",
       "  [('k · kOP', 'be', 'where operator norm')],\n",
       "  [('second term', 'be in', 'Q1 x')],\n",
       "  [('ﬁrst term', 'be in', 'Q3 x'), ('Z zmax zmin', '=', 'u')],\n",
       "  [('Q3 x', 'in term be', 'P4 ='), ('last term', 'be in', 'Q3 x')],\n",
       "  [('DQ kX', 'be with', 'T1')],\n",
       "  [('w → 0', '≤ for', 'kx − ykX → 0'),\n",
       "   ('It', 'be', 'obvious'),\n",
       "   ('b', 'as w be', 'kwkX ≤ 1 for kx − ykX → 0')],\n",
       "  [('term', 'be in', 'T2')],\n",
       "  [('it', 'proceed to', 'next estimate'), ('it', 'observe', 'that')],\n",
       "  [('last addend', 'be in', 'T4')],\n",
       "  [('We', 'have', 'that')],\n",
       "  [('proof', 'be', 'complete')],\n",
       "  [('References NumericCitation Amanna IJ',\n",
       "    'duration of',\n",
       "    'humoral immunity to common viral antigens')],\n",
       "  [('2006 Time delays', 'be in', 'epidemic models')],\n",
       "  [('578 NumericCitation Bhattacharya S', 'fr', 'time'),\n",
       "   ('recovery model', 'be with', 'varying rates of loss of immunity'),\n",
       "   ('578 NumericCitation Bhattacharya S',\n",
       "    'fr since',\n",
       "    'recovery model with varying rates of loss of immunity'),\n",
       "   ('time', 'fr', '2012'),\n",
       "   ('recovery model', 'since fr be', '2012')],\n",
       "  [('2819 NumericCitation Blyuss KB',\n",
       "    'be in',\n",
       "    'epidemic model with varying immunity period'),\n",
       "   ('epidemic model', 'be with', 'varying immunity period')],\n",
       "  [('490 505 NumericCitation Calsina Farkas JZ Steady states',\n",
       "    'be in',\n",
       "    'structured epidemic model with Wentzell boundary condition'),\n",
       "   ('structured epidemic model', 'be with', 'Wentzell boundary condition')],\n",
       "  [('model', 'be with', 'nonlinear individual growth rate')],\n",
       "  [('2012 dynamThe ics', 'be with', 'immune boosting'),\n",
       "   ('MP', 'inﬂuence of', 'increasing life expectancy on JM')],\n",
       "  [('Grenfell BT Antibody dynamics', 'be in', 'childhood diseases'),\n",
       "   ('University', 'dynamics in', 'childhood diseases')],\n",
       "  [('delayed SIR model', 'be with', 'temporary immunity')],\n",
       "  [('S',\n",
       "    'signature of',\n",
       "    'antibody responses derived from systems biology study')],\n",
       "  [('incomplete Freund', 'with', 'adjuvant hydroxide')],\n",
       "  [('result', 'as j be', 'Muller CP Modelling measles re-emergence'),\n",
       "   ('immunity', 'be in', 'vaccinated populations')],\n",
       "  [('impact', 'be in', 'vaccinated populations')],\n",
       "  [('SIR epidemic model', 'be with', 'partial temporary immunity modeled')],\n",
       "  [('Structured Population Models', 'be in', 'Biology')],\n",
       "  [('SEIRS model', 'be with', 'latency'),\n",
       "   ('2013 Threshold dynamics', 'be in', 'SEIRS model with latency')]],\n",
       " 82: [[('Elad Yom-Tov Microsoft Israel',\n",
       "    'model for',\n",
       "    'spread of pathogens in population')],\n",
       "  [('multiple pathogens', 'be with', 'similar symptoms')],\n",
       "  [('Our model',\n",
       "    'allow',\n",
       "    'modeling of population infected by multiple viruses')],\n",
       "  [('population', 'in partition be', 'each representing'),\n",
       "   ('partition', 'be in', 'population into individual components')],\n",
       "  [('average', '% of', 'variance in ILI time series'),\n",
       "   ('our method', 'describe on', '44 % of variance in ILI time series'),\n",
       "   ('variance', 'be in', 'ILI time series')],\n",
       "  [('known viral proﬁles', 'be in', 'populations')],\n",
       "  [('matched components', 'of number be', 'R0'),\n",
       "   ('matched components',\n",
       "    'basic reproductive number of be',\n",
       "    'range known for pathogens')],\n",
       "  [('models', 'be commonly know as', 'SIR')],\n",
       "  [('SIR models', 'model', 'individual pathogens'),\n",
       "   ('SIR models', 'model', 'pathogens')],\n",
       "  [('approach', 'be', 'data-driven')],\n",
       "  [('ILI', 'include', 'diseases'),\n",
       "   ('diseases',\n",
       "    'be with',\n",
       "    'similar symptoms such respiratory syncytial virus')],\n",
       "  [('research', 'bridge', 'gap')],\n",
       "  [('it', 'include', 'more than few strains'),\n",
       "   ('Modeling', 'be in', 'simple history-based model')],\n",
       "  [('we', 'use', 'ensemble of SIR models')],\n",
       "  [('resulting models', 'correspond to', 'known pathogens')],\n",
       "  [('evolution', 'be in', 'population')],\n",
       "  [('case', 'be in', 'this')],\n",
       "  [('speciﬁc elements', 'be in', 'matrices')],\n",
       "  [('speciﬁc elements', 'be in', 'S'), ('its likelihood', 'be', 'negligible')],\n",
       "  [('β', 'be in', 'its matrix form accounts for infection rate')],\n",
       "  [('us', 'capture', 'composite seasonal dynamics')],\n",
       "  [('simple dynamic', 'be in', 'which')],\n",
       "  [('N', 'be preserve for', 'virus type independently')],\n",
       "  [('increase', 'be in', ','),\n",
       "   ('infection', 'be in', 'people infected from other viruses')],\n",
       "  [('inﬂuenza-like illnesses', 'use', '2'), ('illnesses', 'use', '2')],\n",
       "  [('model', 'be', 'how ﬁt to real seasonal data')],\n",
       "  [('coordinates', 'be within', 'England'),\n",
       "   ('GPS', 'coordinate', 'embedded in them'),\n",
       "   ('Tweets', 'originate in', 'England')],\n",
       "  [('them', 'contain', 'only most informative of terms')],\n",
       "  [('largest weight', 'be in', 'model')],\n",
       "  [('terms', 'be list in', 'Table II')],\n",
       "  [('Terms', 'use for', 'ﬁrst stage tweet ﬁltering 2')],\n",
       "  [('We',\n",
       "    'extract',\n",
       "    'weekly ILI rate reported from RCPG in PHE Weekly National Inﬂuenza Reports'),\n",
       "   ('RCPG', 'be in', 'PHE Weekly National Inﬂuenza Reports')],\n",
       "  [('we', 'report by', 'Respiratory DataMart System')],\n",
       "  [('fraction', 'be in', 'population')],\n",
       "  [('solutions', 'be with', 'diﬀerent parameters')],\n",
       "  [('days', 'be in', 'season')],\n",
       "  [('squared error', 'be', 'In our analysis used as criteria for matching')],\n",
       "  [('result', 'be set of', 'functions from dictionary')],\n",
       "  [('time', 'be with', 'SIR components')],\n",
       "  [('value',\n",
       "    'be with',\n",
       "    'peak value of SIR component multiplied by gain parameter'),\n",
       "   ('its positivity', 'be in', 'population')],\n",
       "  [('resulting model parameters', 'be show in', 'Table V')],\n",
       "  [('reported value', 'be in', 'PHE')],\n",
       "  [('spread', 'be in', 'population')],\n",
       "  [('multiple pathogens', 'be with', 'similar symptoms')],\n",
       "  [('daily variance', 'be in', 'predicted ILI')],\n",
       "  [('N', 'thus be constant for', 'virus type')],\n",
       "  [('non-diagonal elements', 'be in', 'model')],\n",
       "  [('non-diagonal elements', 'be in', 'β')],\n",
       "  [('known viral proﬁles', 'be in', 'populations')],\n",
       "  [('matched components', 'be indeed relate to', 'underlying pathogens')],\n",
       "  [('range', 'know for', 'pathogens')],\n",
       "  [('median R0', 'be in', 'ﬂuenza')],\n",
       "  [('R0', 'be in', 'inﬂuenza')],\n",
       "  [('diﬀerences', 'be in', 'population demographics')],\n",
       "  [('NumericCitation W. O. Kermack',\n",
       "    'be in',\n",
       "    'Proceedings of Royal Society of London A')],\n",
       "  [('S. A. Levin', 'journal of', 'mathematical biology 35 825')],\n",
       "  [('B. T. Grenfell', 'proceedings of', 'National Academy')],\n",
       "  [('NumericCitation P. Cr', 'epey be', 'P. T. Boer'),\n",
       "   ('NumericCitation P. Cr', 'epey be', 'M. J. Postma')],\n",
       "  [('W. M. Liu', 'journal of', 'mathematical biology 27 233')],\n",
       "  [('J. Gog', 'journal of', 'mathematical biology 45 471')],\n",
       "  [('J. Swinton', 'journal of', 'mathematical biology 44 169')],\n",
       "  [('Scientiﬁc', 'report', '12760 2015')],\n",
       "  [('J. Shaman', 'journal of', 'Epidemiology 185 395')],\n",
       "  [('D. J. Nokes', 'proceedings of', 'Royal Society')],\n",
       "  [('B. T. Grenfell', 'proceedings of', 'National Academy')],\n",
       "  [('models', 'be in', 'Mathematical approaches emerging')],\n",
       "  [('N. Ferguson', 'journal of', 'Royal Society Interface')],\n",
       "  [('S. Franz', 'journal of', 'Statistical Mechanics')],\n",
       "  [('A. G. McKendrick', 'proceedings of', 'Royal Society')],\n",
       "  [('NumericCitation S. G. Mallat', 'transaction on', 'signal processing')]],\n",
       " 83: [[('latter', 'have', 'corresponding linear approximation'),\n",
       "   ('interesting connection',\n",
       "    'be with',\n",
       "    'its 2n-state nonlinear mean-ﬁeld approximation')],\n",
       "  [('state', 'be', 'ﬁxed'), ('state', 'be', 'stable')],\n",
       "  [('threshold condition', 'be', 'when violated')],\n",
       "  [('immunization', 'improve', 'threshold of epidemic eradication')],\n",
       "  [('ﬁxed-points', 'cost of', 'epidemic')],\n",
       "  [('epoch', 'node in', 'susceptible state'),\n",
       "   ('independent events', 'be with', 'probability β'),\n",
       "   ('nodes', 'be in', 'susceptible state')],\n",
       "  [('nodes', 'be in', 'recovered state'),\n",
       "   ('nodes can transition', 'be with', 'probability γ')],\n",
       "  [('random vaccinations', 'be with', 'probability θ')],\n",
       "  [('Electrical Engineering', 'department of', 'Electrical Engineering'),\n",
       "   ('Electrical Engineering', 'institute of', 'Technology'),\n",
       "   ('Markov chain', 'be with', '3n states'),\n",
       "   ('Electrical Engineering', 'university of', 'Southern California'),\n",
       "   ('at network', 'be with', 'n nodes')],\n",
       "  [('we',\n",
       "    'provide',\n",
       "    'complete global analysis of epidemic dynamics for nonlinear mean-ﬁeld model')],\n",
       "  [('connected network G', 'be with', 'adjacency matrix A'),\n",
       "   ('Ni', 'set of', 'neighbors of node i'),\n",
       "   ('n', 'represent', 'number of nodes')],\n",
       "  [('β', 'be', 'healing probability'),\n",
       "   ('β', 'be', 'immunization γ δ Fig. 1'),\n",
       "   ('single node', 'be in', 'SIRS model')],\n",
       "  [('Wavy arrow', 'represent', 'transition')],\n",
       "  [('S', 'get', 'infected')],\n",
       "  [('Point', 'be', 'ﬁxed')],\n",
       "  [('eigenvalues', 'be in', 'of always βA')],\n",
       "  [('1 − δ', 'be in', 'βA')],\n",
       "  [('+ βA PI', 'in pi be', 'j'),\n",
       "   ('+ βA PI', 'in pi be', '18'),\n",
       "   ('+ βA PI', 'in pi be', 'i'),\n",
       "   ('1 − δ PI', 'be in', '+ βA PI'),\n",
       "   ('+ βA PI', 'in pi be', '17'),\n",
       "   ('upperbound', 'be in', 'addition')],\n",
       "  [('MC', 'of model be', '15'), ('We', 'further show for', '15')],\n",
       "  [('result', 'have', 'has shown for SIS model in NumericCitation'),\n",
       "   ('SIS model', 'be in', 'NumericCitation')],\n",
       "  [('pR', 'tor of', 'Markov chain'),\n",
       "   ('probability vecis', 'be in', 'following forms')],\n",
       "  [('we', 'show', 't')],\n",
       "  [('fi ∈ R2n +1', 'represent', 'ith unit column vector')],\n",
       "  [('Xi', '=', '0'),\n",
       "   ('Xi n', '=', '1'),\n",
       "   ('Xi − n', '=', '1'),\n",
       "   ('Xi n', '=', '0'),\n",
       "   ('Xi − n', '=', '0'),\n",
       "   ('Xi − n', '=', '2')],\n",
       "  [('maps', 'deﬁned above follow', 'properties'), ('i', 'be', 'increasing')],\n",
       "  [('Ψ =', 'be with', 'Ψi')],\n",
       "  [('Ui', 'of point be', '57'), ('we', 'follow to', '57')],\n",
       "  [('It', 'imply', 'i')],\n",
       "  [('it', 'should satisfy', 'Eq')],\n",
       "  [('∗', 'be', 'zi U')],\n",
       "  [('nonlinear map', 'of point be', 'i ='),\n",
       "   ('nonlinear map', 'of point be', '⇒ p ∗ R'),\n",
       "   ('nonlinear map', 'of point be', 'i'),\n",
       "   ('nonlinear map', 'of point be', 'i.e. p ∗ R'),\n",
       "   ('nonlinear map', 'of point be', '52')],\n",
       "  [('us', 'deﬁne', 'K0')],\n",
       "  [('G', 'be', 'connected'), ('j', 'be neighbor of', 'k')],\n",
       "  [('K0', '=', '∅')],\n",
       "  [('Lemma 2.1', 'be in', 'NumericCitation')],\n",
       "  [('Ψ', 'stability of', 'Nontrivial Fixed Point')],\n",
       "  [('second ﬁxed point',\n",
       "    'furthermore be indeed globally stable in',\n",
       "    'variation of SIS model NumericCitation')],\n",
       "  [('V. VACC INAT ION', 'be in', 'section')],\n",
       "  [('single node', 'be in', 'SIRSwithVaccination model')],\n",
       "  [('Wavy arrow', 'represent', 'transition')],\n",
       "  [('θ A. Infection-Dominant Model', 'be in', 'case')],\n",
       "  [('1 − δ', 'be in', 'P ∗ S βA')],\n",
       "  [('a', 'be', 'true'), ('ﬁxed point', 'statement', 'true')],\n",
       "  [('same argument', 'be in', 'proof')],\n",
       "  [('we', 'be', 'interested')],\n",
       "  [('it', 'become', 'vaccinated'),\n",
       "   ('B. Vaccination-Dominant Model', 'be in', 'variation of model')],\n",
       "  [('Yi |', 'be with', 'change'),\n",
       "   ('change', 'with yi be', '101'),\n",
       "   ('enyi graph', 'be with', 'n')],\n",
       "  [('NumericCitation N. T. Bailey et', 'theory of', 'infectious diseases')],\n",
       "  [('NumericCitation M. Richardson',\n",
       "    'be in',\n",
       "    'Proceedings of eighth ACM SIGKDD international conference on Knowledge discovery')],\n",
       "  [('Information Theory', 'transaction on', 'vol'),\n",
       "   ('Information propagation speed', 'be in', 'mobile tolerant networks')],\n",
       "  [('information propagation', 'be in', 'ﬂickr social network'),\n",
       "   ('NumericCitation M. Cha',\n",
       "    'be in',\n",
       "    'Proceedings of 18th international conference on World wide web')],\n",
       "  [('sirs epidemic model', 'be with', 'time delay')],\n",
       "  [('Decision', 'in ahn be', '2013 IEEE 52nd Annual Conference on'),\n",
       "   ('Decision', 'in ahn be', 'CDC'),\n",
       "   ('NumericCitation H. J. Ahn', 'be in', 'Decision')],\n",
       "  [('Decision', 'in numericcitation be', 'CDC'),\n",
       "   ('mixing time',\n",
       "    'on numericcitation be',\n",
       "    '2014 IEEE 53rd Annual Conference on'),\n",
       "   ('Decision',\n",
       "    'in numericcitation be',\n",
       "    '2014 IEEE 53rd Annual Conference on'),\n",
       "   ('mixing time', 'on numericcitation be', 'CDC'),\n",
       "   ('NumericCitation', 'be in', 'Decision')],\n",
       "  [('NumericCitation M. Draief',\n",
       "    'be in',\n",
       "    'Proceedings of 1st international conference on Performance evaluation methodolgies')],\n",
       "  [('Virus spread', 'be in', 'networks')],\n",
       "  [('Epidemic thresholds', 'be in', 'real networks')]],\n",
       " 84: [[('Wang', 'university of', 'Electronic Science'),\n",
       "   ('Wang', '3center for', 'Polymer Studies'),\n",
       "   ('Wang', 'university of', 'Electronic Science')],\n",
       "  [('networks', 'be with', 'positive degree-degree correlations'),\n",
       "   ('networks', 'be with', 'eigenvector localized on high k-core nodes'),\n",
       "   ('networks', 'be with', 'high level of clustering')],\n",
       "  [('many real-world phenomena', 'spread', 'dynamics on complex networks'),\n",
       "   ('real-world phenomena', 'spread', 'dynamics on complex networks')],\n",
       "  [('Researchers',\n",
       "    'have focus because of',\n",
       "    'their important rami ﬁcatio ns in many real-world scenarios'),\n",
       "   ('their important rami ﬁcatio ns', 'be in', 'many real-world scenarios')],\n",
       "  [('global epidemic', 'occur', 'NumericCitation')],\n",
       "  [('us',\n",
       "    'determine',\n",
       "    'effectiveness of given immunization strategy NumericCitation')],\n",
       "  [('us', 'know', 'epidemic threshold'),\n",
       "   ('us', 'more accurately determine', 'optimum source node NumericCitation')],\n",
       "  [('thresholds',\n",
       "    'be in',\n",
       "    'epidemic spreading models such susceptible-infected-recovered SIR model NumericCitation')],\n",
       "  [('approach', 'be accurate in', 'case of tree-like networks')],\n",
       "  [('uncorrelated networks',\n",
       "    'be with',\n",
       "    'strongly heterogeneous degree distribution')],\n",
       "  [('three theoretical methods', 'differ', 'epidemic thresholds')],\n",
       "  [('we',\n",
       "    'determine',\n",
       "    'relationships among three differing outcomes of MFL method')],\n",
       "  [('networks', 'be with', 'eigenvector localized on high k-core nodes')],\n",
       "  [('networks', 'be with', 'eigenvector localized on hubs')],\n",
       "  [('we', 'initiate', 'epidemic')],\n",
       "  [('Infected nodes', 'can also recover with', 'probability γ')],\n",
       "  [('epidemic threshold', 'use', 'only degree distribution')],\n",
       "  [('only input parameter', 'as p be', 'k'),\n",
       "   ('only input parameter',\n",
       "    'as p be',\n",
       "    'theoretical epidemic threshold prediction using')],\n",
       "  [('epidemic threshold', 'be in', 'uncorrelated networks'),\n",
       "   ('λMFL', 'be', 'good')],\n",
       "  [('QMF method',\n",
       "    'capture',\n",
       "    'dynamical correlations among states of neighbors'),\n",
       "   ('column vector', 'be with', 'N elements')],\n",
       "  [('nonreversible epidemic spreading dynamics', 'be in', 'SIR modeled')],\n",
       "  [('DM P method', 'determine', 'complete network structure')],\n",
       "  [('diagonal matrix', 'be with', 'vertex degrees along its diagonal'),\n",
       "   ('0', 'be', 'null'),\n",
       "   ('0', 'be', 'N × N null matrix')],\n",
       "  [('we',\n",
       "    'determine',\n",
       "    'other relationships among three theoretical thresholds')],\n",
       "  [('we', 'obtain', 'c c λA')],\n",
       "  [('we', 'get', 'λDMP c ≥ λQMF c')],\n",
       "  [('νD', 'be', 'ﬁnite')],\n",
       "  [('epidemic threshold λQMF', 'use', 'QMF method')],\n",
       "  [('we', 'use', 'assumptions listed in previous section')],\n",
       "  [('details', 'be in', 'Method'),\n",
       "   ('We', 'determine', 'theoretical epidemic threshold')],\n",
       "  [('nodes', 'be with', 'high k-core index')],\n",
       "  [('Figure 1',\n",
       "    'show',\n",
       "    'systematic study of SIR model on uncorrelated conﬁguration net works')],\n",
       "  [('νD', 'be', 'where degree exponent')],\n",
       "  [('networks', 'be with', 'νD ='),\n",
       "   ('networks', 'be with', 'νD ='),\n",
       "   ('NumericCitation', 'network with', 'νD =')],\n",
       "  [('MFL methods', 'be in', 'general')],\n",
       "  [('absolute errors', 'be in', 'epidemic threshold from MFL methods'),\n",
       "   ('νD', '=', '2.1'),\n",
       "   ('absolute errors decrease', 'be with', 'N')],\n",
       "  [('νD', '=', '3.5')],\n",
       "  [('its performance', 'be even worse than', 'MFL method')],\n",
       "  [('group', 'on performance be', 'e.g. social networks'),\n",
       "   ('three theoretical predictions λMFL',\n",
       "    'of performance be',\n",
       "    'e.g. social networks')],\n",
       "  [('spreading processes', 'be perform on', 'giant connected clusters'),\n",
       "   ('spreading processes', 'be', 'performed')],\n",
       "  [('one', 'produce', 'value closest to λc')],\n",
       "  [('best prediction', 'be in', 'more 40 % of networks')],\n",
       "  [('Theoretical predictions', 'λqmf of', 'epidemic threshold'),\n",
       "   ('epidemic threshold', 'of λqmf be', 'red circles')],\n",
       "  [('b', 'be in', 'entire sample of real-world networks')],\n",
       "  [('λDMP c',\n",
       "    'be',\n",
       "    'of λMFL c c c λMFL c closest to real epidemic threshold in about one-third of real-world networks')],\n",
       "  [('networks', 'have', 'distinct eigenvector localizations NumericCitation'),\n",
       "   ('networks', 'have', 'eigenvector localizations NumericCitation')],\n",
       "  [('they', 'be either localized in', 'real-world networks')],\n",
       "  [('adjacency', 'of ratio be', 'IPR')],\n",
       "  [('b', 'inset of be', 'average absolute errors as function of IPR')],\n",
       "  [('it', 'be slightly accurate than', 'DMP method')],\n",
       "  [('three methods', 'be in', 'theoretical prediction')],\n",
       "  [('MFL method', 'be', 'least')],\n",
       "  [('epidemic thresholds',\n",
       "    'be in',\n",
       "    'networks with positive degree-degree correlations'),\n",
       "   ('networks', 'be with', 'positive degree-degree correlations'),\n",
       "   ('MFL method',\n",
       "    'predict',\n",
       "    'epidemic thresholds in networks with positive degree-degree correlations')],\n",
       "  [('c', 'fraction of', 'λMFL'),\n",
       "   ('collective', 'in c be', 'fraction of λMFL'),\n",
       "   ('c', 'be in', 'collective of LHNs of real-world networks')],\n",
       "  [('we', 'show in', 'a')],\n",
       "  [('networks', 'be in', 'LHNs'), ('networks', 'be in', 'LKNs')],\n",
       "  [('relative errors', 'be in', 'ﬁrst column')],\n",
       "  [('avera ge', 'be in', 'second column')],\n",
       "  [('CONCLUSIONS', 'be in', 'study'),\n",
       "   ('epidemic threshold', 'be in', 'SIR model'),\n",
       "   ('we', 'predict', 'epidemic threshold in SIR model')],\n",
       "  [('networks', 'be with', 'eigenvector localized on high k-core nodes')],\n",
       "  [('relative errors', 'be in', 'ﬁrst column')],\n",
       "  [('average relative errors', 'be in', 'second column')],\n",
       "  [('microscopic topologies',\n",
       "    'be in',\n",
       "    'real-world networks including degree-degree correlations')],\n",
       "  [('QMF method', 'be better than', 'MFL method'),\n",
       "   ('networks', 'be with', 'negative degree-degree correlations')],\n",
       "  [('networks', 'be with', 'positive degree-degree correlations'),\n",
       "   ('QMF method', 'be', 'least')],\n",
       "  [('MFL method', 'be', 'least'), ('networks', 'be with', 'low clustering')],\n",
       "  [('QMF method', 'be', 'least'), ('networks', 'be with', 'high clustering')],\n",
       "  [('accurate epidemic thresholds', 'be in', 'networks'),\n",
       "   ('signi ﬁcant', 'be in', 'ﬁeld spreading')],\n",
       "  [('use',\n",
       "    'be in',\n",
       "    'theoretical methods i.e. performance level of method only proportional to topological information used')],\n",
       "  [('relative errors', 'be in', 'ﬁrst column')],\n",
       "  [('average relative errors', 'be in', 'second column')],\n",
       "  [('METHODS', 'predict', 'numerical threshold')],\n",
       "  [('we', 'determine', 'theoretical epidemic threshold')],\n",
       "  [],\n",
       "  [('Epidemic processes', 'be in', 'complex networks')],\n",
       "  [('Castellano', 'physics of', 'social dynamics')],\n",
       "  [('J. F. F. Critical phenomena', 'be in', 'complex networks'),\n",
       "   ('NumericCitation Dorogovtsev', 'phenomenon in', 'complex networks')],\n",
       "  [('A. Epidemic outbreaks', 'be in', 'complex heterogeneous networks'),\n",
       "   ('Moreno', 'outbreak in', 'complex heterogeneous networks')],\n",
       "  [('NumericCitation Cohen', 'strategy for', 'Computer Networks')],\n",
       "  [('A. Epidemic dynamics', 'be in', 'complex networks'),\n",
       "   ('NumericCitation Pastor-Satorras', 'dynamics in', 'complex networks')],\n",
       "  [('NumericCitation Newman', 'spread on', 'networks'),\n",
       "   ('NumericCitation Newman', 'spread of', 'epidemic disease')],\n",
       "  [('criticality', 'be in', 'absorbing-state phase transitions')],\n",
       "  [('Identi ﬁcation', 'be in', 'complex networks')],\n",
       "  [('Epidemic Thresholds', 'be in', 'Clustered Networks')],\n",
       "  [('M. E. J.Random Graphs', 'be with', 'Clustering'),\n",
       "   ('NumericCitation Newman', 'graphs with', 'Clustering')],\n",
       "  [('Epidemic Threshold', 'be in', 'Scale-Free Networks'),\n",
       "   ('Epidemic Threshold', 'be with', 'Degree Cor-relations'),\n",
       "   ('R.',\n",
       "    'absence of',\n",
       "    'Epidemic Threshold in Scale-Free Networks with Degree Cor-relations')],\n",
       "  [('NumericCitation Miller', 'modelling for', 'infectious disease spread')],\n",
       "  [('Epidemic spreading', 'be with', 'general degree')],\n",
       "  [('Dynamics', 'be with', 'memory of nonredundant information')],\n",
       "  [('Eames',\n",
       "    'be in',\n",
       "    'spread of sexually transmitted 7 for complex networks')],\n",
       "  [('NumericCitation Gross', 'dynamics on', 'Adaptive Network')],\n",
       "  [('NumericCitation G', 'omez be', 'S.')],\n",
       "  [('Discrete-time Markov chain approach', 'be in', 'complex networks')],\n",
       "  [('Ferreira',\n",
       "    'threshold of',\n",
       "    'susceptible-infected-susceptible model on networks')],\n",
       "  [('Epidemic Spreading', 'be in', 'Networks'),\n",
       "   ('NumericCitation Castellano',\n",
       "    'thresholds for',\n",
       "    'Epidemic Spreading in Networks')],\n",
       "  [('Epidemic Thresholds', 'be in', 'Real Networks')],\n",
       "  [('Mieghem', 'spread in', 'Networks'),\n",
       "   ('R. Virus Spread', 'be in', 'Networks')],\n",
       "  [('NumericCitation Lokhov', 'ezard', 'M'),\n",
       "   ('M.', None, 'L.'),\n",
       "   ('NumericCitation Lokhov', 'l.', 'M.')],\n",
       "  [('models', 'be with', 'unidirectional dynamics')],\n",
       "  [('centrality', 'be in', 'networks')],\n",
       "  [('Spectral redemption', 'be in', 'clustering sparse networks')],\n",
       "  [('F. Predicting percolation thresholds', 'be in', 'networks'),\n",
       "   ('NumericCitation Radicchi', 'threshold in', 'networks')],\n",
       "  [('eigenvector localization', 'be in', 'networks'),\n",
       "   ('NumericCitation Pastor-Satorras',\n",
       "    'type of',\n",
       "    'eigenvector localization in networks')],\n",
       "  [('Diseases', 'be in', 'Complex Networks')],\n",
       "  [('W', \"'s ouza be\", 'M. R. Microtransition Cascades to Percolation')],\n",
       "  [('Outstanding Ph. D. Candidate', 'be in', 'Academic Research')]],\n",
       " 85: [[('Infection spreads', 'be with', 'addition')],\n",
       "  [('simultaneous cluster infection',\n",
       "    'can give',\n",
       "    'rise to discontinuous jumps of diﬀerent sizes in number of failed nodes')],\n",
       "  [('jumps', 'emerge at', 'lower infection rates'),\n",
       "   ('Larger jumps', 'emerge at', 'lower infection rates')],\n",
       "  [('we', 'observe', 'hysteresis for networks')],\n",
       "  [('s c i', 'be', 'y h p')],\n",
       "  [('such discontinuous transitions',\n",
       "    'recently propose example of be',\n",
       "    'explosive percolation')],\n",
       "  [('discontinuous transition',\n",
       "    'be in',\n",
       "    'fraction of largest cluster component')],\n",
       "  [('transition', 'be in', 'disease incidence')],\n",
       "  [('healthy nodes', 'thus be absorb into', 'infected cluster all at once'),\n",
       "   ('healthy nodes', 'encapsulate by', 'infected ones')],\n",
       "  [('network topology', 'be', 'when dominated by local connections')],\n",
       "  [('tubercle formation', 'be in', 'lungs NumericCitation')],\n",
       "  [('METHODS', 'capture', 'long time scales'),\n",
       "   ('METHODS', 'capture', 'time scales')],\n",
       "  [('Susceptible nodes',\n",
       "    'become',\n",
       "    'infected at rate p through contact with their infected nearest neighbors'),\n",
       "   ('contact', 'be with', 'their infected nearest neighbors'),\n",
       "   ('Susceptible nodes', 'infect at', 'rate p')],\n",
       "  [('Infected nodes', 'heal at', 'rate q')],\n",
       "  [('I', 'q', 'S')],\n",
       "  [('network', 'be with', 'average degree k')],\n",
       "  [('basic reproduction number', 'be in', 'mean-ﬁeld SIS model'),\n",
       "   ('τ', 'be', 'in mean-ﬁeld SIS model basic reproduction number')],\n",
       "  [('single central node', 'be in', 'system')],\n",
       "  [('sudden jump', 'be in', 'infection'),\n",
       "   ('central node', 'surround by', 'infected nodes')],\n",
       "  [('update schema', 'be in', 'recently introduced no-enclave percolation'),\n",
       "   ('clustering mechanism', 'update schema in', 'NEP')],\n",
       "  [('Gillespie', 'have', 'al 3 Figure 1')],\n",
       "  [('ﬁve additional nodes', 'become too', 'infected'),\n",
       "   ('ﬁve additional nodes', 'become', 'infected')],\n",
       "  [('Recovery', 'be with', 'rate q'),\n",
       "   ('two processes', 'deﬁning', 'dynamics of system'),\n",
       "   ('infection', 'be with', 'rate p')],\n",
       "  [('We', 'determine', 'encapsulated nodes')],\n",
       "  [('Healing processes',\n",
       "    'thus only occur at',\n",
       "    'perimeter of encapsulated regions')],\n",
       "  [('square lattice',\n",
       "    'only account for',\n",
       "    'local nearest-neighbor interactions')],\n",
       "  [('node', 'be with', 'largest degree')],\n",
       "  [('critical value τc',\n",
       "    'be determine by',\n",
       "    'underlying contact process dynamics of simple SIS model')],\n",
       "  [('epidemic jumps', 'be in', 'infection levels'),\n",
       "   ('stationary state', 'be', 'has reached'),\n",
       "   ('transmissibility', 'be', 'when lower')],\n",
       "  [('single realization', 'for tst be', 'τ')],\n",
       "  [('we', 'find in', 'other words')],\n",
       "  [('Discontinuous behavior', 'be in', 'time')],\n",
       "  [('cluster SIS model', 'be with', 'q =')],\n",
       "  [('L × L square lattices', 'be with', 'L = 128 256'),\n",
       "   ('L × L square lattices', 'for jump be', 'binned data')],\n",
       "  [('connected graph', 'be with', 'N nodes')],\n",
       "  [('infected state', 'to tst be', 'τ'), ('function', 'as tst be', 'τ')],\n",
       "  [('straight lines', 'be in', 'inset'), ('eye', 'be with', 'slopes')],\n",
       "  [('Error bars', 'indicate', 'standard error of mean')],\n",
       "  [],\n",
       "  [('it', 'thus can become', 'healthy')],\n",
       "  [('we', 'have', 'contradiction')],\n",
       "  [('We',\n",
       "    'expect',\n",
       "    'abrupt transition from fully healthy to fully infected phase')],\n",
       "  [('parameter', 'imply', 'ﬁrst-order transition')],\n",
       "  [('τc', 'close to ist be', 'τ')],\n",
       "  [('Stationary state', 'be with', 'jump close τc')],\n",
       "  [('Error bars', 'indicate', 'standard error of mean')],\n",
       "  [('Insets',\n",
       "    'show',\n",
       "    'proportions of two possible states for diﬀerent values of τ')],\n",
       "  [('it', 'order', 'τ'),\n",
       "   ('Fig. 8', 'be in', 'Appendix'),\n",
       "   ('single curve <', 'onto ist be', 'τ')],\n",
       "  [('ﬁnite system sizes', 'be in', 'as Fig. 4')],\n",
       "  [('Hysteresis study', 'be with', 'connections')],\n",
       "  [('128 × 128 square lattice', 'be with', 'long-range connections'),\n",
       "   ('2 · 103 square lattice', 'be with', 'diﬀerent samples'),\n",
       "   ('tr = 200', 'lattice with', 'long-range connections')],\n",
       "  [('network', 'be with', 'additional long-range connections')],\n",
       "  [('square lattice', 'be with', 'minimum fraction of active sites =')],\n",
       "  [('characteristic time-scale', 'be in', 'Fig. 3')],\n",
       "  [('topology',\n",
       "    'eﬀects of be',\n",
       "    'transition behavior to changes in network topologies'),\n",
       "   ('changes', 'be in', 'network topologies')],\n",
       "  [('Fig. 9', 'be in', 'Appendix')],\n",
       "  [('node', 'get', 'new connection to another')],\n",
       "  [('r', '=', '0.5')],\n",
       "  [('situation', 'be', 'may more subtle')],\n",
       "  [('system', 'exhibit', 'only two attractors')],\n",
       "  [('level', 'be in', 'system')],\n",
       "  [('transition', 'be still ﬁrst-order on', 'square lattice')],\n",
       "  [('topology', 'strongly inﬂuences', 'cluster dynamics')],\n",
       "  [('E.G.', 'thank h.j.h. for', 'his invitation to Zurich'),\n",
       "   ('E.G.', 'thank', 'H.J.H.')],\n",
       "  [('European Community',\n",
       "    'by',\n",
       "    'H2020 Program under funding scheme FETPROACT-1-2014')],\n",
       "  [('E.G.', 'formulate', 'proof')],\n",
       "  [('Child Health', 'of institute be', 'P01-HD31921')],\n",
       "  [('Health contact', 'add', 'Health')],\n",
       "  [('School network', 'be with', 'Figure 7')],\n",
       "  [('Apollonian network', 'be with', '1096 nodes')],\n",
       "  [('onion-like structures', 'illustrate in', 'Fig. 7')],\n",
       "  [('sixth-generation Apollonian network', 'be with', '1096 nodes')],\n",
       "  [('school network', 'show in', 'Fig. 6')],\n",
       "  [('largest school network', 'be with', '2539 nodes')],\n",
       "  [('parameter L0 .3', 'be in', 'limit of inﬁnite τ < τc')],\n",
       "  [('hysteresis eﬀect', 'describe in', 'main text')],\n",
       "  [('function', 'as ist be', 'τ')],\n",
       "  [('Error bars', 'indicate', 'standard error of mean')],\n",
       "  [('situation', 'however be more subtle regard', 'Apollonian network')],\n",
       "  [('separation', 'disappear for', 'larger values of τ')],\n",
       "  [('line', 'be with', 'observation made in main texts'),\n",
       "   ('transmissibility', 'be', 'when lower')],\n",
       "  [('Apollonian network', 'show', 'hysteresis')],\n",
       "  [('pseudo-linear part', 'be in', 'particular region')],\n",
       "  [('errors', 'be estimate with', '104 samples')],\n",
       "  [('diﬀerent L × L square lattices', 'be with', 'L = 128 256')],\n",
       "  [('distribution', 'start with', 'smallest possible jump of size two')],\n",
       "  [('distribution', 'have', 'two separated regions')],\n",
       "  [('Distribution', '=', '1.65 ≈ τc')],\n",
       "  [('diﬀerent L × L square lattices', 'be with', 'L = 128 256')],\n",
       "  [('Innovations', 'of diﬀusion be', 'Citation'),\n",
       "   ('17 NumericCitation E. M. Rogers', 'diﬀusion of', 'Innovations')],\n",
       "  [('Research', 'on symposium be', '343 1991'),\n",
       "   ('Security', 'in symposium be', '343 1991'),\n",
       "   ('1991 IEEE Computer Society Symposium', 'be in', 'Security')],\n",
       "  [('J. Kert', 'esz be', 'Phys')],\n",
       "  [('NumericCitation D. Stauﬀer', 'introduction to', 'Percolation Theory')],\n",
       "  [('H. Chat', None, 'Phys')],\n",
       "  [('T. Tom', None, 'Phys')],\n",
       "  [('M. Jacomy', 'conference on', 'Weblogs')]],\n",
       " 86: [[('probability', 'strength of', 'target country')],\n",
       "  [('countries', 'be with', 'much lower GDP such Belgium')],\n",
       "  [('today', 'in', 'global economy'), ('other countries', 'be in', 'world')],\n",
       "  [('several signiﬁcant crises', 'be in', 'past few decades')],\n",
       "  [('we', 'generate', 'economic network')],\n",
       "  [('trade', 'be in', 'terms of import/export'), ('trade', 'be', 'large')],\n",
       "  [('network', 'be', 'direct')],\n",
       "  [('investment', 'be', 'key')],\n",
       "  [('International Trade Network', 'version of', 'database'),\n",
       "   ('82 countries', 'be in', 'million US dollars')],\n",
       "  [],\n",
       "  [('subsidiary corporations', 'be in', 'country B')],\n",
       "  [('network', 'be with', 'total 2886 links among 206 nodes')],\n",
       "  [('rest 1516', 'be', 'one directional only')],\n",
       "  [('S1', 'be in', 'Supplementary Information')],\n",
       "  [('total weight', 'be in', 'our network')],\n",
       "  [('we',\n",
       "    'identify',\n",
       "    'uneven roles of diﬀerent countries in global economic network'),\n",
       "   ('uneven roles', 'be in', 'global economic network')],\n",
       "  [('node', 'be in', 'network'),\n",
       "   ('node', 'k', 'its'),\n",
       "   ('network', 'in node be', 'higher its ks')],\n",
       "  [('nodes', 'be in', 'highest shell')],\n",
       "  [('degree k', '=', '1'), ('nodes', 'only have', 'k ≥ 2')],\n",
       "  [('shell ks', '=', '1')],\n",
       "  [('ks', '=', '2')],\n",
       "  [('only links', 'be with', 'weights')],\n",
       "  [('we', 'use', 'diﬀerent threshold values'),\n",
       "   ('we', 'use', 'threshold values'),\n",
       "   ('we', 'show in', 'Fig. 2'),\n",
       "   ('we', 'show in', 'A')],\n",
       "  [('we', 'locate nucleus at', 'ks')],\n",
       "  [('core', 'therefore consist of', 'following twelve countries')],\n",
       "  [('nucleus', 'be almost same as', 'nucleus found for CON')],\n",
       "  [('shell 9', 'be in', 'ITN')],\n",
       "  [('presence', 'be in', 'nucleus of ITN'),\n",
       "   ('their absence', 'be in', 'nucleus of CON')],\n",
       "  [('goods', 'most by', 'especially China'),\n",
       "   ('many large corporations', 'be in', 'global corporate database'),\n",
       "   ('Headquarters', 'be in', 'China'),\n",
       "   ('Headquarters', 'be in', 'Russia')],\n",
       "  [('their total trade volume',\n",
       "    'be',\n",
       "    'On other hand signiﬁcantly high in total global trade')],\n",
       "  [('CON', 'for fig. be', 'C'), ('ITN', 'for fig. be', 'C')],\n",
       "  [('smaller countries', 'be in', 'absolute size')],\n",
       "  [('world', 'of', 'largest companies')],\n",
       "  [('change', 'be in', 'k shell ranking')],\n",
       "  [('ranking', 'be do in', 'units of distance from nucleus')],\n",
       "  [('nuclei', 'use', 'k shell method'),\n",
       "   ('we', 'show in', 'supplementary Fig')],\n",
       "  [('crisis', 'be', 'complex')],\n",
       "  [('role', 'be in', 'spreading')],\n",
       "  [('model spreading', 'be in', 'various networks NumericCitation'),\n",
       "   ('SIR model',\n",
       "    'have',\n",
       "    'has used successfully to model spreading of epidemics in various networks NumericCitation')],\n",
       "  [('it',\n",
       "    'usually assume',\n",
       "    'ﬁxed probability for neighbour-to-neighbour infection')],\n",
       "  [('country', 'overcome', 'crisis')],\n",
       "  [('node i', 'infect', 'its neighbouring node j'),\n",
       "   ('node i', 'infect', 'its node j'),\n",
       "   ('targeted node', 'of weight be', 'strength')],\n",
       "  [('country j', 'be infect by', 'country i'),\n",
       "   ('factor', 'be in', 'probability'),\n",
       "   ('country j', 'be', 'infected')],\n",
       "  [('crisis', 'be in', 'subsidiary'), ('problems', 'be in', 'mother company')],\n",
       "  [('we', 'account for', 'directionality')],\n",
       "  [('cases', 'in w be', 'ij'), ('w', 'be in', 'cases')],\n",
       "  [('undirected links', 'be in', 'resulting network'),\n",
       "   ('we', 'increase', 'value')],\n",
       "  [('crisis spreading process', 'be in', 'CON')],\n",
       "  [('countries', 'be in', 'outer shells'), ('countries', 'be in', 'nucleus')],\n",
       "  [('larger global impact', 'be in', 'comparison to CON')],\n",
       "  [('error bars', 'be in', 'Figs. 3')],\n",
       "  [('they', 'belong in', 'CON')],\n",
       "  [('60 %', 'result of', '50 realizations'),\n",
       "   ('world countries', 'of % be', 'average result')],\n",
       "  [('crises', 'be with', 'small magnitude')],\n",
       "  [('12 countries', 'play in', 'world economy')],\n",
       "  [('economic crisis',\n",
       "    'could spread globally depend',\n",
       "    'depending country of origin')],\n",
       "  [('world', 'on', 'largest companies')],\n",
       "  [('we', 'use', 'k shell decomposition method'),\n",
       "   ('role', 'be in', 'world crisis'),\n",
       "   ('we', 'identify', 'role of diﬀerent countries in world crisis')],\n",
       "  [('crisis', 'be in', 'Indonesia'), ('crisis', 'be in', 'Greece 2')],\n",
       "  [('Correlations', 'be in', 'Finance')],\n",
       "  [('shell 2', 'be in', 'ITN'), ('2 Greece', 'shell', '5')],\n",
       "  [('NumericCitation Hidalgo CA', 'al', 'Barab')],\n",
       "  [('NumericCitation Colizza V', 'm', 'Barth')],\n",
       "  [('layers', 'be schematic drawing of', 'shells obtained by k-shell method'),\n",
       "   ('layers', 'be', 'schematic')],\n",
       "  [('nucleus', 'from distance be', 'CON'),\n",
       "   ('nucleus', 'from distance be', 'ITN')],\n",
       "  [('economic crisis propagation', 'use', 'SIR dynamics')],\n",
       "  [('same number', 'with network be', 'same number of links per node'),\n",
       "   ('simulated network', 'number of', 'links per node'),\n",
       "   ('simulated network', 'be with', 'same number of nodes')],\n",
       "  [('nodes', 'of m. be', 'C')],\n",
       "  [('they', 'belong for', 'CON')],\n",
       "  [('B countries', 'aﬀ', 'Citation')],\n",
       "  [('eﬀect', 'be', 'economic')],\n",
       "  [('infection', 'be in', 'US'),\n",
       "   ('1000 realizations', 'always start', 'infection in US')],\n",
       "  [('it', 'have', 'score ≤ 200')],\n",
       "  [('maximum number', 'be in', 'our runs')],\n",
       "  [('same magnitude', 'be in', 'countries of lower shells')],\n",
       "  [('Citation', 'be in', 'GDP percentage rate')],\n",
       "  [('curve', 'use', '6 point moving average')],\n",
       "  [('larger change', 'be in', 'GDP')],\n",
       "  [('economic crisis Supplementary Information A. Garas1',\n",
       "    'of spreading be',\n",
       "    '54124 Thessaloniki Greece'),\n",
       "   ('economic crisis Supplementary Information A. Garas1',\n",
       "    'of spreading be',\n",
       "    'University'),\n",
       "   ('1 Worldwide spreading', 'university of', 'Thessaloniki')],\n",
       "  [('2Geography Institute', 'university of', 'Lausanne'),\n",
       "   ('2Geography Institute', 'faculty of', 'Geosciences')],\n",
       "  [('3Faculty', 'university of', 'Lausanne'),\n",
       "   ('Business', 'of 3faculty be', 'CH'),\n",
       "   ('Business', 'of 3faculty be', '1015 Lausanne Switzerland'),\n",
       "   ('Business', 'of 3faculty be', 'University')],\n",
       "  [('I', 'result about', 'CON')],\n",
       "  [('we', 'use', 'ORBIS database'), ('we', 'describe in', 'article')],\n",
       "  [('4000 world corporations', 'be with', 'highest turnover')],\n",
       "  [],\n",
       "  [('subsidiary corporations', 'be in', 'country B')],\n",
       "  [('network', 'be with', 'total 2886 links among 206 nodes')],\n",
       "  [('rest 1516', 'be', 'one directional only')],\n",
       "  [('total link weight',\n",
       "    'represent',\n",
       "    'strength of economic ties between two countries in network'),\n",
       "   ('two countries', 'be in', 'network')],\n",
       "  [('we', 'use in', 'our model'), ('network', 'be', 'good')],\n",
       "  [('CON', 'on nature be', '2005')],\n",
       "  [('we', 'use', 'k-shell decomposition method')],\n",
       "  [('we', 'have', 'epidemic outbreak')],\n",
       "  [('crisis', 'be', 'originated')],\n",
       "  [('GDP values', 'be in', 'million US dollars')],\n",
       "  [('we', 'define', 'ijijwE ≡')],\n",
       "  [('trade balance', 'be numerator of', 'Eq')],\n",
       "  [('propagation', 'be in', 'ITN')],\n",
       "  [('country', 'be in', 'crisis')],\n",
       "  [('bilateral trade', 'be with', 'countryi'), ('ijjtotwwɶ', 'be', 'large')],\n",
       "  [('country j',\n",
       "    'be affect by',\n",
       "    'crisis propagating through its link to countryi')],\n",
       "  [('it', 'therefore can take', 'positive value')],\n",
       "  [('error bars', 'be show', 'standard deviation')],\n",
       "  [('error bars', 'be show', 'standard deviation')],\n",
       "  [('error bars', 'be show', 'standard deviation')],\n",
       "  [('current situation', 'be in', 'Greece')],\n",
       "  [('results', 'be average over', '1000 realizations')],\n",
       "  [('easier propagation', 'be in', 'neighbour of Greece')],\n",
       "  [('strong economic ties', 'be with', 'Greece'),\n",
       "   ('countries', 'traditionally have', 'strong economic ties with Greece')]],\n",
       " 87: [[('Disease dynamics', 'be in', 'rural regions')],\n",
       "  [('rural community', 'be in', 'Kansas')],\n",
       "  [('We', 'visit', 'locations'), ('We', 'frequently visit', 'locations')],\n",
       "  [('epidemic size', 'be reduce by', '19 %')],\n",
       "  [('Our results',\n",
       "    'suggest',\n",
       "    'new implementation of highly eﬀective strategy for targeted vaccinations through use of popular locations in rural communities'),\n",
       "   ('use', 'be in', 'rural communities')],\n",
       "  [('dominant inﬂuenza virus', 'be in', 'circulation'),\n",
       "   ('dominant inﬂuenza virus', 'be with', 'many countries'),\n",
       "   ('overseas territories', 'report', 'laboratory')],\n",
       "  [('La Gloria', 'town near', 'Veracruz')],\n",
       "  [('hog barn', 'be in', 'Walworth County')],\n",
       "  [('Various immunization strategies',\n",
       "    'have',\n",
       "    'have formulated for urban populations'),\n",
       "   ('immunization strategies',\n",
       "    'have',\n",
       "    'have formulated for urban populations')],\n",
       "  [('human population distribution', 'be estimate as', 'scale free')],\n",
       "  [('strategy', 'be', 'targeted')],\n",
       "  [('Disease dynamics', 'be in', 'rural regions'),\n",
       "   ('NumericCitation NumericCitation', 'have receive', 'study'),\n",
       "   ('NumericCitation NumericCitation', 'have receive', 'limited study')],\n",
       "  [('predictive strategies', 'be in', 'rural regions')],\n",
       "  [('rural Clay county', 'be in', 'Kansas')],\n",
       "  [('rural Clay', 'be in', 'Kansas')],\n",
       "  [('we', 'characterize', 'contact structure of rural regions')],\n",
       "  [('global targeted strategies',\n",
       "    'mitigate epidemic with',\n",
       "    'limited amount of resources'),\n",
       "   ('partial knowledge', 'be with', 'individual rights')],\n",
       "  [('Random vaccine distribution', 'be in', 'selected popular location')],\n",
       "  [('location-based strategy',\n",
       "    'be',\n",
       "    '55 % as eﬀective as best global target strategy')],\n",
       "  [('contact network', 'analyze', 'epidemic')],\n",
       "  [('SEIR model',\n",
       "    'be with',\n",
       "    'parameters estimated in analysis of recent H1N1 inﬂuenza NumericCitation')],\n",
       "  [('we', 'mail', 'surveys')],\n",
       "  [('Human Sub', 'ject', 'Kansas'),\n",
       "   ('Kansas State University', 'of board be', 'IRB')],\n",
       "  [('informed consent', 'be in', 'cover letter attached to their surveys')],\n",
       "  [('spread', 'be in', 'rural areas'), ('their contacts', 'be with', 'other')],\n",
       "  [('people', 'be with', 'most contacts')],\n",
       "  [('it', 'increase', 'accuracy of our epidemic analysis')],\n",
       "  [('Survey results', 'construct', 'weighted contact network')],\n",
       "  [('respondent', 'make', 'contact for three diﬀerent levels of contact')],\n",
       "  [('both', 'be', 'large'), ('both', 'be', 'when large')],\n",
       "  [('sub-weight wx', 'j between', 'node i')],\n",
       "  [('H1N1 outbreak', 'be in', 'La Gloria')],\n",
       "  [('πDirect − H igh', '=', '1.0')],\n",
       "  [('number',\n",
       "    'be with',\n",
       "    'corresponding simulated new infected individuals in Clay Center'),\n",
       "   ('Clay Center',\n",
       "    'kansas give',\n",
       "    'best estimated values for three contact levels'),\n",
       "   ('corresponding simulated new infected individuals',\n",
       "    'be in',\n",
       "    'Clay Center'),\n",
       "   ('new infected individuals', 'be in', 'La Gloria NumericCitation')],\n",
       "  [('we', 'describe in', 'following sections')],\n",
       "  [('links', 'be with', 'strength greater')],\n",
       "  [('link', 'be with', 'highest strength')],\n",
       "  [('We', 'vary', 'their values')],\n",
       "  [('3.4 % variation', 'be in', 'total infection cases'),\n",
       "   ('variation', 'be in', 'total cases')],\n",
       "  [('contact network', 'be compose of', 'N')],\n",
       "  [('sum', 'j of', 'j ∈ neighbors')],\n",
       "  [('node degree', 'be in', 'binary network')],\n",
       "  [('we', 'compute properly', 'shortest path'),\n",
       "   ('we', 'compute', 'shortest path')],\n",
       "  [('their', 'wi', 'j'), ('their link weight wi', 'be', 'when high')],\n",
       "  [('network diameter', 'be deﬁned as', 'longest of shortest paths')],\n",
       "  [('degree',\n",
       "    'be in',\n",
       "    'binary version of weighted contact network NumericCitation')],\n",
       "  [('we', 'average over', 'individual clustering coeﬃcients')],\n",
       "  [('k-core', 'be', 'maximal')],\n",
       "  [('node', 'be in', 'core of network 7')],\n",
       "  [('node strength', 'size on', 'order of 10 − 6')],\n",
       "  [('matrix', 'be in', 'case'), ('zeros', 'be in', 'main diagonal')],\n",
       "  [('small epidemic threshold',\n",
       "    'be in',\n",
       "    'Susceptible-Infected-Susceptible model NumericCitation'),\n",
       "   ('large maximum eigenvalue',\n",
       "    'correspond to',\n",
       "    'small epidemic threshold in Susceptible-Infected-Susceptible model NumericCitation')],\n",
       "  [('them', 'be', 'divided')],\n",
       "  [('expected number', 'be in', 'same community')],\n",
       "  [('we',\n",
       "    'use',\n",
       "    'weighted version of algorithm described in NumericCitation')],\n",
       "  [('spreading process', 'be in', 'transition from S to E')],\n",
       "  [('remaining transitions', 'δ of', 'disease model'),\n",
       "   ('remaining transitions', 'only depend on', 'rate parameters')],\n",
       "  [('Infected node', 'recover with', 'recovery rate δ')],\n",
       "  [('it', 'recover for', 'remainder of simulation')],\n",
       "  [('blue curve', 'be in', 'Figure 1')],\n",
       "  [('nodes', 'random selection of be', 'simplest method for distribution')],\n",
       "  [('node', 'pass', 'infection on to others'),\n",
       "   ('strength', 'be', 'intuitive')],\n",
       "  [('reduction', 'be in', 'number of cases')],\n",
       "  [('nodes', 'be with', 'deepest coreness'),\n",
       "   ('line', 'be with', 'previous results')],\n",
       "  [('various locations', 'be in', 'county')],\n",
       "  [('we', 'select', 'locations for random strategies'),\n",
       "   ('we', 'be associate with', 'at least 10 % of population')],\n",
       "  [('we',\n",
       "    'select node with',\n",
       "    'highest combined sum for desired metric within locations')],\n",
       "  [('reduction', 'be in', 'number of cases')],\n",
       "  [('small associations', 'be with', 'health')],\n",
       "  [('campaign', 'be', 'when conducted in speciﬁc locations')],\n",
       "  [('node random selection', 'be in', 'entire population'),\n",
       "   ('node random selection', 'be in', 'key location')],\n",
       "  [('they', 'face', 'resource shortages'),\n",
       "   ('they',\n",
       "    'face resource shortage due to',\n",
       "    'smaller budgets than urban areas')],\n",
       "  [('locations', 'be', 'therefore easier')],\n",
       "  [('Epidemic spreading', 'be in', 'scale-free networks'),\n",
       "   ('R. Pastor-Satorras', 'spreading in', 'scale-free networks')],\n",
       "  [('Complex Networks', 'be with', 'Degree Correlations'),\n",
       "   ('Complex Networks',\n",
       "    'in spreading be',\n",
       "    'Springer-Verlag Berlin Heidelberg'),\n",
       "   ('M. Boguna R. Pastor-Satorras A. Vespignani Epidemic Spreading',\n",
       "    'be in',\n",
       "    'Complex Networks with Degree Correlations'),\n",
       "   ('Complex Networks', 'in spreading be', 'LNP 625'),\n",
       "   ('Complex Networks', 'in spreading be', 'Pages 127147')],\n",
       "  [('Preventive Medicine', 'of journal be', 'Pages 182-188 2001'),\n",
       "   ('Preventive Medicine', 'of journal be', 'Issue 3')],\n",
       "  [('P. Caley', 'time for', 'spread')],\n",
       "  [('Black',\n",
       "    'impact of',\n",
       "    'public health nurse intervention on inﬂuenza vaccine acceptance')],\n",
       "  [('Epidemiology', 'be in', 'Citation'),\n",
       "   ('E. D. Kilbourne', 'epidemiology of', 'Inﬂuenza B'),\n",
       "   ('E. D. Kilbourne', 'epidemiology in', 'Citation')],\n",
       "  [('Medicine', 'of journal be', 'Issue 1'),\n",
       "   ('Medicine', 'of journal be', 'Pages 31-38 2003')],\n",
       "  [('V.D. Perminov', 'model for', 'simulation')],\n",
       "  [('R. Pastor-Satorras', 'immunization of', 'complex networks')],\n",
       "  [('D. ben-Avraham', 'strategy for', 'Computer Networks')],\n",
       "  [('N. Wang', 'outbreak in', 'networks'),\n",
       "   ('Model ling disease outbreaks', 'be in', 'networks')],\n",
       "  [('429', 'pp.', '2004')],\n",
       "  [('W. O. Kermack', 'contribution to', 'mathematical theory of epidemics')],\n",
       "  [('aki', 'esz', 'J. Kert'),\n",
       "   ('aki', 'generalization of', 'clustering coeﬃcient'),\n",
       "   ('aki', 'generalization to', 'weighted complex networks')],\n",
       "  [('M. E. J. Newman', 'be in', 'networks')],\n",
       "  [('M. E. J. Newman', 'structure in', 'networks'),\n",
       "   ('Modularity structure', 'be in', 'networks')],\n",
       "  [('M. E. J. Newman', 'spread of', 'epidemic disease'),\n",
       "   ('M. E. J. Newman', 'spread on', 'networks')],\n",
       "  [('A. Barrat', 'process on', 'complex networks')],\n",
       "  [('C. Fraser et', 'potential of', 'Strain')],\n",
       "  [('families', 'be in', 'percentage of respondents medium'),\n",
       "   ('had one at-risk health respondents', 'be with', 'one'),\n",
       "   ('chi-square families', 'be in', 'which')],\n",
       "  [('contacts', 'be with', 'friends')],\n",
       "  [('medium', 'of percentage be', 'percentage'),\n",
       "   ('percentage', 'percentage of', 'fully vaccinated'),\n",
       "   ('high', 'to percentage be', 'percentage of fully vaccinated')],\n",
       "  [('normal times', 'be', 'also likely'),\n",
       "   ('normal times', 'protect by', 'vaccination')],\n",
       "  [('Location-based Mitigation Strategies Random',\n",
       "    'be in',\n",
       "    'Most Popular Location Highest Coreness'),\n",
       "   ('corresponding new infected individuals', 'be in', 'Clay Center'),\n",
       "   ('Location-based Mitigation Strategies Random',\n",
       "    'be in',\n",
       "    'Most Popular Location Random'),\n",
       "   ('Most Popular Location Highest Coreness',\n",
       "    'be in',\n",
       "    'Highest Coreness Location'),\n",
       "   ('Number',\n",
       "    'numericcitation with',\n",
       "    'corresponding new infected individuals in Clay Center given best estimated values for three contact levels'),\n",
       "   ('new infected individuals',\n",
       "    'of number be',\n",
       "    'NumericCitation with corresponding new infected individuals in Clay Center given best estimated values for three contact levels'),\n",
       "   ('NumericCitation',\n",
       "    'be with',\n",
       "    'corresponding new infected individuals in Clay Center given best estimated values for three contact levels'),\n",
       "   ('Most Popular Location Highest Coreness',\n",
       "    'be in',\n",
       "    'Most Popular Location Highest Coreness'),\n",
       "   ('new infected individuals', 'be in', 'La Gloria'),\n",
       "   ('Most Popular Location Random',\n",
       "    'be in',\n",
       "    'Highest Strength Location Highest Strength')],\n",
       "  [('edges', 'be with', 'weights greater than 0.10')],\n",
       "  [('cloud network', 'in node be', 'survey respondents'),\n",
       "   ('network', 'be in', 'Clay Center'),\n",
       "   ('nodes', 'be in', 'cloud network'),\n",
       "   ('locations', 'be in', 'Clay Center')],\n",
       "  [('map', 'be courtesy of', 'Google')]],\n",
       " 88: [[('spread',\n",
       "    'be in',\n",
       "    'two types of structured populations consisting of schools')],\n",
       "  [('part', 'be in', 'each of types')],\n",
       "  [('model', 'correspond to', 'well-studied household-workplace model')],\n",
       "  [('members', 'be in', 'hierarchical model')],\n",
       "  [('branching process approximation', 'be in', 'hierarchical model')],\n",
       "  [('epidemic spreads easier', 'be in', 'independent partition model')],\n",
       "  [('epidemics', 'for model be', 'SIR epidemic in closed population')],\n",
       "  [('epidemics',\n",
       "    'be in',\n",
       "    'large unstructured homogeneously mixing populations')],\n",
       "  [('PT', 'be support by', 'Swedish Research Council')],\n",
       "  [('homogeneous mixing assumption', 'be', 'too strong')],\n",
       "  [('population',\n",
       "    'be',\n",
       "    'In model partitioned into households of relatively small size')],\n",
       "  [('contacts',\n",
       "    'be',\n",
       "    'Within households more frequent than in general population')],\n",
       "  [('model', 'inspire by', 'modeling of spread of childhood diseases')],\n",
       "  [('spread', 'be in', 'schools'),\n",
       "   ('household-school model', 'be natural model in', 'case'),\n",
       "   ('natural model', 'be in', 'case'),\n",
       "   ('model', 'be', 'natural')],\n",
       "  [('partition', 'be in', 'households')],\n",
       "  [('siblings', 'go to', 'same school')],\n",
       "  [('independent partition model',\n",
       "    'be in',\n",
       "    'than more realistic hierarchical model')],\n",
       "  [('epidemic spreads', 'be in', 'independent partition model')],\n",
       "  [('We', 'branch process approximation for', 'conclusion')],\n",
       "  [('branching process approxiamtion', 'be in', 'hierarchical model')],\n",
       "  [('part', 'be in', 'model')],\n",
       "  [('independent numthe individuals', 'be in', 'k =')],\n",
       "  [('we', 'assume for', 'mathematical convenience')],\n",
       "  [('individuals', 'be in', 'population')],\n",
       "  [('number', 'be in', 'i.i.d. sizes'),\n",
       "   ('individuals', 'be in', 'school'),\n",
       "   ('we', 'use', 'independent partitions of N individuals in population'),\n",
       "   ('N individuals', 'be in', 'population'),\n",
       "   ('Nh', 'individual in', 'school')],\n",
       "  [('We', 'consider', 'stochastic SIR epidemic in closed population')],\n",
       "  [('susceptible one', 'become immediately', 'infectious')],\n",
       "  [('individual', 'stay', 'immune forever')],\n",
       "  [('Poisson processes', 'be with', 'intensity λh')],\n",
       "  [('Poisson processes', 'be with', 'intensity λs')],\n",
       "  [('Poisson processes', 'be with', 'intensity λg')],\n",
       "  [('other individuals', 'be', 'susceptible')],\n",
       "  [('household epidemic', 'be always restricted to', 'one household')],\n",
       "  [('other susceptible', 'be', 'contacts')],\n",
       "  [('contacts', 'be with', 'susceptible leads to transmission')],\n",
       "  [('inclused SEIR epidemic models',\n",
       "    'be with',\n",
       "    'non-random infectious period'),\n",
       "   ('individual', 'be', 'infected')],\n",
       "  [('us', 'carefully describe', 'initial phase of epidemic')],\n",
       "  [('interest', 'be in', 'hierarchical model'),\n",
       "   ('approximating branching process', 'be in', 'particular')],\n",
       "  [('schools', 'be relatively small compare to', 'total population'),\n",
       "   ('branching processes', 'make', 'actual computations')],\n",
       "  [('agreement', 'be with', 'theirs'),\n",
       "   ('our results', 'be in', 'agreement with theirs'),\n",
       "   ('our results', 'be in', 'agreement')],\n",
       "  [('We', 'prove', 'following theorem')],\n",
       "  [('hierarchical model', 'in individual be', 'P'),\n",
       "   ('ered individuals', 'be in', 'hierarchical model')],\n",
       "  [('ﬁxed household',\n",
       "    'for n be',\n",
       "    'epidemic spreads easier in independent model')],\n",
       "  [('large outbreak', 'be in', 'hierarchical model'),\n",
       "   ('large outbreak', 'be in', 'independent partition model')],\n",
       "  [('size j', 'be with', 'probability 1'),\n",
       "   ('size 1', 'be with', 'probability 1 −')],\n",
       "  [('Al l households', 'have', 'size 2')],\n",
       "  [('j', 'be with', 'probability')],\n",
       "  [('ph', '=', '1')],\n",
       "  [('We', 'prove theorem in', 'Section 5')],\n",
       "  [('Remark 2.5 examples', 'be in', 'Theorem 2.4')],\n",
       "  [('large school', 'be in', 'hierarchical model')],\n",
       "  [('large school', 'be in', 'hierarchical model'),\n",
       "   ('household members', 'be in', 'large school')],\n",
       "  [('vertices', 'represent', 'individuals')],\n",
       "  [('This', 'be', 'classical')],\n",
       "  [('λh', 'edge to', 'each of other vertices corresponding')],\n",
       "  [('individuals', 'be in', 'its global edges')],\n",
       "  [('edges', 'be', 'school')],\n",
       "  [('vertex', 'be in', 'epidemic')],\n",
       "  [('vertex', 'be in', 'epidemic')],\n",
       "  [('household cluster', 'have', 'similar interpretation as school cluster')],\n",
       "  [('large population limit', 'be with', 'probability tending to 1')],\n",
       "  [('number', 'be in', 'cluster')],\n",
       "  [('vertices', 'be in', 'large cluster')],\n",
       "  [('exact computations', 'be in', 'small populations')],\n",
       "  [('result', 'be special case of', 'p. 281'), ('result', 'be', 'special')],\n",
       "  [('L individuals', 'be', 'susceptible')],\n",
       "  [('us', 'eﬃciently compute', 'P L L')],\n",
       "  [('theorem', 'have', 'Citation'),\n",
       "   ('we', 'will use in', 'analysis of hierarchical model')],\n",
       "  [('k', 'be', 'positive integer'), ('k', 'be', 'positive')],\n",
       "  [('i', '=', '1 1 k')],\n",
       "  [('its infectious period', 'be in', 'case')],\n",
       "  [('SIR epidemics', 'be in', 'large homogeneously mixing populations')],\n",
       "  [('We', 'carry out approximation separately for', 'two types of models')],\n",
       "  [('particle', 'be in', 'branching processes')],\n",
       "  [('units', 'have', 'three types'),\n",
       "   ('branching process approximating', 'be in', 'independent partition model'),\n",
       "   ('4.1 independent partition model', 'be in', 'subsection')],\n",
       "  [('three types', 'be in', 'branching process')],\n",
       "  [('Poisson', 'be', 'distributed with mean µG'),\n",
       "   ('global children', 'be in', 'branching process of particle')],\n",
       "  [('Below', 'follow', 'more detailed approach')],\n",
       "  [('schools', 'be in', 'partition')],\n",
       "  [('schools', 'household from', '1 to n'),\n",
       "   ('n', 'be number of', 'households')],\n",
       "  [('it', 'be', 'given')],\n",
       "  [('ﬁrst repeated index', 'be in', 'sequence'), ('j', 'for x be', 'j')],\n",
       "  [('Th', 'be', 'ﬁrst repeated index in sequence of households'),\n",
       "   ('ﬁrst repeated index', 'be in', 'sequence of households'),\n",
       "   ('We', 'denote sequence by', '1')],\n",
       "  [('n →', 'as p be', 'Ts < nc')],\n",
       "  [('vertex', 'enter', 'process'),\n",
       "   ('we', 'distinguish between', 'three types of vertices')],\n",
       "  [('We', 'associate to', '1')],\n",
       "  [('− 1', 'be in', 'l − 1'), ('ancestor', 'be with', 'type school')],\n",
       "  [('k', 'be with', 'probability k number of school children'),\n",
       "   ('number', 'be with', 'type household')],\n",
       "  [('global children', 'be with', 'expectation λg')],\n",
       "  [('ancestor', 'be in', 'epidemic')],\n",
       "  [('we', 'retain', 'enough independence')],\n",
       "  [('children', 'to x be', 'k')],\n",
       "  [('following households', 'be in', 'sequence'),\n",
       "   ('following schools', 'be in', 'sequence')],\n",
       "  [('construction', 'be in', 'obvious way')],\n",
       "  [('ancestor', 'be in', 'branching process')],\n",
       "  [('household contacts', 'be in', 'x')],\n",
       "  [('vertices', 'be in', 'households')],\n",
       "  [('they', 'belong to', 'to generation 1 vertices')],\n",
       "  [('vertex', 'be', 'such')],\n",
       "  [('global children', 'be with', 'mean λg')],\n",
       "  [('school individual', 'have', 'random number of household children')],\n",
       "  [('individual', 'have', 'random number of school children')],\n",
       "  [('individual', 'be', 'global'),\n",
       "   ('initial infective individual', 'be', 'global individual')],\n",
       "  [('ms', '=', '∞ equally well known')],\n",
       "  [('the', 'be', 'positive')],\n",
       "  [('individuals', 'infect in', 'school epidemic')],\n",
       "  [('We', 'change', 'unit of branching process')],\n",
       "  [('particles', 'be in', 'branching process')],\n",
       "  [('number', 'be in', 'its corresponding cluster')],\n",
       "  [('size', 'be with', 'probability P k of ﬁrst cluster'),\n",
       "   ('l2', 'be with', 'probability P k − l1 − 1'),\n",
       "   ('l2', 'be in', 'way'),\n",
       "   ('l1', 'size of be', 'Indeed l2 with probability P k − l1 − 1 In way')],\n",
       "  [('clusters', 'be with', 'property'), ('individuals', 'be in', 'cluster')],\n",
       "  [('we', 'partition into', 'clusters generated by household edges')],\n",
       "  [('school', 'be with', 'given conﬁguration of households in it'),\n",
       "   ('given conﬁguration', 'be in', 'it'),\n",
       "   ('type', 'be', 'where number of vertices within cluster')],\n",
       "  [('individuals', 'be in', 'type j cluster')],\n",
       "  [('we', 'make', 'sum explicit')],\n",
       "  [('sizes', 'be in', 'school')],\n",
       "  [('households', 'be in', 'the')],\n",
       "  [('other households school part', 'be in', 'of school')],\n",
       "  [('household edges',\n",
       "    'contain',\n",
       "    'a uniformly at random chosen vertex from population')],\n",
       "  [('we', 'finally can describe', 'branching process')],\n",
       "  [('giant component', 'be in', 'epidemic generating means')],\n",
       "  [('initial cases', 'be in', 'schools')],\n",
       "  [('i.i.d. Poisson random variables', 'be with', 'mean λg')],\n",
       "  [],\n",
       "  [('branching process approximations', 'be in', 'previous section')],\n",
       "  [('we', 'uniformly pick', 'schools'),\n",
       "   ('schools', 'of x be', 'x'),\n",
       "   ('we', 'uniformly pick school with', 'replacement'),\n",
       "   ('schools', 'of x be', '1')],\n",
       "  [('households', 'of x be', '1'), ('households', 'of x be', 'x')],\n",
       "  [('Ts > k', 'be in', 'this')],\n",
       "  [('7 We', 'inclusion', 'ity space')],\n",
       "  [('school epidemic run', 'be in', 'school')],\n",
       "  [('independent partition model', 'in x be', 'x'),\n",
       "   ('independent partition model', 'in x be', '1'),\n",
       "   ('x', 'be in', 'independent partition model')],\n",
       "  [('j individuals', 'aﬀected by', 'school epidemic')],\n",
       "  [('individuals', 'be in', 'household')],\n",
       "  [('same quantity', 'be in', 'independent partition model')],\n",
       "  [('next step',\n",
       "    'investigate',\n",
       "    'school epidemics of individuals infected through household contacts')],\n",
       "  [('epidemic generating graph', 'be in', 'hierarchical model')],\n",
       "  [('such cluster 1', 'be in', 'hierarchical partition model'),\n",
       "   ('number', 'be in', 'such cluster 1')],\n",
       "  [('size l.', 'note', 'l ≤ l'),\n",
       "   ('vertices', 'be in', 'C H'),\n",
       "   ('15 next step', 'investigate', 'global edges from vertices in C H')],\n",
       "  [('i.i.d. Poisson numbers', 'be with', 'mean λg'),\n",
       "   ('other l − l vertices', 'be in', 'C I 1'),\n",
       "   ('ﬁrst l vertices', 'be in', 'C'),\n",
       "   ('l vertices', 'be in', 'C H')],\n",
       "  [('k', 'note from', 'C I 1')],\n",
       "  [('new schools', 'be in', 'same way')],\n",
       "  [('cluster', 'be in', 'independent partition model'),\n",
       "   ('we', 'have include', 'k vertices')],\n",
       "  [('we', 'condition on', 'Ak')],\n",
       "  [('We', 'computation with', 'Example 1'),\n",
       "   ('We computations', 'be with', 'Example 1')],\n",
       "  [('j', 'be with', 'probability 1')],\n",
       "  [('We', 'set', 'ps')],\n",
       "  [('j', 'be', 'large')],\n",
       "  [('individuals', 'be in', 'a'), ('It', 'be easy in', 'school/household')],\n",
       "  [('hierarchical model', 'in − be', '1 − ph'),\n",
       "   ('1 −', 'be in', 'hierarchical model'),\n",
       "   ('hierarchical model', 'in − be', '1 − pg'),\n",
       "   ('two vertices', 'be in', 'same household'),\n",
       "   ('we', 'have', 'that'),\n",
       "   ('hierarchical model', 'in − be', '1 − ps')],\n",
       "  [('j', 'be', 'large')],\n",
       "  [('order j', 'be in', 'particular'),\n",
       "   ('largest cluster', 'be in', 'large school')],\n",
       "  [('edge', 'actually end in', 'large household/school cluster')],\n",
       "  [('ph', '=', '1')],\n",
       "  [('This', 'use again', '5')],\n",
       "  [(\"Erd˝os R'enyi graph\", 'be in', 'which'),\n",
       "   ('average share edges', 'be with', '4/3 other vertices')],\n",
       "  [('we', 'can copy argument from', 'previous example')],\n",
       "  [('We',\n",
       "    'have discuss',\n",
       "    'ordering of epidemic severity of infectious diseases in two extreme population models')],\n",
       "  [('comparisons', 'probability of', 'large outbreak')],\n",
       "  [('R0', 'cross', 'threshold 1')],\n",
       "  [('tra jectory', 'be in', 'epidemic models')],\n",
       "  [('Adv.', 'be in', 'Appl')],\n",
       "  [('stochastic sir epidemics', 'be with', 'two levels of mixing')],\n",
       "  [('Epidemics', 'be with', 'two levels of mixing')],\n",
       "  [('random network', 'be with', 'household structure')],\n",
       "  [('Adv.', 'be in', 'Appl')],\n",
       "  [('Adv.', 'be in', 'Appl')],\n",
       "  [('epidemic models', 'be with', 'households')],\n",
       "  [('function', 'as r0 be', 'a'), ('rates', 'proportion of be', 'λg')]],\n",
       " 89: [[('Much eﬀort', 'use', 'mathematical models'),\n",
       "   ('Much eﬀort', 'have', 'has directed')],\n",
       "  [('outbreak', 'be in', 'certain communities')],\n",
       "  [('NumericCitation NumericCitation', 'be in', 'most cases')],\n",
       "  [('Ebola', 'be with', 'relatively small 1 ≤ R0 ≤ 7')],\n",
       "  [('recent simultaneous developments',\n",
       "    'be in',\n",
       "    'few countries around world NumericCitation')],\n",
       "  [('Much eﬀort',\n",
       "    'predict ebola outbreak with',\n",
       "    'help of mathematical modeling NumericCitation')],\n",
       "  [('location',\n",
       "    'removal of',\n",
       "    'infected individuals from community NumericCitation'),\n",
       "   ('dynamics', 'be in', 'speciﬁc communities')],\n",
       "  [('perturbation', 'be in', 'complex network of coupled nodes')],\n",
       "  [('hubs', 'level of', 'communities')],\n",
       "  [('global dynamics', 'be in', 'such system')],\n",
       "  [('progress', 'be in', 'single community')],\n",
       "  [('conjunction', 'be with', 'parameters estimated empirically'),\n",
       "   ('factors', 'include context be', 'quarantine')],\n",
       "  [('there has emerging interest',\n",
       "    'be in',\n",
       "    'network modeling of epidemics NumericCitation')],\n",
       "  [('nodes', 'be in', 'electric grid'),\n",
       "   ('1.2 Connectomics',\n",
       "    'be in',\n",
       "    'models of contagious disease Networked dynamic systems')],\n",
       "  [('system comparable', 'be in', 'size over time'),\n",
       "   ('setup nodes', 'couple at', 'diﬀerent spatio-temporal scales'),\n",
       "   ('population', 'be', 'comparable')],\n",
       "  [('their propagation', 'be in', 'complex network of neurons')],\n",
       "  [('same studies', 'rate of', 'new contacts identiﬁed')],\n",
       "  [('small local perturbations', 'be in', 'connectivity patterns')],\n",
       "  [('way',\n",
       "    'be with',\n",
       "    'propagation of electric charge through 2 brain network during seizure')],\n",
       "  [('one',\n",
       "    'understand',\n",
       "    'source of electric surge propagating through complex brain network'),\n",
       "   ('focal point', 'restore', 'proper function of neural mechanisms'),\n",
       "   ('focal point', 'thus remove', 'source of chaos')],\n",
       "  [('small change', 'be in', 'connectivity')],\n",
       "  [('other new approaches', 'be in', 'ﬁeld')],\n",
       "  [('one infected case', 'be in', 'community'),\n",
       "   ('conjunction', 'be with', 'our model')],\n",
       "  [('basic SIR model', 'be in', 'single relatively small community')],\n",
       "  [('authors',\n",
       "    'represent',\n",
       "    'latent population during virus incubation before development of symptoms'),\n",
       "   ('incubation', 'be in', 'model')],\n",
       "  [('contact', 'be with', 'infected individual')],\n",
       "  [('Citation contagion', 'be in', 'Yambuku')],\n",
       "  [('n nodes', 'represent', 'interconnected communities')],\n",
       "  [('A. Il lustration', 'be with', 'its 4 coupled SEIR variables')],\n",
       "  [('following system', 'to system be', '1')],\n",
       "  [('one', 'within', 'own community')],\n",
       "  [('2.5 %', 'be in', 'our simulations of system')],\n",
       "  [('latent individuals', 'travel out of', 'their own community'),\n",
       "   ('individuals', 'travel out of', 'their own community')],\n",
       "  [('they',\n",
       "    'would only permute between',\n",
       "    'Rk compartments of diﬀerent nodes')],\n",
       "  [('Simulation', 'be in', 'network with n ='),\n",
       "   ('contagion', 'be', 'triggered by only one latent individual in P1')],\n",
       "  [('Rk', 'show in', 'black')],\n",
       "  [('mind typical rural structures', 'be with', 'return')],\n",
       "  [('graph adjacency matrix C',\n",
       "    'induce',\n",
       "    'instantaneous spread of contagion')],\n",
       "  [('eﬀect', 'be in', 'particular on outbreak impact')],\n",
       "  [('ﬁrst model', 'for r be', 't'),\n",
       "   ('two focal points', 'be with', 'one infection')],\n",
       "  [('we', 'fix', 'network size')],\n",
       "  [('edge density ∆', 'for subset be', 'Figure 4'),\n",
       "   ('50 conﬁgurations', 'of subset be', 'Figure 4')],\n",
       "  [('unimodal shape', 'be with', 'peak in intermediate connectivity range'),\n",
       "   ('peak', 'be in', 'intermediate connectivity range')],\n",
       "  [('case', 'be in', 'same node')],\n",
       "  [('case', 'be in', 'same node')],\n",
       "  [('one foci', 'be in', 'low connectivity range'),\n",
       "   ('one focus', 'be in', 'intermediate connectivity range')],\n",
       "  [('n =', 'be in', 'Figure'), ('n =', 'be in', 'Figure 3')],\n",
       "  [('15 20 %', 'be in', 'our ﬁgures')],\n",
       "  [('model', 'reﬂect', 'more realistic trends')],\n",
       "  [('step t', 'connection to', 'j'), ('step t', 'connection from', 'k')],\n",
       "  [('single focal point', 'dependence of', 'impact'),\n",
       "   ('one individual', 'of point be', 'dependence'),\n",
       "   ('single focal point',\n",
       "    'dependence on',\n",
       "    'probability r travel out node along prescribed edges'),\n",
       "   ('single focal point', 'dependence on', 'density of edges')],\n",
       "  [],\n",
       "  [('r', '=', 'black')],\n",
       "  [('dependence', 'be in', 'r 0 range')],\n",
       "  [('we', 'introduce', 'quarantine measures'),\n",
       "   ('2.3 Modular adaptable network', 'be in', 'section'),\n",
       "   ('we', 'introduce', 'more structured quarantine measures')],\n",
       "  [('Model network architecture', 'be with', 'two interconnected modules')],\n",
       "  [('modules', 'each with', 'n =')],\n",
       "  [('two hubs',\n",
       "    'be connect by',\n",
       "    'one single bidirectional edge between their two central nodes')],\n",
       "  [('edges', 'be represent by', 'curved counter clock-wise'),\n",
       "   ('unidirectional edges', 'be represent by', 'curved counter clock-wise')],\n",
       "  [('transmission', 'be in', 'type of network')],\n",
       "  [('connections', 'be with', 'node Pk')],\n",
       "  [('original infection', 'be in', 'network')],\n",
       "  [('size n =', 'dependence of', 'R ∞'),\n",
       "   ('size n =', 'dependence on', 'θ'),\n",
       "   ('we', 'increase from', 'blue to green to red')],\n",
       "  [('bottom panels', 'consider', '50 edges')],\n",
       "  [('two hubs', 'each along', 'inter-modular edges'),\n",
       "   ('two hubs', 'each with', 'n = ='),\n",
       "   ('two hubs', 'each along', 'intra-modular edges =')],\n",
       "  [('two hubs', 'each along', 'inter-modular edges'),\n",
       "   ('two hubs', 'each with', 'n = ='),\n",
       "   ('two hubs', 'each along', 'intra-modular edges =')],\n",
       "  [('0.6 days', 'be in', 'red'), ('0.2 days', 'be in', 'blue')],\n",
       "  [('local delay θ', 'be with', 'same wait time separating')],\n",
       "  [('wide quantitative variations', 'be with', 'structure of network')],\n",
       "  [('people', 'by', 'likelihood travel along existing edges')],\n",
       "  [('curves', 'taper asymptotically', 'oﬀ'),\n",
       "   ('network', 'be with', 'less traveled edges')],\n",
       "  [('3 Discussion', 'be in', 'paper')],\n",
       "  [('context', 'be in', 'population network')],\n",
       "  [('information diﬀusion process', 'be in', 'network of coupled nodes')],\n",
       "  [('impact', 'be in', 'itself')],\n",
       "  [('delay', 'be in', 'global separation')],\n",
       "  [('helpful approach', 'be in', 'hypothetical outbreak')],\n",
       "  [('global separation', 'be', 'imposed faster than common sense')],\n",
       "  [('importance', 'be in', 'network exposed to epidemic outbreak')],\n",
       "  [('corresponding assumption',\n",
       "    'be in',\n",
       "    'our original reference paper NumericCitation')],\n",
       "  [('new infection', 'be with', 'diﬀerent strain'),\n",
       "   ('prior infection',\n",
       "    'be with',\n",
       "    'one strain irrelevant immunologically to new infection')],\n",
       "  [('NumericCitation', 'be with', 'potential of infection'),\n",
       "   ('contact', 'be with', 'dead body')],\n",
       "  [('adequate quarantine systems',\n",
       "    'be customize in',\n",
       "    'response to local details of network architecture')],\n",
       "  [('brain proﬁling', 'be in', 'same way')],\n",
       "  [('may placed investment', 'be in', 'view of potential pandemic'),\n",
       "   ('cost', 'be', 'placed')],\n",
       "  [('NumericCitation Marc Barth', 'elemy be', 'Romualdo Pastor-Satorras'),\n",
       "   ('NumericCitation Marc Barth', 'elemy be', 'Alain Barrat')],\n",
       "  [('Velocity spread', 'be in', 'scale-free networks')],\n",
       "  [('Outbreaks', 'be in', 'africa')],\n",
       "  [('sirs epidemic model', 'be with', 'nonlinear incidence rate')],\n",
       "  [('stochastic sir epidemic models', 'be with', 'random')],\n",
       "  [('Epidemic simulation system', 'be with', 'realistic mobility')],\n",
       "  [('Statistical inference',\n",
       "    'be in',\n",
       "    'stochastic epidemic seir model with control intervention'),\n",
       "   ('stochastic epidemic seir model', 'be with', 'control intervention')],\n",
       "  [('ebola virus transmission', 'be in', 'montserrado')],\n",
       "  [('Epidemic outbreaks', 'be in', 'complex heterogeneous networks')],\n",
       "  [('Bifurcation analysis', 'be in', 'epidemic model')],\n",
       "  [('non-autonomous seirs model', 'be with', 'quarantine')],\n",
       "  [('Theory', 'be in', 'Biosciences 131 1')],\n",
       "  [('its', 'application', '336 3')]],\n",
       " 90: [[('lives', 'minimize', 'health care costs')],\n",
       "  [('deterministic model',\n",
       "    'be',\n",
       "    'For intermediate quantities of vaccine poor estimate of optimal strategy for more realistic stochastic case'),\n",
       "   ('model', 'be', 'poor')],\n",
       "  [('Introduction', 'be', 'growing'),\n",
       "   ('Introduction', 'be', 'global'),\n",
       "   ('Introduction', 'transmission of', 'infectious diseases')],\n",
       "  [('Advances', 'be in', 'biomedical therapies')],\n",
       "  [('disease transmission', 'be in', 'populations')],\n",
       "  [('susceptible individual', 'become', 'infected')],\n",
       "  [('SIR model', 'can', 'can parameterized by single number')],\n",
       "  [('Numerous computational studies', 'have', 'have performed'),\n",
       "   ('Johne', 'have', 'Disease NumericCitation')],\n",
       "  [('delays', 'associate with', '2 immune response')],\n",
       "  [('stochastic SIR dynamics', 'be in', 'two non-interacting populations'),\n",
       "   ('abstract geographic speciﬁc information', 'be in', 'paper'),\n",
       "   ('stochastic SIR dynamics', 'be with', 'prophylactic vaccination')],\n",
       "  [('Keeling', 'obtain', 'striking results'),\n",
       "   ('deterministic SIR model', 'be in', 'scenario')],\n",
       "  [('smaller population', 'be well protect', 'foremost')],\n",
       "  [('infected individual', 'will infect', 'fewer individuals')],\n",
       "  [('sharp transitions', 'be in', 'optimal strategy')],\n",
       "  [('high dimensional state space', 'be with', 'probabilistic transitions'),\n",
       "   ('individuals', 'be in', 'state')],\n",
       "  [('Stochasticity', 'lead to', 'noteworthy diﬀerences in epidemic size'),\n",
       "   ('noteworthy diﬀerences', 'be in', 'epidemic size')],\n",
       "  [('population size', 'be', 'when large'),\n",
       "   ('epidemic',\n",
       "    'progress to',\n",
       "    'large size characterized on average by deterministic results')],\n",
       "  [('given population results',\n",
       "    'be in',\n",
       "    'signiﬁcant diﬀerences for optimal allocation of vaccine')],\n",
       "  [('individuals', 'be in', 'state')],\n",
       "  [('growth', 'be in', 'well-mixed population'),\n",
       "   ('deterministic SIR model NumericCitation',\n",
       "    'system of',\n",
       "    'coupled diﬀerential equations for modeling')],\n",
       "  [('total number individuals', 'be in', 'population')],\n",
       "  [('contact p', 'probability of', 'rate'),\n",
       "   ('contact p', 'probability at', 'which')],\n",
       "  [('βS0 γ', '=', 'c × p × T × S0 N')],\n",
       "  [('single infected individual',\n",
       "    'will produce in',\n",
       "    'completely susceptible')],\n",
       "  [('simulations', 'value of', 'r0')],\n",
       "  [('vaccine', 'be', 'completely eﬀective')],\n",
       "  [('reﬀ', '=', 'c × p × T × S0 − V N S0 − V')],\n",
       "  [('V', 'be', 'vaccinated')],\n",
       "  [('entire population', 'vaccinate', 'entire population')],\n",
       "  [('phenomenon', 'be know as', 'herd immunity')],\n",
       "  [('recovery reactions',\n",
       "    'be',\n",
       "    'In stochastic SIR model modeled as continuous-time Markovian processes')],\n",
       "  [('population', 'be with', 'S susceptible individuals')],\n",
       "  [('evolution', 'be in', 'time')],\n",
       "  [('Goutsias',\n",
       "    'have',\n",
       "    'method NumericCitation of Implicit-Euler integration')],\n",
       "  [('d φ', '=', 'φdt')],\n",
       "  [('components', 'be in', 'such way so For computational eﬃciency')],\n",
       "  [('We',\n",
       "    'enable',\n",
       "    'considerably faster computational speed especially as population size increases'),\n",
       "   ('algorithm', 'count', 'states')],\n",
       "  [('state', 'be in', 'In degree of advancement procedure')],\n",
       "  [('∼ N 2/2', 'be in', 'limit I0 S0')],\n",
       "  [('it', 'integrate', 'system of equations'),\n",
       "   ('diﬀerence', 'be in', 'total number of states between two methods')],\n",
       "  [('various values', 'be in', 'stochastic SIR model')],\n",
       "  [('left vertical scale', 'be in', 'Figure A')],\n",
       "  [('weight', 'be in', 'terminal infection'),\n",
       "   ('weight', 'be in', 'large-scale epidemic')],\n",
       "  [('numerical calculations', 'use', 'MATLAB')],\n",
       "  [('t → ∞ N − S', 'be in', 'stochastic model'),\n",
       "   ('simulation', 'be', 'Once complete'),\n",
       "   ('stochastic model', 'in s be', 't'),\n",
       "   ('E =', 'be in', 'deterministic model')],\n",
       "  [('infection', 'become', 'large scale epidemic')],\n",
       "  [('corresponding vertical dashed line', 'be in', 'Figure 1A')],\n",
       "  [('size', 'be with', 'size of population N')],\n",
       "  [('total weight', 'be in', 'terminal infection peak')],\n",
       "  [('local minimum', 'be in', 'P E')],\n",
       "  [('separation', 'be with', 'little probability observing in between two')],\n",
       "  [('r0', 'of value be', 'greater likelihood')],\n",
       "  [('diﬀerent levels', 'be in', 'stochastic SIR models'),\n",
       "   ('results', 'provide', 'basis for optimization process')],\n",
       "  [('we', 'contrast', 'their properties')],\n",
       "  [('stochastic optimal solution',\n",
       "    'better fulﬁlls measure than',\n",
       "    'deterministic optimum')],\n",
       "  [('Equation 8', 'equation for', 'deterministic SIR model')],\n",
       "  [('incremental reduction',\n",
       "    'be in',\n",
       "    'expected epidemic size per incremental increase'),\n",
       "   ('incremental increase', 'be in', 'allocation')],\n",
       "  [('We', 'deﬁne below as', 'gain G')],\n",
       "  [('corresponding deterministic curve', 'be in', 'case')],\n",
       "  [('there may large separation', 'be with', 'mean size lying somewhere in')],\n",
       "  [('infected individuals', 'be in', 'population')],\n",
       "  [('V', '=', '450'), ('V', '=', '400')],\n",
       "  [('increase', 'be in', 'vaccine allocation'),\n",
       "   ('incremental reduction', 'be in', 'expected epidemic size')],\n",
       "  [('N', 'be', 'when ﬁnite'), ('N', 'be', 'ﬁnite')],\n",
       "  [('I0', 'be in', 'deﬁnition of reﬀ')],\n",
       "  [('corresponding transition', 'be subtler in', 'stochastic model')],\n",
       "  [('It', 'decrease', 'mean size E')],\n",
       "  [('local minimum', 'be in', 'P E'),\n",
       "   ('stochastic model', 'be with', 'value of V'),\n",
       "   ('eﬀective herd immunity', 'be in', 'stochastic model with value of V')],\n",
       "  [('herd immunity transition', 'be in', 'deterministic model')],\n",
       "  [('speciﬁc value', 'be in', 'allocation'),\n",
       "   ('reduction', 'be in', 'average epidemic size')],\n",
       "  [('standard deviation', 'be in', 'large-scale epidemic')],\n",
       "  [('suﬃcient vaccine', 'establish herd immunity in', 'deterministic model'),\n",
       "   ('deterministic size E',\n",
       "    'be generally smaller than',\n",
       "    'average stochastic epidemic size E')],\n",
       "  [('epidemic E',\n",
       "    'be',\n",
       "    'Hence after herd immunity threshold smaller than stochastic average epidemic E')],\n",
       "  [('it', 'spread', 'their disease')],\n",
       "  [('increase', 'be with', 'decreasing r0')],\n",
       "  [('function', 'amount of', 'vaccine allocated to population'),\n",
       "   ('V', 'of function be', 'amount')],\n",
       "  [('having vaccinated individuals', 'reduce', 'reﬀ')],\n",
       "  [('largest uncertainty', 'be in', 'size of large-scale epidemic')],\n",
       "  [('10 G =', 'be in', 'deterministic model'),\n",
       "   ('decrease', 'be in', 'epidemic size'),\n",
       "   ('10 G =', 'be in', 'stochastic model')],\n",
       "  [('r0 =', 'be in', 'Figure 3B')],\n",
       "  [('r0 =', 'be in', 'Figure 3A')],\n",
       "  [('1 curves', 'be in', 'Figures 2A'), ('I0', 'of g be', 'slopes')],\n",
       "  [('large r0 case', 'be in', 'stochastic model')],\n",
       "  [('level', 'be in', 'population'),\n",
       "   ('signiﬁcant increase', 'be in', 'gain G'),\n",
       "   ('the', 'be', 'signiﬁcant')],\n",
       "  [('gain G', 'be in', 'stochastic model')],\n",
       "  [('fundamental tradeoﬀ', 'be in', 'resource management')],\n",
       "  [('We', 'assume in', 'section one population')],\n",
       "  [('one seed infected individual', 'be in', 'population')],\n",
       "  [('deterministic case', 'be in', 'limit of large population sizes')],\n",
       "  [('Optimal Solution', 'be in', 'Stochastic Model')],\n",
       "  [('color scale',\n",
       "    'indicate',\n",
       "    'average epidemic size E corresponding to particular allocation of vaccine')],\n",
       "  [('Results', 'be', 'obtained')],\n",
       "  [('amount', 'be in', 'smaller population V 1')],\n",
       "  [('E', 'be', 'minimized')],\n",
       "  [('number', 'be in', 'population'), ('range', 'be in', 'V 1')],\n",
       "  [('12 Switching', 'be in', 'deterministic model found')],\n",
       "  [('vaccine V', 'amount of be', 'When threshold size')],\n",
       "  [('vaccine V', 'amount of be', 'When size')],\n",
       "  [('behavior', 'r0', '> 1 in deterministic SIR model')],\n",
       "  [('r0 =', 'be in', 'Figure 4A')],\n",
       "  [('second switching point', 'be absent in', 'deterministic model')],\n",
       "  [('relative heights', 'be in', 'next subsection'),\n",
       "   ('second switch', 'be in', 'stochastic model')],\n",
       "  [('given total', 'be in', 'stochastic model')],\n",
       "  [('r0 =', 'be in', 'Figure 4B')],\n",
       "  [('r0', 'be', 'decreased')],\n",
       "  [('r0 dependence', 'be in', 'stochastic models'),\n",
       "   ('presence', 'be in', 'gain curves')],\n",
       "  [('optimal solution',\n",
       "    'approximately distribute vaccine in',\n",
       "    'proportion to population size')],\n",
       "  [('two switches', 'take place for', 'r0 2.9'),\n",
       "   ('two switches', 'take', 'place')],\n",
       "  [('optimal stochastic solutions', 'be in', 'closer detail')],\n",
       "  [('optimal solution', 'be in', 'diﬀerent resource regimes'),\n",
       "   ('We',\n",
       "    'characterize',\n",
       "    'the represented by fraction of diﬀerent strategies employed by optimal solution in diﬀerent resource regimes')],\n",
       "  [('average epidemic size E',\n",
       "    'diﬀers from',\n",
       "    'optimal deterministic solution')],\n",
       "  [('decrease', 'be in', 'epidemic size')],\n",
       "  [('switching', 'do take', 'place')],\n",
       "  [('remaining vaccine', 'be optimally allocate to', 'large population')],\n",
       "  [('diﬀerence', 'be in', 'strategy'),\n",
       "   ('solution', 'maintain', 'diﬀerence in strategy'),\n",
       "   ('gain G', 'decline well before', 'onset of eﬀective herd immunity')],\n",
       "  [('gain', 'be in', 'deterministic model'), ('N 1 2', 'be in', 'contrast')],\n",
       "  [('right vertical scale', 'apply to', 'two measurements')],\n",
       "  [('diﬀerence', 'be in', 'resultant average epidemic size E')],\n",
       "  [('r0 =', 'be in', 'Figure A'), ('r0 =', 'be in', 'Figure B')],\n",
       "  [('earlier decline', 'be in', 'average epidemic size E')],\n",
       "  [('stochastic model', 'be in', 'than deterministic model')],\n",
       "  [('larger gains', 'be in', 'protection')],\n",
       "  [('same behavior', 'be observe for', 'V'),\n",
       "   ('larger values', 'for g be', 'V')],\n",
       "  [('optimal solution',\n",
       "    'increasingly allocate resource to',\n",
       "    'small population')],\n",
       "  [('situation', 'be with', 'excess vaccine')],\n",
       "  [('vaccine', 'be once again allocate to', 'small population')],\n",
       "  [('second abrupt shift',\n",
       "    'be with',\n",
       "    'optimal population based plateau value V 1/V = from above'),\n",
       "   ('second abrupt shift', 'be in', 'resources'),\n",
       "   ('cusp', 'be in', 'optimal V 1')],\n",
       "  [('greater peak', 'be in', 'gain')],\n",
       "  [('enough vaccine available', 'large r0 in', 'stochastic model'),\n",
       "   ('higher gain', 'be in', 'smaller population')],\n",
       "  [('stochastic model', 'curve of', 'Figure 3B')],\n",
       "  [('lower values', 'be in', 'case')],\n",
       "  [('absence', 'be in', 'gain curve')],\n",
       "  [('declines', 'be in', 'gain')],\n",
       "  [('peak', 'be in', 'gain')],\n",
       "  [('presence', 'be in', 'gain curves')],\n",
       "  [('diﬀerences',\n",
       "    'be in',\n",
       "    'optimal vaccination protocols between stochastic models'),\n",
       "   ('substantial diﬀerences', 'be in', 'observed outcomes')],\n",
       "  [('it', 'allocate vaccine in', 'proportion to population size')],\n",
       "  [('optimization', 'be', 'most important'),\n",
       "   ('intermediate amounts', 'be', 'when available')],\n",
       "  [('diﬀerence', 'be in', 'resultant average epidemic size E')],\n",
       "  [('worst case protocol', 'reach', 'herd immunity')],\n",
       "  [('deterministic herd immunity threshold', 'serve as', '15 Figure 6')],\n",
       "  [('two populations', 'be with', 'size N 1 ='),\n",
       "   ('Results', 'be', 'obtained')],\n",
       "  [('r0 =', 'be in', 'Figure A'), ('r0', '=', '2')],\n",
       "  [('color style legends', 'be in', 'B')],\n",
       "  [('diﬀerence', 'be in', 'E')],\n",
       "  [('diﬀerence', 'be in', 'average epidemic size')],\n",
       "  [('diﬀerence', 'be in', 'average epidemic size')],\n",
       "  [('N 1 =', 'with population be', 'two well-separated cities'),\n",
       "   ('two non-interacting populations', 'be with', 'N 1 =')],\n",
       "  [('results', 'be show in', 'Figure 7A')],\n",
       "  [('step-like changes', 'be in', 'probability'),\n",
       "   ('diagonal bands', 'be in', 'Figure 7A')],\n",
       "  [('event', 'be in', 'size range between large-scale epidemic'),\n",
       "   ('discrete steps', 'be in', 'probability'),\n",
       "   ('large population', 'sum of', 'two')],\n",
       "  [('situation', 'be with', 'few resources'),\n",
       "   ('left white triangle', 'be in', 'Figure 7B')],\n",
       "  [('maximum achievable probability', 'be in', 'Figure 7A'),\n",
       "   ('unity', 'be in', 'Emax dependent resource rich regime'),\n",
       "   ('horizontal bands', 'be in', 'Figure 7B')],\n",
       "  [('tradeoﬀs', 'be', 'where critical'),\n",
       "   ('tradeoﬀs', 'be', 'where most critical')],\n",
       "  [('smaller epidemics', 'be in', 'regions')],\n",
       "  [('optimal deterministic protocol', 'be', 'when applied')],\n",
       "  [('deterministic protocol', 'maximize', 'E < Emax')],\n",
       "  [('stochastic optimum', 'be in', 'roughly same regions')],\n",
       "  [('fraction', 'be in', 'smaller population')],\n",
       "  [('two populations', 'be with', 'size N 1 =')],\n",
       "  [('one', 'optimize', 'average epidemic size E')],\n",
       "  [('size', 'be', 'useful'),\n",
       "   ('average epidemic size', 'gauge', 'eﬀects of stochasticity')],\n",
       "  [('We', 'obtain', 'even greater eﬃciency'),\n",
       "   ('We', 'obtain', 'even greater numerical eﬃciency')],\n",
       "  [('construction', 'be in', 'original method')],\n",
       "  [('Our procedure', 'involve', 'three steps')],\n",
       "  [('function', 'be in', 'population')],\n",
       "  [('we', 'distribute between', 'two populations')],\n",
       "  [('corresponding tradeoﬀ', 'be in', 'context of deterministic SIR model')],\n",
       "  [('distribution', 'characteristic of', 'which')],\n",
       "  [('NumericCitation',\n",
       "    'be in',\n",
       "    'limit of distribution of epidemic sizes obtained from stochastic SIR model large population sizes')],\n",
       "  [('diﬀerences',\n",
       "    'be in',\n",
       "    'optimal allocation of vaccine for stochastic model')],\n",
       "  [('diﬀerences', 'be most signiﬁcant for', 'intermediate ranges of vaccine')],\n",
       "  [('Diﬀerences', 'be in', 'optimal allocations')],\n",
       "  [('realistic stochastic description', 'be in', 'two populations')],\n",
       "  [('I0', 'on dependence be', 'number of infected individual'),\n",
       "   ('dependence', 'number of', 'infected individual')],\n",
       "  [('relative weight', 'be in', 'peak describing')],\n",
       "  [('infected individuals', 'be in', 'population')],\n",
       "  [('sharp transitions', 'be in', '19 deterministic models'),\n",
       "   ('we', 'maximize', 'overall gain')],\n",
       "  [('herd immunity', 'be', 'approached')],\n",
       "  [('reduction', 'be in', 'overall epidemic size')],\n",
       "  [('allocations', 'be in', 'proportion to population size'),\n",
       "   ('allocations', 'be likely in', 'contrast')],\n",
       "  [('stochastic model', 'be in', 'Figure 3'),\n",
       "   ('stochastic model',\n",
       "    'exhibit',\n",
       "    'less dramatic switching for smaller values of r0'),\n",
       "   ('reduction', 'be in', 'magnitude of gain')],\n",
       "  [('allocation', 'be in', 'stochastic model'),\n",
       "   ('it', 'be in', 'corresponding deterministic case')],\n",
       "  [('overall epidemic size', 'for tolerance be', 'threshold')],\n",
       "  [('result', 'be', 'somewhat surprising')],\n",
       "  [('stochastic protocol', 'outperform', 'deterministic counterpart'),\n",
       "   ('complete stochastic dynamics', 'be in', 'further studies')],\n",
       "  [('tradeoﬀ', 'be in', 'vaccination allocation')],\n",
       "  [('strong assumptions', 'assumption of', 'non-interacting populations'),\n",
       "   ('Our analysis',\n",
       "    'rely on',\n",
       "    'particularly assumption of non-interacting populations')],\n",
       "  [('extreme switching behavior', 'be in', 'deterministic case')],\n",
       "  [('modest amount', 'be in', 'particularly deterministic case')],\n",
       "  [('epidemic outbreak', 'be in', 'localized geographic region')],\n",
       "  [('vaccine doses', 'have', 'where have prepositioned geographically'),\n",
       "   ('vaccine doses', 'have', 'where have prepositioned'),\n",
       "   ('will intense debate', 'be in', 'situations')]],\n",
       " 91: [[('s c i', 'be', 'y h p')],\n",
       "  [('we', 'utilize', 'state propagation approach'),\n",
       "   ('we', 'infer', 'infection source')],\n",
       "  [('his social page', 'be in', 'recent past'),\n",
       "   ('rumor spreading', 'be in', 'online social network like Facebook')],\n",
       "  [('it', 'have infect', 'neighbors'),\n",
       "   ('infected node', 'stay', 'infected NumericCitation')],\n",
       "  [('distance',\n",
       "    'such as estimator be',\n",
       "    'belief propagation estimator NumericCitation'),\n",
       "   ('distance',\n",
       "    'such as estimator be',\n",
       "    'dynamic message passing estimator NumericCitation'),\n",
       "   ('distance', 'such as estimator be', 'Jordan center NumericCitation')],\n",
       "  [('rumor spreading', 'be in', 'online social network')],\n",
       "  [('node', 'recover', 'multiple times')],\n",
       "  [('additional observations', 'be in', 'periods other than snapshot time')],\n",
       "  [('its applicability',\n",
       "    'be with',\n",
       "    'nodes following different infection models')],\n",
       "  [('SIRI infection spreading', 'use', 'state-space approach NumericCitation'),\n",
       "   ('MODEL', 'be in', 'section')],\n",
       "  [('common underlying probability space',\n",
       "    'be with',\n",
       "    'probability measure P')],\n",
       "  [('We', 'denote', '0 1 n')],\n",
       "  [('case node l', 'be in-neighbor of', 'node k')],\n",
       "  [('set', 'be in', 'network')],\n",
       "  [('probabilities', 'be', 'inferred')],\n",
       "  [('βk', '=', '0'), ('γk', '=', '0')],\n",
       "  [('Our model', 'thus subsume', 'those'),\n",
       "   ('Our model', 'thus subsume', 'those studied in NumericCitation')],\n",
       "  [('we', 'adopt', 'state-space modeling approach')],\n",
       "  [('we', 'use', 'state transitions for SIRI model')],\n",
       "  [('infection', 'utilize', 'mean-ﬁeld approximation')],\n",
       "  [('node k', 'be in', 'state S'), ('node k', 'be in', 'state S at time t1')],\n",
       "  [('we', 'complete', 'model'),\n",
       "   ('we', 'introduce', 'two classes of auxiliary state variables'),\n",
       "   ('we', 'introduce', 'two classes')],\n",
       "  [('DMP equations', 'be in', 'NumericCitation')],\n",
       "  [('k ∈ N S', 'if k be', 't'),\n",
       "   ('k ∈ N S', 'if k be', '10'),\n",
       "   ('k ∈ N I t', 'if k be', '10'),\n",
       "   ('k ∈ N I t', 'if k be', 't')],\n",
       "  [('last observation vector',\n",
       "    'be deﬁned merely for',\n",
       "    'theoretical completeness')],\n",
       "  [('We', 'identify', 'INFECTION SOURCE Wk'),\n",
       "   ('we', 'infer', 'identity of infection source')],\n",
       "  [('slots', 'be in', 'Wk')],\n",
       "  [('set', 'be with', 'at least one observation up to snapshot time')],\n",
       "  [('We',\n",
       "    'formulate source identiﬁcation problem as',\n",
       "    'approximate MAP estimation of infection source')],\n",
       "  [('T c', 'be', 'where set of candidate snapshot times')],\n",
       "  [('joint probability P N O | s ∗ = s N O', 'be', 'given observations')],\n",
       "  [('t ≥ 0', 'for pk be', 't'),\n",
       "   ('k ∈ N c', 'for k be', '0'),\n",
       "   ('objective function', 'in yk be', 't'),\n",
       "   ('observation variables yk', 'be in', 'objective function')],\n",
       "  [('candidate source', 'give o be', 'dTf | N |'),\n",
       "   ('d', 'be', 'where average in-degree of reduced graph')],\n",
       "  [('assumption', 'be', 'where violated')],\n",
       "  [('success', 'be in', 'general network NumericCitation'),\n",
       "   ('similar argument',\n",
       "    'explain',\n",
       "    'success of well-known BP methods in general network NumericCitation')],\n",
       "  [('we', 'hence naively treat siri model as', 'SIR model'),\n",
       "   ('we', 'apply', 'DMP estimator')],\n",
       "  [('source', 'be in', 'distance'),\n",
       "   ('objective function value', 'be in', 'P0'),\n",
       "   ('we', 'compare', 'estimators')],\n",
       "  [('states', 'be', 'observed')],\n",
       "  [('states', 'be', 'observed')],\n",
       "  [('we', 'randomly choose', 'infection source'),\n",
       "   ('1000 nodes', 'be with', \"node 's degree equal to 4 NumericCitation\"),\n",
       "   ('node', 'with', 'degree equal to 4 NumericCitation')],\n",
       "  [('we', 'simulate', 'availability of different side information')],\n",
       "  [('general infection model', 'be in', 'inference')],\n",
       "  [('results', 'be show in', 'Fig. 3')],\n",
       "  [('case HISS', 'may become', 'inferior to DMP')],\n",
       "  [('Its corresponding average', 'normalize', 'rank of true source')],\n",
       "  [('We',\n",
       "    'have introduce',\n",
       "    'state-space description of SIRI infection model')],\n",
       "  [('HISS method', 'do', 'comparisons')],\n",
       "  [('authors', 'also thank wuqiong luo for', 'helpful discussions')],\n",
       "  [('Rumors', 'be in', 'network')],\n",
       "  [('00.20.40.60.8100.10.20.30.40.5 Nnormalizedrankofs',\n",
       "    '∗',\n",
       "    'Infectionrate00 .20.40.60.8101234567 IEEE Trans'),\n",
       "   ('Identifying infection sources', 'be in', 'large networks')],\n",
       "  [('Identifying infection sources', 'be in', 'network'),\n",
       "   ('NumericCitation W. Luo', 'source in', 'network')],\n",
       "  [('NumericCitation J. Leskovec', 'be in', 'Advances'),\n",
       "   ('Advances', 'be in', 'Neural Information Processing Systems 25 2012'),\n",
       "   ('social circles', 'be in', 'ego networks')],\n",
       "  [('herpes infections', 'be in', 'human populations')],\n",
       "  [('Modeling relapse', 'be in', 'infectious diseases')],\n",
       "  [('Topics', 'signal', 'Process.'), ('Topics', 'signal', 'vol')],\n",
       "  [('Information Theory Proceedings', 'on symposium be', 'ISIT'),\n",
       "   ('Information Theory Proceedings', 'on symposium be', 'Citation')],\n",
       "  [('Information source detection', 'be in', 'sir model'),\n",
       "   ('based approach', 'be in', 'Information Theory 2013')],\n",
       "  [('robust information source estimator', 'be with', 'sparse observations')],\n",
       "  [('L. Zdeborov', None, 'Phys')],\n",
       "  [('zero-patient problem', 'be with', 'noisy observations')],\n",
       "  [('siri epidemic model', 'be with', 'nonlinear incidence'),\n",
       "   ('Advances', 'be in', 'Difference Equations')],\n",
       "  [('scaling analysis', 'be in', 'siri epidemiological model')],\n",
       "  [('models', 'be with', 'unidirectional dynamics')],\n",
       "  [('Uncertainty', 'be in', 'Artiﬁcial Intelligence'),\n",
       "   ('empirical study',\n",
       "    'be in',\n",
       "    'Proceedings of Fifteenth Conference on Uncertainty')]],\n",
       " 92: [[('rise', 'be in', 'popularity of OSNs used')],\n",
       "  [('rapid drop', 'be in', 'popularity of Facebook')],\n",
       "  [('arbitrary proportion', 'be in', 'long run')],\n",
       "  [('persistence', 'be', 'important')],\n",
       "  [('dynamics', 'be with', 'inspiration from mathematical epidemiology')],\n",
       "  [('infectious individuals', 'progress to', 'recovered state'),\n",
       "   ('interactions', 'be with', 'infectious individuals')],\n",
       "  [('individuals', 'be in', 'two states')],\n",
       "  [('spontaneous reactions', 'be in', 'epidemic processes')],\n",
       "  [('SIR model', 'be with', 'schematic diagrams in Figs.'),\n",
       "   ('schematic diagrams', 'be in', 'Figs.')],\n",
       "  [('nonlinear system Eq', 'through cooney be', 'FRANCISCO PRIETO')],\n",
       "  [('population size N', 'be', 'constant')],\n",
       "  [('we', 'represent', 'fractions')],\n",
       "  [('we', 'construct', 'representations of epidemic models'),\n",
       "   ('we', 'construct', 'schematic representations of epidemic models')],\n",
       "  [('interaction', 'be with', 'type C individual')],\n",
       "  [('I → R transition', 'be', 'spontaneous')],\n",
       "  [('individuals', 'join osn', 'S-I interaction')],\n",
       "  [('system', 'be in', 'Eq'), ('Their', 'sir', 'irSIR')],\n",
       "  [('they', 'use', 'best-ﬁt techniques'),\n",
       "   ('Google Search interest', 'be in', 'MySpace'),\n",
       "   ('they', 'use', 'techniques')],\n",
       "  [('solutions',\n",
       "    'be in',\n",
       "    'exponential growth phase of irSIR model NumericCitation')],\n",
       "  [('users', 'be in', 'state I'),\n",
       "   ('users', 'be in', 'state R'),\n",
       "   ('single OSN', 'be in', 'isolation')],\n",
       "  [('rise', 'be in', 'Japan'), ('Facebook', 'of', 'popularity')],\n",
       "  [('two OSNs',\n",
       "    'also experience',\n",
       "    'near-term downturn in active membership NumericCitation'),\n",
       "   ('near-term downturn', 'be in', 'active membership NumericCitation')],\n",
       "  [('epidemic model', 'be with', 'infectious-recovery term'),\n",
       "   ('Cintron-Arias', 'describe', 'popularity of viral videos')],\n",
       "  [('interaction', 'be with', 'active users NumericCitation')],\n",
       "  [('we',\n",
       "    'here consider',\n",
       "    'question of long term behavior for infectious-recovery epidemic models')],\n",
       "  [('long-run endemic equilibrium',\n",
       "    'can',\n",
       "    'can sustained in extensions of traditional SIR model')],\n",
       "  [('reasonable augmentations',\n",
       "    'lead to',\n",
       "    'dramatic changes in long time behavior'),\n",
       "   ('dramatic changes', 'be in', 'long time behavior')],\n",
       "  [('underlying patterns', 'be in', 'abundantly available social media data')],\n",
       "  [('Spechler', 'have', 'original irSIR model'),\n",
       "   ('system', 'of', 'equilibria')],\n",
       "  [('irSIRS model', 'be with', 'spontaneous loss of immunity'),\n",
       "   ('recovered individuals', 'can return to', 'susceptibility on their own')],\n",
       "  [('We', 'review', 'steady states of dynamical system')],\n",
       "  [('point', 'be', 'ﬁxed')],\n",
       "  [('dt', '=', '0')],\n",
       "  [('I', '−', '1 − 2I'), ('equilibrium', 'be', 'so unstable')],\n",
       "  [('populations', 'be in', 'irSIR model')],\n",
       "  [('saturation', 'be', 'realistic')],\n",
       "  [('We', 'show', 'this')],\n",
       "  [('Numerical integration', 'be with', 'curves representing in population')],\n",
       "  [('dynamics', 'initial proportion of', '0')],\n",
       "  [('initial exponential growth', 'be in', 'I t')],\n",
       "  [('other hand', 'on infectiousness be', 'dI dt ='),\n",
       "   ('other hand', 'on infectiousness be', 'time t ∗')],\n",
       "  [('t', '=', '0')],\n",
       "  [('irSIR system', 'be with', 'initial condition near point')],\n",
       "  [('γ continual decrease', 'be in', 'number of susceptibles S')],\n",
       "  [('initial rightward motion', 'be in', 'I direction')],\n",
       "  [('we', 'substitute in', 'Eq')],\n",
       "  [('individuals', 'be in', 'removed class'),\n",
       "   ('irSIRS Model', 'be with', 'Spontaneous Loss of Immunity')],\n",
       "  [('interaction', 'be with', 'active user'),\n",
       "   ('they',\n",
       "    'can',\n",
       "    'can convinced to rejoin through interaction with active user')],\n",
       "  [('Schematic Representation', 'be with', 'Spontaneous Loss of Immunity'),\n",
       "   ('R → S transition', 'be', 'spontaneous')],\n",
       "  [('equilibrium', 'be', 'unstable')],\n",
       "  [('discriminant', 'be', 'negative')],\n",
       "  [('equilibrium', 'be', 'stable')],\n",
       "  [('system', 'for', 'unique stable equilibrium for given α')],\n",
       "  [('social phenomenon', 'have', 'catchiness')],\n",
       "  [('Stable Equilibria', 'be with', 'Spontaneous Loss of Immunity')],\n",
       "  [('α', 'be with', 'step size of 0.005'),\n",
       "   ('irSIRS Model', 'be with', 'Spontaneous Loss of Immunity'),\n",
       "   ('computed trajectories', 'be with', 'initial condition ='),\n",
       "   ('computed trajectories', 'be with', 'α between 0.000')],\n",
       "  [('We', 'reduce', 'Eq')],\n",
       "  [('system', 'be in', 'variables')],\n",
       "  [('irSIR Model', 'be with', 'Demography'),\n",
       "   ('new susceptible individuals', 'be spontaneously remove from', 'Fig. 10')],\n",
       "  [('use', 'be in', 'OSN models')],\n",
       "  [('registration', 'once require for', 'example')],\n",
       "  [('Schematic Representation', 'be with', 'Demography')],\n",
       "  [('Individuals', 'be in', 'classes')],\n",
       "  [('We', 'summarize', 'analysis of equilibria for system')],\n",
       "  [('I', 'edge of', 'simplex')],\n",
       "  [('It', 'follow from', 'Eq')],\n",
       "  [('existence condition', 'be in', 'Eq'),\n",
       "   ('We', 'also rewrite', 'existence condition in Eq')],\n",
       "  [('I', '−', 'αI − γ')],\n",
       "  [('we', 'rule out', 'β periodic orbits on interior of simplex')],\n",
       "  [('irSIR Model', 'be with', 'Demography'),\n",
       "   ('unique stable equilibrium point', 'at s be', 'blue'),\n",
       "   ('unique stable equilibrium point', 'at s be', 't')],\n",
       "  [('models', 'be in', 'mathematical epidemiology')],\n",
       "  [('Spechler', 'have', 'irSIR model')],\n",
       "  [('long-run stable equilibrium', 'be with', 'nonzero OSN membership')],\n",
       "  [('irSIRS model', 'be with', 'spontaneous return to infection')],\n",
       "  [('various problems', 'be in', 'population dynamics')],\n",
       "  [('coupled terms', 'be in', 'irSIR model')],\n",
       "  [('questions', 'be in', 'computer science'),\n",
       "   ('Exploration', 'be', 'interesting')],\n",
       "  [('Memes', 'be in', 'Digital Culture')],\n",
       "  [('pages 93 103', 'be in', 'Web Technologies')],\n",
       "  [('Epidemic spreading', 'be in', 'scale-free networks')],\n",
       "  [('NumericCitation Lu', 'bettencourt be', 'on-Arias'),\n",
       "   ('Ariel Cintr', 'on-arias be', 'David I Kaiser')],\n",
       "  [('page 8',\n",
       "    'be in',\n",
       "    'Proceedings of 7th Workshop on Social Network Mining')],\n",
       "  [('infectious diseases', 'be in', 'humans')],\n",
       "  [('Numerical simulation dynamical model',\n",
       "    'be with',\n",
       "    'lotka-volterra linear functional response')],\n",
       "  [('Applied', 'in program be', 'Princeton University'),\n",
       "   ('2 Program', 'be in', 'Applied')],\n",
       "  [('3 Media Lab', 'institute of', 'Technology')]],\n",
       " 93: [[('ring', 'be with', 'next nearest neighbour interactions')],\n",
       "  [('model', 'include', 'mechanism for renewal of susceptibles'),\n",
       "   ('spread', 'be in', 'community')],\n",
       "  [('Models', 'be with', 'immunity waning')],\n",
       "  [('δ', '=', '1')],\n",
       "  [('e-mail', 'know in', 'particular')],\n",
       "  [('inﬁnite', 'be in', 'case')],\n",
       "  [('linear lattices', 'on model be', '1')],\n",
       "  [('contrast', 'be with', 'mean ﬁeld approximation'),\n",
       "   ('simulations', 'be in', 'd ='),\n",
       "   ('simulations', 'be in', 'd =')],\n",
       "  [('lattices', 'be with', 'same coordination number k'),\n",
       "   ('next nearest neighbours', 'be', 'when considered')],\n",
       "  [('problem', 'be in', 'context of modiﬁcation of model')],\n",
       "  [('natural scenario', 'be in', 'epidemiology of diseases')],\n",
       "  [('limit µ =', 'in s. be', '2b'), ('limit µ =', 'in s. be', 'models')],\n",
       "  [('diﬀerent', 'be in', 'opposite limit'),\n",
       "   ('two models', 'dynamics of be', 'In opposite limit drastically diﬀerent')],\n",
       "  [('recovery processes', 'be identical in', 'models')],\n",
       "  [('linear lattice', 'be with', 'periodic boundary conditions'),\n",
       "   ('We', 'have consider', '2'),\n",
       "   ('linear lattice', 'on model be', '2')],\n",
       "  [('θ', 'measure of', 'relative contributions')],\n",
       "  [('lattices', 'be with', 'coordination number k =')],\n",
       "  [('correlated model', 'be with', 'best-ﬁt')],\n",
       "  [('closed form parameter θ', 'depend on', 'only demographic rate µ')],\n",
       "  [('dynamics', 'be govern by', '2')],\n",
       "  [('This', 'follow from', '2'), ('I', 'contact', 'pairs')],\n",
       "  [('tuple', 'ject to', 'normalization condition')],\n",
       "  [('mean-ﬁeld endemic infective probabilities',\n",
       "    'be',\n",
       "    'plotted in Figure 1 as function of infection rate at two values of µ')],\n",
       "  [('two regimes', 'be separate by', 'dashed line')],\n",
       "  [('UPA', 'show in', 'inset')],\n",
       "  [('MFA', 'be for', 'singletons')],\n",
       "  [('stability analysis',\n",
       "    'be',\n",
       "    'For demographic SIR model still tractable by symbolic computation')],\n",
       "  [('lattices', 'be with', 'coordination number k =')],\n",
       "  [('µ', 'tend to', '0')],\n",
       "  [('demographic SIR model',\n",
       "    'be in',\n",
       "    'mechanism for renewal of susceptibles')],\n",
       "  [('randomizing eﬀect',\n",
       "    'be in',\n",
       "    'small-world networks of Watts type NumericCitation')],\n",
       "  [('process', 'of upa be', '1'),\n",
       "   ('square lattice', 'on upa be', '1'),\n",
       "   ('same', 'be true for', '1')],\n",
       "  [('behaviour', 'identify in', 'stochastic simulations')],\n",
       "  [('rameters', 'leave to', 'right')],\n",
       "  [('phase diagrams', 'be show in', 'Figure 3')],\n",
       "  [('square lattice', 'on model be', '1'), ('CPA', 'ﬁtting θ to', '0')],\n",
       "  [('Science', 'for foundation be', 'FCT')],\n",
       "  [('R. M. Anderson', 'disease of', 'Humans')],\n",
       "  [('M. van Baalen', 'approximations for', 'Diﬀerent Spatial Geometries')]],\n",
       " 94: [[('time-optimal control problem', 'be in', 'SIR'),\n",
       "   ('We', 'focus on', 'diﬀerent control policies')],\n",
       "  [('Pontryagin', \"'s principle be\", 'PMP'),\n",
       "   ('Pontryagin', 'have', 'Minimum Principle')],\n",
       "  [('contrast',\n",
       "    'be with',\n",
       "    'previous ﬁndings on unconstrained problems minimizing over outbreak'),\n",
       "   ('optimal strategy', 'use', 'maximal control for entire epidemic')],\n",
       "  [('Recent epidemics', 'be in', 'South Korea'),\n",
       "   ('Recent epidemics', 'be in', 'West Africa')],\n",
       "  [('infectious diseases', 'be in', 'domestic animals')],\n",
       "  [('commerce', 'be in', 'infected areas')],\n",
       "  [('public health priority',\n",
       "    'minimize',\n",
       "    'time period needed for outbreaks eradication in aﬀected areas'),\n",
       "   ('outbreaks eradication', 'be in', 'aﬀected areas')],\n",
       "  [('UK NumericCitation',\n",
       "    'epidemic of',\n",
       "    'high pathogenic avian inﬂuenza in Midwestern USA NumericCitation'),\n",
       "   ('classical swine fever', 'be in', 'Netherlands NumericCitation'),\n",
       "   ('high pathogenic avian inﬂuenza',\n",
       "    'be in',\n",
       "    'Midwestern USA NumericCitation'),\n",
       "   ('Netherlands NumericCitation', 'epidemic of', 'foot-and-mouth'),\n",
       "   ('Netherlands NumericCitation', 'epidemic in', 'UK NumericCitation')],\n",
       "  [('infection events', 'be in', 'livestock')],\n",
       "  [('farms', 'be in', 'infected areas')],\n",
       "  [('export ban', 'be in', 'Hong Kong')],\n",
       "  [('classical swine fever', 'be in', 'Netherlands pig farms')],\n",
       "  [('proportional growth', 'be in', 'number of infected farms')],\n",
       "  [('longer epidemics', 'involve', 'larger number of animals slaughtered')],\n",
       "  [('several infections', 'be in', 'livestock')],\n",
       "  [('control function', 'be in', 'time-optimal framework'),\n",
       "   ('isolation strategies', 'be in', 'subsystem of model proposed')],\n",
       "  [('basic SIR', 'in individual be', 'burden'),\n",
       "   ('infected individuals', 'be in', 'basic SIR')],\n",
       "  [('its goal', 'prevent', 'development of diseases')],\n",
       "  [('Isolation', 'consist in', 'quarantine of infected individuals')],\n",
       "  [('individuals', 'become', 'infected')],\n",
       "  [('it', 'impose', 'limitations')],\n",
       "  [('infectious burden', 'be in', 'unconstrained conditions')],\n",
       "  [('optimal strategy',\n",
       "    'always rely in',\n",
       "    'adoption of maximum control for entire epidemic')],\n",
       "  [('maximum control',\n",
       "    'be',\n",
       "    'In context intended as implementation of control policy at its maximum available rate')],\n",
       "  [('epidemic duration', 'be in', 'SIR models')],\n",
       "  [('control', 'be', 'optimal')],\n",
       "  [('host population',\n",
       "    'be with',\n",
       "    'standard deterministic SIR model NumericCitation')],\n",
       "  [('linear term', 'be in', 'control variable u')],\n",
       "  [('control measures', 'be in', 'place NumericCitation'),\n",
       "   ('susceptible population',\n",
       "    'be with',\n",
       "    'control measures in place NumericCitation'),\n",
       "   ('individual',\n",
       "    'be in',\n",
       "    'completely susceptible population with control measures')],\n",
       "  [('eradication time', 'be in', 'problem')],\n",
       "  [('ε', 'be', 'positive'), ('ε', 'be', 'ﬁxed')],\n",
       "  [('f', 'with problem be', '8'),\n",
       "   ('Pontryagin', 'have', 'Minimum Principle NumericCitation'),\n",
       "   ('optimal control problem', 'be with', 'f')],\n",
       "  [('∗', 'u for', 't ∈'), ('t ∈', 'for u be', '0 T ∗')],\n",
       "  [('controls', 'apply at', 's')],\n",
       "  [('peak', 'be in', 'switching time correspondence of peak')],\n",
       "  [('we', 'will prove in', 'next sections')],\n",
       "  [('0 → umax', 'be with', 'single switching time τ ∗ s'),\n",
       "   ('0 → umax', 'umax for', 't ∈'),\n",
       "   ('single switching time τ ∗ s', 'with umax be', '= umax for t ∈'),\n",
       "   ('t ∈', 'for ∗ be', 't')],\n",
       "  [('depending position', 'be in', 'of addition')],\n",
       "  [('NumericCitation', 'regularization of', 'control function')],\n",
       "  [('identifying bang-bang function u ∈ A', 'be with', 'method')],\n",
       "  [('suitable interval', 'over j be', 'τ')],\n",
       "  [('eradication time Ti', 'relevant to', 'mesh point τi')],\n",
       "  [('we', 'investigate', 'four diﬀerent control policies considered'),\n",
       "   ('k', 'be in', 'following sections')],\n",
       "  [('we', 'set in', 'our numerical simulations'), ('ε', '=', '0.5')],\n",
       "  [('introduced infected individuals', 'be in', 'population')],\n",
       "  [('bang-bang control', 'be with', 'at most one switching time τ ∗'),\n",
       "   ('∗', 'be', 'bang-bang')],\n",
       "  [('vaccination control', 'be with', 'gv as')],\n",
       "  [('t', 'for ˙i be', 't')],\n",
       "  [('it', 'prove', 'following')],\n",
       "  [('color codes', 'be in', 'Fig. 1')],\n",
       "  [('optimal eradication time T ∗', 'be with', 'R0 =')],\n",
       "  [('= gray regions', 'be in', 'Fig. 2')],\n",
       "  [('Fig. 2', 'highlight by', 'analytic results')],\n",
       "  [('T ∗', 'vaccination as', 'functions of maximum eﬀort')],\n",
       "  [('abrupt change', 'be in', 'starting point of optimal changes'),\n",
       "   ('optimal changes', 'be in', 'umax')],\n",
       "  [('variation', 'be in', 'ob jective function'),\n",
       "   ('diﬀerences', 'be in', 'ﬁnal time of epidemic between optimal control')],\n",
       "  [('isolation control', 'with model be', '1'),\n",
       "   ('We', 'consider', '1'),\n",
       "   ('SIR model', 'be with', 'isolation control')],\n",
       "  [('dark gray region', 'be in', 'Fig. 3')],\n",
       "  [('functions', 'as time be', 'τ ∗')],\n",
       "  [('diﬀerences', 'be in', 'ﬁnal time of epidemic between optimal control')],\n",
       "  [('culling control', 'be in', 'optimal control problem deﬁned')],\n",
       "  [],\n",
       "  [('umax region', 'be in', 'Fig. 4'), ('Fig. 4', 'in region be', 'a')],\n",
       "  [('small region', 'be in', 'where starting of optimal strategy can occur'),\n",
       "   ('dark gray region', 'be in', 'box')],\n",
       "  [('functions', 'as time be', 'τ ∗')],\n",
       "  [('diﬀerences', 'be in', 'ﬁnal time between optimal control')],\n",
       "  [('We', 'consider', '1')],\n",
       "  [('strategy', 'be', 'bang-bang'),\n",
       "   ('bang-bang control', 'be with', 'at most one switching time τ ∗ problem')],\n",
       "  [('delayed control', 'be', 'when selected'),\n",
       "   ('control', 'be', 'when selected')],\n",
       "  [('dark gray region', 'be in', 'Fig. 5')],\n",
       "  [('time', 'time for', 'optimal reduction of transmission strategy')],\n",
       "  [('Color meanings', 'be', 'speciﬁed'),\n",
       "   ('Color meanings', 'be speciﬁed in', 'Fig')],\n",
       "  [('the optimal', 'of umax be', 'b')],\n",
       "  [('Other Parameter values', 'be in', 'as Fig. 2')],\n",
       "  [('diﬀerences', 'be in', 'ﬁnal time of epidemic between optimal control')],\n",
       "  [('susceptible individuals', 'of number be', 'umax to isolation problem'),\n",
       "   ('end', 'at number be', 'umax to isolation problem'),\n",
       "   ('S', 'epidemic for', 'optimal reduction of solid curve'),\n",
       "   ('number', 'umax to', 'isolation problem')],\n",
       "  [('Discussion', 'be in', 'work')],\n",
       "  [('may optimal strategy',\n",
       "    'be in',\n",
       "    'even unconstrained optimal control problem')],\n",
       "  [('Fig.', 'be in', 'Figs. 3 4')],\n",
       "  [('prompt intervention', 'reduce rc below', '1')],\n",
       "  [('time-optimal problem', 'be in', 'speciﬁc epidemic contexts'),\n",
       "   ('Our results',\n",
       "    'diﬀer from',\n",
       "    'those previously obtained for time-optimal problem in speciﬁc epidemic contexts')],\n",
       "  [('Pontryagin',\n",
       "    \"'s principle be\",\n",
       "    '15 maximizing for entire epidemic period'),\n",
       "   ('Pontryagin', 'accord to', 'Minimum Principle')],\n",
       "  [('NumericCitation', 'test', 'scenarios'),\n",
       "   ('NumericCitation', 'numerically test', 'scenarios')],\n",
       "  [('Pontryagin', 'accord to', 'Minimum Principle'),\n",
       "   ('diﬀerent works', 'characterize', 'optimal controls'),\n",
       "   ('diﬀerent works', 'characterize', 'controls')],\n",
       "  [('infectious burden', 'be in', 'optimal control problem')],\n",
       "  [('functions', 'as s be', 'T')],\n",
       "  [('small changes', 'be in', 'control parameter umax'),\n",
       "   ('control parameter umax', 'be in', 'Figs. 3')],\n",
       "  [('large changes', 'be in', 'shape of optimal strategies')],\n",
       "  [('SIR framework', 'be with', 'limited resources'),\n",
       "   ('isolation', 'be in', 'SIR framework with limited resources')],\n",
       "  [('abrupt variation', 'be in', 'ob jective function'),\n",
       "   ('catastrophic shift', 'be in', 'shape of isolation strategy')],\n",
       "  [('continuous variations', 'be in', 'ob jective functions'),\n",
       "   ('catastrophic shifts', 'be in', 'shape of control strategies')],\n",
       "  [('spread', 'be in', 'UK farms'),\n",
       "   ('NumericCitation', 'consider', 'diﬀerent control scenarios'),\n",
       "   ('NumericCitation', 'consider', 'control scenarios')],\n",
       "  [('vaccination', 'be', 'when delayed'), ('table 4', 'be in', '13')],\n",
       "  [('classical swine fever', 'be in', 'Switzerland pig farms')],\n",
       "  [('ﬁgure 4', 'be in', '14')],\n",
       "  [('optimal strategy', 'be in', 'epidemiological applications')],\n",
       "  [('host population growth', 'be', 'where density-dependent')],\n",
       "  [('aspects',\n",
       "    'deﬁning',\n",
       "    'optimal protocols of intervention for diseases eradication')],\n",
       "  [('we', 'thanks to', 'generality of model formulation')],\n",
       "  [('remarks', 'improve', 'manuscript')],\n",
       "  [('we', 'detail in', 'Section 2')],\n",
       "  [('it', 'be', 'continuous')],\n",
       "  [('continuous function', 'be in', '0'), ('J', 'be', 'continuous')],\n",
       "  [('controlled problem', 'of x be', 't')],\n",
       "  [('Ix', 'be', 'positive'),\n",
       "   ('Ix', 'be', 'continuous'),\n",
       "   ('it', 'hood of', 'T0')],\n",
       "  [('continuity', 'be in', '0'),\n",
       "   ('proof', 'be in', 'generic starting intervention time τ')],\n",
       "  [('Qk', 'be', 'invariant')],\n",
       "  [('t →', 'as i be', 't')],\n",
       "  [('stationary point', 'be', 'always zero')],\n",
       "  [('infected individuals', 'be in', 'of fact')],\n",
       "  [('control problem', 'be with', 'linear term policy'),\n",
       "   ('control term', 'value of', 'parameters')],\n",
       "  [('control',\n",
       "    'be non-singular',\n",
       "    'namely function ψ vanishes only in isolated points'),\n",
       "   ('function ψ', 'namely vanish in', 'only isolated points')],\n",
       "  [('ψ', 'vanish in', 'open interval B')],\n",
       "  [('contradiction', 'be with', 'Theorem 1')],\n",
       "  [('consequence', 'as ul be', 't'),\n",
       "   ('control', 'be', 'piecewise'),\n",
       "   ('control', 'be', 'constant')],\n",
       "  [('optimal control', 'be in', 'left-neighbourhood of eradication time T')],\n",
       "  [('τs', 'be', 'generic')],\n",
       "  [('Q', 'be', 'decreasing')],\n",
       "  [('τs', 'be', 'generic')],\n",
       "  [('only possible change', 'be in', 'sign of function ψ')],\n",
       "  [('control', 'switch from', '0 to maximum rate umax')],\n",
       "  [('it', 'must', 'must unique')],\n",
       "  [('there can switch', 'be in', 'case')],\n",
       "  [('minimum point mψ', 'be in', 'τs')],\n",
       "  [('extremal point', 'be in', 'interval'), ('it', 'be', 'minimum')],\n",
       "  [('function λS', 'be in', 'the')],\n",
       "  [('Optimal vaccination problem', 'be with', 'delayed control')],\n",
       "  [('interval', 'on λs be', 'λI'), ('interval', 'on λs be', 't')],\n",
       "  [('we', 'substitute value in', 'second derivative of λS'),\n",
       "   ('we', 'denote by', 'MλS')],\n",
       "  [('umax', 'be', 'necessary')],\n",
       "  [('optimal control', 'be in', 'conclusion')],\n",
       "  [('Pontryagin', 'by', 'Minimum Principle')],\n",
       "  [('contradiction', 'be with', 'statement of Theorem 1')],\n",
       "  [('ψ', 'can vanish in', 'only isolated points'),\n",
       "   ('ψ', 'therefore can vanish in', 'only isolated points')],\n",
       "  [('τs', 'be', 'generic')],\n",
       "  [('public health crisis', 'be in', 'Korea'),\n",
       "   ('NumericCitation K. Dong-Hyun', 'journal of', 'Preventive Medicine'),\n",
       "   ('NumericCitation K. Dong-Hyun',\n",
       "    'factor of',\n",
       "    'middle east respiratory syndrome coronavirus outbreak'),\n",
       "   ('NumericCitation K. Dong-Hyun',\n",
       "    'factor as',\n",
       "    'public health crisis in Korea')],\n",
       "  [('impact', 'be in', 'Sierra Leone')],\n",
       "  [('Economic cost', 'be in', 'UK'),\n",
       "   ('NumericCitation DEFRA/DCMS', 'cost in', 'UK'),\n",
       "   ('NumericCitation DEFRA/DCMS', 'cost of', 'foot')],\n",
       "  [('Department', 'of paper be', 'UK'), ('Environment', 'for paper be', 'UK')],\n",
       "  [('NumericCitation M. Hvistendahl', 'race across', 'U.S. Midwest')],\n",
       "  [('Horst', 'evaluation of', 'control')],\n",
       "  [('Hutber', 'prediction for', 'timing of culling'),\n",
       "   ('Research', 'be in', 'Veterinary Science 81 31 36'),\n",
       "   ('Hutber', 'prediction during', 'foot-and-mouth disease epidemic')],\n",
       "  [('Koopmans', 'transmission of', 'H7N7 avian inﬂuenza A virus'),\n",
       "   ('Koopmans',\n",
       "    'transmission during',\n",
       "    'large outbreak in commercial poultry farms in Netherlands'),\n",
       "   ('commercial poultry farms', 'be in', 'Netherlands'),\n",
       "   ('large outbreak', 'be in', 'commercial poultry farms'),\n",
       "   ('Koopmans', 'transmission to', 'human beings')],\n",
       "  [('expectations', 'be in', 'interplay between science'),\n",
       "   ('media', 'sociology of', 'Health 29 46 65')],\n",
       "  [('Otte', 'impact on', 'animal production in developing countries'),\n",
       "   ('animal production', 'be in', 'developing countries'),\n",
       "   ('Perspectives', 'be in', 'Agriculture'),\n",
       "   ('Otte', 'impact of', 'avian inﬂuenza virus')],\n",
       "  [('urr', 'evaluation of', 'control'),\n",
       "   ('urr', 'evaluation for', 'classical swine fever')],\n",
       "  [('bovine tuberculosis surveillance', 'be in', 'dairy farm network'),\n",
       "   ('Rossi',\n",
       "    'modelling for',\n",
       "    'assessment of bovine tuberculosis surveillance in dairy farm network in Emilia-Romagna'),\n",
       "   ('dairy farm network', 'be in', 'Emilia-Romagna')],\n",
       "  [('Longworth', 'analysis of', 'HPAI control in Netherlands I'),\n",
       "   ('HPAI control', 'be in', 'Netherlands I')],\n",
       "  [('NumericCitation C. Jiang', 'control base on', 'cybernetics'),\n",
       "   ('NumericCitation C. Jiang', 'control of', 'SARS epidemics'),\n",
       "   ('NumericCitation C. Jiang', 'journal of', 'Systems Science 38')],\n",
       "  [('SARS transmission patterns', 'be in', 'China'),\n",
       "   ('Zhang', 'model for', 'analysis of SARS transmission patterns in China')],\n",
       "  [('Advances', 'be in', 'Applied Probability 6 494 511'),\n",
       "   ('NumericCitation A. Abakuks', 'policy for', 'epidemics'),\n",
       "   ('NumericCitation A. Abakuks',\n",
       "    'advance in',\n",
       "    'Applied Probability 6 494 511')],\n",
       "  [('optimal control', 'on morton be', 'K. Wickwire'),\n",
       "   ('NumericCitation R. Morton',\n",
       "    'advance in',\n",
       "    'Applied Probability 6 622 635'),\n",
       "   ('Advances', 'be in', 'Applied Probability 6 622 635'),\n",
       "   ('optimal control',\n",
       "    'on morton be',\n",
       "    'Advances in Applied Probability 6 622 635')],\n",
       "  [('NumericCitation K. Wickwire', 'policy for', 'deterministic epidemics')],\n",
       "  [('NumericCitation H. Behncke', 'control of', 'deterministic epidemics')],\n",
       "  [('epidemics', 'be with', 'limited resources'),\n",
       "   ('NumericCitation E. Hansen', 'journal of', 'Mathematical Biology 62'),\n",
       "   ('NumericCitation E. Hansen',\n",
       "    'control of',\n",
       "    'epidemics with limited resources')],\n",
       "  [('infectious diseases', 'be in', 'wildlife'),\n",
       "   ('wildlife', 'journal of', 'Mathematical Biology 69')],\n",
       "  [('NumericCitation R. Anderson', 'biology of', 'infectious diseases')],\n",
       "  [('NumericCitation C. Y. Kaya',\n",
       "    'method for',\n",
       "    'time-optimal switching control'),\n",
       "   ('NumericCitation C. Y. Kaya', 'journal of', 'Optimization Theory')],\n",
       "  [('NumericCitation S. Lenhart', 'applied to', 'Biological Models')],\n",
       "  [('NumericCitation C. Silva',\n",
       "    'regularization of',\n",
       "    'bang-bang optimal control problems'),\n",
       "   ('NumericCitation C. Silva',\n",
       "    'transaction on',\n",
       "    'Automatic Control 55 2488 2499')],\n",
       "  [('NumericCitation M. Shamsi',\n",
       "    'scheme for',\n",
       "    'accurate solution of bang-bang optimal control problems')],\n",
       "  [('NumericCitation R. Thom', 'morphog en', 'ese')],\n",
       "  [('Fixed/free ﬁnal time SIR epidemic models',\n",
       "    'be with',\n",
       "    'multiple controls'),\n",
       "   ('Iacoviello', 'journal of', 'Simulation Modelling 7')],\n",
       "  [('multiple infectious disease outbreaks',\n",
       "    'proceedings of',\n",
       "    'Royal Society B 274 833 837')],\n",
       "  [('eﬀects', 'proceedings of', 'Royal Society B 278 1082 1089')],\n",
       "  [('severity', 'proceedings of', 'Royal Society B 273 2025 2034')],\n",
       "  [('infectious disease emergence', 'for bolzoni be', 'e747')],\n",
       "  [('NumericCitation A. Potapov',\n",
       "    'proceedings of',\n",
       "    'Royal Society B 279 3139 3145')],\n",
       "  [('classical swine fever', 'be in', 'wildlife'),\n",
       "   ('NumericCitation L. Bolzoni',\n",
       "    'analysis of',\n",
       "    'alternative culling strategies for eradication')]],\n",
       " 95: [[('extinction',\n",
       "    'be in',\n",
       "    'generalized susceptible-infected-susceptible model')],\n",
       "  [('problem',\n",
       "    'employ',\n",
       "    'Wentzel-Kramers-Brillouin approximation for master equation')],\n",
       "  [('bifurcation', 'be with', 'exponent 2 for m = for m ≥ 2')],\n",
       "  [('rare-event simulation method', 'validate', 'theory')],\n",
       "  [('ma jor challenges', 'be in', 'population dynamics NumericCitation')],\n",
       "  [('ﬂuctuation probability',\n",
       "    'be in',\n",
       "    'equilibrium systems determined by Boltzmann distribution'),\n",
       "   ('probability', 'be in', 'out-of-equilibrium system')],\n",
       "  [('Many eﬀorts', 'have', 'have done on topic')],\n",
       "  [('infective individuals', 'be in', 'ﬁnite populations'),\n",
       "   ('mathematical epidemic models', 'of time be', 'MET')],\n",
       "  [('them', 'calculate', 'classical action along tra jectory')],\n",
       "  [('population', 'be', 'well-mixed')],\n",
       "  [],\n",
       "  [('threshold', 'be with', 'exponent 2 for m = for m ≥ 2')],\n",
       "  [('we', 'validate', 'theoretical results')],\n",
       "  [('We', 'consider', 'variant of well-known SIS model')],\n",
       "  [('second one',\n",
       "    'refer to',\n",
       "    'recovery process of individual I with recovery rate µ'),\n",
       "   ('individual I', 'be with', 'recovery rate µ')],\n",
       "  [('we', 'set without', 'loss of generality'), ('µ', '=', '1')],\n",
       "  [('Bifurcation diagrams', 'be in', 'standard SIS model')],\n",
       "  [('We', 'proceed', 'our analysis from deterministic mean-ﬁeld theory')],\n",
       "  [('x', 'be', 'steady')],\n",
       "  [('model', 'show in', 'Fig. 1')],\n",
       "  [('xs', '=', '1')],\n",
       "  [('xs', '=', '0'), ('SISm model', 'be with', 'm ≥ 2')],\n",
       "  [('it', 'seek', 'nonvanishing solutions of x')],\n",
       "  [('locations', 'be in', 'y for x ∈ Firstly')],\n",
       "  [('two solutions', 'be with', 'quantity 1/λ')],\n",
       "  [('x − s', 'be', 'unstable'), ('x s', 'be', 'unstable')],\n",
       "  [('it',\n",
       "    'account for',\n",
       "    'process of epidemic extinction induced by large ﬂuctuations')],\n",
       "  [('infected individuals', 'number of be', 'n at time t')],\n",
       "  [('N', 'write', 'n')],\n",
       "  [('function', 'be in', 'As classical mechanics')],\n",
       "  [('S', 'be', 'minimized')],\n",
       "  [('zero-energy', 'be in', 'phase space')],\n",
       "  [('14 three lines',\n",
       "    'determine',\n",
       "    'topology of optimal extinction path on phase plane')],\n",
       "  [('probable tra jectory', 'be in', 'standard SIS model from A to C')],\n",
       "  [('critical infection rate η', 'be in', 'weak infection regime'),\n",
       "   ('17 V. SCALING', 'be in', 'following'),\n",
       "   ('17 V. SCALING', 'be in', 'WEAK INFECTION REGIME'),\n",
       "   ('F =', 'with s be', 'm ≥ 2'),\n",
       "   ('S', 'be with', 'F =')],\n",
       "  [('expansion x ± s', '=', 'xm ± ǫ')],\n",
       "  [('Gillespie', 'by', 'algorithm NumericCitation')],\n",
       "  [('Gillespie', 'of', 'algorithm')],\n",
       "  [('extinction', 'be', 'rare')],\n",
       "  [('conventional brute-force simulation',\n",
       "    'thus become',\n",
       "    'prohibitively ineﬃcient')],\n",
       "  [('we', 'overcome', 'diﬃculty'), ('Gillespie', 'with', 'algorithm')],\n",
       "  [('Nin', 'be', 'where number of interfaces')],\n",
       "  [('function', 'as i/n be', 'm =')],\n",
       "  [('n1', 'be', 'reached')],\n",
       "  [('QNin', 'be in', 'Fig. 4')],\n",
       "  [('action S', 'be', 'Since hT i ∼ eN S comparable to lnhT i/N')],\n",
       "  [('lines', 'be in', 'Fig. 4')],\n",
       "  [('lnhT i/N', 'of plot be', 'm ='), ('function', 'as plot be', 'm =')],\n",
       "  [('epidemic extinction process', 'be in', 'generalized SIS model'),\n",
       "   ('CONCLUSIONS', 'be in', 'conclusion')],\n",
       "  [('distance', 'η for', 'm =')],\n",
       "  [('we', 'validate', 'theoretical results')],\n",
       "  [('model', 'be', 'natural')],\n",
       "  [('selected person', 'take', 'opinion of group')],\n",
       "  [('present theoretical study',\n",
       "    'therefore could provide',\n",
       "    'useful understanding for large ﬂuctuation eﬀect on opinion dynamics')],\n",
       "  [('it',\n",
       "    'consider',\n",
       "    'eﬀect of interacting patterns among agents on extinction of generalized SIS model NumericCitation')]],\n",
       " 96: [[('I', '→', 'R → S'), ('I', '→', 'SIRS')],\n",
       "  [('individuals',\n",
       "    'be separate into',\n",
       "    'three classes determined by their relative states to given disease'),\n",
       "   ('we', 'use', 'stochastic lattice gas model')],\n",
       "  [('socalled susceptible-infected-recovered model NumericCitation',\n",
       "    'model for',\n",
       "    'epidemic'),\n",
       "   ('known models', 'be in', 'context')],\n",
       "  [('contact', 'be with', 'infected individuals')],\n",
       "  [('individuals', 'spontaneously become', 'immune')],\n",
       "  [('extension', 'be main concern of', 'paper')],\n",
       "  [('model', 'be', 'particular'),\n",
       "   ('SIR model', 'will', 'will then particular case of SIRS model')],\n",
       "  [('R', 'become also spontaneously', 'S'),\n",
       "   ('R', 'become', 'S'),\n",
       "   ('I', 'turn into', 'R'),\n",
       "   ('I', 'turn spontaneously into', 'R')],\n",
       "  [('lattice', 'be', 'where fully covered by susceptible')],\n",
       "  [('exponents', 'be obtain from', 'Monte Carlo simulations'),\n",
       "   ('critical exponents', 'be obtain from', 'Monte Carlo simulations')],\n",
       "  [('We',\n",
       "    'have also analyze',\n",
       "    'critical behavior of model occurring around transition from active to nonactive state')],\n",
       "  [('distinct class',\n",
       "    'class in',\n",
       "    'agreement with conjecture by Grassberger NumericCitation'),\n",
       "   ('agreement', 'be with', 'conjecture'),\n",
       "   ('dynamic percolation universality class',\n",
       "    'be in',\n",
       "    'agreement with conjecture by Grassberger NumericCitation')],\n",
       "  [('site', 'be', 'randomly chosen')],\n",
       "  [('S', 'be with', 'probability'),\n",
       "   ('it', 'r spontaneously with', 'probability c'),\n",
       "   ('it', 's spontaneously with', 'probability'),\n",
       "   ('it', 'r with', 'probability c'),\n",
       "   ('R', 'be with', 'probability c')],\n",
       "  [('time', 'be in', 'master equation')],\n",
       "  [('parameters', 'be in', 'way')],\n",
       "  [('phase diagram', 'be in', 'space of parameters'),\n",
       "   ('phase diagram', 'be show in', 'ﬁgure 1')],\n",
       "  [('Both',\n",
       "    'be in',\n",
       "    'agreement with ones of direct percolation universality class'),\n",
       "   ('agreement', 'be with', 'ones of direct percolation universality class')],\n",
       "  [('size scaling', 'propose in', 'reference NumericCitation')],\n",
       "  [('nonactive phase', 'be with', 'three types of individuals'),\n",
       "   ('Color online Phase diagram', 'be in', 'p-c space of parameters showing'),\n",
       "   ('nonactive phase', 'be with', 'only susceptible individuals')],\n",
       "  [('agreement',\n",
       "    'be with',\n",
       "    'ones for directed percolation universality class NumericCitation')],\n",
       "  [('agreement',\n",
       "    'be with',\n",
       "    'ones related to dynamic percolation universality class NumericCitation')],\n",
       "  [('I', '→', 'R'), ('I', '→', 'S')],\n",
       "  [('death process', 'reduce', 'number of degrees of freedom')],\n",
       "  [('description', 'be with', 'just two stochastic variables')],\n",
       "  [('birth', 'be in', 'space of susceptible numbers for SIRS model')],\n",
       "  [('full circle', 'represent', 'absorbing state')],\n",
       "  [('infected individuals', 'number of be', 'm')],\n",
       "  [('birth', 'be in', 'space of susceptible numbers for SIR model')],\n",
       "  [('full circles', 'absorb', 'states')],\n",
       "  [('N', 'be', 'population size')],\n",
       "  [('random walk', 'be in', 'space')],\n",
       "  [('probability A +0', 'with a be', 'm'),\n",
       "   ('probability A +0', 'with a be', 'b'),\n",
       "   ('→', 'be with', 'probability A0 −')],\n",
       "  [('individuals', 'oscillate in', 'time'), ('individuals', 'be in', 'class')],\n",
       "  [('period',\n",
       "    'be conﬁrmed by',\n",
       "    'Fourier analysis of time correlation functions')],\n",
       "  [('active phase', 'be see in', 'ﬁgure 5')],\n",
       "  [('sustained oscillations', 'be in', 'population of individuals')],\n",
       "  [('expansion', 'be in', 'powers of 1/N NumericCitation'),\n",
       "   ('population size N', 'be', 'where large')],\n",
       "  [('Up second order', 'be in', '1/N')],\n",
       "  [('white Gaussian noise functions', 'be with', 'zero mean obeying')],\n",
       "  [('entire population',\n",
       "    'be',\n",
       "    'where composed by susceptible individuals only')],\n",
       "  [('eigenvalues λ', 'correspond to', 'active state')],\n",
       "  [('solution', 'approach', 'asymptotic stable focus')],\n",
       "  [('solution', 'approach', 'asymptotic stable node')],\n",
       "  [('qualitative level', 'be in', 'phase diagram')],\n",
       "  [('we', 'use', 'PMF')],\n",
       "  [('individuals', 'be in', 'population class')],\n",
       "  [('stochastic lattice gas model',\n",
       "    'be in',\n",
       "    'scope of simple mean-ﬁeld approximation'),\n",
       "   ('process',\n",
       "    'be derive from',\n",
       "    'stochastic lattice gas model in scope of simple mean-ﬁeld approximation')],\n",
       "  [('individuals', 'be in', 'class'),\n",
       "   ('random walk', 'be in', 'space of number of susceptible individuals')],\n",
       "  [('region', 'be with', 'few individuals'),\n",
       "   ('contact', 'be with', 'other class of individuals')],\n",
       "  [('helpful discussions', 'be with', 'M. J. de Oliveira')],\n",
       "  [('Biological Populations', 'be in', 'Space')],\n",
       "  [('T. Tom', 'e be', 'J. Phys')],\n",
       "  [('Stochastic Processes', 'be in', 'Physics'),\n",
       "   ('NumericCitation N. G. van Kampen', 'processes in', 'Physics')],\n",
       "  [('NumericCitation C. W. Gardiner', 'handbook of', 'Stochastic Methods'),\n",
       "   ('Stochastic Methods', 'of handbook be', 'Citation')],\n",
       "  [('T. Tom', None, 'Phys')],\n",
       "  [('T. Tom', 'e be', 'J. Math')],\n",
       "  [('T. Tom', None, 'Int')]],\n",
       " 97: [[('Pieter Trapman2 University', 'be in', 'paper')],\n",
       "  [('individual', 'follow', 'mixed Poisson distributions')],\n",
       "  [('individual', 'of', 'infectious period')],\n",
       "  [('homogeneous mixing assumption', 'individual in', 'population contact')],\n",
       "  [('1 2 F. G. BALL', 'sirl of', 'model'), ('connection', 'be in', 'network')],\n",
       "  [('conﬁguration model NumericCitation', 'chapter of', 'NumericCitation')],\n",
       "  [('analysis', 'introduce', 'variable infectious periods')],\n",
       "  [],\n",
       "  [('typical individual', 'be discuss', 'brieﬂy in Section 6'),\n",
       "   ('brieﬂy', 'be in', 'Section 6')],\n",
       "  [('expressions', 'be in', 'terms of Gontcharoﬀ polynomials'),\n",
       "   ('oﬀspring distributions', 'of function be', 'als')],\n",
       "  [('edges', 'be with', 'only one endvertex')],\n",
       "  [('they', 'have', 'same direction')],\n",
       "  [('two directed edges', 'be with', 'same endvertices')],\n",
       "  [('edges', 'remove', 'self-loops'),\n",
       "   ('parallel edges', 'remove', 'self-loops')],\n",
       "  [('interpretation', 'be clear from', 'context')],\n",
       "  [('We', 'denote expectation with respect to', 'random variable X')],\n",
       "  [('We', 'use', 'k')],\n",
       "  [('nonnegative random variable X', 'of variant be', 'provided E X ∈')],\n",
       "  [('Xn ⇒ X', 'denote', 'convergence'),\n",
       "   ('Xn ⇒ X', 'denote convergence in', 'distribution')],\n",
       "  [('we', 'know by', 'Theorem 7.2.19 of NumericCitation')],\n",
       "  [('Reittu', 'have', 'Poissonian random graph model NumericCitation')],\n",
       "  [('Random intersection graphs', 'may', 'may thought')],\n",
       "  [('random undirected multigraphs', 'of sequence be', 'n ∈ N'),\n",
       "   ('random undirected multigraphs', 'of sequence be', 'ω')],\n",
       "  [('j', 'j ≤', '⌊ αn ⌋')],\n",
       "  [('Vertices', 'vi', '∈')],\n",
       "  [('that on j ∈ V ′ ω', 'number of', 'edges')],\n",
       "  [('P', 'be with', '2.4 B j = a.s. as n → ∞'),\n",
       "   ('2.4 B j =', 'with p be', 'B j')],\n",
       "  [('random variables',\n",
       "    'be with',\n",
       "    'empirical distribution of rescaled weights')],\n",
       "  [('n →', 'as number be', 'A ⇒ A')],\n",
       "  [('length 2', 'be in', 'A n'), ('edge', 'be in', 'G')],\n",
       "  [('We', 'note', 'at least one path'),\n",
       "   ('We', 'note', 'at least one path of graph')],\n",
       "  [('edge', 'be in', 'A n')],\n",
       "  [('weights', 'be constant in', 'NumericCitation')],\n",
       "  [('edge', 'be with', 'probability 1 −')],\n",
       "  [('machinery', 'branch', 'process approximations')],\n",
       "  [('vertices', 'be', 'Initially unexplored')],\n",
       "  [('end-vertices', 'be in', 'V ′ of edges'), ('edges', 'of ′ be', 'n')],\n",
       "  [('vertex vl', 'be', 'chosen'),\n",
       "   ('vertex vl', 'be choose with', 'probability Al L n')],\n",
       "  [('end vertex', 'be in', 'V')],\n",
       "  [('of it', 'to ′ be', 'n'), ('P A n', 'be in', 'V ′ of to it')],\n",
       "  [('′', 'be', 'chosen')],\n",
       "  [('chosen vertices', 'be in', 'V ′')],\n",
       "  [('contact', 'be in', 'G'),\n",
       "   ('contact', 'be with', 'each of its neighbours')],\n",
       "  [('infectious contact',\n",
       "    'be',\n",
       "    'contact by infectious individual irrespective of state of contacted individual')],\n",
       "  [('this', 'rescale', 'time')],\n",
       "  [('P', '=', '0')],\n",
       "  [('infectious contact', 'be with', 'member of clique')],\n",
       "  [('G', 'be', 'undirected')],\n",
       "  [('path', 'be in', 'epidemic generated graph')],\n",
       "  [('vertices', 'be in', 'susceptibility set of v0')],\n",
       "  [('branching process', 'be', 'multi-type')],\n",
       "  [('inﬁnite type branching process', 'be study separately in', 'Section 4')],\n",
       "  [('n', 'be', 'large')],\n",
       "  [('SIR epidemics', 'index by', 'population size n'),\n",
       "   ('SIR epidemics', 'prove', 'limit theorems'),\n",
       "   ('SIR epidemics', 'prove', 'associated limit theorems')],\n",
       "  [('theorems', 'prove in', 'Section 5')],\n",
       "  [('standard SIR epidemics',\n",
       "    'be in',\n",
       "    'closed homogeneously mixing populations')],\n",
       "  [('population size n', 'as limit be', 'initial infective')],\n",
       "  [('group', 'be in', 'single clique')],\n",
       "  [('litter', 'be', 'empty')],\n",
       "  [('local epidemics', 'be in', 'cliques')],\n",
       "  [('litters', 'be in', 'limit')],\n",
       "  [('proportional Individual j ∗', 'be in', 'construction of G')],\n",
       "  [('3.1 al l individuals', 'be in', '12 F. G. BALL'),\n",
       "   ('infectious individual',\n",
       "    'be in',\n",
       "    'ﬁrst stages of epidemic NumericCitation')],\n",
       "  [('distinct individuals', 'be in', 'Z f'),\n",
       "   ('oﬀspring', 'be mutually independent of', 'course')],\n",
       "  [('oﬀspring distribution', 'be in', 'generations')],\n",
       "  [('ρ', 'be for', 'x ∈')],\n",
       "  [('equation', 'be in', 'Appendix B. 2'),\n",
       "   ('Appendix B. 2', 'in equation be', 'B. 7')],\n",
       "  [],\n",
       "  [('m other individuals', 'be in', 'clique of size m')],\n",
       "  [('branching process',\n",
       "    'be in',\n",
       "    'much same way as early stages of epidemic')],\n",
       "  [('oﬀspring', 'be in', 'branching process')],\n",
       "  [('repeat', 'be in', 'ﬁrst generation of branching process obtain')],\n",
       "  [('i ∗', 'have', 'local susceptibility set')],\n",
       "  [('local susceptibility set', 'be in', 'contrast to Z f'),\n",
       "   ('b', 'be', 'branching'),\n",
       "   ('b', 'be', 'single-type')],\n",
       "  [('mean number', 'be in', 'Z b')],\n",
       "  [('connection', 'be with', 'computation')],\n",
       "  [('we', 'thus use', 'only notation R ∗')],\n",
       "  [('G ′', 'delete independently', 'directed edges'),\n",
       "   ('G ′', 'delete', 'directed edges')],\n",
       "  [('ρb = ρ', 'be in', 'case')],\n",
       "  [('susceptibility', 'be with', 'probability tending to 1 as n → ∞')],\n",
       "  [('j', 'have', 'susceptibility set'),\n",
       "   (\"j 's susceptibility\", 'set', 'process'),\n",
       "   ('j', 'have', 'susceptibility')],\n",
       "  [('j', 'have', 'susceptibility')],\n",
       "  [('individuals', 'be in', 'generation tn')],\n",
       "  [('j', 'of', 'partially constructed susceptibility set')],\n",
       "  [('it',\n",
       "    'intersect',\n",
       "    \"generation tn of j 's partially constructed susceptibility set\"),\n",
       "   ('it', 'intersect', 'generation tn'),\n",
       "   ('j', 'of', 'partially constructed susceptibility set')],\n",
       "  [('| R', 'number of', 'ultimately recovered individuals in epidemic'),\n",
       "   ('recovered individuals', 'be in', 'epidemic')],\n",
       "  [('its', 'size', 'total progeny'),\n",
       "   ('forancestor', 'include size be', 'total progeny')],\n",
       "  [('forward process', 'approximate', 'early stages of epidemic')],\n",
       "  [('following convergence',\n",
       "    'be in',\n",
       "    'distribution of proportion of individuals')],\n",
       "  [('random variable', 'be with', 'P TF = ='), ('TF', 'be', 'random')],\n",
       "  [('corresponding individual',\n",
       "    'be in',\n",
       "    'epidemic process E belongs to X ∼ MP'),\n",
       "   ('I.e.', 'individual in', 'epidemic process E belongs to X ∼ MP')],\n",
       "  [('individuals', 'be in', 'epidemic process')],\n",
       "  [('we', 'assume for', 'ease of exposition')],\n",
       "  [('individuals', 'be in', 'Z f function'), ('probability', 'be', 'given')],\n",
       "  [],\n",
       "  [('Z f', 'be', 'irreducible')],\n",
       "  [('ˇY', '=', 'k')],\n",
       "  [('Iε = ε ⌊ I ε ⌋', 'be with', 'convention')],\n",
       "  [('individual', 'of', 'own infectious period')],\n",
       "  [],\n",
       "  [('n →', 'as p be', 'X ∈ A')],\n",
       "  [('set', 'be in', 'SIR epidemic on random intersection graph G')],\n",
       "  [('Theorem 3.3', 'take expectation with respect to', 'measure ν')],\n",
       "  [('Theorem 3.4', 'follow similarly from', 'Theorem 5.2')],\n",
       "  [('R', 'of member be', 'n'),\n",
       "   ('exploration process', 'in r be', 'n'),\n",
       "   ('set', 'be in', 'E n'),\n",
       "   ('R 1 0', 'be in', 'exploration process'),\n",
       "   ('subset', 'be in', 'V')],\n",
       "  [('same meaning', 'be in', 'as Section 3.4')],\n",
       "  [('litter', 'be in', 'branching process')],\n",
       "  [('total number', 'be in', 'Z f')],\n",
       "  [('further results', 'also hold for', 'size-biased variants')],\n",
       "  [('typical individual', 'be in', 'Z')],\n",
       "  [('Hk',\n",
       "    'be total number of',\n",
       "    'litters spawned by ﬁrst k vertices evaluated in branching process Z f')],\n",
       "  [('Hk', 'be', 'proper'),\n",
       "   ('l → ∞', 'as p be', 'Hk l'),\n",
       "   ('Hk', 'be', 'random')],\n",
       "  [('total number', 'be in', 'ﬁrst k vertices evaluated in')],\n",
       "  [('l ∈ P', 'for → be', 'l')],\n",
       "  [('V', 'from v be', 'n')],\n",
       "  [('vertices', 'be', 'vertex chosen uniformly')],\n",
       "  [('processes', 'stay', 'coupled')],\n",
       "  [('litter sizes', 'be', 'all independent')],\n",
       "  [('vertices', 'be in', 'l ′')],\n",
       "  [('1 coupling', 'now proceed in', 'obvious way')],\n",
       "  [('total weight', 'be in', 'V')],\n",
       "  [('vertices', 'be in', 'V ′')],\n",
       "  [('number', 'be with', '0 ≤ i < j ≤ k'),\n",
       "   ('above restrictions', 'under v be', 'n')],\n",
       "  [('I', '=', '∞')],\n",
       "  [('k', 'be', 'random')],\n",
       "  [('pointwise nondecreasing', 'be in', 'k')],\n",
       "  [('˜ρ', 'for satisﬁes be', '4.1')],\n",
       "  [('same way', 'in ˆa be', 'n'),\n",
       "   ('ˆA', 'be in', 'same way'),\n",
       "   ('graph', 'be construct from', 'n')],\n",
       "  [('vertices', 'be in', 'A')],\n",
       "  [('We', 'add vertex from', 'n')],\n",
       "  [('We', 'choose', 'two distinct vertices')],\n",
       "  [('two vertices', 'be in', 'ˆG')],\n",
       "  [('individuals', 'be in', 'branching process Z b'),\n",
       "   ('ε 0', 'for o be', 'nε')],\n",
       "  [('tn', 'be in', 'coupling argument')],\n",
       "  [('individual', 'of', 'susceptibility'), ('we', 'see', 'start of Next')],\n",
       "  [('forward process', 'be in', 'G')],\n",
       "  [('We', 'use', 'following lemmas')],\n",
       "  [('we', 'have', 'k')],\n",
       "  [('number', 'be in', 'shortest path connecting'),\n",
       "   ('distance', 'be in', 'graph')],\n",
       "  [('their degrees', 'be in', 'ˆA')],\n",
       "  [('most n', 'be in', 'ˆA')],\n",
       "  [('set', 'be in', 'V within distance 2tn'),\n",
       "   ('distance 2tn', 'within v be', 'n'),\n",
       "   ('set', 'be in', 'V ′ within distance 2tn'),\n",
       "   ('distance 2tn', 'within ′ be', 'n')],\n",
       "  [('total number', 'be in', 'K 1')],\n",
       "  [('ν', 'for ω be', 'Pω')],\n",
       "  [('ˆA', 'number of', 'oﬀspring')],\n",
       "  [('total number', 'be in', 'generations'),\n",
       "   ('ε 0', 'for opν be', 'nε'),\n",
       "   ('Markov', 'have', 'inequality')],\n",
       "  [('we', 'prove', 'Lemma 5.8'), ('we', 'let', 'n')]],\n",
       " 98: [[('model', 'be in', 'sense')],\n",
       "  [('infection', 'be', 'where presents'),\n",
       "   ('particle', 'be in', 'infected'),\n",
       "   ('infection', 'be', 'presents')],\n",
       "  [('positions', 'be in', 'space')],\n",
       "  [('studied case', 'be in', 'mathematical sites of Zd lattice')],\n",
       "  [('They', 'also be infect via', 'contaminated locations')],\n",
       "  [('case',\n",
       "    'be in',\n",
       "    'particular for macroparasite diseases among domestic animal populations')],\n",
       "  [('We', 'other version of', 'it')],\n",
       "  [('infection', 'be in', 'population moving on Zd')],\n",
       "  [('x', 'mx of', 'healthy'),\n",
       "   ('particles', 'for x be', 'number Mx of healthy start with')],\n",
       "  [('it', 'be', 'where initially placed'), ('it', 'be', 'where placed')],\n",
       "  [('particles', 'start at', 'time 0'),\n",
       "   ('y', 'be with', 'home location'),\n",
       "   ('distributed time', 'be with', 'mean β − 1'),\n",
       "   ('chosen site', 'be in', 'Ny')],\n",
       "  [('β', '=', '1')],\n",
       "  [('Sλ', 'γ for', 'event')],\n",
       "  [('We', 'will hence assume', 'that')],\n",
       "  [('Monotonicity', 'be in', 'λ'),\n",
       "   ('≥ Proposition 1.1', 'monotonicity in', 'λ')],\n",
       "  [('psite c', 'here denote', 'critical value for site percolation')],\n",
       "  [('recovery rate', 'be', 'when large enough')],\n",
       "  [('infection', 'be transfer from', 'infected site to neighboring sites')],\n",
       "  [('This', 'explain', 'Theorem 1.1')],\n",
       "  [('This', 'explain', 'Theorem 1.2')],\n",
       "  [('strict diﬀerence', 'be in', 'sense'),\n",
       "   ('it', 'make', 'subcritical model supercritical')],\n",
       "  [('P', '=', '0')],\n",
       "  [('contact', 'be with', 'contaminated site')],\n",
       "  [('types', 'disease among', 'cows')],\n",
       "  [('Our ambition', 'be with', 'paper')],\n",
       "  [('model', 'pick', 'γ small')],\n",
       "  [('We', 'depend on', 'ﬁxed value of λ')],\n",
       "  [('λ', 'be', 'suﬃciently small'), ('we', 'understand', 'this')],\n",
       "  [('phase transition', 'be in', 'λ')],\n",
       "  [('Conjecture 1.1', 'transition in', 'λ'),\n",
       "   ('Phase transition', 'be in', 'λ')],\n",
       "  [('ones', 'be in', 'Corollary 1.1')],\n",
       "  [('Z2', 'be with', 'M =')],\n",
       "  [('K', '=', '100000')],\n",
       "  [('c c', 'be in', 'left-hand picture'),\n",
       "   ('S', 'make', 'model supercritical')],\n",
       "  [('non-trivial phase transition', 'be in', 'γ')],\n",
       "  [('γ', 'be', 'large')],\n",
       "  [('Conjecture 1.2', 'transition in', 'γ'),\n",
       "   ('Phase transition', 'be in', 'γ')],\n",
       "  [('simulations', 'be with', 'M ='),\n",
       "   ('square lattice',\n",
       "    'be with',\n",
       "    'L × L sites under periodic boundary conditions')],\n",
       "  [('process', 'be with', 'recovery rate λ0')],\n",
       "  [('process', 'survive with', 'positive probability for value of λ0')],\n",
       "  [],\n",
       "  [('c m', 'be in', 'c'), ('changes', 'be in', 'settings')],\n",
       "  [('comparison', 'be with', 'site percolation process remains')],\n",
       "  [('small same lines', 'be in', 'as proof of Theorem 1.1')],\n",
       "  [('dependent percolation process',\n",
       "    'rely on',\n",
       "    'direct infections between particles')],\n",
       "  [('Theorem 1.1', 'be', 'true also')],\n",
       "  [('phase transition', 'be in', 'λ'),\n",
       "   ('model', 'have', 'phase transition in λ')],\n",
       "  [('our particles', 'move in', 'only bounded region')],\n",
       "  [('model', 'be with', 'unrestricted movements')],\n",
       "  [('model', 'be with', 'k =')],\n",
       "  [('particles', 'be remove from', 'system')],\n",
       "  [('6 2 Proofs', 'be in', 'section')],\n",
       "  [('processes', 'be with', 'diﬀerent parameter values')],\n",
       "  [('s', 'for γ be', 't'), ('t. Sλ', 'γ for', 's')],\n",
       "  [('Poisson process', 'be with', 'rate λ'),\n",
       "   ('epidemic process', 'be with', 'rate γ1'),\n",
       "   ('process', 'be with', 'rate γ0'),\n",
       "   ('process', 'be with', 'site clearance rate γ1'),\n",
       "   ('Poisson process', 'be with', 'rate γ1'),\n",
       "   ('processes', 'be with', 'parameter values'),\n",
       "   ('sites', 'be in', 'epidemic process with rate γ1')],\n",
       "  [('epidemic process', 'be with', 'rate γ0')],\n",
       "  [('process', 'be with', 'rate γ1'), ('process', 'be with', 'rate γ0')],\n",
       "  [('second claim', 'be in', 'Proposition 1.1')],\n",
       "  [('inﬁnite survival',\n",
       "    'have',\n",
       "    'positive probability starting from more sparse conﬁguration')],\n",
       "  [('particle ξz', 'be with', 'home location'),\n",
       "   ('we', 'shall refer to', 'to center site')],\n",
       "  [('one particle', 'be with', 'z')],\n",
       "  [('white dots', 'represent', 'corner sites of accessible regions'),\n",
       "   ('particles', 'have', 'their home locations'),\n",
       "   ('dots', 'represent', 'corner sites of accessible regions')],\n",
       "  [('region', 'be', 'when activated')],\n",
       "  [('Rz', 'be', 'open')],\n",
       "  [('it', 'be', 'bounded'), ('it', 'be', 'uniformly bounded')],\n",
       "  [('particle ξz', 'visit', 'zact')],\n",
       "  [('ξz', 'be position at', 'zact'), ('ξz', 'once be position at', 'zact')],\n",
       "  [('γ', 'let', 'Bλ'), ('zact', 'be', 'cleared')],\n",
       "  [('P >', 'be in', 'epidemic process'),\n",
       "   ('site percolation process', 'be in', 'independent k')],\n",
       "  [('initial conﬁguration', 'be with', 'fewer particles')],\n",
       "  [('small λ', 'be with', 'γ'),\n",
       "   ('process', 'be supercritical also for', 'suﬃciently small λ with γ'),\n",
       "   ('it', 'mention in', 'Section 1')],\n",
       "  [('case', 'lead to', 'dependencies between regions'),\n",
       "   ('particles', 'be in', 'neighboring regions')],\n",
       "  [('estimates', 'be in', 'such comparison'),\n",
       "   ('Percolation processes', 'be with', 'ﬁnite dependencies')],\n",
       "  [('given site', 'be in', 'Zd')],\n",
       "  [('infected particle recovers',\n",
       "    'be clear at',\n",
       "    'rate γ independently of everything else')],\n",
       "  [('process', 'will', 'will referred to as maximal load epidemic')],\n",
       "  [('maximal possible number', 'be in', '9 epidemic')],\n",
       "  [('neighboring site', 'be in', 'epidemic process')],\n",
       "  [('two processes', 'be in', 'such way')],\n",
       "  [('load epidemic', 'be', 'follows')],\n",
       "  [('Px', 'family of', 'independent Poisson processes'),\n",
       "   ('one Poisson process', 'be with', 'rate 1 ¯ mvk particles at x'),\n",
       "   ('one Poisson process',\n",
       "    'be with',\n",
       "    'rate 1 particles at neighboring sites of x'),\n",
       "   ('one Poisson process', 'be with', 'rate λ')],\n",
       "  [('its particles', 'assign', 'Poisson family Px'),\n",
       "   ('its particles', 't to', 'x'),\n",
       "   ('We', 'all now deﬁne', 'local infection process')],\n",
       "  [('processes', 'be in', 'Px')],\n",
       "  [('set Sx', 't of', 'infectious signals sent from x up until time')],\n",
       "  [('origin', 'receive', 'infectious independent of P0 ,0 infectious signals'),\n",
       "   ('new local', 'be', 'signal'),\n",
       "   ('local', 'be', 'signal')],\n",
       "  [('parallel', 'be with', 'new local process')],\n",
       "  [('given time', 'be in', 'maximal load epidemic')],\n",
       "  [('infectious signals', 'be in', 'modiﬁed process')],\n",
       "  [('elements', 'be in', 'S0 ,0')],\n",
       "  [('individual', 'be in', 'branching process'),\n",
       "   ('individuals',\n",
       "    'hence generate',\n",
       "    'independent identically distributed oﬀspring')],\n",
       "  [('It', 'be', 'infected'),\n",
       "   ('It', 'be', 'infected'),\n",
       "   ('this', 'be', 'infected'),\n",
       "   ('this', 'be', 'infected')],\n",
       "  [],\n",
       "  [],\n",
       "  [('reinforcement signal', 'be', 'received')],\n",
       "  [('reinforcement signal ber', 'clearly dominate', 'number of Furthermore')],\n",
       "  [('it', 'be', 'cleared')],\n",
       "  [('signals', 'estimate as', 'above'),\n",
       "   ('infectious signals', 'estimate as', 'above')],\n",
       "  [('FL', 'be grateful to', 'Proyecto de Postdoctorado N. 3160163')],\n",
       "  [('Springer Lecture Notes', 'be in', 'Statistics 151'),\n",
       "   ('Stochastic epidemic models', 'notes in', 'Statistics 151')],\n",
       "  [('recovery', 'be in', 'D'), ('recovery', 'with epidemic be', 'Ann')],\n",
       "  [('Lecture Notes', 'be in', 'Math')],\n",
       "  [('his', 'birthday', '141-170'),\n",
       "   ('Festschrift', 'be in', 'Honor of Ted Harris')],\n",
       "  [('Branching random walk', 'be with', 'catalysts')],\n",
       "  [('rumor', 'of spread be', 'Ann'), ('spread', 'be in', 'moving population')],\n",
       "  [('phase transition',\n",
       "    'be in',\n",
       "    'model for spread of epidemic Il linois J. Math')]],\n",
       " 99: [[('I', 'β', 'I S N γ I'), ('˙S ˙I ˙R', '=', '− β I S N − γ')],\n",
       "  [('I', '˙r', 'R')],\n",
       "  [('8 remaining task', 'calculate', 'exponential')],\n",
       "  [('proof', 'be', 'simple')],\n",
       "  [('y', 'set', 'x')],\n",
       "  [('us', 'summarize', 'our result')],\n",
       "  [('ei', 'r with', '9'), ('R =', 'be with', '9'), ('9', 'with r be', 't')],\n",
       "  [('We', 'conclude note with', 'one comment')],\n",
       "  [('Our', 'approximation', '4')],\n",
       "  [('This', 'be', 'challenging')],\n",
       "  [('Mathematical Sciences', 'of box be', 'Citation')],\n",
       "  [('book', 'be translate into', 'English')]]}"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "'''The Stanford CoreNLP runs in Java. However, the 'pycorenlp' is a python\n",
    "wrapper to the CoreNLP program.  To use it, you need to download the\n",
    "Stanford coreNLP program library on to your computer, plus an installation\n",
    "of Java 8+ to run the program. To download go here:\n",
    "https://stanfordnlp.github.io/CoreNLP/. Before you can call the CoreNLP\n",
    "program from 'pycorenlp', you need to have the Stanford CoreNLP server\n",
    "running. To do that navigate to the downloaded CoreNLP folder on your\n",
    "terminal and run the following command:\n",
    "\n",
    "java -mx4g -cp '*' edu.stanford.nlp.pipeline.StanfordCoreNLPServer -port 9000 -timeout 500000\n",
    "\n",
    "Once you've done that, you should be ready to go! If you run into problems\n",
    "running the CoreNLP server, see the following webpage\n",
    "(https://stanfordnlp.github.io/CoreNLP/corenlp-server.html#getting-started)'''\n",
    "\n",
    "rdf_obj.extract_raw_tuples()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Load in previosly run rdf_obj (the above process takes a couple hours)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pickle\n",
    "rdf_obj = pickle.load(open('data/rdf_obj_temp.pickle','rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Preprocessing raw relation triples\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n",
      "/Users/taylor_bolt/PycharmProjects/multivac/venv/lib/python3.7/site-packages/scipy/sparse/base.py:582: RuntimeWarning: divide by zero encountered in true_divide\n",
      "  return self.astype(np.float_)._mul_scalar(1./other)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Clustering entities from relation triples\n",
      "\n",
      "8272 entity clusters were found\n",
      "\n",
      "Saving final tuples to .txt files for input to OpenKE\n",
      "['3.3 strategy 3', 'in', 'strategy']\n",
      "['control', 'with', 'strategy 2']\n",
      "['control', 'with', 'strategy 2']\n",
      "['control', 'with', 'strategy 2']\n",
      "['optimal control', 'with', 'strategy 3']\n",
      "['pλ', 'n by', 'lemma 4.1']\n",
      "['pv', 'since δpv', '˜x t']\n",
      "['ebov', 'in', 'guinea']\n",
      "['strategy 3', 'more precisely consist in', 'study of educational campaign about virus coupled with vaccination treatment']\n",
      "['ρd', 'increase function of', 'pd']\n",
      "['campaign', 'with', 'strategy 1']\n",
      "['individuals', 'with', 'strategy']\n",
      "['50 days', 'in', 'strategy 1']\n",
      "['pλ', 'ω for', 'ω ∈ ωd']\n",
      "['pλ', 'ω with', 'measure νω × π']\n",
      "['8 a. rachah', 'e without', 'control strategy 1 strategy']\n",
      "['one node', 'in', 'tu']\n",
      "['d', 'note', 'pλ']\n",
      "['our goal', 'in', 'strategy']\n",
      "['3.1 strategy 1', 'in', 'subsection']\n",
      "['strategy 3', 'of j', 'u']\n",
      "['12 a. rachah', 'e without', 'control strategy 1 strategy']\n",
      "['vr', 'equal to', 'infection eccentricity of node vr']\n",
      "['guinea', 'ci for', 'liberia']\n",
      "['sars', 'emergence of', 'h1n1 swine inﬂuenza']\n",
      "['guinea', 'ci for', 'sierra leone']\n",
      "['wsl', 'have', 'has cornerstone of las vegas outdoor water conservation strategy']\n",
      "['pd', 'for λd', 'r0']\n",
      "['pd', 'for λd', 'r0']\n",
      "['lemma 4.6', 'by m4', 'log d']\n",
      "['citation ebov outbreaks', 'in', 'guinea']\n",
      "['southern nevada water authority', \"'s program\", 'wsl']\n",
      "['s', '→', '0 of ˜p ∞ s']\n",
      "['r0', 'statement regard', 'monotonicity of ρd in pd']\n",
      "['vr', 'be', 'information source']\n",
      "['ρd', 'increase with', 'increasing infectious contact rate λd from symptomatic cases']\n",
      "['snwa', 'have', 'wsl']\n",
      "['wsl', 'have', 'has always completely voluntary incentive based program']\n",
      "['participation', 'in', 'wsl program']\n",
      "['participation', 'in', 'wsl program']\n",
      "['participation', 'in', 'wsl program']\n",
      "['r0', 'for pd', 'r0']\n",
      "['guinea', 'for interval', 'ci']\n",
      "['homes', 'in', 'wsl population']\n",
      "['outbreak', 'in', 'guinea']\n",
      "['the optimal', 'of umax', 'b']\n",
      "['logistic regressions', 'in', 'wsl']\n",
      "['wsl program', 'in', 'las vegas']\n",
      "['ρd', 'in', 'section 4']\n",
      "['simulated epidemic', 'for behaviour', 'the scaling']\n",
      "['infection path y tu', 'with', 'source node u']\n",
      "['case', 'cure of', 'ebola']\n",
      "['populations', 'of class', 'the infected']\n",
      "['ps', 'randomly choose from', 'pi']\n",
      "['wsl rate', 'be', 'eventual wsl participation rate']\n",
      "['lim lim d →', 'ρ for', 'λ ≥ 1 eρ']\n",
      "['optimal control problems', 'with', 'help of acado']\n",
      "['vr', 'infect u with', 'probability q']\n",
      "['node', 'eccentricity of', 'maximum distance from node to infected nodes']\n",
      "['2,3,4 basic reproduction number', 'in', 'sierra leone']\n",
      "['node', 'with', 'minimum infection eccentricity']\n",
      "['node', 'with', 'minimum infection eccentricity']\n",
      "['node', 'with', 'minimum infection eccentricity']\n",
      "['r0', 'number for', 'newly infected symptomatic individuals']\n",
      "['subsequent outbreak', 'in', 'sierra leone']\n",
      "['quantity ρd', 'give', 'indication of importance of symptomatic cases in transmission process in epidemic']\n",
      "['primary case', 'in', 'guinea']\n",
      "['participation', 'in', 'vegas']\n",
      "['node', 'with', 'smaller infection eccentricity']\n",
      "['automatic control', 'for framework', 'optimal control appl']\n",
      "['noise', 'use', 'eqs']\n",
      "['xeric landscapes', 'one of', 'the longest']\n",
      "['ebola', 'with', 'wolfram guage']\n",
      "['control effor ts', 'in', 'liberia']\n",
      "['reported data', 'in', 'guinea']\n",
      "['dynamics', 'with', 'control']\n",
      "['reported cases', 'in', 'guinea']\n",
      "['optimal control', 'with', 'treatment']\n",
      "['node u', 'infect at', 'time u']\n",
      "['node', 'with', 'minimum eccentricity in graph']\n",
      "['ted data', 'in', 'guinea']\n",
      "['symptomatic cases', 'cause', 'most of infections in norovirus scenario']\n",
      "['ebov outbreak', 'in', 'congo']\n",
      "['sn', 'in', 'j − 1']\n",
      "['sn', 'in', 'j']\n",
      "['sn', 'in', 'j']\n",
      "['sn', 'in', 'y 2 0']\n",
      "['sn', 'in', '− y 2 2']\n",
      "['recession', 'inﬂuenced', 'wsl participation']\n",
      "['vr', 'infect', 'u']\n",
      "['suﬃciently small time', 'during analysis', 'probabilities of s']\n",
      "['large d', 'for 1', 'log d']\n",
      "['node', 'of', 'infection']\n",
      "['nodes', 'with', 'minimum infection eccentricity']\n",
      "['homes', 'in', 'neighborhood']\n",
      "['monotonicity', 'in', 'pd']\n",
      "['monotonicity', 'in', 'pd']\n",
      "['changes', 'in', 'wsl participation']\n",
      "['⊆ γ1', 'for d', 'λθ']\n",
      "['d2 n', 'since y', 'γ2']\n",
      "['node', 'with', 'maximum closeness centrality']\n",
      "['nodes', 'in', 'network']\n",
      "['nodes', 'with', 'maximum infection closeness']\n",
      "['nodes', 'in', 'network']\n",
      "['norovirus', 'in', 'symptomatic individuals']\n",
      "['nodes', 'often refer to', 'to spreaders']\n",
      "[\"household 's wsl conversion\", 'have on', 'neighboring homes']\n",
      "['wsl participation rate', 'in', 'neighborhood']\n",
      "['pd', 'for λd', 'd']\n",
      "['infected individuals', 'in', 'of fact']\n",
      "['deterministic center manifold', 'with', 'replacement of noise terms']\n",
      "['deterministic center manifold', 'with', 'replacement of noise terms']\n",
      "['deterministic center manifold', 'with', 'replacement of noise terms']\n",
      "['deterministic center manifold', 'with', 'replacement of noise terms']\n",
      "['our model', 'in analysis', 'probabilities of s']\n",
      "['bd process', 'with', 'prefactor of 1/2']\n",
      "['vacant homes', 'in', 'las vegas']\n",
      "['peer eﬀect', 'fade in', 'time']\n",
      "['section', 'illustrate', 'role of ρd']\n",
      "['household', 'have', 'wsl conversion']\n",
      "['n', 'denote', 'number of nodes']\n",
      "['transmission rate', 'in', 'absence of control interventions']\n",
      "['nodes', 'in', 'network']\n",
      "['pi', 'pr for', 'node v']\n",
      "['pi', 'pr for', 'node v']\n",
      "['time', 'at step', 'node infected']\n",
      "['transmissibility', 'preserve', 'average period of homogeneously mixed population']\n",
      "['transmission rate', 'decay exponentially due to', 'introduction of control measures']\n",
      "['homes', 'require by', 'snwa']\n",
      "['neighborhood 14', 'α with', 'only 9 wsl']\n",
      "['numericcitation', 'eﬀects of', 'spatial correlations']\n",
      "['stochastic change', 'in', 'number of infectives']\n",
      "['jordan infection', 'center as', 'possible candidates of information source']\n",
      "['distribution', 'in', 'limit of n → ∞']\n",
      "['single viral strain', 'represent', 'cluster of co-circulating antigenically close strains']\n",
      "['infected node', 'in', 'network']\n",
      "['noise terms', 'of replacement', 'eqs']\n",
      "['noise terms', 'of replacement', 'eqs']\n",
      "['noise terms', 'of replacement', 'eqs']\n",
      "['noise terms', 'of replacement', 'eqs']\n",
      "['infection', 'in', 'si model']\n",
      "['infection', 'in', 'si model']\n",
      "['optimal sample path', 'node of', 'node with minimum infection eccentricity']\n",
      "['infection closeness centrality', 'be', 'inverse of sum of distances to infected nodes']\n",
      "['depending position', 'in', 'of addition']\n",
      "['ps', 'be', 'randomly chosen']\n",
      "['one state', 'in', 'model']\n",
      "['probability distribunumbers', 'in', 'of tion p']\n",
      "['seir simulations', 'series ×', 'infectious period of pertussis']\n",
      "['homes', 'with', 'single conversions']\n",
      "['source', 'with', 'limited observations of set of infected nodes']\n",
      "['dashed line', 'represent', 'value of t ∗']\n",
      "['symptomatic cases', 'thus meet', 'fewer individuals']\n",
      "['las vegas', 'reduce', 'water consumption']\n",
      "['scale-free networks', 'on zhou', 'b. h. behaviors of susceptible-infected epidemics']\n",
      "['1 λ', 'in', 'terms of λ ±']\n",
      "['homes', 'with', 'multiple conversions']\n",
      "['numericcitation', 'require', 'only unbiased estimate of likelihood']\n",
      "['homes', 'in', 'buﬀer zone']\n",
      "['homes', 'in', 'buﬀer zone']\n",
      "['citation outbreak', 'in', 'west africa']\n",
      "['time', 'in', 'units of number of deaths']\n",
      "['step', 'with', 'of probability µ']\n",
      "['infection source', 'with', 'infection starting at n − 1']\n",
      "['infection source', 'with', 'infection starting at time 1']\n",
      "['infection source', 'with', 'infection starting at time 1']\n",
      "['infection source', 'with', 'infection starting at time 1']\n",
      "['infection source', 'with', 'infection starting at time 1']\n",
      "['infection source', 'with', 'infection starting at time 1']\n",
      "['non-participating house', 'have', 'probability of adoption']\n",
      "['stochastic eﬀects', 'dominate', 'behaviour of time series']\n",
      "['one home', 'have', 'wsl adoption behavior']\n",
      "['numericcitation', 'restrict to', 'set of infected nodes']\n",
      "['stochastic change', 'in', 'number of susceptibles']\n",
      "['activity distribution', 'in', 'community of s']\n",
      "['seir model', 'with', 'symptomatic cases']\n",
      "['∀ t ≥', 't for', 'λ 1 eρ']\n",
      "['epidemic starts', 'with', 'symptomatic outbreak']\n",
      "['of time', 'in rate', 'k2 ∼ poisson of infections']\n",
      "['individuals', 'with', 'given combination of k']\n",
      "['las vegas', 'reduce', 'outdoor water consumption']\n",
      "['diﬀuses', 'with', 'probability 1 of γ']\n",
      "['ebola virus', 'cause', 'acute often fatal haemorrhagic illness']\n",
      "['nodes', 'with', 'minimum observed infection eccentricity']\n",
      "['infected individual', 'become', 'symptomatic case']\n",
      "['numericcitation', 'study', 'malaria transmission in india']\n",
      "['of it', 'to ′', 'n']\n",
      "['numericcitation a. aharony', 'introduction to', 'percolation theory']\n",
      "['numericcitation', 'with', 'potential of infection']\n",
      "['peer eﬀect', 'be', 'present']\n",
      "['reverse infection algorithm', 'base on', 'structure properties of optimal sample paths on trees']\n",
      "['proportional individual j ∗', 'in', 'construction of g']\n",
      "['single infected node', 'in', 'community of size s']\n",
      "['τ', 'use', 'hki']\n",
      "['stochastic normal form coordinate', 'transform', 'eqs']\n",
      "['stochastic normal form coordinate', 'transform', 'eqs']\n",
      "['stochastic normal form coordinate', 'transform', 'eqs']\n",
      "['stochastic normal form coordinate', 'transform', 'eqs']\n",
      "['nodes', 'in', 'state s']\n",
      "['many neighborhoods', 'in', 'las vegas']\n",
      "['susceptible node', 'in', 'community of size s']\n",
      "['equations', 'with', 'replacement of noise terms']\n",
      "['stochastic terms', 'in', 'eqs']\n",
      "['velocity spread', 'in', 'scale-free networks']\n",
      "['control measures', 'be', 'citation respectively']\n",
      "['based conservation program', 'in', 'city of las vegas']\n",
      "['source', 'in', \"numericcitation 's model\"]\n",
      "['asymptomatic cases', 'for e', 'la']\n",
      "['b. vaccination-dominant model', 'in', 'variation of model']\n",
      "['optimal sample path', 'source of', 'jordan infection center']\n",
      "['end-vertices', 'in', 'v ′ of edges']\n",
      "['stochastic dynamics', 'in', 'complete system of transformed variables']\n",
      "['p a n', 'in', 'v ′ of to it']\n",
      "['4 it', 'know', 'reproduction numbers of current ebov outbreak']\n",
      "['household participation', 'in', 'las vegas water smart landscapes program']\n",
      "['ﬁeld model', 'in', 'closed population of n individuals']\n",
      "['persistence transition', 'occur at', 'edge of small world regime']\n",
      "['numericcitation', 'in', 'seir model deterministic recovery']\n",
      "['4.2 vaccination-dominant model', 'in', 'variation of model']\n",
      "['r0', 'for pd', 'd']\n",
      "['incidence oscillations', 'with', 'average period of two years']\n",
      "['16 selected neighborhoods', 'in', 'las vegas']\n",
      "['conﬁguration model numericcitation', 'chapter of', 'numericcitation']\n",
      "['spatial correlations', 'greatly enhance', 'stochastic ﬂuctuations']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['seir model', 'of equation', 'eqs']\n",
      "['abrupt change', 'in', 'levels of infection']\n",
      "['activity class', 'in', 'communities of size s']\n",
      "['two infection paths', 'of pv', 'x tv']\n",
      "['large-scale networks', 'in detection', 'complete snapshot of real world network']\n",
      "['sir model', 'in', 'system composed of two overlapping layers']\n",
      "['adoption environment', 'in', 'las vegas']\n",
      "['α', 'with', 'only 9 wsl']\n",
      "['straight line', 'in', 'phase space of sub-population sizes']\n",
      "['model', 'with', 'two strains']\n",
      "['numericcitation', 'study', 'association between lagged predictors from particular water body to cholera cases in thana']\n",
      "['site', 'in', 'state r']\n",
      "['normal form coordinate', 'transform', 'eqs']\n",
      "['zd', 'with', 'infection rate λ']\n",
      "['population birth rate', '0.27 for', 'recovery rate of diseases']\n",
      "['measles', 'in', 'homogeneously mixed populations']\n",
      "['population', 'with', 'ﬁxed value of c22']\n",
      "['y ∼ 1', 'in', 'region of ridge']\n",
      "['may ideal framework', 'in', 'view of this']\n",
      "['optimal control', 'in', 'left-neighbourhood of eradication time t']\n",
      "['one model', 'dominate', 'behavior of system']\n",
      "['curves', 'in', 'log-linear diagrams of figs 8']\n",
      "['optimal single infection source estimator', 'be', 'jordan center of vi in section iii']\n",
      "['lemma 4.2', 'will', 'where will utilized']\n",
      "['lemma', 'will', 'where will utilized']\n",
      "['zd', 'with', 'length n']\n",
      "['zd', 'with', 'length n']\n",
      "['signiﬁcant diﬀerences', 'in', 'characteristics of homes']\n",
      "['reproduction number', 'in', 'west africa']\n",
      "['reproduction number', 'in', 'west africa']\n",
      "['symptomatic individual', 'i.e cause', 'more infections']\n",
      "['figure 4', 'present', 'time-dependent curve of exposed individuals']\n",
      "['disease dependent period', 'set', 'time scale of model']\n",
      "['vaccination treatment', 'in', 'practical ebola situations']\n",
      "['numericcitation', 'give', 'result announced in eq']\n",
      "['model', 'have', 'phase transition in λ']\n",
      "['neighborhoods', 'in', 'model response to τd']\n",
      "['expression', 'in', 'terms of θ']\n",
      "['numericcitation', 'with', 'corresponding new infected individuals in clay center given best estimated values for three contact levels']\n",
      "['place', 'remove', 'nodes']\n",
      "['node', 'in', 'target of arrow']\n",
      "['optimal sample path', 'in', 'trees']\n",
      "['large extinction-level population collapse', 'in', 'one of species']\n",
      "['quantum formulation', 'in', 'terms of spin operators']\n",
      "['infectious contact', 'with', 'member of clique']\n",
      "['numericcitation k. dong-hyun', 'factor as', 'public health crisis in korea']\n",
      "['numericcitation', 'use', 'eﬀect of intermittent social distancing strategy on propagation of epidemics in adaptive complex networks']\n",
      "['qualitative two-wave process', 'illustrate in', 'fig 2 of section 4']\n",
      "['taylor expansion remainder term', 'in', 'analogue of']\n",
      "['all nodes', 'in', 'network']\n",
      "['infection', 'use', 'above mentioned spreading model']\n",
      "['single infection source', 'in', 'si model']\n",
      "['single infection source', 'in', 'si model']\n",
      "['numericcitation', 'in', 'limit of distribution of epidemic sizes obtained from stochastic sir model large population sizes']\n",
      "['limit', 'correspond to', 'saturation of reservoir of species 2']\n",
      "['neighborhood', 'in', 'context of home foreclosures']\n",
      "['control measures', 'be', 'sufﬁ citation respectively']\n",
      "['following convergence', 'in', 'distribution of proportion of individuals']\n",
      "['our perturbation method', 'explicitly track', 'dynamics of probability distribution of sub-populations in vicinity of cl']\n",
      "['proposed sir model', 'with', 'reinfection']\n",
      "['last equation', 'follow from', 'simply conservation of total number of individuals']\n",
      "['numericcitation l. g. alvarez-zuzek', 'in', 'journal of physics conference series']\n",
      "['individuals', 'represent by', 'nodes of graph']\n",
      "['model', 'include', 'mechanism for renewal of susceptibles']\n",
      "['participation', 'in', 'incentive']\n",
      "['secondary cases', 'in', 'beginning of epidemic']\n",
      "['model', 'inspire by', 'modeling of spread of childhood diseases']\n",
      "['identiﬁcation', 'permit by', 'lemma 4.1']\n",
      "['site', 'belong to', 'cluster of size s']\n",
      "['error bars', 'smaller than', 'size of symbols']\n",
      "['stationary time', 'decrease', 'value of α']\n",
      "['figure 7', 'show detection rate as', 'function of g']\n",
      "['two processes', 'deﬁning', 'dynamics of system']\n",
      "[\"one home 's wsl adoption behavior\", 'inﬂuencing', 'their neighbors participation probabilities']\n",
      "['3.0 ghz', 'with', '4 gb of ram']\n",
      "['d1/3 vertices', 'in', 'γ1']\n",
      "['pr', 'be', 'none-increasing']\n",
      "['our method', 'give', 'explicit description of dynamics of probability distribution of sub-populations in vicinity of cl']\n",
      "['m other individuals', 'in', 'clique of size m']\n",
      "['infected node', 'become', '1the jordan center of infected node set']\n",
      "['jordan infection center', 'with', 'largest infection closeness']\n",
      "['infection source', 'compare result with', 'benchmarks']\n",
      "['d', 'proof of', 'lemma 4.5']\n",
      "['borel-cantelli', 'accord to', 'lemma']\n",
      "['underlying small world network', 'of coeﬃcient', 'citation']\n",
      "['fractional order', 'on model', 'adv diﬀerence equ']\n",
      "['minority phase', 'in', 'process of ostwald ripening numericcitation']\n",
      "['high shell-number nodes', 'in', 'network']\n",
      "['dependent variable', 'be', 'where log of odds of wsl participation']\n",
      "['mathematics', 'in research', 'department of mathematics']\n",
      "['pertussis', 'in', 'pre-vaccination era']\n",
      "['x ∗', 'inﬁnitesimal as', 'function of δ']\n",
      "['of function', 'as r', 't']\n",
      "['estimator', 'with', 'knowledge of infection rates']\n",
      "['infection path x 2', 'with', 'elapsed time 2']\n",
      "['infection path y 3', 'with', 'elapsed time 3']\n",
      "['signiﬁcant diﬀerence', 'in', 'number of susceptible']\n",
      "['mathematics', 'in research', 'university of aveiro']\n",
      "['pi', 'pr from', 'numericcitation']\n",
      "['pi', 'pr from', 'numericcitation']\n",
      "['joint probability', 'β for', 'derivation of eq']\n",
      "['rate µ', 'in', 'sis model']\n",
      "['reduction', 'in', 'number of cases']\n",
      "['reduction', 'in', 'number of cases']\n",
      "['distance', 'for level', 'neighborhoods n ∈ 1 6 8 9']\n",
      "['infection', 'at', 'where endemic equilibrium in reservoir']\n",
      "['individuals', 'in', 'compartments of sir model']\n",
      "['individuals', 'in', 'compartments of sir model']\n",
      "['e', 'denote', 'set of edges']\n",
      "['infector', 'study r0 for', 'beginning of epidemic']\n",
      "['collapse ratio', 'in', 'population of species 1']\n",
      "['such ﬁne scale data', 'collect on', 'timescales of outbreaks']\n",
      "['asymptomatic case', 'in', 'latent stage']\n",
      "['result', 'agree with', 'conclusion of ref']\n",
      "['model', 'determine by', 'dynamics of underlying primary infection']\n",
      "['citation ebola virus outbreak', 'in', 'west africa']\n",
      "['citation ebola virus outbreak', 'in', 'west africa']\n",
      "['node', 'infect', 'probability of vaccination changes']\n",
      "['stochastic model', 'with', 'value of v']\n",
      "['time', 'at vs', 'θt']\n",
      "['sublinear ﬁt', 'may', 'may result of importance of de-phasing eﬀect']\n",
      "['network', 'with', 'average degree k']\n",
      "['degree distributions', 'in', 'of fig 4b']\n",
      "[\"one home 's wsl adoption behavior\", 'inﬂuencing', 'their neighbors eventual participation probabilities']\n",
      "['total number', 'in', 'of range of numericcitation']\n",
      "['measles', 'of oscillation', 'figure 4c']\n",
      "['evolution equations', 'of system', 'eqs']\n",
      "['infection', 'with', 'given probability']\n",
      "['numericcitation', 'would allow for', 'such extensions']\n",
      "['large diﬀerence', 'in', 'terms of rates']\n",
      "['epidemic contribution', 'in', 'dynamics of adoption']\n",
      "['ﬁnal epidemic size', 'reduce', 'value of γ']\n",
      "['given m', 'for a', 'ct']\n",
      "['likely infection path', 'in', 'tree network with single infection source']\n",
      "['e. d. kilbourne', 'epidemiology of', 'inﬂuenza b']\n",
      "['rules', 'equivalent to', 'deﬁnition of model given in beginning of section']\n",
      "['reinfection threshold', 'be', 'at level of mean ﬁeld approximation boundary between two diﬀerent regimes']\n",
      "['only possible change', 'in', 'sign of function ψ']\n",
      "['information source', 'in', 'general networks']\n",
      "['its goal', 'prevent', 'development of diseases']\n",
      "['stochastic realization', 'with', 'population turnover in phase space of i1']\n",
      "['nodes', 'be', 'in case uniformly and independently distributed on unit rectangle of given side lengths']\n",
      "['k', 'be', 'separation of vertex set of system']\n",
      "['β', 'invert', 'eq']\n",
      "['infection rate', 'proportional to', 'size of ful l population of infectious individuals']\n",
      "['isolation', 'consist in', 'quarantine of infected individuals']\n",
      "['numericcitation', 'study', 'malaria transmission']\n",
      "['clustering mechanism', 'update schema in', 'nep']\n",
      "['infection range', 'non-increasing over', 'iterations of mjc']\n",
      "['figure 1', 'show', 'phase plane of system']\n",
      "['node', 'in', 'network']\n",
      "['basic diﬀerential equations', 'in', 'terms of densities']\n",
      "['node', 'get', 'new connection to another']\n",
      "['unemployment rate', 'in', 'las vegas-henderson-paradise']\n",
      "['citation ebola outbreak', 'in', 'west africa']\n",
      "['citation ebola outbreak', 'in', 'west africa']\n",
      "['numericcitation', 'with', 'choices dt']\n",
      "['infectious individual', 'in', 'ﬁrst stages of epidemic numericcitation']\n",
      "['susceptible node', 'successfully contact by', 'one of its infected neighbors']\n",
      "['information', 'spread in', 'network']\n",
      "['mathematical model', 'with', 'considerations of diﬀerent classes of individuals']\n",
      "['same susceptible population', 'in', 'framework of sir model']\n",
      "['node strength', 'size on', 'order of 10 − 6']\n",
      "['problem', 'in', 'context of modiﬁcation of model']\n",
      "['infection', 'use', 'above parameters']\n",
      "['4.4 proof', 'in', 'subsection']\n",
      "['hutber', 'prediction for', 'timing of culling']\n",
      "['optimal control problem', 'minimize', 'u']\n",
      "['si model', 'with', 'single source']\n",
      "['b. source associated', 'with', 'most likely infection path in subsection']\n",
      "['jordan center', 'also optimal for', 'sis model in regular tree networks']\n",
      "['seir model', 'with', 'τr ∈']\n",
      "['relative ﬂuctuations', 'lead to', 'citation']\n",
      "['ebola virus transmission', 'in', 'montserrado']\n",
      "['si model', 'suppose', 'infection process']\n",
      "['system', 'converge asymptotically to', 'all-healthy state']\n",
      "['multiple infection sources', 'in', 'si model']\n",
      "['infective time series', 'exhibit incidence oscillation with', 'average period of two years']\n",
      "['complete system', 'in', 'transformed variables']\n",
      "['infected node', 'in', 'time']\n",
      "['node', 'be', 'then reinfected again for multiple times by observation time']\n",
      "['sample paths', 'in', 'space of absolutely continuous measures']\n",
      "['virus', 'about campaign', 'section 3.3']\n",
      "['1 sources', 'in', 'estimator']\n",
      "['recovery model', 'with', 'varying rates of loss of immunity']\n",
      "['immunization', 'improve', 'threshold of epidemic eradication']\n",
      "['infected nodes', 'in', 'network']\n",
      "['numericcitation j. tang', 'in', 'proceedings of ieee 12th international symposium on world of wireless']\n",
      "['numericcitation w. o. kermack', 'in', 'proceedings of royal society of london a']\n",
      "['separation', 'disappear for', 'larger values of τ']\n",
      "['model', 'correspond to', 'well-studied household-workplace model']\n",
      "['numericcitation numericcitation', 'have receive', 'limited study']\n",
      "['single infection source', 'in', 'sir model']\n",
      "['related model', 'in', 'context of epidemic dynamics']\n",
      "['population birth rate', '− for', 'transmissibility']\n",
      "['agreement', 'with', 'time of most recent common ancestor of genome analysis of ebov outbreak']\n",
      "['source', 'with', 'mild technical assumptions']\n",
      "['numericcitation', 'be', 'adequate']\n",
      "['our model', 'hence reproduce', 'several key features of original sir model']\n",
      "['configuration model', 'with', 'of points']\n",
      "['stochastic s i1 i2s model', 'for s', 'i1 → i1 of transition s → s − 1']\n",
      "['infected nodes', 'hρi', 'i. order of one-thousandth of ρr ﬂuctuation in sir model']\n",
      "['epidemic spreading', 'in', 'networks']\n",
      "['sir model', 'with', 'recovery rate b b ′ in region where']\n",
      "['importance', 'in', 'dynamics of childhood diseases']\n",
      "['numericcitation m. shamsi', 'scheme for', 'accurate solution of bang-bang optimal control problems']\n",
      "['whole multi-dimensional probability distribution', 'in', 'vicinity of cl']\n",
      "['observed infection topology y', 'contain', 'at least one infected node']\n",
      "['node', 'in', 'core of network 7']\n",
      "['noise', 'with', 'intensity σ']\n",
      "['susceptible half-edges', 'of vs', 'θt']\n",
      "['two homes', 'in', 'given neighborhood']\n",
      "['immune population', 'structure by', 'level of immunity']\n",
      "['numericcitation numericcitation', 'have receive', 'study']\n",
      "['coeﬃcients', 'thereby solve', 'eq']\n",
      "['propagation', 'in', 'plots/ﬁelds of same area']\n",
      "['vertical black line', 'derive in', 'max value of r ∞']\n",
      "['population turnover', 'in', 'phase space of i1']\n",
      "['vaccine control', 'in', 'poultry']\n",
      "['normal form coordinate', 'transform method to', 'correctly pro ject noise onto center manifold']\n",
      "['nodes', 'with', 'same degree exhibit']\n",
      "['measles', 'in', 'birmingham']\n",
      "['w. o. kermack', 'contribution to', 'mathematical theory of epidemics']\n",
      "['g', 'vary β from', '0.1 to 0.9']\n",
      "['g', 'β from', '0.1 to 0.9']\n",
      "['d7 20', 'in', 'region of 1 − ξ0 2a']\n",
      "['i', '∞', 'numericcitation']\n",
      "['single family homes', 'in', 'las vegas valley water district service area']\n",
      "['model', '2 on', 'same network']\n",
      "['threshold', 'in', 'limit as n → ∞']\n",
      "['contradiction', 'with', 'statement of theorem 1']\n",
      "['hazard model', 'in', 'as towe']\n",
      "['source', 'with', 'limited observations']\n",
      "['deterministic problem', 'for system', 'eqs']\n",
      "['red arrows', 'indicate', 'direction of branches of infection']\n",
      "['γ n', 'in', 'powers of z']\n",
      "['remaining i − k', 'in', 'set of n − m']\n",
      "['∈ v', 'be', 'infection source']\n",
      "['logarithm', 'in', 'each of eqs']\n",
      "['seir model', 'with', 'parameters estimated in analysis of recent h1n1 inﬂuenza numericcitation']\n",
      "['connected graph', 'with', 'n nodes']\n",
      "['participating household', 'in', 'las vegas valley water district service area']\n",
      "['contrast', 'with', '6.6 at end of campaign']\n",
      "['modiﬁed sir model', 'be', 'introduced in sec 2']\n",
      "['pertussis', 'with', 'p']\n",
      "['4 a. rachah', 'model for', 'non-constant population']\n",
      "['comparison system', 'in', 'proof of nonnegativity']\n",
      "['numericcitation', 'consider', 'diﬀerent control scenarios']\n",
      "['growth rate', 'in', 'eq']\n",
      "['time t', 'in', 'realization of stochastic']\n",
      "['phase transition', 'in', 'model for spread of epidemic il linois j. math']\n",
      "['γ continual decrease', 'in', 'number of susceptibles s']\n",
      "['0 ρ', 'in', 'second order of ε']\n",
      "['0 ρ', 'in', 'second order of ε']\n",
      "['model', 'with', 'zero birth rate numericcitation']\n",
      "['1 ρ', 'in', 'ﬁrst order of ε']\n",
      "['numericcitation k. dong-hyun', 'factor of', 'middle east respiratory syndrome coronavirus outbreak']\n",
      "['asymptomatic individual', 'be', 'infectious at age since infection τ']\n",
      "['algorithm can output', 'with', 'high probability independent of number of infected nodes']\n",
      "['noise', 'be', 'whereby pro']\n",
      "['we will couch', 'in', 'terms of changes']\n",
      "['ﬁrst term', 'in', 'numerator of eq']\n",
      "['speciﬁc form', 'in', 'eqs']\n",
      "['information source', 'with', 'smallest tv']\n",
      "['power law', 'with', 'ﬁxed exponent of 2/3']\n",
      "['following expression', 'in', 'terms of principal']\n",
      "['signiﬁcant spatial clustering', 'in', 'physical characteristics of homes']\n",
      "['dynamics', 'suﬃciently describe by', 'eqs']\n",
      "['6 a. rachah', 'citation by', 'althaus']\n",
      "['adaptive models', 'with', 'sis']\n",
      "['its infection range', 'be', 'node in g with minimum infection range']\n",
      "['average degree', 'hki from', 'eq']\n",
      "['numericcitation', 'regularization of', 'control function']\n",
      "['epidemic starts', 'with', 'small number of infectious individuals']\n",
      "['node', 'in', 'ˆs l − 1']\n",
      "['epidemic spreading models', 'explain', 'propagation of diseases']\n",
      "['grenfell', 'in', 'measles epidemics']\n",
      "['crumpmodejagers', 'process', 'numericcitation']\n",
      "['program', 'have', 'future adoption']\n",
      "['nodes', 'be', 'where distributed in rgg']\n",
      "['measles', 'in', 'england']\n",
      "['node', 'in', 'network with smallest maximum distance']\n",
      "['systems', 'with', 'reinfection']\n",
      "['our eﬀort', 'understand', 'evolution of cooperative behavior']\n",
      "['α', 'with', 'step size of 0.005']\n",
      "['exciting chaos', 'with', 'noise']\n",
      "['sub-populations', 'in', 'vicinity of cl']\n",
      "['sub-populations', 'in', 'vicinity of cl']\n",
      "['sub-populations', 'in', 'vicinity of cl']\n",
      "['sub-populations', 'in', 'vicinity of cl']\n",
      "['balance', 'in', 'derivation of eq']\n",
      "['1 2 f. g. ball', 'sirl of', 'model']\n",
      "['maximum susceptible degree', 'in', 'population of size n']\n",
      "['numericcitation', 'consider', 'control scenarios']\n",
      "['grenfell', 'determinant of', 'measles metapopulation dynamics in england']\n",
      "['infected node set', 'center of', 'optimal single source estimator']\n",
      "['second term', 'correspond to', 'term of eq']\n",
      "['infectious contact', 'be', 'contact by infectious individual irrespective of state of contacted individual']\n",
      "['estimator', 'in', 'accordance to si model']\n",
      "['continuous-time reed-frost models', 'with', 'deterministic latent periods']\n",
      "['clark county tax assessors records', 'in', 'wsl conservation program records']\n",
      "['network', 'use', 'adjacency list']\n",
      "['other households school part', 'in', 'of school']\n",
      "['numericcitation numericcitation', 'in', 'most cases']\n",
      "['recovered nodes', 'in', 'ba networks of size n']\n",
      "['tree network', 'with', 'single infection source']\n",
      "['exclusion principle', 'eigenvalue of', 'particle operators']\n",
      "['τd neighborhood distance metric aic', 'α no', 'st eα τd wsl rate']\n",
      "['τd neighborhood distance metric aic', 'α', 'st eα τd wsl rate']\n",
      "['d independent exponential times', 'with', 'rate λθ 2d']\n",
      "['information source detection', 'in', 'sir model']\n",
      "['infected node', 'in', 'early stages of epidemic outbreak']\n",
      "['cauchy', 'have', 'construction of m representing']\n",
      "['available outbreak data', 'obtain', 'estimates for parameters of model numericcitation']\n",
      "['i site', 'have', 'four nearest-neighbor sites']\n",
      "['inﬁnitesimal rates', 'in', 'of time']\n",
      "['hopf bifurcation', 'with', 'noise']\n",
      "['distributed latent periods', 'with', 'constant latent periods']\n",
      "['incidence oscillations', 'in', 'real data for similar population sizes numericcitation']\n",
      "['result', 'provide', 'additional support of network model']\n",
      "['distribution', 'start with', 'smallest possible jump of size two']\n",
      "['our model', 'overestimate', 'size of outbreak']\n",
      "['multiplicative hazard model', 'in', 'as towe']\n",
      "['jordan center', 'in', 'corresponding super node graph ˜g']\n",
      "['578 numericcitation bhattacharya s', 'fr since', 'recovery model with varying rates of loss of immunity']\n",
      "['information source detection problem', 'can', 'similar to si model can formalized as mle problem']\n",
      "['reverse infection algorithm', 'outperform signiﬁcantly', 'random guesses']\n",
      "['epidemic outbreak', 'in', 'sis model numericcitation']\n",
      "['network', 'have', 'clustering coeﬃcient']\n",
      "['sir model', 'with', 'transmissibility β for ﬁrst infection']\n",
      "['siri model', 'in', 'lemma a 1']\n",
      "['probability pi', 'with', 'probability pr']\n",
      "['rapid drop', 'in', 'popularity of facebook']\n",
      "['normalization constant probability distribution p xt', 'x0 in', 'of our analysis']\n",
      "['s1 collapse', 'in', 'case of uni-directional transmission']\n",
      "['sir model', 'in', 'multilayer networks numericcitation']\n",
      "['κ hk2i/hki', 'branch factor of', 'degree distribution of network']\n",
      "['other words', 'in kµ', 'sum of φ p β of latently infected individuals']\n",
      "['model', 'ﬁxed as', 'total population size in si1 i2s model']\n",
      "['σiφi terms', 'in', 'eqs']\n",
      "['system', 'with', 'certain spectral requirements']\n",
      "['mathematics', 'in center', 'university of aveiro']\n",
      "['community', 'large ing from', 'rest of network']\n",
      "['ct t ≥ 0', 'with', 'state space']\n",
      "['eﬀective transmission rate', 'number per', 'time step divided by product of instantaneous densities of susceptibles']\n",
      "['infection', 'with', 'virus']\n",
      "['its existence', 'depend on', 'topology of network']\n",
      "['equilibrium', 'with', 'current value of n/s']\n",
      "['network epidemic models', 'with', 'two levels of mixing']\n",
      "['block', 'in', 'class of activity']\n",
      "['network', 'with', 'additional long-range connections']\n",
      "['distribution', 'in', 'number of recovered individuals']\n",
      "['r m spatial heterogeneity', 'in', 'epidemic models']\n",
      "['numericcitation k. dong-hyun', 'journal of', 'preventive medicine']\n",
      "['model', 'with', 'epidemic eﬀect']\n",
      "['increasing functions', 'on r0', 'λd']\n",
      "['ni', 'set of', 'neighbors of node i']\n",
      "['infectious periods', 'give rise to', 'markov models']\n",
      "['generating function g', 'in', 'powers of z']\n",
      "['rate', 'in', 'population due to susceptible-infected contact']\n",
      "['node', 'with', 'incoming edge from u']\n",
      "['limit', 'in', 'number of vaccines']\n",
      "['numericcitation', 'with', '3 − 5 days most common numericcitation']\n",
      "['same susceptible population', 'in', 'framework of sis model numericcitation']\n",
      "['new variables', 'substitute into', 'eqs']\n",
      "['term', 'contribute to', 'increase of φi in eq']\n",
      "['research', 'for center', 'university of aveiro']\n",
      "['mathematics', 'in center', 'department of mathematics']\n",
      "['τr', 'in', 'months']\n",
      "['ach node', 'in', 'hv']\n",
      "['distribution', 'in', 'presence of rewiring']\n",
      "['proof', 'utilize', 'following proposition given in numericcitation']\n",
      "['two strains', 'in', 'context of sis model']\n",
      "['information source', 'give x', '0 t']\n",
      "['method', 'also describe', 'whole multi-dimensional probability distribution of system in vicinity of cl']\n",
      "['nodes', 'with', 'large number of links']\n",
      "['number', 'in', 'network']\n",
      "['hmf', 'result', 'c of adjacent matrix']\n",
      "['numericcitation', 'in', 'eq']\n",
      "['λθ', 'rate of', 'x']\n",
      "['constant distance', 'with', 'high probability independent of number of infected nodes']\n",
      "['constant distance', 'with', 'high probability independent of number of infected nodes']\n",
      "['bifurcation point', 'near fluctuation', 'number of infectives']\n",
      "['positive fraction', 'in', 'population at beginning of epidemic']\n",
      "['infectious homes', 'require by', 'snwa']\n",
      "['research', 'for center', 'department of mathematics']\n",
      "['markov stochastic dynamics', 'in', 'discrete state space of sub-population sizes']\n",
      "['ratio β/µ', 'drive', 'spreading of disease']\n",
      "['similar analysis', 'with', 'that of equation']\n",
      "['similar analysis', 'with', 'that of equation']\n",
      "['node', 'in', 'target of arrow']\n",
      "['void node', 'remove due to', 'occupancy of link']\n",
      "['sis model', 'with', 'homogeneous infection']\n",
      "['time t', 'with', 'very small numbers of infected hosts in one']\n",
      "['set estimator', 'in', 'tree network for si model']\n",
      "['infection sources', 'compare result with', 'benchmarks']\n",
      "['its ultimate steady state population', 'in', 'endemic state of diseases']\n",
      "['agreement', 'with', 'those of isotropic percolation']\n",
      "['node', 'on', 'degree']\n",
      "['weak convergence', 'in', 'topology of d']\n",
      "['fraction', 'of series', 'eqs']\n",
      "['standard ﬁnite-size scaling analysis', 'obtain', 'critical behavior of model']\n",
      "['measles', 'for series', 'p']\n",
      "['unique root', 'in', '0 1 of ∞ xk']\n",
      "['i site', 'in', 'sis model']\n",
      "['periods', 'give rise to', 'markov models']\n",
      "['cutoff value rc', 'correspond to', 'ﬁrst peak of ∆']\n",
      "['node', 'with', 'maximum pagerank centrality']\n",
      "['multiple infection sources', 'in', 'numericcitation']\n",
      "['model', 'can', 'can theoretical explanation for nonlocal interactions in epidemic spreading']\n",
      "['opinion formation', 'in', 'network b']\n",
      "['hazard rate', 'in', 'additive-multiplicative hazard model höhle et al']\n",
      "['discrete time', 'model for', 'states of nodes in network']\n",
      "['conjunction', 'with', 'model of susceptible']\n",
      "['such connections', 'hamper', 'spreading of disease']\n",
      "['second order inﬁnitesimal', 'with', 'since development of technology']\n",
      "['dark-gray node', 'in', 'figure 5']\n",
      "['k ≥ 1', 'for ϑ', 'αk log d']\n",
      "['θ', 'represent', 'probability of direct immunization']\n",
      "['node', 'with', 'maximum eigenvector centrality']\n",
      "['rhodes', 'in', 'lattice models of epidemic spread']\n",
      "['pair', 'remove from', 'set of free half-edges']\n",
      "['state', 'in', 'in degree of advancement procedure']\n",
      "['second time series', 'use', 'reduced stochastic system of equations']\n",
      "['l ∈ tv', 'for a', 'b']\n",
      "['sir model', 'with', 'given recovery rate b']\n",
      "['sir model', 'with', 'given recovery rate b']\n",
      "['water', 'of m3', '300,000 acre-feet']\n",
      "['edge eij', 'in', 'community of size s ber of edges activated']\n",
      "['large changes', 'in', 'shape of optimal strategies']\n",
      "['random graph model', 'allow for', 'greater inhomogeneity with rather arbitrary distribution of number of contacts for diﬀerent persons']\n",
      "['epidemic outbreaks', 'in', 'complex heterogeneous networks']\n",
      "['network', 'of', 'detailed topology']\n",
      "['ten gillespie simulations', 'perform for', 'value of τ']\n",
      "['information spreading rate', 'be', 'when function of time']\n",
      "['increased coeherence', 'be', 'in our model due to loss of stability of endemic equilibrium']\n",
      "['ct', '∅ for', 't']\n",
      "['subtree', 'with', 'ﬁrst link of path from u v in removed d']\n",
      "['sigmoid function', 'with', 'increase of connection radius in similar way as in case of rggs']\n",
      "['n', 'denote', 'number of individuals in population']\n",
      "['symmetry', 'in', 'asymptomatic cases']\n",
      "['susceptible home', 'in', 'model']\n",
      "['numericcitation', 'attract', 'interest']\n",
      "['single source estimation', 'with', 'priori knowledge of set of suspect nodes']\n",
      "['92.3 correction factor', 'essentially double', 'number of cases']\n",
      "['isolation strategies', 'in', 'subsystem of model proposed']\n",
      "['y x', 'by x', 'y ∈ γ1']\n",
      "['size', 'with', 'size of population n']\n",
      "['node', 'with', 'largest degree']\n",
      "['hazard model', 'in', 'as scheike']\n",
      "['contact rate', 'incorporate seasonality in', 'transmission of infection']\n",
      "['behavior', 'in', 'limit of inﬁnite system']\n",
      "['sis model numericcitation', 'in outbreak', 'endemic state']\n",
      "['figure 1', 'example of', 'infection process']\n",
      "['pauli operators', 'commute at', 'diﬀerent lattice sites']\n",
      "['full lines', 'correspond to', 'theoretical evaluation of eq']\n",
      "['four cases', 'for pr', '1 − pi']\n",
      "['model independent parameters', 'be', 'when taken for measles']\n",
      "['increased frequency', 'in', 'tight clusters of small-world network']\n",
      "['deterministic model', 'from', 'behaviour']\n",
      "['n +1', 'in', 'terms of ψ']\n",
      "['κh', '+1 in', 'terms of ψ']\n",
      "['5 proof', 'in', 'section']\n",
      "['exponential time', 'with', 'rate 2d ρ for hd']\n",
      "['sir model', 'oﬀers', 'simple approach by set of evolution equations numericcitation']\n",
      "['nevada', 'have', 'water conservation program']\n",
      "['netherlands numericcitation', 'epidemic in', 'uk numericcitation']\n",
      "['numericcitation', 'test', 'scenarios']\n",
      "['numericcitation', 'numerically test', 'scenarios']\n",
      "['numericcitation defra/dcms', 'cost in', 'uk']\n",
      "['numericcitation defra/dcms', 'cost of', 'foot']\n",
      "['our estimator', 'outperform', 'that in numericcitation in terms of average error distance']\n",
      "['step', 'however irrelevant for', 'course of epidemic']\n",
      "['model', 'have', 'has used before in diﬀerent contexts numericcitation']\n",
      "['its steady state value', 'in', 'endemic state of disease']\n",
      "['zeros', 'in', 'reconstruction of edges']\n",
      "['zeros', 'in', 'reconstruction of edges']\n",
      "['infectious disease', 'be', 'in particular present in population beyond its initial phase of outbreak']\n",
      "['w variable cw', 'with', 'convergence of moments']\n",
      "['epidemic model', 'with', 'several diﬀerent types of infective']\n",
      "['fluctuations', 'become', 'important of network']\n",
      "['β γ', 'in', 'correspondence of lmax']\n",
      "['γ', 'in', 'correspondence of lmax']\n",
      "['seir model', 'oscillation with', 'periods']\n",
      "['tree-like structure', 'with', 'branches of infection']\n",
      "['model', 'have', 'free parameters']\n",
      "['population birth rate', '0.27 for', 'reinfection parameter']\n",
      "['sir model', 'in', 'two networks']\n",
      "['information source detection', 'in', 'large-scale networks']\n",
      "['node', 'in', 'layer a/b of connectivity k']\n",
      "['individuals', 'with', 'diﬀerent levels of immunity']\n",
      "['large d', 'for ∈', '1 λeρ']\n",
      "['numericcitation', 'attract', 'increasing interest']\n",
      "['epidemic model', 'predict', \"program 's future adoption\"]\n",
      "['p', 'plot', 'imprecision function of degree']\n",
      "['sir model', 'characterize by', 'non-equilibrium phase transition occurring as function of infection rate']\n",
      "['information source detection problem', 'study under', 'si model']\n",
      "['network', 'with', 'set of nodes v']\n",
      "['transparent method', 'transformation of', 'master equation']\n",
      "['connection', 'with', 'theory of volterra integral equations']\n",
      "['hµ', 'ψi for', 'integral of ψ']\n",
      "['persistence', 'fraction of', 'simulations surviving']\n",
      "['average number', 'in', 'population of susceptibles numericcitation']\n",
      "['catastrophic shifts', 'in', 'shape of control strategies']\n",
      "['network v', 'centrality of', 'node i satisﬁes xi']\n",
      "['population', 'be', 'constant in period of time under study']\n",
      "['possible nodes states', 'in', 'siri model']\n",
      "['shift', 'in', 'low-w regions of sir model']\n",
      "['zt', 'number of', 'of proof']\n",
      "['watson process', 'k for', 'of simplicity']\n",
      "['g small world eﬀect', 'in', 'epidemiological model']\n",
      "['home', 'have', 'structural characteristics']\n",
      "['more individuals', 'with', 'low level of immunity']\n",
      "['expansion', 'in', 'powers of 1/n numericcitation']\n",
      "['paper', 'in center', 'university of science']\n",
      "['infection rates', 'in', 'sir']\n",
      "['same quantity', 'in', 'of discrete time']\n",
      "['linear odes', 'with', 'addition of quadratic terms for number of new infections']\n",
      "['eyes', 'with', 'blood of person']\n",
      "['c/c', 'in', 'framework of inria']\n",
      "['comments', 'signiﬁcantly contribute to', 'quality of paper']\n",
      "['infection rate βs', 'in', 'sir simulations']\n",
      "['rate ct → λ 2d y', 'at ct', 'x']\n",
      "['sir model', 'in', 'esemble of multilayer networks 4.1']\n",
      "['scenes', 'with', 'ﬁeld of view']\n",
      "['role', 'in', 'emergence of infectious diseases']\n",
      "['individuals', 'in', 'state']\n",
      "['individuals', 'recover in', 'in exactly time-step']\n",
      "['neighborhoods', 'with', 'small lot sizes']\n",
      "['southern nevada water authority', 'have', 'water smart landscapes program']\n",
      "['nodes', 'in', 'network']\n",
      "['node', 'have', 'potential infection-activity measured by possibly maximal contribution to propagation process within one time step']\n",
      "['argument', 'in', 'eqs']\n",
      "['critical threshold βc', 'at', 'intersection of two eqs']\n",
      "['6 critical threshold βc', 'at', 'intersection of two eqs']\n",
      "['period', 'in', 'independently model']\n",
      "['network', 'with', 'power law degree distribution']\n",
      "['pi', 'be', 'randomly chosen']\n",
      "['model', 'in', 'case of basic reproduction number r0 1']\n",
      "['network', 'with', 'cycles']\n",
      "['joint probability', 'β at', 'time t']\n",
      "['neighborhoods', 'with', 'large lot sizes']\n",
      "['signiﬁcant simpliﬁcation', 'in', 'study of model']\n",
      "['model', 'in', 'höhle']\n",
      "['cascading failures', 'can occur', 'numericcitation']\n",
      "['immunization strategy', 'stop', 'spreading of disease']\n",
      "['cumulative wsl conversion area', 'in', 'acres']\n",
      "['sir model', 'will', 'will then particular case of sirs model']\n",
      "['lower susceptibility', 'consider', '30 of population']\n",
      "['dynamics', 'with', 'memory of nonredundant information']\n",
      "['blue curves', 'show', 'fast extinction of epidemic']\n",
      "['state r', 'in', 'state']\n",
      "['proportional growth', 'in', 'number of infected farms']\n",
      "['stochastic terms', 'in', 'governing equations']\n",
      "['attention', 'have', 'recently has focussed on impact of network topology on dynamics of processes running on it with emphasis on spreading of infectious diseases']\n",
      "['susceptibleexposed-infected-recovered seir epidemiological model', 'with', 'stochastic forcing']\n",
      "['infection term', 'in nonlinearity', 'immigration of infectives']\n",
      "['boundary problems', 'in', 'identiﬁcation of peers']\n",
      "['catastrophic shift', 'in', 'shape of isolation strategy']\n",
      "['quantity', 'deﬁnes', 'conditions necessary for spreading of illness']\n",
      "['bolker', 'chaos in', 'measles models']\n",
      "['simultaneous cluster infection', 'can give', 'rise to discontinuous jumps of diﬀerent sizes in number of failed nodes']\n",
      "['node', 'in', 'h']\n",
      "['node', 'in', 'h']\n",
      "['node', 'in', 'h']\n",
      "['node', 'for pr', 'v']\n",
      "['node', 'for pr', 'v']\n",
      "['selected person', 'take', 'opinion of group']\n",
      "['difference', 'in', 'terms of average number of hops']\n",
      "['apollonian network', 'with', '1096 nodes']\n",
      "['sixth-generation apollonian network', 'with', '1096 nodes']\n",
      "['threshold value', 'independent of', 'topology of underlying network']\n",
      "['model', 'have', 'performance on test period']\n",
      "['world', 'for number', 'all of canada']\n",
      "['contact', 'with', 'infected one through link of network']\n",
      "['base node', 'susceptible at', 'time t']\n",
      "['rare event', 'in', 'tail of noise distribution']\n",
      "['disease', 'fade away as', 'fraction of infected individuals']\n",
      "['linear odes', 'with', 'addition of quadratic terms for number of new infections similar to si term in sir model']\n",
      "['nodes', 'of collection', 'number in set υ']\n",
      "['irsirs model', 'with', 'spontaneous loss of immunity']\n",
      "['irsirs model', 'with', 'spontaneous loss of immunity']\n",
      "['irsirs model', 'with', 'spontaneous loss of immunity']\n",
      "['c3', 'with', 'population size of n']\n",
      "['practical interest', 'in', 'theoretical understanding of behaviour of near-critical epidemics']\n",
      "['sir model', 'with', 'β']\n",
      "['role', 'in', 'prediction of global epidemics']\n",
      "['infected half-edges', 'in', 'also sub critical case']\n",
      "['nodes', 'resist', 'infection sprea ding']\n",
      "['time', 'time for', 'optimal reduction of transmission strategy']\n",
      "['networks', 'use', 'molloy-reed algorithm numericcitation']\n",
      "['panic', 'negatively eﬀect', 'economy of region']\n",
      "['largest spread', 'with', 'half of nodes having']\n",
      "['o nαn', 'in', 'proof of case']\n",
      "['ﬁxation', 'in', 'two-strain extensions of sis models']\n",
      "['sirws model', 'in', 'numericcitation']\n",
      "['changes', 'in', 'demand of product/service']\n",
      "['n', 'be', 'where number of nodes in network']\n",
      "['death process', 'reduce', 'number of degrees of freedom']\n",
      "['scenario 1', 'for e', 'la']\n",
      "['scenario 2', 'for e', 'la']\n",
      "['stochastic terms', 'thus justify', 'our use of deterministic center manifold for particular model']\n",
      "['same approach', 'in', 'as proposition 2 of armbruster']\n",
      "['function', 'with', 'hki']\n",
      "['function', 'with', 'hki']\n",
      "['4 simpler standard seir model', 'demonstrate', 'power of our method']\n",
      "['node', 'with', 'degree k']\n",
      "['optimal infection source estimator', 'in', 'ﬁnite regular tree']\n",
      "['individuals', 'with', 'diﬀerent topological features']\n",
      "['jordan center1', 'give', 'optimal estimator associated with most likely infection path']\n",
      "['network', 'be', 'simpliﬁed']\n",
      "['our optimal control problem', 'minimize', 'u']\n",
      "['ni', 'in', 'rest of section']\n",
      "['home', 'in', 'recovered status from its construction onwards']\n",
      "['small same lines', 'in', 'as proof of theorem 1.1']\n",
      "['reduction', 'in', 'transmissibility of infectious disease']\n",
      "['insets', 'show', 'proportions of two possible states for diﬀerent values of τ']\n",
      "['impact', 'in', 'sierra leone']\n",
      "['third term', 'denote', 'recovery of i-nodes of degree k']\n",
      "['offspring distribution', 'be', 'where function of τ']\n",
      "['jordan infection center', 'in', 'tree networks with inﬁnitely many levels']\n",
      "['abrupt change', 'in', 'persistence']\n",
      "['degree', 'number of', 'contacts of corresponding individual']\n",
      "['changes', 'in', 'population of compartment']\n",
      "['largest uncertainty', 'in', 'size of large-scale epidemic']\n",
      "['ﬁrst time series', 'use', 'complete stochastic system of transformed equations']\n",
      "['ﬁrst time series', 'use', 'complete stochastic system of transformed equations']\n",
      "['seir model numericcitation', 'with', 'epidemiological parameters']\n",
      "['dashed lines', 'show', 'possible solutions of eq']\n",
      "['last option', 'get', 'rid of stochastic ﬂuctuations']\n",
      "['one recovery', 'in', 'd units of time']\n",
      "['problem', 'signiﬁcantly more challenge than', 'that in si model']\n",
      "['mean ﬁeld model', 'capture', 'eﬀects of low connectivity']\n",
      "['node', 'with', 'high eigenvector centrality']\n",
      "['probability measure', 'with', 'rate λθ']\n",
      "['nodes', 'be', 'populations']\n",
      "['f2 ≤', 'with', 'solutions ϕ1']\n",
      "['similar analysis', 'with', 'that leads to lemma 3.4 of numericcitation']\n",
      "['stochastic eﬀects', 'in', 'seasonally forced models']\n",
      "['theorem 2', 'consistent with', 'theorem 1 for infection in si model']\n",
      "['death processes', 'with', 'application to variability in compartmental models']\n",
      "['numericcitation', 'in', 'same parameter region']\n",
      "['noise terms', 'evaluate at', 'µτ']\n",
      "['similar analysis', 'with', 'that leads to lemma 4.1']\n",
      "['following analytical form numericcitation', 'in', 'sir model']\n",
      "['instance', 'in', 'case of equine inﬂuenza numericcitation']\n",
      "['model', 'reproduce', 'qualitative features of random disease propagation observed in true data']\n",
      "['euclidean network abdul khaleque1', 'on model', '1department of physics']\n",
      "['s iri model', 'with', 'nonlinear incidence of infection']\n",
      "['nodes', 'be', 'single individuals']\n",
      "['demographic sir model', 'in', 'mechanism for renewal of susceptibles']\n",
      "['species survival', 'in', 'endemic state of disease']\n",
      "['average epidemic size', 'gauge', 'eﬀects of stochasticity']\n",
      "['aii ≤ pj 6', 'result on', 'eigenvalues of system']\n",
      "['size', 'number of', 'nodes reached by infection']\n",
      "['work', 'in', 'numericcitation']\n",
      "['taylor series expansion', 'equate', 'terms of same order']\n",
      "['11 error', 'in', 'number of new infections']\n",
      "['multiple jordan center', 'set', 'mjc']\n",
      "['rumor source', 'in', 'online social networks']\n",
      "['numericcitation', 'in', 'model']\n",
      "['peaks', 'in', 'number of nodes']\n",
      "['t ∗', 'vaccination as', 'functions of maximum eﬀort']\n",
      "['c', 'in', 'collective of lhns of real-world networks']\n",
      "['euclidean network abdul khaleque1', 'on model', 'university of calcutta']\n",
      "['data', 'ﬁt', 'predictive model of seir type']\n",
      "['elad yom-tov microsoft israel', 'model for', 'spread of pathogens in population']\n",
      "['decay', 'in', 'distribution of distances']\n",
      "['k infected nodes', 'in', 'set of m susceptible nodes']\n",
      "['solutions', 'give by', 'intersection of r.h.s of equations with identity']\n",
      "['diﬀer', 'in', 'their values for rate of']\n",
      "['system', 'introduce', 'small fraction of individuals infected by invading strain']\n",
      "['belief propagation', 'perform very well in', 'case of degree-based observation']\n",
      "['reinforcement signal ber', 'clearly dominate', 'number of furthermore']\n",
      "['longer epidemics', 'involve', 'larger number of animals slaughtered']\n",
      "['result', 'as 9', 'probability of node i ﬁrst becoming']\n",
      "['pi', 'randomly choose from', '0.5 1']\n",
      "['numericcitation maslov s', 'in', 'dynamic kill-the-winner model of microbial ecosystems']\n",
      "['diﬀerence', 'in', 'knowledge of states of nodes']\n",
      "['inﬂuential spreaders', 'in', 'networks']\n",
      "['healing processes', 'thus only occur at', 'perimeter of encapsulated regions']\n",
      "['square lattice', 'with', 'minimum fraction of active sites']\n",
      "['diﬀerent behaviors', 'in', 'spread of epidemic']\n",
      "['mathematical analysis', '214 on', 'variation with parameter of zeros of bessel']\n",
      "['logistic regression', 'with', 'citation']\n",
      "['delay', 'represent', 'average duration of disease-induced immunity']\n",
      "['a. tour e. ebola outbreak', 'in', 'conakry']\n",
      "['nodes', 'in', 'communities of size s drawn from size-distribution p s']\n",
      "['resonant ampliﬁcation', 'observe', 'numericcitation']\n",
      "['natural scenario', 'in', 'epidemiology of diseases']\n",
      "['one infection sources', 'in', 'tree network']\n",
      "['xtobs x0', 'in', 'of our analysis']\n",
      "[\"network 's clustering coeﬃcient\", 'in', 'when small world region']\n",
      "['r immune hosts', 'with', 'high level of immunity']\n",
      "['reduction', 'in', 'systems with reinfection']\n",
      "['network', 'with', 'adjacency matrix']\n",
      "['invasion', 'in', 'multiple strain infections']\n",
      "['quarantine model', 'allow for', 'chance of infection']\n",
      "['infected nodes', 'recover at', 'rate µ']\n",
      "['march 31', 'from number', 'all of canada']\n",
      "['neighbor', 'with', 'degree k']\n",
      "['higher order moments', 'in', 'terms of ﬁrst']\n",
      "['latent periods', 'in', 'sections']\n",
      "['r immune hosts', 'with', 'intermediate level of immunity']\n",
      "['imprecision', 'function over', 'all of our networks including ca-hepth network']\n",
      "['rf model', 'describe in', 'section 3.3.1']\n",
      "['its understanding', 'in', 'context of time-varying graphs']\n",
      "['various models', 'describe', 'spreading process of infection']\n",
      "['network diameter', 'deﬁned as', 'longest of shortest paths']\n",
      "['x2', 'with', 'solutions ϕ1']\n",
      "['b seasonality', 'in', 'epidemic model']\n",
      "['our secondary infection', 'spread on', 'subgraph of hosts infected with primary']\n",
      "['interesting choice', 'in', 'strategy of secondary infection']\n",
      "['global targeted strategies', 'mitigate epidemic with', 'limited amount of resources']\n",
      "['uk numericcitation', 'epidemic of', 'high pathogenic avian inﬂuenza in midwestern usa numericcitation']\n",
      "['sequence', 'in', 'in view of hypothesis']\n",
      "['he sir process', 'in', 'limit of large n']\n",
      "['susceptible individual', 'have', 'rate of infection']\n",
      "['small world probability', 'with', 'other site on lattice']\n",
      "['program', 'over', 'history']\n",
      "['program', 'throughout', 'history']\n",
      "['figure 9', 'of', 'estimators identiﬁed by reverse infection algorithm']\n",
      "['single jordan center estimation algorithm', '19 with', 'computational complexity of o']\n",
      "['speciﬁc time', 'in', 'conditional probability ps']\n",
      "['parameter l0 .3', 'in', 'limit of inﬁnite τ τc']\n",
      "['rise', 'in', 'popularity of osns used']\n",
      "['example', 'in', 'numericcitation']\n",
      "['siir model', 'with', 'effective recovery rate b ′']\n",
      "['effective recovery rate', 'in', 'siir model']\n",
      "['susceptibility', 'in', 'sir']\n",
      "['families', 'in', 'percentage of respondents medium']\n",
      "['monte brownian motion', 'with', 'choice of convex hull as function of chosen d']\n",
      "['spatial eﬀects', 'in', 'transmission of diseases']\n",
      "['barth elemy', 'in', 'scale-free networks']\n",
      "['scale-free networks', 'with', 'identical infectivity']\n",
      "['percolation', 'in', 'networks']\n",
      "['central nodes', 'in', 'networks']\n",
      "['optimal sample path', 'in', 't ∗']\n",
      "['extinction time distribution', 'be', 'for extinction time distribution global feature of epidemic process']\n",
      "['process', 'survive with', 'positive probability for value of λ0']\n",
      "['infection path x t', 'give supernode', 's']\n",
      "['social eﬀects', 'in', 'diﬀusion of solar photovoltaic technology']\n",
      "['average degree', 'decrease with', 'elongation of rectangle']\n",
      "['time', 'in', 'as view of eqs']\n",
      "['time', 'with', 'ﬁnite recovery rates on order of few months to year']\n",
      "['random walk', 'in', 'space of number of susceptible individuals']\n",
      "['routh-hurwitz conditions', 'obtain', 'sign of real part of eigenvalues of jacobian']\n",
      "['increase', 'in', 'spatial correlations']\n",
      "['numerical simulations', 'conﬁrmed', 'theoretical analysis of equilibria for model']\n",
      "['stochasticity', 'in', 'transmission of infection']\n",
      "['line', 'in', 'deterministic limit']\n",
      "['one infected node', 'in', 'y']\n",
      "['pre-vaccination incidence records', 'well reproduce by', 'simpler deterministic models with seasonal forcing']\n",
      "['initial outbreak', 'successfully capture by', 'reduced system']\n",
      "['susceptible population', 'with', 'control measures in place numericcitation']\n",
      "['disruption', 'in', 'part of world numericcitation']\n",
      "['two points', 'with', 'same image of function']\n",
      "['forward process', 'approximate', 'early stages of epidemic']\n",
      "['λ', 'exponent of', 'power law of degree distribution']\n",
      "['error bars', 'indicate', 'standard error of mean']\n",
      "['error bars', 'indicate', 'standard error of mean']\n",
      "['error bars', 'indicate', 'standard error of mean']\n",
      "['threshold', 'with', 'exponent 2 for m for m ≥ 2']\n",
      "['sir', 'in', 'ﬁnal state']\n",
      "['our results', 'with', 'those of pervious authors']\n",
      "['same trend', 'with', 'elongation of rectangle']\n",
      "['time', 'with', 'sir components']\n",
      "['number', 'recover at', 'time t']\n",
      "['variation', 'with', 'value of exponent ˜ν']\n",
      "['network', 'with', 'p']\n",
      "['network', 'with', 'p']\n",
      "['network', 'with', 'p']\n",
      "['tree network', 'with', 'multiple sources for si model']\n",
      "['authors', 'in', 'numericcitation']\n",
      "['endemic eﬀect', 'be', 'thus ma jor component of individual homes activation probability']\n",
      "['model', 'with', 'epidemic component']\n",
      "['n', 'represent', 'number of nodes']\n",
      "['stationary solution', 'be', 'in fact already known in context of genetic propagation of mutant allele numericcitation']\n",
      "['small world networks', 'with', 'sir']\n",
      "['small world networks', 'with', 'sir']\n",
      "['stochastic solutions', 'be', 'in way very robust to noise']\n",
      "['maximum', 'in', 'number of infectious individual']\n",
      "['example', 'in', 'context of mode competition']\n",
      "['numericcitation g. f. cooper', 'method for', 'induction of probabilistic networks from data']\n",
      "['vacant homes', 'of number', 'problem']\n",
      "['ﬁnite size scaling', 'in', 'types of systems numericcitation']\n",
      "['estimator', 'in', 'ﬁnite tree network']\n",
      "['model', \"'s parameter\", 'p']\n",
      "['situation', 'more subtle in', 'condition for existence of maximum in number of infectious individual']\n",
      "['tree', 'be', 'branching']\n",
      "['their model', 'fall into', 'one of three universal classes due to behavior of ﬁxed point curves']\n",
      "['numericcitation', 'understand of', 'causative virus']\n",
      "['victims', 'die', 'numericcitation']\n",
      "['165 victims', 'die', 'numericcitation']\n",
      "['d', 'be', 'where independent of size of infected subnetwork']\n",
      "['increase', 'in', 'spatial correlations']\n",
      "['ﬁxed rate ζ', 'in', 'relation to size of exposed subpopulation size']\n",
      "['weights', 'constant in', 'numericcitation']\n",
      "['phenomena', 'in', 'population of interacting individuals']\n",
      "['biocomputation', 'for institute', 'university of zaragoza']\n",
      "['right', 'with', 'elongation of rectangle']\n",
      "['λ1', 'determine', 'conditions of invasion']\n",
      "['reduced system', 'with', 'mortality']\n",
      "['optimal strategy', 'always rely in', 'adoption of maximum control for entire epidemic']\n",
      "['method', 'in', 'context of population genetics numericcitation']\n",
      "['epidemic model', 'on', 'likelihood integrated with house level covariates']\n",
      "['social contagion', 'in', 'adoption of solar photovoltaic technology']\n",
      "['correlations', 'in', 'transmission of infection']\n",
      "['partial observation', 'consider', 'connected subgraph of 30 of nodes']\n",
      "['cases', 'thus meet', 'fewer individuals']\n",
      "['zmin', 'correspond to', 'low level of immunity']\n",
      "['node', 'with', 'degree equal to 4 numericcitation']\n",
      "['deﬁnite sw structure', 'with', 'small number of nodes']\n",
      "['central node', 'surround by', 'infected nodes']\n",
      "['roc curve', 'start from', 'origin of axes']\n",
      "['mathematical model', 'describe', 'characteristics of hbv disease transmission']\n",
      "['further increase', 'in', 'fraction of recovered nodes']\n",
      "['pool', 'presence of', 'next most signiﬁcant factor associated with wsl participation']\n",
      "['threshold', 'in', 'circumstances']\n",
      "['citation house', 'with', 'pool']\n",
      "['pre-vaccination incidence oscillations', 'with', 'broad 2.5 year period']\n",
      "['si1 i2r model', 'with', '0.02 solid line']\n",
      "['continuum family', 'with', 'arbitrary proportion of strains']\n",
      "['spatial sir model', 'depend on', 'only three additional parameters']\n",
      "['small epidemic threshold', 'in', 'susceptible-infected-susceptible model numericcitation']\n",
      "['stochastic s i1 i2s model', 'for s', 's → s with strain 1 with strain 2 of i1 κ1 i1 recovery of i2 κ2 i2 table i. transition rates']\n",
      "['b chaos', 'in', 'measles models']\n",
      "['changes', 'in', 'latent periods']\n",
      "['convenient deterministic framework', 'model', 'spread of epidemic']\n",
      "['our next step', 'bind', 'zn − ¯ z']\n",
      "['sis model', 'in', 'sis model']\n",
      "['partial epidemic realization', 'ﬁned by', 'complete knowledge of infected']\n",
      "['partial epidemic realization', 'de ﬁned by', 'complete knowledge of infected']\n",
      "['numericcitation a. barrat', 'process on', 'complex networks']\n",
      "['much recent work', 'thus have focus on', 'role of network of disease-transmitting contacts']\n",
      "['t', 'consider', 'the following six possible cases for y t +1']\n",
      "['aspects', 'deﬁning', 'optimal protocols of intervention for diseases eradication']\n",
      "['network structure', 'have', 'an in ﬂuence on ratio too']\n",
      "['quantum approach', 'in', 'quantum hamiltonian representation since combined inﬂuence of noise of freedom']\n",
      "['memory eﬀects', 'with', 'diﬀerent values of α']\n",
      "['small-world networks', 'on model', 'faculty of science']\n",
      "['small-world networks', 'on model', 'faculty of science']\n",
      "['existing diﬀerential equations', 'in', 'ode systems']\n",
      "['r0', 'with', 'initial condition t0']\n",
      "['test period', 'as year', 'citation citation']\n",
      "['x2 dynamics', 'in', 'new coordinate system']\n",
      "['time', 'in', 'master equation']\n",
      "['participating households', 'in', 'study area']\n",
      "['non-participating home', 'infect by', 'neighbor']\n",
      "['zhang', 'model for', 'analysis of sars transmission patterns in china']\n",
      "['contact', 'with', 'each of its neighbours']\n",
      "['non-convergence', 'in', 'instances of epidemic process']\n",
      "['other parameter values', 'in', 'as fig 2']\n",
      "['up time', 'to collision', 't. given evolution of process']\n",
      "['network', 'with', 'elements']\n",
      "['our method', 'describe on', '44 of variance in ili time series']\n",
      "['rossi', 'model for', 'assessment of bovine tuberculosis surveillance in dairy farm network in emilia-romagna']\n",
      "['inequalities', 'in', 'lemma 5']\n",
      "['model', 'in', 'numericcitation']\n",
      "['cintron-arias', 'describe', 'popularity of viral videos']\n",
      "['expected state h nx', 'in', 'our stochastic model']\n",
      "['abrupt change', 'in', 'starting point of optimal changes']\n",
      "['time', 'in', 'recurrent epidemics']\n",
      "['d1/4 vertices', 'in', 'γ2']\n",
      "['lines', 'represent', 'best-ﬁt model to data']\n",
      "['minimum eccentricity', 'in', 'graph']\n",
      "['sources', 'apply', '105 realizations of sir model']\n",
      "['set', 'in', 'g with minimum infection range']\n",
      "['gt', 'be', 'random']\n",
      "['random', 'at probability', 'ni']\n",
      "['h g sustained oscillations', 'in', 'stochastic systems']\n",
      "['conﬁguration model', 'have', 'has used in study of epidemics numericcitation']\n",
      "['information source detection problem', 'in', 'scenarios']\n",
      "['algorithm 1', 'require', 'speciﬁcation of tuning parameter λ']\n",
      "['present results', 'with', 'those of static percolation on same network']\n",
      "['long range edges', 'in', 'case of swn']\n",
      "['diﬀerences', 'in', 'ﬁnal time of epidemic between optimal control']\n",
      "['diﬀerences', 'in', 'ﬁnal time of epidemic between optimal control']\n",
      "['diﬀerences', 'in', 'ﬁnal time of epidemic between optimal control']\n",
      "['a. self-similar community structure', 'in', 'network of human interactions']\n",
      "['epidemiology', 'in', 'coevolution of parasites']\n",
      "['graphs', 'occur in', 'propagation of infectious disease']\n",
      "['eﬀect', 'lead', 'overestimation of probability of long extinction times']\n",
      "['ﬁrst term', 'in', 'r.h.s accounts for recovery of infected individuals']\n",
      "['recurrent epidemics', 'in', 'small world networks j. a. verdascaa']\n",
      "['nin', 'be', 'where number of interfaces']\n",
      "['approach', 'accurate in', 'case of tree-like networks']\n",
      "['distribution', 'in', 'system']\n",
      "['will crucial importance', 'in', 'proof of theorem 1']\n",
      "['jordan center', 'in', 'section iii']\n",
      "['increase', 'in', 'period of recurrent epidemic peaks reported after mass vaccination']\n",
      "['networks', 'with', 'heterogeneous power law degree distribution']\n",
      "['second term', 'correspond to', 'recovery of infected individuals at rate γ']\n",
      "['system', 'decrease', 'value of α including memory']\n",
      "['infinite population', 'in', 'limit of inﬁnite system size']\n",
      "['s', 'denote', 'number of susceptible individuals']\n",
      "['si', 'in', 'heterogeneous networks']\n",
      "['epidemic dynamics', 'faster than', 'dynamics of opinions']\n",
      "['proposed method', 'accurately reconstruct', 'topology of 05100.50.60.70.80.91 node degreea']\n",
      "['10 a. rachah', 'torres to', 'population']\n",
      "['2.2 deterministic symptom-response epidemic models', 'with', 'as nearly all epidemic models']\n",
      "['sustained ﬂuctuations', 'with', 'well deﬁned period']\n",
      "['bias', 'in', 'favor of slow strain for which']\n",
      "['epidemics', 'with', 'two levels of mixing']\n",
      "['effects', 'in', 'country']\n",
      "['expressions', 'in', 'terms of gontcharoﬀ polynomials']\n",
      "['stochastic propagation processes', 'can', 'can modeled as markov chains of exponential dimensions in size of network']\n",
      "['opinion dynamics models numericcitation propagation', 'synchronization of', 'numericcitation']\n",
      "['schematic representation', 'with', 'spontaneous loss of immunity']\n",
      "['stable equilibria', 'with', 'spontaneous loss of immunity']\n",
      "['multilayer networks', 'with', 'number of layers m']\n",
      "['individuals', 'oscillate in', 'time']\n",
      "['lloyd', 'variability in', 'models for recurrent epidemics']\n",
      "['erdős-rényi numericcitation degree distribution', 'with', 'hki']\n",
      "['model', 'divide to', 'individuals of population in three categories']\n",
      "['size outbreak', 'diverge in', 'vicinity of threshold']\n",
      "['sir', 'in', 'case']\n",
      "['infectious contacts', 'with', 'susceptible individuals']\n",
      "['incidence oscillations', 'with', 'similar population size']\n",
      "['initial conditions', 'construct in', 'one of two ways']\n",
      "['numericcitation', 'determine', 'upper-and lower bound for symptom-response seir model']\n",
      "['solution', 'in', 'comparison to expected fraction calculated for increasing number of nodes']\n",
      "['em', 'bat as', 'reservoirs of ebola virus']\n",
      "['our second model deals', 'with', 'two infectious strains competing for same susceptible population in framework of sir model']\n",
      "['random graph numericcitation', 'for l', 'total number of vertices n']\n",
      "['probability φi', 'neighbor at', 'time t']\n",
      "['vertices', 'in', 'susceptibility set of v0']\n",
      "['our theoretical results', 'conﬁrmed by', 'extensive simulations of sir process']\n",
      "['non-linear dynamics', 'in', 'speciﬁcally tractability of likelihood']\n",
      "['de', 'ﬁne', 'right hand of eq']\n",
      "['eigenvalues', 'in', 'of always βa']\n",
      "['disease', 'in', 'framework of sir mechanism']\n",
      "['more research', 'in', 'long-time behavior of epidemic models']\n",
      "['proposition 1 proposition 2', 'in', 'numericcitation']\n",
      "['largest school network', 'with', '2539 nodes']\n",
      "['goutsias', 'have', 'method numericcitation of implicit-euler integration']\n",
      "['their proof', 'identify', 'inﬂuence spread of single node under sir model']\n",
      "['wide quantitative variations', 'with', 'structure of network']\n",
      "['nodes', 'with', 'outgoing edges to u']\n",
      "['our exact results', 'obtain for', 'thermodynamic limit of inﬁnite population size']\n",
      "['recovered vertices', 'recover for', 'rest of simulation']\n",
      "['section ii', 'provide', 'complete description of model']\n",
      "['deterministic system', 'have', 'two ﬁxed points']\n",
      "['hyper-node', 'with', 'n edges']\n",
      "['5 nodes', 'with', '4 edges']\n",
      "['air', 'in', 'case of inﬂuenza']\n",
      "['nonlocal interaction', 'in', 'spreading of outbreak']\n",
      "['numericcitation w. luo', 'source in', 'network']\n",
      "['terms', 'in', 'normal form coordinate']\n",
      "['susceptibility χ', 'estimate', 'epidemic threshold of sir model']\n",
      "['variability ∆', 'determine', 'epidemic threshold of sir model']\n",
      "['great importance', 'with', 'vaccination treatment']\n",
      "['epidemic threshold', 'obtain via', 'numerical simulations']\n",
      "['model', 'speciﬁed by', 'k k − 1 state transition probabilities given in 3x3 stochastic matrix 0 pi']\n",
      "['s', 'epidemic for', 'optimal reduction of solid curve']\n",
      "['disc', 'be', 'ml estimator for regular trees under si model with single source']\n",
      "['mckendrick', 'study on', 'theory of continuous probabilities']\n",
      "['process', 'in', 'idle state decays with time']\n",
      "['free susceptible half-edge', 'with pair', 'expected square of jump satisﬁes']\n",
      "['other ma jor environmental changes', 'in', 'las vegas metro area']\n",
      "['transformed system', 'give by', 'eqs']\n",
      "['small world network', 'in', 'section']\n",
      "['bistability regions iii', 'vastly diﬀerent depend', 'depending value of re-susceptibility rate q']\n",
      "['sw model', 'arouse', 'new wave of epidemiological research']\n",
      "['gillingham', 'address', 'unobservables']\n",
      "['network model', 'with', 'p']\n",
      "['network model', 'with', 'p']\n",
      "['networks', 'with', 'large initial conditions']\n",
      "['van kampen', 'have', 'expansion of master equation of corresponding stochastic model']\n",
      "['mechanistic epidemic model', 'with', 'autoinfection']\n",
      "['control measures', 'in', 'place numericcitation']\n",
      "['parameters', 'estimate', 'states of such stochastic large-dimension phenomena']\n",
      "['epidemic pathways', 'in', 'global outbreaks of infectious diseases']\n",
      "['data', 'of year', 'citation citation']\n",
      "['epidemic spreading', 'in', 'population of large size n']\n",
      "['telecomunicazioni', 'laboratory for', 'modeling of biological systems']\n",
      "['equilibrium point', 'with', 'entry equal to 0']\n",
      "['phase diagram', 'in', 'space of parameters']\n",
      "['associated vaccination', 'with', 'autism numericcitation']\n",
      "['optimal sample path', 'with', 'node']\n",
      "['two sums', 'in', 'exponential of equation']\n",
      "['fraction', 'hence contribute negatively to', 'fraction of infected individuals']\n",
      "['theoretical solution', 'thank to', 'mapping of model to bond percolation numericcitation']\n",
      "['state', 'with', 'probability computed from knowledge of states of observed nodes']\n",
      "['part', 'in', 'each of types']\n",
      "['stochastic terms', 'in', 'systems']\n",
      "['transmission', 'in', 'type of network']\n",
      "['numericcitation a. rachah', 'in', 'west africa']\n",
      "['such model', 'new in', 'ebola context numericcitation']\n",
      "['numericcitation', 'in', 'real populations']\n",
      "['number', 'in', 'thee network']\n",
      "['numericcitation', 'in', 'terms of average error distance']\n",
      "['recovered population', 'be', 'in contrast coupled to infected population']\n",
      "['following lemma', 'be', 'crucial 4.3']\n",
      "['excellent agreement', 'with', 'solution of original stochastic system for temporal scale']\n",
      "['r', 'recover individual at', 'time t. of course']\n",
      "['theorem 1', 'will', 'will proved in § 2']\n",
      "['theorem 2', 'will', 'will proved in § 3']\n",
      "['absolute difference', 'in', 'number of infected individuals']\n",
      "['complex dynamical system', 'with', 'a minimal of technical syntax']\n",
      "['sir dynamics', 'with', 'diﬀerent mechanisms of replenishment of susceptibles']\n",
      "['knowledge spreading', 'with', 'sir model']\n",
      "['infected individual', 'have during', 'its infectious period']\n",
      "['t b β', 'w for', 'identical values of β']\n",
      "['ﬁrst repeated index', 'in', 'sequence of households']\n",
      "['plot simulated', 'with', 'cutoﬀ of equation']\n",
      "['kolmogorov 3 equations', 'system of', 'n2 diﬀerential equations']\n",
      "['oscillations', 'in', 'numbers of three compartments']\n",
      "['infected patient', 'on', 'knowledge of consequences in contact with healthy individuals']\n",
      "['12 positive entries', 'h on', 'entries in h i']\n",
      "['only healthy nodes', 'with', 'stationary density of infected nodes']\n",
      "['our main tool', 'be', 'lemma 5']\n",
      "['sir model', 'in', 'four real networks']\n",
      "['municipal governance', 'in', 'las vegas metro area']\n",
      "['0 exp', 'drop below', 'one individual']\n",
      "['nodes', 'in', 'graph g']\n",
      "['peak', 'in', 'switching time correspondence of peak']\n",
      "['ﬁrst term', 'in', 'left-hand side of eq']\n",
      "['models', 'of type', 'deterministic numericcitation']\n",
      "['r0', 'duration of', 'infection']\n",
      "['numericcitation h. w. hethcote', 'mathematics of', 'infectious diseases']\n",
      "['recovered vertices', 'stay', 'recovered for rest of simulation']\n",
      "['information source detection', 'in', 'sir model']\n",
      "['i0', 'on dependence', 'number of infected individual']\n",
      "['numericcitation l. borio', 'virus as', 'biological weapons']\n",
      "['more researchers', 'rigorously analyze', 'accuracy of compartmental models']\n",
      "['infection spreading', 'for 4', 'i']\n",
      "['threshold condition', 'be', 'when violated']\n",
      "['η1', 'η of', 'state']\n",
      "['state', 'of permutation', '0 → 2 → 1']\n",
      "['numerical estimation', 'in', 'of infected compartment']\n",
      "['individuals', 'can move between', 'two nodes connected by link']\n",
      "['simulations results', 'in', 'ﬁnal state of process']\n",
      "['ﬁrst identiﬁ citation', 'in', 'guongdong province of china numericcitation']\n",
      "['collective', 'in c', 'fraction of λmfl']\n",
      "['thick red curve', 'show clearly', 'bottom of sharp valley']\n",
      "['susceptible individuals', 'in', 'sir model']\n",
      "['grassberger', 'p on', 'critical behaviour of general epidemic process']\n",
      "['marked contrast', 'with', 'results for measles for same contact network']\n",
      "['models', 'with', 'scenario of limited vaccines availability']\n",
      "['u', 'have', 'larger infection eccentricity']\n",
      "['t', 'be', 'where time length of infection data']\n",
      "['dynamics', 'in', 'general stochastic sir epidemic x']\n",
      "['epidemics', 'with', 'pronounced 3.5 year period in line']\n",
      "['u', 'susceptible at', 'time slot t']\n",
      "['neighborhood', 'within', 'results']\n",
      "['figure 1', 'show', 'systematic study of sir model on uncorrelated conﬁguration net works']\n",
      "['networks', 'with', 'inﬂuence produced by shape variation of plots']\n",
      "['ﬁrst sum', 'in', 'exponential of equation']\n",
      "['maximum', 'in', 'number of infectious particles']\n",
      "['infected individuals', 'recover at', 'ﬁxed time tr']\n",
      "['epidemic thresholds', 'in', 'real networks']\n",
      "['parameter', 'in', 'case of fastsir']\n",
      "['local susceptibility set', 'in', 'contrast to z f']\n",
      "['infection', 'take ﬁrstly', 'control of large degree vertices']\n",
      "['infected node', 'in', 'state i']\n",
      "['xn', 'variable by', 'infected node of degree n numericcitation']\n",
      "['abel equation', 'thus lead to', 'signiﬁcant simpliﬁcation in study of model']\n",
      "['simple deterministic models', 'with', 'seasonal forcing']\n",
      "['model', 'estimate', 'initial growth rate of sars epidemic']\n",
      "['convergence', 'in', 'probability']\n",
      "['one poisson process', 'with', 'rate 1 particles at neighboring sites of x']\n",
      "['discrete-time epidemic spreading model', 'with', 'autoinfection']\n",
      "['infected population', 'grow more slowly in', 'stochastic model']\n",
      "['diﬀusion limit', 'derive for', 'ﬁnal size of near-critical epidemic']\n",
      "['from graphs', 'with', 'speciﬁed sequence of contact numbers']\n",
      "['guimera', 'structure in', 'network of human interactions']\n",
      "['two features', 'increase in', 'number of recognized cases']\n",
      "['recurrent epidemics', 'in', 'unforced spatially extended models']\n",
      "['sir model', 'in', 'sir model']\n",
      "['n', 'a. towards', 'characterization of behaviordisease models']\n",
      "['px', 'd for', 'x']\n",
      "['epidemic thresholds', 'in', 'real networks']\n",
      "['sustained oscillations', 'with', 'periods']\n",
      "['sustained oscillations', 'with', 'periods']\n",
      "['as spatial sir model', 'with', 'following modiﬁcations']\n",
      "['supplementary information', 'available in', 'online version of paper']\n",
      "['ﬂuctuations', 'in', 'number of infective neighbors']\n",
      "['sars outbreaks', 'in', 'diﬀerent parts of world']\n",
      "['solid square curve', 'distribuin', 'tion of whole epidemic network']\n",
      "['expression', 'same as', 'ω1 in eq']\n",
      "['onion-like structures', 'illustrate in', 'fig 7']\n",
      "['our results', 'suggest', 'new implementation of highly eﬀective strategy for targeted vaccinations through use of popular locations in rural communities']\n",
      "['l', 'theory from', 'displacement measurements of animal populations']\n",
      "['discontinuous transition', 'in', 'fraction of largest cluster component']\n",
      "['ﬁnal size', 'treat speciﬁcally in', 'numericcitation']\n",
      "['infected node', 'stay', 'infected numericcitation']\n",
      "['class', 'with', 'exponentially distributed latent periods']\n",
      "['class', 'with', 'latent periods exponentially distributed']\n",
      "['nodes', 'in', 'state s']\n",
      "['numericcitation e. hansen', 'control of', 'epidemics with limited resources']\n",
      "['same argument', 'in', 'as proof of corollary 4 of lalley']\n",
      "['time behavior', 'in', 'supercritical regime for diﬀerent values of r0']\n",
      "['time behavior', 'in', 'supercritical regime for diﬀerent values of r0']\n",
      "['agreement', 'with', 'branching process for small values of ϕ']\n",
      "['repeated connections', 'in', 'ego-centered networks of node']\n",
      "['epidemic threshold', 'be', 'through simulation-based analysis given as function of spectral radius of network']\n",
      "['si model', 'in', 'network satisfying']\n",
      "['ﬁ citation', 'in', 'democratic republic of congo near ebola river']\n",
      "['second scenario', 'represent', 'measles in england in vaccine era with subclinical infections']\n",
      "['simpler deterministic models', 'with', 'seasonal forcing']\n",
      "['local information based behavioral responses', 'reduce', 'prevalence of epidemic numericcitation']\n",
      "['epidemic threshold', 'reduce', 'ﬁnal fraction of recovered nodes']\n",
      "['stochastic sir epidemics', 'with', 'two levels of mixing']\n",
      "['enhancement', 'in', 'number of neighy']\n",
      "['other resources', 'curb', 'impact of disease']\n",
      "['diﬀerent individuals', 'lead to', 'diﬀerent numbers of contacts']\n",
      "['numericcitation l. bolzoni', 'analysis of', 'alternative culling strategies for eradication']\n",
      "['generic node', 'in', 'of terms of average messages']\n",
      "['sustained oscillations', 'in', 'population of individuals']\n",
      "['quasi-neutral competition', 'in', 'broad class of stochastic competition models']\n",
      "['proposed sparse structure learning method', 'design for', 'such occurs in propagation of infectious disease']\n",
      "['network topology', 'be', 'when dominated by local connections']\n",
      "['mark newman', 'have', 'data repository at university of michigan']\n",
      "['stochastic lattice gas model', 'in', 'scope of simple mean-ﬁeld approximation']\n",
      "['numericcitation m. barry', 'outbreak in', 'conakry']\n",
      "['numericcitation l. borio', 'group on', 'civilian biodefense']\n",
      "['numericcitation c. j. peters', 'introduction to', 'ebola']\n",
      "['models', 'with', 'unidirectional dynamics']\n",
      "['reasonable models', 'apply to', 'data on initial spread of disease']\n",
      "['mean-ﬁeld endemic infective probabilities', 'be', 'plotted in figure 1 as function of infection rate at two values of µ']\n",
      "['change', 'in', 'spreading rate']\n",
      "['similar analysis', 'with', 'that leads to equation']\n",
      "['individuals', 'in', 'diﬀerent compartments']\n",
      "['y-axis', 'scale by', 'theoretical scaling law of eq']\n",
      "['µk', 'satisfy', 'hypotheses of theorem 1']\n",
      "['r immune hosts', 'with', 'critically low level of immunity']\n",
      "['curves', 'scale for', 'comparison of peak times']\n",
      "['case node l', 'in-neighbor of', 'node k']\n",
      "['model', 'be', 'two-strain']\n",
      "['susceptible-infected-recovered sir epidemic model', 'in', 'one dimension']\n",
      "['inﬁnite', 'in', 'sis processes for values of β']\n",
      "['extinction time', 'would', 'indeed in limit of high transmission probabilities would exactly proportional to average eccentricity weighted by size of components']\n",
      "['outgoing probabilities', 'in', 'construction of kuulasmaa numericcitation']\n",
      "['individuals', 'be', 'initially susceptible']\n",
      "['our model', 'in', 'hamiltonian transition description contribution to birth of susceptible individual']\n",
      "['αn', '≥ with', 'α0']\n",
      "['change', 'in', 'fraction of infected individuals']\n",
      "['probability distribution', 'in', 'terms of initial distribution at time t']\n",
      "['model', 'with', 'tunable modularity']\n",
      "['model', 'with', 'tunable modularity']\n",
      "['small world model', 'with', 'p']\n",
      "['gillingham', 'address', 'correlated unobservables']\n",
      "['depending trend', 'in', 'opinion of society']\n",
      "['bifurcation', 'with', 'exponent 2 for m for m ≥ 2']\n",
      "['have spatio-temporal graphical model selection 3', 'in', 'limit of large populations with equal mixing rates']\n",
      "['its dynamics', 'can', 'can conceived in diﬀerent manners on networks leading to conservation of diﬀerent physical quantities']\n",
      "['ω', 'be', 'proved in detail in appendix c']\n",
      "['mean-ﬁeld model', 'with', 'small renormalized infection rate βmf']\n",
      "['i', 'depend on', 'proportion of i in same population']\n",
      "['i0', 'in', 'deﬁnition of reﬀ']\n",
      "['reduction', 'in', 'magnitude of gain']\n",
      "['red free half-edge pairs', 'with', 'free susceptible half-edge at vertex of degree k']\n",
      "['their geographical large-scale spread', 'way of', 'goals of work']\n",
      "['theoretical models', 'describe', 'spread of infectious diseases']\n",
      "['distributed random time', 'with', 'rate β i']\n",
      "['t', 'with', 'probability θ']\n",
      "['fluctuations', 'in', 'sirs model']\n",
      "['its corresponding average', 'normalize', 'rank of true source']\n",
      "['optimal infection sources', 'in', 'paper']\n",
      "['section 3.5', 'prove', 'convergence']\n",
      "['section', 'prove', 'convergence']\n",
      "['three approaches', 'commonly adopt in', 'analysis of propagation models']\n",
      "['epidemic model', 'in', 'discrete time kalbﬂeisch']\n",
      "['one ma jor challenge', 'in', 'very practical ﬁeld of disease control']\n",
      "['theorem 2.1', 'calculate', 'inﬂuence spread of single node under sir models']\n",
      "['numericcitation', 'part of', 'facebook network with 4039 nodes']\n",
      "['observations', 'with', 'large number of infected']\n",
      "['si1 i2r model', 'for 54', '0.2 dashed line']\n",
      "['numericcitation t. nedelea', 'work in', 'progress']\n",
      "['left ﬁgure', 'show information propagation over', 'time']\n",
      "['increase', 'in', 'as case of rggs']\n",
      "['role', 'in', 'emergence of infectious diseases']\n",
      "['proof', 'in', 'si models']\n",
      "['collisions', 't. give', 'evolution of process']\n",
      "['information diﬀusion process', 'in', 'network of coupled nodes']\n",
      "['number', 'in', 'model']\n",
      "['our model', 'see in', 'tra jectory of predicted fraction of infected individuals']\n",
      "['1000 nodes', 'with', \"node 's degree equal to 4 numericcitation\"]\n",
      "['infected people', 'infect', 'susceptible individuals with probability q connected directly to them within same time scale']\n",
      "['sir model', 'with', 'partial immunity']\n",
      "['two close values', 'in', 'vicinity of deterministic period']\n",
      "['spatial strucuture', 'be', 'when taken into account']\n",
      "['d1/4 initial infectious vertices', 'in', 'γ2']\n",
      "['previous studies', 'mainly focus on', 'spreading ability of individuals']\n",
      "['behaviour', 'in', 'pre periods']\n",
      "['same contact network', 'for measles', 'figure 4']\n",
      "['closed form solution', 'be', 'numericcitation']\n",
      "['single source', 'in', 'section iii']\n",
      "['normal prior distributions', 'with', 'standard deviations of 5']\n",
      "['former results', 'in', 'population']\n",
      "['bp', 'be', 'in general more accurate in classiﬁcation of individual marginals than in estimate of average epidemic size']\n",
      "['recognition rate', 'in', 'fig 7']\n",
      "['incidence peak', 'in', 'amplitude']\n",
      "['detailed proof', 'be', 'given there']\n",
      "['infection', 'spread as', 'generational']\n",
      "['infection', 'in', 'closed connected community']\n",
      "['epidemic threshold', 'in', 'numerical simulations of infection processes on complex networks']\n",
      "['two-type cbp', 'in', 'context of epidemic processes']\n",
      "['small region', 'in', 'where starting of optimal strategy can occur']\n",
      "['best prediction', 'in', 'more 40 of networks']\n",
      "['deviations', 'at', 'most in range of few percent']\n",
      "['dominant inﬂuenza virus', 'in', 'circulation']\n",
      "['dominant inﬂuenza virus', 'with', 'many countries']\n",
      "['rewiring probability', 'proportional to', 'degree of infected nodes']\n",
      "['discontinuous ﬁnal size distribution', 'in', 'siir model']\n",
      "['set', 'in', 'network']\n",
      "['ﬂuctuations', 'disappear by', 'law of large numbers']\n",
      "['outbreaks', 'in', 'vicinity of epidemic threshold']\n",
      "['attitude', 'in', 'face of vaccines']\n",
      "['consequences', 'in', 'spread of diseases']\n",
      "['consensus', 'in', 'favor of vaccine']\n",
      "['consensus', 'in', 'favor of vaccine']\n",
      "['extremists', 'in', 'favor of vaccination']\n",
      "['extremist', 'in', 'favor of vaccine']\n",
      "['extremist', 'in', 'favor of vaccination']\n",
      "['global children', 'in', 'branching process of particle']\n",
      "['further decrease', 'in', 'number of susceptible individuals']\n",
      "['dynamical process', 'randomly select', 'one infected node']\n",
      "['proof', 'in', 'sis model']\n",
      "['system', 'successfully reproduce', 'main features of observed time series for measles incidence']\n",
      "['total number', 'with', 'initial infectious degree xi ,0 of potential infectious contacts at beginning of epidemic number of acquaintances of initially infectious individuals']\n",
      "['pr', 'be', 'set']\n",
      "['ma jor development', 'in', 'application of network theory to real-world problems']\n",
      "['numericcitation cdc', 'center for', 'disease control']\n",
      "['deterministic processes', 'dominate', 'time series']\n",
      "['greater inhomogeneity', 'with', 'rather arbitrary distribution of number of contacts for diﬀerent persons']\n",
      "['work', 'verify', 'theoretical analysis of critical point']\n",
      "['complicomparison', 'with', 'real epidemiological data']\n",
      "['spread', 'in', 'two types of structured populations consisting of schools']\n",
      "['quantity ξs ir', 'deﬁnes', 'epidemic threshold of process']\n",
      "['463,658 single family homes', 'in', 'clark county tax assessors records']\n",
      "['sources', 'in', 'larg e networks']\n",
      "['insets', 'show', 'magniﬁed image of temporal fraction of infected individuals']\n",
      "['studied case', 'in', 'mathematical sites of zd lattice']\n",
      "['model', 'with', 'endemic components']\n",
      "['small increase', 'in', 'fraction of immunized individuals']\n",
      "['dramatic drop', 'in', 'number of recovered individuals']\n",
      "['speciﬁc type', 'in', 'limit of inﬁnite population']\n",
      "['trade', 'in', 'terms of import/export']\n",
      "['rate δ', 's at', 'rate γ']\n",
      "['values', 'determine by', 'self-consistent set of equations']\n",
      "['offspring distribution', 'in', 'case of interest']\n",
      "['population size', 'ﬂuctuates with', 'time']\n",
      "['school network', 'with', 'figure 7']\n",
      "['la', 'be', 'where long relative']\n",
      "['discontinuous transition', 'in', 'fraction of recovered individuals']\n",
      "['other two neighborhoods', 'with', 'measurable epidemic eﬀect']\n",
      "['corresponding convex hull', 'resort to', 'algorithm proposed in numericcitation']\n",
      "['discrete-time epidemic spreading model seir', 'with', 'autoinfection']\n",
      "['23 dk l', 'refer to', 'component of probability density found']\n",
      "['solid curves', 'compute by', 'numerical integration of eqs']\n",
      "['individuals', 'in', 'epidemic process']\n",
      "['likely infection path', 'in', 'g']\n",
      "['l estimating variability', 'in', 'models for recurrent epidemics']\n",
      "['identifying infection sources', 'in', 'network']\n",
      "['numericcitation s. p. sethi', 'control of', 'simple deterministic epidemic models']\n",
      "['change', 'in', 'fraction of susceptible individuals']\n",
      "['epidemic threshold', 'consequently correspond to', 'epidemic threshold of bond percolation problem']\n",
      "['three models', 'one for', 'each of diﬀerent methods selecting']\n",
      "['green solid lines', 'represent', 'case of unlimited vaccines']\n",
      "['eﬀective transmission rate', 'number of', 'new infectives']\n",
      "['exponential time', 'with', 'rate one for x']\n",
      "['clusters', 'allow for', 'exact analytical treatment of whole hierarchy of evolution equations']\n",
      "['eigenvector centrality', 'consistently outperform', 'other centrality measures usually independent of β']\n",
      "['such models', 'can', 'in deterministic setting can described with integral equations']\n",
      "['larger noise', 'lead to', 'signiﬁcant diﬀerences between two reduced model solutions']\n",
      "['mechanistic discrete-time seir model', 'with', 'autoinfection']\n",
      "['theorem 1', 'in', 'numericcitation']\n",
      "['5.2 reﬂect', 'with', 'additional values of τr presented in appendix tab']\n",
      "['general stochastic sir epidemic', 'equivalent to', 'markovian sir epidemic on complete graph']\n",
      "['numericcitation', 'in', 'proc']\n",
      "['model', 'with', 'bacteria']\n",
      "['comparison', 'with', 'predictions of original sir model deﬁned by set of diﬀerential equations']\n",
      "['formalism', 'accommodate', 'patterns of latency']\n",
      "['space structure', 'explicitly take into', 'account numericcitation']\n",
      "['→ 0', 'as 0', 'n → z0']\n",
      "['linear model', 'provide only', 'upper bound on marginal probabilities of marginals']\n",
      "['dependance', 'with', 'topological parameters of rrgs']\n",
      "['b. multiple infection sources', 'in', 'subsection']\n",
      "['people', 'have', 'global location in networks']\n",
      "['undirected graph', 'in', 'network']\n",
      "['modiﬁed model', 'in', 'sec 3']\n",
      "['numericcitation a. a. lashari', 'control of', 'vector borne disease with horizontal transmission']\n",
      "['predictability', 'in', 'global outbreaks of infectious diseases']\n",
      "['study', 'in', 'real networks numericcitation']\n",
      "['ﬁxed household', 'for n', 'epidemic spreads easier in independent model']\n",
      "['node', 'initially allocate', 'quintuple of stubs']\n",
      "['epidemic spreading', 'with', 'sir model on complex networks']\n",
      "['there has emerging interest', 'in', 'network modeling of epidemics numericcitation']\n",
      "['barab', 'asi', 'proceedings of national academy of sciences 104 7332 2007']\n",
      "['severity', 'in', 'populations of two host species']\n",
      "['case', 'call', 'incubation-latent model']\n",
      "['stochastic process', 'in', 'thus double contrast to evolution studied in mean ﬁeld approach']\n",
      "['epidemic model', 'in', 'two-layer network']\n",
      "['d t 3 similar result', 'with', 'that in theorem 2.1 for bond percolation model']\n",
      "['α0 ∈ γ3', '−', 'ej']\n",
      "['lloyd', 'heterogeneity in', 'epidemic models']\n",
      "['one layer', 'with', 'scale-free distribution of intralayer degrees k numericcitation']\n",
      "['plots', 'be', 'done in lin-log scale for better observation of structure of power spectra']\n",
      "['factor', 'in', 'lemma 4']\n",
      "['nodes can transition', 'with', 'probability γ']\n",
      "['corresponding integrals', 'in', 'exponential of equation']\n",
      "['real seasonal variations', 'in', 'disease activity of ili']\n",
      "['aic', 'measure of', 'model ﬁt']\n",
      "['solution θt', 'of existence', 'constant function θ ∞']\n",
      "['southern nevada water authority', \"'s program\", 'snwa']\n",
      "['random walks', 'converge to', 'those of super-brownian motion']\n",
      "['analysis', 'in', 'numericcitation']\n",
      "['household-school model', 'natural model in', 'case']\n",
      "['re', 'drop below', 'unity']\n",
      "['respondent', 'make', 'contact for three diﬀerent levels of contact']\n",
      "['scaling behavior', 'apply in', 'vicinity of epidemic threshold']\n",
      "['community structure', 'therefore will slow down', 'speed of sir spreading']\n",
      "['infectiousness parameter', 'in', 'our model']\n",
      "['schools', 'household from', '1 to n']\n",
      "['degree', 'in', 'binary version of weighted contact network numericcitation']\n",
      "['same scaling relationship', 'be', 'noted in numericcitation for simple sir']\n",
      "['numericcitation c. silva', 'regularization of', 'bang-bang optimal control problems']\n",
      "['susceptible individuals', 'in', 'contact networks']\n",
      "['n', 'be', 'where number of random simulations']\n",
      "['steady state', 'in computation', 'fraction of vaccinated individuals']\n",
      "['deal', 'with', 'limit of vaccine units']\n",
      "['epidemics', 'in', 'networks of spatially correlated three-dimensional root-branching structures']\n",
      "['existence', 'in', 'many interaction networks including networks of social contacts']\n",
      "['maximum control', 'be', 'in context intended as implementation of control policy at its maximum available rate']\n",
      "['other columns', 'represent', 'time evolution of epidemic process in speciﬁc realization']\n",
      "['autonomous system', 'consist of', 'eqs']\n",
      "['τ', 'scale behavior in', 'ﬁnite system of size n']\n",
      "['vertex', 'in', 'as numericcitation']\n",
      "['50018 zaragoza', 'institute for', 'biocomputation of complex systems']\n",
      "['ignorant people', 'be', 'individuals unaware of rumor']\n",
      "['k shell index ks', 'predict', 'average susceptibility of individuals']\n",
      "['nearest node', 'in', 'ˆs l − 1']\n",
      "['nearest node', 'in', 'ˆs l − 1']\n",
      "['dynamical properties', 'in', 'context of single networks numericcitation']\n",
      "['numericcitation r. anderson', 'biology of', 'infectious diseases']\n",
      "['numericcitation v. privman', 'in', 'phase transitions']\n",
      "['model', 'be', 'simple']\n",
      "['missing information', 'apply to', 'problem of epidemic forecasting']\n",
      "['degree', 'outperform', 'number']\n",
      "['changes', 'in', 'numericcitation']\n",
      "['results', 'in', 'case of bounded vertex degrees']\n",
      "['epidemic threshold', 'in', 'rrgs']\n",
      "['numericcitation', 'threshold of', 'sir model']\n",
      "['sir', 'epidemic on', 'random graph']\n",
      "['7 we', 'inclusion', 'ity space']\n",
      "['agreement', 'with', 'data shown in figure 2 of who data']\n",
      "['time slot tint', 'in', 'x t']\n",
      "['time-varying spreading rate β1', 'for control', 'state evolution with control without control']\n",
      "['sharp contrast', 'with', 'homogeneously mixed stochastic models']\n",
      "['important issue', 'in', 'theoretical understanding of biological processes']\n",
      "['sir dynamics', 'in', 'random networks']\n",
      "['single switching time τ ∗ s', 'with umax', 'umax for t ∈']\n",
      "['whole stochastic binary system', 'specify', 'pathsets for g']\n",
      "['population', 'in', 'numericcitation']\n",
      "['locatin g', 'in', 'ﬂuential nodes of complex networks']\n",
      "['abbreviation', 'with', 'for probability']\n",
      "['results', 'agree with', 'large accumulation of empirical evidence about role of plots/ﬁelds elongation on propagation of diseases on plants']\n",
      "['observed threshold', 'be', 'however for certain networks different from threshold for stability of nonlinear model']\n",
      "['graph adjacency matrix c', 'induce', 'instantaneous spread of contagion']\n",
      "['numericcitation k. diethelm', 'subroutine for', 'numerical solution of diﬀ citation s. heinzel']\n",
      "['pair', 'remove from', 'pool of free half-edges']\n",
      "['ﬁnite sum', 'in', 'last line of 3.13']\n",
      "['example', 'consider at', 'end of section 2.1']\n",
      "['conditioning', 'use', 'method of moments']\n",
      "['epidemic curve', 'in', 'extreme setting']\n",
      "['t0', 'on 0', 'x0 − xt']\n",
      "['estimates', 'increase', 'number of simulated samples']\n",
      "['node i', 'be', 'susceptible respectively']\n",
      "['adverse economic impact', 'in', 'parts of east asia']\n",
      "['immediate comparison', 'with', 'power spectra of simulations']\n",
      "['their propagation', 'in', 'complex network of neurons']\n",
      "['claim', 'apply', \"specialized form of gronwall 's inequality\"]\n",
      "['infected particle recovers', 'clear at', 'rate γ independently of everything else']\n",
      "['model', 'with', 'lower akaike information criterion']\n",
      "['festschrift', 'in', 'honor of ted harris']\n",
      "['fundamental growth rate β', 'essentially constant from', 'one region of world to another']\n",
      "['ni', 'be', 'where number of its infected neighbors']\n",
      "['as sis model', 'study previously by', 'gross et al']\n",
      "['decrease', 'in', 'fraction of vaccinated individuals']\n",
      "['ω ∗', 'increase in', 'fraction of immunized individuals']\n",
      "['corresponding product', 'to e', 'si']\n",
      "['information dynamics', 'shape', 'sexual networks of internet-mediated prostitution']\n",
      "['diﬃculty', 'with', 'equations']\n",
      "['ω', 'depend on', 'time ω']\n",
      "['stochastic fourth-order runge-kutta scheme', 'with', 'constant time step size']\n",
      "['ﬁnite', 'in', 'sirs model t']\n",
      "['curves', 'be', 'scaled in arbitrary units for comparison of peak times']\n",
      "['siir model', 'in', 'region']\n",
      "['brunet e', 'statistics at', 'tip of branching random walk']\n",
      "['substantial negative economic impact', 'in', 'parts of east asia']\n",
      "['sis model ρr', 'in', 'sir model for λ']\n",
      "['number', 'with', 'corresponding simulated new infected individuals in clay center']\n",
      "['sir model', 'with', 'infection']\n",
      "['ﬁrst four de ﬁnitions', 'repeat here for', 'convenience of reader']\n",
      "['previous work numericcitation', 'predict', 'eﬀect of order-four structure on epidemic dynamics']\n",
      "['rumors', 'in', 'network']\n",
      "['spectral radius', 'bind by', 'average degree of rgg']\n",
      "['terms', 'in', 'our rescaled system']\n",
      "['3.3 epidemic curve', 'in', 'section']\n",
      "['partial observation', 'in', 'early stage of dynamics']\n",
      "['epidemic spreading', 'in', 'sf networks']\n",
      "['mean-ﬁeld convergence results', 'use', 'only basic ode theory']\n",
      "['convergence results', 'use', 'only basic ode theory']\n",
      "['spreading process', 'in', 'transition from s to e']\n",
      "['behavior', 'with', 'those obtained from mean-ﬁeld approximation']\n",
      "['±', 'unrelated to', 'inverse widths of bi-gaussian λ ± given by eq']\n",
      "['focal point', 'thus remove', 'source of chaos']\n",
      "['infection attempts', 'in', 'coupled epidemic']\n",
      "['coordinate', 'transform', 'more transparent']\n",
      "['collapse', 'with', 'its steady state value in endemic state of disease']\n",
      "['s', 'with', 'respect to number of links l']\n",
      "['4.3 illustration', 'with', 'norovirus']\n",
      "['60 citation -1966', 'from datum', 'citation']\n",
      "['long-run endemic equilibrium', 'can', 'can sustained in extensions of traditional sir model']\n",
      "['b. most likely infection path sources estimator', 'in', 'subsection']\n",
      "['13 citation citation', 'bureau of', 'economic analysis']\n",
      "['n', 'be', 'where number of individuals with new strain']\n",
      "['non-linear term', 'in', 'sir model']\n",
      "['rules', 'with', 'limited vaccines for small network of size n']\n",
      "['random variables', 'with', 'empirical distribution of rescaled weights']\n",
      "['susceptibility', 'with', 'standard set of equations of sir dynamics']\n",
      "['center', 'be', 'universal']\n",
      "['center', 'be', 'universal']\n",
      "['inﬁnite population limit', 'with', 'assumptions of spatial homogeneity']\n",
      "['eﬀective herd immunity', 'in', 'stochastic model with value of v']\n",
      "['network epidemic models', 'with', 'two levels of mixing']\n",
      "['ﬁrst link', 'in', 'path']\n",
      "['epidemic size', 'increase', 'modularity']\n",
      "['authorities', 'preserve', 'topological features of sf networks']\n",
      "['individuals', 'in', 'network']\n",
      "['its latent period', 'with', 'probability p']\n",
      "['data', 'ignore ﬁrst at_time', '40 years of transient solution']\n",
      "['single focal point', 'dependence on', 'density of edges']\n",
      "['bistability', 'observe previously for', 'sis model numericcitation']\n",
      "['mechanisms', 'in', 'spread of infectious diseases']\n",
      "['bypass', 'fortunately possible for', 'whole class of quasineutral competition models']\n",
      "['c', 'be', 'when convex hull of at time t']\n",
      "['most likely infection path', 'in', 'subsection']\n",
      "['china', 'in', 'southern province of guangdong']\n",
      "['continuous-time markov chain', 'with', 'transition rates as in table 1']\n",
      "['line', 'represent', 'theoretical prediction of eq']\n",
      "['similar argument', 'explain', 'success of well-known bp methods in general network numericcitation']\n",
      "['play', 'in', 'prediction of large outbreaks6 ,7,8']\n",
      "['cholera transmission model', 'with', 'time-varying environmental force of infection']\n",
      "['hidden sirs model', 'with', 'time-varying environmental force of infection']\n",
      "['hidden sirs model', 'with', 'time-varying environmental force of infection']\n",
      "['social online networks', 'can have in', 'actual spreading of viral epidemic']\n",
      "['special reference', 'with mckendrick', 'a.g. studies on theory of continuous probabilities']\n",
      "['total number', 'cost of', 'treatment']\n",
      "['process', 'in', 'work']\n",
      "['model', 'study', 'information provided by model']\n",
      "['model', 'pick', 'γ small']\n",
      "['sir model', 'with', 'schematic diagrams in figs']\n",
      "['x ∈ b', 'for pλ', 'y inf u']\n",
      "['susceptible nodes', 'describe', 'individuals']\n",
      "['same method', 'predict', 'future evolution of epidemic outbreak from partial observations at early stage of dynamics']\n",
      "['result', 'have', 'has shown for sis model in numericcitation']\n",
      "['bd process', 'with', 'birth rate α set to 1']\n",
      "['important role', 'in', 'modeling of infectious diseases']\n",
      "['link', 'of', 'appearance in infection hierarchies numericcitation']\n",
      "['dotted lines', 'use as', 'guide to eyes']\n",
      "['results', 'in', 'numericcitation']\n",
      "['exponent one', 'with', 'distance to criticality φs']\n",
      "['b', 'display', 'result for configuration model with of points']\n",
      "['moore', 'in', 'small world networks']\n",
      "['order', 'in', 'models']\n",
      "['local correlations', 'diﬀer from', 'those of standard percolation']\n",
      "['two special classes', 'class with', 'exponentially distributed latent periods']\n",
      "['two special classes', 'class with', 'latent periods exponentially distributed']\n",
      "['epidemic threshold', 'strongly depend on', 'degree correlations of multilayer structure']\n",
      "['our analysis', 'consider', 'governing equations for stochastic seir model given by eqs']\n",
      "['networks', 'with', 'same degree distribution of clustering']\n",
      "['pei', 'search for', 'superspreaders of information in real-world social media']\n",
      "['big-v algorithm', 'still exhibit', 'mild sensitivity to removal of edges']\n",
      "['two rrgs', 'with', 'diﬀerent values of rectangle side length']\n",
      "['endemic steady state', 'lifetime of', 'lifetime of endemic steady state']\n",
      "['change', 'with', 'change of connection radius for diﬀerent values of rectangle elongation']\n",
      "['representation', 'in', 'terms of second-quantized bosonic operators']\n",
      "['second sum', 'zero in', 'probability']\n",
      "['nodes can transition', 'with', 'probability γ']\n",
      "['endemic model', 'with', 'peers at all']\n",
      "['population', 'in', 'subject of campaign']\n",
      "['transition diagram', 'depict in', 'upper panel of fig 1']\n",
      "['contact network epidemiology', 'model', 'epidemic spreading']\n",
      "['r', 'with', 'probability c']\n",
      "['probability', 'in', 'site']\n",
      "['n → ∞', 'population of', 'infected individuals']\n",
      "['grenfell numericcitation', 'overcome', 'intractability']\n",
      "['realistic epidemic simulations', 'have become', 'very important application of high-performance computing in epidemic predictions']\n",
      "['exponential growth', 'in', 'time φ']\n",
      "['cluster', 'eventually become', 'composed of recovered individuals only due to spontaneous immunization']\n",
      "['numericcitation', 'with', 'creation rate λ']\n",
      "['process', 'derive from', 'stochastic lattice gas model in scope of simple mean-ﬁeld approximation']\n",
      "['multiplicative noise', 'in', 'infectives']\n",
      "['5 conclusions phase transitions', 'in', 'modiﬁed versions of sirs model for epidemics']\n",
      "['two vertices', 'in', 'component of randomly chosen vertex']\n",
      "['evolution', 'with', 'varying initial fraction of infected individuals']\n",
      "['inﬂection points', 'in', 'number of recovered agents']\n",
      "['virus transmission', 'in', 'networks of computers']\n",
      "['school', 'with', 'given conﬁguration of households in it']\n",
      "['g', 'be', 'where largest component of g']\n",
      "['susceptible-infected-recovered model', '1department of', 'physics']\n",
      "['single peak', 'in', 'as fig 4']\n",
      "['our model', 'allow', 'modeling of population infected by multiple viruses']\n",
      "['increase', 'in', 'number of infected individuals']\n",
      "['node', 'in', 'state']\n",
      "['signiﬁcant contribution', 'in', 'problems of source de tection']\n",
      "['epidemic threshold', 'in', 'sir model']\n",
      "['death', 'result in', 'population']\n",
      "['host', 'have', 'population']\n",
      "['contract de-ac05-00or22725', 'with', 'us department of energy']\n",
      "['proof', 'rely on', 'operator semigroup techniques']\n",
      "['network', 'cycle through', 'edge in list']\n",
      "['susceptible nodes', 'become', 'infected at rate p through contact with their infected nearest neighbors']\n",
      "['transmission', 'in heterogeneity', 'value quite similar to that computed for strains of inﬂuenza numericcitation']\n",
      "['ﬁnite system sizes', 'in', 'as fig 4']\n",
      "['only difference', 'with', 'si model']\n",
      "['analytic results', 'with', 'numerical results obtained by numerical solution of master equation for si1 i2s model']\n",
      "['sis model', 'in', 'regular tree networks']\n",
      "['sir model', 'with', 'quantities deﬁned in section']\n",
      "['infections', 'in', 'norovirus scenario']\n",
      "['their corresponding constraints', 'be', 'summarized in table 1']\n",
      "['model analysis', 'result', 'much more challenging than one of classical size-structured models']\n",
      "['sars', 'be', 'ﬁrst identiﬁ citation in guongdong province of china numericcitation']\n",
      "['noticeable increase', 'in', 'fraction of infected individuals']\n",
      "['λdmp c', 'be', 'of λmfl c c c λmfl c closest to real epidemic threshold in about one-third of real-world networks']\n",
      "['solution', 'will reveal', 'important eﬀects of correlations in properties of epidemic spreading in multilayer networks']\n",
      "['10 3.3.1 continuous-time reed-frost model', 'in', 'continuous-time reed-frost model']\n",
      "['networks', 'be', 'by construction loop-less in limit of large network size']\n",
      "['could practical importance', 'in', 'setup of dynamic control strategies']\n",
      "['aij', 'be', 'where element of matrix a. degree']\n",
      "['r', 'recover', 'individuals']\n",
      "['r', 'recover', 'individuals']\n",
      "['continuous-time reed-frost', 'with', 'deterministic latent periods']\n",
      "['propagation processes', 'stop', 'numericcitation']\n",
      "['size', 'with', 'probability p k of ﬁrst cluster']\n",
      "['underlying small world network', 'for coeﬃcient', 'dashed line']\n",
      "['such collapse-driven extinctions', 'depend on', 'factors diﬀerent than those in endemic steady state of disease']\n",
      "['epidemic simulation problem', 'compute', 'ﬁrst n moments of random variable x']\n",
      "['individuals', 'represent by', 'vertices in graph g with edges corresponding to potentially infectious contacts']\n",
      "['square lattice', 'with', 'small world interaction rules']\n",
      "['network sis model', 'with', 'infection rate β']\n",
      "['our analysis', 'rely on', 'particularly assumption of non-interacting populations']\n",
      "['most initial conditions', 'with', 'very low number of infectives']\n",
      "['numericcitation c. y. kaya', 'method for', 'time-optimal switching control']\n",
      "['hbv', 'be', 'dna virus classiﬁed in virus family of hepadnaviridae']\n",
      "['vertical lines', 'point out', 'positions of numerical effective epidemic thresholds']\n",
      "['our proofs', 'also involve', 'couplings of evolution of total infective degree with suitable brownian motions']\n",
      "['models', 'with', 'constant delay such those in numericcitation']\n",
      "['description', 'be', 'in leading order in 1 √ n given by eq']\n",
      "['deterministic coexistence line', 'in', 'three-dimensional phase space i1 i2s']\n",
      "['sirs epidemic model', 'with', 'time delay']\n",
      "['nodes', 'with', 'large degree']\n",
      "['5 gain', 'in', 'power']\n",
      "['i. introduction', 'role in', 'modeling of infectious diseases']\n",
      "['grenfell', 'interplay in', 'childhood diseaeses']\n",
      "['model', 'describe', 'well-localized disease outburst']\n",
      "['a2', 'with', 'its counterpart xα t']\n",
      "['conditional probability ps', 'in time', 'x t']\n",
      "['infection', 'transfer from', 'infected site to neighboring sites']\n",
      "['important eﬀects', 'in', 'properties of epidemic spreading']\n",
      "['ﬁnal epidemic size', 'be', 'when small']\n",
      "['initial conditions', 'with', 'equal rates of deaths']\n",
      "['problem', 'with', 'diﬀerential equations']\n",
      "['epidemics', 'news/rumors in', 'social networks']\n",
      "['number', 'numericcitation with', 'corresponding new infected individuals in clay center given best estimated values for three contact levels']\n",
      "['its critical behavior', 'coincide with', 'that of static isotropic percolation on regular lattices numericcitation']\n",
      "['work', 'inspire', 'more research in long-time behavior of epidemic models']\n",
      "['re', 'in', 'parts such urban areas']\n",
      "['irsirs model', 'with', 'spontaneous return to infection']\n",
      "['new infected individuals', 'in', 'la gloria numericcitation']\n",
      "['θ a. infection-dominant model', 'in', 'case']\n",
      "['much eﬀort', 'predict ebola outbreak with', 'help of mathematical modeling numericcitation']\n",
      "['14 three lines', 'determine', 'topology of optimal extinction path on phase plane']\n",
      "['following ﬂow chart', 'show', 'basic procession of sir model numericcitation']\n",
      "['neighborhood', 'have', 'id']\n",
      "['sir epidemic model', 'with', 'pulse vaccination']\n",
      "['seir model', 'with', 'mortality']\n",
      "['variation', 'with', 'change of connection radius for diﬀerent values of rectangle elongation obtained by computational realizations of rrgs']\n",
      "['two sampled nodes', 'in', 'figure 3']\n",
      "['numericcitation m. keeling', 'disease in', 'humans']\n",
      "['simulations', 'most closely match', 'qualitative features of observed data']\n",
      "['process', 'be', 'simple']\n",
      "['probability', 'in', 'eq']\n",
      "['cluster equations', 'simplify under', 'natural assumption of translation invariant initial distribution']\n",
      "['propagation', 'use', 'high enough values of w']\n",
      "['deterministic model', 'be', 'for intermediate quantities of vaccine poor estimate of optimal strategy for more realistic stochastic case']\n",
      "['jan 1 1995', 'between archive', 'symmetrized snapshot of structure of internet at level of autonomous systems']\n",
      "['critical threshold', 'be', 'how much more underestimated in sir model of two-layer sf networks than in siir model']\n",
      "['average degree', 'in', 'selected subset of aps dataset']\n",
      "['measles metapopulation dynamics', 'in', 'england']\n",
      "['eigenvector centrality', 'perform best for', 'network']\n",
      "['eigenvalue centrality', 'perform best for', 'network']\n",
      "['eigenvalue centrality', 'perform best for', 'network']\n",
      "['eigenvalue centrality', 'perform best for', 'network']\n",
      "['eigenvalue centrality', 'perform best for', 'network']\n",
      "['eigenvalue centrality', 'perform best for', 'network']\n",
      "['in si1 i2s model', 'with', 'r1']\n",
      "['infected nodes', 'in', 'sis model']\n",
      "['numericcitation', 'be', 'eﬃcient']\n",
      "['transmission rate β', 'at_time', '2.4 day − 1']\n",
      "['houses', 'of combination', 'j']\n",
      "['convergence', 'in', 'mean-square']\n",
      "['numericcitation studies', 'of ¨', 'generalized reed-frost epidemic model']\n",
      "['secondary infection', 'of 100', 'p blue curve']\n",
      "['i. susceptible-infected-recovered model', 'in', 'sir model']\n",
      "['additive noise terms', 'in', 'equation']\n",
      "['rules', 'as', 'as spatial sir model with following modiﬁcations']\n",
      "['color online phase diagram', 'in', 'p-c space of parameters showing']\n",
      "['si models', 'in', 'homogeneous networks']\n",
      "['key observation', 'in', 'numericcitation']\n",
      "['terms', 'in', 'evolution equations']\n",
      "['one poisson process', 'with', 'rate λ']\n",
      "['number', 'in', 'network numericcitation']\n",
      "['part', 'in', 'model']\n",
      "['d1/3 φ0', 'd1/3 in', 'sense of coupling']\n",
      "['ni ,0', 'delete', 'initially infective vertices of degree 0']\n",
      "['network', 'of', 'adjacency matrix']\n",
      "['ns', 'be', 'where number of clusters of size s per site on lattice']\n",
      "['new additive noise term', 'with', 'variance diﬀerent from original noise process']\n",
      "['phase transition', 'in', 'λ']\n",
      "['phase transition', 'in', 'λ']\n",
      "['non-trivial phase transition', 'in', 'γ']\n",
      "['phase transition', 'in', 'γ']\n",
      "['phase transition', 'in', 'λ']\n",
      "['s', '− with', 'probability pi']\n",
      "['pair', 'with', 'infection starting at time 1']\n",
      "['pair', 'with', 'infection starting at time 1']\n",
      "['stochastic model', 'exhibit', 'less dramatic switching for smaller values of r0']\n",
      "['numericcitation d. koschutzki', 'in', 'network analysis']\n",
      "['results', 'with', 'those of stochastic sis model']\n",
      "['opinion', 'with', 'diﬀerent orientation approaches to opinion of neighbor']\n",
      "['poisson process', 'with', 'rate λ']\n",
      "['koopmans', 'transmission during', 'large outbreak in commercial poultry farms in netherlands']\n",
      "['network', 'be', 'still small world here']\n",
      "['sir process', 'in', 'random network with heterogeneous connectivity']\n",
      "['time', 'be', 'where discretized into unit intervals']\n",
      "['distribution', 'with', 'infection at time 0']\n",
      "['si term', 'in', 'sir model']\n",
      "['pre-vaccination pattern', 'be', 'for measles accurately reproduced by simple deterministic models with seasonal forcing']\n",
      "['two special classes', 'class of', 'markov models']\n",
      "['two special classes', 'class of', 'markov models']\n",
      "['next time step', 'with', 'probability λ']\n",
      "['step', 'with', 'probability q']\n",
      "['delﬁ 2 a. rachah', 'torres for', 'humans']\n",
      "['time', 'divide into', 'discrete slots']\n",
      "['increase', 'in', 'number of exposed individuals']\n",
      "['u1 ∈ z τ', 'root of', 'survived b 1 branching process']\n",
      "['infected 7', 'recover from', 'disease']\n",
      "['norovirus outbreak', 'in', 'hospital setting']\n",
      "['agreement', 'with', 'numericcitation']\n",
      "['strategy a', 'b. with', 'largest rewiring probability w']\n",
      "['roc area increase', 'in', 'ﬁrst stage of epidemics for prediction methods']\n",
      "['dashed line', 'predict by', 'eq numericcitation']\n",
      "['children', 'in', 'subsequent generation of branching process']\n",
      "['same relation', 'in', 'case of random initial conditions']\n",
      "['sis model', 'in', 'homogeneous networks']\n",
      "['susceptible individuals', 'with', 'topological indicators']\n",
      "['square lattice', 'on process', 'model']\n",
      "['healthy nodes', 'thus absorb into', 'infected cluster all at once']\n",
      "['densities', 'show in', 'fig 2 for two sets of parameter values']\n",
      "['rumors', 'in', 'network']\n",
      "['expansions', 'in', 'limit of high infection rates']\n",
      "['β', 'threshold of', 'network']\n",
      "['previous studies', 'have ignore', 'characteristic of information epidemics']\n",
      "['our method', 'give', 'seasonal epidemic data consisting of number of infectious individuals']\n",
      "['two features', 'increase after', 'march 31st in growth of new recognized cases']\n",
      "['our model', 'thus subsume', 'those studied in numericcitation']\n",
      "['single node', 'in', 'sirs model']\n",
      "['single node', 'in', 'sirswithvaccination model']\n",
      "['spreaders', 'in', 'non-monotonic models of diffusion processes such voter model']\n",
      "['birth', 'in', 'space of susceptible numbers for sir model']\n",
      "['epidemic spreading', 'in', 'scale-free networks']\n",
      "['si models', 'in', 'heterogeneous networks']\n",
      "['circles', 'for', '3-stage analysis mentioned in fig 2']\n",
      "['agents', 'reveal', 'eﬀect of biased movements']\n",
      "['eﬀective spreading rate', 'be', 'when constant over time']\n",
      "['eﬀective spreading rate β', 'be', 'when constant over time']\n",
      "['more uncertainty', 'in', 'prediction for fraction of susceptibles']\n",
      "['numericcitation s. lenhart', 'apply to', 'biological models']\n",
      "['autonomous figure 9', 'show', 'results on internet autonomous systems network']\n",
      "['randomizing eﬀect', 'in', 'small-world networks of watts type numericcitation']\n",
      "['nearest neighbor links', 'always present in', 'model']\n",
      "['cd', 'be', 'where constant characteristic of rgg in d-dimensional cube']\n",
      "['inset', 'show', 'zoom for ﬁrst case of eq']\n",
      "['behavior', 'r0', '1 in deterministic sir model']\n",
      "['large maximum eigenvalue', 'correspond to', 'small epidemic threshold in susceptible-infected-susceptible model numericcitation']\n",
      "['asymptomatics', 'also have', 'very intense infectious pressure']\n",
      "['asymptomatics', 'have', 'very intense infectious pressure']\n",
      "['identifying infection sources regions', 'in', 'large networks']\n",
      "['sir', 'show', 'same criticality as isotropic 3 percolation problem']\n",
      "['probability', 'in yn/f', 'n']\n",
      "['yn/f → 0', 'in', 'probability']\n",
      "['64 dqn dp n − → 1', 'in', 'probability']\n",
      "['3 − d/2', '→ in', 'probability']\n",
      "['our models', 'be', 'included in supplemental materials']\n",
      "['its variation', 'in', 'second line of eq']\n",
      "['figure 2', 'show', 'result of high movement case']\n",
      "['same argument', 'in', 'as proposition 4 of lalley']\n",
      "['k', 'be', 'number of contacts per unit time']\n",
      "['model', 'be', 'diﬀerent']\n",
      "['initial value problem', 'of solution', 'gn t']\n",
      "['i', 'be', 'number of red free half-edges at vertex i']\n",
      "['φr', 'be', 'negligible at beginning of process']\n",
      "['pronounced 3.5 year period', 'in', 'line with epidemiological data']\n",
      "['17 figures', 'figure', '1 persistence transition']\n",
      "['fraction', 'tend to', 'zero']\n",
      "['tra jectory', 'in', 'epidemic models']\n",
      "['rumor spreading', 'in', 'social networks']\n",
      "['individuals', 'in', 'z f function']\n",
      "['non-zero imaginary parts', 'in', 'regions of parameter space']\n",
      "['numericcitation e. hansen', 'journal of', 'mathematical biology 62']\n",
      "['color online', 'λc as', 'function of recovery rate µ on real-world networks']\n",
      "['second one', 'refer to', 'recovery process of individual i with recovery rate µ']\n",
      "['network', 'generate in', 'following way']\n",
      "['recovered node', 'in', 'state r']\n",
      "['epidemic peaks', 'in', 'our model']\n",
      "['periodic functions', 'with', 'same period t of limit cycle']\n",
      "['analogous results', 'in', 'simpler setting of mean-ﬁeld models']\n",
      "['observations', 'with', 'suﬃciently large number of infected']\n",
      "['spontaneous recovery', 'with', 'rate γ']\n",
      "['recovery', 'with', 'rate γ']\n",
      "['b', 'in', 'entire sample of real-world networks']\n",
      "['system', 'in', 'separated block form']\n",
      "['dashed curves', 'have', 'constant community size equal to average value of power law']\n",
      "['recovery rates', 'be', 'equal to one as in inﬁnite-population limit']\n",
      "['greatest heterogeneity', 'in', 'distribution of diagonal squares']\n",
      "['double-starred equality', 'follow from', '18 in eq']\n",
      "['average behavior', 'with', 'deterministic behavior of mean ﬁeld model']\n",
      "['relaxation time', 'depend on', 'initial conditions through initial density of susceptibles']\n",
      "['information available', 'involve', 'only fraction of overall population']\n",
      "['low order', 'in', 'normal form']\n",
      "['source location estimation problem', 'core of', 'thus determination of likelihood of observed epidemic realization initiated at source location θ']\n",
      "['model', 'with', 'memory']\n",
      "['numericcitation c. jiang', 'journal of', 'systems science 38']\n",
      "['fast time', 'in', 'evolution equations']\n",
      "['existence', 'in', 'many interaction networks including networks of social contacts']\n",
      "['neighboring node', 'in', 'ρ']\n",
      "['page 8', 'in', 'proceedings of 7th workshop on social network mining']\n",
      "['social networks', 'have', 'have used in previous studies']\n",
      "['white dots', 'represent', 'corner sites of accessible regions']\n",
      "['dots', 'represent', 'corner sites of accessible regions']\n",
      "['contact', 'with', 'other class of individuals']\n",
      "['u', 'use', 'center manifold equation given by eq']\n",
      "['u', 'use', 'center manifold equation given by eq']\n",
      "['individuals', 'in', 'branching process z b']\n",
      "['h5', 'n with', 'variance of exposed fraction on left hand side']\n",
      "['members', 'fall into', 'one of three classes']\n",
      "['time', 'lag for', 'mean ﬁeld model with multiplicative noise']\n",
      "['8 global clustering', 'deﬁned as', 'ratio of total number of closed triples to total number of connected structures with 3 nodes']\n",
      "['nonactive phase', 'with', 'three types of individuals']\n",
      "['model', 'show in', 'fig 1']\n",
      "['ﬁnite borel measure µ', 'against hµ', 'ψi for integral of ψ']\n",
      "['disease', 'will spread among', 'population']\n",
      "['two simple models', 'have', 'have extended in diﬀerent directions numericcitation']\n",
      "['recovery period', 'in', 'inﬂuence']\n",
      "['vaccinated agents', 'stay', 'pinned in opinion +2']\n",
      "['h. a. spreading dynamics', 'in', 'complex networks']\n",
      "['number', 'in', 'population']\n",
      "['number', 'in', 'population']\n",
      "['optimal control', 'on morton', 'advances in applied probability 6 622 635']\n",
      "['ﬁrst order', 'in', 'dt']\n",
      "['their lawns', 'desert', 'landscaping based upon size of converted area']\n",
      "['topology', 'eﬀects of', 'transition behavior to changes in network topologies']\n",
      "['networks', 'with', 'high level of clustering']\n",
      "['case fatality rate', 'give by', 'f']\n",
      "['2 sir model', 'in', 'as kitsak et al']\n",
      "['changes', 'in', 'reported number of cases']\n",
      "['increase', 'in', \"incidence peak 's amplitude\"]\n",
      "['r →', 'with', 'θt for t 6 ˆa0']\n",
      "['models', 'with', 'small birth rates']\n",
      "['stochastic eﬀects', 'be', 'quite prominent']\n",
      "['numericcitation f. vel', 'jas', 'physical review e 95 052315 2017']\n",
      "['individuals', 'in', 'infected stage']\n",
      "['equations', 'in', 'u']\n",
      "['model', 'in', 'sense']\n",
      "['model', 'with', 'unrestricted movements']\n",
      "['model', 'with', 'k']\n",
      "['only root', 'in', 'numericcitation']\n",
      "['distribution', 'reach maximum at', 'larger times']\n",
      "['possible information sources', 'in', 'sir model']\n",
      "['modules size', 'far smaller than', 'net-n n']\n",
      "['method', 'however provide', 'only rough approximation of structured population']\n",
      "['estimator', 'in', 'general graphs']\n",
      "['degree', 'however only reﬂects', 'nodes number of neighbors']\n",
      "['astrophysics e-print archive', 'snapshot at', 'level of autonomous systems']\n",
      "['obtained delay', 'of', 'in case indeed state-dependent type']\n",
      "['nodes', 'in', 'susceptible state']\n",
      "['independent noise processes', 'with', 'diﬀerent variance']\n",
      "['control', 'switch from', '0 to maximum rate umax']\n",
      "['kuperman', 'eﬀect in', 'epidemiological model']\n",
      "['susceptible-infected-recovered model', 'university of', 'calcutta']\n",
      "['numericcitation a. abakuks', 'policy for', 'epidemics']\n",
      "['difference', 'in', 'ability of starting node spread even at seven times']\n",
      "['possible sources', 'in', 'si model numericcitation']\n",
      "['perturbation', 'in', 'complex network of coupled nodes']\n",
      "['direct recruitment', 'in', 'sir epidemic process']\n",
      "['epidemic model', 'with', 'time-dependent transmission rate']\n",
      "['17 figures', 'figure', '1 epidemic persistence transition']\n",
      "['israel science foundation', 'ministry with', 'italy ministry of foreign aﬀairs']\n",
      "['spreading processes', 'be', 'how aﬀected by features of real networks']\n",
      "['stochastic lattice-gas model', 'with', 'asynchronous dynamics']\n",
      "['computation', 'in', 'steady state of our model']\n",
      "['configurations', 'give entirely in', 'terms of occupation numbers']\n",
      "['c', 'be', 'two survived nodes']\n",
      "['health', 'of ministry', 'outbreak in uganda']\n",
      "['b', 'in', 'as proof of theorem 2.6 of solution follow from lipschitz continuity of pi']\n",
      "['show', 'be', 'for tree networks closer to actual source than one identiﬁed by closeness centrality heuristic']\n",
      "['ιk ≡ xk', 'with', 'ℓk']\n",
      "['one multiplicative noise term', 'in', 'one equation']\n",
      "['diﬀerence', 'in', 'watering requirements of mesic landscaping']\n",
      "['m', 'for pair', 'expected square of jump satisﬁes']\n",
      "['plot', 'illustrate', 'interplay of hidden states of underlying compartmental model']\n",
      "['susceptible-infected-recovered model', 'model of', 'epidemics numericcitation']\n",
      "['resulting solution', 'with', 'data']\n",
      "['effect', 'lead to', 'decrease of effective epidemic threshold with recovery probability µ']\n",
      "['solutions', 'in', 'exponential growth phase of irsir model numericcitation']\n",
      "['similar methods', 'identify', 'inﬂuential spreader in social networks 9']\n",
      "['stochastic model reduction', 'be', 'done correctly']\n",
      "['symptoms', 'be', 'in ﬁnal stage followed by death numericcitation']\n",
      "['dynamics', 'describe suﬃciently by', 'eqs']\n",
      "['transmission rate β', 'at_time', '2.4 day − 1']\n",
      "['transmission rate β', 'at_time', '1.5 day − 1']\n",
      "['higher order moments', 'in', 'terms of lower order ones']\n",
      "['results', 'analogous to', 'those well-known for scalar case']\n",
      "['state', 'be', 'ﬁxed']\n",
      "['similar result', 'in', 'numericcitation']\n",
      "['other parameters', 'in', 'as figure 4a']\n",
      "['asynchronous sir model', 'in', 'universality class of standard percolation through behavior of cluster size distribution']\n",
      "['numericcitation k. wickwire', 'policy for', 'deterministic epidemics']\n",
      "['numericcitation h. behncke', 'control of', 'deterministic epidemics']\n",
      "['local neighbourhood', 'in', 'as lattices']\n",
      "['function', 'amount of', 'vaccine allocated to population']\n",
      "['sir model', 'have', 'has used successfully to model spreading of epidemics in various networks numericcitation']\n",
      "['klemm', 'searchability of', 'central nodes in networks']\n",
      "['facebook network', 'with', '4039 nodes']\n",
      "['issue', 'have', 'for animal epidemics has investigated experimentally for instance in case of equine inﬂuenza numericcitation']\n",
      "['one infection', 'have', 'strict obligate relationship with another']\n",
      "['nodes', 'be', 'signiﬁcant']\n",
      "['numericcitation c. silva', 'transaction on', 'automatic control 55 2488 2499']\n",
      "['elongated shapes', 'decrease', 'impact of epidemics on plant populations']\n",
      "['linear behavior', 'in', 'fig 1 for larger s']\n",
      "['5.2 inference performance', 'in', 'tab']\n",
      "['individuals', 'in', 'sir']\n",
      "['inﬂuence mi', 'deﬁned as', 'average size of population infected in spreading process originating at node i']\n",
      "['stochastic models', 'with', 'synchronous update numericcitation']\n",
      "['evolution', 'average', 'over 103 realizations of sirs process']\n",
      "['news/rumors', 'in', 'social networks']\n",
      "['local clustering', 'be', 'measure of interconnectivity between neighbours of given node']\n",
      "['stochastic ampliﬁcation', 'in', 'epidemics']\n",
      "['p', 'in', 'terms of original variables']\n",
      "['infected individuals', 'in', 'population']\n",
      "['infected individuals', 'in', 'population']\n",
      "['conservation', 'target', 'outdoor landscaping']\n",
      "['infection rate βr', 'in', 'rumor spreading simulations']\n",
      "['given half-edge', 'still free at', 'time t']\n",
      "['birth', 'in', 'space of susceptible numbers for sirs model']\n",
      "['individuals', 'in', 'generation tn']\n",
      "['pa', 'correctly represent', 'results of stochastic sirs process for small γ']\n",
      "['sir model', 'use', 'method']\n",
      "['two models', 'dynamics of', 'in opposite limit drastically diﬀerent']\n",
      "['susceptibility', 'thus also diﬀers from', 'one to another']\n",
      "['repeat', 'in', 'ﬁrst generation of branching process obtain']\n",
      "['good agreement', 'with', 'data']\n",
      "['importance', 'in', \"formation of people 's opinions\"]\n",
      "['network', 'at', 'most l − 1']\n",
      "['change', 'in', 'interest level of population']\n",
      "['percolation theory', 'describe', 'steady state of sir model on complex networks']\n",
      "['our results', 'diﬀer from', 'those previously obtained for time-optimal problem in speciﬁc epidemic contexts']\n",
      "['populations', 'less susceptible to', 'epidemic outbreaks']\n",
      "['can three possible states', 'in', 'rumor spreading model']\n",
      "['d', 'be', 'where average in-degree of reduced graph']\n",
      "['corresponding transition', 'subtler in', 'stochastic model']\n",
      "['detecting sources', 'in', 'networks']\n",
      "['simulated observed data', 'show in', 'right plot of figure 2']\n",
      "['increase', 'induce', 'increase in number of exposed individuals']\n",
      "['march 31st', 'in', 'growth of new recognized cases']\n",
      "['similar results', 'have', 'interestingly have recently obtained in models of time-varying networks characterised by correlated topological features induced by reinforcement of speciﬁc ties numericcitation']\n",
      "['inﬂuenza epidemics', 'use', 'search engine query data']\n",
      "['locations', 'in', 'y for x ∈ firstly']\n",
      "['deterministic case', 'in', 'limit of large population sizes']\n",
      "['limiting fraction', 'so θk', 'θ ∈ numericcitation']\n",
      "['disease', 'lose', 'immunity against disease-causing pathogen']\n",
      "['lemma 3.2', 'with', '˜αn yield for ε']\n",
      "['4.19 sup', 'lemma with', '˜αn yield for ε']\n",
      "['running time', 'in', 'seconds for live journal network']\n",
      "['dominance', 'by xx', 'position of initial particles of our epidemic model']\n",
      "['sir epidemic model', 'in', 'section 3']\n",
      "['sir epidemic model', 'in', 'section 3']\n",
      "['particles', 'for x', 'number mx of healthy start with']\n",
      "['continuous sample paths', 'in', 'd']\n",
      "['behavior', 'in', 'higher dimension']\n",
      "['behavior', 'in', 'figs 3']\n",
      "['model', 'with', 'vital dynamics']\n",
      "['model', 'with', 'vital dynamics']\n",
      "['diﬀerences', 'most signiﬁcant for', 'intermediate ranges of vaccine']\n",
      "['th', 'be', 'ﬁrst repeated index in sequence of households']\n",
      "['classical swine fever', 'in', 'netherlands numericcitation']\n",
      "['state', 'with', 'xi']\n",
      "['exact day', 'might', 'in other words might aﬀected by error of several days']\n",
      "['particular point', 'in', 'time']\n",
      "['performance', 'in', 'description of steady states']\n",
      "['memory eﬀects', 'of', 'model with memory for diﬀerent values of ﬁnite time t']\n",
      "['local neighbourhood', 'in', 'as lattices']\n",
      "['epidemic curve', 'with', 'ﬁxed mean']\n",
      "['front yards', 'be', 'turf']\n",
      "['transparent method', 'transformation into', 'quantum language']\n",
      "['critical value τc', 'determine by', 'underlying contact process dynamics of simple sis model']\n",
      "['time', 'discretize into', 'unit intervals']\n",
      "['result', 'will', 'will proved in lemma 3 in section vi']\n",
      "['result', 'will', 'will proved in lemma 4 in section vi']\n",
      "['sir dynamics', 'in', 'random networks with heterogeneous connectivity']\n",
      "['numericcitation m. barth', 'elemy', 'phys']\n",
      "[\"r'enyi g graphs numericcitation\", 'those with', 'local household structure numericcitation']\n",
      "['epidemic model', 'with', 'exposure-dependent severities']\n",
      "['longworth', 'analysis of', 'hpai control in netherlands i']\n",
      "['model', 'in', 'study']\n",
      "['nodes', 'in', 'b']\n",
      "['recovery probability', 'be', 'when large']\n",
      "['framework', 'involve', 'large scale simulation of contagion spreading processes from set of potential source locations']\n",
      "['complete convergence theorem', 'hold for', 'contact processes on open clusters of z d × z']\n",
      "['present results', 'of', 'could practical importance in setup of dynamic control strategies']\n",
      "['theoretical results', 'substitute', 'eq']\n",
      "['numericcitation c. jiang', 'control of', 'sars epidemics']\n",
      "['numerical integration', 'with', 'curves representing in population']\n",
      "['rrg', 'connect', 'changes as sigmoid function with increase of connection radius in similar way as in case of rggs']\n",
      "['term', 'with', 'm equal to m ′']\n",
      "['results', 'in', 'numericcitation']\n",
      "['infectious vertices', 'die out with', 'probability one']\n",
      "['sir model', 'in', 'multiplex system composed by two networks']\n",
      "['case', 'ﬁrst', 'case']\n",
      "['combined eﬀect', 'in', 'observation of many short loops']\n",
      "['new infected individuals', 'of number', 'numericcitation with corresponding new infected individuals in clay center given best estimated values for three contact levels']\n",
      "['characterization', 'in', 'ensemble of multilayer networks']\n",
      "['simulated epidemic curve', 'in', 'model']\n",
      "['eq', 'in', 'case']\n",
      "['change', 'in', 'relative amplitude of main stochastic peaks']\n",
      "['sir model', 'with', 'vital dynamics to abel type equation']\n",
      "['proof', 'in', 'appendix']\n",
      "['has similarity', 'in', 'evolution of epidemics infect']\n",
      "['complete graph', 'in', 'thermodynamic limit of inﬁnite population size']\n",
      "['susceptible individuals', 'of number', 'umax to isolation problem']\n",
      "['corresponding tradeoﬀ', 'in', 'context of deterministic sir model']\n",
      "['deterministic sir model numericcitation', 'system of', 'coupled diﬀerential equations for modeling']\n",
      "['plot', 'in', 'bottom row of figure 3']\n",
      "['plot', 'in', 'bottom row of figure 5']\n",
      "['time', 'be', 'standard']\n",
      "['technique', 'apply to', 'large ﬂuctuations of multiscale problems17']\n",
      "['infected individuals', 'number of', 'n at time t']\n",
      "['covariates', 'show with', 'lag of three weeks']\n",
      "['control', 'non-singular', 'namely function ψ vanishes only in isolated points']\n",
      "['numericcitation buldyrev s. v', 'cascade of', 'failures in interdependent networks']\n",
      "['recovered individuals', 'in', 'present model']\n",
      "['color scale', 'indicate', 'average epidemic size e corresponding to particular allocation of vaccine']\n",
      "['evolution', 'with', 'vital dynamics']\n",
      "['evolution', 'with', 'vital dynamics']\n",
      "['1 −', 'in', 'time']\n",
      "['time τ', 'in', 'x t']\n",
      "['d', 'in', 'time']\n",
      "['second equation', 'in', 'eq']\n",
      "['stochastic fourth-order runge-kutta integrator', 'with', 'constant time step size']\n",
      "['rapid roll-over', 'in', 'growth rate of recognized cases']\n",
      "['autoinfection', 'with model', 'seir']\n",
      "['numericcitation a. abakuks', 'advance in', 'applied probability 6 494 511']\n",
      "['numericcitation r. morton', 'advance in', 'applied probability 6 622 635']\n",
      "['study', 'in', 'numericcitation de']\n",
      "['large population sizes', 'with', 'ever increasing role of stochastic ﬂuctuations as n decreases']\n",
      "['many assumptions', 'in', 'epidemic model']\n",
      "['numericcitation a. potapov', 'proceed of', 'royal society b 279 3139 3145']\n",
      "['r. competing activation mechanisms', 'in', 'epidemics on networks']\n",
      "['theorem 1', 'in', 'numericcitation']\n",
      "['epidemic model', 'with', 'infector-dependent severity']\n",
      "['period', 'conﬁrmed by', 'fourier analysis of time correlation functions']\n",
      "['r. susceptible nodes', 'in', 'contact with infected individuals']\n",
      "['changes', 'in', 'network topologies']\n",
      "['school network', 'show in', 'fig 6']\n",
      "['apollonian network', 'show', 'hysteresis']\n",
      "['susceptible-infected-recovered epidemic model m. saeedian ,1 m. khalighi ,1 n. azimi-tafreshi ,2 g. r. jafari ,1', 'institute for', 'advanced studies 3school of biological sciences']\n",
      "['near-criticality', 'discuss in', 'numericcitation']\n",
      "['conjecture', 'in', 'numericcitation']\n",
      "['analogous result', 'in', 'numericcitation']\n",
      "['diﬀerences', 'in', 'epidemic models']\n",
      "['behavior', 'survive', 'population of found one in toy networks']\n",
      "['spreading inﬂue nce', 'in', 'complex networks']\n",
      "['shortest path', 'in', 'graph g']\n",
      "['random network', 'with', 'heterogeneous connectivity']\n",
      "['presence', 'in', 'adoption dynamics']\n",
      "['nodes', 'with', 'degree k ≤ 2']\n",
      "['strain 1', 'with', 'strain 2 of i1 κ1 i1 recovery of i2 κ2 i2 table i. transition rates']\n",
      "['power-law degree distributions', 'be', 'have observed in various networks numericcitation']\n",
      "['vertices', 'in', 'graph g with edges corresponding to potentially infectious contacts']\n",
      "['numericcitation c. y. kaya', 'journal of', 'optimization theory']\n",
      "['random graph', 'may', 'may coupled to branching model with high probability']\n",
      "['connection', 'with', 'numericcitation']\n",
      "['epidemic spread', 'in', 'metapopulation model']\n",
      "['collection', 'in', 'v from time 1 to t']\n",
      "['analysis', 'have provide', 'crucial insights in to full nonlinear process']\n",
      "['mathematical formalism', 'describe', 'spread of infectious diseases']\n",
      "['discrete diﬀusion equation', 'with', 'diﬀusion coeﬃcient equal to size of infected population']\n",
      "['pioneering essay diﬀerent vaccines', 'prevent', 'long list of infectious diseases']\n",
      "['infected probability', 'average over', '105 sir realizations']\n",
      "['parameters', 'in', 'models']\n",
      "['grenfell', 'interplay between', 'determinism']\n",
      "['σi', 'mean', 'diﬀusion rate of infected agents']\n",
      "['individuals', 'be', 'infertile']\n",
      "['k. j. pitt', 'algorithm for', 'direct solution of optimal control problems']\n",
      "['infectious path', 'with', 'length n']\n",
      "['israel ministry', 'with', 'italy ministry of foreign aﬀairs']\n",
      "['most networks', 'heterogeneous in', 'reality']\n",
      "['childhood epidemics', 'to dynamics', 'citation']\n",
      "['susceptible-infected-recovered-susceptible sirs model numericcitation', 'more closer to', 'fmd']\n",
      "['study', 'with', 'larger network numericcitation']\n",
      "['long time', 'in', 'epidemic phase']\n",
      "['contact', 'with', 'infected individuals']\n",
      "['state', 'with', 'larger ri']\n",
      "['probability 1 − µ', 'with', 'nodes in other groups']\n",
      "['value', 'with', 'peak value of sir component multiplied by gain parameter']\n",
      "['simulation scenario', 'in', 'which for models']\n",
      "['transmission probabilities', 'vary over', 'time']\n",
      "['system', 'more prone to', 'disease spreading']\n",
      "['local connections', 'with', 'small world probability']\n",
      "['epidemic process', 'spread slowly at', 'early time']\n",
      "['number', 'in', 'epidemic spreading on arbitrary network']\n",
      "['snwa', 'of', 'citation drought declaration']\n",
      "['johne', 'have', 'disease numericcitation']\n",
      "['two infectious stages', 'in', 'our modiﬁed metapopulation sir model']\n",
      "['such attributes', 'have', 'critical eﬀects']\n",
      "['value', 'slightly less than', 'that observed from 1-d original model']\n",
      "['regular tree', 'with', 'inﬁnitely many levels']\n",
      "['u', 'recover in', 'second time slot']\n",
      "['recovered nodes', 'ﬁnal fraction of', 'above which ﬁnite numericcitation']\n",
      "['proof', 'in', 'numericcitation']\n",
      "['susceptible individuals', 'with', 'probability q connected directly to them within same time scale']\n",
      "['two-point distribution', 'with', 'probability mass fraction infected at 0']\n",
      "['survival probabilities', 'in', 'infections transmitted by members of its own species']\n",
      "['sir model', 'with', 'population turnover numericcitation']\n",
      "['similar ﬁnite size scaling effects', 'have', 'have observed in other coevolving infection models']\n",
      "['joint probability', 'β at', 'site y']\n",
      "['joint probability', 'β at', 'site w']\n",
      "['ﬁrst term', 'in', 'right side of eq']\n",
      "['cosine terms', 'in', 'β with eqs']\n",
      "['data time series', 'between ln', 'σ']\n",
      "['spread', 'in', 'affected countries']\n",
      "['t', 'rate in', 'additive-multiplicative hazard model höhle et al']\n",
      "['rumor spreading model', 'with', 'βr']\n",
      "['susceptibility', 'in', 'rumor spreading model']\n",
      "['vertices', 'with', 'l1 norm']\n",
      "['netherlands numericcitation', 'epidemic of', 'foot-and-mouth']\n",
      "['numericcitation m. hvistendahl', 'race across', 'u.s. midwest']\n",
      "['numericcitation c. jiang', 'control base on', 'cybernetics']\n",
      "['higher epidemic size', 'in', 'sis models']\n",
      "['counters', 'account for', 'deterministic recovery']\n",
      "['simple random graph', 'with', 'low average connectivity']\n",
      "['nodes', 'in', 'network increases']\n",
      "['14 supplementary information', 'with', 'critical values obtained by model']\n",
      "['analysis', 'introduce', 'variable infectious periods']\n",
      "['sum', 'in', 'time']\n",
      "['vector', 'in', 'similar ﬂavor as in linear regression']\n",
      "['adaptive networks leah b. shaw department', 'on epidemic', 'col lege of wil liam']\n",
      "['solution', 'in', 'ﬁrst equation']\n",
      "['total variance', 'in', 'system']\n",
      "['total variance', 'in', 'system']\n",
      "['color-bar values', 'in', 'fig 10']\n",
      "['color-bar values', 'have', 'have normalized by 10 − 3']\n",
      "['color-bar values', 'have', 'have normalized by 10 − 3']\n",
      "['color-bar values', 'have', 'have normalized by 10 − 4']\n",
      "['color-bar values', 'have', 'have normalized by 10 − 4']\n",
      "['difference', 'in', 'variation of effective epidemic threshold for different networks']\n",
      "['sars', 'number', 'classical epidemiological measure associated with reproductive power of disease']\n",
      "['sudan virus', 'in', 'uganda']\n",
      "['epidemic dynamics', 'in', 'social contact networks']\n",
      "['given population results', 'in', 'signiﬁcant diﬀerences for optimal allocation of vaccine']\n",
      "['n', 'with', 'variance of exposed fraction on left hand side']\n",
      "['diﬀerences', 'in', 'optimal allocation of vaccine for stochastic model']\n",
      "['decrease', 'with', 'recovery probability µ']\n",
      "['recovered nodes', 'in', 'sir model']\n",
      "['our theoretical predictions', 'illustrate', 'novel feature about dynamics of adaptive networks in sir models']\n",
      "['tree networks', 'with', 'inﬁnitely many levels']\n",
      "['node', 'in', 'infected state']\n",
      "['giant susceptible cluster', 'exist at', 'given time']\n",
      "['speed-up', 'dependent on', 'network structure']\n",
      "['network', 'with', 'n nodes']\n",
      "['nodes', 'in', 'network']\n",
      "['sir model', 'eﬀect of', 'intermittent social distancing strategy on propagation in adaptive complex networks']\n",
      "['sir', 'relate to', 'percolation phenomena']\n",
      "['node i', 'of rate', 'probability']\n",
      "['independent events', 'with', 'probability β']\n",
      "['probability', 'for i', 't']\n",
      "['pr', 'i for', 'probability']\n",
      "['bernstein', 'by', 'inequality for sums of bounded independent centred random variables']\n",
      "['system', 'have', 'linear part']\n",
      "['µ', 'r. by', 'arguments of preceding paragraph']\n",
      "['particles', 'in', 'ball of radius 3c1 k1/6']\n",
      "['colorado river water', 'be', 'how shared']\n",
      "['k-shell number', 'generally outperform by', 'eigenvector centrality']\n",
      "['distribution', 'have', 'two separated regions']\n",
      "['λh', 'edge to', 'each of other vertices corresponding']\n",
      "['s i r model', 'use throughout', 'paper']\n",
      "['diﬀerent recovery times τr', 'in', 'months']\n",
      "['new variables', 'substitute into', 'eqs']\n",
      "['numericcitation report', 'estimate', 'reproduction numbers']\n",
      "['susceptible nodes', 'infect at', 'rate p']\n",
      "['infected nodes', 'heal at', 'rate q']\n",
      "['model', 'be', 'discrete']\n",
      "['initial disease outbreak', 'correctly capture by', 'reduced model']\n",
      "['gronwall', 'of', 'inequality']\n",
      "['t gronwall', 'have', 'inequality']\n",
      "['recovered nodes', 'in', 'sir model']\n",
      "['conclusions', 'in', 'prospective analysis of data']\n",
      "['k', 'with', 'probability k number of school children']\n",
      "['sis model', 'with', 'population turnover numericcitation']\n",
      "['we can neglect', 'in', 'regime t τ s']\n",
      "['φ d1/3', 'in', 'sense of coupling']\n",
      "['numericcitation d. stauﬀer', 'introduction to', 'percolation numericcitation d. j. watts']\n",
      "['instance', 'in', 'numericcitation']\n",
      "['our algorithm', 'can', 'can extended to virtually such compartment model']\n",
      "['π', 'be', 'where probability measure of i.i.d exponential times with rate λθ']\n",
      "['infection', 'in', 'population moving on zd']\n",
      "['sir model', 'in', 'single multilayer network 3.1']\n",
      "['u', 'remain', 'infected for i steps']\n",
      "['node', 'pass', 'infection on to others']\n",
      "['colizza', 'in', 'heterogeneous networks']\n",
      "['signiﬁcant variance', 'in', 'home density']\n",
      "['tree networks', 'with', 'inﬁnitely many levels']\n",
      "['models', 'with', 'externally applied changes in structure']\n",
      "['castellano', 'mechanism in', 'epidemics on networks']\n",
      "['f2 ≤', 'with', 'constant l']\n",
      "['epidemic duration', 'in', 'sir models']\n",
      "['system', 'have', 'asymptotic behavior']\n",
      "['real edges', 'in', 'network']\n",
      "['models', 'with', 'ﬁxed total population size numericcitation']\n",
      "['irregular dynamics', 'be', 'why observed in corresponding stochastic system']\n",
      "['mapping', 'with', 'annealed approximation predictions of epidemic spreading of sis model on multilayer networks']\n",
      "['typical individual', 'discuss', 'brieﬂy in section 6']\n",
      "['nodes', 'with', 'positive opinion state']\n",
      "['recovered nodes', 'of r', 't']\n",
      "['sites', 'in', 'epidemic process with rate γ1']\n",
      "['numericcitation uganda ministry', 'outbreak of', 'ebola']\n",
      "['numericcitation uganda ministry', 'outbreak in', 'uganda']\n",
      "['epidemic variability', 'in', 'complex networks arxiv']\n",
      "['period', 'of', 'magnitude']\n",
      "['branching process', 'in', 'much same way as early stages of epidemic']\n",
      "['following lemma', 'justiﬁes', 'time change in section 3']\n",
      "['oscillatory phase', 'in', 'regime of slow driving through introduction of new susceptible individuals']\n",
      "['reﬂection', 'with', 'similar adoption times']\n",
      "['number', 'in', 'state x']\n",
      "['multilayer networks', 'with', 'arbitrary number of links across layers']\n",
      "['basic reproduction ratio r0', 'in', 'models for infectious diseases']\n",
      "['arbitrary networks', 'over spread', 'namely sis']\n",
      "['bond percolation', 'with', 'arbitrary number of links across layers']\n",
      "['second part', 'from', 'random selection of k1 neighbors']\n",
      "['epidemic extinction process', 'in', 'generalized sis model']\n",
      "['system', 'with', 'separated stable']\n",
      "['intertwined sis model', 'in', 'numericcitation']\n",
      "['robust information source estimator', 'with', 'sparse observations']\n",
      "['numericcitation c. w. gardiner', 'handbook of', 'stochastic methods']\n",
      "['empirical study', 'in', 'proceedings of fifteenth conference on uncertainty']\n",
      "['1 1 introduction', 'in', 'face of increasing climate variability']\n",
      "['dynamical vaccination', 'thus make', 'more eﬀective usage of available vaccines']\n",
      "['conditional probability', 'in', 'discrete time']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['their absence', 'in', 'nucleus of con']\n",
      "['j. f. critical phenomena', 'in', 'complex networks']\n",
      "['epidemic model', 'faculty of', 'science']\n",
      "['epidemic model', 'faculty of', 'science']\n",
      "['linear regression', 'with', 'ﬁxed eﬀects']\n",
      "['analytical expression', 'in', 'arbitrary number of layers']\n",
      "['model', 'with', 'nonmarkovian disease dynamics']\n",
      "['model', 'with', '2700 × 2700 nodes']\n",
      "['k-shell number', 'be', 'generally outperformed']\n",
      "['ranking spreaders', 'in', 'complex networks']\n",
      "['model framework', 'in', 'section 2']\n",
      "['high-dimensional sis', 'with', 'random edge weights']\n",
      "['multivariate normal random walk proposal distribution', 'in', 'ﬁnal run of pmmh algorithm']\n",
      "['ω0', 'be', 'where vaccination probability at beginning of process']\n",
      "['authors', 'get', 'measures of local reproductive rates of infection']\n",
      "['k-jordan center set', 'be', 'solution']\n",
      "['exact program', 'to', 'application date']\n",
      "['connection', 'with', 'standard sir model']\n",
      "['s', 'be', 'where connected to t']\n",
      "['population', 'for model', 'infectious i evolving under stochastic dynamics']\n",
      "['large t', 'at o', 't one can approximate w θ due sharpness of front to leading order for large near front']\n",
      "['solution', 'approach', 'asymptotic stable node']\n",
      "['results', 'be', 'how organized in two subsections']\n",
      "['may placed investment', 'in', 'view of potential pandemic']\n",
      "['periodic boundary condition', 'have', 'has used in simulation']\n",
      "['simplest branching process', 'hence call', 'birth-death process']\n",
      "['new noise-induced phenomena', 'in', 'generic stochastic systems']\n",
      "['its numerical resolution', 'study in', 'numericcitation']\n",
      "['one contiguous outbreak', 'with', 'probability 1 − ν − 1']\n",
      "['function', 'as r', 'ba networks for diﬀerent attachment number m']\n",
      "['economics 18', 'in note', 'department of applied mathematics']\n",
      "['process', 'will', 'will referred to as maximal load epidemic']\n",
      "['department', 'with', 'center for control']\n",
      "['nonzero value', 'in', 'bimodal part of epidemic phase diagram']\n",
      "['case hiss', 'may become', 'inferior to dmp']\n",
      "['c', 'show', 'outcome for erdős-rényi model']\n",
      "['joint outbreak size distribution ﬁrst', 'become', 'relevant for two-type cbp']\n",
      "['dynamical systems', 'in', 'spatial eﬀects numericcitation']\n",
      "['other hand', 'on similarity', 'computer science propagation of virus']\n",
      "['models', 'with', 'diﬀerent assumptions on values of φs']\n",
      "['immune-admitting model', 'go to', 'steady state']\n",
      "['p', 'revent', 'emergence of antiviral drug resistance in hsv-2']\n",
      "['branching random walk', 'be', 'often referred to as branching envelope of epidemic']\n",
      "['k-jordan center set', 'in', 'g']\n",
      "['contribution', 'in', 'context of disease transmission model']\n",
      "['models', 'arise from', 'mean ﬁeld analysis of probabilistic models']\n",
      "['contact', 'with', 'average of km others']\n",
      "['models', 'be', 'then introduced']\n",
      "['power-law degree distributions', 'be', 'recently have observed in various networks numericcitation']\n",
      "['origin', 'receive', 'infectious independent of p0 ,0 infectious signals']\n",
      "['choosing t large enough forces close agreement', 'in', 'maximum eigenvalues of corresponding production operators']\n",
      "['infected x', 'with', 'x']\n",
      "['network connections', 'of importance', 'ideia']\n",
      "['process', 'with', 'rate γ0']\n",
      "['process', 'with', 'rate γ1']\n",
      "['process', 'with', 'rate γ0']\n",
      "['has studied epidemic model', 'in', 'literature numericcitation']\n",
      "['sim ulation', 'verify', 'effectiveness of proposed estimators']\n",
      "['dramatic changes', 'in', 'behavior of magnitudes at steady state']\n",
      "['infection process', 'with', 'rate β']\n",
      "['node k', 'in', 'state s at time t1']\n",
      "['gps', 'coordinate', 'embedded in them']\n",
      "['calculations', 'in', 'numericcitation']\n",
      "['temporal evolution', 'with', 'local vaccination for er network']\n",
      "['φ', 'be', 'average number of shortcuts per bond in lattice']\n",
      "['average mean error distances', 'with', 'error bars of 95 conﬁdenc interval for various networks under different infection spreading models']\n",
      "['average mean error distances', 'with', 'error bars of 95 conﬁdenc interval for various networks under different infection spreading models']\n",
      "['can infected multiple times', 'in', 'sis model']\n",
      "['qmf method', 'capture', 'dynamical correlations among states of neighbors']\n",
      "['sir epidemic model', 'with', 'nonlinear incidence']\n",
      "['sir case', 'to approach', 'citation']\n",
      "['optimal control', 'evolution without', 'control for sis epidemic']\n",
      "['optimal control', 'evolution without', 'control for sis epidemic']\n",
      "['optimal control', 'evolution without', 'control for sis epidemic']\n",
      "['noise impacts predictability', 'in', 'climate']\n",
      "['new susceptible individuals', 'spontaneously remove from', 'fig 10']\n",
      "['numericcitation', 'use', 'stochastic transmission models']\n",
      "['convergence p p st0', 'in', 'just as proof of theorem 2.6']\n",
      "['population', 'be', 'in model partitioned into households of relatively small size']\n",
      "['single infected individual', 'will produce in', 'completely susceptible']\n",
      "['sir process', 'in', 'random network with heterogeneous connectivity']\n",
      "['unknown coeﬃcients', 'equate', 'terms of same order']\n",
      "['choosing t large enough forces', 'close agreement in', 'maximum eigenvalues of corresponding production operators']\n",
      "['population size', 'in', 'simulations']\n",
      "['mckendrick', 'with', 'special reference to its bearing on natural phenomena of progressive nature']\n",
      "['numericcitation w. feller', 'introduction to', 'probability theory']\n",
      "['peaks', 'increase signiﬁcantly in', 'line with real data']\n",
      "['population size', 'remain', 'virtually unchanged']\n",
      "['binomial random trees 6', 'on infection', 'ri']\n",
      "['susceptible state s', 'with', 'one node chosen at']\n",
      "['pt', 'support by', 'swedish research council']\n",
      "['wetted sites', 'with', 'correct probability']\n",
      "['numericcitation n. t. j. bailey', 'theory of', 'infectious diseases']\n",
      "['coupled equation', 'of jacobian', 'k']\n",
      "['case', 'with', 'w constant']\n",
      "['area', 'be', 'related to extreme statistics of one dimensional branching brownian motion with death']\n",
      "['primary infection', 'in', 'new host']\n",
      "['infection', 'spread between', 'nodes in diﬀerent layers']\n",
      "['stochastic sis model', 'with', 'edge weights']\n",
      "['certain real-world scenarios', 'in', 'like analysis of disease propagating through plants']\n",
      "['evolution', 'in', 'time']\n",
      "['network', 'of connectivity', 'mean node degree']\n",
      "['node k', 'in', 'state s']\n",
      "['g', 'with', 'minimum infection range']\n",
      "['minimum infection range', 'in', '˜g']\n",
      "['minimum infection range', 'in', '˜g']\n",
      "['equations', 'model', 'epidemic dynamics']\n",
      "['f3 expressions', 'find in', 'eqs']\n",
      "['xt i. introduction many real-world systems', 'range from', 'nature to society']\n",
      "['super node supernode', 'in', '˜g']\n",
      "['siir epidemic model', 'in', 'two-layer network']\n",
      "['ren', 'allocation for', 'ranking spreaders in complex networks']\n",
      "['confusions', 'with', 'community size distribution p s']\n",
      "['moran model', 'model of', 'random genetic variations']\n",
      "['solution', 'maintain', 'diﬀerence in strategy']\n",
      "['same way', 'in', 'as proof of similar result']\n",
      "['healthy nodes', 'encapsulate by', 'infected ones']\n",
      "['jump', 'in', 'data for ontario']\n",
      "['susceptibility', 'with', 'probability tending to 1 as n → ∞']\n",
      "['ℓk', 'with xk', 'k d']\n",
      "['r', 'number of', 'ultimately recovered individuals in epidemic']\n",
      "['nodes', 'in', 'layer']\n",
      "['recovery nodes', 'fraction of', 'vaccinated nodes']\n",
      "['infected individual', 'of individual', 'rate']\n",
      "['ms', 'number of', 'infectees given initially infected node s']\n",
      "['nearest nodes', 'in', 'ˆs l − 1']\n",
      "['piece', 'in', 'network']\n",
      "['network', 'be', 'sparse']\n",
      "['branching processes', 'can', 'can studied in discrete time']\n",
      "['one-to-one correspondence', 'with', 'static conﬁguration of individual infection times']\n",
      "['measles', 'in', 'vaccine era with subclinical infections']\n",
      "['infectious diseases', 'spread through', 'human networks']\n",
      "['monte carlo direct sampling', 'in', 'case of sir model on random graphs for diﬀerent observation methods']\n",
      "['diﬀerent models', 'study', 'epidemic spreading']\n",
      "['parameter values', 'simulate', 'possible realizations of test data']\n",
      "['nodes', 'in', 'recovered state']\n",
      "['introduced infected individuals', 'in', 'population']\n",
      "['z τ l', 'l give', 'kl ≥ 2 for l ≤ l']\n",
      "['ﬂuctuations', 'in', 'number of infectives lagged behind ﬂuctuations']\n",
      "['presence', 'in', 'nucleus of itn']\n",
      "['easier propagation', 'in', 'neighbour of greece']\n",
      "['hidden state processes', 'to x', 'x ∗ particle weights of smc algorithm']\n",
      "['model', 'be', 'generative']\n",
      "['phase', 'be', 'many orders of magnitude longer than typical relaxation time']\n",
      "['stochastic sir model', 'with', 'edge weights']\n",
      "['there can large heterogeneity', 'in', 'population']\n",
      "['simulations', 'in', 'section']\n",
      "['simulated epidemics', 'in', 'empirical spatiotemporal network of 50,185 sexual contacts']\n",
      "['based estimator', 'in', 'general graphs']\n",
      "['analytical results', 'with', 'those obtained from direct simulations of stochastic model']\n",
      "['k individuals', 'with', 'probability']\n",
      "['distribution', 'much diﬀerent from', 'binomial distribution of standard percolation eq']\n",
      "['host population', 'with', 'standard deterministic sir model numericcitation']\n",
      "['basic reproduction number r0', 'be', 'eﬀective reproduction number in fully-healthy susceptible population']\n",
      "['two infectious stages', 'in', 'our modiﬁed metapopulat ion sir model']\n",
      "['zhou', 'dynamics on', 'complex networks']\n",
      "['defense', 'in', 'complex networks']\n",
      "['pei', 'dynamics in', 'complex networks']\n",
      "['identiﬁcation', 'in', 'complex networks']\n",
      "['chen', 'node in', 'complex networks']\n",
      "['t. identifying inﬂuential nodes', 'in', 'complex networks']\n",
      "['dorogovtsev', 'phenomenon in', 'complex networks']\n",
      "['c. synchronization', 'in', 'complex networks']\n",
      "['arenas', 'synchronization in', 'complex networks']\n",
      "['numericcitation pastor-satorras r', 'in', 'complex networks']\n",
      "['infected sites', 'in', 'stochastic model']\n",
      "['vaccination control', 'with', 'gv as']\n",
      "['observed states', 'be', 'numbers of cholera cases reported']\n",
      "['algorithm', 'provide', 'good estimates of true parameter values']\n",
      "['virus spread', 'in', 'networks']\n",
      "['bifurcation analysis', 'in', 'epidemic model']\n",
      "['johansen', 'model of', 'recurrent epidemics']\n",
      "['v. source e st imat ion', 'in', 'section']\n",
      "['barthélemy m', 'velocity of', 'epidemic outbreaks in scale-free networks']\n",
      "['optimal solution', 'approximately distribute vaccine in', 'proportion to population size']\n",
      "['allocations', 'in', 'proportion to population size']\n",
      "['square lattice', 'with', 'l × l sites under periodic boundary conditions']\n",
      "['graph', 'be', 'random']\n",
      "['simulation', 'with', 'demographic eﬀects']\n",
      "['scheme', 'disregard', 'existing correlations between 90 of nodes']\n",
      "['new infections', 'cause by', 'symptomatics']\n",
      "['new infections', 'cause by', 'asymptomatics']\n",
      "['school individual', 'have', 'random number of household children']\n",
      "['numericcitation', 'impact of', 'asymptomatic infection on coupled disease-behavior dynamics']\n",
      "['large outbreak', 'in', 'commercial poultry farms']\n",
      "['node', 'recover', 'multiple times']\n",
      "['its state changes', 'with', 'probability p']\n",
      "['relationship', 'in accordance with', 'results presented in fig']\n",
      "['comparison', 'with', 'results of stochastic simulations on rrg-4 plane']\n",
      "['subsequent interval', 'during walker', '0 dt']\n",
      "['dt', 'in', '∆ x']\n",
      "['numericcitation m. yuan', 'in', 'gaussian graphical model']\n",
      "['permanent immunity', 'in', 'sir model']\n",
      "['infection', 'should', 'in last step should transmitted to k1 neighboring nodes']\n",
      "['deterministic eﬀects', 'in', 'population dynamics']\n",
      "['one dimensional branching brownian motion', 'with', 'death']\n",
      "['play', 'in', 'prediction of large outbreaks']\n",
      "['recovery', 'deterministic after', 'τi time steps']\n",
      "['that on j ∈ v ′ ω', 'number of', 'edges']\n",
      "['role', 'in', 'pathogen antigenic variation']\n",
      "['their landscape', 'so visible to', 'their neighbors']\n",
      "['1 − λi', 'in', 'seir model']\n",
      "['epidemic generating graph', 'in', 'hierarchical model']\n",
      "['networks', 'with', 'fig 8 shows in particular even in case of complete obthe regular trees ing on observation']\n",
      "['a epidemic spreading', 'in', 'scale free networks']\n",
      "['behavioural changes', 'in', 'models of infectious disease dynamics']\n",
      "['spread', 'in', 'networks']\n",
      "['random networks', 'with', 'heterogeneous connectivity']\n",
      "[\"erd˝os r'enyi\", 'with', 'average degree hki']\n",
      "['average degree hki', \"with r'enyi\", 'er']\n",
      "['death', 'with', 'same parameters']\n",
      "['death', 'with', 'same parameters']\n",
      "['vertex', 'with', 'degree k proportional to k']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['probable tra jectory', 'in', 'standard sis model from a to c']\n",
      "['various networks', 'in', 'homogeneous networks under sis model']\n",
      "['numericcitation a. karnik', 'proceed of', 'ieee communication systems']\n",
      "['generating functions formalism', 'branch', 'theory numericcitation']\n",
      "['models', 'evolve for', '10000 time steps']\n",
      "['4.2 network sis model', 'in', 'section']\n",
      "['near-critical sir', 'with', 'vaccination']\n",
      "['infected individual', 'will infect', 'fewer individuals']\n",
      "['size', 'in', 'case']\n",
      "['mean ﬁeld equations', 'can', 'for comparison can also considered near equilibrium in stochastic form']\n",
      "['mean ﬁeld type recursion relation', 'with', 'discrete time steps for sir model']\n",
      "['ﬁ citation', 'in', 'congo']\n",
      "['problem', 'have', 'for classical critical sir process has investigated in numericcitation']\n",
      "['susceptibility', 'in', 'phase transitions numericcitation']\n",
      "['network', 'with', 'less traveled edges']\n",
      "['eames', 'in', 'spread of sexually transmitted 7 for complex networks']\n",
      "['initial time', 'in', 'sir model']\n",
      "['optimal strategy', 'use', 'maximal control for entire epidemic']\n",
      "['vertex', 'with', 'l1']\n",
      "['kmax', 'be', 'where largest degree of network']\n",
      "['example numericcitation', 'in', 'mice']\n",
      "['model selection', 'in', 'networks']\n",
      "['node i probability', 'in', 'one discrete time step number of simulations for speciﬁc sir process']\n",
      "['9 susceptible population', 'be', 'once reduced']\n",
      "['sir epidemic spreading', 'in', 'facebook social network']\n",
      "['results', 'present in', 'tab']\n",
      "['expected fraction', 'in', 'even limit as n → ∞']\n",
      "['epidemic spreading', 'in', 'multilayer networks 3 2']\n",
      "['epidemic spreading', 'in', 'multilayer networks']\n",
      "['current outbreak', 'in', 'west africa']\n",
      "['node spreading inﬂuence', 'with', 'largest 19']\n",
      "['epidemic chain', 'record', 'infection fact from seed to last patient']\n",
      "['our ﬁndings', 'may partially answer', 'key challenges mentioned in ﬁrst paragraph proposed in numericcitation']\n",
      "['t c', 'be', 'where set of candidate snapshot times']\n",
      "['inﬂuential spreader', 'in', 'social networks 9']\n",
      "['chen', 'node in', 'large-scale directed networks']\n",
      "['t. identifying inﬂuen tial nodes', 'in', 'large-scale directed networks']\n",
      "['l', 'leaders in', 'social networks']\n",
      "['t. leaders', 'in', 'social networks']\n",
      "['ide', 'centrality in', 'interconnected networks']\n",
      "['a. diffusion centrality', 'in', 'interconnected networks']\n",
      "['comparison', 'with', 'results of stochastic simulations on rrg-4 for parameter values in region ii']\n",
      "['jensen', 'have', 'inequality']\n",
      "['ﬁnal infection speed', 'proportional to', 'degree centrality']\n",
      "['sir model', 'with', 'same parameters']\n",
      "['numericcitation e. volz', 'journal of', 'mathematical biology 56 293']\n",
      "['total stochasticity', 'in', 'system']\n",
      "['total stochasticity', 'in', 'system']\n",
      "['numericcitation j. c. miller', 'journal of', 'mathematical biology']\n",
      "['dynamical system', 'with', 'discrete time']\n",
      "['interesting references', 'in', 'numericcitation']\n",
      "['there may large separation', 'with', 'mean size lying somewhere in']\n",
      "['reproduction number', 'in', 'west africa']\n",
      "['recovered nodes', 'acquire', 'permanent immunity in sir model']\n",
      "['two osns', 'also experience', 'near-term downturn in active membership numericcitation']\n",
      "['details', 'will', 'will given in next section']\n",
      "['epidemic models', 'with', 'uncorrelated random walk']\n",
      "['non-ignorable lump', 'in', 'ﬂuential in simulated determination of sir epi demic threshold']\n",
      "['19822154 numericcitation anderson r', 'disease of', 'humans']\n",
      "['infected indviduals', 'highly infectious during', 'time']\n",
      "['population', 'with', 'infective individuals passing on to their neighbours']\n",
      "['simulation', 'in', 'network with n']\n",
      "['w.p', 'be', 'where abbreviation for with probability']\n",
      "['bp', 'overestimate', 'epidemic size']\n",
      "['active growth site', 'spread to', 'unoccupied nearest neighbor with given bond probability']\n",
      "['equations', 'govern', \"epidemic 's growth\"]\n",
      "['approach', 'result in', 'deterministic network model']\n",
      "['discontinuity', 'in', 'ﬁnal size distribution of siir mode l']\n",
      "['color online a', 'value for', 'spectra of daily mean temperature']\n",
      "['numericcitation h. behncke', 'control of', 'deterministic epidemics']\n",
      "['numericcitation h. gaﬀ', 'apply for', 'various epidemiological models']\n",
      "['numericcitation h. gaﬀ', 'apply to', 'vaccination']\n",
      "['numericcitation a. karnik', 'control of', 'information epidemics']\n",
      "['eﬀect', 'have', 'has already observed before in spectra of ﬂuctuations around stable node in unforced systems numericcitation']\n",
      "['section 5', 'study', 'computational complexity of extended/generalized epidemiological problems']\n",
      "['gain g', 'decline well before', 'onset of eﬀective herd immunity']\n",
      "['size', 'in', 'so case']\n",
      "['time', 'at kstm', 'k']\n",
      "['comparison', 'with', 'data series from google flu trends']\n",
      "['r0', 'give', 'number of secondary cases produced by lone infectious individual in population of susceptible individuals during one infectious period']\n",
      "['sir model', 'with', 'random edge']\n",
      "['phase transitions', 'in', 'modiﬁed versions of sirs model deﬁned on swn']\n",
      "['infection', 'be', 'where presents']\n",
      "['arbitrary network structure', 'with', 'reduced running time']\n",
      "['epidemics', 'predict', 'essential proteins in protein interaction networks 13']\n",
      "['contact', 'infect with', 'probability β']\n",
      "['susceptible individuals', 'in', 'sir process']\n",
      "['numericcitation v. g. subramanian', 'in', 'proc']\n",
      "['numericcitation c. milling', 'in', 'proc']\n",
      "['process', 'supercritical also for', 'suﬃciently small λ with γ']\n",
      "['distributed random time', 'with', 'ﬁxed rate γ']\n",
      "['varying effects', 'in', 'different areas']\n",
      "['20553944 numericcitation khan m', 'model with', 'migration eﬀect']\n",
      "['noisy oscillations', 'with', 'same frequency as damped oscillations of deterministic system']\n",
      "['spreading dynamics', 'in online', 'effective epidemic threshold λc as function of recovery rate µ on real-world networks']\n",
      "['infected node', 'recover with', 'recovery rate δ']\n",
      "['numericcitation pastor-satorras r', 'in', 'scale-free networks']\n",
      "['inference', 'in', 'disease transmission experiments']\n",
      "['individuals', 'possibly have', 'symptoms']\n",
      "['individuals', 'have', 'symptoms']\n",
      "['model', 'in', 'thermodynamic limit']\n",
      "['ngc', 'be', 'where size of giant component']\n",
      "['dependence', 'number of', 'infected individual']\n",
      "['larger modules', 'yield', 'higher epidemic size']\n",
      "['degree', 'outperform', 'shell number']\n",
      "['setup', 'in', 'numericcitation']\n",
      "['diﬀerential equations', 'model', 'epidemic dynamics']\n",
      "['at network', 'with', 'n nodes']\n",
      "['epidemic process', 'in', 'speciﬁc realization']\n",
      "['susceptible individuals', 'in', 'steady state']\n",
      "['layer α', 'neighbour of', 'susceptible node in layer β']\n",
      "['noise', 'in', 'simulations of stochastic attractors near endemic state']\n",
      "['n smax', 'since t', 'out-degree ko n for time']\n",
      "['errant infection attempts', 'of number', 'n x']\n",
      "['rumor source det ection', 'with', 'multiple observations']\n",
      "['stochastic model', 'in', 'figure 3']\n",
      "['numericcitation a. ganesh', 'in', 'proc']\n",
      "['spread', 'in', 'west africa']\n",
      "['public health priority', 'minimize', 'time period needed for outbreaks eradication in aﬀected areas']\n",
      "['total number', 'with', 'immunity level in at time']\n",
      "['complete convergence theorem', 'hold for', 'contact processes in random environment on z d × z']\n",
      "['adaptive networks', 'in', 'sir models']\n",
      "['ﬁrst order terms', 'in', 'ia s ra we obtain']\n",
      "['monte carlo simulations', 'with', 'death']\n",
      "['aparicio', 'oscillation in', 'stochastic systems']\n",
      "['states', 'in', 'j from time slots']\n",
      "['square lattice', 'only account for', 'local nearest-neighbor interactions']\n",
      "['embedded chain', 'with', 'random edge weights']\n",
      "['internet number', 'on infection', 'ri']\n",
      "['st. louis', 'of bank', 'st. louis']\n",
      "['site', 'be', 'in which occupied by just one individual']\n",
      "['results earlier studies', 'make on', 'same network']\n",
      "['ﬁrst equation', 'in', 'eq']\n",
      "['ﬁrst equation', 'in', 'eq']\n",
      "['points', 'in', 'ball of radius 1/2 centered at origin']\n",
      "['bang-bang control', 'with', 'at most one switching time τ ∗']\n",
      "['original infection', 'in', 'network']\n",
      "['2.2.1 markov models', 'in', 'often modelling community']\n",
      "['state', 'with', 'xi infective vertices recover at rate ρn']\n",
      "['phase transition', 'in', 'model']\n",
      "['result', 'have', 'important eﬀect in vaccination policies']\n",
      "['epidemics', 'for model', 'sir epidemic in closed population']\n",
      "['sir model', 'include', 'following processes']\n",
      "['extreme model d', 'd for', 'symmetry reasons']\n",
      "['recovery time', 'in', '≤ t']\n",
      "['order', 'in', 'sir model']\n",
      "['behaviour', 'in', 'limit']\n",
      "['epidemic models', 'with', 'explicit individual movement']\n",
      "['process', 'with', 'recovery rate λ0']\n",
      "['earlier scenario', 'with', 'isolation in layers']\n",
      "['probability', 'in', 'out-of-equilibrium system']\n",
      "['following notations', 'will', 'will used in later analysis']\n",
      "['equations 1', 'with', 'β11']\n",
      "['code ¸ co numericcitation', 'use sir model with', 'r +1 s-1']\n",
      "['range', 'in', 'v 1']\n",
      "['regular tree', 'evolution of', 'z τ']\n",
      "['ks', 'be', 'maximum shell present in network']\n",
      "['temporal shift', 'result of', 'diﬀerence in initial condition']\n",
      "['proper system', 'with', 'same variables on right hand side as on left hand side']\n",
      "['t1/2', 'number of', 'infected cases']\n",
      "['structure', 'in', 'our model']\n",
      "['c', 'number of', 'infectious contacts']\n",
      "['ﬁnal outcome', 'in', 'large population']\n",
      "['model', 'deﬁned on', 'swn']\n",
      "['bond percolation', 'in', 'generic multilayer network given locally tree-like multilayer network of m layers']\n",
      "['9 8 m.f laguna g. abramson', 'in', 'model of social inﬂuence physica a']\n",
      "['spread', 'in', 'networks']\n",
      "['distances', 'in', 'components of erdős-rényi model scales like log n']\n",
      "['inclusion', 'explain', 'jump in data for ontario on march 31st']\n",
      "['inhomogeneous models', 'construct for', 'four cases studied in previous section']\n",
      "['mixed si epidemic dynamics', 'in', 'random graph s']\n",
      "['otte', 'impact on', 'animal production in developing countries']\n",
      "['infected individuals', 'in', 'population weeks observed number of cholera cases increases']\n",
      "['stable spiral', 'in', 'direction transverse to cl']\n",
      "['sir model', 'in', 'also percolation class']\n",
      "['time', 'proceed in', 'discrete time steps']\n",
      "['seir simulations', 'with', 'p']\n",
      "['seir simulations', 'series on', 'n']\n",
      "['numericcitation', 'from pr', 'v']\n",
      "['numericcitation', 'from pr', 'v']\n",
      "['epidemic model', 'with', 'infectious-recovery term']\n",
      "['sir model', 'with', 'population turnover']\n",
      "['sir model', 'with', 'population turnover']\n",
      "['ﬁnal epidemic size', 'fraction of', 'recovered agents']\n",
      "['adequate quarantine systems', 'customize in', 'response to local details of network architecture']\n",
      "['individuals', 'can', 'in multilayer networks can actors']\n",
      "['vaccine trial', 'test', 'eﬃcacy of experimental vaccine against ebola']\n",
      "['sis-like diseases', 'in', 'local groups of tightly connected individuals']\n",
      "['asymptotics', 'in', 'high dimension for fortuin-kasteleyn cluster model']\n",
      "['citation', 'deadliest epidemic in', 'history']\n",
      "[\"r'enyi random graph\", 'with', 'edge probability λ/n']\n",
      "['u', 'use', 'center manifold equation given by eq']\n",
      "['similar population size', 'with oscillation', 'figure 4c']\n",
      "['beta-binomial distribution', 'with', 'parameters']\n",
      "['infected node u', 'infect', 'adjacent susceptible node v in one discrete time step']\n",
      "['analytical results', 'can quantitatively describe', 'impacts of diﬀerent factors on epidemic threshold']\n",
      "['disease', 'totally eradicate from', 'population']\n",
      "['type', 'be', 'where number of vertices within cluster']\n",
      "['size', 'increase', 'exponentially fast with number of nodes']\n",
      "['binomial distribution', 'with', '3 binomial random trees']\n",
      "['k individuals', 'infect at', 'time t']\n",
      "['peaks', 'correspond to', 'raised points for ﬁnal epidemic size']\n",
      "['e.g', 'thank h.j.h for', 'his invitation to zurich']\n",
      "['future time slot', 'with', 'positive probability']\n",
      "['population size n', 'be', 'when large enough']\n",
      "['algebra', 'after 4', 'following differential equations for']\n",
      "['diﬀerent mathematical models', 'study', 'epidemic spreading']\n",
      "['susceptible nodes', 'describe', 'healthy individuals']\n",
      "['following fractional differential equations', 'can', 'can obtained for sir model']\n",
      "['full circles', 'absorb', 'states']\n",
      "['k', 'with', 'results of stochastic simulations on rrg-4 for several system sizes']\n",
      "['independent events', 'with', 'probability β']\n",
      "['single susceptible s', 'catch disease in', 'when contact with infectious individuals situated on neighboring nodes']\n",
      "['total number', 'in', 'ﬁrst k vertices evaluated in']\n",
      "['sir dynamics', 'for threshold', 'message-passing approach numericcitation']\n",
      "['ﬁrst term', 'in', 'ob jective']\n",
      "['herd immunity transition', 'in', 'deterministic model']\n",
      "['than initial epidemics', 'in parameter', 'relative population sizes']\n",
      "['opportunity', 'in', 'to ﬂuence']\n",
      "['epidemiology', 'in', 'citation']\n",
      "['number', 'in', 'network']\n",
      "['new transmission mode explosive growth', 'in', 'contrast to transmission']\n",
      "['underlying network g', 'be', 'where general graph']\n",
      "['12 5 network susceptible-infected-recovered model', 'in', 'section']\n",
      "['results', 'in', 'large-n limit']\n",
      "['16 0.2 0.15 β', 'in', 'plane β − w']\n",
      "['size scaling', 'propose in', 'reference numericcitation']\n",
      "['infective vertices', 'also recover at', 'rate ρ']\n",
      "['infective vertices', 'recover at', 'rate ρ']\n",
      "['joint probability pt', 'in', 'eq']\n",
      "['infection', 'occur between', 'connected sites']\n",
      "['infection', 'in', 'susceptible class']\n",
      "['infected individuals', 'move after', 'latent period']\n",
      "['11 increase', 'in', 'line with real data']\n",
      "['size', 'consider', 'particle']\n",
      "['branching brownian motion', 'with', 'death']\n",
      "['monte brownian motion', 'with', 'death']\n",
      "['resonance eﬀects', 'in', 'ecological time series']\n",
      "['ˆi0', 'clearly', 'αi']\n",
      "['2 basic model equations', 'in', 'section']\n",
      "['class np-complete', 'consist of', 'hardest problems in np']\n",
      "['population', 'into campaign', 'section 3.3']\n",
      "['stochastic susceptible-exposed-infected-recovered seir epidemiological model', 'with', 'contact rate']\n",
      "['n → ∞', 'as d/2', '64 dqn dp n − → 1 in probability']\n",
      "['p n', 'under d/2', '64 dqn dp n − → 1 in probability']\n",
      "['numericcitation c. w. gardiner', 'handbook of', 'stochastic methods']\n",
      "['2819 numericcitation blyuss kb', 'in', 'epidemic model with varying immunity period']\n",
      "['rgg', 'with', 'n nodes connected']\n",
      "['k', 'for ns', 'k − εn remaining susceptible vertices of degree k']\n",
      "['propagation', 'be', 'in general formulation stochastic process on complex network']\n",
      "['δ σ × − σ', 'measure of', 'diﬀerence between two competing strains']\n",
      "['micro-outbreak', 'be', 'where analogous to customer be-associated with single imported infection']\n",
      "['second population s2', 'of state', 'steady state']\n",
      "['i.o', 'in', 'subsection']\n",
      "['numericcitation castellano', 'thresholds for', 'epidemic spreading in networks']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['case', 'number of', 'edges']\n",
      "['infective individuals', 'in', 'ﬁnite populations']\n",
      "['fig 8 shows', 'in', 'even case of complete obthe regular trees ing on observation']\n",
      "['average error distances', 'with', 'error bars of 95 conﬁdence int erval']\n",
      "['average error distances', 'with', 'error bars of 95 conﬁdence int erval']\n",
      "['average error distances', 'with', 'error bars of 95 conﬁdence int erval']\n",
      "['average error distances', 'with', 'error bars of 95 conﬁdence int erval']\n",
      "['neighboring site', 'choose at', 'random']\n",
      "['critical birth-death process', 'in', 'section']\n",
      "['gillespie', 'on', 'method numericcitation']\n",
      "['outbreak', 'in', 'bdi process']\n",
      "['outbreak', 'in', 'bdi process']\n",
      "['simulations', 'with', 'm']\n",
      "['inﬁnite', 'with', 'probability 1 − ν − 1']\n",
      "['data', 'of year', 'citation']\n",
      "['global stability analysis', 'with', 'discretization approach for age-structured multigroup sir epidemic model']\n",
      "['maximal extinction time', 'in', 'sir simulations on networks']\n",
      "['driven sir process', 'with', '3 outbreaks']\n",
      "['cavity theory', 'obtain', 'epidemic threshold for sauir model numericcitation']\n",
      "['models', 'for 1', 'ξ0']\n",
      "['models', 'for 1', 'π +1']\n",
      "['models', 'for ln', 'ξ0']\n",
      "['eqs', 'with β', 't']\n",
      "['β', 'with', 'eqs']\n",
      "['rumor spreading', 'in', 'online social network']\n",
      "['infected neighbors', 'in', 'sir model']\n",
      "['contribution', 'be', 'novel']\n",
      "['20553944 numericcitation khan m', 'model of', 'hepatitis b virus']\n",
      "['comparison', 'with', 'numericcitation']\n",
      "['critical point', 'in', 'equilibrium phase on magnetic system']\n",
      "['results', 'show in', 'figure 8']\n",
      "['random vaccinations', 'with', 'probability θ']\n",
      "['contact processes', 'in', 'random environment on z d × z']\n",
      "['sir models', 'in', 'homogeneous networks']\n",
      "['calculations', 'use matlab routine for', 'fast fourier transform']\n",
      "['level', 'in', 'system']\n",
      "['moran model', 'model in', 'haploid population']\n",
      "['minimalist model', 'in', 'haploid population']\n",
      "['uganda', 'in virus', 'sudv']\n",
      "['recovery time', 'set', 'time scale']\n",
      "['reittu', 'have', 'poissonian random graph model numericcitation']\n",
      "['seeds', 'in', 'sis model']\n",
      "['distribution', 'in', 'sis model']\n",
      "['sis model', 'in', 'fig 2']\n",
      "['ρi', 'in', 'sis model']\n",
      "['bang-bang control', 'with', 'at most one switching time τ ∗ problem']\n",
      "['recent methodological developments', 'in', 'inference']\n",
      "['new i sites', 'in', 'third step']\n",
      "['short average path length', 'in', 'as random graphs']\n",
      "['tutorial comprehensive treatment', 'with', 'comprehensive statements for network si']\n",
      "['sir dynamics', 'in', 'random networks with heterogeneous connectivity']\n",
      "['pascal cr', 'epey', 'epidemic variability in complex networks arxiv']\n",
      "['such ill individuals', 'infect', 'only susceptible ones in same population']\n",
      "['our deterministic network models', 'be', 'derived from markov-chain models through mean-ﬁeld approximation']\n",
      "['epidemic phase', 'second take', 'place']\n",
      "['duration', 'with', 'high probability p1 ,2']\n",
      "['sei model', 'in', 'epidemiology']\n",
      "['their changed landscape', 'so visible to', 'their neighbors']\n",
      "['epidemic outbreak', 'in', 'relatively realistic situations']\n",
      "['predictive power', 'construct', 'model']\n",
      "['infected state', 'in', 'figure 1']\n",
      "['effective epidemic threshold', 'also decrease with', 'recovery rate µ in synchronous updating process']\n",
      "['corresponding individual', 'in', 'epidemic process e belongs to x ∼ mp']\n",
      "['e', 'in', 'y2']\n",
      "['time t', 'at s', 't']\n",
      "['infected state', 'in random', 'seed']\n",
      "['prediction', 'have', 'performance']\n",
      "['complex dynamical transitions', 'in', 'epidemics']\n",
      "['mean ﬁeld model', 'with', 'multiplicative noise']\n",
      "['its neighbors', 'with', 'probability ω']\n",
      "['our treatment', 'establish', 'numerous properties including novel characterization of epidemic threshold']\n",
      "['susceptible individual', 'correspond to', 'void node in link percolation']\n",
      "['5 connection', 'with', 'ode models']\n",
      "['tra jectory', 'in', 'time']\n",
      "['individuals', 'in', 'state']\n",
      "['individuals', 'in', 'state']\n",
      "['bp', 'give', 'better results for observations']\n",
      "['death probabilities', 'be', 'for critical process equal']\n",
      "['outbreak', 'in', 'uganda']\n",
      "['campaign', 'be', 'when conducted in speciﬁc locations']\n",
      "['near-critical behaviour', 'in', 'context of susceptibleinfectiverecovered epidemic on random graph on n vertices with given degree sequence']\n",
      "['inequalities', 'can', 'can found in appendix c as eq']\n",
      "['random 2-regular graph', 'with', 'n − n1/2 vertices']\n",
      "['weekly infection number', 'be', 'two parts']\n",
      "['interval numericcitation', 'in', 'time']\n",
      "['given large inter m', 'with', 'high probability']\n",
      "['random graph', 'with', 'given vertex degrees']\n",
      "['∞', 'point of', 'pa-sis epidemic process']\n",
      "['phenomenon', 'depend on', 'exact topological structure of networks']\n",
      "['ring vaccination', 'be', 'more suitable for fmd than mass vaccination numericcitation']\n",
      "['methods', 'in', 'paper']\n",
      "['qn n ≥ 1', 'with', 'α0']\n",
      "['distribution', 'acquire', 'many diﬀerent shapes']\n",
      "['interesting future research direction', 'study', 'sensitivity of control strategies to estimation errors']\n",
      "['ρr ﬂuctuation', 'in', 'sir model']\n",
      "['larger λ', 'in', 'sir model']\n",
      "['distribution', 'in', 'sir model']\n",
      "['1000 realizations', 'always start', 'infection in us']\n",
      "['our methodology', 'potentially useful during', 'cholera surveillance in bangladesh']\n",
      "['our methodology', 'useful during', 'cholera surveillance in bangladesh']\n",
      "['closed form solution', 'be', 'available numericcitation']\n",
      "['numericcitation', 'with', 'link occupancy probability t']\n",
      "['citation ﬁrst limits', 'in', 'new construction']\n",
      "['event', 'of f', 'ti']\n",
      "['ma jor challenges', 'in', 'population dynamics numericcitation']\n",
      "['epidemiological model', 'with', 'seasonal ﬂuctuations']\n",
      "['broader peaks', 'due to', 'resonant ampliﬁcation of stochastic ﬂuctuations around limit cycle']\n",
      "['epidemics', 'in', 'small-world networks']\n",
      "['long range infections', 'in', 'model']\n",
      "['sir model', 'with', 'variable transmission rate']\n",
      "['sir model', 'with', 'time-dependent indirect transmission rate']\n",
      "['snwa', 'have', 'eﬀorts marketing']\n",
      "['use', 'in', 'theoretical methods i.e performance level of method only proportional to topological information used']\n",
      "['diﬀerent layers', 'with', 'diﬀerent contacts in layer']\n",
      "['interparticle correlation', 'in', 'death processes with application to variability']\n",
      "['phase-space transport', 'in', 'population dynamics of virus spread']\n",
      "['system', 'respond adiabatically to', 'changes in susceptible fraction']\n",
      "['infection', 'become', 'large scale epidemic']\n",
      "['3.1 al l individuals', 'in', '12 f. g. ball']\n",
      "['sir model', 'with', 'isolation control']\n",
      "['similar arguments', 'in', 'as t']\n",
      "['susceptible nodes', 'become', 'infected with rate β']\n",
      "['p', 'measure of', 'reduced susceptibility to sars in class s2']\n",
      "['nodes', 'in', 'network']\n",
      "['network', 'with', 'n nodes']\n",
      "['probability πiα', 'β of', 'link between p']\n",
      "['endogenous diseases', 'with', 'vital dynamics']\n",
      "['12 epidemic spreading', 'in', 'multilayer networks 6']\n",
      "['d', 'use', 'notiﬁcation data in numericcitation']\n",
      "['contact processes', 'with', 'random vertex weights on oriented lattices']\n",
      "['patterns', 'with', 'realistic amplitudes']\n",
      "['evolution', 'in similarity', 'computer science propagation of virus']\n",
      "['number', 'in', 'system']\n",
      "['sir model', 'can', 'can parameterized by single number']\n",
      "['broad distribution degree', 'in', 'contact network numericcitation']\n",
      "['focal point', 'restore', 'proper function of neural mechanisms']\n",
      "['model', 'in curve', 'black dots']\n",
      "['pair ξ', 'ζ of', 'infected individuals']\n",
      "['time', 'in', 'random process']\n",
      "['epoch', 'node in', 'susceptible state']\n",
      "['infection', 'can take oﬀ above', 'threshold']\n",
      "['graph', 'with', 'given degrees']\n",
      "['simulations', 'agree well with', 'analytic results']\n",
      "['analytic results', 'agree well with', 'simulations']\n",
      "['b', 'inset of', 'average absolute errors as function of ipr']\n",
      "['separation', 'with', 'little probability observing in between two']\n",
      "['1 reduction', 'in', 'nonlinear dynamical systems']\n",
      "['15 it', 'can', 'can seen']\n",
      "['simulated network', 'with', 'same number of nodes']\n",
      "['sites', 'in', 'i state change into r state']\n",
      "['epidemic spreading', 'in', 'scale-free networks']\n",
      "['nodes', 'in', 'susceptible state']\n",
      "['similar result', 'have', 'has also shown in continuous-time case']\n",
      "['transition rates', 'in', 'as table 1 with constraint']\n",
      "['same magnitude', 'in', 'countries of lower shells']\n",
      "['its initial state x0', 'in', 'time µt i p']\n",
      "['one dimensional network', 'with', 'additional random links']\n",
      "['results', 'hold without', 'even it']\n",
      "['15 it', 'can', 'can seen from eq']\n",
      "['full deterministic equations', 'in', 'variables']\n",
      "['two outbreaks', 'in', 'congo']\n",
      "['equations', 'can', 'in fact can directly obtained by eqs']\n",
      "['our approach', 'also yield', 'explicit formula for transmission rate']\n",
      "['ﬂuctuations', 'in', 'birth rate manifest itself as additive noise']\n",
      "['n pl', 'be', 'average outbreak size in power-law regime']\n",
      "['natural model', 'in', 'case']\n",
      "['general multilayer network', 'with', 'arbitrary number of links across layers']\n",
      "['s5', 'connectivity of', 'susceptible individuals']\n",
      "['epidemic spreads', 'in', 'independent partition model']\n",
      "['observed snapshot y', 'contain', 'at least one infected node']\n",
      "['spread', 'in', 'given population']\n",
      "['stochastic si1 i2r model', 'with', 'population turnover']\n",
      "['fraction', 'in', 'smaller population']\n",
      "['markov process x', 'with', 'transition rate matrix q']\n",
      "['model', 'be', 'how related to percolation around criticality']\n",
      "['scale-free networks', 'with', 'degree correlations']\n",
      "['numerical solutions', 'result of', 'simulations in solid lines']\n",
      "['hazard rate si', 'with s', 't']\n",
      "['3 susceptible-infected model', 'in', 'section']\n",
      "['4 susceptible-infected-susceptible model', 'in', 'section']\n",
      "['h3', 'n in', 'such 9']\n",
      "['high-dimensional lattices', 'with', 'random edge weights']\n",
      "['susceptible nodes', 'in', 'contact with infected individuals']\n",
      "['15 next step', 'investigate', 'global edges from vertices in c h']\n",
      "['real-world social contact networks', 'however exhibit', 'complex patterns of interconnection between individuals']\n",
      "['real-world social contact networks', 'exhibit', 'complex patterns of interconnection between individuals']\n",
      "['s', 'be', 'in case changed into exposed e state']\n",
      "['→ 0', 'as', 'dotted line']\n",
      "['only dimensionless parameter', 'in', 'model']\n",
      "['t', 'be', 'where chosen from numericcitation']\n",
      "['ﬁt', 'treat k as', 'free parameter']\n",
      "['number', 'in', 'square']\n",
      "['models', 'with', 'diﬀerent']\n",
      "['possible events', 'in', 'network superinfection model']\n",
      "['vaccination programs numericcitation', 'provide immunity to', 'individuals']\n",
      "['spatio-temporal epidemic modelling', 'use', 'additive-multiplicative intensity models']\n",
      "['agreement', 'with', 'ones of direct percolation universality class']\n",
      "['scale-free phenomena', 'be', 'unlike traditional deterministic models observed in epidemic network']\n",
      "['theorem', 'have', 'citation']\n",
      "['independent partition model', 'in', 'than more realistic hierarchical model']\n",
      "['ω λ', 'be', 'indicator function for epidemic threshold']\n",
      "['normal distribution', 'with', 'mean 0 deviation ση']\n",
      "['diﬀerence', 'in', 'total number of states between two methods']\n",
      "['there can most one errant infection attempt', 'in', 'modiﬁed process']\n",
      "['ring vaccination', 'eradicate fmd for', 'even high infection probability']\n",
      "['1 introduction infectious diseases', 'spread through', 'human networks']\n",
      "['epidemic outbreaks', 'in', 'networks']\n",
      "['other hypothetical scenarios', 'can', 'can also explored in straightforward way']\n",
      "['stochastic population models', 'in', 'ecology']\n",
      "['sir epidemics', 'index by', 'population size n']\n",
      "['random variables', 'for e', 'y 2z']\n",
      "['one layer', 'with', 'poisson degree distribution k numericcitation']\n",
      "['stochastic changes', 'in', 's']\n",
      "['sir model', 'with', 'vaccination']\n",
      "['one seed infected individual', 'in', 'population']\n",
      "['epidemic process', 'with', 'rate γ1']\n",
      "['epidemic process', 'with', 'rate γ0']\n",
      "['step', 'be', 'at time preserved in figure 5-a']\n",
      "['interpretation', 'number of', 'secondary cases produced by single infectious individual in population of susceptibles in one infectious period']\n",
      "['toronto data', 'of feature', 'steep increase in number after march 31st in growth']\n",
      "['two ﬁrst terms', 'in', 'functional ob jective']\n",
      "['optimal control', 'evolution without', 'control for sis epidemics for time-varying spreading rate β2']\n",
      "['optimal control', 'evolution without', 'control for sis epidemics for time-varying spreading rate β3']\n",
      "['discrepancy', 'can', 'can explained by invalidation of assumptions']\n",
      "['impressive improvement', 'in', 'time series prediction']\n",
      "['spread', 'in', 'human networks']\n",
      "['infected node', 'recover in', 'one discrete time step']\n",
      "['solid curves', 'calculate from', 'eq']\n",
      "['correlated model', 'with', 'best-ﬁt']\n",
      "['t', 'in', 'purely multiplicative model used']\n",
      "['quasineutral model', 'of line', 'cl']\n",
      "['expression', 'in', 'terms of infectivities λ given by eq']\n",
      "['preferential rewiring', 'more eﬃcient than', 'case with w constant']\n",
      "['bifurcation', 'in', 'hopf bifurcation for q']\n",
      "['nodes', 'random selection of', 'simplest method for distribution']\n",
      "['high pathogenic avian inﬂuenza', 'in', 'midwestern usa numericcitation']\n",
      "['connection radius', 'value of', 'for which rgg with n nodes connected with high probability']\n",
      "['pp 259', 'in', 'proceedings of fifth berkeley symposium on mathematical statistics']\n",
      "['chierichetti', 'spread in', 'social networks']\n",
      "['optimal control function u', 'with', 'initial conditions']\n",
      "['targets', 'target of', 'other simultaneous infection attempts']\n",
      "['would case', 'with', 'markov assumption']\n",
      "['gephi numericcitation', 'visualize', 'sample networks generated by proposed algorithms']\n",
      "['partial snapshot y', 'contain', 'at least one infected node']\n",
      "['importance', 'in', 'network exposed to epidemic outbreak']\n",
      "['λ', 'corollary of', 'theorem 2.1']\n",
      "['resulting time series', 'also show in', 'fig 2']\n",
      "['rumor spreading', 'in', 'online social network']\n",
      "['quantity', 'zero for', 'large n']\n",
      "['model spreading', 'in', 'various networks numericcitation']\n",
      "['substantial growth', 'in', 'residential housing']\n",
      "['h4', 'n in', 'numericcitation such']\n",
      "['complete agreement', 'with', 'numerical results obtained in numericcitation']\n",
      "['eq', 'of ∗', '∞']\n",
      "['instance', 'instance of', 'π epidemic']\n",
      "['shooting method numericcitation', 'in', 'matlab']\n",
      "['population collapse', 'in', 'species 1']\n",
      "['dynamical models', 'use', 'freely available wrapper around odeint function in scipy']\n",
      "['hierarchical model', 'in −', '1 − ps']\n",
      "['6 comparison', 'with', 'data measles']\n",
      "['model', 'also consider by', 'antal et al']\n",
      "['references numericcitation amanna ij', 'duration of', 'humoral immunity to common viral antigens']\n",
      "['such extinction regimes', 'use', 'fokker-planck approach numericcitation']\n",
      "['holme numericcitation', 'use', 'simple sir model']\n",
      "['578 numericcitation bhattacharya s', 'fr', 'time']\n",
      "['1 ≤ 𝑘 ≤ 𝑁 crete distribution', 'with', 'probability mass function']\n",
      "['numericcitation t. e. harris', 'theory of', 'branching processes']\n",
      "['s2 ∈', 'with', 's1 s2']\n",
      "['infected individual', 'in', 'layer α']\n",
      "['susceptibility', 'with', 'standard set of equations spreading']\n",
      "['two-strain competition', 'in', 'quasi-neutral stochastic disease dynamics']\n",
      "['second critical point', 'in', 'process separating']\n",
      "['numericcitation n. antulov-fantulin', 'c', 'h. ˇstefanˇci']\n",
      "['poisson process', 'with', 'rate γ1']\n",
      "['previous case', 'with', 'exception']\n",
      "['spreading process', 'use', 'discrete time step model']\n",
      "['ks', 'be', 'so in real scenario still eﬀective predictor for highly susceptible individuals']\n",
      "['epidemic chain', 'model of', 'social contacts']\n",
      "['fect ion s pread ing', 'in', 'tree s']\n",
      "['internet autonomous system network', 'obtain base on', '31 citation']\n",
      "['only diﬀerence', 'eﬀect of', 'transformation on stochastic term']\n",
      "['eﬀective reproduction number', 'in', 'deterministic network sir model']\n",
      "['activity distribution', 'with', 'corresponding quantities']\n",
      "['system', 'with', 'inﬁnite population size']\n",
      "['primary infection', 'in', 'child']\n",
      "['primary infection', 'in', 'parent']\n",
      "['fraction', 'in', 'disease state']\n",
      "['degree/motif distribution', 'of pgf', 'probability generating function']\n",
      "['appendix a', 'contain', 'technical lemma on time change in section 3']\n",
      "['changes', 'in', 'environment']\n",
      "['parameters', 'modiﬁed to', 'diﬀerent infectious disease dynamics']\n",
      "['models', 'be', 'also introduced']\n",
      "['process', 'with', 'site clearance rate γ1']\n",
      "['ﬁnitesimal f raction', 'in', 'sir model numericcitation']\n",
      "['decision processes', 'series in', 'quantitative methods for decision making']\n",
      "['parameters', 'for simulation', 'left']\n",
      "['numericcitation r. m. anderson', 'diseases in', 'humans']\n",
      "['contagion', 'be', 'where indeﬁnite']\n",
      "['two distinct methods', 'describe for', 'disease propagation dynamics']\n",
      "['susceptible vertices', 'with', 'k free half-edges']\n",
      "['shortcuting neighbours', 'be', 'ﬁxed beforehand']\n",
      "['figure 1', 'schematic demonstrate', 'our approach to problem']\n",
      "['epidemic threshold', 'in', 'question']\n",
      "['one poisson process', 'with', 'rate 1 ¯ mvk particles at x']\n",
      "['individuals', 'in', 'population m']\n",
      "['expected number', 'with', 'β']\n",
      "['parameters', 'for distribution', 'right']\n",
      "['˙pm', 'in', 'as deterministic equations']\n",
      "['distribution', 'in', 'bd process']\n",
      "['numericcitation a. vespignani', 'structure of', 'complex networks']\n",
      "['eﬀects', 'proceed of', 'royal society b 278 1082 1089']\n",
      "['diﬀerences', 'in', 'ﬁnal time between optimal control']\n",
      "['time domain', 'in percolation', 'void node']\n",
      "['node void percolation', 'in', 'time domain']\n",
      "['next step', 'investigate', 'school epidemics of individuals infected through household contacts']\n",
      "['x', 'of', 'infected neighbors']\n",
      "['stochastic peaks', 'be', 'for largest system size predicted correctly by theory']\n",
      "['model', 'with', 'initial condition s']\n",
      "['2013 threshold dynamics', 'in', 'seirs model with latency']\n",
      "['i', 'become', 'r at rate b']\n",
      "['er', 'with', 'same hki for']\n",
      "['stochasticity', 'lead to', 'noteworthy diﬀerences in epidemic size']\n",
      "['dynamical system', 'in', 'vector form']\n",
      "['probability', 'in', 'sir network']\n",
      "['vertex', 'with', 'l1 norm lager']\n",
      "['interdegree', 'perform', 'random permutation of corresponding interdegree sequence']\n",
      "['phase diagram', 'in', 'plane β − w']\n",
      "['phase diagram', 'in', 'plane β − w']\n",
      "['phase diagram', 'in', 'plane β − w']\n",
      "['phase diagram', 'in', 'plane β − w']\n",
      "['phase diagram', 'in', 'plane β − w']\n",
      "['claim', 'be', 'now proved']\n",
      "['geometric structure', 'be', 'when applied to population']\n",
      "['saddle-node point', 'in', 'question']\n",
      "['proofs', 'in', 'kurtz']\n",
      "['non-equilibrium second order phase transition', 'in', 'statistical physics numericcitation']\n",
      "['t1', 'on it0', 'a.s. for t1 0']\n",
      "['ferreira', 'threshold of', 'susceptible-infected-susceptible model on networks']\n",
      "['numericcitation', 'with', '2 λ']\n",
      "['fractions', 'in', 'case']\n",
      "['case ρ', 'in 20', 'recoveries']\n",
      "['g. again', 'for theorem', 'case']\n",
      "['numericcitation m. cha', 'in', 'proceedings of 18th international conference on world wide web']\n",
      "['infected nodes', 'ﬁnal fraction of', 'above which ﬁnite numericcitation']\n",
      "['correlation coeﬃcient', 'be', 'larger within ma jority of subintervals than for distributions taken for whole interval']\n",
      "['transmission rate', 'proportion in', 'same population']\n",
      "['stochastic s i1 i2s model', 'for s', 'i2 → i2']\n",
      "['stochastic s i1 i2s model', 'for s', 's → s']\n",
      "['similarities', 'in', 'infectious waning numericcitation']\n",
      "['infection', 'be', 'presents']\n",
      "['remaining transitions', 'δ of', 'disease model']\n",
      "['contact', 'with', 'its neighbors']\n",
      "['square lattice', 'with', 'n sites']\n",
      "['many problems', 'in', 'epidemiology numericcitation']\n",
      "['network', 'with', 'smallest maximum distance']\n",
      "['susceptible individuals', 'become', 'infected at given rate through contact with infected individuals']\n",
      "['measles', 'in', 'england']\n",
      "['theorem 1', 'will', 'will proved']\n",
      "['theorem 2', 'will', 'will proved']\n",
      "['numericcitation pastor-satorras', 'type of', 'eigenvector localization in networks']\n",
      "['networks', 'with', 'same degree distribution']\n",
      "['networks', 'with', 'same degree distribution']\n",
      "['results', 'can', 'however can likely improved by higher computational power']\n",
      "['connections', 'break', 'links from non-infected nodes to infected nodes']\n",
      "['economic crisis', 'could spread globally depend', 'depending country of origin']\n",
      "['mult iple source s e st imat ion', 'in', 'section']\n",
      "['random graphs', 'with', 'given vertex degrees']\n",
      "['matrix', 'in', 'case']\n",
      "['s', 'be', \"where node 's community size\"]\n",
      "['zeros', 'in', 'reconstruction of edges 200 node synthetic smal l world network']\n",
      "['zeros', 'in', 'reconstruction of edges 200 node synthetic smal l world network']\n",
      "['error', 'be', 'where based upon statistical error of data']\n",
      "['similar way', 'in', 'as regression analysis']\n",
      "['10 g', 'in', 'deterministic model']\n",
      "['10 g', 'in', 'stochastic model']\n",
      "['sir model', 'with', 'equal death']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'deaths']\n",
      "['sir model', 'with', 'equal death']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'equal death']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'deaths']\n",
      "['sir model', 'with', 'deaths']\n",
      "['sir model', 'with', 'vital dynamics']\n",
      "['sir model', 'with', 'births']\n",
      "['squared error', 'be', 'in our analysis used as criteria for matching']\n",
      "['3 phase transitions', 'in', 'epidemic models']\n",
      "['b. west', 'application of', 'fractional calculus in physics']\n",
      "['will state i', 'in', 'next time step']\n",
      "['probability mass fraction', 'with distribution', 'small outbreaks']\n",
      "['application', 'be', 'more of our perturbation method here leads again to eﬀective one-dimensional fokker-planck equation along cl']\n",
      "['poisson process', 'with', 'rate ν α']\n",
      "['pontryagin', 'have', 'minimum principle numericcitation']\n",
      "['i.e', 'individual in', 'epidemic process e belongs to x ∼ mp']\n",
      "['total number', 'in', 'focal area']\n",
      "['dynamics', 'in', 'higher dimensional deterministic models']\n",
      "['disease dynamics', 'be', 'how related to environmental covariates']\n",
      "['numericcitation m. draief', 'in', 'proceedings of 1st international conference on performance evaluation methodolgies']\n",
      "['process', 'perform on', 'complete motif with 4 members']\n",
      "['4.1 infection-dominant model', 'in', 'case']\n",
      "['infection seed', 'be', 'where located']\n",
      "['single individuals', 'in', 'contact network']\n",
      "['appendix b', 'summary of', 'main notation used in paper']\n",
      "['simulations', 'with', 'exact age-of-infection model']\n",
      "['proof', 'provide in', 'appendix g']\n",
      "['small outbreak', 'be', 'really small']\n",
      "['interest', 'in', 'hierarchical model']\n",
      "['pourbohloul b. heterogeneous bond percolation', 'with', 'application to epidemic dynamics']\n",
      "['similar model', 'in', 'two dimensions']\n",
      "['branching process approximation', 'in', 'hierarchical model']\n",
      "['small black squares', 'in', 'figure 1']\n",
      "['poisson degree distribution k numericcitation', 'with', 'average degree smaller than 1']\n",
      "['contrast', 'with', 'standard epidemic models']\n",
      "['problem', 'employ', 'wentzel-kramers-brillouin approximation for master equation']\n",
      "['infection', 'with', 'rate λi']\n",
      "['slow strain', 'have', 'competitive advantage for uniformly distributed initial conditions']\n",
      "['38 epidemic spreading', 'in', 'multilayer networks 10']\n",
      "['i', 'have', 'neighbors']\n",
      "['s', '1 for', 's − individuals']\n",
      "['r − individuals', 'for s', '0 1']\n",
      "['s', '0 for', 's − individuals']\n",
      "['distributed time', 'with', 'mean β − 1']\n",
      "['e', 'set of', 'directed links']\n",
      "['2 · 103 square lattice', 'with', 'diﬀerent samples']\n",
      "['annealed network epidemic spreading', 'in', 'multilayer networks']\n",
      "['time varying information spreading rate', 'in', 'our formulation']\n",
      "['two long range contacts', 'with', '2 other nodes']\n",
      "['saturation value rsat', 'attain', 'ﬁnite value']\n",
      "['increase', 'in', 'speed of infection per infectious period']\n",
      "['numericcitation marc barth', 'elemy', 'romualdo pastor-satorras']\n",
      "['numericcitation marc barth', 'elemy', 'alain barrat']\n",
      "['infection', 'become', 'chronic']\n",
      "['contact network numericcitation', 'in cluster', 'many short cycles']\n",
      "['end', 'at number', 'umax to isolation problem']\n",
      "['various networks', 'in', 'homogeneous networks']\n",
      "['various networks', 'in', 'homogeneous networks']\n",
      "['ghanbarnejad', 'impact of', 'individual nodes in boolean network dynamics']\n",
      "['duration t', 'be', 'integer uniformly chosen from numericcitation']\n",
      "['deterministic peaks', 'in', 'power spectra obtained from real data records of childhood diseases']\n",
      "['u-nodes', 'become', 'i-nodes with rate δ']\n",
      "['other site', 'with probability', 'p']\n",
      "['there can switch', 'in', 'case']\n",
      "['density', 'in', 'typical single realization of stochastic model starting from triennial limit cycle']\n",
      "['simple replacement', 'with', 'deterministic realization']\n",
      "['odes', 'use function ode from', 'package desolve']\n",
      "['random variable', 'with', 'distribution p']\n",
      "['results', 'be', 'how aﬀected by infection probability β']\n",
      "['number', 'in', 'instantaneous network']\n",
      "['inhomogeneous models', 'be', 'then introduced']\n",
      "['eigenvalue λ1', 'be', 'known as principal eigenvalue of adjacency matrix also as perron-frobenius eigenvalue']\n",
      "['critical behavior', 'ﬁnd', 'slope − 0.05524 consistent with percolation scaling']\n",
      "['simplest model', 'with', 'duplication corresponding to speciations']\n",
      "['results', 'also hold for', 'sir epidemic on g']\n",
      "['latent period', 'incorporate as', 'exposed state']\n",
      "['recovery reactions', 'be', 'in stochastic sir model modeled as continuous-time markovian processes']\n",
      "['2006 time delays', 'in', 'epidemic models']\n",
      "['studied processes', 'in', 'networks having']\n",
      "['transmission rate', 'proportion of', 'i']\n",
      "['nodes', 'in', 'recovered state']\n",
      "['bond percolation', 'in', 'multilayer networks']\n",
      "['bond percolation', 'in', 'multilayer networks']\n",
      "['a. epidemic dynamics', 'in', 'complex networks']\n",
      "['recovery rate', 'be', 'when large enough']\n",
      "['red free half-edge pairs', 'with', 'free susceptible half-edge']\n",
      "['terms', 'in', 'our rescaled system']\n",
      "['better source detection rates', 'in', 'numerical experiments']\n",
      "['in-community probability', 'deviate from', 'gaussian distribution']\n",
      "['boundary conditions t', 'with d', 'ξ0 t ξ0']\n",
      "['1 2 d', 'with', 'boundary conditions t']\n",
      "['patterns', 'with', 'slightly irregular periods']\n",
      "['patterns', 'with', 'slightly irregular periods']\n",
      "['initially-infected population', 'recover at', 'rate γ']\n",
      "['vertex', 'with', 'degree']\n",
      "['populations', 'with', 'similar sizes']\n",
      "['mean', 'with', 'standard deviation shown in parentheses']\n",
      "['same number', 'with network', 'same number of links per node']\n",
      "['seasonality', 'with epidemic', 'model reduction 1 introduction as dynamical process in epidemics']\n",
      "['generalized network sir model', 'with', 'vital dynamics']\n",
      "['individuals', 'can in', 'can three possible states']\n",
      "['prediction', 'have', 'one more parameter']\n",
      "['k', 'be', 'average degree']\n",
      "['t', 'be', 'where uniformly chosen from numericcitation']\n",
      "['epoch', 'node in', 'susceptible state']\n",
      "['we computations', 'with', 'example 1']\n",
      "['values', 'in', 'region above curve β']\n",
      "['cities', 'connect through', 'airline transportation network']\n",
      "['s', 'with', 'probability']\n",
      "['r', 'with', 'probability c']\n",
      "['probability a +0', 'with a', 'm']\n",
      "['probability a +0', 'with a', 'b']\n",
      "['ba networks', 'with', 'diﬀerent size n']\n",
      "['hki', 'be', 'average degree']\n",
      "['vaccination coverage', 'with', '70 eﬃcacy']\n",
      "['plots', 'in', 'section 6.3 show variation of cost functional j']\n",
      "['agreement', 'with', 'ones related to dynamic percolation universality class numericcitation']\n",
      "['r', 'absence of', 'epidemic threshold in scale-free networks with degree cor-relations']\n",
      "['saddle', 'with', 'two attracting directions in i2s plane']\n",
      "['ts', 'set of', 'corresponding feasible elapsed times']\n",
      "['infected node', 'infect sequentially', 'its neighbors']\n",
      "['notiﬁcations', 'of proportion', 'number']\n",
      "['element', 'in', 'x3']\n",
      "['cluster probability distribution', 'apply to', 'standard percolation models']\n",
      "['limit theorems', 'approximate', 'ordinary diﬀerential processes']\n",
      "['more details', 'will', 'will given in future publication']\n",
      "['outbreak size distributions', 'in', 'epidemics with multiple stages']\n",
      "['stochastic methods', 'of handbook', 'citation']\n",
      "['individuals', 'be', 'latent']\n",
      "['model', 'with', 'sir structure']\n",
      "['tc', 'in', 'er network with hki for tc for tc']\n",
      "['systems', 'with', 'size n']\n",
      "['similar algorithm', 'give by', 'grassberger numericcitation']\n",
      "['parameter values', 'summarize in', 'table 2']\n",
      "['full circle', 'represent', 'absorbing state']\n",
      "['information source detection', 'snapshot of', 'real world network']\n",
      "['two solutions', 'relatively synchronize with', 'one another']\n",
      "['evaluations', 'with', 'dash line estimated weekly as squares also in fig 3']\n",
      "['average distances', 'with', 'given expected degrees']\n",
      "['parameter values', 'in', 'as fig 2a']\n",
      "['such structures', 'eﬀect of', 'signiﬁcant numericcitation']\n",
      "['epidemic spreads easier', 'in', 'independent partition model']\n",
      "['multiplicative noise eﬀects', 'generate', 'similar results to those reported for additive noise']\n",
      "['rumor', 'in', 'social network']\n",
      "['exponential time', 'with', 'rate y']\n",
      "['results', 'provide', 'basis for optimization process']\n",
      "['siri infection spreading', 'use', 'state-space approach numericcitation']\n",
      "['line', 'with', 'epidemiological data']\n",
      "['nodes', 'act as', 'spreaders']\n",
      "['network si model', 'in', '− diag x']\n",
      "['local contact', 'individual from', 'background with probability b dt']\n",
      "['movements', 'be', 'in metapopulation sir models taken into account numericcitation']\n",
      "['can epidemic-free phase', 'in', 'epidemic phase']\n",
      "['numericcitation m. e. j. newman', 'structure of', 'complex networks']\n",
      "['rumor spreading', 'in', 'online social network like facebook']\n",
      "['population collapse', 'be', 'when especially severe']\n",
      "['sir model', 'for size', '1 − st']\n",
      "['parameters', 'with recover', 'r']\n",
      "['excellent agreement', 'in', 'amplitude']\n",
      "['recovered state r. infected state c', 'in', 't t t t0']\n",
      "['such errors', 'in', 'stochastic epidemic modeling impact']\n",
      "['may optimal strategy', 'in', 'even unconstrained optimal control problem']\n",
      "['epidemic spreading', 'in', 'multilayer network ensembles']\n",
      "['such infected individuals', 'in', 'latent period']\n",
      "['ﬂuctuations', 'diﬀerentiating between', 'values']\n",
      "['mathematical epidemic models', 'of time', 'met']\n",
      "['active immunization', 'achieve by', 'vaccine administration numericcitation']\n",
      "['infected individuals', 'in', 'basic sir']\n",
      "['keywords epidemics', 'introduction as', 'dynamical process in epidemics']\n",
      "['epidemic model', 'with', 'varying immunity period']\n",
      "['large population limit', 'with', 'probability tending to 1']\n",
      "['infective vertex', 'with', 'k free half-edges']\n",
      "['individuals', 'can in', 'where can three possible states']\n",
      "['numerical simulations', 'in', 'section']\n",
      "['second abrupt shift', 'in', 'resources']\n",
      "['clusters', 'with', 'infected nodes']\n",
      "['simulation', 'use', 'diﬀerent parameters numericcitation']\n",
      "['model', 'with', 'positive rates']\n",
      "['traditional epidemic models', 'with', 'random diﬀusion']\n",
      "['population size n', 'as limit', 'initial infective']\n",
      "['n', 'be', 'binary']\n",
      "['→ 0', 'as 0', 'right hand side goes to zero']\n",
      "['random graph', 'with', 'given degrees svante janson']\n",
      "['epidemic centrality numericcitation a.lan ˇci', 'c', 'c physica a']\n",
      "['epidemic threshold', 'in', 'case']\n",
      "['system', 'always stay in', 'epidemic regime']\n",
      "['epidemic threshold calculation', 'in', 'madar et']\n",
      "['derivation', 'give in', 'appendix']\n",
      "['recovered individual birth', 'of individual', 'rate']\n",
      "['people', 'with', 'most contacts']\n",
      "['virus spread', 'in', 'networks']\n",
      "['3 such switching', 'absent in', 'stochastic system']\n",
      "['markovian arrival process', 'stand for', 'general service time distribution']\n",
      "['quasi-neutral si1 i2s model', 'in', 'rescaled variables']\n",
      "['quasi-neutral si1 i2r model', 'in', 'rescaled variables']\n",
      "['si1 i2r model', 'in', 'rescaled variables x']\n",
      "['success', 'in', 'general network numericcitation']\n",
      "['tra jectory', 'in', 'epidemic models']\n",
      "['epidemic models', 'with', 'households']\n",
      "['close physical contact', 'with', 'infected bodily ﬂuids']\n",
      "['close physical contact', 'with', 'infected bodily ﬂuids']\n",
      "['epidemics', 'with', 'multiple stages']\n",
      "['posterior probability', 't give', 'observation xtobs at time tobs']\n",
      "['g', 'be', 'in formalism ﬁxed graph g with labelling pr']\n",
      "['i', 'o', 'numericcitation']\n",
      "['sir model', 'with', 'ﬁxed population']\n",
      "['sir model', 'in model', 'ﬁxed population n']\n",
      "['similar problems', 'in', 'bollinger']\n",
      "['total link weight', 'represent', 'strength of economic ties between two countries in network']\n",
      "['single central node', 'in', 'system']\n",
      "['extinction', 'in', 'generalized susceptible-infected-susceptible model']\n",
      "['pontryagin', 'have', 'maximum principle numericcitation']\n",
      "['pontryagin', 'have', 'maximum principle numericcitation']\n",
      "['β', 'be', 'where obtained']\n",
      "['error bars', 'show', '95 credible regions for parameters']\n",
      "['recovered individuals', 'can return to', 'susceptibility on their own']\n",
      "['susceptible agents', 'get', 'when infected']\n",
      "['robustness', 'in', 'tree networks']\n",
      "['system', 'in', 'state']\n",
      "['epidemic outbreak distribution', 'in', 'sis models effectiveness of simulated identi ﬁcation measure unde r cutoff hypothesis']\n",
      "['quantities', 'obey', 'same scaling laws used in percolation theory']\n",
      "['infectious individual', 'with', 'probability ni']\n",
      "['larger global impact', 'in', 'comparison to con']\n",
      "['i', 'become', 'r with recovery rate γ']\n",
      "['time', 'in', 'modiﬁed process']\n",
      "['5 connection', 'in', 'sect 1']\n",
      "['connection', 'with', 'state-dependent delay']\n",
      "['branching process approxiamtion', 'in', 'hierarchical model']\n",
      "['many paths', 'in', 'network pass']\n",
      "['bovine tuberculosis surveillance', 'in', 'dairy farm network']\n",
      "['dairy farm network', 'in', 'emilia-romagna']\n",
      "['dotted lines', 'be', 'within active phase numerical plots of hopf bifurcation curves']\n",
      "['unifo rmly', 'choose', 'value for cumulative distribution']\n",
      "['probability', 'in', 'theorem 5.3']\n",
      "['rewiring algorithms construct', 'with', 'more heterogeneity in clustering']\n",
      "['er network', 'with', 'hki']\n",
      "['er network', 'with', 'hki']\n",
      "['er network', 'with', 'hki']\n",
      "['deterministic sir dynamics', 'with', 'renormalized infection rate βmf']\n",
      "['results', 'also hold for', 'sir epidemic on g']\n",
      "['ties', 'break at', 'random']\n",
      "['numericcitation n. t. j. bailey', 'theory of', 'infectious diseases']\n",
      "['epidemic dynamics', 'in', 'adaptive networks']\n",
      "['recent simultaneous developments', 'in', 'few countries around world numericcitation']\n",
      "['theoretical results', 'with', 'monte carlo simulations on er network in fig 2']\n",
      "['induced death rate', 'in', 'basic sir model']\n",
      "['mean number', 'in', 'general stochastic sir epidemic']\n",
      "['psite c', 'here denote', 'critical value for site percolation']\n",
      "['diagram', 'show', 'relations between variables used in compartmental model']\n",
      "['1.2 connectomics', 'in', 'models of contagious disease networked dynamic systems']\n",
      "['corrected weekly notiﬁcation data', 'hence should provide', 'good approximation for total weekly infections']\n",
      "['eigenvalues λ', 'correspond to', 'active state']\n",
      "['role', 'with', 'zero birth rate']\n",
      "['left vertical scale', 'in', 'figure a']\n",
      "['individuals', 'with', 'very large connectivity']\n",
      "['many neighbors', 'with', 'high degree']\n",
      "['15 epidemic spreading', 'in', 'multilayer networks node']\n",
      "['nearest-neighbor sites', 'in', 'state i']\n",
      "['n', 'be', \"neighborhood 's id\"]\n",
      "['numericcitation m. richardson', 'in', 'proceedings of eighth acm sigkdd international conference on knowledge discovery']\n",
      "['simulation', 'be', 'performed in same conditions as fig 2']\n",
      "['sirs epidemic model', 'with', 'nonlinear incidence rate']\n",
      "['beta distribution', 'with', 'density 1 2 x − 1/2 on numericcitation']\n",
      "['spectral analysis', 'in', 'random geometric networks']\n",
      "['epidemic threshold', 'eﬀect of', 'diﬀerent immunization strategies investigated']\n",
      "['number', 'be', 'when increased']\n",
      "['comparison', 'with', 'random walk as in supercritical case']\n",
      "['numericcitation r.m anderson', 'disease in', 'humans']\n",
      "['gillespie', 'have', 'al 3 figure 1']\n",
      "['numericcitation s. eubank', 'outbreak in', 'realistic urban social networks']\n",
      "['age-structured sir epidemic model', 'with', 'time periodic coeﬃcients']\n",
      "['system', 'with', '100 × 100 nodes']\n",
      "['number', 'with', 'degree k']\n",
      "['≥', 'be', 'random']\n",
      "['vaccination programs numericcitation', 'provide', 'immunity']\n",
      "['population', 'of ea', 't']\n",
      "['table 1', 'show', 'basic information for above mentioned networks']\n",
      "['members', 'in', 'hierarchical model']\n",
      "['hierarchical model', 'in individual', 'p']\n",
      "['ered individuals', 'in', 'hierarchical model']\n",
      "['large outbreak', 'in', 'hierarchical model']\n",
      "['large school', 'in', 'hierarchical model']\n",
      "['large school', 'in', 'hierarchical model']\n",
      "['hierarchical model', 'in −', '1 − ph']\n",
      "['1 −', 'in', 'hierarchical model']\n",
      "['hierarchical model', 'in −', '1 − pg']\n",
      "['stochastic model', 'curve of', 'figure 3b']\n",
      "['larger values', 'in', 'figure 16']\n",
      "['random walk', 'in', 'as supercritical case']\n",
      "['4 discussion', 'in', 'section']\n",
      "['r. infected individuals', 'recover in', 'one step']\n",
      "['contact', 'with', 'single infected node']\n",
      "['numericcitation newman m. e. j', 'with', 'arbitrary degree distributions']\n",
      "['numericcitation newman m. e. j', 'with', 'arbitrary degree distributions']\n",
      "['rumors', 'in', 'network']\n",
      "['as table 1', 'with', 'constraint']\n",
      "['corresponding expression', 'in', 'standard sir model pn']\n",
      "['conclusions', 'be', 'contained respectively in sec']\n",
      "['mc', 'stand for', 'markov chain model']\n",
      "['li', 'spreader by', 'weighted leaderrank']\n",
      "['circle', 'for', 'full infection epidemic network simulated in geographical map with 100 × 100 nodes']\n",
      "['x', \"vector 's\", \"xi 's\"]\n",
      "['zn m', 'in', 'branching model']\n",
      "['three-dimensional phase space i1 i2s', 'in line', 'cl']\n",
      "['n', 'be', 'where set to 104']\n",
      "['results', 'in accordance with', 'above analysis']\n",
      "['vespigniani', 'spread in', 'scale free networks']\n",
      "['sir model', 'in', 'paper']\n",
      "['stochastic model', 'in', 'than deterministic model']\n",
      "['graph', 'with', 'degree sequence']\n",
      "['data', 'with', 'seasonal epidemic peaks']\n",
      "['numericcitation p. sen', 'in', 'recent developments']\n",
      "['numericcitation p. sen', 'in', 'handbook eds']\n",
      "['two populations', 'with', 'size n 1']\n",
      "['two populations', 'with', 'size n 1']\n",
      "['model', 'in', 'our experiments']\n",
      "['terms', 'in', 'sum']\n",
      "['infection', 'in', 'people infected from other viruses']\n",
      "['system', 'correspond to', 'epidemic threshold']\n",
      "['distributions', 'for bind', 'd']\n",
      "['kendall', 'have', 'tau τ for ds centrality']\n",
      "['system', 'tend toward', 'memoryless system']\n",
      "['last line', 'in', 'eq']\n",
      "['control', 'be', 'when selected']\n",
      "['correlations', 'in', 'infected neighbors']\n",
      "['susceptible nodes', 'how fast react to', 'quarantine probability']\n",
      "['hutber', 'prediction during', 'foot-and-mouth disease epidemic']\n",
      "['population size', 'be', 'when large']\n",
      "['system', 'in', 'behavior']\n",
      "['severe population collapses', 'in', 'multi-host epidemic dynamics']\n",
      "['infected node', 'recover at', 'rate γ']\n",
      "['4.1 scalar sis model', 'in', 'scalar sis model']\n",
      "['outbreak statistics', 'be', 'qualitatively diﬀerent for externally driven system than for simple sir']\n",
      "['threshold epidemic value β ∗', 'with', 'tr']\n",
      "['single pathogen', 'with', 'infection rates']\n",
      "['pxi ,0 infective vertices', 'with', 'at least one half-edge']\n",
      "['epidemic simulation system', 'with', 'realistic mobility']\n",
      "['propagation', 'in', 'plants']\n",
      "['individuals', 'in', 'population infected under ic model']\n",
      "['phase transitions', 'in', 'models']\n",
      "['results', 'in', 'section']\n",
      "['mfl method', 'predict', 'epidemic thresholds in networks with positive degree-degree correlations']\n",
      "['infectives', 'with', 'distribution predicted by eq']\n",
      "['clusters', 'with', 'property']\n",
      "['tuberculosis model', 'with', 'reinfection']\n",
      "['stochastic epidemic models', 'note in', 'statistics 151']\n",
      "['response rate', 'in', 'study']\n",
      "['complete motifs', 'must', 'must compensated with other parts of network decomposed into much more random graph-type structure']\n",
      "['system', 'in', 'model at equilibrium state']\n",
      "['node random selection', 'in', 'entire population']\n",
      "['1 introduction infectious diseases', 'impact', 'public health']\n",
      "['nodes', 'in', 'electric grid']\n",
      "['n nodes', 'represent', 'interconnected communities']\n",
      "['second site', 'infective in', 'i']\n",
      "['outbreak', 'be', 'once deterministic']\n",
      "['rate βn', 'in', 'sir']\n",
      "['importance', 'in', 'real epidemics']\n",
      "['method', 'in', 'section iii c']\n",
      "['value', 'be', 'used for following prediction']\n",
      "['v', 'infect node at', 'lowest level on sub-tree']\n",
      "['infected node', 'recover in', 'one discrete time step']\n",
      "['seasonality', 'in', 'epidemic model']\n",
      "['individuals', 'separate into', 'three classes determined by their relative states to given disease']\n",
      "['population', 'in', 'infected state']\n",
      "['local connections', 'with', 'k neighbouring sites']\n",
      "['various problems', 'in', 'population dynamics']\n",
      "['immunization', 'achieve by', 'vaccine administration numericcitation']\n",
      "['given connected network g', 'with', 'n nodes']\n",
      "['c', 'a. in', 'populations with heterogeneous infection rates']\n",
      "['macri p. a', 'in', 'populations with heterogeneous infection rates']\n",
      "['authors', 'in', 'numericcitation aim compute']\n",
      "['sis model', 'in', 'numericcitation']\n",
      "['random graphs', 'with', 'tunable clustering']\n",
      "['vaccine', 'with', 'probability ω']\n",
      "['probability', 'denote by', 'φr']\n",
      "['vaccination', 'with', 'probability ω']\n",
      "['results exact', 'with', 'results obtained for sis dynamics using']\n",
      "['such networks', 'with', 'ﬁnite size numericcitation']\n",
      "['total population size', 'in', 'si1 i2s model']\n",
      "['identical infectivity a', 'of larger', 'higher for same λ from fig 1']\n",
      "['sir model', 'in x', 'word-of-mouth']\n",
      "['1 2 u2ma x', 'in', 'sir model']\n",
      "['109 independent runs', 'in', 'populations as large as n']\n",
      "['k', 'for zti', '− 1 0 with probability 2ε/µ otherwise']\n",
      "['same quantity', 'in', 'independent partition model']\n",
      "['numericcitation g. csardi', 'package for', 'research']\n",
      "['numericcitation n. meinshausen', 'annals of', 'statistics']\n",
      "['numericcitation a. raftery', 'trend in', 'hiv/aids']\n",
      "['numericcitation r. tibshirani', 'shrinkage via', 'lasso']\n",
      "['numericcitation p. tseng', 'method for', 'nonsmooth separable minimization']\n",
      "['models', 'in', 'swn']\n",
      "[\"1 instituto de f'ısica\", 'point of', 'stochastic asynchronous susceptible-infected-recovered model on square lattice']\n",
      "['1 left', 'in', 'network']\n",
      "['pagerank', 'in', 'network']\n",
      "['sism model', 'with', 'm ≥ 2']\n",
      "['sir-d model', 'with', 'village size n']\n",
      "['seasonal epidemic data 13', 'with', 'speciﬁc eigenvalue structure']\n",
      "['seasonal inﬂuenza', 'in', 'general population']\n",
      "['graph g', 'with', 'edges corresponding to potentially infectious contacts']\n",
      "['percolation', 'in', 'random graphs with given vertex degrees']\n",
      "['d', 'with', 'probability']\n",
      "['w', \"'s ouza\", 'm. r. microtransition cascades to percolation']\n",
      "['logarithmic increase', 'with', 'system size numericcitation']\n",
      "['sum', 'in', 'equation']\n",
      "['bethe lattice', 'with', 'coordination number']\n",
      "['dt', 'evolution of', 'fraction']\n",
      "['borge-holthoefer', 'absence of', 'inﬂuential spreaders in rumor dynamics']\n",
      "['memory integrals', 'be', 'fast-time processes']\n",
      "['discrete-time markov chain approach', 'in', 'complex networks']\n",
      "['time t', 'have', 'has rescaled by 1/κ1']\n",
      "['system', 'be', 'nonequilibrium']\n",
      "['remaining transitions', 'only depend on', 'rate parameters']\n",
      "['s i rs', 'refer to', 'three state stochastic process']\n",
      "['graphs', 'with', 'given degrees']\n",
      "['vaccination probability', 'control', 'epidemic with available immunization resources']\n",
      "['pi', 'be', 'hence continuous on θ ∞ 1']\n",
      "['infection time', 'with', 'distribution exp']\n",
      "['s', 'make', 'model supercritical']\n",
      "['other parameters', 'ﬁxed as', 'as table i']\n",
      "['e. d. kilbourne', 'epidemiology in', 'citation']\n",
      "['average rate', 'as β', 'τ']\n",
      "['sir model', 'with', 'vital rates']\n",
      "['normalization factor', 'in', 'contrast to two-dimensional case']\n",
      "['ni', 'k of', 'initially infected vertices 29']\n",
      "['bdi process', 'with', 'ν']\n",
      "['micro-outbreak', 'in', 'bdi process']\n",
      "['probability 0.3', 'with i', 't']\n",
      "['networks', 'with', 'more cycles']\n",
      "['networks', 'with', 'more cycles']\n",
      "['infectious diseases', 'in', 'heterogeneous populations']\n",
      "['dmp equations', 'in', 'numericcitation']\n",
      "['− αk ek i ′ k', 'with', 'pa']\n",
      "['its own community', 'with', 'probability µ']\n",
      "['line', 'with', 'real data']\n",
      "['large-dimensional contact process', 'with', 'random recovery rates on open clusters']\n",
      "['cml', 'with', 'delay numericcitation']\n",
      "['particle', 'in', 'cp model']\n",
      "['smal l time', 'for βλmax', 'weighted average t fast with rate βλmax']\n",
      "['function', 'in', 'population']\n",
      "['network numericcitation', 'of set', 'geodesics']\n",
      "['other words', 'fraction of', 'survivors exp']\n",
      "['increase', 'with', 'increasing k for given m']\n",
      "['evolution', 'in', 'networks with community structure']\n",
      "['event', 'in', 'size range between large-scale epidemic']\n",
      "['spontaneous reactions', 'in', 'epidemic processes']\n",
      "['their susceptible neighbors', 'with', 'probability β']\n",
      "['numericcitation g', 's', 'borge-holthoefer j']\n",
      "['numericcitation g', 's', 'arenas a']\n",
      "['numericcitation g', 's', 'meloni s']\n",
      "['case', 'well describe by', 'directed percolation numericcitation']\n",
      "['second matrix block', 'will possess', 'eigenvalues with negative real part']\n",
      "['epidemic threshold', 'in', 'con ﬁguration model numericcitation']\n",
      "['third matrix block', 'will possess', 'eigenvalues with zero real part']\n",
      "['epidemic model', 'with', 'stage structure']\n",
      "['sample probability', 'be', 'when larger than 6']\n",
      "['individuals', 'be', 'where are continuously infected']\n",
      "['parameter values', 'in', 'reasonable ranges taken from literature']\n",
      "['biased random walk', 'as ≥', 'ρ']\n",
      "['st', 'number at', 'time']\n",
      "['inﬁnite subharmonic bifurcations', 'in', 'seir epidemic model']\n",
      "['connections', 'large jority of', 'vertices in same group']\n",
      "['virus spread', 'in', 'networks']\n",
      "['variations', 'in', 'autoinfection probabilities']\n",
      "['disease', 'can', 'can transmitted from one plant to its nearest neighbors']\n",
      "['random multigraph', 'with', 'given degree sequence']\n",
      "['τ', 'be', 'in mean-ﬁeld sis model basic reproduction number']\n",
      "['spread', 'in', 'outbreak settings']\n",
      "['epidemic threshold', 'in', 'wm networks']\n",
      "['seasonally forced deterministic seir epidemic model', 'in', 'section']\n",
      "['average distances', 'in', 'random graphs']\n",
      "['results', 'in', 'section iii']\n",
      "['citation -2012', 'name', '95 interval']\n",
      "['l1', 'size of', 'indeed l2 with probability p k − l1 − 1 in way']\n",
      "['sir', 'with', 'ﬁxed recovery time']\n",
      "['outbreak', 'be', 'once suﬃciently large']\n",
      "['infected individuals', 'be', 'infertile']\n",
      "['other layer', 'have', 'poisson interdegree distribution with average smaller than one']\n",
      "['conﬁguration model', 'with', 'bounded vertex degrees']\n",
      "['internet autonomous systems', 'between communication', '31 citation']\n",
      "['mathematical model', 'in', 'population']\n",
      "['numericcitation centers', 'in', 'chief']\n",
      "['sir epidemic model', 'with', 'deaths']\n",
      "['links', 'in', 'network']\n",
      "['reasonable augmentations', 'lead to', 'dramatic changes in long time behavior']\n",
      "['dynamic percolation universality class', 'in', 'agreement with conjecture by grassberger numericcitation']\n",
      "['weighted average t', 'with', 'rate βλmax']\n",
      "['si1 i2r model', 'with', 'population turnover']\n",
      "['b. si1 i2r model', 'with', 'population turnover']\n",
      "['si1 i2r model', 'with', 'population turnover']\n",
      "['c. si1 i2r model', 'with', 'population turnover']\n",
      "['si1 i2r model', 'with', 'population turnover']\n",
      "['individuals', 'in', 'two states']\n",
      "['same function', 'in', 'mathematically models']\n",
      "['series', 'in', 'quantitative methods for decision making']\n",
      "['epidemic spreading', 'in', 'school']\n",
      "['α', 'be', 'where infection rate']\n",
      "['measles', 'in', 'england']\n",
      "['measles', 'in', 'england']\n",
      "['bartlett m s stochastic population models', 'in', 'ecology']\n",
      "['coupled systems', 'with', 'time delayes']\n",
      "['numericcitation volz e. sir dynamics', 'in', 'random networks with heterogeneous connectivity']\n",
      "['discussion', 'in', 'appendix sec.ii']\n",
      "['distinct individuals', 'in', 'z f']\n",
      "['recovery time tr', 'also replace by', 'recovery rate γ']\n",
      "['riley s', 'model of', 'infectious disease']\n",
      "['f', 'dr dt from', 'then eq']\n",
      "['fig', 'summarize', 'tab']\n",
      "['fig 5', 'summarize', 'tab']\n",
      "['populations', 'with', 'spatial structures in humans']\n",
      "['1 n threshold', 'with', 'ω tending to inﬁnity slowly as population size']\n",
      "['function', 'as r', '∞']\n",
      "['function', 'as r', 'n']\n",
      "['function', 'as r', '∞']\n",
      "['function', 'as r', 'b']\n",
      "['limit distribution', 'with', 'edge probability p']\n",
      "['sensitive dependence', 'in', 'short noisy non-chaotic time series such epidemic data']\n",
      "['one cluster', 'in', 'epidemic sample']\n",
      "['infections', 'have spread up to', 'time t']\n",
      "['numericcitation n. g. van kampen', 'process in', 'physics']\n",
      "['infected individual', 'once have recover from', 'infection']\n",
      "['system', 'have', 'nonlinearities']\n",
      "['o self-avoiding paths', 'with', 'length n staring at o']\n",
      "['variance', 'in', 'ili time series']\n",
      "['numericcitation n. t. j. bailey', 'theory of', 'infectious diseases']\n",
      "['forced seir model', 'in', 'realistic parameter region for childhood infectious diseases']\n",
      "['c. fraser et', 'potential of', 'strain']\n",
      "['stochastic epidemic-type model', 'with', 'enhanced connectivity']\n",
      "['model', 'be', 'how ﬁt to real seasonal data']\n",
      "['sl −', 'in', 'voronoi']\n",
      "['results', 'with', 'substituted temperature-dependent reaction rate mentioned above']\n",
      "['scientiﬁc', 'report', '19767 2016']\n",
      "['single site probabilities', 'in', 'stochastic sirs epidemic process']\n",
      "['work', 'with', 'deterministic setting']\n",
      "['t ≥ 0', 'with', 'random edge b']\n",
      "['i', 'dτ', 'dµ dτ']\n",
      "['sir model', 'with', 'waning immunity s ′']\n",
      "['p', 'set of', 'counting problems associated with decision problems in set np']\n",
      "['secondary averaging', 'consider', '100 diﬀerent network conﬁgurations']\n",
      "['random graphs', 'with', 'given degree sequence']\n",
      "['random graphs', 'with', 'given degree sequence']\n",
      "['random graphs', 'with', 'given degree sequence']\n",
      "['similar results', 'in', 'current literature restricted to tree networks']\n",
      "['s-node', 'be', 'then infected by i-node again']\n",
      "['transition', 'show up later in', 'stochastic system']\n",
      "['t', 'be', 'at time represented by variable xt process']\n",
      "['numerous studies', 'in', 'stochastic models with synchronous update numericcitation']\n",
      "['network sis model sections', 'include', 't']\n",
      "['rsat', 'have', 'scaling form similar to magnetization']\n",
      "['numerical spreading experiments', 'in', 'ba networks']\n",
      "['degree distribution', 'of result', 'kmin at diﬀerent times']\n",
      "['numericcitation m. kimmel', 'process in', 'biology']\n",
      "['numericcitation g. odor', 'universality in', 'nonequilibrium lattice systems']\n",
      "['kermackmckendrick sir model', 'with', 'poisson transmission']\n",
      "['informed consent', 'in', 'cover letter attached to their surveys']\n",
      "['transition temperature', 'show', 'logarithmic increase with system size numericcitation']\n",
      "['hollow circle', 'for', 'system with 100 × 100 nodes']\n",
      "['time change', 'in', 'process']\n",
      "['comparison', 'hold also for', 'general stochastic sir epidemic']\n",
      "['their shape', 'correlate for', 'even such simple model']\n",
      "['their shape', 'suﬃciently correlate for', 'even such simple model']\n",
      "['larger τ', 'correspond to', 'better performance']\n",
      "['concentration', 'in', 'as theorem 3.1']\n",
      "['isolation', 'in', 'sir framework with limited resources']\n",
      "['our algorithm', 'can', 'can also extended']\n",
      "['individual s', 'in', 'contact with infected individual']\n",
      "['t', 'denote at', 'time']\n",
      "['epidemic outbreaks', 'obviously involve', 'ﬁnite populations']\n",
      "['epidemic outbreaks', 'involve', 'ﬁnite populations']\n",
      "['model selection', 'in', 'gaussian graphical models']\n",
      "['ways disease spreads', 'in', 'biological network']\n",
      "['ξ1', '· for', 'given connected network g with n nodes']\n",
      "['curves', 'for', 'one with 1000 × 1000 nodes']\n",
      "['nonreversible epidemic spreading dynamics', 'in', 'sir modeled']\n",
      "['plots', 'split into', 'two ﬁgures for clarity']\n",
      "['si1 i2r model', 'with', 'population turnover involved']\n",
      "['nr', 'k of', 'vertices']\n",
      "['st', 'number of', 'vertices']\n",
      "['b simulated distribution', 'in', 'sis model ρr']\n",
      "['sis model ρr', 'in distribution', 'λ']\n",
      "['eﬀective way', 'in', 'other words']\n",
      "['better results', 'introduce degree-dependence in', 'infection probability inferred from observation']\n",
      "['disease', 'with', 'ﬁxed β']\n",
      "['s s f i r epidemiological model', 'in', 'complex networks']\n",
      "['signiﬁcant turf restrictions', 'in', 'place']\n",
      "['line', 'with', 'observation made in main texts']\n",
      "['¯ x', 'be', 'equilibrium points']\n",
      "['more detail', 'in', 'section v']\n",
      "['step 2', 'consider', 'i.e y ∈ y n +1']\n",
      "['k ≥ 3', 'for d', 'ηk − ηk − 1']\n",
      "['technical report', 'bank of', 'st. louis']\n",
      "['technical report', 'bank of', 'st. louis']\n",
      "['technical report', 'bank of', 'st. louis']\n",
      "['indication', 'in', 'stochastic system']\n",
      "['basic reproduction number', 'in', 'mean-ﬁeld sis model']\n",
      "['sir epidemic model', 'with', 'partial temporary immunity modeled']\n",
      "['lower red dotted curve', 'of upper', 'green dashed-dotted curve']\n",
      "['490 505 numericcitation calsina farkas jz steady states', 'in', 'structured epidemic model with wentzell boundary condition']\n",
      "['eigenvalue', 'with', 'largest absolute value in above jacobian matrix']\n",
      "['ﬂuctuation probability', 'in', 'equilibrium systems determined by boltzmann distribution']\n",
      "['networks', 'with', 'mean connectivity 4.07 in giant component']\n",
      "['models', 'in', 'munz et']\n",
      "['id', 'in', 'message']\n",
      "['erd ¨ os', 'of feature', 'degree hki']\n",
      "['upper-and lower bounds', 'd for', 'details']\n",
      "['k core', 'of size', 'diameter numericcitation']\n",
      "['free infective half-edge pairs', 'with', 'free recovered half-edge']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['downton', 'at', 'carrier-borne epidemic model']\n",
      "['infection', 'spread over', 'whole lattice']\n",
      "['jansson s', 'law of', 'large numbers for sir epidemic on random graph with given degrees']\n",
      "['mandatory notiﬁcation', 'in', 'uk citation']\n",
      "['new-product adoption model', 'journal of', 'optimization theory']\n",
      "['contradictory results', 'again obtain in', 'two diﬀerent studies']\n",
      "['time scales', 'in', 'fractional-and integer-order dynamics']\n",
      "['sir processes', 'be', 'indeed always characterized']\n",
      "['uniqueness', 'be', 'for iterative algorithm rigorously proved']\n",
      "['nontrivial stable ﬁxed point', 'with', 'high probability']\n",
      "['infectious burden', 'in', 'optimal control problem']\n",
      "['thresholds', 'in', 'epidemic spreading models such susceptible-infected-recovered sir model numericcitation']\n",
      "['reports', 'in', 'physics 80 036603']\n",
      "['such cluster 1', 'in', 'hierarchical partition model']\n",
      "['way', 'with', 'propagation of electric charge through 2 brain network during seizure']\n",
      "['individuals', 'exchange', 'information']\n",
      "['individuals', 'activate', 'previous connections']\n",
      "['individuals', 'activate', 'connections']\n",
      "['mathematics', 'in research', '3810-193 aveiro']\n",
      "['results', 'mean', 'rmse']\n",
      "['driven models', 'be', 'in their basic formulation activity memoryless']\n",
      "[\"collectif l'epid\", \"d'eveloppement en\", 'france']\n",
      "['p', 'with', '2.4 b j a.s. as n → ∞']\n",
      "['cross-correlation', 'measure', 'similarity between two time series']\n",
      "['5.1 scalar sir model', 'in', 'model individuals']\n",
      "['stable annual limit cycles', 'in', 'deterministic model']\n",
      "['single node', 'in', 'sirs-with-vaccination model']\n",
      "['seir epidemic model', 'with', 'historical citation -1966']\n",
      "['actual births', 'in', 'our model']\n",
      "['information theory proceedings', 'on symposium', 'citation']\n",
      "['numericcitation z. wang', 'in', 'proc']\n",
      "['y ∗', 'be', 'equilibrium']\n",
      "['6 2 proofs', 'in', 'section']\n",
      "['lattice-based sir model', 'be', 'where relevant']\n",
      "['24 eric forgoston', '−', 'i']\n",
      "['24 eric forgoston', '−', 'i']\n",
      "['cosine terms', 'in', 'eq']\n",
      "['nodes changes', 'in', 'response to node information']\n",
      "['xk', 'k of', 'degree k vertices']\n",
      "['0 ∞ xk', 'k of', 'infective vertices']\n",
      "['β γ', 'trajectory from', 'initial condition x 0 converges to exponential ly stable equilibrium x ∗']\n",
      "['our simulations', 'average', 'over 10000 network realizations']\n",
      "['simulation', 'result on', 'diﬀerent networks']\n",
      "['drastic reduction', 'in', 'βef f n']\n",
      "['xt i. introduction many real-world systems', 'can', 'can described by complex networks']\n",
      "['small increases', 'in', 'indoor size metrics including indoor area']\n",
      "['21 6 connection', 'with', 'dde models delay models']\n",
      "['epidemic threshold', 'give by', 'eq']\n",
      "['convergence', 'in', 'probability']\n",
      "['mixture distribution yk − 1', 'with', 'probability k']\n",
      "['probability', 'in', 'section']\n",
      "['yk − 1 0', 'with', 'probability µr/µ']\n",
      "['probability', 'αm before', 'log n']\n",
      "['ηn → 0', 'consider', 'sir epidemic on random graph']\n",
      "['theorem 4.2', 'uniqueness of', 'solution to sis model']\n",
      "['contacts', 'with', 'susceptible leads to transmission']\n",
      "['random variables', 'in', 'numericcitation']\n",
      "['location', 'removal of', 'infected individuals from community numericcitation']\n",
      "['survival probabilities', 'with', 'random edge weights xiaofeng xue ∗ beijing jiaotong university abstract']\n",
      "['lattices', 'with', 'same coordination number k']\n",
      "['contact', 'with', 'infected individuals']\n",
      "['number xi', 't of', 'free infective half-edges']\n",
      "['average distances', 'in', 'isolated components']\n",
      "['average degrees', 'in', 'non-largest components']\n",
      "['reed-frost assumption', 'in', 'temporal behaviour']\n",
      "['its opinion', 'with', 'probability p']\n",
      "['epidemic spreading', 'in', 'real networks']\n",
      "['r0', 'in', 'figure a']\n",
      "['r0', 'in', 'figure b']\n",
      "['r0', 'in', 'figure a']\n",
      "['variables s', 'stand for', 'probability']\n",
      "['stochastic sir epidemic models', 'with', 'random']\n",
      "['disease spreading', 'be', 'where described by sir-model']\n",
      "['numericcitation j. mcauley', 'in', 'proc']\n",
      "['spread', 'in', 'population']\n",
      "['spread', 'in', 'population']\n",
      "['size', 'in', 'supercritical random graph']\n",
      "['size j', 'with', 'probability 1']\n",
      "['size 1', 'with', 'probability 1 −']\n",
      "['j', 'with', 'probability']\n",
      "['j', 'with', 'probability 1']\n",
      "['states', 'in', 'at principle']\n",
      "['mean cluster size', 'in', 'percolation theory']\n",
      "['sfns', 'with', 'structural cutoff in figs 4']\n",
      "['unknown probability', 'in', 'sir network']\n",
      "['e → numericcitation probability', 'in', 'sir network']\n",
      "['e → numericcitation probability', 'in', 'sir network']\n",
      "['susceptibleˆae probability', 'in', 'sir network']\n",
      "['diﬀerent parameters', 'in', 'than initial epidemics']\n",
      "['distinct class', 'class in', 'agreement with conjecture by grassberger numericcitation']\n",
      "['random one', 'be', 'when established']\n",
      "['r', 'be', 'where given']\n",
      "['their eﬃcacy', 'in', 'related epidemic model']\n",
      "['initial conditions', 'with u', 't']\n",
      "['initial conditions', 'with u', 't ∈']\n",
      "['stability', 'with', 'high probability']\n",
      "['expected degree', 'with', 'high probability']\n",
      "['ω ∈ ωd', 'for ω', '·']\n",
      "['citation detection', 'parameter of', 'model identiﬁed from world health organization data']\n",
      "['x', 'be', 'random']\n",
      "['universit', 'sabatier', 'france e-mail']\n",
      "['various immunization strategies', 'have', 'have formulated for urban populations']\n",
      "['immunization strategies', 'have', 'have formulated for urban populations']\n",
      "['threshold', 'in', 'sense']\n",
      "['figure 5', 'show', 'results']\n",
      "[\"i 's activity\", 'decrease', 'due to illness']\n",
      "['people', 'be', 'when infected']\n",
      "['people', 'be', 'when infected']\n",
      "['2012 outbreak size distributions', 'in', 'epidemics with multiple stages']\n",
      "['two solutions', 'with', 'quantity 1/λ']\n",
      "['re-optimization step', 'in', 'iteration']\n",
      "['diﬀerence', 'with', 'less dramatic eﬀect for s i s epidemics']\n",
      "['ω', 'with', 'measure νω × π']\n",
      "['numericcitation newman', 'spread of', 'epidemic disease']\n",
      "['simulation', 'use', 'diﬀerent starting parameters numericcitation']\n",
      "['epidemic jumps', 'in', 'infection levels']\n",
      "['single key parameter', 'r0 to', 'βs']\n",
      "['d', 'be', 'random']\n",
      "['knk', 'be', 'random']\n",
      "['sir model', 'with', 'arbitrary recovery rate']\n",
      "['sir model', 'with', 'arbitrary recovery rate']\n",
      "['great importance', 'in', 'countries']\n",
      "['more theoretical developments', 'have', 'since then have presented in literature']\n",
      "['perra et al', 'self-initiated social distance into', 'classical sir model']\n",
      "['culling control', 'in', 'optimal control problem deﬁned']\n",
      "['analysis', 'in', 'section 4']\n",
      "['slow strain', 'observe earlier in', 'models with ﬁxed total population size numericcitation']\n",
      "['third matrix block', 'have', 'eigenvalues with zero real part30 ,31']\n",
      "['epidemic ﬂows', 'in', 'two social networks with community structure']\n",
      "['graphs', 'with', 'given degree sequence']\n",
      "['≤ r ≤', 'in', 'simulations']\n",
      "['random graphs', 'with', 'ﬁxed degree sequence']\n",
      "['diﬀerent levels', 'in', 'stochastic sir models']\n",
      "['second matrix block', 'have', 'eigenvalues with negative real part']\n",
      "['qy', 'be', 'cube centered at y with length 1']\n",
      "['results', 'in', 'eqs']\n",
      "['main stochastic peak', 'be', 'where signiﬁcantly namics']\n",
      "['infected individuals', 'become', 'immune']\n",
      "['t', 'for ∅', 'τ0 τk']\n",
      "['ω ∈ a', 'for ∅', 'τ0 τk']\n",
      "['ξ0', 'in', 'agreement with refs']\n",
      "['running time', 'bind by', 'polynomial']\n",
      "['running time', 'bind by', 'polynomial']\n",
      "['si1 i2r model', 'for pm', 'n']\n",
      "['si1 i2r model', 'of realization', 'upper panel']\n",
      "['geometric ratchet mechanism', 'importantly independent of', 'speciﬁc competition model']\n",
      "['si1 i2r model', 'for v', 'ξ']\n",
      "['si1 i2s model', 'for v', 'ξ']\n",
      "['si1 i2s model', 'for −', 'ξ0']\n",
      "['eﬀect', 'especially pronounce in', 'si1 i2r model']\n",
      "['si1 i2r model', 'behave diﬀerently at', '→ 0']\n",
      "['expression', 'very cumbersome for', 'si1 i2r model']\n",
      "['si1 i2r model', 'for point', 't ξ0']\n",
      "['negative opinion', 'in', 'favor on vaccination']\n",
      "['poisson degree distribution k numericcitation', 'with layer', 'layer 2']\n",
      "['branching process', 'with', 'single cal ly 1 −']\n",
      "['facebook 0 0 50 100 150 200 0.01 0.02 0.03 0.04 0.05 0.06 0.07 0.08 0.09 0.1 time t i n f c t e d p r o p o r t i n o ρ sir facebook', 'rewire', 'facebook fig 2']\n",
      "['critical process', 'with', 'three oﬀspring']\n",
      "['interactions', 'with', 'k neighbouring sites']\n",
      "['will part', 'with', 'probability p1']\n",
      "['model', 'with', 'time-varying environmental covariate']\n",
      "['model', 'only consider', 'direct transmission']\n",
      "['other new approaches', 'in', 'ﬁeld']\n",
      "['disease', 'with', 'β']\n",
      "['little vaccines', 'stop', 'disease']\n",
      "['reduced markov chain', 'in', 'steady state']\n",
      "['phase transition', 'with', 'arbitrary recovery rate']\n",
      "['assessing node risk', 'in', 'epidemics on networks']\n",
      "['agents', 'in', 'system']\n",
      "['infection rate β', 'depend on', 'population size n']\n",
      "['outbreak sizes', 'be', 'ﬁnite']\n",
      "['basic reproduction number', 'in', 'deterministic scalar sis model']\n",
      "['initial conditions', 'in', 'plot denoted by a']\n",
      "['shell number', 'predict', 'spreaders']\n",
      "['number changes', 'with', 'β']\n",
      "['rich spatial detail', 'in', 'our dataset']\n",
      "['phase', 'with', 'one another']\n",
      "['l', 'number of', 'latently infected/exposed individuals']\n",
      "['incoming individuals', 'of proportion', 'cp']\n",
      "['figure 1c', 'show', 'basic model proposed by munz et']\n",
      "['science', 'of school', 'x1 at rate 1']\n",
      "['number', 'in', 'branching envelope']\n",
      "['md', 'provide', 'largest variance in local clustering']\n",
      "['disease', 'have', 'fewer initial vaccinated nodes']\n",
      "['real epidemics', 'in importance', 'ideia']\n",
      "['contact', 'with', 'infected individuals']\n",
      "['line', 'with', 'real data']\n",
      "['degree', 'one of', 'three measures']\n",
      "['standard binomial random graph', 'in', 'sparse regime']\n",
      "['methods', 'predict', 'numerical threshold']\n",
      "['dashed lines', 'represent', 'limit']\n",
      "['model', 'ﬁnding', 'excellent agreement with stochastic simulations']\n",
      "['1000 simulations', 'with', 'mean degree c']\n",
      "['power', 'on infection', 'ri']\n",
      "['wiley series', 'in', 'probability']\n",
      "['time', 'in', 'mcs']\n",
      "['sum', 'with', 'mean nβ si ∆ t']\n",
      "['many models', 'have', 'have considered']\n",
      "['ring vaccination', 'always cost', 'less than mass vaccination']\n",
      "['interest', 'in', 'sir model on networks']\n",
      "['smaller epidemics', 'in', 'regions']\n",
      "['contact model', 'in', 'other words m']\n",
      "['nodes dynamical inﬂuences', 'in', 'susceptible-infected-r ecovered spreading model 32']\n",
      "['j', 'behavior of', 'hepatitis b virus transmission model with vaccination']\n",
      "['small system', 'with', '100 × 100 nodes']\n",
      "['m. leng', 'region in', 'large networks']\n",
      "['main results', 'in', 'sir model']\n",
      "['function', 'in', 'as classical mechanics']\n",
      "['stochastic processes', 'in', \"spatially structured systems based on gillespie 's method numericcitation\"]\n",
      "['expected number', 'in', 'same community']\n",
      "['power-law distribution r', 'with', 'cutoff at value']\n",
      "['set', 'with', 'at least one observation up to snapshot time']\n",
      "['logit transformation', 'use for', 'probability ρ']\n",
      "['p', 'in', 'scaling depending on number introducing']\n",
      "['random walk', 'in', 'space']\n",
      "['fractions', 'in', 'diﬀerent disease states']\n",
      "['identifying infection sources', 'in', 'large networks']\n",
      "['individuals', 'once recover from', 'infection']\n",
      "['susceptible individual', 'in', 'contact with infected one']\n",
      "['initial infectives', 'set of', 'where single individual']\n",
      "['individual nodes', 'in', 'boolean network dynamics']\n",
      "['pontryagin', \"'s principle\", '15 maximizing for entire epidemic period']\n",
      "[\"people 's global location\", 'in', 'networks']\n",
      "['infected individuals', 'recover permanently with', 'rate µ']\n",
      "['synthetic models', 'with', 'real world system']\n",
      "['ebola virus', 'be', \"transmitted to initial human by contact with infected animal 's body ﬂuid\"]\n",
      "['counters', 'be', 'set']\n",
      "['action s', 'be', 'since ht i ∼ en s comparable to lnht i/n']\n",
      "['ba networks', 'with', 'diﬀerent sizes']\n",
      "['spreading rate', 'be', 'when small']\n",
      "['probability vecis', 'in', 'following forms']\n",
      "['optimal control', 'in', 'conclusion']\n",
      "['optimal control', 'on morton', 'k. wickwire']\n",
      "['square lattice l2', 'with', 'periodic boundary conditions']\n",
      "['parameter estimation', 'apply to', 'dynamical models']\n",
      "['parameter estimation', 'in', 'dynamical models']\n",
      "['epidemic threshold', 'clearly larger than', 'than ml systems']\n",
      "['sir epidemic dynamics', 'with', 'arbitrary recovery rate on scale-free networks']\n",
      "['eﬀective reproduction number', 'in', 'fully-healthy susceptible population']\n",
      "['flu', 'example of', 'model']\n",
      "['δ', 'in', 'model']\n",
      "['results', 'give in', 'fig']\n",
      "['taylor series expansion', 'in', 'v']\n",
      "['unexpected dynamics', 'in', 'epidemic outbreaks']\n",
      "['researchers', 'have focus because of', 'their important rami ﬁcatio ns in many real-world scenarios']\n",
      "['periodic change', 'in', 'disease transmission']\n",
      "['propagation', 'in', 'adaptive complex networks']\n",
      "['remark 2.5 examples', 'in', 'theorem 2.4']\n",
      "['line', 'in', 'previous paragraph']\n",
      "['diﬀerent mathematical models', 'have', 'have used']\n",
      "['epidemic outbreaks', 'with', 'synchronous updating on scale-free networks']\n",
      "['our discussion', 'in', 'section v']\n",
      "['infected nodes', 'of queue', 'v']\n",
      "['infected nodes', 'of queue', 'v']\n",
      "['node', 'have', 'neighbors']\n",
      "['recovery rate', 'have', 'remarkable effect on sir epidemic dynamics with synchronous updating']\n",
      "['ω1', 'in', 'eq']\n",
      "['vaccinated neighbors', 'with', 'probability']\n",
      "['oscillatory phase', 'be', 'why seen as absorbing phase in stochastic simulations']\n",
      "['predictive distribution', 'number of', 'infected individuals']\n",
      "['helper lines', 'mark', 'frequencies for triennial limit cycle']\n",
      "['average imprecision', 'with', 'β']\n",
      "['average imprecision', 'with', 'β']\n",
      "['average imprecision', 'with', 'β']\n",
      "['household epidemic', 'always restrict to', 'one household']\n",
      "['information spreading', 'in', 'population']\n",
      "['gamma-distributed infectious period', 'with', 'constant infectiousness']\n",
      "['factors', 'be', 'when zero']\n",
      "['pastor-santorras', 'in', 'complex networks']\n",
      "['lattices', 'with', 'coordination number k']\n",
      "['lattices', 'with', 'coordination number k']\n",
      "['setting y', 'in', 'sum on smallest k term']\n",
      "['opinion', 'with', 'probability q']\n",
      "['p xt t', 'xt at', 'time t']\n",
      "['node', 'in', 'state b']\n",
      "['node', 'in', 'state a']\n",
      "['multi-type branching process', 'with', 'production operator m arising as above from kernel µ']\n",
      "['multiple infectious disease outbreaks', 'proceed of', 'royal society b 274 833 837']\n",
      "['diagonal form', 'with', 'eigenvalue structure']\n",
      "['crops', 'be', 'when cultivated in country ﬁelds']\n",
      "['disease', 'of', 'endemic state']\n",
      "['sir dynamics', 'in', 'random networks']\n",
      "['random networks', 'with', 'heterogeneous connectivity']\n",
      "['nodes', 'in', 'network']\n",
      "['network', 'have', 'nodes']\n",
      "['conclu s ion', 'in', 'paper']\n",
      "['rules', 'of demonstration', 'recovery time tr']\n",
      "['essential proteins', 'in', 'protein interaction networks 13']\n",
      "['chaos', 'in', 'epidemics']\n",
      "['results', 'be', 'simple']\n",
      "['seir', 'epidemic', 'idea about their clinical features']\n",
      "['b d', 'distribution of', 'infected probability']\n",
      "['infected probability', 'of value', 'base 10']\n",
      "['time', 'in', 'its randomized version considering']\n",
      "['deﬁnition', 'in', 'two models']\n",
      "['as algorithm', 'in', 'section iii']\n",
      "['inﬁnite', 'in', 'size']\n",
      "['deterministic size e', 'generally smaller than', 'average stochastic epidemic size e']\n",
      "['epidemic threshold', 'shift to', 'larger values']\n",
      "['case', 'for 9', 'combining eqs']\n",
      "['notiﬁcation data', 'in', 'numericcitation']\n",
      "['precise methodology', 'use in', 'numericcitation']\n",
      "['epidemic threshold', 'in', 'basic activity']\n",
      "['diﬀerent l × l square lattices', 'with', 'l 128 256']\n",
      "['diﬀerent l × l square lattices', 'with', 'l 128 256']\n",
      "['interactions', 'be', 'predominantly local']\n",
      "['interactions', 'in', 'spatio-temporal graphical model']\n",
      "['mixed si epidemic dynamics', 'with', 'general degree distributions']\n",
      "['random graphs', 'with', 'power law degree distributions']\n",
      "['present theoretical study', 'therefore could provide', 'useful understanding for large ﬂuctuation eﬀect on opinion dynamics']\n",
      "['numericcitation n. bailey', 'theory of', 'infectious diseases']\n",
      "['stochastic eﬀects', 'in', 'seasonally forced epidemic model g. rozhnova1 2']\n",
      "['reduction', 'with', 'vital dynamics to abel type equation']\n",
      "['random networks', 'with', 'heterogeneous connectivity']\n",
      "['single networks', 'in', 'locally tree-like approximation']\n",
      "['transition', 'in', 'sir model deﬁned on square lattice']\n",
      "['network', 'with', 'probability pr']\n",
      "['i.i.d', 'with', 'binomial']\n",
      "['basic sir model', 'with', 'vital dynamics']\n",
      "['our work', 'model', 'sars data for hk']\n",
      "['number', 'in', 'general stochastic epidemic']\n",
      "['k', 'denote', 'time']\n",
      "['populations', 'with', 'seasonal forcing']\n",
      "['inﬁnite regime', '10 for', 'only ν']\n",
      "['stochastic epidemic seir model', 'with', 'control intervention']\n",
      "['diﬀerent works', 'characterize', 'optimal controls']\n",
      "['susceptible network', 'overcome', 'second order transition with mean ﬁeld exponents']\n",
      "['− → 0', 'of sup', 'number']\n",
      "['number', 'in', 'section 4.3.1']\n",
      "['disease', 'with', 'rate ρ1']\n",
      "['epidemics', 'in', 'large unstructured homogeneously mixing populations']\n",
      "['diﬀerences', 'in', 'initial conditions']\n",
      "['size', 'with', 'given vertex degrees under weakened assumptions']\n",
      "['time', 'be', 't 1 c']\n",
      "['i', 'denote', 'individuals']\n",
      "['lemma 6', 'lemma in', 'sugitani']\n",
      "['lemma 3.1', 'in', 'sugitani']\n",
      "['0 xz pi', 'proof of', 'lemma 9']\n",
      "['l', 'event', 'at most log n vertices will infected']\n",
      "['background', 'with', 'probability b dt']\n",
      "['ηx dt', 'with', 'probability 1 − γ']\n",
      "['r0', 'in', 'agreement with expressions']\n",
      "['team modeling', 's without', 'control s with control x']\n",
      "['type', 'with', 'edge pointing towards individual under consideration']\n",
      "['epidemic simulations', 'be', 'mutually independent']\n",
      "['computed trajectories', 'with', 'α between 0.000']\n",
      "['unimodal shape', 'with', 'peak in intermediate connectivity range']\n",
      "['kµ', 'in', 'other words for disease-free equilibrium state']\n",
      "['individuals', 'in', 'removed class']\n",
      "['number', 'in', 'n']\n",
      "['number', 'in', 'example']\n",
      "['spread', 'in', 'moving population']\n",
      "['random variable y', 'in', 'numericcitation']\n",
      "['contacts', 'with', 'infected people']\n",
      "['shape', 'in', 'diﬀerent scenarios']\n",
      "['mean ﬁeld based analysis', 'provide', 'wrong result']\n",
      "['susceptibility', 'be', 'for sir spreading highly correlated to nodes inﬂuence']\n",
      "['precise size', 'in', 'subcritical case']\n",
      "['sis model', 'in', 'continuous-time case']\n",
      "['second gain term', 'in', 'rate equation']\n",
      "['corresponding assumption', 'in', 'our original reference paper numericcitation']\n",
      "['time-optimal control problem', 'in', 'sir']\n",
      "['ba networks', 'with', 'diﬀerent average degrees']\n",
      "['connected network g', 'with', 'adjacency matrix a']\n",
      "['sis model', 'with', 'β']\n",
      "['simple sir process', 'describe', 'data']\n",
      "['cholera transmission dynamics', 'in', 'our model']\n",
      "['random multigraph', 'be', 'simple']\n",
      "['character', 'also veriﬁed in', 'our model']\n",
      "['desired scaling', 'in', 'large ϕ']\n",
      "['agreement', 'with', 'ones for directed percolation universality class numericcitation']\n",
      "['standard errors', 'in', 'parentheses']\n",
      "['0.18 0.32 0.33 0.34 0.30 0.23 0.09 0.60 0.12 0.28 0.26 0.43 0.28 0.05 0.22 0.15 0.23 0.23 standard errors', 'in', 'parentheses']\n",
      "['population', 'represent by', 'graph']\n",
      "['work', 'analyze', \"system 's asymptotic behavior\"]\n",
      "['infected animal', 'be', 'so once discovered']\n",
      "['ﬁnal shape', 'be', 'precisely convex hull associated to set']\n",
      "['simplest model', 'in', 'epidemiology']\n",
      "['mathematical model', 'with', 'control']\n",
      "['homogeneous mixing assumption', 'be', 'too strong']\n",
      "['l × l square lattices', 'for jump', 'binned data']\n",
      "['similar processes', 'in', 'ﬁelds such host-pathogen biology']\n",
      "['changes', 'in', 'spectra for diﬀerent population sizes']\n",
      "['root', 'mean', 'square error between average participation rate over 100 monte carlo realizations']\n",
      "['recovered nodes', 'acquire', 'permanent immunity']\n",
      "['small changes', 'in', 'control parameter umax']\n",
      "['number', 'in', 'set υ']\n",
      "['number', 'in', 'set']\n",
      "['epidemic phase', 'increase signiﬁcantly in', 'four cases']\n",
      "['new infected individuals', 'in', 'la gloria']\n",
      "['infected individuals', 'in', 'ith patch']\n",
      "['quasi-neutral competition', 'in', 'spatial systems']\n",
      "['experimental analysis', 'be', 'made on ﬁve di ﬀerent empirical complex network on single core processor']\n",
      "['sample probability', 'be', 'when large']\n",
      "['important application', 'in', 'epidemic predictions']\n",
      "['computer', 'may recover from', 'virus attack after anti-virus software']\n",
      "['nodes', 'with', 'same topological measures']\n",
      "['conclusions', 'in', 'work']\n",
      "['infections', 'in', 'full epidemic']\n",
      "['numericcitation use', 'in', 'their analysis']\n",
      "['numericcitation colizza v', 'm', 'barth']\n",
      "['deterministic model', 'in', 'nonhomogeneous population']\n",
      "['control', 'be', 'piecewise']\n",
      "['horst', 'evaluation of', 'control']\n",
      "['urr', 'evaluation of', 'control']\n",
      "['as left hand side', 'on variable', 'e s']\n",
      "['proof', 'be', 'simple']\n",
      "['lines', 'represent', 'limit']\n",
      "['i1', 'recover with', 'immunity']\n",
      "['overall epidemic size', 'for tolerance', 'threshold']\n",
      "['generic saddle-node bifurcation', 'in', 'one dimension']\n",
      "['susceptibles', 'with', 'τ']\n",
      "['susceptibles', 'of fraction', 'γ']\n",
      "['random graph', 'with', 'given degree sequence']\n",
      "['τ 6 ∞', 'be', 'stopping time']\n",
      "['our results', 'in', 'section iv']\n",
      "['branching process approximating', 'in', 'independent partition model']\n",
      "['than regular lattice', 'with', 'same z']\n",
      "['epidemic outbreaks', 'in', 'scale-free networks']\n",
      "['τ continuous time markov chain', 'with', 'ﬁnite state space e']\n",
      "['constant matrix', 'with', 'eigenvalues']\n",
      "['constant matrix', 'with', 'eigenvalues']\n",
      "['population', 'be', 'where divided into four groups']\n",
      "['susceptible vertices', 'of op', 'n']\n",
      "['numericcitation gallos l. k', 'in', 'reaction-diﬀusion processes']\n",
      "['numericcitation cohen r', 'strategy for', 'computer networks']\n",
      "['numericcitation granell c', 'in', 'multiplex networks']\n",
      "['numericcitation cozzo e', 'in', 'multiplex networks']\n",
      "['numericcitation anderson', 'disease of', 'humans']\n",
      "['numericcitation cohen r', 'handbook of', 'graphs']\n",
      "['numericcitation valdez l. d', 'in', 'correlated interdependent networks']\n",
      "['numericcitation brummitt c. d', 'in', 'interdependent networks']\n",
      "['numericcitation cozzo e', 'a. of', 'boolean multilevel networks']\n",
      "['edge', 'with', 'probability 1 −']\n",
      "['n', 'assume', 'randomly connected network']\n",
      "['involved si1 i2r model', 'with', 'population turnover']\n",
      "['contact process', 'with', 'random vertexdependent infection rates']\n",
      "['university', 'dynamics in', 'childhood diseases']\n",
      "['node', 'have', 'inﬂuence only accounting for 6 local surroundings']\n",
      "['b', 'as w', 'kwkx ≤ 1 for kx − ykx → 0']\n",
      "['authors', 'represent', 'latent population during virus incubation before development of symptoms']\n",
      "['second switching point', 'absent in', 'deterministic model']\n",
      "['control term', 'value of', 'parameters']\n",
      "['size', 'in', 'barely supercritical random graph']\n",
      "['d. h. zanette', 'in', 'globally coupled systems with time delayes']\n",
      "['analytical results', 'in', 'good agreement with numerics']\n",
      "['memory eﬀects', 'in', 'susceptible-infected-recovered sir epidemic model']\n",
      "['individuals', 'in', 'classes']\n",
      "['nontrivial ﬁxed point', 'with', 'high probability for ﬁxed β']\n",
      "['sir dynamics', 'in', 'random networks with heterogeneous connectivity']\n",
      "['poor agreement', 'in', 'phase']\n",
      "['quantitative diﬀerences', 'in', 'monte carlo simulations']\n",
      "['sfns', 'with', 'natural cutoff']\n",
      "['sfns', 'with', 'natural cutoff c fig 4']\n",
      "['theoretical predictions', 'in', 'networks with disassortative mixing']\n",
      "['phase diagram', 'in', 'plane for pa sirs model']\n",
      "['standard deviation', 'in', 'large-scale epidemic']\n",
      "['one infected node', 'with', 'k neighbors']\n",
      "['real networks', 'have', 'community structure numericcitation']\n",
      "['u-or i-neighbor', 'with', 'rate β']\n",
      "['u-or i-neighbor', 'with', 'rate γ β']\n",
      "['u-nodes', 'i-node with', 'rate δ']\n",
      "['i-nodes', 'with', 'rate δ']\n",
      "['number', 'in', 'corresponding deterministic epidemic']\n",
      "['steady states', 'use', 'continuation package numericcitation']\n",
      "['absence', 'in', 'scale-free networks with degree correlations']\n",
      "['disease outbreak', 'be', 'where important']\n",
      "['number', 'umax to', 'isolation problem']\n",
      "['multilayer networks', 'with', 'two layers']\n",
      "['fraction', 'in', 'giant component numericcitation']\n",
      "['long time series', 'in', 'monte carlo']\n",
      "['jav', 'acknowledge', 'support from fct']\n",
      "['farm', 'be', 'once infected']\n",
      "['sir model', 'of evolution', 'tr']\n",
      "['sir model', 'of evolution', 'vl']\n",
      "['problem formulat ion', 'in', 'section']\n",
      "['random recovery time', 'with', 'distribution exp']\n",
      "['appreciable change', 'in', 'scaling with system size at criticality here']\n",
      "['bauer', 'in', 'epidemic models']\n",
      "['contact tracing control', 'in', 'organized populations with local structure']\n",
      "['context', 'in', 'population network']\n",
      "['outbreaks', 'with', 'only a few infections']\n",
      "['information propagation', 'in', 'ﬂickr social network']\n",
      "['spreads', 'in', 'diﬀerent ways']\n",
      "['l × l square lattices', 'with', 'l 128 256']\n",
      "['transition', 'still ﬁrst-order on', 'square lattice']\n",
      "['exponent values', 'in', 'diﬀerent dimensions']\n",
      "['tr 200', 'lattice with', 'long-range connections']\n",
      "['sum', 'behavior of', 'occupation time statistics branching']\n",
      "['its applicability', 'with', 'nodes following different infection models']\n",
      "['spread', 'in', 'population with random susceptibility']\n",
      "['changes', 'apply only to', 'new construction']\n",
      "['fig 3', 'illustrate', 'corner allocation for example node']\n",
      "['real social networks', 'have', 'community structure numericcitation']\n",
      "['number', 'in', 'observation']\n",
      "['inductive step', 'hold for', '¯ d']\n",
      "['inductive step', 'hold for', '¯ d']\n",
      "['heterogeneous sir model', 'with', 'sparse observations']\n",
      "['j. zhu', 'estimation by', 'joint numericcitation numericcitation sparse regression models']\n",
      "['inhomogeneous models', 'be', 'also introduced']\n",
      "['vertex', 'in', 'b']\n",
      "['homogeneous sir model', 'with', 'complete snapshot']\n",
      "['additional observation', 'with', 'reference to control signals']\n",
      "['model', 'with', 'nonlinear individual growth rate']\n",
      "['asymptotic scaling', 'in', 'as eq']\n",
      "['1 − r −', 'in', 'as eq']\n",
      "['∞', 'with', 'high probability']\n",
      "['stability analysis', 'be', 'for demographic sir model still tractable by symbolic computation']\n",
      "['epidemic thresholds', 'in', 'networks with positive degree-degree correlations']\n",
      "['large outbreak', 'in', 'independent partition model']\n",
      "['4.1 independent partition model', 'in', 'subsection']\n",
      "['independent partition model', 'in x', 'x']\n",
      "['x', 'in', 'independent partition model']\n",
      "['cluster', 'in', 'independent partition model']\n",
      "['sir processes', 'inhibit by', 'highly modular connectivity patterns']\n",
      "['change', 'in', 'values']\n",
      "['vaccinated agent', 'be', 'when infected']\n",
      "['optimal vaccination problem', 'with', 'delayed control']\n",
      "['dependence', 'in', 'our notation']\n",
      "['human population distribution', 'estimate as', 'scale free']\n",
      "['our simulations', 'average', 'over 1000 network realizations']\n",
      "['probability models', 'series in', 'decision processes']\n",
      "['single equations', 'with', 'same content']\n",
      "['degree', 'however only reﬂects', 'nodes number']\n",
      "['three theoretical methods', 'differ', 'epidemic thresholds']\n",
      "['4 infected', 'give by', '1 − β']\n",
      "['strucuture', 'be', 'when taken into account']\n",
      "['contact', 'with', 'infected individual']\n",
      "['infected individuals', 'convince', 'susceptibles']\n",
      "['analysis', 'in', 'our model']\n",
      "['there will node', 'in', 'randomly chosen edge traversed by disease']\n",
      "['t', 'with', 'probability θ boosting to rw']\n",
      "['second abrupt shift', 'with', 'optimal population based plateau value v 1/v from above']\n",
      "['color-bar values', 'in', 'figs 4']\n",
      "['color-bar values', 'in', 'figs 6']\n",
      "['color-bar values', 'in', 'fig 9']\n",
      "['epidemic e', 'be', 'hence after herd immunity threshold smaller than stochastic average epidemic e']\n",
      "['bifurcation diagrams', 'in', 'standard sis model']\n",
      "['n set', 'l-set of', 'links']\n",
      "['individuals', 'with', 'new strain']\n",
      "['binomial distribution', 'with', 'parameters di']\n",
      "['disease-free ﬁxed point', 'be', 'when globally stable for mean-ﬁeld model']\n",
      "['population', 'with', 's susceptible individuals']\n",
      "['distance', 'such as estimator', 'jordan center numericcitation']\n",
      "['disease dynamics', 'in', 'rural regions']\n",
      "['disease dynamics', 'in', 'rural regions']\n",
      "['great importance', 'in', 'poor countries']\n",
      "['g', 'be', 'random']\n",
      "['vaccine v', 'amount of', 'when size']\n",
      "['contacts', 'with', 'infected animals']\n",
      "['weighted graph', 'with', 'link weights chosen proportionally according sir parameters']\n",
      "['time-dependent distribution', 'in', 'strictly sub-critical case']\n",
      "['general sir model', 'with', 'complete observations']\n",
      "['power-law degree distribution', 'with', 'exponent 2']\n",
      "['clay center', 'kansas give', 'best estimated values for three contact levels']\n",
      "['random graph numericcitation', 'with', 'coordination number']\n",
      "['contrast', 'with', 'previous ﬁndings on unconstrained problems minimizing over outbreak']\n",
      "['multigraphs', 'with', 'given degree sequence deﬁned by conﬁguration model']\n",
      "['basic reproduction number', 'above', 'when epidemic threshold']\n",
      "['processes', 'with', 'diﬀerent parameter values']\n",
      "['partial observations', 'in', 'sir model']\n",
      "['removed subpopulation', 'in', 'models']\n",
      "['models', 'be', 'signiﬁcant']\n",
      "['research', 'in', 'mathematics']\n",
      "['research', 'department of', 'mathematics']\n",
      "['eﬀective reproduction number', 'in', 'deterministic scalar sir model']\n",
      "['simulation results', 'kmin at', 'diﬀerent times']\n",
      "['contradictory results', 'be', 'again obtained']\n",
      "['contact', 'with', 'other individuals']\n",
      "['g', 'be', 'random']\n",
      "['example', 'be', 'random']\n",
      "['de ﬁnitions', 'be', 'repeated here']\n",
      "['contacts', 'of network', 'graph']\n",
      "['random graphs', 'with', 'ﬁxed degree sequence']\n",
      "['spreading process', 'in', 'sir models']\n",
      "['dynamics', 'in', 'montserrado']\n",
      "['lipschitz', 'in', 'x']\n",
      "['time', 'on sn', 't']\n",
      "['scaling exponents', 'in', 'multi-stage sir']\n",
      "['transmission', 'in', 'communication networks']\n",
      "['use', 'in', 'epidemic models']\n",
      "['following taylor series expansion', 'in', 'v']\n",
      "['application', 'in', 'hand']\n",
      "['statistical inference', 'in', 'stochastic epidemic seir model with control intervention']\n",
      "['susceptible population', 'in', 'one infectious period']\n",
      "['his opinion', 'with', 'probability p']\n",
      "['k', 'infect by', 'node']\n",
      "['differences', 'in', 'population structure']\n",
      "['size o', 'epidemic of', 'size']\n",
      "['linear system size l. parameter', 'in', 'standard percolation theory']\n",
      "['spread', 'in', 'uk farms']\n",
      "['swn', 'be', 'then shown']\n",
      "['spread', 'in', 'populations']\n",
      "['recovered vertices', 'in', 'sir dynamics']\n",
      "['number', 'in', 'population']\n",
      "['sis model', 'also know as', 'contact process']\n",
      "['critical random graph', 'with', 'given degree sequence']\n",
      "['parallelization', 'in', 'like similar algorithms numericcitation']\n",
      "['sir epidemics', 'prove', 'limit theorems']\n",
      "['single individual results', 'in', 'ﬁnite outbreak']\n",
      "['non-integer t', 'in', 'by here distribution']\n",
      "['household level variations', 'thus control for', 'heterogeneity']\n",
      "['model ling disease outbreaks', 'in', 'networks']\n",
      "['low number', 'be', 'when observed']\n",
      "['epidemic dynamics', 'in', 'complex networks']\n",
      "['ˆi sim t 6 conclusions', 'in', 'work']\n",
      "['epidemics', 'in', 'complex networks']\n",
      "['fmd', 'can', 'also can transmitted']\n",
      "['right hand side', 'on variable', 'e s']\n",
      "['discrete-time epidemics', 'of spread', 'namely sis']\n",
      "['experimental results', 'in', 'section 5']\n",
      "['diﬀerent values', 'for 30', 'right']\n",
      "['diﬀerent values', 'for 30', 'right']\n",
      "['sir process', 'with', 'rates']\n",
      "['rich phase space', 'with', 'multiple epidemic peaks']\n",
      "['statistical model', 'use', 'methods']\n",
      "['their total trade volume', 'be', 'on other hand signiﬁcantly high in total global trade']\n",
      "['than initial epidemics', 'in parameter', 'growth rates']\n",
      "['contact', 'with', 'infected individuals']\n",
      "['infected individuals', 'number of', 'm']\n",
      "['tr', 'eﬀective probability of', 'contagion']\n",
      "['many people', 'have', 'where have previously infected']\n",
      "['memory prone sir model dynamics', 'use', 'fractional derivatives']\n",
      "['r', 'with', 'probability γ']\n",
      "['spreading rate', 'be', 'when very small']\n",
      "['mean ﬁeld approximation', 'in', 'eq']\n",
      "['recovery', 'with epidemic', 'ann']\n",
      "['individual i', 'with', 'recovery rate µ']\n",
      "['pa', 'clearly give', 'results closer to those obtained from stochastic model']\n",
      "['condition 1', 'prevent', 'zero denominastep 3']\n",
      "['experimental results', 'in', 'section']\n",
      "['situation', 'with', 'few resources']\n",
      "['n', 'be', 'when large']\n",
      "['castellano', 'physics of', 'social dynamics']\n",
      "['pathogens', 'in', 'human network']\n",
      "['can susceptible compartments', 'in', 'sir model nodes']\n",
      "['components', 'in', 'such way so for computational eﬃciency']\n",
      "['model network architecture', 'with', 'two interconnected modules']\n",
      "['time', 'with', 'l']\n",
      "['2 results', 'with', 'varying spreading rate β']\n",
      "['marginal probability', 'in', 'network']\n",
      "['large r0 case', 'in', 'stochastic model']\n",
      "['section 8', 'contain', 'few remarks on random 2 time shift used in our proof']\n",
      "['sir epidemics', 'prove', 'associated limit theorems']\n",
      "['distance', 'such as estimator', 'belief propagation estimator numericcitation']\n",
      "['β', 'be', 'healing probability']\n",
      "['population', 'in', 'three categories']\n",
      "['sir epidemics', 'in', 'large homogeneously mixing populations']\n",
      "['diﬀerence', 'translate into', 'more modular structure for rewired networks']\n",
      "[\"link 's appearance\", 'in', 'infection hierarchies numericcitation']\n",
      "['watanabe', 'have', 'theorem']\n",
      "['watanabe', 'have', 'theorem']\n",
      "['watanabe', 'of', 'theorem']\n",
      "['watanabe', 'from', 'theorem']\n",
      "['watanabe', 'from', 'theorem']\n",
      "['watanabe', 'in', 'theorem']\n",
      "['sugitani', 'of', 'theorem']\n",
      "['sugitani', 'have', 'theorem']\n",
      "['d/2 sugitani', 'have', 'theorem']\n",
      "['sugitani', 'of', 'theorem']\n",
      "['watanabe', 'of', 'theorem']\n",
      "['sugitani', 'have', 'theorem']\n",
      "['θn', 'follow from', 'theorem 1']\n",
      "['exponential distribution', 'with', 'parameter 2λ − 1']\n",
      "['other event', 'in', 'factor']\n",
      "['studies', 'design', 'vaccination campaigns']\n",
      "['scaling power laws', 'will', 'will muddied']\n",
      "['edge', 'in', 'g']\n",
      "['edge', 'in', 'a n']\n",
      "['models', 'commonly know as', 'sir']\n",
      "['infectious disease emergence', 'for bolzoni', 'e747']\n",
      "['degree', 'be', 'when large']\n",
      "['other hand', 'in', 'fig 10']\n",
      "['state function', 'represent', 'latency']\n",
      "['structured epidemic model', 'with', 'wentzell boundary condition']\n",
      "['10 −', 'in', 'transient regime']\n",
      "['p2', 'can', 'can non-zero']\n",
      "['analysis', 'make on', 'ﬁ ve diﬀerent empirical complex networks']\n",
      "['location-based strategy', 'be', '55 as eﬀective as best global target strategy']\n",
      "['vaccination', 'be', 'when dominant']\n",
      "['infectious sars coronavirus spread', 'in', 'quickly public with few protections']\n",
      "['stochastic sir dynamics', 'in', 'two non-interacting populations']\n",
      "['only input parameter', 'as p', 'theoretical epidemic threshold prediction using']\n",
      "['information', 'propagate for', 'time slots']\n",
      "['our results', 'in', 'agreement with theirs']\n",
      "['ti distribution', 'with', 'size iti']\n",
      "['de ﬁnitions', 'repeat here for', 'completeness']\n",
      "['1999 opposite patterns', 'in', 'sympatric disease metapopulations']\n",
      "['low value', 'quantitatively suggest', 'poor agreement between two solutions']\n",
      "['zeros', 'in', 'main diagonal']\n",
      "['ties', 'be', 'broken']\n",
      "['simulation results', 'in', 'section']\n",
      "['cb', 'also receive', 'partial support from asu/sfi center for biosocial complexity']\n",
      "['recovery processes', 'identical in', 'models']\n",
      "['β', 'in', 'our model']\n",
      "['comparison', 'with', 'numerical simulations']\n",
      "['simulations', 'in', 'solid lines']\n",
      "['mathematical literature', 'in', 'connection with acute infectious diseases modeling']\n",
      "['high dimensional state space', 'with', 'probabilistic transitions']\n",
      "['common probability space', 'in', 'way']\n",
      "['10 − 2', 'network in', 'twitter']\n",
      "['10 − 2', 'network of', 'communications']\n",
      "['trivial solution θ', 'in', 'self-consistent equation']\n",
      "['exponential growth', 'with', 'rate βλmax']\n",
      "['βλmax', 't with', 'rate βλmax']\n",
      "['infectious individuals', 'progress to', 'recovered state']\n",
      "['global children', 'with', 'mean λg']\n",
      "['results', 'with', 'those obtained from deterministic mfa']\n",
      "['second switch', 'in', 'stochastic model']\n",
      "['random graph', 'with', 'given degrees svante janson']\n",
      "['complete ﬂexibility', 'in', 'modeling transmission dynamics']\n",
      "['optimal eradication time t ∗', 'with', 'r0']\n",
      "['i ∈ ˆηj spatio-temporal graphical model selection 9', 'with', 'step size']\n",
      "['eigenvalues', 'with', 'zero real part']\n",
      "['291737 0.010 standard errors', 'in', 'parentheses']\n",
      "['equilibrium', 'in', 'stochastic form']\n",
      "['chaos', 'in', 'childhood epidemics']\n",
      "['analysis', 'in', 'two epidemic models']\n",
      "['sir models nodes', 'in susceptible', 's']\n",
      "['simulations', 'in', 'naive sir algorithm']\n",
      "['eigenvalues', 'with', 'negative real part']\n",
      "['scaling power laws', 'will', 'will muddied via interference from logarithmic factors']\n",
      "['classical s i r model', 'into class', 's f']\n",
      "['its area', 'be', 'divergent']\n",
      "['st', 'be', 'continuous-time']\n",
      "['simulation', 'result base on', 'eqs']\n",
      "['general sir model', 'with', 'vaccination']\n",
      "['random multigraph', 'with', 'given degree sequence']\n",
      "['common underlying probability space', 'with', 'probability measure p']\n",
      "['its short range contacts', 'value of', 'nodes linked']\n",
      "['spectra', 'be', 'fully described by theory only for very large populations']\n",
      "['set', 'in', 'sir epidemic on random intersection graph g']\n",
      "['non-infected nodes', 'be', 'where assumed']\n",
      "['more extremist', 'with', 'probability p']\n",
      "['total number n', 'in', 'network']\n",
      "['epidemic thresholds', 'in', 'real networks']\n",
      "['haq z. transmission model', 'with', 'migration eﬀect']\n",
      "['bayesian parameter estimation', 'apply to', 'dynamical models']\n",
      "['ratio', 'with', 'c']\n",
      "['probability ρi', 'with', 'probability 1 − ρi']\n",
      "['state-dependent type', 'in', 'case']\n",
      "['condition', 'be', 'also suﬃcient']\n",
      "['transition rate', 'probability of', 's f']\n",
      "['regular graphs', 'can display only', 'clustering property']\n",
      "['modiﬁed recovery algorithm', 'have', 'ﬁve steps together with three conditions']\n",
      "['hk', 'total number of', 'litters spawned by ﬁrst k vertices evaluated in branching process z f']\n",
      "['dynamics', 'in', 'reductions']\n",
      "['error bars', 'would', 'would have smaller than symbol size']\n",
      "['heterogeneous networks', 'with', 'identical infectivity rui yang']\n",
      "['− 1', 'with', 'probability p1']\n",
      "['− 1', 'with', 'probability p1']\n",
      "['− 2', 'with', 'probability p1']\n",
      "['eigenvalues', 'with', 'positive real part']\n",
      "['models', 'with', 'high dimension']\n",
      "['contacts', 'be', 'within households more frequent than in general population']\n",
      "['classic sis model', 'with', 'ρ ≡ 1']\n",
      "['uncertainty', 'in', 'values']\n",
      "['numerical results', 'in', 'single multilayer networks']\n",
      "['application', 'in', 'compartmental models']\n",
      "['nodes', 'with', 'high k']\n",
      "['ﬁnite networks', 'with', 'loops']\n",
      "['4 methods', 'in', 'section']\n",
      "['scale-free networks', 'with', 'power-law degree distribution p']\n",
      "['k-shell', 'outperform', 'degree']\n",
      "['degree', 'outperform', 'k-shell']\n",
      "['degree', 'be', 'radial']\n",
      "['elimination', 'with', 'degree 1']\n",
      "['size', 'with', 'given degree sequence']\n",
      "['agreement', 'with', 'available data']\n",
      "['m', 'serrano', 'phys']\n",
      "['extended problems', 'in', 'epidemiology epidemic on networks with time-varying transmission probabilities']\n",
      "['number', 'in', 'system']\n",
      "['number', 'in', 'system']\n",
      "['hiv', 'in', 'africa numericcitation such druggers']\n",
      "['situation', 'be', 'may more subtle']\n",
      "['temporal evolution', 'with', 'unlimited vaccines']\n",
      "['synchronization', 'in', 'random geometric graphs']\n",
      "['same way', 'in', 'as lemma 2.2']\n",
      "['u', 'n. by', 'induction assumption']\n",
      "['conclusions', 'summarize in', 'section 5']\n",
      "['inclused seir epidemic models', 'with', 'non-random infectious period']\n",
      "['theory', 'in', 'proc']\n",
      "['delayed control', 'be', 'when selected']\n",
      "['biological populations', 'in', 'space']\n",
      "['mathematics', 'in research', 'portugal e-mail']\n",
      "['individuals', 'join osn', 's-i interaction']\n",
      "['behavior', 'in', 'populations starting from small initial infection fraction']\n",
      "['unoccupied nearest neighbor', 'with', 'given bond probability']\n",
      "['ﬁnite time', 'with', 'probability one']\n",
      "['good agreement', 'with', 'results obtained by numerical simulations in fig 2']\n",
      "['cw', 'in', 'conditional probability p']\n",
      "['numericcitation e. tielkes', 'competition in', 'changing world']\n",
      "['two curves', 'be', 'qualitatively similar']\n",
      "['populations', 'in', 'irsir model']\n",
      "['21 6 connection', 'with', 'constant delay']\n",
      "['population', 'with', 'constant infectiousness']\n",
      "['mathematical models', 'in', 'biology']\n",
      "['outbreaks eradication', 'in', 'aﬀected areas']\n",
      "['new neighbors', 'introduce', 'local structure']\n",
      "['number', 'in', 'r1']\n",
      "['number', 'in', 'r2']\n",
      "['outbreak', 'in', 'n/m']\n",
      "['section 4', 'describe', 'analytical methods used']\n",
      "['limit process xt', 'be', 'where dawson-watanabe process with initial conﬁguration variance parameter σ2']\n",
      "['critical infection rate η', 'in', 'weak infection regime']\n",
      "['recovered individuals', 'in', 'ﬁnal state']\n",
      "['diﬀerences', 'in', 'higher-order structure impact on epidemic threshold']\n",
      "['constant matrix', 'with', 'eigenvalues']\n",
      "['constant matrix', 'with', 'eigenvalues']\n",
      "['numericcitation aderson r m', 'in', 'humans']\n",
      "['clustering eﬀect', 'also observe in', 'stochastic sis process numericcitation']\n",
      "['research', 'university of', 'aveiro']\n",
      "['stochastic integration', 'volume of', 'stochastic model ling']\n",
      "['model', 'in', 'section']\n",
      "['eﬀects', 'in', 'nonlinear oscillations close bifurcations numericcitation']\n",
      "['infected individuals', 'infect by', 'neighbour']\n",
      "['pa', 'with', 'results from stochastic sirs process']\n",
      "['given total', 'in', 'stochastic model']\n",
      "['phase transitions', 'study in', 'section 3']\n",
      "['sl − 1', 'η for', 'predetermined small positive value η']\n",
      "['theorem 1', 'discuss in', 'section iii']\n",
      "['theorem', 'discuss in', 'section iii']\n",
      "['limit', 'be', 'ﬁxed']\n",
      "['results', 'apply to', 'more general multiple-type critical branching processes']\n",
      "['kendall d g deterministic epidemics', 'in', 'closed populations']\n",
      "['complex networks', 'in', 'preparation']\n",
      "['complex networks', 'in c', 'h. ˇstefan ˇci c physica a']\n",
      "['c', 'in', 'complex networks']\n",
      "['models', 'with', 'constant delay']\n",
      "['b interplay', 'in', 'childhood diseaeses']\n",
      "['h x y', '2 for', 'si1 i2r model eq']\n",
      "['number', 'in', 'shortest path']\n",
      "['number', 'in', 'g']\n",
      "['regime', 'study in', 'monte carlo simulations']\n",
      "['infection', 'monotonically vanish for', 'arbitrary ﬁxing topology graphs']\n",
      "['has sub ject', 'in', 'recent years']\n",
      "['many mechanisms', 'generate', 'scale free networks']\n",
      "['epidemics', 'in', 'individual-based models from partial observations']\n",
      "['configuration model', 'with', 'exponent 2']\n",
      "['graphs', 'with', 'speciﬁed degree distributions']\n",
      "['number', 'of sum', 'asymptotic behavior']\n",
      "['wolfram', 'with', 'mathematica language']\n",
      "['10 harrington', 'with', 'u']\n",
      "['multistage sir model', 'in', 'ﬁnite population']\n",
      "['same meaning', 'in', 'as section 3.4']\n",
      "['auc', 'even decrease with', 'time much faster compared to direct sampling with complete observation']\n",
      "['2.3 modular adaptable network', 'in', 'section']\n",
      "['two processes', 'in', 'such way']\n",
      "['naive sir algorithm', 'of e', 't n']\n",
      "['stubs', 'with', 'probability p1']\n",
      "['processes', 'with', 'parameter values']\n",
      "['time', 'in', 'state η']\n",
      "['epidemic outbreaks', 'in', 'complex heterogeneous networks']\n",
      "['j', 'intervention during', 'emerging epidemic']\n",
      "['epidemic outbreaks', 'in', 'complex heterogeneous networks']\n",
      "['childhood epidemics', 'in', 'cities']\n",
      "['betweenness centrality plot', 'show', 'slightly higher spread for md networks']\n",
      "['damped oscillatory behavior', 'in', 'parameter ranges']\n",
      "['s', 'in', 'de ﬁnitions']\n",
      "['topology', 'strongly inﬂuences', 'cluster dynamics']\n",
      "['electrical engineering', 'institute of', 'technology']\n",
      "['ν', 'apply', 'property given by eq']\n",
      "['sir epidemics', 'in', 'contact networks']\n",
      "['s imulat ion re sult s', 'in', 'section']\n",
      "['untrimmed process satisand', 'hence also survive with', 'positive probability']\n",
      "['core', 'in', 'random graphs']\n",
      "['random graphs', 'with', 'arbitrary degree distributions']\n",
      "['early spreading', 'in', 'singapore numericcitation']\n",
      "['heterogeneity', 'in', 'square-lattice sir model']\n",
      "['newborn', 'immune for', 'several months after birth numericcitation']\n",
      "['its mortality rate', 'be', 'fortunately numericcitation']\n",
      "['s', 'in', 'ﬁnite tree']\n",
      "['recovered individuals', 'lose immunity at', 'rate γ']\n",
      "['numericcitation d. stauﬀer', 'introduction to', 'percolation theory']\n",
      "['error bars', 'show', 'standard error']\n",
      "['counters', 'set to', '-1']\n",
      "['deterministic herd immunity threshold', 'serve as', '15 figure 6']\n",
      "['epidemic threshold', 'below 1', 'α']\n",
      "['numericcitation j. stehl', 'e', 'n. voirin']\n",
      "['s. gupta', 'proceed of', 'national academy']\n",
      "['interest', 'of case', 'poisson distribution with mean 1']\n",
      "['susceptible vertex', 'with', 'ℓ half-edges']\n",
      "['spread', 'in', 'contact networks']\n",
      "['more work', 'address', 'issue']\n",
      "['many people', 'have', 'have previously infected']\n",
      "['function', 'with', 'exponential cutoﬀ c']\n",
      "['cumulative distribution', 'in', 'ﬁgure 1']\n",
      "['extinction events', 'in', 'article']\n",
      "['fixed/free ﬁnal time sir epidemic models', 'with', 'multiple controls']\n",
      "['wireless networks', 'symposium on', 'pp 628-635']\n",
      "['tubercle formation', 'in', 'lungs numericcitation']\n",
      "['general idea', 'in', 'proof presented']\n",
      "['a. vespignani', 'process on', 'complex networks']\n",
      "['noteworthy diﬀerences', 'in', 'epidemic size']\n",
      "['decrease', 'in', 'epidemic size']\n",
      "['decrease', 'in', 'epidemic size']\n",
      "['given degrees', 'with graph', 'critical window']\n",
      "['newman numericcitation', 'use', 'generating function formalism']\n",
      "['particles', 'start at', 'time 0']\n",
      "['science', 'for foundation', 'fct']\n",
      "['contagion', 'be', 'triggered by only one latent individual in p1']\n",
      "['increase', 'in', 'r0']\n",
      "['borge-holthoefer', 'spreader on', 'online social network']\n",
      "['multi-type branching process', 'with', 'production kernel given by equation']\n",
      "['global invasion threshold', 'in', 'metapopulation system']\n",
      "['nucleus', 'almost same as', 'nucleus found for con']\n",
      "['˙pm', 'with', 'time rescaled by 1/µ']\n",
      "['various values', 'in', 'stochastic sir model']\n",
      "['host', 'with', 'immunity level y']\n",
      "['host', 'with', 'immunity level y']\n",
      "['ε', 'be', 'ﬁxed']\n",
      "['same argument', 'in', 'as one-dimensional case']\n",
      "['cidma pro ject uid/mat/04106', 'within foundation', 'fct']\n",
      "['t → ∞ n − s', 'in', 'stochastic model']\n",
      "['stochastic model', 'in s', 't']\n",
      "['e', 'in', 'deterministic model']\n",
      "['suﬃcient vaccine', 'establish herd immunity in', 'deterministic model']\n",
      "['gain g', 'in', 'stochastic model']\n",
      "['optimal solution', 'in', 'stochastic model']\n",
      "['gain', 'in', 'deterministic model']\n",
      "['enough vaccine available', 'large r0 in', 'stochastic model']\n",
      "['allocation', 'in', 'stochastic model']\n",
      "['graph', 'of slope', 'death rate']\n",
      "['questions', 'in', 'computer science']\n",
      "['structure', 'interplay between', 'immune response']\n",
      "['time-varying transmission probabilities', 'can', 'can encoded as pr']\n",
      "['role', 'in', 'epidemic model o f mobile individuals on networks']\n",
      "['sauis model', 'for fig', 'color online']\n",
      "['random initial condition', 'with', 'given node']\n",
      "['full infection process', 'with', '1000 × 1000 nodes']\n",
      "['local connections', 'with', 'k']\n",
      "['result', 'in', 'fig 3']\n",
      "['numericcitation marro j', 'in', 'lattice models']\n",
      "['host population growth', 'be', 'where density-dependent']\n",
      "['universit', 'sabatier', 'f-31062 toulouse cedex 9']\n",
      "['r', 'fraction of', 'population']\n",
      "['use', 'in', 'epidemiological models']\n",
      "['stochastic effects', 'in', 'childhood diseases epidemiology']\n",
      "['romeros', 'ﬁlms', 'citation']\n",
      "['linear term', 'in', 'control variable u']\n",
      "['numericcitation r. m. anderson', 'disease of', 'humans']\n",
      "['numericcitation d. stauﬀer', 'introduction to', 'percolation theory']\n",
      "['kmax', 'be', 'maximum degree']\n",
      "['error probabilities', 'in', 'their case']\n",
      "['proof', 'in', 'multigraph case']\n",
      "['theorems', 'in', 'supercritical case']\n",
      "['contact', 'with', 'infected one']\n",
      "['optimal control problem', 'with', 'f']\n",
      "['contact', 'with', 'infected node']\n",
      "['random network', 'with', 'heterogeneous connectivity']\n",
      "['dynamical system', 'in', 'addition']\n",
      "['science', 'for foundation', 'fct']\n",
      "['k', 'be', 'when visited']\n",
      "['k', 'be', 'when visited']\n",
      "['infected node', 'in', 'layer α']\n",
      "['infected node', 'with', 'lifetime τ']\n",
      "['time-optimal problem', 'in', 'speciﬁc epidemic contexts']\n",
      "['troduc t ion', 'in', 'paper']\n",
      "['removed population', 'inevitably recycle into', 'zombie subpopulation']\n",
      "['zombie population', 'restore', 'civilization']\n",
      "['following diﬀerential equations numericcitation', 'impact of', 'asymptomatic infection']\n",
      "['vector ﬁeld', 'of part', 'jacobian']\n",
      "['bump', 'be', 'ﬁnite']\n",
      "['can diﬀerent stages', 'in', 'multistage model infected individuals']\n",
      "['our proof', 'base on', 'gw']\n",
      "['corresponding new infected individuals', 'in', 'clay center']\n",
      "['population', 'classiﬁed into', 'three classes']\n",
      "['branching process spreading', 'in', 'random time-sensitive environment']\n",
      "['vaccine v', 'amount of', 'when threshold size']\n",
      "['networks', 'with', 'power-law degree distribution generated by configuration model']\n",
      "['absolute errors', 'in', 'epidemic threshold from mfl methods']\n",
      "['cases', 'discuss respectively in', 'sec']\n",
      "['remarkable quantitative diﬀerences', 'with', 'results from direct sampling']\n",
      "['outbreak', 'in', 'susceptible pool']\n",
      "['partition step', 'in', 'iteration']\n",
      "['following quantities', 'also important in', 'our analysis']\n",
      "['many more pools', 'be', 'somewhat more valuable']\n",
      "['integral term', 'in', 'model']\n",
      "['5.3 large values', 'in', 'kitsak et']\n",
      "['third term', 'correspond to', 'diﬀusion']\n",
      "['coeﬃcients change', 'in', 'time']\n",
      "['given ﬁxed number', 'know as', 'connection radius']\n",
      "['contact', 'with', 'infected neighbor']\n",
      "['contact', 'with', 'infected neighbor']\n",
      "['undirected graph', 'in', 'figure 5']\n",
      "['van kampen', 'have', 'system size expansion']\n",
      "['stochastic shifts', 'in', 'time']\n",
      "['models', 'obtain from', 'm1']\n",
      "['two hubs', 'connect by', 'one single bidirectional edge between their two central nodes']\n",
      "['fig 8', 'in', 'appendix']\n",
      "['fig 9', 'in', 'appendix']\n",
      "['system', 'for', 'unique stable equilibrium for given α']\n",
      "['total number individuals', 'in', 'population']\n",
      "['intimate contacts', 'lead to', 'infection numericcitation']\n",
      "['contact p', 'probability of', 'rate']\n",
      "['their results', 'identical to', 'those following from our eqs']\n",
      "['bistability regime', 'find in', 'region iii']\n",
      "['individuals', 'infect in', 'school epidemic']\n",
      "['j individuals', 'aﬀected by', 'school epidemic']\n",
      "['immunity', 'in', 'population']\n",
      "['reduction', 'in', 'average epidemic size']\n",
      "['average epidemic size e', 'diﬀers from', 'optimal deterministic solution']\n",
      "['earlier decline', 'in', 'average epidemic size e']\n",
      "['diﬀerence', 'in', 'average epidemic size']\n",
      "['diﬀerence', 'in', 'average epidemic size']\n",
      "['models', 'often describe by', 'pde']\n",
      "['agent-based models', 'with', 'explicit movement']\n",
      "['interactions', 'with', 'infectious individuals']\n",
      "['b', 'for', 'configuration model with added triangles']\n",
      "['case', 'in', 'particular for macroparasite diseases among domestic animal populations']\n",
      "['2008 numericcitation a. legr', 'disease under', 'changing climate']\n",
      "['susceptibles', 'in', 'contact with infected individual']\n",
      "['ﬁnite-size analysis', 'introduce in', 'section 4']\n",
      "['sir models', 'model', 'individual pathogens']\n",
      "['unobserved nodes', 'with', 'at least one susceptible neighbour']\n",
      "['pa', 'with k', '7 k']\n",
      "['γ', 'be', 'immunization loss probability']\n",
      "['2n t', 'with', 'following values']\n",
      "['epidemic threshold', 'use', 'only degree distribution']\n",
      "['epidemics', 'seed with', '5 infectious nodes']\n",
      "['numericcitation p. cr', 'epey', 'p. t. boer']\n",
      "['numericcitation p. cr', 'epey', 'm. j. postma']\n",
      "['numericcitation s. g. mallat', 'transaction on', 'signal processing']\n",
      "['r ⋆', 'be', 'its recovery time']\n",
      "['run sir simulation', 'with', 'θ']\n",
      "['next step', 'in', 'our study']\n",
      "['other second order phase transitions', 'in', 'physics like relaxation time near curie temperature']\n",
      "['synchronous updating method', 'be', 'when used']\n",
      "['g', 'odor', 'phys']\n",
      "['t. tom', 'e', 'j. phys']\n",
      "['information', 'account for', 'correlations between thanas']\n",
      "['e hk', 'in', 'multilayer networks']\n",
      "['sir dynamics', 'in', 'multilayer networks']\n",
      "['scaling laws', 'in', 'inﬁnite regime']\n",
      "['pair', 'each from', 'diﬀerent layer']\n",
      "['performance', 'with', 'sparse observations']\n",
      "['phase transition', 'in', 'sir model']\n",
      "['control function', 'in', 'time-optimal framework']\n",
      "['t increases', 'with', 'recovery rate µ']\n",
      "['phase diﬀerences', 'be', 'where important']\n",
      "['phase diﬀerences', 'be', 'where important']\n",
      "['level', 'in', 'population']\n",
      "['sars', 'be', 'public health crisis on scale rarely seen']\n",
      "['branching random walk', 'compose of', 'ﬁve individuals']\n",
      "['giant susceptible component', 'overcome', 'temporal second order phase transition with mean ﬁeld exponents']\n",
      "['facebook network', 'with', 'community structure']\n",
      "['many real-world phenomena', 'spread', 'dynamics on complex networks']\n",
      "['real-world phenomena', 'spread', 'dynamics on complex networks']\n",
      "['std-gcc', 'be', 'online']\n",
      "['infectiousness', 'independent of', 'infection duration']\n",
      "['extinction probability', 'with', 'numbers independent']\n",
      "['second transition', 'describe', 'recovery process']\n",
      "['p. modeling infectious diseases', 'in', 'humans']\n",
      "['all networks', 'with', 'power-law degree distribution swae']\n",
      "['contact', 'with', 'infected node']\n",
      "['following condition', 'have', 'non-trivial solution']\n",
      "['one sequence', 'in', 'increasing order']\n",
      "['solution', 'together with one', 'black solid line']\n",
      "['same calculation', 'in', 'as lalley']\n",
      "['mode competition', 'in', 'lasers numericcitation']\n",
      "['numericcitation e. tielkes', 'competition for', 'resources']\n",
      "['numericcitation m.s bartlett', 'model in', 'ecology']\n",
      "['time t', 'have', 'has rescaled']\n",
      "['→', 'with', 'probability a0 −']\n",
      "['results boundary', 'with', 'region iii']\n",
      "['r1', 'be', 'greater than 1 if c2 c3']\n",
      "['response rate βf', 'be', 'when larger']\n",
      "['process', 'reach', 'steady state']\n",
      "['stability conditions', 'd for', 'a16']\n",
      "['βnxi', 'in', 'state with xi infective vertices recover at rate ρn']\n",
      "['sir process', 'in', 'random network with heterogeneous connectivity']\n",
      "['algorithm', 'with', 'polynomial running time']\n",
      "['class np', 'consist of', 'non-deterministic polynomial time problems']\n",
      "['homogeneous graph', 'with', 'k']\n",
      "['occupation numbers', 'as nx', 'i']\n",
      "['its peak value', 'increase with', 'fig 3']\n",
      "['inﬂuential spreaders', 'in', 'networks']\n",
      "['zombies', 'see in', 'previous model proposed']\n",
      "['distribution', 'in', 'skorohod space d']\n",
      "['memory', 'corroborate', 'important role played by non-markovian dynamics on dynamical processes unfolding on temporal networks numericcitation']\n",
      "['eigenvalues', 'with', 'negative real part']\n",
      "['links', 'represent', 'two individuals']\n",
      "['analytical approach', 'in', 'er network with hki']\n",
      "['fraction', 'with', 'large similarity index to observed incomplete snapshot']\n",
      "['importance', 'in', 'transmission process']\n",
      "['ring vaccination programme', 'control', 'disease propagation']\n",
      "['system', 'for solution', 'a8']\n",
      "['2-species system', 'of solution', 'a4']\n",
      "['connection', 'in', 'network']\n",
      "['random time shift', 'in', 'theorems 2.9']\n",
      "['results', 'show in', 'fig 3']\n",
      "['jacobian', 'ignore', 'µ components']\n",
      "['dynamics', 'in', 'speciﬁc communities']\n",
      "['results', 'in', 'figs']\n",
      "['sis case', 'in', 'its entirety']\n",
      "['model', 'in', 'section 2']\n",
      "['model', 'in', 'section 2']\n",
      "['model', 'in', 'section 3']\n",
      "['random varibles', 'have', 'statistical independence']\n",
      "['success probability iti ρ', 'probability of', 'infected individuals seeking']\n",
      "['time change', 'in', 'section 3']\n",
      "['time change', 'in', 'section 3']\n",
      "['eigenvalues', 'with', 'zero real part30 ,31']\n",
      "['pde', 'in', 'model']\n",
      "['model', 'with', 'delay']\n",
      "['ubiquitous source', 'in', 'epidemics']\n",
      "['more ﬂexibility', 'in', 'modeling assumptions']\n",
      "['plots', 'with', 'tw 0 for two er networks']\n",
      "['sharp transitions', 'in', '19 deterministic models']\n",
      "['small γ', 'in', 'present case']\n",
      "['simulations', 'in', 'region ii']\n",
      "['k-shell', 'can', 'can obtained by so-called k-core decomposition 50']\n",
      "['5 conclusion', 'in', 'paper']\n",
      "['basis step', 'hold for', 'd']\n",
      "['basis step', 'hold for', 'd']\n",
      "['basis step', 'hold for', '¯ d']\n",
      "['c', 'for', 'erdős-rényi model']\n",
      "['dead time', 'of shaun', 'hours']\n",
      "['r ev er s e', 'in', 'fec t ion a lgor']\n",
      "['two hundred simulation runs', 'average for', 'data point']\n",
      "['e-state', 'in', 'standard seir model']\n",
      "['case', 'in', 'paper']\n",
      "['china', 'of foundation', 'program by uestc']\n",
      "['mean ﬁeld system', 'in', 'eqs']\n",
      "['similarity sampling', 'give', 'largest average diﬀerence']\n",
      "['degree values', 'in ρ', 'kth']\n",
      "['ρ', 'in', 'degree values']\n",
      "['results', 'show in', 'figure 7a']\n",
      "['error distance', 'show in', 'fig 5']\n",
      "['absorbing phase', 'in', 'stochastic simulations']\n",
      "['single run', 'with', 'computed mean']\n",
      "['active individual', 'be', 'where always infectious']\n",
      "['eigenvalues', 'with', 'positive real part']\n",
      "['set sx', 't of', 'infectious signals sent from x up until time']\n",
      "['probability', 'in', 'ebcm approach']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['foundation', 'program of', 'outstanding ph d. candidate in academic research']\n",
      "['steady state change', 'with', 'vaccination probability ω']\n",
      "['network', 'be', 'more robust']\n",
      "['sis process', 'in', 'original twitter network']\n",
      "['oriented percolation', 'in', 'dimensions']\n",
      "['critical point', 'order of', 'size']\n",
      "['outbreak', 'in', 'uganda']\n",
      "['new babies', 'be', 'when born']\n",
      "['128 × 128 square lattice', 'with', 'long-range connections']\n",
      "['giant component', 'in', 'epidemic generating means']\n",
      "['logarithmic divergence', 'in', 'time']\n",
      "['time', 'at x', 'tm']\n",
      "['time', 'set', 't']\n",
      "['population', 'have', 'opinion']\n",
      "['largest absolute value', 'in', 'above jacobian matrix']\n",
      "['nodes', 'be', 'where all decoupled']\n",
      "['multilayer network', 'with', 'same interlayer network structure']\n",
      "['physics', 'ﬁcation of', 'effective epidemic threshold']\n",
      "['solutions', 'with', 'diﬀerent parameters']\n",
      "['other susceptible', 'be', 'contacts']\n",
      "['c3', 'in', 'appendix c']\n",
      "['reduction', 'in', 'sir network']\n",
      "['initial infective individual', 'be', 'global individual']\n",
      "['poisson interdegree distribution', 'with', 'average smaller than one']\n",
      "['distance', 'such as estimator', 'dynamic message passing estimator numericcitation']\n",
      "['contact', 'with', 'infectious individuals situated on neighboring nodes']\n",
      "['graphs', 'with', 'speciﬁed degree distributions']\n",
      "['epidemic spreading', 'in', 'department']\n",
      "['proposed graphical model selection', 'in', 'paper']\n",
      "['sir epidemic process', 'graph with', 'given degree sequence']\n",
      "['optimal changes', 'in', 'umax']\n",
      "['optimal strategy', 'in', 'epidemiological applications']\n",
      "['application', 'in', 'virus propagation']\n",
      "['local minimum', 'in', 'p e']\n",
      "['local minimum', 'in', 'p e']\n",
      "['second order phase transition', 'in', 'susceptible network']\n",
      "['prevalence', 'be', 'still very small']\n",
      "['nodes', 'method such as', 'one given']\n",
      "['asymptotic results', 'apply to', 'more general multiple-type critical branching processes']\n",
      "['infected animal', 'with', 'body ﬂuid']\n",
      "['infected individuals', 'in', 'layers']\n",
      "['our discussion', 'in', 'endemic state']\n",
      "['n', 'in', 'such 9']\n",
      "['such 9', 'in n', 't']\n",
      "['random walk', 'be', 'thus absorbed there']\n",
      "['other susceptible peers', 'with', 'probability w']\n",
      "['virulence', 'be', 'even when small']\n",
      "['virulence', 'be', 'even when small']\n",
      "['differences', 'in', 'assumptions']\n",
      "['infected nodes', 'can also recover with', 'probability γ']\n",
      "['infectious signals', 'in', 'modiﬁed process']\n",
      "['two diﬀerent runs', 'with', 'population size 1000']\n",
      "['values', 'be', 'where maximal']\n",
      "['values', 'be', 'where maximal']\n",
      "['β', 'in', 'infection probability']\n",
      "['coupled terms', 'in', 'irsir model']\n",
      "['stochastic eﬀects', 'play', 'more fundamental role']\n",
      "['volz equations', 'may', 'may derived as corollary']\n",
      "['second-order ode', 'with', 'outer excitation']\n",
      "['martin-l öf', 'size of', 'nearly critical ep wiener process to parabolic barrier']\n",
      "['proposed graphical model selection method', 'in', 'paper']\n",
      "['epidemic will', 'diverge signiﬁcantly from', 'branching envelope']\n",
      "['bene ﬁcial', 'in', 'real world scenarios']\n",
      "['dependent quantity', 'in', 'real world applications']\n",
      "['lemma 10', 'by pn', 'x']\n",
      "['diagonal form', 'with', 'separated eigenvalue structure']\n",
      "['color online', 'in', 'spreading dynamics with synchronous updating']\n",
      "['color online', 'in', 'spreading dynamics with synchronous updating']\n",
      "['color online', 'in', 'spreading dynamics with synchronous updating']\n",
      "['percolation', 'in', 'random clustered networks']\n",
      "['statistics', 'thus conditional on', 'ﬁnite sized outbreak']\n",
      "['individuals', 'in', 'its global edges']\n",
      "['right vertical scale', 'apply to', 'two measurements']\n",
      "['disease parameters', 'correspond to', 'typical measles']\n",
      "['case', 'count in', 'test data']\n",
      "['application', 'be', 'when ﬁled']\n",
      "['bp', 'finally usually approximate', 'extinction time probability distribution better than other methods']\n",
      "['can actors', 'in', 'multilayer networks']\n",
      "['populations', 'with', 'heterogeneous infection rates']\n",
      "['information source detection', 'in', 'sir model']\n",
      "['above results', 'from d', '1 − d']\n",
      "['infection graph reports', 'with', 'probability σ']\n",
      "['l', 'be', 'yi g y']\n",
      "['lump', 'distribution of', 'dirac delta function in theory']\n",
      "['t. tom', 'e', 'j. math']\n",
      "['3.1 preliminaries', 'in', 'subsection']\n",
      "['pictorial description', 'in', 'probability']\n",
      "['dawson-watanabe process', 'with', 'location-dependent killing']\n",
      "['dawson-watanabe process', 'with', 'location-dependent killing']\n",
      "['dawson-watanabe process', 'with', 'initial conﬁguration variance parameter σ2']\n",
      "['dawson-watanabe process', 'with', 'initial conﬁguration µ']\n",
      "['lattice', 'be', 'where fully covered by susceptible']\n",
      "['two curves', 'in', 'equation']\n",
      "['numericcitation hidalgo ca', 'al', 'barab']\n",
      "['socia l circles', 'in', 'ego networks']\n",
      "['v', 'reach', 'its limit value']\n",
      "['vaccine era', 'with', 'subclinical infections']\n",
      "['immunization strategies', 'have', 'have developed']\n",
      "['site percolation process', 'in', 'independent k']\n",
      "['other hand', 'on infectiousness', 'time t ∗']\n",
      "['curves', 'in', 'fig 4b']\n",
      "['model ling disease outbreaks', 'in', 'realistic urban social networks']\n",
      "['role', 'in', 'epidemic threshold']\n",
      "['n', 'in', 'numericcitation such']\n",
      "['numericcitation such', 'in n', 't']\n",
      "['left white triangle', 'in', 'figure 7b']\n",
      "['t', 'be', 'at given time displayed as red dots']\n",
      "['its infectious period', 'in', 'case']\n",
      "['extinction', 'can', 'can seen']\n",
      "['optimal solution', 'in', 'diﬀerent resource regimes']\n",
      "['babies', 'be', 'when born']\n",
      "['kind o f equation', 'also call', 'forward kolmogorov differential equation']\n",
      "['fourier', 'transform of', 'our recovered transmission rate function']\n",
      "['eﬀect', 'in', 'sir processes']\n",
      "['global children', 'with', 'expectation λg']\n",
      "['i. sir', 'in', 'activity driven networks']\n",
      "['i.i.d sequence', 'with', 'distribution y']\n",
      "['αn ¯ z', 'in', 'distribution']\n",
      "['60 cities', 'in', 'england']\n",
      "['12 switching', 'in', 'deterministic model found']\n",
      "['such random graphs', 'study in', 'numericcitation']\n",
      "['second term', 'in', 'eq']\n",
      "['two social networks', 'with', 'community structure']\n",
      "['delays', 'in', 'previous two factors']\n",
      "['fraction', 'in', 'population']\n",
      "['changes', 'in', \"system 's behavior\"]\n",
      "['severity', 'in', 'diﬀerent people']\n",
      "['poisson degree distribution', 'with', 'mean λ']\n",
      "['poisson degree distribution', 'with', 'mean λ']\n",
      "['successional dynamics', 'in', 'ecology numericcitation']\n",
      "['sirs numericcitation', 'dpt for', 'γ → ∞']\n",
      "['sm', 'also support by', 'mineco through juan de la cierva program']\n",
      "['nodes', 'with', 'degree k']\n",
      "['closed isola', 'in', 'two parameters']\n",
      "['rectangle', 'in', 'rrg model']\n",
      "['experimental evidence', 'in', 'model system']\n",
      "['heterogeneity', 'in', 'transmission']\n",
      "['from graphs', 'with', 'degree sequence']\n",
      "['method', 'propose by', 'newman et al']\n",
      "['individuals', 'with', 'vaccine-induced immunity']\n",
      "['diﬀerent mathematical frameworks', 'study', 'epidemic spreading']\n",
      "['contact', 'with', 'infected individual']\n",
      "['contact', 'with', 'infected individual']\n",
      "['recovery time', 'in', 'tr']\n",
      "['weight', 'in', 'large-scale epidemic']\n",
      "['setup nodes', 'couple at', 'diﬀerent spatio-temporal scales']\n",
      "['random network', 'with', 'household structure']\n",
      "['individuals', 'in', 'susceptible state']\n",
      "['complete stochastic dynamics', 'in', 'further studies']\n",
      "['value modeling', 'control of', 'ebola virus spread 7']\n",
      "['parameter', 'in', 'inference procedure for source location estimation']\n",
      "['spread', 'in', 'schools']\n",
      "['curves', 'be', 'standard']\n",
      "['countries', 'traditionally have', 'strong economic ties with greece']\n",
      "['our results', 'in', 'original networks']\n",
      "['stochastic ﬂuctuations', 'of spectrum', 'power spectral density']\n",
      "['our results', 'in', 'two formats']\n",
      "['maximum achievable probability', 'in', 'figure 7a']\n",
      "['remaining probability mass 1 − ν − 1', 'with', 'inﬁnite']\n",
      "['σi', 'be', 'when changed']\n",
      "['γ', 'be', 'when high']\n",
      "['γ', 'be', 'when high']\n",
      "['σi', 'be', 'when signiﬁcant']\n",
      "['n', 'be', 'when varied']\n",
      "['n', 'be', 'when varied']\n",
      "['disease free equiin particular', 'librium in', 'x']\n",
      "['chosen susceptible node', 'diﬀerent from', 'its present neighbors']\n",
      "['results', 'average', 'over 102 × 104 independent realizations on 102 different networks']\n",
      "['results', 'average', 'over 102 × 104 independent realizations on 102 different networks']\n",
      "['diseases', 'in', 'epidemiology research']\n",
      "['population size n', 'be', 'where large']\n",
      "['nonlocal interactions', 'in', 'epidemic spreading']\n",
      "['epidemic spreading', 'in', 'population']\n",
      "['scaling results', 'show in', 'figure 10']\n",
      "['seir model', 'with', 'vital rates']\n",
      "['seir model', 'with', 'vital rates']\n",
      "['networks', 'with', 'power-law degree distributions']\n",
      "['results', 'achieve without', 'parallelization']\n",
      "['time inequality', 'consider', 'inﬁnite tree rooted at vr']\n",
      "['underestimation', 'can strongly aﬀect', 'spreading dynamics']\n",
      "['properties', 'have', 'eﬀect on higher-order structure']\n",
      "['such results', 'in', 'literature']\n",
      "['infected nodes', 'in', 'wm networks']\n",
      "['important topological measures', 'in', 'network theory']\n",
      "['figure 1', 'illustrate for', 'numerical example']\n",
      "['irsir model', 'with', 'demography']\n",
      "['irsir model', 'with', 'demography']\n",
      "['spechler', 'have', 'irsir model']\n",
      "['simulated hidden states', 'of plot', 't. simulation with seasonally varying α']\n",
      "['ps sns', 'in', 'standard percolation theory']\n",
      "['two social dynamics', 'one for', 'opinion formation']\n",
      "['equation', 'in plus', 'minus']\n",
      "['other set', 'start from', 'random initial condition']\n",
      "['control problem', 'with', 'linear term policy']\n",
      "['mixing time', 'on numericcitation', '2014 ieee 53rd annual conference on']\n",
      "['epidemic outbreak', 'growth for', 'small times']\n",
      "['awarenessunawareness-awareness spreading', 'in', 'social online network']\n",
      "['susceptible-infected-susceptible spreading', 'in', 'social contact network']\n",
      "['homogeneous mixing assumption', 'individual in', 'population contact']\n",
      "['source', 'can', 'where can node from set s']\n",
      "['individuals', 'act as', 'spreaders via advertisements in mass media']\n",
      "['reduction', 'in', 'gorilla populations numericcitation']\n",
      "['keywords epidemics', 'with', 'seasonality']\n",
      "['limit µ', 'in s.', 'models']\n",
      "['r0', 'cross', 'threshold 1']\n",
      "['corresponding vertical dashed line', 'in', 'figure 1a']\n",
      "['modeling relapse', 'in', 'infec tious diseases']\n",
      "['poisson distributions', 'with', 'means φs']\n",
      "['new variables', 'in ρ', 'x t']\n",
      "['∂ t ρ', 'in', 'new variables']\n",
      "['large oscillations', 'in', 'system']\n",
      "['stationary state', 'be', 'has reached']\n",
      "['numericcitation', 'numericcitation in', 'discrete-time case']\n",
      "['numericcitation', 'in', 'discrete-time case']\n",
      "['single node', 'in', 'sis model']\n",
      "['single node', 'in', 'sirs model']\n",
      "['numericcitation j. leskovec', 'in', 'advances']\n",
      "['individuals', 'in', 'ﬁxed network']\n",
      "['evidence', 'in', 'than general multilayer networks']\n",
      "['bayesian stochastic modelling', 'in', 'python']\n",
      "['odeint function', 'in', 'scipy']\n",
      "['running time ratio', 'be', 'where strictly less']\n",
      "['running time ratio', 'be', 'where strictly less']\n",
      "['running time ratio', 'be', 'where strictly less']\n",
      "['running time ratio', 'be', 'where strictly less']\n",
      "['multivariate process', 'with', 'joint probability function given']\n",
      "['infection events', 'in', 'livestock']\n",
      "['34 recovery algorithm 12', 'in', 'case']\n",
      "['contagious disease spreading', 'in', 'community']\n",
      "['mp', 'inﬂuence of', 'increasing life expectancy on jm']\n",
      "['dynamical properties', 'in', 'section v. ii']\n",
      "['susceptible node', 'in', 'layer β']\n",
      "['numericcitation barth', 'm', 'barrat a']\n",
      "['numericcitation barth', 'm', 'pastor-satorras r']\n",
      "['gamma distributed recovery time', 'with', 'constant infectiousness']\n",
      "['running time ratio', 'be', 'where strictly less than one']\n",
      "['standard sir epidemics', 'in', 'closed homogeneously mixing populations']\n",
      "['infection', 'in', 'time']\n",
      "['homogeneous system', 'with', 'a periodic']\n",
      "['two hubs', 'each along', 'inter-modular edges']\n",
      "['two hubs', 'each along', 'intra-modular edges']\n",
      "['two hubs', 'each along', 'inter-modular edges']\n",
      "['two hubs', 'each along', 'intra-modular edges']\n",
      "['seir model', 'with', 'variable vital rates']\n",
      "['seir model', 'with', 'variable vital rates']\n",
      "['red path', 'be', 'therefore regular one dimensional brownian motion']\n",
      "['method', 'base on', 'time scale separation']\n",
      "['function', 'as area', 'c']\n",
      "['function', 'as area', 'c']\n",
      "['experimental analysis', 'make on', 'ﬁ ve diﬀerent empirical complex networks']\n",
      "['s', 'be', 'where connected']\n",
      "['stochastic process', 'in', 'general formulation']\n",
      "['their network structure', 'with', 'heterogeneous degree distributions']\n",
      "['frontiers', 'in', 'applied mathematics 28']\n",
      "['our arguments', 'will also require', 'following estimates on discretized green kernel']\n",
      "['a. epidemic outbreaks', 'in', 'complex heterogeneous networks']\n",
      "['resulting susceptible network', 'be', 'as consequence more homogeneous than original']\n",
      "['sample standard intervals', 'be', 'eﬀect']\n",
      "['red dots', 'in', 's']\n",
      "['outbreak size distribution', 'of scale', 'ν +2']\n",
      "['infectious diseases', 'in', 'domestic animals']\n",
      "['infectious diseases', 'in', 'wildlife']\n",
      "['agreement', 'with', 'stochastic simulations']\n",
      "['agreement', 'with', 'stochastic simulations']\n",
      "['nearest-neighbor i', 'with', 'probability c']\n",
      "['increase', 'in', 'probability']\n",
      "['network', 'with', 'identical interlayer degree sequences maximally anti-correlated']\n",
      "['size', 'in', 'fact']\n",
      "['excellent agreement', 'with', 'previous calculations']\n",
      "['contact', 'with', 'contaminated site']\n",
      "['epidemic spreading', 'with', 'general degree']\n",
      "['n such update steps', 'deﬁne', 'one monte-carlo time step']\n",
      "['step-like changes', 'in', 'probability']\n",
      "['entire population', 'be', 'where composed by susceptible individuals only']\n",
      "['analytical results', 'also veriﬁed by', 'numerical simulations']\n",
      "['recent epidemics', 'in', 'west africa']\n",
      "['chromatic number', 'with', 'ﬁxed degree sequence']\n",
      "['numericcitation j. leskovec', 'advance in', 'neural information processing systems 539']\n",
      "['corresponding simulated new infected individuals', 'in', 'clay center']\n",
      "['equal increment', 'in', 'activity']\n",
      "['contact networks', 'with', 'strongly-connected topologies']\n",
      "['linear stability', 'in', 'variables']\n",
      "['subset', 'in', 'variables']\n",
      "['integration', 'in', 'y variables']\n",
      "['euclidean network abdul khaleque1', 'on model', '92 apc road']\n",
      "['euclidean network abdul khaleque1', 'on model', 'kolkata 700009']\n",
      "['euclidean network abdul khaleque1', 'on model', 'india']\n",
      "['power-law degree distribution', 'with', 'parameter values']\n",
      "['energy', 'of department', 'doe']\n",
      "['graph theoretic methods', 'in', 'multiagent networks']\n",
      "['our ﬁrst aim', 'illustrate', 'possible eﬀects']\n",
      "['distributions', 'use for', 'several networks']\n",
      "['infected individuals', 'in', 'population']\n",
      "['step', 'in', 'time']\n",
      "['critical point', 'in', 'equilibrium phase on magnetic system31']\n",
      "['eradication time', 'in', 'problem']\n",
      "['network', 'have', 'temporal properties']\n",
      "['total number', 'in', 'infection']\n",
      "['grenfell bt antibody dynamics', 'in', 'childhood diseases']\n",
      "['α', 'in', 'our case']\n",
      "['used temperature variations', 'enforce', 'desired oscillations']\n",
      "['contact', 'with', 'infected individual']\n",
      "['contact', 'with', 'infected individual']\n",
      "['summary statistics', 'show in', 'table 2']\n",
      "['infectives', 'can', 'can introduced']\n",
      "['function', 'as τ', 'solid line']\n",
      "['fig 3', 'show r as', 'function ω']\n",
      "['function', 'as τ', 'solid line']\n",
      "['function', 'as τ', 'solid line']\n",
      "['three methods', 'in', 'theoretical prediction']\n",
      "['conﬁguration xt', 'of probability', 'xtobs time t given observation xtobs at time tobs']\n",
      "['time series', 'longer than', '105 mcs']\n",
      "['st', 'number of', 'given type']\n",
      "['other sequence', 'in', 'decresing order']\n",
      "['1 curves', 'in', 'figures 2a']\n",
      "['20 harrington', 'spread of', 'epidemic disease']\n",
      "['idle period', 'thus transient in', 'case']\n",
      "['immunity level', 'in', 'at time']\n",
      "['two distributions', 'use', 'quantitative measure']\n",
      "['correlations', 'in', 'un-forced epidemic models']\n",
      "['integrand', 'in', 'second term']\n",
      "['connectivity 4.07', 'in', 'er network with hki']\n",
      "['gillespie', 'by', 'algorithm numericcitation']\n",
      "['ebola', 'of', 'citation']\n",
      "['citation', 'in', 'capital city']\n",
      "['networks', 'with', 'more heterogeneous degree distributions']\n",
      "['mathematical sciences', 'of box', 'citation']\n",
      "['much memory', 'in', 'simulations']\n",
      "['vaccines', 'in', 's i r model with local vaccination']\n",
      "['weight', 'in', 'terminal infection']\n",
      "['one edge', 'through t', 'λ']\n",
      "['many more pools', 'be', 'somewhat valuable']\n",
      "['watson process', 'with', 'yk individuals instead one']\n",
      "['diﬀerent predicted extinction time distributions', 'with', 'maximum value']\n",
      "['equation', 'with', 'diﬀerent initial conditions']\n",
      "['random networks', 'with', 'heterogeneous connectivity']\n",
      "['gmin', 'qmin', 'lower bound on q']\n",
      "['hiv model', 'journal of', 'diﬀerential equations']\n",
      "['more detail', 'in', 'net section']\n",
      "['sis processes', 'in inﬁnite', 'endemic state']\n",
      "['outbreak', 'in', 'animal epidemics eric dumonteil ∗']\n",
      "['joint probability p n o s ∗ s n o', 'be', 'given observations']\n",
      "['vanishing threshold', 'in', 'thermodynamic limit for γ ≤ 3']\n",
      "['diseases', 'with', 'different infection durations']\n",
      "['their link weight wi', 'be', 'when high']\n",
      "['cycles', 'number of', 'sample from geometric distribution with expectation 1/q']\n",
      "['result', 'special case of', 'p. 281']\n",
      "['larger values', 'for g', 'v']\n",
      "['global epidemic', 'occur', 'numericcitation']\n",
      "['initial condition ξ0', 'vs. online', 'rn']\n",
      "['infected nodes', 'be', 'when observed']\n",
      "['transition', 'describe', 'recovery process']\n",
      "['total number', 'in', 'b']\n",
      "['vector borne disease', 'with', 'horizontal transmission']\n",
      "['sir', 'in', 'temporal networks kaiyuan sun ,1 andrea baronchelli ,2']\n",
      "['notations', 'in', 'subsection']\n",
      "['real networks numericcitation', 'of ipr', 'λ']\n",
      "['data', 'correspond to', 'average over 108 realizations']\n",
      "['total number', 'in', 'generation']\n",
      "['13 a. temporal evolution', 'in', 'fig 3']\n",
      "['expression', 'be', 'simple']\n",
      "['cholera commensal', 'in', 'number zooplankton species numericcitation']\n",
      "['stage', 'in', 'seir model']\n",
      "['new sis model', 'with', 'delay']\n",
      "['infected individual', 'transmit disease to', 'their neighbors']\n",
      "['dynamics', 'with', 'inspiration from mathematical epidemiology']\n",
      "['homophily-driven diﬀusion', 'in', 'dynamic networks']\n",
      "['network', 'with', 'converged component-size distribution']\n",
      "['theoretical predictions', 'λqmf of', 'epidemic threshold']\n",
      "['epidemic processes', 'in', 'complex networks']\n",
      "['extinction', 'in', 'recurrent epidemics']\n",
      "['immune population', 'in', 'model m1']\n",
      "['n ≥ 1', 'for ln', 'α0']\n",
      "['same fraction', 'in', 'case']\n",
      "['propagation', 'in', 'population']\n",
      "['ξ1', 'state of', 'health']\n",
      "['equilibrium point', 'call', 'disease-free equilibrium']\n",
      "['averhost', 'in', 'fully susceptible population']\n",
      "['βn n', 'tend to', 'strictly positive ﬁnite limit as n → ∞']\n",
      "['same factors', 'in', 'denominator']\n",
      "['our observation', 'with', 'frequencies']\n",
      "['susceptibility χ', 'in', 'standard percolation theory']\n",
      "['recovery algorithm', 'use', 'two simulated data sets']\n",
      "['τ l', 'distance between', 'v therefore']\n",
      "['connection', 'with', 'approach de-present veloped by kermack']\n",
      "['problem', 'reproduce', 'exactly results from computational simulations']\n",
      "['infection', 'of period', 'ﬂu']\n",
      "['spatial interactions', 'in', 'simple ecological models']\n",
      "['poisson', 'be', 'distributed with mean µg']\n",
      "['prevalence', 'be', 'exponential distribution']\n",
      "['independent variables', 'in', 'case one']\n",
      "['numericcitation n. t. bailey et', 'theory of', 'infectious diseases']\n",
      "['inﬁnite survival', 'have', 'positive probability starting from more sparse conﬁguration']\n",
      "['k', 'force of', 'infection']\n",
      "['infection', 'of force', 'citation']\n",
      "['p', 'in', 'epidemic process']\n",
      "['neighboring site', 'in', 'epidemic process']\n",
      "['circumstances', 'in threshold', 'deterministic epidemic overestimates']\n",
      "['number', 'in', 'layer β']\n",
      "['model assumptions', 'time after', 'recovery immune hosts']\n",
      "['i', 'contact', 'pairs']\n",
      "['information propagation speed', 'in', 'mobile tolerant networks']\n",
      "['∆ zti', '0 with', 'probability 2ε/µ otherwise']\n",
      "['yk − 1 0', 'with', 'probability αskpk']\n",
      "['susceptibility', 'can', 'therefore can calculated by method eﬃciently']\n",
      "['large populations', 'with', 'equal mixing rates']\n",
      "['random graph g', 'with', 'degree sequence']\n",
      "['near-term downturn', 'in', 'active membership numericcitation']\n",
      "['interaction', 'with', 'active users numericcitation']\n",
      "['two er multilayer networks numericcitation', 'with', 'average degree hkai ≃ hkb i ≃ 2.31']\n",
      "['two er multilayer networks numericcitation', 'with', 'average degree hka i ≃ hkb i ≃ 2.31']\n",
      "['y', 'matrix with', 'eigenvalues']\n",
      "['system', 'simulate for', '2 × 104 mcs']\n",
      "['system', 'behave like', 'focus']\n",
      "['incremental reduction', 'in', 'expected epidemic size']\n",
      "['knowledge', 'obtain in', 'practice']\n",
      "['γ', 'can', 'thus can used']\n",
      "['r0', 'in', 'figure 3b']\n",
      "['r0', 'in', 'figure 3a']\n",
      "['r0', 'in', 'figure 4a']\n",
      "['r0', 'in', 'figure 4b']\n",
      "['diagonal bands', 'in', 'figure 7a']\n",
      "['horizontal bands', 'in', 'figure 7b']\n",
      "['bernoulli type diﬀerential equation', 'with', 'general solution given by φ']\n",
      "['t', 'start', 'blue line']\n",
      "['l2', 'with', 'probability p k − l1 − 1']\n",
      "['i', 'hence make', 'three simplifying assumptions']\n",
      "['diﬀerence', 'in', 'resultant average epidemic size e']\n",
      "['diﬀerence', 'in', 'resultant average epidemic size e']\n",
      "['similar performance', 'with', 'ri']\n",
      "['one jump', 'in', 'x ′ x ′']\n",
      "['h one root', 'in', '0 1']\n",
      "['recurrent epidemics', 'in', 'infectious diseases']\n",
      "['corresponding product', 'from e', 's i']\n",
      "['synchrony', 'of pattern', 'science 286 968-971']\n",
      "['higher gain', 'in', 'smaller population']\n",
      "['variables', 'of set', 's']\n",
      "['algorithm', 'require', 'two conditions']\n",
      "['general solution', 'with', 'matrix function a']\n",
      "['same activity distribution', 'in', 'cases']\n",
      "['poisson', 'with', 'average degree smaller than one']\n",
      "['optimizing chemotherapy', 'in', 'hiv model']\n",
      "['fister', 'chemotherapy in', 'hiv model']\n",
      "['∈ g xi', 'number of', 'shortest paths']\n",
      "['geographical map', 'with', '100 × 100 nodes']\n",
      "['geographical map', 'with', '100 × 100 nodes']\n",
      "['vertical helper lines', 'mark', 'frequencies predicted by eq']\n",
      "['vertical helper lines', 'mark', 'frequencies predicted by eq']\n",
      "['vertical helper lines', 'mark', 'frequencies predicted by eq']\n",
      "['vertical helper lines', 'mark', 'frequencies predicted by eq']\n",
      "['may contact', 'with', 'infected individual']\n",
      "['ﬁrst time', 'in', 'other words']\n",
      "['homogeneous populations', 'with', 'seasonal forcing']\n",
      "['sparse inverse covariance estimation numericcitation', 'with', 'graphical lasso']\n",
      "['solution', 'lead to', 'n log n']\n",
      "['individuals', 'with', 'non-uniform population density']\n",
      "['n r', 'in', 'standard fokker-planck form']\n",
      "['eﬀect', 'in', 'particular on outbreak impact']\n",
      "['random multigraph', 'be', 'simple']\n",
      "['µ', 'for r', 'µ 0.6 1.0 down triangles with asynchronous updating']\n",
      "['λ', 'vs. r', 'µ 0.6 1.0 down triangles with asynchronous updating']\n",
      "['coupled disease-behavior interaction model', 'in', 'complex networks']\n",
      "['otte', 'impact of', 'avian inﬂuenza virus']\n",
      "['fraction', 'in', 'population']\n",
      "['infection', 'utilize', 'mean-ﬁeld approximation']\n",
      "['diﬀerent factors', 'of inﬂuence', 'awareness rate']\n",
      "['deterministic sir model', 'in', 'scenario']\n",
      "['equation 8', 'equation for', 'deterministic sir model']\n",
      "['ebola outbreak', 'in', 'west africa']\n",
      "['particle', 'with', 'rate γ']\n",
      "['function', 'as t', 'w']\n",
      "['function', 'as a', 't']\n",
      "['∆ n n', 'in', 'sum theorem']\n",
      "['non-linear approximations', 'have', 'have proposed']\n",
      "['transition matrix', 'be', 's']\n",
      "['randomness', 'in', 'infection']\n",
      "['several approaches', 'have', 'have developed']\n",
      "['result', 'set of', 'functions from dictionary']\n",
      "['delay term', 'make', 'analysis signiﬁcantly more complicated']\n",
      "['infection', 'after β', 'τ']\n",
      "['mary', 'model for', 'epidemics on adaptive network']\n",
      "['nodes', 'with', 'ﬁxed degree k']\n",
      "['mass vaccination programme', 'have', 'had applied']\n",
      "['transition', 'occur at', 'certain system size ζ']\n",
      "['effective epidemic thresholds', 'with', 'different recovery rates']\n",
      "['decision', 'in ahn', '2013 ieee 52nd annual conference on']\n",
      "['distribution', 'in', 'discussion']\n",
      "['network', 'in', 'twitter']\n",
      "['early detection', 'in', 'west africa']\n",
      "['early detection', 'in', 'west africa']\n",
      "['data set', 'can', 'can freely downloaded from web site']\n",
      "['pr', 'in', 'other words']\n",
      "['other words', 'in pr', 'pr']\n",
      "['case', 'in', 'same node']\n",
      "['case', 'in', 'same node']\n",
      "['models', 'be', 'key']\n",
      "['average outbreak size', 'in', 'power-law regime']\n",
      "['average outbreak size', 'in', 'power-law regime']\n",
      "['above prediction', 'in', 'most cases on sfns numericcitation']\n",
      "['β', 'for parameter', 'data obtained from google trends']\n",
      "['seir compartmental model', 'with', 'periodic forcing']\n",
      "['2 sirs model', 'with', 'environmental predictors']\n",
      "['hidden variable', 'in', 'sirs model']\n",
      "['number', 'in', 'sis model']\n",
      "['1 ζi', 'in', 'g']\n",
      "['optimal control', 'in', 'rabies metapopulation model']\n",
      "['numericcitation miller', 'model for', 'infectious disease spread']\n",
      "['errors', 'estimate with', '104 samples']\n",
      "['worm epidemics', 'in', 'wireless ad hoc networks']\n",
      "['memory eﬀects decay', 'in', 'time like power-law function']\n",
      "['similar properties', 'with', 'epidemic maps']\n",
      "['generated graphs', 'multigraph', 'where allowed']\n",
      "['lecture notes', 'in', 'computer science']\n",
      "['several values', 'in', 'range k']\n",
      "['few infected nodes', 'be', 'when observed']\n",
      "['similar results', 'have', 'interestingly have recently obtained']\n",
      "['asynchronous sir model', 'follow', 'procedure']\n",
      "['section 2', 'with', 'numerical simulations']\n",
      "['via percolation method', 'formula for', 'epidemic threshold']\n",
      "['individuals', 'have', 'partial immunization']\n",
      "['ﬁnal outbreak size', 'be', 'n ot affected by recovery rate']\n",
      "['modeling info rmation dissemination', 'in', 'generalized social networks']\n",
      "['mathematical models', 'in', 'biology']\n",
      "['contact', 'with', 'infected agents']\n",
      "['contact', 'with', 'infected agents']\n",
      "['plague epidemics', 'in', 'london']\n",
      "['four cases', 'for pr', 'v']\n",
      "['interaction', 'with', 'type c individual']\n",
      "['blythe', 'spread as', 'contact process on lattice']\n",
      "['increasing functions', 'in', 'example']\n",
      "['hepatitis b virus transmission model', 'with', 'vaccination']\n",
      "['key tool', 'in', 'epidemiology']\n",
      "['random graphs', 'with', 'given degree sequence']\n",
      "['rumor', 'of spread', 'ann']\n",
      "['time-varying state variable', 'in', 'cost function']\n",
      "['jacobian determinant', 'with', 'eigen values λi']\n",
      "['2 new class', 'in', 'section']\n",
      "['research', 'in', 'security']\n",
      "['equilibria', 'with', 'form']\n",
      "['lines', 'in', 'fig 4']\n",
      "['fundamental system', 'in', 'epidemiological modeling']\n",
      "['epidemiological models', 'with', 'age structure']\n",
      "['diﬀerent contexts', 'in', 'shao']\n",
      "['d', 'with', 'probability']\n",
      "['effective sir epidemic threshold', 'with', 'constant value µ']\n",
      "['scale', 'in', 'inﬁnite regime from eq']\n",
      "['scalar si model', 'with', 'β 0']\n",
      "['process', 'be', 'simple']\n",
      "['notation', 'be', 'simple']\n",
      "['assumptions', 'be', 'simple']\n",
      "['e.g multigraph', 'be', 'simple']\n",
      "['g ∗', 'be', 'simple']\n",
      "['multigraph', 'be', 'simple']\n",
      "['adv', 'in', 'appl']\n",
      "['epidemic threshold', 'in', 'uncorrelated networks']\n",
      "['epidemic threshold', 'of λqmf', 'red circles']\n",
      "['epidemic thresholds', 'in', 'clustered networks']\n",
      "['epidemic threshold', 'in', 'scale-free networks']\n",
      "['epidemic threshold', 'with', 'degree cor-relations']\n",
      "['epidemic spreading', 'in', 'networks']\n",
      "['k', 'be', 'branching']\n",
      "['k', 'be', 'branching']\n",
      "['infection', 'also infer from', 'fig 3']\n",
      "['reduction', 'in', 'overall epidemic size']\n",
      "['t p t ≤ t', 'yn of', 'outbreak']\n",
      "['outbreak', 'in', 'inﬁnite regime']\n",
      "['decision', 'in numericcitation', '2014 ieee 53rd annual conference on']\n",
      "['b. outbreak duration', 'with', 'eﬀective transmission rate α']\n",
      "['chebyshev', 'accord to', 'inequality']\n",
      "['chebyshev', 'by', 'inequality']\n",
      "['only input parameter', 'as p', 'k']\n",
      "['theorem 3.4', 'follow similarly from', 'theorem 5.2']\n",
      "['value independent', 'with', 'high probability']\n",
      "['agreement', 'with', 'refs']\n",
      "['susceptible-infected-recovered model', 'with', 'local dynamic vaccination']\n",
      "['model', 'of evolution', 'vl']\n",
      "['s i r model', 'with', 'local vaccination']\n",
      "['model', 'use', 'ebcm']\n",
      "['2.4 generalized contact model 2.5 immune-admitting model', 'in', 'section']\n",
      "['model', 'reﬂect', 'more realistic trends']\n",
      "['such active set updates', 'have', 'have used successfully']\n",
      "['ﬁeld experiments', 'with', 'aerially disseminated pathogens']\n",
      "['curve', 'with', 'blue circles']\n",
      "['sis process', 'of time', 'l']\n",
      "['moment', 'in', 'other words']\n",
      "['time-dependent indirect transmission rate', 'with model', 'joh et']\n",
      "['cluster sis model', 'with', 'q']\n",
      "['citation', 'be', 'day']\n",
      "['number', 'in', 'c1']\n",
      "['use', 'in', 'low-dimensional ode approximations such pairwise model']\n",
      "['stochastic methods', 'of handbook', 'citation']\n",
      "['type b', 'just tackle', 'eq']\n",
      "['3.2 parameter distributions shown', 'in', 'figures']\n",
      "['curves', 'converge to', 'same epidemic threshold']\n",
      "['observed infected set', 'in', 'network']\n",
      "['ﬁnal outbreak size', 'increase with', 'recovery rate']\n",
      "['observed state', 'prediction of', 'epidemic evolution']\n",
      "['unique counts', 'be', 'when used']\n",
      "['country j', 'affect by', 'crisis propagating through its link to countryi']\n",
      "['approach', 'in', 'step']\n",
      "['known models', 'in', 'context']\n",
      "['cornel l university', 'laboratory of', 'atomic state physics']\n",
      "['steady state', 'average over', '104 mcs']\n",
      "['endemic state', 'lose', 'stability']\n",
      "['edge', 'in', 'component ai']\n",
      "['∆ x', 'in', 'last case']\n",
      "['recovered individuals', 'of number', 'mean value']\n",
      "['one infected case', 'in', 'community']\n",
      "['activity distribution', 'in', 'network']\n",
      "['λ ∆ p', 'with', 'prediction from hmf theory']\n",
      "[\"l'industrie et la physique\", 'sabatier', 'universit']\n",
      "['expectations', 'in', 'interplay between science']\n",
      "['ﬁxation scales', 'with', 'characteristic population size n numericcitation']\n",
      "['t i n', '2 with', 'initial values s']\n",
      "['17 numericcitation e. m. rogers', 'diﬀusion of', 'innovations']\n",
      "['results', 'be', 'yet still cumbersome']\n",
      "['our ﬁnding', 'applicable to', 'other spreading dynamics']\n",
      "['e astrof ısica da universidade de lisboa', '‡ in', 'paper']\n",
      "['p. r. china m. k. mak ‡', 'in', 'paper']\n",
      "['3 behavior', 'in', 'limit']\n",
      "['m. e. j. newman', 'spread of', 'epidemic disease']\n",
      "['our bayesian inference procedure', 'use', 'simulated cholera data generated from model with time-varying environmental covariate']\n",
      "['multi-type branching process', 'with', 'type space t']\n",
      "['branching process', 'of region', 'pale green box']\n",
      "['0 → umax', 'with', 'single switching time τ ∗ s']\n",
      "['transient periodicity', 'in', 'biological dynamics']\n",
      "['eﬃciency', 'in', 'england']\n",
      "['rarity', 'in', 'typical human population']\n",
      "['n', 'be', 'when ﬁnite']\n",
      "['nodes', 'with', 'extremely large degree']\n",
      "['links', 'in', 'real-world networks']\n",
      "['two attracting directions', 'in', 'i2s plane']\n",
      "['preliminary scaling results', 'show in', 'figure 10']\n",
      "['optimal deterministic protocol', 'be', 'when applied']\n",
      "['color online r', 'network for', 'diﬀerent attachment number m']\n",
      "['sharp transitions', 'in', 'optimal strategy']\n",
      "['exponential distribution', 'with', 'rate ds']\n",
      "['agreement', 'with', 'expected scaling']\n",
      "['its dynamical behavior', 'be', 'given by lemma below']\n",
      "['state evolutions', 'with', 'control']\n",
      "['initial condition', 'be', '40 randomly selected nodes']\n",
      "['impact', 'in', 'case']\n",
      "['evolution', 'in', 'population']\n",
      "['epidemic size', 'get', 'non-zero value']\n",
      "['time horizon', 'with', 'trajectories resampled']\n",
      "['time horizon', 'with', 'trajectories resampled']\n",
      "['births', 'show', 'large annual variations']\n",
      "['t', 'period of', 'limit cycle']\n",
      "['comparative results', 'in', 'following tables']\n",
      "['approach', 'be', 'compartmental']\n",
      "['i-nodes', 's-state with', 'recovery rate µ']\n",
      "['immune hosts equation', 'in', 'model m1']\n",
      "['bounded number', 'with', 'uniformly bounded degrees']\n",
      "['good agreement', 'with', 'our predictions from eq']\n",
      "['non-negative population sizes', 'with ﬁxed', 'fps']\n",
      "['smaller population', 'well protect', 'foremost']\n",
      "['amount', 'in', 'smaller population v 1']\n",
      "['running time', 'be', 't 1 c']\n",
      "['µ', 'be', 'when small']\n",
      "['branching process approximations', 'in', 'previous section']\n",
      "['vector', 'in', 'similar way']\n",
      "['ck', 'be', 'dependent']\n",
      "['sir dynamics', 'with', 'arbitrary recovery rate']\n",
      "['case', 'with', 'µ']\n",
      "['contact opportunities', 'with', 'infected agents']\n",
      "['contact opportunities', 'with', 'infected agents']\n",
      "['edge density ∆', 'for subset', 'figure 4']\n",
      "['poisson', 'with', 'average degree one']\n",
      "['school epidemic run', 'in', 'school']\n",
      "['individuals', 'become', 'immune']\n",
      "['on their neighbours', 'with', 'rate β1']\n",
      "['average duration', 'grow logarithmically with', 'fig 2']\n",
      "['results', 'show in', 'figure 4b']\n",
      "['many eﬀorts', 'have', 'have done on topic']\n",
      "['g1', 'be', 'where scaling function']\n",
      "['infectious individuals', 'recover at', 'rate ρ 0']\n",
      "['fraction', 'of series', 'i']\n",
      "['fraction', 'of series', 'i']\n",
      "['fraction', 'of series', 'i']\n",
      "['fraction', 'of series', 'i']\n",
      "['fraction', 'of series', 'i']\n",
      "['fraction', 'of series', 'i']\n",
      "['campaign deadline', 'be', 'when suﬃciently small']\n",
      "['endemic ﬁxed point', 'be', 'where now located at origin']\n",
      "['random graph', 'with', 'given degrees']\n",
      "['comparative results', 'in', 'particular situation']\n",
      "['multilayer network', 'with', 'two layers']\n",
      "['discontinuous behavior', 'in', 'time']\n",
      "['asano', 'control in', 'rabies metapopulation model']\n",
      "['iα k', 'spread in', 'multilayer networks node']\n",
      "['expected number', 'in', 'stochastic model']\n",
      "['better agreement', 'with', 'simulations']\n",
      "['largest weight', 'in', 'model']\n",
      "['terms', 'use for', 'ﬁrst stage tweet ﬁltering 2']\n",
      "['research', 'on symposium', 'pages 2 15']\n",
      "['iv', 'in', 'lemma 6']\n",
      "['patches', 'be', 'where diﬀerent']\n",
      "['contact', 'with', 'pathogen']\n",
      "['small-world networks', 'with', 'connectivity ﬂuctuations']\n",
      "['delayed sir model', 'with', 'temporary immunity']\n",
      "['random graph', 'with', 'mean degree c']\n",
      "['ﬁnite state space e', 'with chain', 'j ∈ e']\n",
      "['u p', 'independent of', 'system size']\n",
      "['branching processes', 'in', 'disease epidemics']\n",
      "['n', 'be', 'where corner-stub cardinality']\n",
      "['realistic stochastic description', 'in', 'two populations']\n",
      "['block groups', 'with', 'higher homeownership rates']\n",
      "['result', 'announce in', 'eq']\n",
      "['monte carlo simulations', 'with', 'results']\n",
      "['various models', 'in', 'literature']\n",
      "['diﬀerence', 'in', 'initial condition']\n",
      "['hpai control', 'in', 'netherlands i']\n",
      "['recovered individuals', 'which as', 'expected decrease']\n",
      "['inset ﬁgures', 'zoom in on', '0 25']\n",
      "['numerical simulation dynamical model', 'with', 'lotka-volterra linear functional response']\n",
      "['models', 'with', 'immunity waning']\n",
      "['seijr model', 'with', 'parameters α']\n",
      "['l', 'with time', 'seijr model']\n",
      "['connections', 'break', 'links from non-infected nodes']\n",
      "['its immunization', 'cause', 'its infection']\n",
      "['actual distributions', 'in', 'fig 4b']\n",
      "['higher order nonlinearities', 'in', 'infection term']\n",
      "['pa-sirs system', 'in', 'eq']\n",
      "['epidemics', 'accelerate', 'innovation 8']\n",
      "['zhou', 'behaviors of', 'susceptible-infected epidemics']\n",
      "['dependent percolation process', 'rely on', 'direct infections between particles']\n",
      "['general multi-group sir model', 'with', 'birth']\n",
      "['models', 'resort to', 'cumbersome monte carlo simulations']\n",
      "['convergence', 'in', 'case']\n",
      "['hadeler', 'manipulate', 'our derivation']\n",
      "['more viral campaigns', 'less costly in', 'other words']\n",
      "['role', 'in', 'seasonally forced stochastic seir models']\n",
      "['stochastic sir dynamics', 'with', 'prophylactic vaccination']\n",
      "['vaccine-induced immunity', 'in', 'our framework']\n",
      "['spatial sir epidemics', 'in', 'dimensions']\n",
      "['number', 'with', '0 ≤ i j ≤ k']\n",
      "['number', 'in', 'shortest path connecting']\n",
      "['3.4 variation', 'in', 'total infection cases']\n",
      "['conclusions', 'in', 'conclusion']\n",
      "['lecture notes', 'in', 'statistics']\n",
      "['conclusions', 'in', 'present paper']\n",
      "['epidemic spreads', 'number of', 'susceptible targets declines']\n",
      "['ﬁrst model', 'for r', 't']\n",
      "['second order phase transition', 'with', 'mean ﬁeld exponents at tc']\n",
      "['susceptible nodes', 'infect with', 'rate β']\n",
      "['contact processes', 'have', 'critical value 0']\n",
      "['ν', 'with', 'single infectious host at beginning']\n",
      "['user', 'may delete rumor from', 'her/his blog']\n",
      "['function', 'as plot', 'n']\n",
      "['sir epidemics', 'with', 'given degree sequence']\n",
      "['ring vaccination', 'eradicate', 'fmd']\n",
      "['ring vaccination', 'more suitable for', 'fmd']\n",
      "['important eﬀect', 'in', 'vaccination policies']\n",
      "['ring vaccination', 'eradicate', 'fmd']\n",
      "['edges', 'in', 'graph']\n",
      "['β', 'in', 'its matrix form accounts for infection rate']\n",
      "['15 20', 'in', 'our ﬁgures']\n",
      "['global dynamics', 'in', 'such system']\n",
      "['sir epidemic dynamics', 'with', 'synchronous updating']\n",
      "['immune individuals', 'with', 'immunity level']\n",
      "['t', 'display as', 'red dots']\n",
      "['red curve', 'correspond to', 'critical regime']\n",
      "['growth', 'in', 'well-mixed population']\n",
      "['entire population', 'vaccinate', 'entire population']\n",
      "['expected number', 'in', 'otherwise healthy population']\n",
      "['predicted fraction', 'in', 'population']\n",
      "['global characteristics', 'can', 'can estimated']\n",
      "['mathematical models', 'often establish with', 'static parameters']\n",
      "['references numericcitation barrat a', 'm on', 'complex networks']\n",
      "['wang', 'outbreak in', 'west africa']\n",
      "['actual number', 'in', 'observation']\n",
      "['actual number', 'in', 'observation']\n",
      "['ˆit equation', 'with', 'unique solution ˆit']\n",
      "['0.2 days', 'in', 'blue']\n",
      "['convergence', 'positivity of', 'infection probabilities']\n",
      "['panel a community sizes probability density function', 'in', 'original network']\n",
      "['discussion', 'in', 'paper']\n",
      "['dashed lines', 'give in', 'eq']\n",
      "['dashed lines', 'give in', 'eq']\n",
      "['point', 'be', 'ﬁxed']\n",
      "['age structure', 'with model', 'j. math']\n",
      "['classical sirs model', 'with', 'constant delay s ′']\n",
      "['directional walking pattern', 'can', 'can described by biased random walk']\n",
      "['table 1 prior knowledge form', 'size of', 'neighborhood max possible size']\n",
      "['spread', 'in', 'rural areas']\n",
      "['p. caley', 'time for', 'spread']\n",
      "['above table', 'use', 'unique counts']\n",
      "['doob', 'by', 'inequality']\n",
      "['chebyshev', 'have', 'inequality']\n",
      "['jensen', 'by', 'inequality']\n",
      "['realistic models', 'could', 'more generally could taken into account']\n",
      "['estimates', 'with', 'non-zero value']\n",
      "['0 0n', '0n for', 'ii']\n",
      "['0 0n', '0n for', 'ii']\n",
      "['spechler', 'have', 'original irsir model']\n",
      "['parameter ν', 'should away from', 'should far critical points']\n",
      "['0 16 processes', 'in', 'their original time scaling']\n",
      "['our result', 'in', 'item']\n",
      "['other timescales', 'in', 'system']\n",
      "['coeﬃcient', 'exp in', 'large-η limit']\n",
      "['sars transmission patterns', 'in', 'china']\n",
      "['disease', 'in', 'surviving population']\n",
      "['work', 'in', 'progress']\n",
      "['nd', 'origins of', 'ma jor human infectious diseases']\n",
      "['sars', 'be', 'public']\n",
      "['percentage', 'in', 'block']\n",
      "['dots', 'numerically solve', 'eq']\n",
      "['dots', 'numerically solve', 'eq']\n",
      "['y', 'matrix with', 'eigenvalues']\n",
      "['effective epidemic threshold', 'attract', 'increasing attentions']\n",
      "['infectious burden', 'in', 'unconstrained conditions']\n",
      "['degree distribution', 'do apparently influence considerably', 'outcome']\n",
      "['o', 'with', 'length']\n",
      "['o', 'with', 'length n']\n",
      "['vaccines', 'use of', 'through years has always controversial issue']\n",
      "['k − 1', 'with', 's1 ,1']\n",
      "['epidemic bond percolation growth', 'in', 'sir language']\n",
      "['system', 'exhibit', 'only two attractors']\n",
      "['parallel', 'with', 'new local process']\n",
      "['estimates', 'converge to', 'actual value']\n",
      "['death', 'in', 'zombie models']\n",
      "['good agreement', 'with', 'numerical effective epidemic thresholds identiﬁe by variability measure']\n",
      "['their features', 'very low number of', 'infectives between recurrent epidemics']\n",
      "['intermediate amounts', 'be', 'when available']\n",
      "['sir', 'epidemic on', 'random multigraph g ∗']\n",
      "['random network', 'with', 'heterogeneous connectivity']\n",
      "['discontinuous ﬁnal size distribution', 'with', 'jump at r0']\n",
      "['our system', 'with', '104 nodes']\n",
      "['simple graph', 'with', 'n vertices']\n",
      "['compartment diagram', 'with', 'vital dynamics']\n",
      "['propagation dynamics', 'with', 'death']\n",
      "['validation', 'with', 'vital dynamics']\n",
      "['detailed study', 'make in', 'two dimensions']\n",
      "['third equation', 'be', 'bit more complicated']\n",
      "['anderson', 'disease of', 'humans']\n",
      "['pronounced bump', 'in', 'simulations']\n",
      "['extension', 'main concern of', 'paper']\n",
      "['size', 'in', 'g']\n",
      "['our model', 'assume', 'quadratic cost']\n",
      "['change', 'in', 'susceptible individuals']\n",
      "['hubs', 'level of', 'communities']\n",
      "['ﬁnite diﬀerence', 'in', 'ﬁrst equation']\n",
      "['individuals', 'in', 'layers']\n",
      "['n individuals', 'in', 'layers']\n",
      "['small associations', 'with', 'health']\n",
      "['tradeoﬀs', 'be', 'where critical']\n",
      "['above analysis', 'repeat for', 'inactive outbreaks']\n",
      "['number', 'in', 'its corresponding cluster']\n",
      "['humans', 'in diseases', 'citation']\n",
      "['region', 'be', 'when activated']\n",
      "['connections', 'with', 'full models']\n",
      "['international series', 'in', 'decision processes']\n",
      "['node', 'become', 'healthy from infected']\n",
      "['g', 'with', 'p for proof see']\n",
      "['information source', 'in', 'figure 3']\n",
      "['function ψ', 'namely vanish in', 'only isolated points']\n",
      "['numbers', 'present as', 'base 10 exponential notation']\n",
      "['opinions', 'model of', 'where consensus']\n",
      "['such graphs', 'study in', 'numericcitation']\n",
      "['contrast', 'with', 'traditional models']\n",
      "['voter model', 'in', 'antal et']\n",
      "['people', 'with', 'sars']\n",
      "['parameter values', 'in', 'region ii']\n",
      "['mean number', 'value of', 'square']\n",
      "['hi', 'n around', 'zero']\n",
      "['zero', 'around n', 't']\n",
      "['red free half-edge pairs', 'square of', 'jump satisﬁes']\n",
      "['ζm', 'number of', 'new red free half-edges']\n",
      "['pair', 'in', 'two diﬀerent ways']\n",
      "['active phase', 'with', 'oscillatory behavior']\n",
      "['sir epidemic threshold', 'inversely proportional to', 'average connectivity hki numericcitation']\n",
      "['source', 'in', 'distance']\n",
      "['assumptions', 'in', \"watanabe 's theorem\"]\n",
      "['proof', 'in', 'lalley']\n",
      "['diﬀerent works', 'characterize', 'controls']\n",
      "['convex optimization solvers', 'can', 'can applied']\n",
      "['disease propagation', 'in', 'species']\n",
      "['1 x', 'in', 'set u']\n",
      "['percolation processes', 'with', 'ﬁnite dependencies']\n",
      "['inﬁnite', 'in', 'case']\n",
      "['sir models', 'model', 'pathogens']\n",
      "['two random vertices', 'connect by', 'shortcut']\n",
      "['epidemic outbreak', 'in', 'localized geographic region']\n",
      "['most well-known propagation models', 'in', 'literature']\n",
      "['similar recursive formula', 'can', 'can used']\n",
      "['no-one', 'remain', 'infectious forever']\n",
      "['recovery rates', 'be', 'also known']\n",
      "['transmissibility', 'be', 'when lower']\n",
      "['transmissibility', 'be', 'when lower']\n",
      "['exponent − ν', 'with ρ', 'ko']\n",
      "['out-community ρ', 'with', 'exponent − ν']\n",
      "['state evolution', 'with', 'control']\n",
      "['state evolution', 'with', 'control']\n",
      "['state evolution', 'with', 'control']\n",
      "['state evolution', 'with', 'control']\n",
      "['size', 'in', 'g']\n",
      "['free parameter', 'in', 'our study']\n",
      "['peaks', 'in', 'middle']\n",
      "['peaks', 'thus exist at', 'medium r. iv']\n",
      "['description', 'with', 'just two stochastic variables']\n",
      "['k', 'with', 'rate p along susceptible-infected link']\n",
      "['responses', 'class of', 'individuals']\n",
      "['individuals', 'take', 'measures']\n",
      "['individuals', 'take', 'protective measures']\n",
      "['chain', 'be', 'scale-invariant sw network']\n",
      "['hysteresis eﬀect', 'describe in', 'main text']\n",
      "['numbers', 'present as', '10-base exponential notation']\n",
      "['i-state', 'with', 'evident clinical symptoms labelled as i']\n",
      "['c2a', 'in', 'appendix c']\n",
      "['appendix c', 'in c2a', 'c2c']\n",
      "['two hubs', 'each with', 'n']\n",
      "['two hubs', 'each with', 'n']\n",
      "['discussion', 'in', 'paper']\n",
      "['nodes', 'with', 'higher susceptibility']\n",
      "['nodes', 'with', 'large betweenness centrality']\n",
      "['nodes', 'neighbor', 'pagerank']\n",
      "['collection', 'number of', 'nodes']\n",
      "['nodes', 'with', 'combined k-shell ks']\n",
      "['nodes', 'for susceptibility', 'base 10']\n",
      "['set', 'in', 'example']\n",
      "['individuals', 'have about', 'disease']\n",
      "['israel science foundation', 'ministry of', 'science']\n",
      "['a24', 'in', 'case']\n",
      "['hosts', 'with', 'waning immunity']\n",
      "['markov process', 'with', '6 probability measure p']\n",
      "['our method', 'overestimate', 'nodes ceptibility']\n",
      "['two vertices', 'in', 'same household']\n",
      "['strain 1', 'with', 'rate']\n",
      "['proposed numerical solution', 'in', 'paper']\n",
      "['excellent agreement', 'with', 'stochastic simulations']\n",
      "['recovered individuals', 'lose', 'immunity']\n",
      "['t', 'n for', 'time']\n",
      "['node types', 'have', 'poissonian degree distribution']\n",
      "['none-increasing function', 'in', 'summary']\n",
      "['0 kni', 'k of', '∞ xk']\n",
      "['partial knowledge', 'with', 'individual rights']\n",
      "['r', 'be', 'even when low']\n",
      "['example', 'in', 'figure 2']\n",
      "['networks', 'with', 'heterogeneous degree distributions']\n",
      "['household cluster', 'have', 'similar interpretation as school cluster']\n",
      "['mean time', 'in', 'appendix d. figure 7']\n",
      "['such hyper-graph counting', 'quickly become', 'too tedious']\n",
      "['litter sizes', 'be', 'all independent']\n",
      "['model', 'as lm', 'koopman j. percolation on heterogeneous networks']\n",
      "['our approach', 'with', 'changing n/s']\n",
      "['other hand', 'on infectiousness', 'di dt']\n",
      "['section 6', 'be', 'conclusion']\n",
      "['given time', 'in', 'maximal load epidemic']\n",
      "['fraction', 'in', 'endemic state']\n",
      "[\"j 's susceptibility\", 'set', 'process']\n",
      "['transitions', 'with', 'probability ρi']\n",
      "['ϕ', 'transition with', 'probability ρi']\n",
      "['1 n', 'number of', 'simulations']\n",
      "['special case', 'in', 'sect 3.1']\n",
      "['w', 'with', 'three diﬀerent hki']\n",
      "['recovered β', 'in', 'panels']\n",
      "['s i r processes', 'model', 'transmission events']\n",
      "['statistics', 'can', 'can calculated']\n",
      "['multilayer network', 'be', 'more fragile']\n",
      "['dynamic void node percolation process', 'with', 'instantaneous void control parameter']\n",
      "['method', 'underestimate', 'true diﬀerence']\n",
      "['vertex vl', 'choose with', 'probability al l n']\n",
      "['diﬀusion', 'in', 'uk']\n",
      "['closeness', 'like measure', 'outperform degree']\n",
      "['network', 'have', 'structure']\n",
      "['terms', 'in', 'ith log-likelihood']\n",
      "['second sum', 'in', 'likelihood ratio']\n",
      "['current models', 'of', 'generally two types']\n",
      "['community structure', 'extract by', 'fast heuristic method based on modularity optimization numericcitation']\n",
      "['exact parametric form', 'in', 'section iii']\n",
      "['resulting model parameters', 'show in', 'table v']\n",
      "['underreporting bias', 'with', 'estimated mean reporting rate']\n",
      "['cc', 'cluster of', 'recovered individuals']\n",
      "['order parameter p', 'number of', 'recovered individuals']\n",
      "['agent', 'be', 'when vaccinated']\n",
      "['our scaling predictions', 'in', 'agreement']\n",
      "['general problem', 'even embed', 'geographical map numericcitation']\n",
      "['situation', 'with', 'excess vaccine']\n",
      "['total tendency', 'same with', 'other results']\n",
      "['ﬁrst cases', 'in', 'toronto']\n",
      "['situation', 'with', 'diﬀerent rates']\n",
      "['than cases', 'with', 'diﬀerent r0']\n",
      "['networks', 'with', 'ﬁnite threshold pc']\n",
      "['transmissibility', 'decrease', 'overlap']\n",
      "['changes', 'in', 'population size n']\n",
      "['monte carlo simulation points', 'in', 'fig 1']\n",
      "['communicable disease spreads', 'in', 'population']\n",
      "['algorithm', 'count', 'states']\n",
      "['evolution', 'in', 'uncontrolled sis process']\n",
      "['second observation', 'study', 'diﬀerent community structures']\n",
      "['koopmans', 'transmission of', 'h7n7 avian inﬂuenza a virus']\n",
      "['eradication time ti', 'relevant to', 'mesh point τi']\n",
      "['auc', 'remain below', 'considerably one']\n",
      "['control parameter umax', 'in', 'figs 3']\n",
      "['static networks', 'in', 'large population']\n",
      "['∗ ming tang ,1 3', 'school of', 'physics']\n",
      "['further infection', 'result in', 'shortest extinction time']\n",
      "['us western states power grid', 'network of', 'coauthorships between scientists posting on astrophysics e-print archive between jan 1 1995 reconstructed from bgp tables posted by university']\n",
      "['results', 'in', 'section v']\n",
      "['non', 'be', 'system formed by single networks interacting through external connections between them']\n",
      "['contagion process', 'in', 'sis models']\n",
      "['aproximation pc ∼', 'in', 'only networks larger than 109 nodes']\n",
      "['corresponding bond percolation problem', 'use', 'message passing technique']\n",
      "['susceptible ones', 'in', 'same population']\n",
      "['its interaction', 'with', 'mass action terms']\n",
      "['g ∗', 'be', 'thus simple']\n",
      "['control signal', 'in', 'sis model']\n",
      "['network', 'be', 'when rewired']\n",
      "['numericcitation lomonosov m', 'combinatorics of', 'reliability monte random structures algorithms']\n",
      "['n ∞', 'be', 'average outbreak size condieq']\n",
      "['sars cases', 'in', 'hk']\n",
      "['good agreement', 'in', 'most cases']\n",
      "['disease outbreak', 'in', 'sis models']\n",
      "['same parameter values', 'for equation', 'blue solid line']\n",
      "['probabilities', 'in', 'section iv']\n",
      "['diseases', 'in', 'complex networks']\n",
      "['temperature dependence', 'could', 'correspondingly could considered as leading factor']\n",
      "['targeted mailers', 'with', 'concurrently semi-annual citywide advertising push remind']\n",
      "['diﬀusion process', 'in', 'two dimensions']\n",
      "['upper limit', 'in', 'sum']\n",
      "['same behavior', 'observe for', 'v']\n",
      "['non-active nodes', 'can still receive', 'connections from other active vertices']\n",
      "['05.45.-a i. introduction epidemic spread', 'in', 'human populations']\n",
      "['proof', 'in', 'generic starting intervention time τ']\n",
      "['lower epidemic threshold', 'in', 'latter case']\n",
      "['pro ject', 'be', 'where started']\n",
      "['epidemic threshold λqmf', 'use', 'qmf method']\n",
      "['variation', 'with', 'increasing ﬁnite time']\n",
      "['waning immunity', 'in', 'england']\n",
      "['branching processes', 'in', 'biology']\n",
      "['high loss', 'in', 'national income']\n",
      "['infectives', 'can', 'can easily introduced']\n",
      "['problem', 'time-varying version of', 'thus np-hard']\n",
      "['studies', 'with', 'given degree sequence']\n",
      "['given degree sequence', 'with', 'bounded vertex degrees']\n",
      "['results', 'display as', 'solid lines']\n",
      "['results', 'display as', 'solid lines']\n",
      "['last term', 'in', '3rd equation']\n",
      "['discrete-time markov chain approach', 'in', 'complex networks']\n",
      "['contact', 'with', 'infected individual']\n",
      "['ﬁrst author', 'acknowledge', 'support from nih grant k25 ai079404-02']\n",
      "['contact', 'with', 'infective one']\n",
      "['two switches', 'take place for', 'r0 2.9']\n",
      "['two switches', 'take', 'place']\n",
      "['n 1', 'with population', 'two well-separated cities']\n",
      "['two non-interacting populations', 'with', 'n 1']\n",
      "['general infection model', 'in', 'inference']\n",
      "['total weight', 'in', 'terminal infection peak']\n",
      "['basic sir', 'in individual', 'burden']\n",
      "['total weight', 'in', 'our network']\n",
      "['geometric distribution', 'with', 'parameter p']\n",
      "['individuals', 'hence generate', 'independent identically distributed oﬀspring']\n",
      "['size eﬀects', 'study through', 'additional simulations']\n",
      "['expected number susceptible', 'underestimate', 't']\n",
      "['various reasons', 'in', 'practice']\n",
      "['remaining vaccine', 'optimally allocate to', 'large population']\n",
      "['results', 'in', 'figs 7']\n",
      "['standard dawson-watanabe process', 'with', 'same variance parameter']\n",
      "['2 ˙pr', 'in', 'ﬁrst equation']\n",
      "['numerous computational studies', 'have', 'have performed']\n",
      "['lower values', 'in', 'case']\n",
      "['conditions', 'so avoid', 'transients']\n",
      "['f', 'be', 'where maximum']\n",
      "['κ increases', 'with', 'k']\n",
      "['connected 1-dimensional lattice', 'with', 'periodic boundary conditions']\n",
      "['sample', 'extract in', 'dec. 2008']\n",
      "['numerical simulations', 'conﬁrm', 'theoretical analysis']\n",
      "['changes', 'in', 'customer awareness']\n",
      "['science', 'for foundation', 'fct']\n",
      "['ﬁve steps', 'with', 'three conditions']\n",
      "['contact', 'with', 'healthy individuals']\n",
      "['allowed transitions', 'in', 'state']\n",
      "['current epidemiological models', 'of', 'generally two types']\n",
      "['tment control', 'with', '0 de ﬁnition 1']\n",
      "['minimal level', 'in', 'immune hosts']\n",
      "['commerce', 'in', 'infected areas']\n",
      "['farms', 'in', 'infected areas']\n",
      "['2 prediction', 'seem', 'rather paradoxical at ﬁrst glance']\n",
      "['social circles', 'in', 'ego networks']\n",
      "['clustered networks', 'of gcc', 'giant connected component']\n",
      "['differences', 'be', 'summarized below']\n",
      "['layers', 'schematic draw of', 'shells obtained by k-shell method']\n",
      "['statistics', 'in', 'system']\n",
      "['complex network interactions', 'can', 'when can safely neglected']\n",
      "['ﬁxed-points', 'cost of', 'epidemic numericcitation']\n",
      "['immunization strategy', 'get', 'more eﬀective against disease']\n",
      "['individuals', 'with', 'infectious symptoms']\n",
      "['three diﬀerent types', 'in', 'g']\n",
      "['difference', 'in', 'effective epidemic threshold']\n",
      "['transition rate matrix q', 'with x', 't']\n",
      "['further model development', 'for option', 'citation']\n",
      "['straight lines', 'in', 'inset']\n",
      "['network', 'have', 'topology']\n",
      "['network', 'be', 'coherent']\n",
      "['prior infection', 'with', 'one strain irrelevant immunologically to new infection']\n",
      "['other hand', 'on compact', 'µkgkt']\n",
      "['proposed x ∗', 'weight of', 'smc algorithm']\n",
      "['diﬀusion', 'in', 'competitive economic networks']\n",
      "['cbp', 'be', 'branching']\n",
      "['r', 'with', 'subscript']\n",
      "['subscript', 'with r', 't']\n",
      "['corporate recovery', 'with', 'immunity']\n",
      "['values pr', 'with', 'integer coeﬃcients']\n",
      "['susceptible individuals', 'be', 'born']\n",
      "['monte carlo simulations', 'discard', 'transients']\n",
      "['β', 'for β', 'y ∈ zd as 1 φl']\n",
      "['vaccine', 'be', 'completely eﬀective']\n",
      "['ﬂuctuations', 'exhibit', 'power law behavior']\n",
      "['1-d chain', 'with', 'periodic boundary conditions']\n",
      "['numericcitation lederberg s. suppression', 'in', 'lysogenic bacteria']\n",
      "['physicists', 'for introduction', 'citation']\n",
      "['vaccine doses', 'have', 'where have prepositioned']\n",
      "['equation', 'with', 's']\n",
      "['case counts', 'observe at_time', 'once 14 days']\n",
      "['inset data', 'with', 'w rescaled by appropriate wc calculated from eq']\n",
      "['poisson infection', 'be', 'where assumed']\n",
      "['fraction', 'of 30', 'd']\n",
      "['case', 'distribution with', 'mean 1']\n",
      "['non-diagonal elements', 'in', 'model']\n",
      "['basic sir model', 'in', 'single relatively small community']\n",
      "['o', 'with', 'high probability']\n",
      "['transition rates', 'with chain', 'x t']\n",
      "['transition rates', 'with chain', 'y']\n",
      "['last process', 'in', 'eq']\n",
      "['n. wang', 'outbreak in', 'networks']\n",
      "['typical time', 'in', 'single evolution step']\n",
      "['agreement', 'with', 'our monte carlo simulations']\n",
      "['results', 'in', 'figure 1 show']\n",
      "['c 1', 'be', 'where constant']\n",
      "['population', 'in partition', 'each representing']\n",
      "['o', 'belong to', 'inﬁnite open cluster']\n",
      "['realistic models', 'resort to', 'cumbersome monte carlo simulations']\n",
      "['199 cases', 'in', '34 states']\n",
      "['bracketed term', 'in', 'integrand']\n",
      "['ε', 'be', 'chosen']\n",
      "['com/rebates', 'access', 'jan 3rd 2018']\n",
      "['index node', 'clearly have', 'many long range linkages']\n",
      "['r', 'be', 'even when relatively low']\n",
      "['susceptible hosts', 'become', 'infective themselves']\n",
      "['pseudo-linear part', 'in', 'particular region']\n",
      "['inﬁnitesimal rates', 'in', 'equation']\n",
      "['simulations', 'in', 'two dimensional map']\n",
      "['systems', 'in', 'study']\n",
      "['based approach', 'in', 'proc']\n",
      "['fraction', 'in', 'steady state']\n",
      "['susceptible-infective-recovered-susceptible model', 'for equation', 'pa']\n",
      "['three conventional methods', 'be', 'ﬁ nite-size scaling analysis']\n",
      "['system', 'in s', 'susceptible experiences']\n",
      "['ideas', 'with', 'other 5 individuals']\n",
      "['quantitative comparison', 'with', 'mean-ﬁeld solution']\n",
      "['two constants', 'in', 'solution']\n",
      "['∞ σbt', 'in', 'simple graph case']\n",
      "['sir framework', 'with', 'limited resources']\n",
      "['m. boguna r. pastor-satorras a. vespignani epidemic spreading', 'in', 'complex networks with degree correlations']\n",
      "['parameter values', 'in', 'region iii']\n",
      "['function', 'in', 'p t']\n",
      "['their characteristic features', 'very low number of', 'infectives between recurrent epidemics']\n",
      "['average area', 'in', 'eq']\n",
      "['average area', 'in', 'eq']\n",
      "['characterization', 'in', 'eq']\n",
      "['tw', 'in', 'eq']\n",
      "['pt t', 'in', 'eq']\n",
      "['reported value', 'in', 'phe']\n",
      "['disease', 'with', 'immunization']\n",
      "['recovery rate', 'represent by', 'γi']\n",
      "['infection', 'with', 'rate p']\n",
      "['pieter trapman2 university', 'in', 'paper']\n",
      "['terms', 'in', 'square bracket cancel']\n",
      "['new phenomena', 'may', 'may observed']\n",
      "['17 v. scaling', 'in', 'weak infection regime']\n",
      "['path', 'in', 'epidemic generated graph']\n",
      "['conclusion', 'in', 'paper']\n",
      "['several approaches', 'study', 'age-of-infection models']\n",
      "['several approaches', 'study', 'models']\n",
      "['incubation', 'in', 'model']\n",
      "['recovery', 'in', 'd']\n",
      "['spread', 'in', 'community']\n",
      "['conclusions', 'in', 'study']\n",
      "['link', 'with', 'highest strength']\n",
      "['structured population models', 'in', 'biology']\n",
      "['extensive simulations', 'in', 'real-world social networks']\n",
      "['original networks', 'in', 'fig 4']\n",
      "['facebook', 'rewire', 'enron networks']\n",
      "['their degrees', 'in', 'ˆa']\n",
      "['than homogeneous networks', 'with', 'same mean degree hki']\n",
      "['introduced dynamical processes', '0 such as', 'random walk 22 23']\n",
      "['concluding remarks', 'make in', 'last section']\n",
      "['several related works', 'in', 'area']\n",
      "['extent', 'to epidemic', 'idea about their clinical features']\n",
      "['initial conditions', 'so avoid', 'transients']\n",
      "['i', 'hence make', 'three assumptions']\n",
      "['following households', 'in', 'sequence']\n",
      "['following schools', 'in', 'sequence']\n",
      "['opinion leadership', 'in', 'new product diﬀusion']\n",
      "['time evolution', 'in', 'population']\n",
      "['time evolution', 'in', 'population']\n",
      "['bernoulli raﬄe', 'with', 'probability given by value c']\n",
      "['normal times', 'be', 'also likely']\n",
      "['zhu', 'control under', 'delayed model']\n",
      "['s. p. sethi', 'advertise in', 'new-product adoption model']\n",
      "['optimal advertising', 'in', 'new-product adoption model']\n",
      "['numbers', 'close to', 'very true data numericcitation']\n",
      "['infected nodes', 'of o', 'ρi']\n",
      "['equation', 'in', 'model m1']\n",
      "['cone d', 'positively invariant for', 'model m1']\n",
      "['pde', 'in', 'model m1']\n",
      "['r', 'in', 'model m1']\n",
      "['seirs model', 'with', 'latency']\n",
      "['particles', 'remove from', 'system']\n",
      "['innovation diﬀusion', 'in', 'heterogeneous populations']\n",
      "['node', 'within', 'community']\n",
      "['acknowledgements m.s', 'acknowledge', 'ﬁnancial support from james s. mcdonnell foundation']\n",
      "['neighbor', 'with', 'state xj']\n",
      "['∗ ming tang ,1 3', 'school of', 'sciences']\n",
      "['underlying graph', 'be', 'when tree']\n",
      "['results', 'average over', '10 × 104 independent realizations on different 10 networks']\n",
      "['whose opinion', 'choose at', 'random between four possible opinion states']\n",
      "['number', 'number of', 'carriers']\n",
      "['greatest portion', 'in', 'expectation']\n",
      "['equal increment', 'in ρ', 'kth']\n",
      "['ρ', 'in', 'equal increment']\n",
      "['added edges', 'in', 'configuration model']\n",
      "['configuration model', 'with', 'added triangles']\n",
      "['larger gains', 'in', 'protection']\n",
      "['agreement', 'with', 'previous central value']\n",
      "['giant susceptible component', 'be', 'destroyed']\n",
      "['giant susceptible component', 'be', 'destroyed']\n",
      "['one-dimensional problem', 'be', 'once solved']\n",
      "['recovered x', 'r', 't']\n",
      "['2 exact epidemic thresholds', 'in', 'markovian epidemic models']\n",
      "['remarkable agreement', 'with', 'numerical simulations']\n",
      "['networks', 'with', 'disassortative mixing patterns']\n",
      "['ﬁnancial support', 'with', 'grant number ksrc16006536']\n",
      "['ﬁnancial support', 'with', 'grant number 11501542']\n",
      "['maximum number', 'in', 'our runs']\n",
      "['10000 network realizations', 'with', 'n']\n",
      "['bifurcations', 'in', 'ﬂuctuating systems']\n",
      "['optimal solution', 'increasingly allocate resource to', 'small population']\n",
      "['g', 'for theorem', 'ii']\n",
      "['accurate epidemic thresholds', 'in', 'networks']\n",
      "['ebola outbreak', 'in', 'conakry']\n",
      "['theory', 'in', 'biosciences 131 1']\n",
      "['mean ﬁeld equations', 'with', 'noise strength ǫ']\n",
      "['fi ∈ rn +1', 'be', 'i-th unit column vector']\n",
      "['entire parametric space', 'on ratio', 'citation simulations']\n",
      "['1 2 kanako mizuno', 'magnitude of', 'outbreak']\n",
      "['average outbreak sizes', 'in', 'two-and three-type cbps']\n",
      "['three main stable attractors', 'cycle of', 'period 1 2']\n",
      "['ﬂuctuations', 'in', 'infectives near bifurcation point']\n",
      "['two directed edges', 'with', 'same endvertices']\n",
      "['fractional diﬀerential equations', 'with', 'caputo-type derivative']\n",
      "['siblings', 'go to', 'same school']\n",
      "['new schools', 'in', 'same way']\n",
      "['two auxiliary processes', 'in', 'lemma']\n",
      "['purple line', 'show line with', 'slope 1']\n",
      "['epidemic outbreaks', 'accelerate', 'information dissemination']\n",
      "['rates', 'proportion of', 'λg']\n",
      "['opinions', 'can', 'can transmitted via other media']\n",
      "['vertical lines', 'be', 'epidemic threshold']\n",
      "['immunity', 'in', 'vaccinated populations']\n",
      "['sis', 'in', 'real time varying networks']\n",
      "['similar relations', 'hold for', 'bi']\n",
      "['similarities', 'in', 'ways disease spreads']\n",
      "['comparison', 'with', 'site percolation process remains']\n",
      "['connection', 'with', 'acute infectious diseases modeling']\n",
      "['above full infection', 'with', 'ﬁxed parameters']\n",
      "['contact', 'with', 'their infected nearest neighbors']\n",
      "['observed tra jectory', 'of distribution', 'l p']\n",
      "['vaccination', 'be', 'when delayed']\n",
      "['stage', 'have', '15 days']\n",
      "['results', 'average', 'over 104 independent realizations on network']\n",
      "['results', 'average', 'over 106 independent realizations on network']\n",
      "['contact', 'with', \"infected animal 's body ﬂuid\"]\n",
      "['x0 i i ∈ v', 'number of', 'diﬀerent realizations']\n",
      "['number', 'in', 'such cluster 1']\n",
      "['diﬀerent', 'in', 'opposite limit']\n",
      "['hysteresis study', 'with', 'connections']\n",
      "['our approach', 'predict through', 'eq']\n",
      "['discussion', 'in', 'work']\n",
      "['advances', 'in', 'applied probability 622 635']\n",
      "['g1', 'be', 'satisﬁed']\n",
      "['exact computations', 'in', 'small populations']\n",
      "['outbreak sizes', 'indeed obey', 'power-law distribution with exponent -1.5']\n",
      "['same formalism', 'explain above for', 'social networks']\n",
      "['infected nodes', 'of stepnumber', '1 − θ']\n",
      "['discussion', 'in', 'work']\n",
      "['great agreement', 'with', 'analytical predictions']\n",
      "['variation', 'in', 'total cases']\n",
      "['vespignani a epidemic spreading', 'in', 'scale-free networks']\n",
      "['many lower parts', 'in', 'fig 1']\n",
      "['simplicity', 'in', 'our analysis']\n",
      "['considerable noise', 'in', 'simulations']\n",
      "['nonlinear science center', 'in', 'paper']\n",
      "['has great deal', 'in', 'recent years']\n",
      "['h', 'be', 'where unknown function']\n",
      "['its origin', 'in', 'mckendrick']\n",
      "['its origin', 'in', 'mckendrick']\n",
      "['its branching envelope', 'in', 'way']\n",
      "['use', 'in', 'clustering-type ratios']\n",
      "['case counts', 'of peak', 'citation citation']\n",
      "['recovered nodes', 'in', 'layers']\n",
      "['θ', 'be', 'where heaviside step function']\n",
      "['impact', 'in', 'vaccinated populations']\n",
      "['ψ', 'can vanish in', 'only isolated points']\n",
      "['ψ', 'therefore can vanish in', 'only isolated points']\n",
      "['forcing amplitude', 'correspond to', 'annual limit cycle']\n",
      "['sis', 'in', 'figs 9 12']\n",
      "['epidemics', 'achieve with', 'message passing techniques']\n",
      "['a 12', 'by ξi', 'x ∗']\n",
      "['density process', 'of lt', 'x']\n",
      "['dawson-watanabe process xt', 'with', 'killing rate θ']\n",
      "['simulations', 'between r', '∞']\n",
      "['phenomena', 'may', 'may observed']\n",
      "['systems', 'with', 'certain spectral requirements']\n",
      "['ﬁnal results', 'be', 'obtained numerically']\n",
      "['diseases', 'with', 'similar symptoms such respiratory syncytial virus']\n",
      "['pear', 'existence of', 'endemic state']\n",
      "['pathogen', 'reach more easily', 'endemic state']\n",
      "['r0 dependence', 'in', 'stochastic models']\n",
      "['state variables', 'plot with', 'respect control signals']\n",
      "['state variables', 'plot with', 'respect control signals']\n",
      "['epidemic outbreak', 'growth of', 't → x']\n",
      "['γ', 'part of', 'reproductive number']\n",
      "['delta function δ', 'in', 'classical markov process']\n",
      "['size', 'so sme', '∞']\n",
      "['group', 'on performance', 'e.g social networks']\n",
      "['second order terms', 'in', 'µ']\n",
      "['networks', 'with', 'power-law scaling p']\n",
      "['networks', 'with', 'disassortative mixing']\n",
      "['total number', 'in', 'z f']\n",
      "['total number', 'in', 'k 1']\n",
      "['total number', 'in', 'generations']\n",
      "['our results', 'in', 'case nr']\n",
      "['function', 'as i/n', 'm']\n",
      "['function', 'as plot', 'm']\n",
      "['sfns', 'with', 'degree exponents γ']\n",
      "['rrns', 'with', 'constant degree']\n",
      "['may variations', 'in', 'contact rates']\n",
      "['remaining edges', 'in', 'g ∗']\n",
      "['parallel edges', 'in', 'g ∗']\n",
      "['order', 'in', 'µ']\n",
      "['value', 'show by', 'right scale mark']\n",
      "['value', 'show by', 'left scale mark']\n",
      "['results', 'in', 'ﬁgure 10']\n",
      "['nonactive phase', 'with', 'only susceptible individuals']\n",
      "['50 conﬁgurations', 'of subset', 'figure 4']\n",
      "['n', 'in', 'figure']\n",
      "['n', 'in', 'figure 3']\n",
      "['normal prior distribution', 'with', 'mean log']\n",
      "['normal prior distribution', 'with', 'mean logit']\n",
      "['epidemic spreading', 'in', 'scale-free networks']\n",
      "['m ag', 'in', 'opinion dynamic']\n",
      "['k · kop', 'be', 'where operator norm']\n",
      "['k · kop', 'be', 'where operator norm']\n",
      "['continuous martingales', 'volume of', 'grund lehren der mathematischen wissenschaften']\n",
      "['its stability', 'use', 'routh-hurwitz theorem']\n",
      "['methods', 'capture', 'time scales']\n",
      "['incremental reduction', 'in', 'expected epidemic size per incremental increase']\n",
      "['presence', 'in', 'network']\n",
      "['probability', 'strength of', 'target country']\n",
      "['factor', 'in', 'probability']\n",
      "['k', 'be', 'positive integer']\n",
      "['linear preference function', 'be', 'applied here']\n",
      "['size n', 'dependence of', 'r ∞']\n",
      "['size n', 'dependence on', 'θ']\n",
      "['most popular location highest coreness', 'in', 'most popular location highest coreness']\n",
      "['physiological structure', 'in', 'immune population']\n",
      "['dynamics', 'exhibit in', 'their paper']\n",
      "['k large', 'for hµk', 'xl']\n",
      "['k large', 'for hµk', 'xl kt']\n",
      "['t', 'be', 'linear']\n",
      "['models', 'in', 'mathematical approaches emerging']\n",
      "['gt', 'be', 'simple']\n",
      "['matlab', 'have', 'initial value problem']\n",
      "['closed form parameter θ', 'depend on', 'only demographic rate µ']\n",
      "['infection', 'in', 'early stage']\n",
      "['distance', 'with', 'probability p']\n",
      "['enyi graph', 'with', 'n']\n",
      "['st', 's0 similarly for', 'other processes']\n",
      "['convenient quantities', 'in', 'numerical spreading experiments']\n",
      "['quantity', 'be', 'random']\n",
      "['must appropriate type', 'in', 'mean ﬁeld']\n",
      "['cascading failures', 'in', 'power grids']\n",
      "['small λ', 'with', 'γ']\n",
      "['supplementary ﬁgures', 'in', 'fig 9']\n",
      "['edges', 'with', 'only one endvertex']\n",
      "['initial evolution', 'be', 'more predictable']\n",
      "['evolution', 'be', 'more predictable']\n",
      "['µn → µ', 'with', 'µ']\n",
      "['two solutions', 'solve equation for', 'variable ωt above t ∗']\n",
      "['largest cluster', 'in', 'large school']\n",
      "['i-neighbor', 'with', 'awareness rate βf']\n",
      "['conclusion', 'draw in', 'last section']\n",
      "['black half-edges', 'pair', 'oﬀ']\n",
      "['second order transition', 'with', 'mean ﬁeld exponents']\n",
      "['stochastic optimum', 'in', 'roughly same regions']\n",
      "['challenge', 'in', 'our further work']\n",
      "['ﬁrst ingredient', 'in', 'calculation']\n",
      "['independent poisson initial distributions', 'with', 'means φs']\n",
      "['science', 'for foundation', 'fct']\n",
      "['k', 'be', 'when greater than 1']\n",
      "['β', 'number of', 'infected nodes']\n",
      "['fractional derivatives', 'in', 'corresponding nonlinear fractional diﬀerential evolutio 7 1 0 2']\n",
      "['inﬁnite type branching process', 'study separately in', 'section 4']\n",
      "['two layers', 'each of', 'size n']\n",
      "['statistics', 'use', 'established methods']\n",
      "['epidemic thresholds', 'in', 'real networks']\n",
      "['global characteristics', 'can', 'can estimated statistically']\n",
      "['models', 'in', 'section']\n",
      "['variety', 'in', 'epidemiology']\n",
      "['networks', 'with', 'strong structural heterogeneity numericcitation']\n",
      "['squares', 'in', 'also fig 3']\n",
      "['epidemics', 'with', 'limited resources']\n",
      "['paper', 'in center', 'pr china']\n",
      "['paper', 'in center', 'hefei 230026']\n",
      "['temporal shift transformation', 'without s9', 'solid black line']\n",
      "['physics', 'university of', 'fribourg']\n",
      "['optoelectronic engineering', '5department of', 'physics']\n",
      "['physics', 'ee', 'chemin du mus']\n",
      "['our aim', 'study', 'phase transitions']\n",
      "['value', 'may', 'may larger than one']\n",
      "['messages', 'pay', 'advertising']\n",
      "['information theory', 'transaction on', 'vol']\n",
      "['their higher degree nodes', 'ﬁrst as', 'as intentional attack']\n",
      "['based approach', 'in', 'information theory 2013']\n",
      "['isolation parameter', 'time of', 'isolation']\n",
      "['dawson-watanabe processes', 'with', 'variable-rate killing']\n",
      "['β1', 'correspond to', 'stable annual limit cycle']\n",
      "['nonlinear functions', 'in', 'x']\n",
      "['plk', 'acknowledge', 'support from nsf grant ccf-0829541']\n",
      "['reality', 'study', 'corresponding controlling measures']\n",
      "['m. ausloos5 6', '1department of', 'physics']\n",
      "['fractional calculus', 'in', 'physics']\n",
      "['time', 'at', 'z']\n",
      "['determinant', 'in', 'mycoplasma pneumoniae incidence']\n",
      "['β ∗', 'of computation', 'fraction']\n",
      "['isolated infected nodes', 'in', 'layer a']\n",
      "['integer coeﬃcients', 'with pr', 'e']\n",
      "['two vertices', 'in', 'ˆg']\n",
      "['b', 'in', 'cases']\n",
      "['hiss method', 'do', 'comparisons']\n",
      "['exact system diﬀers', 'in', 'only time shifts']\n",
      "['two regimes', 'separate by', 'dashed line']\n",
      "['epidemics', 'with', 'multiple stages']\n",
      "['threshold', 'in', 'original network']\n",
      "['household edges', 'contain', 'a uniformly at random chosen vertex from population']\n",
      "['distributions', 'converge to', 'steady-state distribution as t increases']\n",
      "['undirected network', 'with', 'self loops']\n",
      "['emergence', 'be', 'public']\n",
      "['organized populations', 'with', 'local structure']\n",
      "['role', 'in', 'case']\n",
      "['seasonal cholera epidemics', 'in', 'bangladesh']\n",
      "['agent', 'with', 'positive moderate opinion become']\n",
      "['individuals', 'in', 'population']\n",
      "['individuals', 'in', 'population']\n",
      "['individuals', 'in', 'population']\n",
      "['disease', 'completely dissapear from', 'environment']\n",
      "['disease', 'once spread to', 'entire population']\n",
      "['disease', 'once spread to', 'entire population']\n",
      "['independent variables', 'in', 'operator']\n",
      "['simulation results', 'show in', 'figure 4c']\n",
      "['simulation results', 'show in', 'figure 4d']\n",
      "['simple example', 'in', 'appendix a. several iterations']\n",
      "['discussion', 'in', 'noisy limit cycles']\n",
      "['animal production', 'in', 'developing countries']\n",
      "['one man', 'with', 'sars made']\n",
      "['r.m may', 'disease of', 'humans']\n",
      "['algebraic behavior', 'in', 'small-η limit']\n",
      "['exponential behavior', 'in', 'large-η limit']\n",
      "['several seasons', 'through series', 'bottom']\n",
      "['interplay', 'in', 'childhood diseases']\n",
      "['sars cases', 'of number', 'ontario']\n",
      "['sars cases', 'of number', 'canada']\n",
      "['sars cases', 'of number', 'hong kong']\n",
      "['sars cases', 'of number', 'top row']\n",
      "['his opinion', 'in', 'other layer']\n",
      "['results', 'prove throughout', 'paper']\n",
      "['people', 'in', 'society']\n",
      "['people', 'of', 'opinions']\n",
      "['0 ≤ γ 1', 'be', 'where nodes discount factor']\n",
      "['tradeoﬀs', 'be', 'where most critical']\n",
      "['increase', 'in', 'eq']\n",
      "['increase', 'in', 'eq']\n",
      "['dashed lines', 'scale', 'laws']\n",
      "['p0', 'datum as', 'hidden complete data at time t']\n",
      "['explanatory power', 'in', 'cluster-size vs. distance metric']\n",
      "['two power-law networks', 'with', 'exponential cutoﬀ c']\n",
      "['degree k', 'with', 'wk given by general form wk']\n",
      "['its increasing continuous inverse', 't for', 't']\n",
      "['infections', 'in', 'parent']\n",
      "['infections', 'in', 'parent']\n",
      "['comparison', 'with', 'earlier works']\n",
      "['selected statements', 'in', 'theorem 5']\n",
      "['three nodes', 'in', 'diﬀerent communities']\n",
      "['countries', 'with', 'much lower gdp such belgium']\n",
      "['β', 'be', 'when higher than pc']\n",
      "['gaussian noise terms', 'with', 'zero mean hlj i']\n",
      "['one particle', 'with', 'z']\n",
      "['computation', 'in', 'large time limit']\n",
      "['infectivity', 'be', 'when very high']\n",
      "['gronwall', 'have', 'inequality yields']\n",
      "['memory eﬀects', 'with', 'memoryless case']\n",
      "['there will multiple regularization parameters', 'one for', 'neighborhood']\n",
      "['latter', 'in', 'its most general sense']\n",
      "['economic crisis propagation', 'use', 'sir dynamics']\n",
      "['random contacts', 'in', 'ﬁnite well-mixed populations']\n",
      "['solutions', 'with', 'negative values']\n",
      "['cycle', 'in', 'when idle state']\n",
      "['arrival', 'occur in', 'when idle state']\n",
      "['approximation', 'in', 'eq']\n",
      "['b ′', 'in', 'region where']\n",
      "['larger system sizes', 'with', 'same node']\n",
      "['ﬁrst problem', 'in', 'network epidemiology known']\n",
      "['v. discussion', 'in', 'manuscript']\n",
      "['v. vacc inat ion', 'in', 'section']\n",
      "['r. m. may', 'disease of', 'humans']\n",
      "['numerical solution', 'reproduce', 'exactly analytical solution']\n",
      "['zombie', 'can', 'can permanently removed']\n",
      "['average degree', 'have', 'dynamics']\n",
      "['statistics', 'important for', 'several reasons']\n",
      "['increase', 'in', 'transmission']\n",
      "['new clustering type metrics', 'use', 'structures larger than triangles']\n",
      "['ﬁxed population n', 'with', 'only three compartments']\n",
      "['anhui university', 'spread of', 'infectious disease']\n",
      "['pathogenic h5n1 inﬂuenza virus infection', 'in', 'migratory birds']\n",
      "['scaling analysis', 'in', 'siri epidemiological model']\n",
      "['sample', 'between', 'nodes']\n",
      "['stationary point', 'be', 'always zero']\n",
      "['its particles', 'assign', 'poisson family px']\n",
      "['components', 'be', 'cycles']\n",
      "['1 ≤ i ≤ n', 'for p', 'ai']\n",
      "['vaccine doses', 'have', 'where have prepositioned geographically']\n",
      "['dimensionality', 'in', 'between 4']\n",
      "['transmission', 'in', '-20 0 20 40 60 80 100 0 20 40 60 80 100 120 sars data for hk revised data']\n",
      "['partition', 'in', 'population into individual components']\n",
      "['ebola outbreaks', 'have appear sporadically in', 'africa']\n",
      "['i0', 'of g', 'slopes']\n",
      "['individual role', 'in', 'collective dynamics']\n",
      "['solution', 'obtain in', 'exact parametric form']\n",
      "['spreading dynamics', 'with', 'synchronous updating']\n",
      "['spreading dynamics', 'with', 'synchronous updating']\n",
      "['spreading dynamics', 'with', 'synchronous updating']\n",
      "['integrated series', 'in', 'information systems']\n",
      "['continuous variations', 'in', 'ob jective functions']\n",
      "['immune individuals', 'of r', 't']\n",
      "['lasso', 'journal of', 'royal statistical society series b-methodological']\n",
      "['delayed outbreaks', 'observe in', 'rewired system']\n",
      "['connected network', 'with', 'low clustering']\n",
      "['same studies', 'rate of', 'new contacts identiﬁed']\n",
      "['scale', 'with', 'village size n']\n",
      "['wildlife', 'journal of', 'mathematical biology 69']\n",
      "['results', 'summarize in', 'table 1']\n",
      "['interaction', 'with', 'neighbor 6 j']\n",
      "['5 discussion presented', 'in', 'paper']\n",
      "['bifurcation regions', 'in', 'detail']\n",
      "['other countries', 'in', 'world']\n",
      "['siri epidemic model', 'with', 'nonlinear incidence']\n",
      "['rest 1516', 'be', 'one directional only']\n",
      "['rest 1516', 'be', 'one directional only']\n",
      "['inference', 'in', 'epidemics on temporal contact networks']\n",
      "['o', 'in', 'other words']\n",
      "['multilayer network m', 'with', 'm layers α']\n",
      "['l × l matrix links', 'in', 'multilayer network']\n",
      "['multilayer network', 'with', 'm']\n",
      "['their important rami ﬁcatio ns', 'in', 'many real-world scenarios']\n",
      "['a.r', 'acknowledge', 'ﬁnancial support']\n",
      "['d. f. m. torres', 'dynamics of', 'dengue epidemics']\n",
      "['main aspects', 'in', 'reality']\n",
      "['traditional theoretical study', 'base on', 'hmf']\n",
      "['expression', 'in', 'ensured by degree cutoﬀs k']\n",
      "['networks', 'with', 'markedly diﬀerent order-four motif distributions']\n",
      "['local epidemics', 'in', 'cliques']\n",
      "['population', 'with', 'random susceptibility']\n",
      "['improvement', 'even qualitative in', 'one dimension']\n",
      "['conjunction', 'with', 'our model']\n",
      "['computation', 'fraction of', 'vaccinated individuals']\n",
      "['other parameters', 'be', 'set']\n",
      "['n 1 2', 'in', 'contrast']\n",
      "['allocations', 'likely in', 'contrast']\n",
      "['log n pn', 'in', 'therefore dimension 2']\n",
      "['connections', 'with', 'node pk']\n",
      "['small-world networks', 'on model', 'egypt 2mathematics department']\n",
      "['solid lines', 'correspond to', 't']\n",
      "['other words', 'in kµ', 'rate']\n",
      "['example', 'for period', 'ﬂu']\n",
      "['ones', 'in', 't i']\n",
      "['population size', 'in', 'compartment']\n",
      "['population size', 'in', 'compartment']\n",
      "['nodes', 'in', 'large networks']\n",
      "['world', 'for number', 'ontario']\n",
      "['world', 'for number', 'top row']\n",
      "['world', 'for number', 'canada']\n",
      "['world', 'for number', 'hong kong']\n",
      "['l. massouli e', 'threshold for', 'virus spread on networks']\n",
      "['lives', 'minimize', 'health care costs']\n",
      "['decaying function', 'with', 'f']\n",
      "['support function', 'in', 'direction θ']\n",
      "['epidemics', 'duration of', 'when maximum']\n",
      "['susceptible agents', 'pass through', 'infectious agents']\n",
      "['result', 'as j', 'muller cp modelling measles re-emergence']\n",
      "['relation', 'with', 'fisher information matrix pawitan']\n",
      "['infection duration', 'of function', 'pdf']\n",
      "['probability', 'in', 'one given network x']\n",
      "['delays', 'in', 'biochemical systems']\n",
      "['many network models', 'use', 'synthetic networks']\n",
      "['t', 'edge in', 'network gt']\n",
      "['extensive monte carlo simulations', 'with', 'synchronous update']\n",
      "['clusters', 'show', 'eﬀect from nearest neighbour connections']\n",
      "['users', 'in', 'state i']\n",
      "['users', 'in', 'state r']\n",
      "['excellent agreement', 'with', 'our expectations']\n",
      "['other magnitudes', 'also show', 'standard behavior']\n",
      "['g', 'be', 'how chosen']\n",
      "['discussion', 'in', 'section 2.2']\n",
      "['whiskey', 'for', 'drinking']\n",
      "['i', 'even become', 'much larger than 1']\n",
      "['t', 'be', 'even when known']\n",
      "['observed nodes', 'with', 'same attributes']\n",
      "['populations', 'with', 'complete mixing']\n",
      "['populations', 'with', 'homogeneous mixing']\n",
      "['recovery individuals', 'in', 'steady state']\n",
      "['recovery individuals', 'in', 'steady state']\n",
      "['will intense debate', 'in', 'situations']\n",
      "['dramatic changes', 'in', 'long time behavior']\n",
      "['comparison', 'with', 'stochastic simulations']\n",
      "['t. tom', 'e', 'j. phys']\n",
      "['inverse method', 'in', 'paper']\n",
      "['other cholera models', 'in', 'literature']\n",
      "['ν', 'away from', 'far its critical values']\n",
      "['cdb', 'support by', 'john templeton foundation']\n",
      "['its logarithmic derivative', 'be', 'always bounded below']\n",
      "['its interaction', 'with', 'resonant frequencies']\n",
      "['parameter values', 'in', 'oscillatory phase']\n",
      "['ecological dynamics3', 'in', 'cities4 ,5']\n",
      "['density sampling', 'in', 'regimes']\n",
      "['his social page', 'in', 'recent past']\n",
      "['terms', 'list in', 'table ii']\n",
      "['excellent agreement', 'with', 'theoretical results']\n",
      "['m', 'lead follow', 'to 1 4 1 16 1 32 1 32 1 64 1 128 5 256 5 512 3 512 5 2048 7 512 7 1024 9 2048 5 2048 7 8192 21 2048 21 4096 7 2048 35 16384 35 32768 21 65536 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 one']\n",
      "['zmax zmin ψ maximal level', 'in', 'immune hosts']\n",
      "['solid curve', 'in', 'vl − β plane']\n",
      "['results', 'average over', '3 × 104 mcs']\n",
      "['world countries', 'of', 'average result']\n",
      "['methods', 'capture', 'long time scales']\n",
      "['enyi graph', 'with', 'n']\n",
      "['similar results', 'in', 'detection performance']\n",
      "['processes', 'in', 'px']\n",
      "['20 harrington', 'spread on', 'networks']\n",
      "['poisson', 'with', 'average degree two']\n",
      "['martingale', 'with', 'same quadratic variation as super-brownian motion']\n",
      "['nonlinear functions', 'in', 'x']\n",
      "['t6a σbt', 'in', 'other words']\n",
      "['data', 'start at_time', 'march 31']\n",
      "['relative heights', 'in', 'next subsection']\n",
      "['control', 'for center', 'shadi mohagheghi']\n",
      "['set-theoretic methods', 'in', 'control']\n",
      "['v', 'succeed with', 'probability quv']\n",
      "['ξ0', 'be', 'where determined']\n",
      "['tr', 'for 13', 'solid line']\n",
      "['er network', 'in approach', 'solid line']\n",
      "['connectivity', 'in', 'dense networks conﬁned within right prisms']\n",
      "['r1', 'greater if', 'c2 c3']\n",
      "['would o outbreak', 'with', 'ν']\n",
      "['fi ∈ r2n +1', 'represent', 'ith unit column vector']\n",
      "['k0', 'be', 'empty set']\n",
      "['θi', 'j to', 'continuous valued variable lying between its discrete values log']\n",
      "['all deal', 'with', 'unforced systems']\n",
      "['network', 'with', 'κ']\n",
      "['u. s. health oﬃcials', 'currently investigate', '199 cases']\n",
      "['step', 'in', 'its discrete formulation']\n",
      "['two-type cbp', 'in', 'ﬁnite populations']\n",
      "['three-type cbp', 'in', 'ﬁnite populations']\n",
      "['immunization', 'cause', 'its infection']\n",
      "['epidemic spreading', 'in', 'complex networks']\n",
      "['edge-based compartmental model', 'in', 'appendix']\n",
      "['terms', 'yield', 'eqs']\n",
      "['standard benchmark', 'class of', 'np-complete problems']\n",
      "['networks', 'with', 'eigenvector localized on hubs']\n",
      "['epidemic ﬁnal size', 'between relationship', 'n']\n",
      "['most popular location highest coreness', 'in', 'highest coreness location']\n",
      "['p', 'be', 'when small']\n",
      "['time t recruitment rate natural death rate disease-induced death rate transmission rate recovery rate g', 'at n', 't']\n",
      "['30 fall', 'in', 'second case']\n",
      "['different values', 'in', 'fig 3']\n",
      "['multiplicative noise eﬀects', 'generate', 'similar results']\n",
      "['realization', 'with', 'origin']\n",
      "['movement phase', 'first take', 'place']\n",
      "['function', 'in', 'ﬁgures 4']\n",
      "['h', 'be', 'where unknown function']\n",
      "['much eﬀort', 'have', 'has directed']\n",
      "['epidemic-free phase', 'with', 'only small outbreaks']\n",
      "['i', 'must at', 'must even least']\n",
      "['theorem 1.1', 'be', 'true also']\n",
      "['distribution function', 'satisﬁes', 'closed evolution equation']\n",
      "['state xtobs', 'in', 'vobs']\n",
      "['i', 'of', 'infected neighbors']\n",
      "['distributions', 'have', 'universal algebraic tails']\n",
      "['amc studios', 'circle of', 'confusion']\n",
      "['t0', 'be', 'where arbitrary integration constant']\n",
      "['one positive entry', 'in', 'row']\n",
      "['bolker', 'have', 'measles data archive numericcitation']\n",
      "['guinea', 'of', 'population']\n",
      "['numto', 'in', 'other words']\n",
      "['curves', 'guide', 'eye']\n",
      "['erd ¨ os number project', 'in', 'oakland university']\n",
      "['lcc', 'interestingly appear earlier in', 'ml networks']\n",
      "['β µ ≥ ξs', 'in', 'ml networks']\n",
      "['r ∞', 'in', 'ml networks']\n",
      "['infectives', 'boost to', 'other level']\n",
      "['signiﬁcant variation', 'with', 'p']\n",
      "['temporal second order phase transition', 'with', 'mean ﬁeld exponents']\n",
      "['vaccinated individuals', 'in', 'steady state']\n",
      "['jordan infection center', 'with', 'observed infection eccentricity 1']\n",
      "['regular one-dimensional chain', 'with', 'periodic boundary conditions']\n",
      "['network growth', 'in', 'integrated network']\n",
      "['regimes', 'follow by', 'initial growth']\n",
      "['real networks', 'with', 'assortative mixing']\n",
      "['set', 'in', 'v within distance 2tn']\n",
      "['set', 'in', 'v ′ within distance 2tn']\n",
      "['epidemic processes', 'in', 'complex networks']\n",
      "['our results', 'in', 'agreement']\n",
      "['scientiﬁc', 'report', '12760 2015']\n",
      "['classical sis process', 'in', 'complex networks']\n",
      "['3 big data research center', 'university of', 'electronic science']\n",
      "['∗ ming tang ,1 3', 'university of', 'technology']\n",
      "['objective function value', 'in', 'p0']\n",
      "['qc', 'reach', '1 slightly above δ']\n",
      "['sars', 'mode of', 'transmission']\n",
      "['sars', 'number', 'r0']\n",
      "['sars', 'in', 'hong kong']\n",
      "['nature', 'in', 'controversial region']\n",
      "['opinion', 'of', 'formation']\n",
      "['vaccinated agents', 'pin in', 'opinion +2']\n",
      "['underlying patterns', 'in', 'abundantly available social media data']\n",
      "['citation', 'in', 'gdp percentage rate']\n",
      "['single realization', 'for tst', 'τ']\n",
      "['single curve', 'onto ist', 'τ']\n",
      "['branching random walk', 'with', 'offspring distribution rn']\n",
      "['deﬁnition', 'by probability', 'πr']\n",
      "['exploration process', 'in r', 'n']\n",
      "['r 1 0', 'in', 'exploration process']\n",
      "['more viral campaigns', 'run than', 'less viral campaigns']\n",
      "['rényi random graphs', 'with', 'mean degree c']\n",
      "['sir spreading', 'in', 'ml']\n",
      "['most popular location random', 'in', 'highest strength location highest strength']\n",
      "['edges incident', 'of number', 'equivalently ki pj aij']\n",
      "['links', 'be', 'fast']\n",
      "['13 recovered individuals', 'be', 'permanently immune']\n",
      "['similar observations', 'hold during', 'decay']\n",
      "['t ≥ 0', 'with', 'state space']\n",
      "['reduced susceptibility', 'in', 'class s2']\n",
      "['discrete steps', 'in', 'probability']\n",
      "['simulation', 'be', 'once complete']\n",
      "['center', 'in', 'mathematics']\n",
      "['center', 'in', 'mathematics']\n",
      "['center', 'department of', 'mathematics']\n",
      "['mathematics', 'in center', '3810-193 aveiro']\n",
      "['mathematics', 'in center', 'd. f. m. torres']\n",
      "['3 3 4 2 0 ti 0 6 0 12 24 table 1', 'number of', 'local links li']\n",
      "['equation 24', 'can', 'finally can written']\n",
      "['online sex community', 'in', 'brazil']\n",
      "['corresponding simulation', 'result respectively in', 'one run']\n",
      "['expression', 'be', 'harmonic']\n",
      "['such active set updates', 'estimate', 'sparse partial correlations']\n",
      "['e. profumo', 'proceed of', 'national academy']\n",
      "['i', 'be', 'where n × n unit matrix']\n",
      "['removal rate', 'exceed', 'infection rate r']\n",
      "['calculation cost', 'be', 'proportional']\n",
      "['rate', 'be', 'periodic']\n",
      "['rate', 'be', 'sinusoidal']\n",
      "['measure-valu', 'branch', 'diffusion processes']\n",
      "['facebook', 'rewire respectively', 'enron social networks']\n",
      "['facebook', 'rewire', 'enron social networks']\n",
      "['developed theory', 'can describe very well', 'phenomena']\n",
      "['biotechnology', 'of institute', 'university']\n",
      "['helpful approach', 'in', 'hypothetical outbreak']\n",
      "['theorem 2.9', 'of part', 'i']\n",
      "['expression', 'in', 'theorem 2.9']\n",
      "['1 gk ni', 'k in', 'theorem 5.3']\n",
      "['k', 'in', 'theorem 5.3']\n",
      "['theorem 5.3', 'consequently apply to', '¯ g ∗']\n",
      "['1 kni', 'w.h.p by', 'theorem 2.9']\n",
      "['good agreement', 'with', 'numerics']\n",
      "['terms', 'in +1', 'x']\n",
      "['diﬀerent factors', 'of inﬂuence', 'network structure']\n",
      "['mean number', 'in', 'z b']\n",
      "['w 0.8', 'in', 'scenario presented']\n",
      "['jordan infection center', 'in', 'inﬁnite trees']\n",
      "['b ′', 'in', 'case']\n",
      "['well-known article', 'in', 'nature magazine4']\n",
      "['little empirical evidence', 'support', 'their assumed functions']\n",
      "['little empirical evidence', 'currently support', 'their assumed functions']\n",
      "['component ai', 'in edge', 'u']\n",
      "['could principle deal', 'with', 'networked markov processes']\n",
      "['i.i.d poisson random variables', 'with', 'mean λg']\n",
      "['50 000 simulations', 'in', 'ﬁgure 2']\n",
      "['single server', 'in', 'queue system']\n",
      "['overview', 'with', 'different recovery rates']\n",
      "['ignoring age-of-infection eﬀects', 'lose', 'details']\n",
      "['vicinity', 'of character', 'relaxation time scale']\n",
      "['excellent agreement', 'with', 'values obtained from stochastic simulations']\n",
      "['link percolation', 'in', 'two layers']\n",
      "['analogous representation', 'in', 'one-dimensional case']\n",
      "['lawler', 'in', 'case']\n",
      "['comparison', 'with', 'simulated thresholds on rrns']\n",
      "['comparison', 'with', 'simulated thresholds on sfns']\n",
      "['resulting models', 'correspond to', 'known pathogens']\n",
      "['network', 'have', 'community structure']\n",
      "['network', 'have', 'community structure']\n",
      "['its dynamical behavior', 'in', 'lemma below']\n",
      "['three types', 'in', 'branching process']\n",
      "['heavy dashed line', 'line of', 'limit points']\n",
      "['two power law networks', 'with', 'exponential cutoﬀ c']\n",
      "['k shell index', 'reﬂect', 'nodes susceptibility than degree k']\n",
      "['k shell index', 'better reﬂect', 'nodes susceptibility than degree k']\n",
      "['number', 'cost of', 'vaccination']\n",
      "['our simplifying assumptions', 'in', 'section 3.1']\n",
      "['intimate contacts', 'with', 'sars patient']\n",
      "['rare-event simulation method', 'validate', 'theory']\n",
      "['wang', 'university of', 'electronic science']\n",
      "['wang', 'university of', 'electronic science']\n",
      "['our work', 'model', 'sars data']\n",
      "['ψk', 'in', 'local uniform topology']\n",
      "['its randomized version', 'in time', 'l']\n",
      "['light-gray common ﬂu activity', 'in', 'berlin']\n",
      "['advances', 'in', 'global change research']\n",
      "['leading asymptotic behavior', 'in', 'small-η limit']\n",
      "['series a', 'paper of', 'mathematical character']\n",
      "['time step', 'be', 'discrete']\n",
      "['final outbreak size r', 'µ down', 'triangles']\n",
      "['ﬁrst term', 'in', 'q3 x']\n",
      "['monotonical ly', 'in', 'solution x']\n",
      "['maximal possible number', 'in', '9 epidemic']\n",
      "['instances', 'with', 'observed infected']\n",
      "['curve', 'use', '6 point moving average']\n",
      "['w. luo', 'with', 'currently micron semiconductor asia']\n",
      "['µ', 'in', 'such networks']\n",
      "['much eﬀort', 'use', 'mathematical models']\n",
      "['susceptibility', 'in', 'susceptible-infectious-recovered spreading dynamics']\n",
      "['contact', 'with', 'susceptible category']\n",
      "['largest component', 'in', 'resulting graph']\n",
      "['two piecewise linear parts', 'in', 'log-linear graph']\n",
      "['our analytical approach', 'predict through', 'eq']\n",
      "['small-world networks', 'on model', 'a. s. hegazi1']\n",
      "['small-world networks', 'on model', 'al-ain p. o. box 17551']\n",
      "['small-world networks', 'on model', 'mansoura 35516']\n",
      "['conclusions', 'in', 'present work']\n",
      "['pages 93 103', 'in', 'web technologies']\n",
      "['2 introduction', 'in', '1796 edward jenner']\n",
      "['daily variance', 'in', 'predicted ili']\n",
      "['w. luo', 'with', 'micron semiconductor asia']\n",
      "['network', 'characterize by', 'well connected core surrounded by many low degree nodes']\n",
      "['infection rate', '1 in', 'other words']\n",
      "['disease', 'in', 'endemic state']\n",
      "['βnxi', 'in', 'time-changed process']\n",
      "['isolated individuals', 'in', 'layer a']\n",
      "['information source', 'be', 'randomly selected']\n",
      "['b o i l i t y p 1', 'number of', 'days']\n",
      "['conjunction', 'with', 'global sars data']\n",
      "['a a1 s a2', 'into supernode', 's']\n",
      "['n 1', 'be', 'where ﬁxed total population size']\n",
      "['log n 1/3', 'in', 'm']\n",
      "['diﬀerent time intervals', 'in', 'data observation period']\n",
      "['square', 'with', 'diagonal']\n",
      "['standard dawson-watanabe process xt', 'with', 'variance parameter σ2']\n",
      "['second term', 'in', 'q1 x']\n",
      "['modularity', 'slow down', 'spreading process']\n",
      "['facebook', 'of', 'popularity']\n",
      "['contrast', 'with', 'mean ﬁeld approximation']\n",
      "['central paradigm', 'in', 'ecology']\n",
      "['complex conjugate', 'in', 'table']\n",
      "['mathematical tools', 'model', 'epidemics']\n",
      "['tools', 'model', 'epidemics']\n",
      "['epidemics', 'with', 'tr']\n",
      "['common problem', 'in', 'chemical kinetics literature']\n",
      "['degree k', 'number of', 'connections']\n",
      "['recovery', 'with', 'rate q']\n",
      "['j. j. trujillo', 'theory of', 'fractional diﬀerential equations']\n",
      "['stability analysis', 'show', 'eqs']\n",
      "['disease transmission', 'in', 'contact networks']\n",
      "['larger jumps', 'emerge at', 'lower infection rates']\n",
      "['global separation', 'be', 'imposed faster than common sense']\n",
      "['nevertheless bp', 'better than', 'other methods']\n",
      "['extinction time', 'see in', 'fig 3']\n",
      "['graph', 'with', 'degree sequence']\n",
      "['multi-type galton', 'with', 'types']\n",
      "['s. franz', 'journal of', 'statistical mechanics']\n",
      "['immune hosts', 'of g', 'zmin']\n",
      "['infection rate', '1 in', 'words']\n",
      "['two er layers', 'with', 'hka i ≃ hkb i ≃ 2.31']\n",
      "['two er networks', 'with', 'hka i ≃ hkb i ≃ 2.31']\n",
      "['two er networks', 'with', 'kmin']\n",
      "['two er networks', 'with', 'hka i ≃ hkb i ≃ 2.31']\n",
      "['two er networks', 'with', 'kmin']\n",
      "['two er networks', 'with', 'hka i ≃ hkb i ≃ 2.31']\n",
      "['two er networks', 'with', 'kmin']\n",
      "['two er networks', 'with', 'hka i ≃ hkb i ≃ 2.31']\n",
      "['weak degree heterogeneity', 'with', 'γ']\n",
      "['simulation', 'in', 'fig 1']\n",
      "['helpful discussions', 'with', 'm. j. de oliveira']\n",
      "['processes', 'in', 'nature']\n",
      "['extreme switching behavior', 'in', 'deterministic case']\n",
      "['proposed method', 'with', 'global speciﬁc penalties']\n",
      "['quantities', 'in', 'table 1']\n",
      "['similar phase diagram', 'with', 'hopf bifurcation critical line bounding']\n",
      "['τ', 'with', 'modiﬁed transition rates']\n",
      "['individuals', 'in', 'class']\n",
      "['individuals', 'in', 'class']\n",
      "['system comparable', 'in', 'size over time']\n",
      "['figure 4', 'illustrate', 'phenomenon']\n",
      "['our results', 'characterize', 'mechanisms']\n",
      "['graph', 'with', 'vertex set v']\n",
      "['decision problems', 'in', 'set np']\n",
      "['second line', 'in', 'eq']\n",
      "['second line', 'in', 'eq']\n",
      "['extremal point', 'in', 'interval']\n",
      "['poisson distribution', 'with', 'mean 1']\n",
      "['offspring distribution', 'poisson with', 'mean 1']\n",
      "['poisson distribution', 'with', 'mean 1']\n",
      "['measles', 'in', 'humans']\n",
      "['strong ties', 'with', 'their peers']\n",
      "['homogeneous networks', 'with', 'parameters']\n",
      "['in-degree time dependence', 'easily approximate with', 'probabilistic consideration']\n",
      "['βc', 'in', 'our simulations']\n",
      "['deﬁnition', 'in', 'our research']\n",
      "['approach', 'in', 'ﬁeld']\n",
      "['inﬁnitesimal rates', 'with process', 'citation']\n",
      "['inhomogeneous markov process', 'with', 'inﬁnitesimal rates']\n",
      "['inﬁnitesimal rates', 'poisson of', 'infections']\n",
      "['systems', 'with', 'constant delay']\n",
      "['origin', 'take', 'given direction θ']\n",
      "['our particles', 'move in', 'only bounded region']\n",
      "['environmental data', 'be', 'also collected approximately two weeks from six water bodies']\n",
      "['pathogens', 'typically infect', 'species']\n",
      "['last option', 'have', 'more reliable community structure']\n",
      "['fraction', 'in', 'control scenario']\n",
      "['relative weight', 'in', 'peak describing']\n",
      "['q', 'be', 'ﬁxed']\n",
      "['non-sexually diseases', 'in', 'well-mixed populations']\n",
      "['size distribution', 'be', 'thus purely exponential']\n",
      "['isolation', 'take place in', 'one network']\n",
      "['coupled ﬂuctuations', 'in', 'stacked contact processes']\n",
      "['iacoviello', 'journal of', 'simulation modelling 7']\n",
      "['smaller countries', 'in', 'absolute size']\n",
      "['diﬀerent contacts', 'in', 'layer']\n",
      "['nodes', 'in', 'diﬀerent layers']\n",
      "['similar phenomena', 'common in', 'our daily lives']\n",
      "['future work', 'envision in', 'ref']\n",
      "['i1 i2 endemic plane', 'on line', 'cl']\n",
      "['modeling infectious diseases', 'in', 'humans']\n",
      "['obvious question', 'in', 'such crisis 2 sars']\n",
      "['ones', 'in', 'corollary 1.1']\n",
      "['networks', 'with', 'eigenvector localized on high k-core nodes']\n",
      "['networks', 'with', 'eigenvector localized on high k-core nodes']\n",
      "['networks', 'with', 'eigenvector localized on high k-core nodes']\n",
      "['ignoring age-of-infection eﬀects', 'lose', 'important details']\n",
      "['s ir process', 'with', 'source θ']\n",
      "['same value b', 'with', 'b ∈']\n",
      "['two focal points', 'with', 'one infection']\n",
      "['behaviour', 'identify in', 'stochastic simulations']\n",
      "['cases', 'with', 'µ']\n",
      "['cases', 'with', 'synchronous updating']\n",
      "['solid black dots', 'represent', 'outbreak origins']\n",
      "['sum', 'in n', 'x']\n",
      "['˜xt', 'be', 'linear']\n",
      "['˜st', 'be', 'linear']\n",
      "['scaling trick', 'in', 'section iii']\n",
      "['sf networks', 'with', 'k']\n",
      "['4 only nonzero eigenvalue', 'with', 'non-negative eigenfunction']\n",
      "['additional probabilities', 'in', 'table ii']\n",
      "['competing effects', 'handle analytically with', 'even approximate approach']\n",
      "['other eigenvalue', 'than maxx', 't']\n",
      "['may rm', 'dynamics on', 'scale-free networks']\n",
      "['corresponding transition rates', 'be', 'diﬀerent']\n",
      "['dm p method', 'determine', 'complete network structure']\n",
      "['shortcuts', 'be', 'ﬁxed beforehand']\n",
      "['he exits', 'in', 'other words']\n",
      "['presence', 'in', 'gain curves']\n",
      "['presence', 'in', 'gain curves']\n",
      "['i.i.d poisson numbers', 'with', 'mean λg']\n",
      "['two three weeks', 'in', 'advance']\n",
      "['sudden jump', 'in', 'infection']\n",
      "['set', 'with network', 'example']\n",
      "['explicit time dependence', 'in', 'transition rates']\n",
      "['messages', 'use', 'mass communication']\n",
      "['saddle-node points', 'of line', 'limit points']\n",
      "['infected state', 'to tst', 'τ']\n",
      "['pmmh algorithm', 'use', 'data']\n",
      "['half', 'in', 'high movement case']\n",
      "['infectious disease epidemics', 'use', 'networks']\n",
      "['infectious disease epidemics', 'use', 'temporal networks']\n",
      "['coupled disease-behavior interaction models', 'in', 'multiplex networks']\n",
      "['z', 'with', 'village size n']\n",
      "['law', 'with', 'village size n']\n",
      "['b countries', 'aﬀ', 'citation']\n",
      "['22 6.2 new class', 'with', 'constant delay']\n",
      "['node', 'with', 'k connections']\n",
      "['recent epidemics', 'in', 'south korea']\n",
      "['region', 'with', 'limit cycle solutions']\n",
      "['authors', 'support by', 'vr']\n",
      "['lemma 3', 'in', 'section vi']\n",
      "['vaccinated agents', 'in', 'opinion state']\n",
      "['case', 'lead to', 'dependencies between regions']\n",
      "['simulations results', 'with', 'tr']\n",
      "['transmission', 'in', 'population']\n",
      "['its creation', 'in', 'early nineties']\n",
      "['ϕ', 'transition to', 'state ii +1']\n",
      "['cases', 'in w', 'ij']\n",
      "['function', 'as lcc', 'lcc/n']\n",
      "['value', 'in', 'product']\n",
      "['example', 'in', 'fig 3']\n",
      "['˜αn αn → ∞', 'in', 'case ν']\n",
      "['case', 'of proof', 'i']\n",
      "['times', 'in', 'z τ']\n",
      "['branching random walk', 'with', 'location-dependent killing']\n",
      "['red dashed line', 'in', 'fig 1']\n",
      "['susceptible individual η', 'for n', 'x']\n",
      "['high-resolution measurements', 'in', 'primary school']\n",
      "['introduction', 'transmission of', 'infectious diseases']\n",
      "['ﬁrst term', 'in', 'series']\n",
      "['network science', 'for ,1', 'institute']\n",
      "['use', 'in', 'osn models']\n",
      "['transition', 'will eventually disappear for', 'large heterogeneous networks']\n",
      "['absence', 'in', 'gain curve']\n",
      "['ﬁrst two terms', 'in', 'eq']\n",
      "['spread', 'in', 'west africa']\n",
      "['universality', 'in', 'power-law characteristics']\n",
      "['direct sampling', 'monotonically decrease', 'auc values']\n",
      "['pofc-qren', 'with', 'compete reference fcomp-01-0124-feder-028894']\n",
      "['optimization', 'be', 'most important']\n",
      "['fundamental sciences', 'in', 'basic sciences']\n",
      "['biological sciences', 'institute for', 'research']\n",
      "['distribution', 'in', 'simulations']\n",
      "['equation', 'in', 'appendix b 2']\n",
      "['change', 'in', 's']\n",
      "['change', 'in', 'n/s']\n",
      "['same ﬁxed p1 ,2 rest curves', 'in', 'figs 5']\n",
      "['exponents', 'obtain from', 'monte carlo simulations']\n",
      "['critical exponents', 'obtain from', 'monte carlo simulations']\n",
      "['changes', 'in', 'susceptible fraction']\n",
      "['new class', 'with', 'constant delay']\n",
      "['mathematical sciences', 'of department', 'university']\n",
      "['mathematical sciences', 'of department', 'university']\n",
      "['particles', 'have', 'their home locations']\n",
      "['multi-edge', 'will', 'will created']\n",
      "['direction', 'in', 'real lives']\n",
      "['white gaussian noise functions', 'with', 'zero mean obeying']\n",
      "['markov chain', 'with', 'transition rates']\n",
      "['small birth term', 'with', 'saturation']\n",
      "['unexpected tail', 'comprise of', 'few very large outbreaks']\n",
      "['r', 'fraction of', 'vaccinated individuals']\n",
      "['r', 'fraction of', 'vaccinated individuals']\n",
      "['r', 'fraction of', 'vaccinated individuals']\n",
      "['lower panels', 'show', 'corresponding power spectra']\n",
      "['other terms', 'in', 'eqs']\n",
      "['large numbers result', 'in', 'spirit']\n",
      "['summary', 'in', 'paper']\n",
      "['contact', 'with', 'infectives']\n",
      "['contact', 'with', 'infectives']\n",
      "['contact', 'with', 'infectives']\n",
      "['contact', 'with', 'infectives']\n",
      "['initial-times linear evolution satisﬁes', 'in', 'rn']\n",
      "['blythe parasites', 'spread as', 'contact process restricted sites currently occupied hosts']\n",
      "['i ∗', 'have', 'local susceptibility set']\n",
      "['two countries', 'in', 'network']\n",
      "['disease dynamics', 'evolve without', 'late-time constrains']\n",
      "['addition', 'in', 'table 3']\n",
      "['82 countries', 'in', 'million us dollars']\n",
      "['ﬁrst one', 'generate function of', 'k']\n",
      "['asymptotic analysis', 'scale', 'ex-fig']\n",
      "['kansas state university', 'of board', 'irb']\n",
      "['only jumps', 'in', 's ′ τ']\n",
      "['constraint symmetry θi', 'of form', 'j ∈ ηi known non-interactions min possible size']\n",
      "['local delay θ', 'with', 'same wait time separating']\n",
      "['research', 'for center', 'd. f. m. torres']\n",
      "['research', 'for center', '3810-193 aveiro']\n",
      "['y', 'represent', 'ln p ≥ s']\n",
      "['level', 'by immunity', 'w']\n",
      "['t', 'with', 'immunity level']\n",
      "['β', 'be', 'immunization γ δ fig 1']\n",
      "['corresponding deterministic curve', 'in', 'case']\n",
      "['k', 'be', 'where ﬁgure']\n",
      "['results', 'in', 'sec 2']\n",
      "['epidemic threshold', 'above c', 'individuals']\n",
      "['increasing trend', 'in', 'ﬁrst stage']\n",
      "['reports', 'with', 'monitoring agencies']\n",
      "['unique stable equilibrium point', 'at s', 'blue']\n",
      "['sum', 'j of', 'j ∈ neighbors']\n",
      "['lecture notes', 'in', 'math']\n",
      "['edges', 'represent by', 'curved counter clock-wise']\n",
      "['units', 'have', 'three types']\n",
      "['four days', 'in', 'june']\n",
      "['bifurcation point', 'in', 'simulations']\n",
      "['human sub', 'ject', 'kansas']\n",
      "['j', 'have', 'susceptibility']\n",
      "['j', 'have', 'susceptibility']\n",
      "['several other well-known radial measures', 'citation include', 'closeness']\n",
      "['poisson distribution', 'with', 'average degree c']\n",
      "['multilayer network framework', 'characterize', 'sis dynamics']\n",
      "['latent individuals', 'travel out of', 'their own community']\n",
      "['individuals', 'travel out of', 'their own community']\n",
      "['h', 'with', 'largest degree centralities']\n",
      "['other hand', 'in', 'ref']\n",
      "['0 kni', 'fraction of', 'more general ly']\n",
      "['νd', 'be', 'where degree exponent']\n",
      "['dashed line', 'represent', 'exact solution']\n",
      "['dashed line', 'be', 'exact solution']\n",
      "['unidirectional edges', 'represent by', 'curved counter clock-wise']\n",
      "['lr', 'selection in', 'paper']\n",
      "['sir', 'with', 'neighborhood speciﬁc 1 respectively']\n",
      "['k', 'be', 'where degree cutoﬀ']\n",
      "['several recent analytical studies', 'deal with', 'unforced systems']\n",
      "['vertices', 'in', 'same group']\n",
      "['inﬂuence', 'in', 'england 1500-1667']\n",
      "['compromise', 'in', 'same proportion']\n",
      "['2.2.2.2 global stability', 'point of', 'system']\n",
      "['origin', 'point of', 'system']\n",
      "['non-autonomous seirs model', 'with', 'quarantine']\n",
      "['infections', 'bias towards', 'young infection ages']\n",
      "['only links', 'with', 'weights']\n",
      "['propagation', 'in', 'animals']\n",
      "['excellent agreement', 'with', 'simulations']\n",
      "['shell', 'in', 'fact']\n",
      "['data description', 'in', 'materials']\n",
      "['suicide', 'be', 'where commonplace']\n",
      "['mutualism', 'promote', 'pathogen evolution']\n",
      "['equations', 'system of', 'computationally only mildly more diﬃcult than standard sir equations']\n",
      "['deterministic epidemics', 'in', 'closed populations']\n",
      "['bt', 'be', 'standard']\n",
      "['more ﬂat', 'with', 'increasing ν']\n",
      "['cumulative number', 'all of', 'canada']\n",
      "['cumulative number', 'in', 'ontario']\n",
      "['cumulative number', 'in', 'hong kong']\n",
      "['their clients', 'invite', 'others']\n",
      "['isolation', 'in', 'only one layer']\n",
      "['isolation', 'occur in', 'only one layer']\n",
      "['n', 'preserve for', 'virus type independently']\n",
      "['∗ ming tang ,1 3', 'university of', 'electronic science']\n",
      "['site', 'be', 'randomly chosen']\n",
      "['diﬀerent frameworks', 'study', 'epidemic spreading']\n",
      "['phase transitions', 'in', 'sirs versions']\n",
      "['2004 sexual networks', 'in', 'contemporary western societies']\n",
      "['squares', 'k for', 'sis']\n",
      "['sis', 'for λc', 'squares']\n",
      "['time evolution', 'with', 'equal birth']\n",
      "['inﬂuential spreaders', 'in', 'rumor dynamics']\n",
      "['nodes', 'in', 'other groups']\n",
      "['spread', 'in', 'population']\n",
      "['contagion process', 'in', 'modular activity']\n",
      "['ebola outbreak', 'kill', '5000 gorillas']\n",
      "['389 cases', 'in', '26 countries']\n",
      "['rényi random graphs', 'in', 'limit']\n",
      "['brieﬂy', 'in', 'section 6']\n",
      "['theorems', 'prove in', 'section 5']\n",
      "['fl', 'grateful to', 'proyecto de postdoctorado n. 3160163']\n",
      "['infective vertices', 'recover at', 'rate ρn 0']\n",
      "['true variation', 'in', 'control strategy']\n",
      "['our approach', 'lead to', 'fairly simple proofs']\n",
      "['above formulas probabilities pm', 'n for', 'm']\n",
      "['ξ0', 'be', 'where fully determined']\n",
      "['eqs', 'of s1', 'blue solid line']\n",
      "['px', 'family of', 'independent poisson processes']\n",
      "['largest variance', 'in', 'local clustering']\n",
      "['line', 'with', 'previous results']\n",
      "['methods', 'directly applicable to', 'real-world disease dynamics']\n",
      "['al l households', 'have', 'size 2']\n",
      "['solution φ', 'know as', 'malthusian parameter']\n",
      "['brief discussion', 'with', 'births']\n",
      "['electrical engineering', 'department of', 'electrical engineering']\n",
      "['conclusion', 'in', 'paper']\n",
      "['formidable task', 'in', 'even absence branching']\n",
      "['change', 'in', 'attack ratio']\n",
      "['color codes', 'in', 'fig 1']\n",
      "['color meanings', 'be', 'speciﬁed']\n",
      "['color meanings', 'speciﬁed in', 'fig']\n",
      "['dawson-watanabe measure-valued diffusion', 'in', 'd']\n",
      "['networks', 'with', 'tunable clustering']\n",
      "['networks', 'with', 'low clustering']\n",
      "['networks', 'with', 'φ']\n",
      "['our rates', 'use', 'τ']\n",
      "['close match', 'with', 'analytic results']\n",
      "['i-individuals', 'recover by', 'rate q3']\n",
      "['j', 'have', 'susceptibility set']\n",
      "['key step', 'in', 'proof']\n",
      "['contact', 'with', 'dead body']\n",
      "['brain proﬁling', 'in', 'same way']\n",
      "['solution', 'in', 'problems']\n",
      "['solution', 'thus unique for', 't']\n",
      "['disease', 'of', 'parameters']\n",
      "['disease', 'of', 'parameters']\n",
      "['less terms', 'in', 'product']\n",
      "['mathematical sciences', 'of school', 'uk']\n",
      "['reference parameter', 'in', 'mathematical epidemiology used']\n",
      "['12 countries', 'play in', 'world economy']\n",
      "['today', 'of', 'computer systems']\n",
      "['today', 'with', 'computer systems']\n",
      "['nonnegative function r', 'with', 'r integrable']\n",
      "['excellent agreement', 'in', 'cases']\n",
      "['9 terms', 'include', 't']\n",
      "['non-linear terms', 'in', 'eq']\n",
      "['system', 'with', 'o']\n",
      "['transition', 'disappear for', 'only large very heterogeneous networks']\n",
      "['pa', 'be', 'such schemes']\n",
      "['b. yu', 'selection in', 'gaussian graphical models']\n",
      "['theoretical predictions', 'in', 'section iii']\n",
      "['changes', 'in', 'settings']\n",
      "['new local', 'be', 'signal']\n",
      "['gillespie', 'of', 'algorithm']\n",
      "['gillespie', 'with', 'algorithm']\n",
      "['b. t. grenfell', 'proceed of', 'national academy']\n",
      "['b. t. grenfell', 'proceed of', 'national academy']\n",
      "['two scale free networks', 'with', 'kmin']\n",
      "['scenario', 'in', 'one layer']\n",
      "['isolation', 'in', 'one layer']\n",
      "['behavior', 'in', 'subsection']\n",
      "['behavior', 'in', 'statement']\n",
      "['nodes', 'can in', 'can three possible states']\n",
      "['numericcitation', 'in', 'continuous-time']\n",
      "['numericcitation n', 'j at', 'origin']\n",
      "['sir', 'with', 'single penalty']\n",
      "['controlled problem', 'of x', 't']\n",
      "['network', 'with', 'large size n']\n",
      "['networks', 'with', 'ﬁnite size']\n",
      "['c network size', 'in', 'figs 1']\n",
      "['outbreak size', 'with', 'ρr ≤ rc']\n",
      "['complex networks', 'with', 'ﬁnite size']\n",
      "['druggers', 'such as numericcitation', 'high-risk population']\n",
      "['rise', 'in', 'japan']\n",
      "['value', 'may', 'may larger']\n",
      "['infections diseases', 'in', 'humans']\n",
      "['agents', 'change', 'their opinion']\n",
      "['list', 'in', 'ﬁrst step']\n",
      "['many diseases', 'in', 'virus transmission']\n",
      "['∼ n 2/2', 'in', 'limit i0 s0']\n",
      "['major centrality measures', 'in', 'literature']\n",
      "['random regular graphs', 'with', 'uniform epidemic parameters']\n",
      "['fraction', 'in', 'addition']\n",
      "['links', 'be', 'very fast']\n",
      "['uncorrelated networks', 'for p', 'k']\n",
      "['contact', 'with', 'i1']\n",
      "['ma jor outbreak', 'in', 'supercritical case']\n",
      "['state ϕ1', 'follow by', 'particular successor state ϕ2']\n",
      "['ni', 'approach', 'zero']\n",
      "['second order phase transition', 'with', 'critical threshold transmissibility tc']\n",
      "['void network', 'undergo', 'second order phase transition']\n",
      "['markov', 'have', 'inequality']\n",
      "['3 3 4 2 0 ti 0 6 0 12 24 table 1', 'number of', 'global stubs σi']\n",
      "['proofs', 'with', 'slight modiﬁcations']\n",
      "['al l t', 'for 0n', 'then x 0n']\n",
      "['epidemic stops', 'with', 'zm']\n",
      "['terms', 'in', 'pgf']\n",
      "['ili', 'include', 'diseases']\n",
      "['g', 'be', 'strongly connected']\n",
      "['next nearest neighbours', 'be', 'when considered']\n",
      "['functions', 'as time', 'τ ∗']\n",
      "['functions', 'as time', 'τ ∗']\n",
      "['functions', 'as s', 't']\n",
      "['location-based mitigation strategies random', 'in', 'most popular location highest coreness']\n",
      "['changes', 'with', 'slowly n']\n",
      "['adv', 'in', 'appl']\n",
      "['typical individual', 'in', 'z']\n",
      "['md networks', 'particularly dependent on', 'isolated motifs']\n",
      "['jordan infection center', 'with', 'partial observation']\n",
      "['i.i.d', 'with', 'mean 0']\n",
      "['research', 'on symposium', '343 1991']\n",
      "['speciﬁc value', 'in', 'allocation']\n",
      "['λ', 'be', 'suﬃciently small']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['result', 'in', 'fig 2a']\n",
      "['normal times', 'protect by', 'vaccination']\n",
      "['main observations', 'be', 'overall four']\n",
      "['multiple pathogens', 'with', 'similar symptoms']\n",
      "['multiple pathogens', 'with', 'similar symptoms']\n",
      "['independent gaussian white noises ηx', 'with', 'zero mean']\n",
      "['infective half-edge', 'be', 'paired oﬀ']\n",
      "['disease transmission', 'in', 'populations']\n",
      "['initial cases', 'in', 'schools']\n",
      "['many eﬀorts', 'study', 'ﬁeld1 ,2']\n",
      "['s', 'signature of', 'antibody responses derived from systems biology study']\n",
      "['meter', 'square', 'area']\n",
      "['future cases', 'be', 'then predicted until next cut oﬀ']\n",
      "['j. f. f. critical phenomena', 'in', 'complex networks']\n",
      "['set', 'in', 'e n']\n",
      "['ﬁrst repeated index', 'in', 'sequence']\n",
      "['readers', 'in', 'ﬁeld']\n",
      "['construction', 'in', 'original method']\n",
      "['our ¯ zm', 'be', 'right continuous']\n",
      "['h1n1 outbreak', 'in', 'la gloria']\n",
      "['slow strain', 'reach', 'ﬁxation']\n",
      "['r', 'approach to', 'zero']\n",
      "['x t', 's follow', 'y ∈ tz']\n",
      "['y ∈ tz', 'follow s', 'τ − 1']\n",
      "['simulation', 'be', 'random']\n",
      "['nontrivial ﬁxed point', 'also stable in', 'most cases']\n",
      "['number', 'in', 'parentheses']\n",
      "['periodic sinusoidal function', 'with', 'period']\n",
      "['growth', 'in', 'mfa']\n",
      "['extensive numerical experiments', 'validate', 'proposed theoretical predictions']\n",
      "['r', 'with', 'recovery rate γ']\n",
      "['idea', 'choose number from', 'distribution']\n",
      "['asano', 'control of', 'vaccine distribution']\n",
      "['night', 'for simulation', 'left']\n",
      "['shaun', 'for simulation', 'left']\n",
      "['wormald', 'have', 'diﬀerential equations method']\n",
      "['simulations', 'with', 'full infection data']\n",
      "['network', 'in node', 'higher its ks']\n",
      "['analysis', 'probability of', 's']\n",
      "['observations', 'with', 'few infected']\n",
      "['data points', 'with', 'best ﬁt lines']\n",
      "['quintuple', 'arrange into', 'one complete square corner']\n",
      "['gdp values', 'in', 'million us dollars']\n",
      "['immunity levels', 'in', 'interval']\n",
      "['individuals', 'in', 'population class']\n",
      "['phenomenon', 'call', 'infor mation epidemics']\n",
      "['multilayer ensemble', 'with', 'given multilayer degree sequence']\n",
      "['numericcitation', 'numericcitation under', 'homogeneous susceptible-infection']\n",
      "['correlation', 'integrate', 'joint distribution']\n",
      "['algebraic distribution', 'in', 'n · p']\n",
      "['survey results', 'construct', 'weighted contact network']\n",
      "['our procedure', 'involve', 'three steps']\n",
      "['random vaccine distribution', 'in', 'selected popular location']\n",
      "['bounded second moment', 'on assumption', 'g1']\n",
      "['ﬁrst order', 'in', 'ε']\n",
      "['one focus', 'in', 'intermediate connectivity range']\n",
      "['assumption', 'be', 'where violated']\n",
      "['blue curve', 'in', 'figure 1']\n",
      "['τ1', 'in', 'x t']\n",
      "['sars patient', 'in', 'hospital']\n",
      "['lecture notes', 'in', 'mathematics']\n",
      "['mckendrick', 'application of', 'mathematics']\n",
      "['communities', 'with', 'smaller average sizes']\n",
      "['common paradigm', 'emerge from', 'simple deterministic framework']\n",
      "['vaccines', 'be', 'when suddenly exhausted']\n",
      "['infection spreads', 'with', 'addition']\n",
      "['numericcitation laumanns m', 'complexity of', 'impact size estimation']\n",
      "['memory eﬀects', 'in', 'dynamical systems']\n",
      "['network', 'with', 'bigger communities']\n",
      "['optimal control', 'evolution with', 'control']\n",
      "['optimal control', 'evolution with', 'control']\n",
      "['optimal control', 'evolution with', 'control']\n",
      "['optimal control', 'evolution with', 'control']\n",
      "['optimal control', 'evolution with', 'control']\n",
      "['n ode uninfe', 'cted', 'n od']\n",
      "['r. c. brunham', 'journal of', 'theoretical biology 232 71']\n",
      "['two dimensional dynamical systems', 'in', 'x n invariant plane']\n",
      "['unique positive root', 'in', 'cases']\n",
      "['statistical properties', 'be', 'formidable problem']\n",
      "['mean connectivity 4.07', 'in', 'giant component']\n",
      "['λ θ 1 log θ 0 s.t', 'with', 'λ']\n",
      "['b · log s', 'constraint of', 'θ 1 ≤ s']\n",
      "['minimum point mψ', 'in', 'τs']\n",
      "['corner cases', 'of number', '1 −']\n",
      "['invasion', 'in', 'spatial epidemics']\n",
      "['population turnover', 'with realization', 'upper panel']\n",
      "['function ρ', 'with', 'yet unknown f']\n",
      "['trees', 'with', 'uniform epidemic parameters']\n",
      "['trees', 'with', 'uniform epidemic parameters']\n",
      "['b', 'with', 'uniform epidemic parameters']\n",
      "['pontryagin', 'have', 'maximum principle']\n",
      "['pontryagin', 'have', 'maximum principle']\n",
      "['same way', 'in ˆa', 'n']\n",
      "['ˆa', 'in', 'same way']\n",
      "['immune system', 'move to', 'rf']\n",
      "['non-pharmaceutical interventions', 'protect', 'society']\n",
      "['exposed people', 'zombie within', 'minutes']\n",
      "['observed infection topology y', 'generate by', 'unbiased sampling algorithm']\n",
      "['countries', 'in', 'outer shells']\n",
      "['countries', 'in', 'nucleus']\n",
      "['their high similarity', 'in', 'shape']\n",
      "['suppression', 'in', 'region']\n",
      "['recurrence formula b', 'ζi in', 'g']\n",
      "['economic cost', 'in', 'uk']\n",
      "['poisson', 'with', 'mean 1']\n",
      "['poisson', 'with', 'mean 1']\n",
      "['distant vertex', 'in', 'same compo']\n",
      "['tex', 'in', 'same component']\n",
      "['progress', 'in', 'single community']\n",
      "['sars', 'of spread', 'acquisition']\n",
      "['edges', 'with', 'weights greater than 0.10']\n",
      "['optimal control strategy', 'with', 'constant control strategy']\n",
      "['connected graph', 'with', 'adjacency matrix a']\n",
      "['hierarchical dynamics', 'with', 'three plateaus']\n",
      "['hierarchical dynamics', 'with', 'three plateaus']\n",
      "['e. g. measles', 'in', 'highly vaccinated populations']\n",
      "['lecture notes', 'in', 'biomathematics']\n",
      "['obvious difference', 'in', 'ﬁnal outbreak size']\n",
      "['two critical exponents', 'determine', 'universality class']\n",
      "['agreement', 'with', 'result']\n",
      "['system', 'in s', 'r']\n",
      "['system', 'in s', 'i']\n",
      "['s', 'in', 'system']\n",
      "['2 ∼ m', 'in', 'ﬁnite-sized system']\n",
      "['recent developments', 'in', 'theoretical physics']\n",
      "['introduction ebola virus', 'currently aﬀecting', 'several african countries']\n",
      "['new clustering type metrics', 'use', 'structures']\n",
      "['additional observations', 'in', 'periods other than snapshot time']\n",
      "['classical swine fever', 'in', 'netherlands pig farms']\n",
      "['classical swine fever', 'in', 'switzerland pig farms']\n",
      "['urr', 'evaluation for', 'classical swine fever']\n",
      "['classical swine fever', 'in', 'wildlife']\n",
      "['response', 'in', 'turn']\n",
      "['analysis', 'scale', 'ex-fig']\n",
      "['m. van baalen', 'approximations for', 'diﬀerent spatial geometries']\n",
      "['s. percolation transition', 'in', 'dynamical trafﬁc net work']\n",
      "['network', 'in', 'clay center']\n",
      "['light dashed line', 'denote', 'bp']\n",
      "['light dashed line', 'denote', 'transcritical bifurcations']\n",
      "['interpretation', 'clear from', 'context']\n",
      "['isolation period', 'be', 'when maximum']\n",
      "['direct sampling', 'be', 'then employed']\n",
      "['1 introduction', 'in', 'paper']\n",
      "['subdivision', 'with', 'common characteristic features called']\n",
      "['r. pastor-satorras', 'spread in', 'scale-free networks']\n",
      "['m. e. j. newman', 'in', 'networks']\n",
      "['m. e. j. newman', 'structure in', 'networks']\n",
      "['modularity structure', 'in', 'networks']\n",
      "['m. e. j. newman', 'spread on', 'networks']\n",
      "['color online competitive advantage', 'in', 'case']\n",
      "['infectious individuals', 'for infect', 'i']\n",
      "['diﬀerences', 'in', 'optimal vaccination protocols between stochastic models']\n",
      "['having vaccinated individuals', 'reduce', 'reﬀ']\n",
      "['0 1 2', 'represent', 'susceptibility']\n",
      "['35 obs obs obs', 'in', 'factor graph representation']\n",
      "['true sars infection distribution', 'in', 'hk']\n",
      "['modiﬁed sir-d epidemic', 'be', 'after feller-rescaling indistinguishable']\n",
      "['information diﬀusion', 'in', 'overlaying social-physical networks']\n",
      "['second order phase transition', 'with', 'control parameter β']\n",
      "['t', 'be', 'random']\n",
      "['th', 'support by', 'epsrc']\n",
      "['individual nodes', 'in', 'network']\n",
      "['ﬁrst depiction', 'in', 'popular culture']\n",
      "['corresponding decision problem', 'solve', 'enumeration problem']\n",
      "['research', 'in', 'veterinary science 81 31 36']\n",
      "['populations', 'of fraction', 'y']\n",
      "['ds centrality', 'locate in', 'ﬂuential spreaders']\n",
      "['saddle', 'with', 'only one attracting direction']\n",
      "['g', 'be', 'ﬁxed']\n",
      "['.020.040.060.080.166.577.588.59 sample probability σaverage distance', 'in', 'section']\n",
      "['ui', '−', '1 such']\n",
      "['single focal point', 'dependence on', 'probability r travel out node along prescribed edges']\n",
      "['interaction', 'in', 'population']\n",
      "['instances', 'in', 'equations']\n",
      "['ak', 'acknowledge', 'ﬁnancial support']\n",
      "['distance l', 'number of', 'vertices n']\n",
      "['er network', 'for plot', 'n']\n",
      "['er network', 'with', 'hkigc']\n",
      "['abstract geographic speciﬁc information', 'in', 'paper']\n",
      "['people', 'in', 'compartment']\n",
      "['model development', 'of stage', 'propensity']\n",
      "['networks', 'with', 'time-varying transmission probabilities']\n",
      "['f', 'ﬁnalizes decomposition with', 'third shell']\n",
      "['performances', 'in', 'fig 6']\n",
      "['performances', 'in', 'fig 8']\n",
      "['phase diagrams', 'show in', 'figure 3']\n",
      "['koopmans', 'transmission to', 'human beings']\n",
      "['distribution', 'in', 'log-log']\n",
      "['distribution', 'in', 'fig 5']\n",
      "['location-based mitigation strategies random', 'in', 'most popular location random']\n",
      "['computed trajectories', 'with', 'initial condition']\n",
      "['deterministic perturbations decay', 'in', 'damped oscillatory way']\n",
      "['related problem', 'in', 'chen et al']\n",
      "['distribution', 'with', 'γ']\n",
      "['new class', 'with', 'delay']\n",
      "['matched components', 'indeed relate to', 'underlying pathogens']\n",
      "['rectangle', 'be', 'when elongated']\n",
      "['rectangle', 'be', 'when elongated']\n",
      "['today', 'in', 'global economy']\n",
      "['wc increase', 'with', 'β']\n",
      "['clear region', 'with', 'co-existing p']\n",
      "['other l − l vertices', 'in', 'c i 1']\n",
      "['average share edges', 'with', '4/3 other vertices']\n",
      "['susceptibleinfectiverecovered epidemics', 'with', 'strictly obligate relationship such']\n",
      "['remaining edges', 'in', 'g ∗']\n",
      "['a. modular activity', 'drive', 'networks']\n",
      "['activity potential', 'belong to', 'com']\n",
      "['γ', 'in', 'notation used above']\n",
      "['science', 'for foundation', 'fct']\n",
      "['contact patterns', 'in', 'high school']\n",
      "['w', 'in', 'cases']\n",
      "['γ', 'be', 'where degree exponent']\n",
      "['k', 'be', 'where sub ﬁgure']\n",
      "['discussion', 'with', 'wen-xu wang']\n",
      "['sexual experience', 'with', 'other individual']\n",
      "['nonequilibrium phase transitions', 'in', 'lattice models']\n",
      "['c. faloutsos', 'threshold in', 'real networks']\n",
      "['rate δ', 'be', 'sars disease-induced mortality']\n",
      "['unforced systems', 'with', 'stable focus']\n",
      "['agents', 'with', 'negative opinion']\n",
      "['agents', 'with', 'extremist opinion']\n",
      "['agents', 'convince agent with', 'moderate opinion']\n",
      "['agents', 'have', 'moderate opinion']\n",
      "['solid black curve', 'numerical solution of', 'eq']\n",
      "['unity', 'in', 'emax dependent resource rich regime']\n",
      "['nontrivial fixed point', 'of δ', 'trivial ﬁxed point']\n",
      "['one foci', 'in', 'low connectivity range']\n",
      "['conclusions', 'in', 'paper']\n",
      "['chebyshev', 'have', 'yields']\n",
      "['final outbreak size r', 'µ with', 'asynchronous updating']\n",
      "['color online susceptibility χ', 'with', 'cutoff']\n",
      "['homogeneous models', 'in', 'epidemiology']\n",
      "['our algorithm', 'signiﬁcantly outperform', 'dmp']\n",
      "['diﬀerent activity potential', 'belong to', 'com']\n",
      "['high impact', 'in', 'mainly guinea']\n",
      "['information theory proceedings', 'on symposium', 'isit']\n",
      "['black line', 'in', 'fig 2']\n",
      "['transition', 'eventually disappear for', 'large enough systems']\n",
      "['l. a. braunstein', 'journal of', 'physics 20 013007']\n",
      "['machinery', 'branch', 'process approximations']\n",
      "['branching process', 'be', 'multi-type']\n",
      "['litter', 'in', 'branching process']\n",
      "['forward process', 'in', 'g']\n",
      "['bit count', 'with', 'biran-kernignan method']\n",
      "['university', 'franqu es', 'mart']\n",
      "['university', 'institute for', 'scientiﬁc interchange']\n",
      "['time-scales', 'in', 'opposite limit µ → 1 networks']\n",
      "['parameter p', 'number of', 'recovered sites']\n",
      "['upperbound', 'in', 'addition']\n",
      "['emergence', 'be', 'key']\n",
      "['prediction', 'in uncertainty', 'citation citation']\n",
      "['prediction', 'overall result', 'look promising']\n",
      "['m. lax', 'in', 'statistical physics']\n",
      "['612 diagnosed cases', 'with', 'l']\n",
      "['same trend', 'with', 'larger rewiring']\n",
      "['abrupt transition', 'with', 'β']\n",
      "['abrupt transition', 'in', 'fig 5']\n",
      "['jumps', 'emerge at', 'lower infection rates']\n",
      "['expression', 'in', 'eq']\n",
      "['rameters', 'leave to', 'right']\n",
      "['sis-like diseases', 'in', 'tightly connected clusters']\n",
      "['difference', 'in', 'work']\n",
      "['abbreviations', 'explain in', 'methods section']\n",
      "['power law', 'with', 'exponent one']\n",
      "['respect 3', 'in', 'z']\n",
      "['stage', 'of day', 'march 20 2003']\n",
      "['spatial structures', 'in', 'humans']\n",
      "['wavy arrow', 'represent', 'transition']\n",
      "['wavy arrow', 'represent', 'transition']\n",
      "['v. s imu lat ion s', 'in', 'section']\n",
      "['education', 'of ministry', 'citation r1a1a2010067']\n",
      "['y', 'with', 'length 1']\n",
      "['traditional topic', 'in', 'mathematical epidemiology']\n",
      "['α', 'in', 'four dimensional case']\n",
      "['simulation', 'proceed to', 'next step']\n",
      "['work', 'part of', 'pict 0293/2008']\n",
      "['3 deterministic phase τ', 'in', 'section']\n",
      "['google search interest', 'in', 'myspace']\n",
      "['results', 'in', 'fig 2']\n",
      "['simulation', 'make', 'approximations']\n",
      "['non-linear diﬀerential equations systems', 'give by', 'eqs']\n",
      "['models', 'with', 'parameter assumptions']\n",
      "['nonlinear science center', 'university of', 'science']\n",
      "['nodes', 'with', 'high activity rate']\n",
      "['management', 'of school', 'university road']\n",
      "['out-community degree ko', 'treat separately in', 'fact']\n",
      "['robust information source estimator', 'with', 'sparse observations kai zhu']\n",
      "['system', 'in', 'eq']\n",
      "['existence condition', 'in', 'eq']\n",
      "['ϕ', 'transition to', 'state r']\n",
      "['exposed people', 'become', 'zombies within minutes']\n",
      "['betweenness', 'be', 'hence medial centrality measure']\n",
      "['similarities', 'in', 'contagion mechanisms']\n",
      "['homogeneous networks', 'with', 'hki']\n",
      "['relative errors', 'in', 'ﬁrst column']\n",
      "['relative errors', 'in', 'ﬁrst column']\n",
      "['relative errors', 'in', 'ﬁrst column']\n",
      "['several signiﬁcant crises', 'in', 'past few decades']\n",
      "['mixing time', 'on numericcitation', 'cdc']\n",
      "['poisson processes', 'with', 'intensity λh']\n",
      "['poisson processes', 'with', 'intensity λs']\n",
      "['poisson processes', 'with', 'intensity λg']\n",
      "['branching processes', 'make', 'actual computations']\n",
      "['particle', 'in', 'branching processes']\n",
      "['use', 'in', 'rural communities']\n",
      "['modeling infectious disease', 'in', 'humans']\n",
      "['non-linear third line', 'in', 'eq']\n",
      "['third line', 'in', 'eq']\n",
      "['2012 dynamthe ics', 'with', 'immune boosting']\n",
      "['vmax ≤ vmax', 'in', 'real part']\n",
      "['j. shaman', 'journal of', 'epidemiology 185 395']\n",
      "['n × 1 vector', 'with', 'only ith element 1']\n",
      "['y y y diﬀerential equation', 'with', 'deﬁnitions']\n",
      "['swae', 'expect eccentricity of', 'random vertex']\n",
      "['erdős p', 'a on', 'random graphs']\n",
      "['maximum', 'in', 'derivatives']\n",
      "['annealed approximation', 'in', 'ref']\n",
      "['diﬀerences', 'in', 'population demographics']\n",
      "['substantial diﬀerences', 'in', 'observed outcomes']\n",
      "['infectious disease', 'in', 'humans']\n",
      "['core', 'therefore consist of', 'following twelve countries']\n",
      "['what general procedure', 'outline in', 'appendix c']\n",
      "['spreads', 'in', 'n ext']\n",
      "['authors', 'describe', 'sole waning immunity process']\n",
      "['sipser', 'with', 'ﬁxed degree sequence']\n",
      "['infectious diseases', 'in', 'humans']\n",
      "['function', 'as tst', 'τ']\n",
      "['function', 'as ist', 'τ']\n",
      "['astrophysics e-print archive', 'snapshot of', 'structure']\n",
      "['preventive medicine', 'of journal', 'pages 182-188 2001']\n",
      "['pathogens', 'typically infect', 'multiple species']\n",
      "['their structural isomorphism', 'with', 'certain problems concerning']\n",
      "['intersection', 'with', 'identity']\n",
      "['hyper-graph counting', 'quickly become', 'too tedious']\n",
      "['solution', 'approach', 'asymptotic stable focus']\n",
      "['half-edge', 'be', 'when paired oﬀ']\n",
      "['entertaining example', 'that of', 'impending so-called zombie apocalypse']\n",
      "['economic crisis supplementary information a. garas1', 'of spread', 'university']\n",
      "['its particles', 't to', 'x']\n",
      "['distinct states', 'of sequence', 'i1']\n",
      "['springer lecture notes', 'in', 'statistics 151']\n",
      "['n', 'be', 'ﬁnite']\n",
      "['estimates', 'in', 'such comparison']\n",
      "['na ∝ n 3 11', 'n for', 'three-type cbp']\n",
      "['source localization', 'in', 'network']\n",
      "['discussion', 'in', 'summary']\n",
      "['signiﬁcant alterations', 'in', 'network structure']\n",
      "['nuclei', 'use', 'k shell method']\n",
      "['classical markov process', 'in δ', 't − t ′']\n",
      "['many large corporations', 'in', 'global corporate database']\n",
      "['public health crisis', 'in', 'korea']\n",
      "['network', 'with', 'total 2886 links among 206 nodes']\n",
      "['network', 'with', 'total 2886 links among 206 nodes']\n",
      "['changes', 'in', 'b']\n",
      "['medicine', 'of journal', 'pages 31-38 2003']\n",
      "['fleischmann', 'behavior of', 'measure']\n",
      "['xm', 'be', 'initially 28 infective']\n",
      "['evolution', 'in', 'network']\n",
      "['barab asi', 'proceed of', 'national academy']\n",
      "['a. baronchelli', 'proceed of', 'national academy']\n",
      "['their interplay', 'introduce', 'non-trivial effects']\n",
      "['competition', 'in', 'changing world']\n",
      "['details', 'in', 'method']\n",
      "['qmf method', 'better than', 'mfl method']\n",
      "['effect', 'in', 'infectious period']\n",
      "['department', 'of paper', 'uk']\n",
      "['µ', 'with', 'i-th entry']\n",
      "['gray node', 'in', 'b']\n",
      "['their solutions', 'work for', 'epidemiologists']\n",
      "['tra jectories', 'on', 'level sets']\n",
      "['our trick', 'in', 'section 3.1 n n']\n",
      "['our proof', 'in', 'section 3']\n",
      "['m ∗ ∗', 'k by', 'assumption']\n",
      "['assumption', 'by k', 'k p']\n",
      "['assigned value', 'in', 'rank c ∈ numericcitation']\n",
      "['synchronization problems', 'in', 'complex netwrks']\n",
      "['message passing', 'simplify', 'calculations']\n",
      "['master equation', 'transform via', 'quantum spin operator formulation']\n",
      "['similar result', 'show in', 'fig 2']\n",
      "['business', 'of 3faculty', 'university']\n",
      "['rcpg', 'in', 'phe weekly national inﬂuenza reports']\n",
      "['predicted hidden states', 'base on', 'data collected']\n",
      "['x', 'be', 'random']\n",
      "['n', 'be', 'random']\n",
      "['y', 'be', 'random']\n",
      "['z', 'be', 'random']\n",
      "['group', 'in', 'single clique']\n",
      "['markov chain', 'with', '2n states']\n",
      "['l. a. braunstein', 'report in', 'physics 80 036603']\n",
      "['form', 'in 13', 'i']\n",
      "['m. rosvall', 'journal of', 'complex networks 3 177']\n",
      "['o', 'with', 'control r 0']\n",
      "['control r 0', 'with i', 't']\n",
      "['control s', 'with i', 't']\n",
      "['t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s', 'with', 'control i']\n",
      "['control i', 'with s', 't']\n",
      "['control r 0', 'with i', 't']\n",
      "['word-of-mouth control', 'in', 'fig 2b']\n",
      "['control i', 'with i', 't']\n",
      "['control s', 'with i', 't']\n",
      "['t i n o 1 4 5 0.05 0.01 0.02 0.03 0.04 c n o t r o l s', 'with', 'control i']\n",
      "['control i', 'with s', 't']\n",
      "['control i', 'with i', 't']\n",
      "['control s', 'with i', 't']\n",
      "['t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s', 'with', 'control i']\n",
      "['control i', 'with s', 't']\n",
      "['control i', 'with i', 't']\n",
      "['control s', 'with i', 't']\n",
      "['t i n o 1 4 5 0.06 0.01 0.02 0.03 0.04 0.05 c n o t r o l s', 'with', 'control i']\n",
      "['control i', 'with s', 't']\n",
      "['graph', 'with', 'adjacency matrix a']\n",
      "['degree distribution', 'in', 'layer']\n",
      "['interval', 'on each', 'i']\n",
      "['interval', 'on each', '0 t']\n",
      "['based approach', 'locate information source with', 'sparse observations']\n",
      "['markov chain', 'with', '3n states']\n",
      "['vertex', 'enter', 'process']\n",
      "['models', 'in', 'mathematical epidemiology']\n",
      "['table i', 'summary of', 'bifurcation diagram']\n",
      "['co-evolves', 'with', 'social context']\n",
      "['1-dimensional lattice', 'with', 'periodic boundary conditions']\n",
      "['pa', 'give', 'consistently better results than mfa']\n",
      "['stochastic eﬀects', 'be', 'once unimportant']\n",
      "['extinct', 'strongly inﬂuenced by', 'stochastic eﬀects']\n",
      "['extinct', 'inﬂuenced by', 'stochastic eﬀects']\n",
      "['epidemic threshold', 'of ﬁcation', 'chengdu']\n",
      "['µ → 1', 'spread of', 'disease']\n",
      "['results', 'average over', '1000 realizations']\n",
      "['their statistical analysis lecture notes', 'in', 'statistics']\n",
      "['n', 'give n/mb for', 'average 6 fig 1']\n",
      "['newborns', 'with', 'passive immunity']\n",
      "['generating function', 'in', 'limit']\n",
      "['large-η limit', 'in exp', '− π2 η']\n",
      "['˜φ 2π2 exp', 'in', 'large-η limit']\n",
      "['objective functionals', 'in', 'our models']\n",
      "['contact', 'with', 'i']\n",
      "['litters', 'in', 'limit']\n",
      "['bottom panels', 'consider', '50 edges']\n",
      "['k p', 'with', 'indicator variable z i']\n",
      "['variable selection', 'with', 'lasso']\n",
      "['connected digraph', 'with', 'adjacency matrix a']\n",
      "['connected digraph', 'with', 'adjacency matrix a']\n",
      "['direct sampling', 'with', 'complete information']\n",
      "['direct sampling', 'with', 'complete information']\n",
      "['direct sampling', 'with', 'complete information']\n",
      "['simulations', 'value of', 'r0']\n",
      "['may contact', 'in', 'random strategy']\n",
      "['problems', 'in', 'mother company']\n",
      "['curve', 'in', 'log-log diagram']\n",
      "['net change', 'in', 'zt']\n",
      "['4center institute', 'in', 'fundamental sciences']\n",
      "['wide occurrence', 'in', 'real world']\n",
      "['community structure', 'slow down', 'spreading processes']\n",
      "['eﬀective predictor', 'in', 'real scenario']\n",
      "['lcad', 'have', 'higher r2-values than swd']\n",
      "['7 v. fluctuations', 'in', 'previous sections']\n",
      "['drift terms', 'in', 'ˆl']\n",
      "['health contact', 'add', 'health']\n",
      "['condition', 'in', 'theorem 2.4']\n",
      "['ψ', 'stability of', 'nontrivial fixed point']\n",
      "['several stages', 'in', 'paper']\n",
      "['log scales', 'use on', 'axes']\n",
      "['shell index ks', 'in', 'most cases']\n",
      "['µ → 1', 'repetition of', 'same contacts within communities']\n",
      "['pa', 'give', 'quantitative improvement over mfa']\n",
      "['same argument', 'by op', 'n α']\n",
      "['has similarity', 'propagation of', 'virus']\n",
      "['a δ', 'point of', 'system equation']\n",
      "['competition', 'in', 'important situation']\n",
      "['individuals', 'in', 'population']\n",
      "['n individuals', 'in', 'population']\n",
      "['connected structures', 'with', '3 nodes']\n",
      "['connected structures', 'to square', 'φ2 4']\n",
      "['collapse ratio', 'with', 'ﬁxed intra-species collapse factor c11']\n",
      "['µ', 'in', 'our set-in']\n",
      "['local interaction', 'with', 'nearest neighbours']\n",
      "['ring', 'with', 'next nearest neighbour interactions']\n",
      "['uncorrelated networks', 'with', 'strongly heterogeneous degree distribution']\n",
      "['cholera outbreaks', 'in', 'bangladesh']\n",
      "['forancestor', 'include size', 'total progeny']\n",
      "['total weight', 'in', 'v']\n",
      "['latter', 'have', 'corresponding linear approximation']\n",
      "['i/t', 'keep', 'ﬁxed']\n",
      "['one-to-one nodes', 'in', 'network a']\n",
      "['zombies', 'in', 'popular culture ﬁlms']\n",
      "['nodes', 'in', 'diﬀerent layers']\n",
      "['network', 'with', 'adaptive rewiring']\n",
      "['control', 'in', 'second stage']\n",
      "['computational complexity', 'in', 'section']\n",
      "['update schema', 'in', 'recently introduced no-enclave percolation']\n",
      "['x0 terms', 'in', 'latter expression']\n",
      "['eﬃcient isolation', 'with', 'high degree']\n",
      "['spread', 'in', 'plants']\n",
      "['its positivity', 'in', 'population']\n",
      "['electrical engineering', 'university of', 'southern california']\n",
      "['monte carlo simulations', 'with', '2 · 106 realizations']\n",
      "['barab', 'asi', 'rev mod']\n",
      "['connection', 'with', 'percolation theory']\n",
      "['advances', 'in', 'neural information processing systems 2007']\n",
      "['j', 'of', 'partially constructed susceptibility set']\n",
      "['j', 'of', 'partially constructed susceptibility set']\n",
      "['social phenomenon', 'have', 'catchiness']\n",
      "['advances', 'in', 'applied probability 6 494 511']\n",
      "['advances', 'in', 'applied probability 6 622 635']\n",
      "['barab', 'asi', 'nature physics 8']\n",
      "['uniqueness solution', 'with', 'direct recruitment']\n",
      "['stubs', 'in', 'random edge']\n",
      "['uneven roles', 'in', 'global economic network']\n",
      "['orange line', 'in', 'fig 1g']\n",
      "['synchronous updating', 'with 0.6', 'circles']\n",
      "['individuals', 'spontaneously become', 'immune']\n",
      "['last point', 'in', 'plot']\n",
      "['2010 monte carlo simulation', 'in', 'statistical physics']\n",
      "['contact', 'in', 'g']\n",
      "['severity', 'proceed of', 'royal society b 273 2025 2034']\n",
      "['small local perturbations', 'in', 'connectivity patterns']\n",
      "['citation contagion', 'in', 'yambuku']\n",
      "['swn', 'model', 'several real systems']\n",
      "['β †', 'explain in', 'main text']\n",
      "['number', 'in', 'i.i.d sizes']\n",
      "['number', 'in', 'cluster']\n",
      "['number', 'with', 'type household']\n",
      "['last term', 'in', 'q3 x']\n",
      "['registration', 'once require for', 'example']\n",
      "['strong correlation', 'with', 'topology']\n",
      "['multinomial distribution', 'with', 'parameters pi']\n",
      "['graphs', 'of handbook', 'citation']\n",
      "['stationary state', 'with', 'jump close τc']\n",
      "['m', 'in', 'scale']\n",
      "['nonlinear system eq', 'through cooney', 'francisco prieto']\n",
      "['t', 'control to', 'problem']\n",
      "['problem', 'in u1', 'u2']\n",
      "['u1', 'in', 'problem']\n",
      "['ﬁrst inequality', 'hold by', 'b']\n",
      "['kernel', 'be', 'so strictly positive']\n",
      "['graphs', 'with', 'small-world properties']\n",
      "['∞ k', 'with', 'ﬁnite mean λ']\n",
      "['similar arguments', 'in', 'related situation']\n",
      "['con-n → ∞ ˆit', 'to mi', 'dominated convergence theorem']\n",
      "['scale', 'in', 'way']\n",
      "['l. a. epidemics', 'in', 'partially overlapped multiplex networks']\n",
      "['information engineering', 'university of', 'padova']\n",
      "['electrical engineering', 'department of', 'information engineering']\n",
      "['fluctuating epidemics', 'lege of', 'wil liam']\n",
      "['same degree', 'of graph', 'rrg']\n",
      "['points', 'in', 'curves']\n",
      "['discussion', 'in', 'conclusion']\n",
      "['various parameters', 'in', 'sis models']\n",
      "['third equation', 'redundant because of', 't']\n",
      "['5 simulation', 'test pmmh algorithm on', 'simulated infectious disease data']\n",
      "['t. simulation', 'with', 'seasonally varying α']\n",
      "['exponential distribution', 'with', 'rate 1']\n",
      "['nonlinear mapping', 'with', '2n states rather 3n states']\n",
      "['leading order', 'in', '1 √ n']\n",
      "['figure 4', 'count in', 'mathbaria']\n",
      "['random intersection graphs', 'may', 'may thought']\n",
      "['several infections', 'in', 'livestock']\n",
      "['zero steady state solution', 'stable for', 'γ']\n",
      "['nodes', 'with', 'high betweenness centrality']\n",
      "['removal', 'with', 'high betweenness centrality']\n",
      "['nodes', 'with', 'high betweenness centrality']\n",
      "['nodes', 'with', 'high betweenness centrality']\n",
      "['term', 'in', 't2']\n",
      "['geometric distribution', 'with', 'expectation 1/q']\n",
      "['0 0n', '0n for', 'al l t']\n",
      "['al l t ≥ 0', 'for s', 't']\n",
      "['robust information source estimator', 'with', 'sparse observations']\n",
      "['outstanding ph d. candidate', 'in', 'academic research']\n",
      "['people', 'by', 'likelihood travel along existing edges']\n",
      "['several weeks', 'in', 'advance']\n",
      "['brain', 'burn', 'body']\n",
      "['outbreak', 'in', 'england']\n",
      "['probability', 'in', 'framework']\n",
      "['probability', 'for a', 'b']\n",
      "['layers', 'with', 'probability w']\n",
      "['probability', 'for equation', 'fa']\n",
      "['urse notes', 'in', 'economics 18']\n",
      "['np-hard 7 path τ', 'in', 'g']\n",
      "['total control eﬀorts', 'be', 'however considerably diﬀerent']\n",
      "['fig 2', 'highlight by', 'analytic results']\n",
      "['increase', 'in', 'similar way']\n",
      "['outbreaks', 'in', 'africa']\n",
      "['li', 'transition in', 'dynamical trafﬁc net work']\n",
      "['zero-energy', 'in', 'phase space']\n",
      "['method', 'develop in', 'ref']\n",
      "['numericcitation cohen', 'strategy for', 'computer networks']\n",
      "['threshold', 'be', 'interestingly same i.e ξs i s ξs ir']\n",
      "['offspring distribution', 'be', 'poisson']\n",
      "['offspring distribution', 'have', 'exponentially decaying tail']\n",
      "['p', 'consist of', 'single vertex']\n",
      "['precise numerical determination', 'in', 'absorbing-state phase transitions']\n",
      "['sitions', 'in', 'lattice models']\n",
      "['threshold', 'in', 'sis dynamics']\n",
      "['pair ξ', 'ζ at', 'neighboring sites']\n",
      "['left panel', 'in', 'fig 7']\n",
      "['l. a. meyers', 'proceed of', 'royal society']\n",
      "['version', 'with', 'interventions']\n",
      "['comparison', 'with', 'eigenvector centrality']\n",
      "['eigenvector centrality', 'outperform', 'k-shell']\n",
      "['eigenvector centrality', 'typically outperform', 'k-shell']\n",
      "['centrality measure', 'in', 'bonacich']\n",
      "['diseases', 'with', 'multiple host species']\n",
      "['pathogens', 'with', 'multiple host species']\n",
      "['days', 'in', 'season']\n",
      "['initial conditions x', 'with', 'constants ni']\n",
      "['initial conditions', 'with', 'x']\n",
      "['n3', 'with', 'initial condition']\n",
      "['change', 'in', 'k shell ranking']\n",
      "['ﬁrst term', 'in', 'eq']\n",
      "['ﬁrst term', 'in', 'eq']\n",
      "['convergence', 'be', 'locally uniform']\n",
      "['management', 'of school', 'united kingdom 7 ehumanities group']\n",
      "['other reasonable problems', 'arise from', 'practical applications']\n",
      "['k', 'be', 'random']\n",
      "['k', 'be', 'random']\n",
      "['x', 'be', 'random']\n",
      "['spitzer f', 'principle of', 'random']\n",
      "['r0', 'in', 'inﬂuenza']\n",
      "['recent ebola outbreak', 'in', 'west africa']\n",
      "['ccm', 'long way from', 'randomly introduced clustering']\n",
      "['patient-zero problem', 'with', 'noisy observations']\n",
      "['one individual', 'of point', 'dependence']\n",
      "['βf', 'be', 'when increased']\n",
      "['βf', 'be', 'when reduced']\n",
      "['contact network', 'compose of', 'n']\n",
      "['node degree', 'in', 'binary network']\n",
      "['cloud network', 'in node', 'survey respondents']\n",
      "['nodes', 'in', 'cloud network']\n",
      "['results', 'in', 'communications network reliability']\n",
      "['complex networks', 'with', 'degree correlations']\n",
      "['complex networks', 'in spread', 'springer-verlag berlin heidelberg']\n",
      "['complex networks', 'in spread', 'lnp 625']\n",
      "['complex networks', 'in spread', 'pages 127147']\n",
      "['r. pastor-satorras', 'immunization of', 'complex networks']\n",
      "['d. ben-avraham', 'strategy for', 'computer networks']\n",
      "['a. barrat', 'process on', 'complex networks']\n",
      "['further results', 'also hold for', 'size-biased variants']\n",
      "['cases', 'in', 'real world']\n",
      "['undirected links', 'in', 'resulting network']\n",
      "['fraction', 'in', 'stationary state']\n",
      "['6.1 sirs', 'with', 'constant delay']\n",
      "['ﬁxed graph g', 'with', 'labelling pr']\n",
      "['ﬁxed graph g', 'in', 'formalism']\n",
      "['information', 'in', 'population']\n",
      "['buzz', 'in', 'population']\n",
      "['population', 'of s', 't']\n",
      "['network science', 'for ,1', '45195-1159 zanjan']\n",
      "['problem', 'answer correctly in', 'simulations']\n",
      "['heterogeneity', 'in', 'contact']\n",
      "['common approach', 'in', 'epidemiology']\n",
      "['irsir system', 'with', 'initial condition near point']\n",
      "['instance people', 'with', 'aids']\n",
      "['signiﬁcant lagged relationship', 'with', 'cholera incidence']\n",
      "['18 proposition 2', 'instability of', 'dfe']\n",
      "['scaled sizes', 'in', 'eq']\n",
      "['management', 'of school', 'joan muyskenweg 25']\n",
      "['expected average degree', 'in', 'rrg']\n",
      "['diﬀuses', 'with', 'diﬀusion constant d']\n",
      "['stationary distribution', 'with', 'power-law tail']\n",
      "['front advances', 'with', 'constant velocity']\n",
      "['aspects', 'in', 'many systems']\n",
      "['many systems', 'in', 'ﬁelds']\n",
      "['1000 network realizations', 'with', 'n']\n",
      "['1000 network realizations', 'with', 'n']\n",
      "['node i', 'infect', 'its node j']\n",
      "['survivors', 'of variance', 'table ii']\n",
      "['proportion', 'in', 'same population']\n",
      "['step t', 'connection to', 'j']\n",
      "['step t', 'connection from', 'k']\n",
      "['w. m. liu', 'journal of', 'mathematical biology 27 233']\n",
      "['unique stable equilibrium point', 'at s', 't']\n",
      "['decreasing function', 'in', 'y']\n",
      "['decreasing function', 'in', 'y']\n",
      "['depending position', 'in', 'phase diagram']\n",
      "['fracion', 'in', 'steady state']\n",
      "['complex systems', 'of institute', 'ubics']\n",
      "['ebola', 'with', 'relatively small 1 ≤ r0 ≤ 7']\n",
      "['nontrivial ﬁxed point', 'be', 'unique']\n",
      "['nontrivial fixed point', 'of δ', 'origin']\n",
      "['markov epidemic spreading', 'with', 'λ']\n",
      "['a. il lustration', 'with', 'its 4 coupled seir variables']\n",
      "['criticality', 'in', 'absorbing-state phase transitions']\n",
      "['global clustering', 'use', 'purely unique counts']\n",
      "['endemic equilibrium', 'in', 'reservoir']\n",
      "['viral campaigns', 'run than', 'less viral campaigns']\n",
      "['2geography institute', 'university of', 'lausanne']\n",
      "['2geography institute', 'faculty of', 'geosciences']\n",
      "['1 introduction', 'in', 'bangladesh']\n",
      "['norm', 'with', 'linear inequality constraints']\n",
      "['locatin g', 'with', 'great theoretical']\n",
      "['relative increase', 'in', 'j']\n",
      "['virus', 'be', 'how transmitted']\n",
      "['seasonal cholera outbreaks', 'in', 'bangladesh']\n",
      "['t', 'solution to', 'optimal control problem']\n",
      "['initialise node', 'with', 'ﬁve stubs']\n",
      "['epidemic models', 'in', 'dimensions']\n",
      "['adjacency matrix', 'with share', 'connection']\n",
      "['phase diagram', 'in', 'plane vl − ω']\n",
      "['signi ﬁcant', 'in', 'ﬁeld spreading']\n",
      "['series expansion', 'in', 'frac']\n",
      "['role', 'in', 'world crisis']\n",
      "['zhu', 'control of', 'computer virus']\n",
      "['nodes', 'with', 'high betweenness centrality rank']\n",
      "['critical point', 'in', 'fig 2']\n",
      "['tra jectories', 'with', 'initial conditions out invariant plane']\n",
      "['quick action', 'in', 'early stage']\n",
      "['numbers', 'in', 'sir-d epidemic']\n",
      "['well-know procedures', 'in', 'algorithmic graph theory']\n",
      "['β1', 'in', 'fig 1']\n",
      "['hamiltonian maximizing condition', 'u', 't']\n",
      "['conclusions', 'in', 'paper']\n",
      "['numericcitation', 'introduction to', 'ﬁeld']\n",
      "['x k', 'be', 'corresponding counting measure']\n",
      "['european community', 'by', 'h2020 program under funding scheme fetproact-1-2014']\n",
      "['other settings', 'same as', 'fig 1']\n",
      "['other settings', 'same as', 'fig 1']\n",
      "['outstanding ph d. candidate', 'in', 'academic research']\n",
      "['zero-patient problem', 'with', 'noisy observations']\n",
      "['vertices', 'be', 'initially unexplored']\n",
      "['node degree', 'with', 'trajectories resampled']\n",
      "['reconstruction', 'in', '200 node synthetic scale free network']\n",
      "['reconstruction', 'in', '200 node synthetic scale free network']\n",
      "['inverse problem', 'under-determined in', 'sense']\n",
      "['wang', '3center for', 'polymer studies']\n",
      "['szendroi b', 'in', 'contact networks']\n",
      "['direct sampling', 'predict', 'future states']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['direct sampling', 'with', 'complete observation']\n",
      "['random variable tm', 'of pt', 'tm']\n",
      "['gg', 'spread in', 'human populations']\n",
      "['identifying bang-bang function u ∈ a', 'with', 'method']\n",
      "['k', 'in', 'following sections']\n",
      "['y', 'with', 'diﬀerent order approximations obtained']\n",
      "['networks', 'with', 'positive degree-degree correlations']\n",
      "['networks', 'with', 'positive degree-degree correlations']\n",
      "['networks', 'with', 'positive degree-degree correlations']\n",
      "['higher values', 'with', 'decreasing α']\n",
      "['extensions', 'in', 'section iv b']\n",
      "['transition', 'in', 'disease incidence']\n",
      "['l', 'theory of', 'home range estimation']\n",
      "['diﬀerent approaches', 'in', 'refs']\n",
      "['conclusions', 'in', 'conclusion']\n",
      "['biu center', 'in', 'applied cryptography']\n",
      "['real networks', 'however have', 'ﬁnite critical transmisibilities']\n",
      "['average number', 'in', 'lattice']\n",
      "['biological sciences', 'institute in', 'fundamental sciences']\n",
      "['motifs', 'be', 'how counted']\n",
      "['phenomenon', 'particularly visible in', 'solid blue curve']\n",
      "['many other reasonable problems', 'arise from', 'practical applications']\n",
      "['emergence', 'in', 'hsv-2']\n",
      "['larger change', 'in', 'gdp']\n",
      "['approximation', 'in', 'case']\n",
      "['power law', 'with', 'continuously varying exponent']\n",
      "['range', 'know for', 'pathogens']\n",
      "['sustained oscillations', 'in', 'thermodynamic limit']\n",
      "['s. a. levin', 'journal of', 'mathematical biology 35 825']\n",
      "['numericcitation dorogovtsev', 'phenomenon in', 'complex networks']\n",
      "['numericcitation pastor-satorras', 'dynamics in', 'complex networks']\n",
      "['identi ﬁcation', 'in', 'complex networks']\n",
      "['convergence criteria', 'repeat', 'step 1']\n",
      "['correlation coeﬃcient r', 'in', 'fig 5']\n",
      "['advances', 'in', 'neural information processing systems 25 2012']\n",
      "['sugitani', \"'s theorem\", 'z t 0 xs ds zx lt ψ dx']\n",
      "['our ambition', 'with', 'paper']\n",
      "['k chosen terminals', 'be', 'mutually wise connected']\n",
      "['k chosen terminals', 'be', 'mutually connected']\n",
      "['particles', 'neighbor site accord to', 'law rn described above']\n",
      "['function', 'as s1', 't']\n",
      "['function', 'as s1', 't']\n",
      "['function', 'as s1', 't']\n",
      "['function', 'as tc', 'tr']\n",
      "['function', 'as tc', 'tr']\n",
      "['stochastic processes', 'in', 'physics']\n",
      "['f. predicting percolation thresholds', 'in', 'networks']\n",
      "['its performance', 'even worse than', 'mfl method']\n",
      "['constraint symmetry θi', 'of form', 'i known interactions θi']\n",
      "['randomization process', 'remove', 'topological structures']\n",
      "['trees', 'with', 'branching ratio k']\n",
      "['trees', 'with', 'branching ratio 3']\n",
      "['trees', 'with', 'branching ratio 3']\n",
      "['mckendrick', 'have', 'ode model']\n",
      "['milho film', 'news on', 'request']\n",
      "['introduction', 'in', 'modeling framework']\n",
      "['second lump', 'be', 'when ignored']\n",
      "['complete set', 'for ρ', 'k']\n",
      "['base', 'be', 'its neighbor']\n",
      "['small change', 'in', 'connectivity']\n",
      "['durable-good duopoly', 'journal of', 'operations research']\n",
      "['logarithmic increase', 'with', 'n']\n",
      "['weir advanced research centre', 'university of', 'strathclyde']\n",
      "['diﬃculties', 'with', 'statistical inference']\n",
      "['working precision', 'typically set up in', 'integrations']\n",
      "['control signal', 'take value in', '0 uma x']\n",
      "['optimal control', 'on morton', 'k. h. wickwire']\n",
      "['sample path based estimators', 'in', 'section']\n",
      "['ancestor', 'in', 'branching process']\n",
      "['particles', 'in', 'branching process']\n",
      "['umax region', 'in', 'fig 4']\n",
      "['media', 'sociology of', 'health 29 46 65']\n",
      "['systems', '2department of', 'mathematics']\n",
      "['applied changes', 'in', 'structure']\n",
      "['k k', 'be', 'hypergeometric function']\n",
      "['diﬀerent initial conditions x', 'for z', 't']\n",
      "['same bifurcation diagram', 'in', 'fig 1']\n",
      "['form', 'with', 'u']\n",
      "['enough information', 'in', 'data']\n",
      "['triangle curve', 'in', 'fig 2 reﬁnes']\n",
      "['massive employment', 'in', 'network epidemiology']\n",
      "['complete motif', 'with', '4 members']\n",
      "['three theoretical predictions λmfl', 'of performance', 'e.g social networks']\n",
      "['branching processes', 'with', 'uniformly positive kernel']\n",
      "['dynamic percolation', 'with', 'instantaneous void transmissibility v s ≡']\n",
      "['diﬀerence', 'in', 'φ1 4']\n",
      "['problem', 'be', 'computationally intractable']\n",
      "['problem', 'in', 'class p']\n",
      "['aki', 'generalization to', 'weighted complex networks']\n",
      "['castellano', 'physics of', 'social dynamics']\n",
      "['1 pascal cr epey', 'variability in', 'complex networks arxiv']\n",
      "['c2', 'in', 'closed form ω1']\n",
      "['dashed blue line', 'in', 'fig 5-b']\n",
      "['ten years', 'of average', 'solid black line']\n",
      "['ﬁrst l vertices', 'in', 'c']\n",
      "['last addend', 'in', 't4']\n",
      "['diﬀerent strategies', 'with', 'varying parameter values']\n",
      "['actor interchanges information', 'with', 'its pairs']\n",
      "['spatial eﬀects', 'must', 'must necessarily considered']\n",
      "['impact', 'in', 'adaptive networks']\n",
      "['investment', 'be', 'key']\n",
      "['conventional brute-force simulation', 'thus become', 'prohibitively ineﬃcient']\n",
      "['evolution equation', 'with', 'vital dynamics']\n",
      "['higher concentration', 'in', 'environment']\n",
      "['87 1974 integral operators', 'with', 'nonnegative kernels']\n",
      "['node i', 'infect', 'its neighbouring node j']\n",
      "['haitian creole term zonbi', 'in', 'caribbean']\n",
      "['north mbundu term nzumbe', 'in', 'africa']\n",
      "['region', 'with', 'few individuals']\n",
      "['many short loops', 'in', 'network']\n",
      "['claim', 'rigorously prove by', 'sahneh et al']\n",
      "['avera ge', 'in', 'second column']\n",
      "['average relative errors', 'in', 'second column']\n",
      "['average relative errors', 'in', 'second column']\n",
      "['delay', 'in', 'global separation']\n",
      "['∼ 1 4α π √', 'with', 'exponential cutoﬀ around ξ']\n",
      "['their transformation', 'in', 'epidemics']\n",
      "['i1 κ1 i1 recovery', 'of strain', 'β2']\n",
      "['information spread', 'in', 'two diﬀerent models']\n",
      "['branching random walk', 'with', 'catalysts']\n",
      "['similarity sampling', 'underestimate at', 'late times']\n",
      "['observation', 'occur at', 'later times']\n",
      "['outbreak', 'in', 'certain communities']\n",
      "['same argument', 'in', 'proof']\n",
      "['large scale climate ﬂuctuations inﬂuence cholera transmission', 'in', 'bangladesh']\n",
      "['r. virus spread', 'in', 'networks']\n",
      "['phase diagram vl', 'with', 'ω']\n",
      "['delay term', 'in', 'equation']\n",
      "['ch .4', 'in', 'climate change']\n",
      "['trade balance', 'numerator of', 'eq']\n",
      "['longest self-avoiding path', 'in', 'g']\n",
      "['topological information', 'in', 'observation']\n",
      "['j. gog', 'journal of', 'mathematical biology 45 471']\n",
      "['j. swinton', 'journal of', 'mathematical biology 44 169']\n",
      "['γ', 'rate of', 'recovery']\n",
      "['educational campaigns', 'journal of', 'diﬀerential equations']\n",
      "['j', 'hull of', 'n planar brownian 12']\n",
      "['crisis spreading process', 'in', 'con']\n",
      "['its relative frequency', 'in', 'experiment']\n",
      "['recent advances', 'in', 'technology']\n",
      "['studies', 'nonetheless have reveal', 'rich phenomenology']\n",
      "['variation', 'in', 'ob jective function']\n",
      "['abrupt variation', 'in', 'ob jective function']\n",
      "['transmission', 'be', 'when unidirectional 2 → 1']\n",
      "['world', 'of', 'largest companies']\n",
      "['world', 'on', 'largest companies']\n",
      "['independent numthe individuals', 'in', 'k']\n",
      "['databases lecture notes', 'in', 'computer science 6323 99']\n",
      "['threshold', 'rive with', 'diﬀerent approaches refs']\n",
      "['e. m. volz', 'journal of', 'royal society interface 9 890']\n",
      "['0.1 1 online version', 'in', 'colour']\n",
      "['0.1 1 online version', 'in', 'colour']\n",
      "['0.1 1 online version', 'in', 'colour']\n",
      "['online version', 'in', 'colour']\n",
      "['online version', 'in', 'colour']\n",
      "['online version', 'in', 'colour']\n",
      "['online version', 'in', 'colour']\n",
      "['parallel version', 'with', 'interventions']\n",
      "['heterogeneity', 'in', 'nodes activities']\n",
      "['dynamical properties', 'in', 'activity']\n",
      "['coverage', 'remain below', 'threshold c']\n",
      "['epidemic decreases', 'with', 'decreasing α']\n",
      "['random variable', 'with', 'moment generating']\n",
      "['modeling relapse', 'in', 'infectious diseases']\n",
      "['a/2 -11-1-2012-0001 naframework ropean social fund', 'in', 'tional excel lence program']\n",
      "['moreno', 'outbreak in', 'complex heterogeneous networks']\n",
      "['chaotic regimes', 'be', 'characterized respectively']\n",
      "['sparsity', 'induce', '16 priors']\n",
      "['m terms', 'proportional to', 'm − 1']\n",
      "['changing climate', 'under disease', 'ed']\n",
      "['well-established theorems', 'in', 'matrix analysis']\n",
      "['according de ﬁnition', 'above ξr', 't']\n",
      "['dark gray region', 'in', 'fig 3']\n",
      "['dark gray region', 'in', 'box']\n",
      "['dark gray region', 'in', 'fig 5']\n",
      "['symbol φ', 'denote', 'empty set']\n",
      "['speciﬁc elements', 'in', 'matrices']\n",
      "['speciﬁc elements', 'in', 's']\n",
      "['realizations', 'obtain', 'curve']\n",
      "['incubation', 'be', 'included separately']\n",
      "['system', 'control over', 'whole campaign period']\n",
      "['14 ζ t ti', 'xt with', 'ζ t i i']\n",
      "['ebola virus transmission', 'in', 'montserrado']\n",
      "['higher order nonlinearities', 'immigration of', 'infectives']\n",
      "['p', 'branch of', 'endemic steady states']\n",
      "['known viral proﬁles', 'in', 'populations']\n",
      "['known viral proﬁles', 'in', 'populations']\n",
      "['three exclusive stochastic moves', 'in', '∞ 0 nonnegative']\n",
      "['phase diagram', 'show in', 'fig 1']\n",
      "['phase diagram', 'give in', 'fig 2']\n",
      "['cluster', 'in', 'link percolation theory']\n",
      "['200 node smal l-world network', 'with', 't']\n",
      "['contact', 'with', 'zombie']\n",
      "['comparison', 'with', 'prevaccination era']\n",
      "['optimal path', 'in', 'random weighted networks']\n",
      "['s. j. clark', 'annals of', 'applied statistics']\n",
      "['interesting connection', 'with', 'its 2n-state nonlinear mean-ﬁeld approximation']\n",
      "['mind typical rural structures', 'with', 'return']\n",
      "['linear ﬁt correlation coeﬃcient r', 'give', '− 0.989']\n",
      "['tuple', 'ject to', 'normalization condition']\n",
      "['systems', 'less vulnerable to', 'disease spreading']\n",
      "['system', 'more fragile to', 'disease spreading']\n",
      "['system', 'more resilient to', 'disease spreading']\n",
      "['systems', 'more prone to', 'disease invasion']\n",
      "['numericcitation barlow re', 'theory of', 'reliability']\n",
      "['n', 'thus constant for', 'virus type']\n",
      "['critical point', 'give by', '1 − wc']\n",
      "['current situation', 'in', 'greece']\n",
      "['available immunization resources', 'in', 'medical institutes']\n",
      "['models', 'with', 'unidirectional dynamics']\n",
      "['pext', 'non-trivial at_time', 'peculiar topological structure']\n",
      "['local clustering', 'of box-plot', 'fig 5']\n",
      "['map h', 'in', 'set s ∈ rn 0n ≤ s']\n",
      "['random regular graphs', 'on b', 'µ']\n",
      "['7 updating', 'for clive', 'primary hiv/aids practical guide care']\n",
      "['ﬁniteness', 'also play', 'crucial role']\n",
      "['ma jor breakthrough', 'in', 'theoretical computer science']\n",
      "['b o i l i t y p 1', 'number from', 'march 12 2003']\n",
      "['pr increases', 'in', 'clockwise direction']\n",
      "['s1', 'in', 'supplementary information']\n",
      "['strong economic ties', 'with', 'greece']\n",
      "['β', 'in', 'ucm networks']\n",
      "['nodes', 'have', 'low local clustering']\n",
      "['regular lattice', 'swn with', 'k']\n",
      "['regular lattice', 'swn with', 'k']\n",
      "['crises', 'with', 'small magnitude']\n",
      "['numericcitation eubank s. network', 'base model of', 'infectious disease spread']\n",
      "['numericcitation eubank s. network', 'model of', 'infectious disease spread']\n",
      "['4000 world corporations', 'with', 'highest turnover']\n",
      "['12 jan 04apr 04jul 04oct 04jan 05apr 05jul 05oct 05jan 06apr 06jul 06oct 06jan 07apr 07jul 07oct 07jan 08apr 08jul 08oct 08jan 09apr 09jul 09oct 09jan 10apr 10jul 10oct 10jan 11apr 11jul 11oct 11jan 12apr 12jul 12oct 12jan 13apr 13jul 130246810number cholera cases', 'in', 'mathbaria − 2 −']\n",
      "['table 1', 'show', 'expected motif allocation per node']\n",
      "['economic crisis supplementary information a. garas1', 'of spread', '54124 thessaloniki greece']\n",
      "['µ', 'at ﬁxed', 'γ']\n",
      "['i', '¯', 't']\n",
      "['zeroth-order', 'in', 'µ']\n",
      "['zeroth-order', 'in', 'µ']\n",
      "['i', 'show in', 'b']\n",
      "['s', '∈', 'v']\n",
      "['u', 'in', 'x t']\n",
      "['u', 'of tint', 'u']\n",
      "['u', 'in', 'y t +1']\n",
      "['˜g', 'in supernode', 's ′']\n",
      "['s', 'be', 'k-jordan']\n",
      "['v', 'of tint', 'v']\n",
      "['u', 'in', 'x t']\n",
      "['u', 'of tint', 'u']\n",
      "['u', 'in', '˜x t']\n",
      "['x t', 'in tint', 'u']\n",
      "['x', 'slot to', 't']\n",
      "['t', 'be', 'none-increasing']\n",
      "['y 2 0', 'in sn', 't']\n",
      "['∀ t ≥ 0', 't for', 'λ']\n",
      "['i', 'o', 't']\n",
      "['n ≥ 0', 'for ≥', 'ρ']\n",
      "['o', 'from y', 'u']\n",
      "['i', 'o', 't t']\n",
      "['i', 'o there', 't t']\n",
      "['ω ∈ f', 'for m2', 'd']\n",
      "['ω ∈ f', 'for m2', 'p φ d1/3']\n",
      "['u', 'ω for', 'ω ∈ ω']\n",
      "['x ∈ b', 'for pλ', 'd']\n",
      "['i', 'for ln', 'x']\n",
      "['x', 'ln for', 'i']\n",
      "['σ', 'deﬁne for', 'x']\n",
      "['≥', 'be', 'self-avoiding']\n",
      "['x', 'for ρ', 'x y']\n",
      "['f', 'as f', 'β']\n",
      "['ai', 'probability of', 'ai → u i']\n",
      "['s-node', 'become', 'a-node']\n",
      "['insets', 'in', 'bottom subﬁgures']\n",
      "['u-nodes', 'enter', 'i-state']\n",
      "['u-nodes', 'quickly enter', 'i-state']\n",
      "['latent period', 'take into', 'account']\n",
      "['r0', 'in', 'group']\n",
      "['latent period', 'take into', 'account']\n",
      "['spread', 'with', 'latent period ¶ t sn']\n",
      "['¶ t', 'in', 'asn']\n",
      "['latent period ¶ t sn', 't in', 'asn']\n",
      "['latent period ¶ t sn', 't in', 'nn − bin']\n",
      "['¶ t', 'in', 'nn − bin']\n",
      "['nn − mhn', 't in', 'mhn − bin']\n",
      "['m ¶ t', 'in', 'mhn − bin']\n",
      "['p ∼ k', 'with', 'g']\n",
      "['b', 'in', '4 o o i i t t']\n",
      "['b', 'in', 'high-w region']\n",
      "['jump', 'in', 'attack ratio']\n",
      "['spread', 'with', 'latent period']\n",
      "['i', 'be', 'immobile']\n",
      "['pro ﬁle', 'in', 'fig 2']\n",
      "['curve', 'in', 'fig 3']\n",
      "['r.m anderson', 'disease of', 'humans']\n",
      "['people', 'on', 'susceptibility']\n",
      "['outbreaks', 'accelerate', 'information dissemination']\n",
      "['people', 'have', 'susceptibility']\n",
      "['people', 'on', 'susceptibility']\n",
      "['diﬀerent colors', 'in', 'fig 1']\n",
      "['eight communities', 'distinguish by', 'diﬀerent colors']\n",
      "['communities', 'distinguish by', 'diﬀerent colors']\n",
      "['sources', 'in', 'diﬀerent communities']\n",
      "['colors', 'indicate', 'base 10']\n",
      "['permanent immunity', 'with', 'probability λ']\n",
      "['sources', 'apply', '105 realizations']\n",
      "['spreaders', 'with', 'probability βr']\n",
      "['stiﬂers', 'with', 'probability λ']\n",
      "['inset', 'present', 'their correlation coeﬃcient']\n",
      "['theoretical predictions', 'highly correlate with', 'simulated susceptibility']\n",
      "['theoretical predictions', 'be', 'highly correlated']\n",
      "['slope k', 'however vary as', 'βs increases']\n",
      "['phenomenon', 'also reﬂected in', 'd']\n",
      "['collection', 'number in', 'set υ']\n",
      "['s', 'of value', 'base 10']\n",
      "['diﬀerent fraction f', 'for r', 'f']\n",
      "['relationship', 'in', 'fig 6']\n",
      "['βr', 'be', 'set']\n",
      "['diﬀerent fraction f', 'for r', 'f']\n",
      "['barab', 'asi', 'nature 406 378']\n",
      "['advances', 'in', 'neural information processing systems 539']\n",
      "['powers', 'in g', 'z']\n",
      "['equation', 'be', 'discrete']\n",
      "['t → ∞', 'as hr', 't']\n",
      "['eq', 'in φ', 'ξ']\n",
      "['tion φ', 'in', 'eq']\n",
      "['taylor series', 'in', 'conjugate variables']\n",
      "['present work', 'fall in', 'latter category']\n",
      "['high accuracy', 'in', 'd']\n",
      "['site x', 'of neighborhood', 'nearest neighbor sites']\n",
      "['site x', 'in', 'eqs']\n",
      "['site', 'in', 'd-dimensional cubic latz −']\n",
      "['t → ∞', 'as pt', 'i']\n",
      "['damped oscillatory pattern', 'in', 'figs 7']\n",
      "['i', '→', 'r ni']\n",
      "['∼ s', '− τ so', 'ps ∼ s1 − τ']\n",
      "['∼ s', '− τ so that', 'ps ∼ s1 − τ']\n",
      "['properties', 'in', 'detail']\n",
      "['i', '→ r with', '1 − c']\n",
      "['dashed lines', 'be', 'branches']\n",
      "['dashed lines', 'be', 'unstable branches']\n",
      "['n − k ′', '− for', 'n']\n",
      "['rdi', 'n for', 'n']\n",
      "['r nodes', 'in', 'numbers matching']\n",
      "['recovereds', 'with', 'degree 0']\n",
      "['a. fluctuations', 'number of', 'infectives']\n",
      "['fluctuations', 'in', 'infectives']\n",
      "['oscillations', 'could', 'could ignored']\n",
      "['peaks', 'in', 'infective fraction']\n",
      "['electrical engineering', 'department at', 'santa barbara']\n",
      "['computation', 'university of', 'california']\n",
      "['entries', 'be', 'x1']\n",
      "['diagonal entries', 'be', 'x1']\n",
      "['weighted digraph', 'with', 'adjacency matrix a']\n",
      "['x', 'n for', 't']\n",
      "['nagumo', 'of', 'theorem']\n",
      "['t ∈', 'for x', 't']\n",
      "['x', 'with x', 't']\n",
      "['˙x β', 'in', '− diag']\n",
      "['− γ', 'in', 'x']\n",
      "['nagumo', 'of', 'theorem']\n",
      "['x ∗', 'be', 'γ ax']\n",
      "['∗', 'be', 'γ']\n",
      "['vmax ≤ vmax', 'in', '− diag']\n",
      "['ˆa', 'stable', 'schur']\n",
      "['β γ y', 'by rn', 'k']\n",
      "['taylor expansions', 'in', 'parts']\n",
      "['iiib', 'in', 'part']\n",
      "['y', 'be', 'scalar']\n",
      "['y ∗', 'assume', 'y']\n",
      "['intersection', 'with', 's']\n",
      "['t ≥ τ', 'for x', 't']\n",
      "['t ∈', 'for 0', '0 τ']\n",
      "['s', 'for x', 'tra jectory s']\n",
      "['t ≥ 0', 'for δs', 't']\n",
      "['adjacency matrix a', 'be', 'binary']\n",
      "['singapore 138671', 'c', 'mile ˇsiki']\n",
      "['data', 'with', 'o']\n",
      "['k', 'be', 'big enough']\n",
      "['calculation cost', 'proportional to', 'kma x 4']\n",
      "['minimum precision', 'be', '64 bits']\n",
      "['algorithms', 'implement in', 'c']\n",
      "['star', 'republic of', 'singapore']\n",
      "['barab', 'asi', 'rev mod']\n",
      "['barab', 'asi', 'science 286 508 1999']\n",
      "['c', 'a', 'h. ˇstefan ˇci']\n",
      "['imprecision', 'with', 'β']\n",
      "['k-shell', 'measure of', 'kitsak et']\n",
      "['p', 'with', 'β']\n",
      "['ms', 'number with', 'β']\n",
      "['s', 'in', 'g']\n",
      "['literature', 'have', 'k']\n",
      "['topic', 'in', 'castellano']\n",
      "['betweenness', 'in', 'kitsak et']\n",
      "['place', 'remove node into', 'shell vs']\n",
      "['1 technically', 'work of', 'antal et']\n",
      "['betweenness', 'be', 'medial']\n",
      "['i', 'of distance', 'metric counts']\n",
      "['q', 'for distance', 'metric counts']\n",
      "['package', 'in', 'r']\n",
      "['imprecision', 'with', 'β']\n",
      "['imprecision', 'with', 'β']\n",
      "['imprecision', 'with', 'β']\n",
      "['imprecision', 'with', 'β']\n",
      "['imprecision', 'with', 'β']\n",
      "['e astrof ısica da universidade de lisboa', 'institute of', 'vocational education']\n",
      "['e astrof ısica da universidade de lisboa', 'c8', 'ediﬁc']\n",
      "['scientist', 'in', 'fact']\n",
      "['ordinary diﬀerential equations numericcitation', 'of system', 'dy dt']\n",
      "['constants ni', 'with x', 'y']\n",
      "['traveling wave ordinary diﬀerential equations', 'in', 'usual spatial']\n",
      "['latter', 'eliminate y from', 'eqs']\n",
      "['x', 'of variation', 'dotted curve']\n",
      "['β', 'for z', 't']\n",
      "['x', 'of variation', 'dotted curve']\n",
      "['ψ', 'with', 'ψ0']\n",
      "['diﬀerent order approximations', 'with y', 't']\n",
      "['s c i', 'be', 'y h p']\n",
      "['multilayer networks', 'network of', 'networks']\n",
      "['actors', 'in', 'layers']\n",
      "['pi ∼ k', 'with', 'λa']\n",
      "['w', 'in', 'blue']\n",
      "['kmin', 'with network', 'λb']\n",
      "['λa', 'with network', 'λb']\n",
      "['jacobian', 'with', 'ji']\n",
      "['plot', 'with', 'λa']\n",
      "['λb', 'with', 'kmin']\n",
      "['multiplex networks', 'with', 'βc ≈ 0.043']\n",
      "['w', 'with', 'wc']\n",
      "['kmin', 'in', 'fig 3']\n",
      "['kmin', 'in', 'fig 3']\n",
      "['λb', 'with', 'kmin']\n",
      "['tw', 'with', 'λa']\n",
      "['synchronization mechanism', 'as a.', 'macri p. a']\n",
      "['reaction-diﬀusion processes', 'in', 'scale-free networks']\n",
      "['failures', 'in', 'interdependent networks']\n",
      "['multiplexity-facilitated cascades', 'in', 'networks']\n",
      "['coevolution multiplexity', 'in', 'multiplex networks']\n",
      "['emergence', 'feature from', 'multiplexity']\n",
      "['n', 'm. of', 'small-world networks']\n",
      "['scenario', 'in', 'other layer']\n",
      "['λb', 'with', 'kmin']\n",
      "['multiplex networks', 'with', 'βc ≈ 0.043']\n",
      "['plot', 'with', 'λa']\n",
      "['curves', 'in', 'fig 5']\n",
      "['isolation', 'in', 'layers']\n",
      "['λb', 'with', 'kmin']\n",
      "['tw', 'with', 'tr']\n",
      "['isolation', 'in', 'layers']\n",
      "['w', 'in', 'scenario']\n",
      "['isolation', 'in', 'layers']\n",
      "['b', 'be', 'life-threatening']\n",
      "['b', 'be', 'life-threatening']\n",
      "['fulminant hepatitis', 'require', 'liver transplantation']\n",
      "['albeit', 'can vary from', '30-180 days']\n",
      "['kµ', 'sum of', 'φ p β']\n",
      "['hepatitis b virus', 'in', 'china']\n",
      "['ﬁnite-si ze networks', 'on ﬁcation', 'chengdu']\n",
      "['lot', 'in', 'classic susceptible-infected-recovered']\n",
      "['s c i', 'be', 'y h p']\n",
      "['networks', 'with', 'power-law scaling p']\n",
      "['sec', 'give', 'conclusions']\n",
      "['sec 5', 'give', 'conclusions']\n",
      "['effective simulated identi ﬁcation measure', 'in', 'simulations']\n",
      "['simulations', 'use', 'synchronous updating scheme']\n",
      "['rrn', 'with', 'constant degree k']\n",
      "['triangleups', 'in', 'circles']\n",
      "['large k', 'for ≃', 'k −']\n",
      "['k −', 'in', 'ref']\n",
      "['further comparison', 'with', 'susceptibility measure']\n",
      "['λ ∆ p', 'λs', 'i r c']\n",
      "['λ ∆ p', 'be', 'closer']\n",
      "['rrn', 'with', 'k']\n",
      "['fig 2', 'in hki', 'b']\n",
      "['hki −', 'in', 'fig 2']\n",
      "['λ λc', 'distribution of', 'outbreak sizes']\n",
      "['power-law distribution', 'with', 'exponent -1.5']\n",
      "['outbreak sizes', 'of distribution', 'λ']\n",
      "['λ', 'for distribution', 'λ']\n",
      "['complete lump', 'in', 'distribution']\n",
      "['susceptibility', 'in', 'fig 3']\n",
      "['dirac delta function', 'in', 'theory']\n",
      "['rc', 'with', 'p']\n",
      "['applications', 'in', 'section']\n",
      "['structural cutoff', 'in', 'figs 4']\n",
      "['largest degree kmax', 'variation with', 'ref']\n",
      "['whose variation', 'with', 'ref']\n",
      "['small outbreak sizes', 'with', 'ρr ≤ rc']\n",
      "['contents 9 n', 'depend strongly on', 'γ']\n",
      "['sfns', 'with', 'structural cutoff']\n",
      "['variations', 'with', 'γ']\n",
      "['sfns', 'with', 'structural cutoff']\n",
      "['sfns', 'with', 'structural cutoff']\n",
      "['kmax', 'be', 'maximum degree']\n",
      "['conclusions', 'in', 'summary']\n",
      "['threshold', 'with', 'λ']\n",
      "['power-law distribution', 'with', 'exponent -1.5']\n",
      "['λ', 'distribution of', 'outbreak sizes']\n",
      "['lump', 'in', 'simulated scattergram']\n",
      "['simulated outbreak probability', 'in', 'theory']\n",
      "['effectiveness', 'analyze', 'critical distribution']\n",
      "['cases', 'in', 'asynchronous updating scheme']\n",
      "['foundation', 'program by', 'uestc']\n",
      "['korea', 'of foundation', 'nrf']\n",
      "['appendix', 'with', 'steady-state']\n",
      "['distribution', 'in', 'bond percolation']\n",
      "['general network', 'with', 'arbitrary degree distribution pk']\n",
      "['g0', 'in', 'addition']\n",
      "['data', 'from discovery', 'acm tkdd']\n",
      "['2004 phys', 'as m', 'pastor-satorras r']\n",
      "['2004 phys', 'as m', 'diaz-guilera a']\n",
      "['explicit movement', 'with model', 'abm']\n",
      "['directional movement', 'in', 'one-dimensional space']\n",
      "['space', 'be', 'dimensional']\n",
      "['i', 'have', 'activity']\n",
      "['infection', 'always take place within', 'radius']\n",
      "['movies', 'in', 'supplemental materials']\n",
      "['i', 'be', 'agent']\n",
      "['n', 'be', 'varied']\n",
      "['bn', 'be', 'binary']\n",
      "['convention', 'use in', 'work']\n",
      "['k → ∞', 'type of', 'network']\n",
      "['p ≈ k − 2 − γ', 'with', 'γ']\n",
      "['investigations', 'in', 'closed community']\n",
      "['news', 'in', 'internet']\n",
      "['bernoulli raffle', 'with', 'probability c']\n",
      "['survivors', 'of variance', 'v']\n",
      "['sum', 'in', 'row']\n",
      "['networks', 'be', 'equivalent']\n",
      "['networks', 'be', 'equivalent']\n",
      "['lm', 'with', 'sm']\n",
      "['electrical engineering', 'institute of', 'technology']\n",
      "['confusion', 'facilitate', 'reading']\n",
      "['proofs', 'postpone to', 'appendix']\n",
      "['2 s', 'be', 'ep idem ics fig 1']\n",
      "['s', 'be', 'ep idem ics fig 1']\n",
      "['wavy arrow', 'represent', 'transition']\n",
      "['origin', 'be', 'ﬁxed']\n",
      "['t', 'in', 'βa p']\n",
      "['origin', 'be', 'unique']\n",
      "['βa', 'in λmax', '1 − δ']\n",
      "['λmax', 'in', 'βa']\n",
      "['ξi ∂ xj', 'in', 'g']\n",
      "['nj', 'in', 'g']\n",
      "['→ rn', 'with', 'ξ']\n",
      "['βa', 'in λmax', '1 − δ']\n",
      "['λmax', 'in', 'βa']\n",
      "['p', 'in', 'βa p t']\n",
      "['βa p t', 'in p', '1 − δ']\n",
      "['βa p t', 'in', 'βa']\n",
      "['βa p t', 'in p', 't']\n",
      "['βa', 'in time', '1 − δ']\n",
      "['λmax', 'for time', '1 − δ']\n",
      "['5 2.3.2 par tial ordering µx ≥', 'in', 'section']\n",
      "['u', 'by u', 'r']\n",
      "['r1', 'u by', 'u']\n",
      "['equation', 'in n', 'y p ξi']\n",
      "['0,1 n', 'in', 'equation above']\n",
      "['lemma 2.7', 'also true for', 'm']\n",
      "['lemma 2.7', 'true for', 'm']\n",
      "['βa', 'in λmax', '1 − δ']\n",
      "['λmax', 'in', 'βa']\n",
      "['i ∈ nj', 'in', 'g']\n",
      "['proof', 'for p', 'n']\n",
      "['wavy arrow', 'represent', 'transition']\n",
      "['1 − δ', 'in', 'βa']\n",
      "['1 − δ', 'in', 'βa']\n",
      "['origin', 'be', 'ﬁxed']\n",
      "['pr', 'relationship between', 'probabilities']\n",
      "['wavy arrow', 'represent', 'transition']\n",
      "['presence', 'in i', 't']\n",
      "['m', 'in', 'p ∗ s βa']\n",
      "['m', 'in', '− θp ∗ s βa']\n",
      "['t', 'in', '0n 0n × n p ∗ r 1n']\n",
      "['2 u', 'in', 'βa']\n",
      "['1 − δ', 'in', 'βa']\n",
      "['vaccination', 'be', 'dominant']\n",
      "['i', '0 since', 'x ∗ ∈ u']\n",
      "['i ∈', 'for ψi', 'x ∗']\n",
      "['node', 'in', 'k0']\n",
      "['t', '∗ go to', 'inﬁnity']\n",
      "['p', 'converge to', 'x ∗']\n",
      "['ni', 'j for', 'j 1']\n",
      "['ni', 'x', 'small 0 such']\n",
      "['property', 'by hold', 'a 87']\n",
      "['v', 'in λmax', '1 − δ']\n",
      "['λmax', 'in', '∗']\n",
      "['∗', 'in λmax', '1 − δ']\n",
      "['λmax', 'in', 'v']\n",
      "['1 − p ∗ r u', 'in', 'n × 2n']\n",
      "['γ', 'proof of', 'theorem 4.3']\n",
      "['theorem 4.3', 'of proof', 'a 120']\n",
      "['pi pi', 'in', 'βa pi']\n",
      "['βa pi', 'in pi', 't']\n",
      "['βa pi', 'in pi', '1 − δ']\n",
      "['1 − δ', 'in', 'p ∗ s βa']\n",
      "['statement b', 'also follow from', 'upper bounding eq']\n",
      "['outbreaks', 'in', 'developing countries']\n",
      "['sheep', 'be', 'essential source']\n",
      "['sheep', 'essential source for', 'human food']\n",
      "['herds', 'form', 'clusters']\n",
      "['systems', 'often display', 'spatiotemporal chaotic behaviour']\n",
      "['shortcuts', 'with', 'length k']\n",
      "['individuals', 'with', 'shortcuts']\n",
      "['i', '−', 'q3 i']\n",
      "['i', '−', 'q2r']\n",
      "['swn', 'with', 'l']\n",
      "['swn', 'describe', 'interactions']\n",
      "['coupled systems', 'in zanette', 'cond-mat/0003174']\n",
      "['k', 'with', 'degree k']\n",
      "['nr', 'k with', 'degree k']\n",
      "['e ′ ⊆ e', 'on y', 't']\n",
      "['event en', 'hold', 'd1']\n",
      "['assumptions k', 'further', 'k n → αsλ']\n",
      "['assumptions k', 'further', '0 kns']\n",
      "['susceptibles', 'with', 'non-zero degree']\n",
      "['uniform summability', 'in', 'remark 2.2']\n",
      "['g ∗', 'bind as', 'n → ∞']\n",
      "['justiﬁcation', 'in', 'proof below']\n",
      "['hi', 'with ∈', '0 1']\n",
      "['unique θ ∞ ∈', 'with', 'hi']\n",
      "['deterministic limits', 'in', 'theorems']\n",
      "['hi', 'with ∈', '0 1']\n",
      "['unique θ ∞ ∈', 'with', 'hi']\n",
      "['hi', 'with ∈', '0 1']\n",
      "['unique θ ∞ ∈', 'with', 'hi']\n",
      "['t0', 'on xs', '∞']\n",
      "['threshold', 'in', 'r0']\n",
      "['giant component', 'in', 'g']\n",
      "['v2', 'denote', 'ordered component sizes']\n",
      "['unique giant component', 'with', 'n − op']\n",
      "['n − op susceptibles', 'in', 'giant component']\n",
      "['k', 'vaccinate with', '0 1']\n",
      "['p ∞', 'mean', 'τ ∧ τ ∗']\n",
      "['gronwall', 'have', 's ′']\n",
      "['jumps', 'in', 'ms']\n",
      "['jumps', 'in', 's ′ τ']\n",
      "['jump', 'in', 's ′ τ']\n",
      "['doob', 'have', 'l2']\n",
      "['z τ gronwall', 'have', 'inequality yields']\n",
      "['jumps', 'in', 'x ′ τ']\n",
      "['jumps', 'in', 'mx']\n",
      "['infective vertex', 'with', 'k free half-edges']\n",
      "['gronwall', 'have', 'inequality']\n",
      "['0 knr', 'µr by', 'd4']\n",
      "['3.16 nβ eτ', 'in', 'mr']\n",
      "['d5', 'by mr', 'τ']\n",
      "['x ′ i', 'of convergence', 'n − x ′ s']\n",
      "['τ ∗', 'bind', 'w.h.p']\n",
      "['h', 'be', 'concave']\n",
      "['pi', 'hence continuous on', 'θ ∞ 1']\n",
      "['limit', 'equal', 'θ ∞']\n",
      "['satisﬁes', 'ˆτ', 't']\n",
      "['inverse', 'follow', 'ˆa']\n",
      "['c adl ag martingale', 'with', 'mi']\n",
      "['doob', 'have', 'inequality']\n",
      "['distribution', 'in', 'c']\n",
      "['n', 'in', 'd']\n",
      "['d', 'in n', '0 ∞']\n",
      "['con-n → ∞ ˆit', 'to mi', 't']\n",
      "['1 − ˆit1 − vs', 'w.h.p for', 't']\n",
      "['ˆit', 'be', 'non-decreasing']\n",
      "['t1', 'in', 'summary']\n",
      "['statement', 'be', 'immediate']\n",
      "['unique root', 'in', '0 1']\n",
      "['0 τ ˆτ ∞', 'of →', '0 ˆτ ∞']\n",
      "['process', 'of τ', 't']\n",
      "['convention', 'for part', 'c']\n",
      "['right-hand side', 'diverge as', 'ε → 0']\n",
      "['t ∈', 'over ˆτ', 't']\n",
      "['distribution', 'in', 'd']\n",
      "['it0', 'in', 'distribution along original sequence']\n",
      "['ˆit it0', 'a.s. for', 't1 0']\n",
      "['t', 'for n', 'vs']\n",
      "['xs ,0 − xs', '∞ of', 'half-edges']\n",
      "['t0', 'on op', 'n']\n",
      "['ns', 'vertex of', 'degree k']\n",
      "['infective vertex', 'with', 'ℓ']\n",
      "['normalization urn', 'with', 'n balls']\n",
      "['i → ∞', 'as p', '0 ˆzi n']\n",
      "['ni', 'individual for', 'k']\n",
      "['ni', 'individual of', 'type k']\n",
      "['proofs', 'in', 'section']\n",
      "['w d − → cw', 'with', 'convergence']\n",
      "['method', 'by that', 'w1 l']\n",
      "['e w m', 'be', 'd']\n",
      "['p p l', 'in', '¯ g ∗']\n",
      "['diﬀerential equation', 'to ˜θt', 't']\n",
      "['pk ∼ k − α − 1', 'with', 'α 6 2']\n",
      "['≍ xα − 2', 's', '1 − x']\n",
      "['x → 0', 'as s', '1 − x']\n",
      "['r0', 'be', 'ﬁnite']\n",
      "['t0', 'in', 'example 7.1']\n",
      "['mathematics', 'of department', 'sweden e-mail address']\n",
      "['∼ svante', 'university of', 'london']\n",
      "['london', 'of university', 'uk']\n",
      "['cbp', 'be', 'peculiar']\n",
      "['b', 'b into', 'c']\n",
      "['∼ c 4π n', 'with', 'c']\n",
      "['good agreement', 'with', 'simulations']\n",
      "['100101102103104105101102103104105106107108n na nb 100101102103104105106101102103104105106107108n na nb nc 7', 'average of', 'outbreaks']\n",
      "['l', 'with l', 'm − 1']\n",
      "['l', 'with', 'l']\n",
      "['universality', 'in', 'nonequilibrium lattice systems']\n",
      "['γ', 'with', 'added triangles']\n",
      "['curves', 'in', 'fig 3c']\n",
      "['weak trends', 'in', 'directions']\n",
      "['vertices', 'of pair', 'edges']\n",
      "['brent', 'have', 'method locate']\n",
      "['distance measures', 'in', 'graphs']\n",
      "['name efficiency', 'in', 'ref']\n",
      "['pastor-santorras r', 'spread in', 'scale-free networks']\n",
      "['buckley f', 'distance in', 'graphs']\n",
      "['harary f distance', 'in', 'graphs']\n",
      "['p', 'vulnerability of', 'complex networks']\n",
      "['latora v', 'behavior of', 'small-world networks']\n",
      "['error bars', 'in', '𝜏']\n",
      "['2complex lab', 'university of', 'electronic science']\n",
      "['g', 'be', 'promising']\n",
      "['rate β', 'recover with', 'recovering rate µ']\n",
      "['derivation', 'in', 'materials']\n",
      "['elements', 'in', 'columns']\n",
      "['node', 'have', 'inﬂuence']\n",
      "['kendall', 'have', 'tau τ 36']\n",
      "['kendall', 'by', 'tau']\n",
      "['kendall', 'by', 'tau']\n",
      "['kendall', 'have', 'tau']\n",
      "['kendall', 'by', 'tau 36']\n",
      "['sgn', 'be', 'piecewise']\n",
      "['spain', 'of virgili', 'urv']\n",
      "['protein 53', 'version of', 'proteome-scale map']\n",
      "['superspreaders', 'in', 'real-world social media']\n",
      "['china', 'of foundation', 'shanghai']\n",
      "['special appointment', 'of professor', 'eastern scholar']\n",
      "['regions', 'with', 'poor medical care systems']\n",
      "['ma', 'in', 'r e su lt s']\n",
      "['ma', 'in', 'section']\n",
      "['notation', 'in', 'numericcitation']\n",
      "['path', 'be', 'robust']\n",
      "['machine', 'with', '1.8 gb memory']\n",
      "['algorithms', 'implement in', 'python 2.7']\n",
      "['ri', 'much faster than', 'dmp']\n",
      "['ri', 'therefore outperform', 'dmp']\n",
      "['tree', 'root in', 'v']\n",
      "['inequality', 'hold for', 'k ≤ n']\n",
      "['l', 'node at', 'bottom survived']\n",
      "['φ u', 'in', 'event 2']\n",
      "['adler', 'of', '1993 conjecture']\n",
      "['integer lattice zd', 'in', 'd dimensions']\n",
      "['particles', 'be', 'initially red']\n",
      "['site zd', 'in', 'generation']\n",
      "['variance parameter σ2', 'with xt', 'super-brownian motion']\n",
      "['d', 'in path', '0 ∞']\n",
      "['xt', 'have', '0 ∞']\n",
      "['proposition 2.15', 'in', 'etheridge']\n",
      "['t', 'x for', 't']\n",
      "['xt', 'be', 'singular']\n",
      "['reader', 'for', 'convenience']\n",
      "['x0', 'in', 't ≥ 0']\n",
      "['sugitani', 'have', 'theorems']\n",
      "['super-brownian motion', 'with', 'variance parameter σ2']\n",
      "['sufﬁcient condition s', 'give below in', 'proposition 1']\n",
      "['xt', 'super-brownian motion in', 'dimension d']\n",
      "['proposition 35', 'in', 'lalley']\n",
      "['super-brownian motion', 'with', 'variance parameter σ2']\n",
      "['distributional', 'prove in', 'various cases']\n",
      "['initial fn α µn', 'with', 'compact support']\n",
      "['α', 'in', 'dimensions d 5']\n",
      "['loca l t ime', 'in', 'd 3 2.1']\n",
      "['gauss kernel', 'in', 'rd']\n",
      "['γ ≤ 1', 'for 1', 'pn x −']\n",
      "['monotone', 'in', 'ψ']\n",
      "['origin', 'in', 'zd']\n",
      "['origin', 'include in', 'collection']\n",
      "['β', 'zd as', '1 φl']\n",
      "['h ≥ 2', 'for 35', 'n ∈']\n",
      "['h ≥ 2', 'for 35', 'n']\n",
      "['h ≥ 2', 'for 35', 'κh']\n",
      "['κ1', 'in', 'fact']\n",
      "['k', 'for b', '√ k']\n",
      "['assumption', 'by b', '0 r']\n",
      "['positive integers', 'of tuple', 'h1']\n",
      "['claim', 'in', 'part a']\n",
      "['proposition 1', 'in', 'sugitani']\n",
      "['x ∈ zd', 'for y', 'ψk t']\n",
      "['ψk t − ψk s 2ε', 'for ε', 't ≤ δ']\n",
      "['t ≥ δ', 'for t', 'x']\n",
      "['c', 'in compact', 'µkgkt']\n",
      "['qσ2 t', 'in', 'c']\n",
      "['c', 'in t', 'x']\n",
      "['k2 − d/2', 'verify', 'ascoli-arzela criterion']\n",
      "['spitzer', 'proposition on', 'p72']\n",
      "['t ≥ δ', 'for time', 'µkgkt']\n",
      "['y ∈ zd', 'with', 'y']\n",
      "['k', 'for y', 'c1 k1/6']\n",
      "['f', 'have', 'support']\n",
      "['f', 'have', 'compact support']\n",
      "['decreasing functions', 'in', 'compact support']\n",
      "['sup i1', 'in', 'n xk']\n",
      "['proposition 3', 'in', 'lalley']\n",
      "['proposition 28', 'in', 'lalley']\n",
      "['site x', 'at n', 'x']\n",
      "['collisions', 'of n', 'x']\n",
      "['58 xn ≤ n α t xx', 'position of', 'initial particles']\n",
      "['α ≤ 2/3', 'since o', 'n × n α']\n",
      "['α ≤ 1/2', 'since o', 'n × n α']\n",
      "['dimension 2', 'in pn', 'x']\n",
      "['pairs ̺', 'of n', 'x']\n",
      "['dimension 2', 'in', 'dimension 3']\n",
      "['increments', 'in', 't']\n",
      "['n', 'in', 'equation']\n",
      "['equation', 'in n', 'x']\n",
      "['quantities', 'furthermore ∆', 'x']\n",
      "['equation', 'in', 'lalley']\n",
      "['lalley', 'of corollary', 'xn xx ∆ n n ̺n n z z θn']\n",
      "['watanabe', 'have', 'λn n']\n",
      "['watanabe', \"'s n\", 'x']\n",
      "['dimension 2', 'in', 'dimension 3']\n",
      "['λj', 'with', 'λi']\n",
      "['h1', 'for φl', 'u']\n",
      "['boston', 'in adler', 'superprocess local l ocal times']\n",
      "['boston', 'in adler', 'r. j']\n",
      "['birkhäuser adler', 'in', 'boston']\n",
      "['dawson', 'in', 'dence']\n",
      "['etheridge', 'introduction to', 'superprocesses']\n",
      "['springer-verlag 2nd ed', 'text in', 'mathematics']\n",
      "['graduate texts', 'in', 'mathematics']\n",
      "['berlin', 'in walsh', 'xiv']\n",
      "['walsh', 'in', 'berlin']\n",
      "['lecture notes', 'in', 'math']\n",
      "['mathematics', 'of department', 'canada v6t 1z2 e-mail address']\n",
      "['heterogeneity', 'in', 'tie strengths']\n",
      "['c emergence', 'in', 'ml']\n",
      "['ml', 'in spread', 'blue circles']\n",
      "['sis spreading', 'in', 'ml']\n",
      "['ml', 'in spread', 'blue circles']\n",
      "['heterogeneity', 'in', 'ties strength']\n",
      "['enormous amount', 'in', 'theoretical epidemiology']\n",
      "['network', 'have', 'operation']\n",
      "['computers', 'in', 'p']\n",
      "['communications network g', 'set of', 'computers']\n",
      "['relevant concepts', 'in', 'computational complexity']\n",
      "['instance n', 'of ℓ', 'n']\n",
      "['c', 'be', 'correct']\n",
      "['many problems', 'in', 'np']\n",
      "['hardest problems', 'in', 'np']\n",
      "['γ', 'solve', 'π']\n",
      "['np-hard problems', 'be', 'found']\n",
      "['g edge', 'set', 'e']\n",
      "['u', 'with form', 'u']\n",
      "['edge', 'with', 'ϕ1']\n",
      "['formalism', 'extend period of', 'infectivity']\n",
      "['states', 'of op', 'g ′']\n",
      "['states', 'of op', 'g']\n",
      "['edges', 'in', 'pathset']\n",
      "['links', 'in', 'network']\n",
      "['problems', 'belong to', 'class p-complete numericcitation']\n",
      "['networks', 'with', 'disease latency']\n",
      "['πrel poly', 'be', 'polynomially reducible']\n",
      "['πrel poly', 'polynomially reducible to', 'πi']\n",
      "['i', 'non-trivial pattern of', 'infectivity']\n",
      "['unknowns', 'coeﬃcients of', 'reliability polynomial']\n",
      "['jpn j', 'infect', 'dis']\n",
      "['la', 'epidemic on', 'directed contact networks']\n",
      "['lm', 'percolation on', 'heterogeneous networks']\n",
      "['industrial engineering', 'university of', 'toronto']\n",
      "['numericcitation', 'harm', 'devitt js']\n",
      "['experiments', 'with', 'symbolic algebra environment']\n",
      "['numericcitation gertsbakh ib', 'model of', 'network reliability']\n",
      "['technology zurich', 'of institute', 'eth z ¨ urich']\n",
      "['swiss federal institute', 'institute of', 'bioinformatics']\n",
      "['technology zurich', 'of institute', 'basel institute']\n",
      "['department', 'university of', 'liverpool']\n",
      "['mathematical sciences', 'of department', 'liverpool l69 7zl']\n",
      "['department', 'university of', 'liverpool']\n",
      "['mathematical sciences', 'of department', 'liverpool l69 7zl']\n",
      "['t', 'for e', 'x smes s t']\n",
      "['s0', 'with', 'i 6 j']\n",
      "['imes', 'with rme', '˙rmes']\n",
      "['rmes', 'with', 'imes']\n",
      "['imes', 'with rme', 't']\n",
      "['al l t', 'for sme', 't']\n",
      "['al l t', 'for sme', 't']\n",
      "['t', 'for t', 'then lemma 2']\n",
      "['x ∈', 'for x', '0 1']\n",
      "['t', 'for ˙f2', 't']\n",
      "['t', 'for ˙f2', 'whence f2 s2 t']\n",
      "['t', 'for ˙f1', 'whence f1 s1 t']\n",
      "['t', 'for ˙f1', 't']\n",
      "['t 0', 'for x', 'f2']\n",
      "['acknowledgements r.r.w', 'acknowledge', 'support']\n",
      "['springer lecture notes', 'in', 'statistics']\n",
      "['lecture notes', 'in', 'biomathematics']\n",
      "['mckendrick', 'application to', 'medical problems']\n",
      "['n 1/d', 'in', 'd dimensions']\n",
      "['f', 'be', 'universal']\n",
      "['interaction', 'with', 'distant neighbors']\n",
      "['swn', 'with', 'k']\n",
      "['swn', 'with', 'k']\n",
      "['swn', 'with', 'n']\n",
      "['tow phases', 'appear', 'i']\n",
      "['swn', 'with', 'n']\n",
      "['swn', 'with', 'n']\n",
      "['swn', 'with', 'n']\n",
      "['i', '→', 'r']\n",
      "['immunization', 'be', 'permanent']\n",
      "['m/s 2', 'ﬁnd', 'order parameter p']\n",
      "['site', 'belong to', 'inﬁnite percolating cluster']\n",
      "['ˆa', 'be', 'universal']\n",
      "['u p', 'as expect by', 'arguments']\n",
      "['u p', 'see in', 'ﬁgure 9']\n",
      "['biological populations', 'in', 'space']\n",
      "['t. tom', 'e', 'physica a 389 1142']\n",
      "['school', 'university of', 'london']\n",
      "['direction', 'in', 'future publications']\n",
      "['n m × n m supra-adjacency matrix aiα', 'β of', 'elements']\n",
      "['node', 'in', 'layer α']\n",
      "['node', 'in', 'layer β']\n",
      "['node', 'in', 'layer α']\n",
      "['node', 'in', 'layer β']\n",
      "['node', 'in', 'layer α']\n",
      "['node', 'in', 'layer β']\n",
      "['approach', 'be', 'straightforward']\n",
      "['nodes', 'in', 'giant component']\n",
      "['message', 'pass', 'eqs']\n",
      "['nodes', 'in', 'giant component s']\n",
      "['z', 'be', 'normalization factor']\n",
      "['fact', 'in pα', 'k']\n",
      "['pα', 'in', 'fact']\n",
      "['nodes', 'in', 'giant component']\n",
      "['scale-free distribution', 'with layer', 'layer 1']\n",
      "['ﬁgure 1', 'of panel', 'b']\n",
      "['correlated interdegree', 'for threshold', 'panel']\n",
      "['power-law distribution', 'with', 'exponent γ']\n",
      "['heterogeneity', 'in', 'connectivity patterns']\n",
      "['temporal changes', 'in', 'networks structures']\n",
      "['interactions network', 'in', 'school']\n",
      "['presence', 'in', 'network']\n",
      "['social interaction', 'with', 'others']\n",
      "['∆ t', 'with', 'n']\n",
      "['nodes', 'of kc', 't']\n",
      "['s t ∼', 'value of', 'in-community degree']\n",
      "['terms', 'in', 'µ']\n",
      "['panel b', 'function of', 'µ']\n",
      "['95 conﬁdence interval', 'in', 'gray']\n",
      "['contacts', 'with', 'nodes']\n",
      "['network', 'with', 'many connected components']\n",
      "['network', 'with', 'larger modules']\n",
      "['temporal network', 'in', 'american physical society']\n",
      "['recovery', 'describe by', 'γ − → s']\n",
      "['a. vespignani', 'process on', 'complex networks']\n",
      "['pan', 'esz', 'j. kert']\n",
      "['c. faloutsos', 'in', 'databases lecture notes']\n",
      "['barab', 'asi', 'epl']\n",
      "['w911nf-15-1-0267', 'with', 'drs a. garcia']\n",
      "['e.u', 'review', 'manuscript']\n",
      "['i', \"'s community\", 'in-community degree']\n",
      "['integrated network', 'in growth', 'node']\n",
      "['1 c s', '−', '1 − 1']\n",
      "['1 − c s', '−', '1 − 1']\n",
      "['possible edges', 'in', 'network']\n",
      "['degree', 'in', 'degree']\n",
      "['nodes', 'in', 'network ﬁre']\n",
      "['node', 'in', 'instantaneous network']\n",
      "['t τc s', '− for', 's t τc s']\n",
      "['terms', 'give', 'k']\n",
      "['nodes', 'for k', 's t']\n",
      "['− ν', 'with', 'ν']\n",
      "['ω', 'with', 'ω']\n",
      "['95 conﬁdence interval', 'in', 'gray']\n",
      "['conﬁdence interval', 'in', 'gray']\n",
      "['functions', 'in', 'panels a-b']\n",
      "['kc → k c', 'kc kc in', 'panel d']\n",
      "['t τc s', 'for po', 'k']\n",
      "['oslom ﬁnds communities', 'in', 'non-deterministic way']\n",
      "['integrated aps network', 'be', 'modular']\n",
      "['contact', 'with', 'i']\n",
      "['particles', 'by nx', 'i']\n",
      "['b. cluster', 'use', 'eq']\n",
      "['maximum', 'in', 'i']\n",
      "['maximum', 'in', 'i']\n",
      "['g', 'with', 'eq']\n",
      "['eq', 'with g', 't']\n",
      "['overdamped motion', 'in', 'potential dr dt']\n",
      "['eq', 'in', 'limit t']\n",
      "['arbitrary lattice site j', 'be', 'chosen randomly']\n",
      "['eﬀect', 'in', 'initial distribution']\n",
      "['propagation', 'in', 'environment']\n",
      "['proliferation', 'in', 'environment']\n",
      "['administrative subdistricts', 'with', 'police station']\n",
      "['multiple thanas', 'in', 'rural bangladesh']\n",
      "['association', 'in', 'thana']\n",
      "['tractability', 'in dynamics', 'citation']\n",
      "['malaria transmission', 'in', 'india']\n",
      "['cholera transmission', 'in', 'bangladesh']\n",
      "['cholera surveillance', 'in', 'bangladesh']\n",
      "['amount', 'in', 'environment']\n",
      "['estimated likelihood', 'in', 'acceptance ratio']\n",
      "['weights', 'of set', 'citation']\n",
      "['plots', 'simulation with', 'seasonally varying α']\n",
      "['line', 'estimate', 'parameters']\n",
      "['10000 iterations', 'run of', '10000 iterations']\n",
      "['water bodies', 'in temperature', 'wt']\n",
      "['temperature', 'in', 'water bodies']\n",
      "['t0', 't0 of', 'lag']\n",
      "['full cholera data', 'with', 'smoothed covariates']\n",
      "['cholera transmission', 'in', 'bangladesh']\n",
      "['concentration', 'in', 'environment']\n",
      "['vaccine trial', 'in', 'guinea']\n",
      "['limited amount', 'with', 'aim reducing']\n",
      "['probabilities', 'in', 'table i']\n",
      "['v', 'match exactly with', 'vl']\n",
      "['local vaccination', 'with evolution', 'vl']\n",
      "['ω', 'for evolution', 'tr']\n",
      "['ω', 'for evolution', 'vl']\n",
      "['local vaccination', 'with evolution', 'tr']\n",
      "['ω', 'for evolution', 'vl']\n",
      "['β', 'of range', 'vl']\n",
      "['vl', 'supply of', 'immunization units']\n",
      "['er', 'with', 'hki']\n",
      "['fig 9', 'in', 'appendix e']\n",
      "['vl', 'in', 'figs 6']\n",
      "['vl', 'in', 'figs 6']\n",
      "['vaccines-depletion region', 'in', 'purple']\n",
      "['suﬃcient vaccines', 'in', 'yellow']\n",
      "['medical institutions', 'run out of', 'immunization units']\n",
      "['vl', 'in', 'plane']\n",
      "['†', 'in', 'b']\n",
      "['ωc', 'in', 'b']\n",
      "['height', 'depend on', 'vl']\n",
      "['appendix', 'in model', 'ebcm']\n",
      "['ω', 'prove in', 'detail']\n",
      "['er', 'with', 'hki']\n",
      "['l. a. braunstein', 'report on', 'progress']\n",
      "['occurrence', 'in', 'chesapeake bay']\n",
      "['their role', 'in', 'cholera']\n",
      "['scientiﬁc reports', 'in', 'press']\n",
      "['s c i', 'be', 'y h p']\n",
      "['p', 'with network', 'sf']\n",
      "['pure scale-free network', 'with', 'p']\n",
      "['networks', 'with', 'λ']\n",
      "['void node', 'in', 'link percolation']\n",
      "['network', 'economy of', 'region']\n",
      "['root node', 'with', 'connectivity k']\n",
      "['a. temporal percolation', 'in', 'ref']\n",
      "['cluster', 'reach nth generation follow', 'link']\n",
      "['void cluster', 'lose', 'node']\n",
      "['connectivity 4.07', 'in', 'giant component']\n",
      "['b sf network', 'with', 'λ']\n",
      "['g1', 'be', 'concave']\n",
      "['connectivity 4.07', 'in', 'giant component']\n",
      "['eqs', 'of s1', 't']\n",
      "['v s c', 'obtain from', 'eq']\n",
      "['unmdp', 'of ject', 'pict 0293/2008']\n",
      "['neighbor', 'transmit disease to', 'root node']\n",
      "['sf', 'with', 'λ']\n",
      "['node void percolation', 'in percolation', 'void node']\n",
      "['node void percolation', 'in', 'node void percolation']\n",
      "['sf network', 'with', 'λ']\n",
      "['er network', 'with', 'hki']\n",
      "['power law', 'with', 'exponent τ ≈ 2.5']\n",
      "['er network', 'with', 'hki']\n",
      "['connectivity 4.07', 'in', 'giant component']\n",
      "['e. m. volz', 'journal of', 'royal society interface 9 890']\n",
      "['bu t', 'with', 'synchronous updating']\n",
      "['fig 1', 'show with', 'asynchronous updating']\n",
      "['µ 0.6 1.0', 'with', 'asynchronous updating']\n",
      "['synchronous updating method', 'in', 'fig 2']\n",
      "['sfns', 'with', 'γ']\n",
      "['sfns', 'with', 'γ']\n",
      "['sfns', 'with', 'γ']\n",
      "['r', 'be', 'pearson correlation coefﬁcient']\n",
      "['c', 'be', 'clustering coefﬁcient']\n",
      "['conclusion', 'in', 'paper']\n",
      "['zero mean', 'with ηx', 't']\n",
      "['convex hull', 'of a', 't']\n",
      "['cauchy', 'of', 'integral geometric formulae']\n",
      "['cauchy', 'have', 'formulae']\n",
      "['xm dimensional stochastic process', 'in', 'x direction']\n",
      "['isotropy', 'in', 'space']\n",
      "['path', 'be', 'marked']\n",
      "['tm', 'give component', 'clearly y2 2d tm']\n",
      "['tra jectory', 'of component', 'clearly y2 2d tm']\n",
      "['path', 'be', 'brownian']\n",
      "['path', 'be', 'dimensional']\n",
      "['interval', 'with', 'probability qt']\n",
      "['power-law tail', 'due to m', 't']\n",
      "['tm', 'determine in', 'supplementary material']\n",
      "['cauchy', 'of', 'formula']\n",
      "['underlying process', 'be', 'rotationally invariant']\n",
      "['supplementary materials cauchy', 'have', 'formula']\n",
      "['convex hull', 'with', '0 ≤ τ ≤ t']\n",
      "['stick', 'of m', 'θ']\n",
      "['cauchy', 'over', 'formulas']\n",
      "['latter condition', 'obtain by', 'laplace transxm 0']\n",
      "['∼ ez', 'iﬁed', 'bessel functions']\n",
      "['ez', 'iﬁed', 'bessel functions']\n",
      "['fraser c', 'potential of', 'strain']\n",
      "['colizza v', 'm', 'barth']\n",
      "['t', 'be', 'displayed']\n",
      "['convex hull', 'of a', 't']\n",
      "['monte carlo', 'display as', 'symbols']\n",
      "['monte carlo', 'be', 'displayed']\n",
      "['component', 'be', 'brownian']\n",
      "['realizations', 'number of', '2 · 106']\n",
      "['changing interest level', 'in', 'campaign']\n",
      "['information epidemics', \"principle 's\", 'pontryagin']\n",
      "['individuals', 'exchange', 'messages']\n",
      "['individuals', 'give', 'rise']\n",
      "['individuals', 'give rise to', 'information epidemics']\n",
      "['spread', 'in', 'daley-kendall models']\n",
      "['t', 'let', 't']\n",
      "['objective functional', 'choose', 't']\n",
      "['enthusiastic individuals', 'promote', 'product']\n",
      "['advertisements', 'in', 'mass media']\n",
      "['u', 'be', 'lebesgue integrable']\n",
      "['cost', 'in', 'cost functional']\n",
      "['details', 'in', 'appendix a 6 5']\n",
      "['maximization 1', 'to problem', 'u ∗ r ∗ h']\n",
      "['shapes', 'in', 'sections']\n",
      "['details', 'in', 'appendix b 6']\n",
      "['tree', 'in', 'fig 1']\n",
      "['r0', 'in', 'fig 2a']\n",
      "['newbie', 'in', 'market']\n",
      "['individuals', 'convince', 'susceptibles']\n",
      "['plots', 'in', 'section']\n",
      "['variable t', 'drop for', 'notational brevity']\n",
      "['variable t', 'drop for', 'notational brevity']\n",
      "['t', 'drop for', 'notational brevity']\n",
      "['optimal singular controls', 'on ledzewicz', 'h. sch ¨ attler']\n",
      "['design', 'conference on', 'computer communications']\n",
      "['rumours', 'university of', 'adelaide']\n",
      "['s. belen', 'behaviour of', 'rumours']\n",
      "['krishnamoorthy', 'price in', 'durable-good duopoly']\n",
      "['optimal pricing', 'in', 'durable-good duopoly']\n",
      "['urse notes', 'department of', 'applied mathematics']\n",
      "['applied mathematics', 'university of', 'washington']\n",
      "['homogeneous degree k', 'graph of', 'degree k']\n",
      "['δ', 'for model', 'k']\n",
      "['δ', 'for model', 'k']\n",
      "['rrg-4', 'with', 'n']\n",
      "['phd dissertation', 'university of', 'warwick']\n",
      "['szab', 'o', 'g']\n",
      "['qc', 'increase for', '1 δ']\n",
      "['variation', 'with', 'q']\n",
      "['variation', 'with', 'q']\n",
      "['continuous phase transition', 'take here', 'place']\n",
      "['functions', 'in', 'fig 5']\n",
      "['variation', 'with', 'δ']\n",
      "['variation', 'with', 'n']\n",
      "['variation', 'with', 'q']\n",
      "['variation', 'with', 'q']\n",
      "['discussion', 'in', 'preceding subsection']\n",
      "['rsat', 'of φ', 'rsat']\n",
      "['variation', 'with', 'q']\n",
      "['variation', 'with', 'q']\n",
      "['scales', 'with', 'n']\n",
      "['exponents', 'dependent', 'δ']\n",
      "['stronger dependence', 'in', 'region 1']\n",
      "['variation', 'with', 'δ']\n",
      "['variation', 'in', 'intermediate region']\n",
      "['tr', 'be', 'β']\n",
      "['phase diagrams', 'in', 'w − β plane']\n",
      "['β', 'higher than', 'pc']\n",
      "['ni', 'of plot', 'n']\n",
      "['averages', 'over', '104 realizations']\n",
      "['averages', 'over', '104 realizations']\n",
      "['tr', 'with', 'k']\n",
      "['quarantine', 'isolate', 'superspreaders']\n",
      "['barab', 'asi', 'nature 401 130']\n",
      "['plant pathology', 'of department', 'my 14853']\n",
      "['n', 'be', 'ﬁxed']\n",
      "['event s', 'i1 of', 'transition s → s − 1']\n",
      "['event s', 's with', 'strain 1']\n",
      "['strain 2', 'with strain', 'β1']\n",
      "['s → s', 'with', 'strain 1']\n",
      "['r1 r2', 'character of', 'ﬁxed points']\n",
      "['cl', 'of n', '0 1 − 1/r']\n",
      "['three ﬁxed points', 'with', 'non-negative pop']\n",
      "['property', 'lead to', 'eq']\n",
      "['eﬀect', 'show in', 'fig 3']\n",
      "['example', 'for pm', 'n']\n",
      "['explanations', 'in', 'subsection']\n",
      "['sharp peak', 'evolve into', 'cl']\n",
      "['o', 'of scale', 'ε − 2']\n",
      "['unknown f', 'with ρ', 'x τ']\n",
      "['ρ', 'with', 'yet unknown f']\n",
      "['quadratic form', 'in', 'y']\n",
      "['full analogy', 'with', 'eq']\n",
      "['47 calculation', 'in', 'eq']\n",
      "['equation', 'in', 'variable ξ']\n",
      "['→ 0', 'as v', 'ξ']\n",
      "['cl', 'in', 'cases']\n",
      "['eq', 'from v', 'ξ']\n",
      "['integrals', 'in', 'eq']\n",
      "['si1 i2r', 'in', 'cases']\n",
      "['origin', 'in', 'fig 1']\n",
      "['quasineutral competition', 'in', 'epidemiology']\n",
      "['our interest', 'in', 'quasi-neutral competition']\n",
      "['orx y z', 'to 15', 'a4']\n",
      "['x', 'be', 'frozen']\n",
      "['integration', 'in', 'eq']\n",
      "['equation', 'generalize', 'eq']\n",
      "['our starting point', 'in', 'appendix']\n",
      "['o', 'of scale', 'ε − 2']\n",
      "['unknown f', 'with ρ', 'x τ']\n",
      "['eq', 'from cij', 'x']\n",
      "['diﬀusion term', 'in', 'full fokker-planck equation']\n",
      "['total derivatives', 'with', 'ξi']\n",
      "['ˆl †', 'in', 'eq']\n",
      "['− 3 ln', 'ξ0', '1 − ξ0']\n",
      "['rn', 'rescale in', 'everywhere appendix']\n",
      "['ln ln 1', 'to simpliﬁes', 'd4']\n",
      "['rural development', 'for drive', 'citation']\n",
      "['ostwald ripening', 'in', 'material processing']\n",
      "['formation', 'in', 'favor']\n",
      "['spread', 'in', 'layer a']\n",
      "['tr', 'go to', 'blue']\n",
      "['s', 'vaccinate', 'gray']\n",
      "['r', 'of derivative', 'dashed line']\n",
      "['moderate opinions', 'in', 'favor']\n",
      "['fig 2', 'inset in', 'ones']\n",
      "['r', 'of derivative', 'dashed line']\n",
      "['extremist', 'in', 'opinions']\n",
      "['r', 'of derivative', 'dashed line']\n",
      "['syndrome', 'be', 'respiratory']\n",
      "['2003 oﬃcials', 'in', 'hong kong']\n",
      "['fever', 'shortness of', 'breath']\n",
      "['erasmus medical center', 'in', 'rotterdam']\n",
      "['researchers', 'in', 'erasmus medical center']\n",
      "['rates', 'estimate', 'r0']\n",
      "['except transient response', 'in', 'diagnostic rate α']\n",
      "['rate', 'in', 'δ']\n",
      "['situation', 'in', 'toronto']\n",
      "['comparison', 'with', 'epidemiological statistics']\n",
      "['march 31', 'from number', 'hong kong']\n",
      "['march 31', 'from number', 'ontario']\n",
      "['march 31', 'from number', 'top row']\n",
      "['march 31', 'from number', 'canada']\n",
      "['canada', 'of all', 'third row']\n",
      "['march 31st', 'day of', 'jump']\n",
      "['singapore', 'have', 'table 1']\n",
      "['6 school', 'academy of', 'arts']\n",
      "['management', 'of school', 'le1 7rh']\n",
      "['management', 'of school', 'royal netherlands academy']\n",
      "['management', 'of school', 'leicester']\n",
      "['6 school', 'university of', 'leicester']\n",
      "['s c i', 'be', 'y h p']\n",
      "['sides', 'on −', 'α − 1']\n",
      "['y', 'of each', '0 t']\n",
      "['y', 'of each', 'i']\n",
      "['period', 'in', 'past']\n",
      "['network', 'with', 'degree exponent λ']\n",
      "['scale-free network', 'with', 'degree exponent λ']\n",
      "['network', 'with', 'degree exponent λ']\n",
      "['networks', 'with', '2 λ']\n",
      "['g. r. jafari', 'journal of', 'artiﬁcial society']\n",
      "['epidemics', 'in', 'ch2olera']\n",
      "['hamilton', 'have', 'forgotten epidemics']\n",
      "['prediction', 'plot in', 'fig 2']\n",
      "['3 stages', 'in', 'fig 2']\n",
      "['black dots', 'curve of', 'revised data']\n",
      "['high-density housing estate', 'in', 'hong kong']\n",
      "['signiﬁcant statistical ﬂuctuations', 'in', 'data']\n",
      "['method', 'widely apply to', 'various epidemiological data']\n",
      "['3-stage analysis', 'mention in', 'fig 2']\n",
      "['clusteringly scatter', 'in', 'map']\n",
      "['piecewise linear', 'in', 'b']\n",
      "['piecewise linear ﬁts', 'have r in', 'b']\n",
      "['hollow triangle', 'in', 'figs 5']\n",
      "['hk', 'for datum', 'grey column']\n",
      "['scaling behaviour', 'in', 'log-log coordinates']\n",
      "['changes', 'in', 's/n']\n",
      "['changes', 'in', 'n/s']\n",
      "['changes', 'in', 'n/s']\n",
      "['c', 'be', 'random']\n",
      "['process', 'be', 'memoryless']\n",
      "['equations', 'in', 'appendix a']\n",
      "['n/s', 'remain', 'ﬁxed']\n",
      "['c', 'be', 'random']\n",
      "['eﬀects', 'be', 'once unimportant']\n",
      "['transition', 'in', '§ 4']\n",
      "['similarity', 'in', 'notation']\n",
      "['changes', 'in', 'n/s']\n",
      "['diﬀerence', 'in', 'timing β']\n",
      "['recoveries', 'happen in', 'interval']\n",
      "['diﬀerence', 'be', 'skellam']\n",
      "['many events', 'happen in', 'interval']\n",
      "['generation interval distribution', 'close to', 'when discrete']\n",
      "['sir', 'with', 'birth']\n",
      "['diﬀerent orders', 'in', 'n']\n",
      "['noisier dyfor n', 'in', 'fig 3']\n",
      "['ebola virus', 'in', 'guinea']\n",
      "['ebola virus', 'of detection', 'parameters']\n",
      "['symptoms', 'include', 'diarrhoea']\n",
      "['mathematical formulation', 'in', 'section']\n",
      "['i', '−', 'γs']\n",
      "['detection', 'in', 'guinea']\n",
      "['s', 'with', 's']\n",
      "['i', 'with infect', 'b i t']\n",
      "['c r', 'with', 'r']\n",
      "['r', 'with r', 't']\n",
      "['s', 'with s', 't']\n",
      "['control s', 'with', 'control x']\n",
      "['control r', 'with', 'control x']\n",
      "['control s', 'without s', 't']\n",
      "['control', 'without i', 't']\n",
      "['current situation', 'in', 'guinea']\n",
      "['problem', 'for u', 't ∈']\n",
      "['problem', 'for u', 't']\n",
      "['control n', 'with', 'control']\n",
      "['engineering center', 'in citation', 'optec']\n",
      "['m. vukov', 'citation in', 'engineering center']\n",
      "['c. j. peters', 'transmission of', 'ebola hemorrhagic fever']\n",
      "['a. p. galvani', 'dynamics in', 'montserrado']\n",
      "['a. p. galvani', 'dynamics of', 'ebola virus transmission']\n",
      "['center', 'university of', 'aveiro']\n",
      "['β2/ρ2', 'also', 'α']\n",
      "['boundary', 'be', 'boundary']\n",
      "['k', 'be', 'hypergeometric']\n",
      "['c', 'c for', 'ϕ → 0']\n",
      "['space', 'over cb', '¯ t']\n",
      "['ψ', 'for n', 'ψ']\n",
      "['m t', 'of λ', 't']\n",
      "['˜m t', 'of ˜λ', 't']\n",
      "['cb', 'to m', 't']\n",
      "['λ', 'zero with', 't']\n",
      "['β1', 'be', 'hypergeometric']\n",
      "['ρ2', 'be', 'hypergeometric']\n",
      "['β2', 'be', 'hypergeometric']\n",
      "['ρ1', 'be', 'hypergeometric']\n",
      "['λ', 'top eigenvalue of', 'm']\n",
      "['functions', 'of family', 'ψ']\n",
      "['equating coefﬁcients', 'in', 'determines']\n",
      "['production kernel', 'with process', 'zn']\n",
      "['functions', 'with', 'appropriate ﬁnite limits']\n",
      "['bounds', 'in', 'lemma 3']\n",
      "['bound', 'for u', 'ϕ']\n",
      "['limits', 'of l', 'ϕ']\n",
      "['url factsheets/hepatitis-d/en', 'with', 'formulas']\n",
      "['phase transitions', 'in', 'cooperative coinfections']\n",
      "['algorithm', 'with', 'historic uk measles data']\n",
      "['increasing peaks', 'in', 'panel']\n",
      "['panels', 'in β', 't']\n",
      "['increasing peaks', 'in', 'panel c']\n",
      "['moduli', 'transform of', 't']\n",
      "['underreporting bias', 'in', 'uk measles data']\n",
      "['patrick harrington †', 'university of', 'michigan']\n",
      "['nodes', 'in', 'bioinformatics']\n",
      "['examples', 'include', 'gmms']\n",
      "['research', 'in', 'mrfs']\n",
      "['nodes', 'in', 'network']\n",
      "['k − 1', 'with', 'z']\n",
      "['j', '− with', 'z']\n",
      "['symmetry', 'in', 'interactions j']\n",
      "['symmetry', 'in', 'θi']\n",
      "['constraint symmetry θi', 'of form', 'j']\n",
      "['m', 'with g', 'm']\n",
      "['m', 'with g', 'm']\n",
      "['g', 'with', 'm']\n",
      "['h', 'with', 'α']\n",
      "['α', 'with h', 'm']\n",
      "['entries', 'in', 'h i']\n",
      "['entries', 'on h', 'm']\n",
      "['j', 'with', 'surrogate 2']\n",
      "['co-ordinate descent', 'in', 'group lasso logistic regression']\n",
      "['non-zero entries', 'in', 'estimator']\n",
      "['environment', 'in', 'matlab']\n",
      "['images', 'in', 'figures']\n",
      "['discussions', 'with', 'ami wiesel']\n",
      "['s. j. clark', 'projection of', 'hiv prevalence']\n",
      "['d. ben avraham', 'strategy for', 'computer networks']\n",
      "['r. tibshirani', 'numericcitation with', 'graphical lasso']\n",
      "['k. eames', 'journal of', 'royal society interface']\n",
      "['s. boyd', 'method for', 'large-scale l1regularized logistic regression']\n",
      "['i. yang', 'journal of', 'computational']\n",
      "['d. koller', 'learn of', 'markov networks']\n",
      "['lee', 'in', 'advances']\n",
      "['p. b ¨ uhlmann', 'lasso for', 'logistic regression']\n",
      "['j. zhu', 'journal of', 'american statistical association']\n",
      "['estimating trends', 'in', 'hiv/aids']\n",
      "['high-dimensional consistency', 'in', 'nips']\n",
      "['i. m. longini', 'transmissibility of', 'pandemic inﬂuenza']\n",
      "['r', 'be', 'random']\n",
      "['θk', 'with', 'maximum likelihood ˆp']\n",
      "['nodes', 'in', 'r1']\n",
      "['nodes', 'in', 'r2']\n",
      "['p', 'gree of', 'variability']\n",
      "['notomers', 'enter immediately into', 'service']\n",
      "['arrival', 'in', 'inﬁnite server queue']\n",
      "['arrivals', 'in', 'analogous m']\n",
      "['y', 'around h', 'y']\n",
      "['simpliﬁcation', 'in', 'last step']\n",
      "['eq', 'in h', 'y']\n",
      "['pgf h', 'in', 'eq']\n",
      "['inset', 'scale by', 'n 2/3']\n",
      "['presence', 'in', 'bump']\n",
      "['tional powers', 'in', 'expansion']\n",
      "['minishes', 'with', 'n']\n",
      "['n', 'in', 'ﬁgure 7']\n",
      "['ν', 'for scale', 'ν +2']\n",
      "['m ∼', 'by scale', 'ν +2']\n",
      "['exponent', 'in', 'eq']\n",
      "['n 2/3', 'at collapse', 'exponential cutoﬀs collapse']\n",
      "['power-laws', 'in', 'p n']\n",
      "['derivation', 'in', 'eq']\n",
      "['power-law regime', 'thus only depend on', 'n']\n",
      "['inﬁnite regime', 'in 10', 'n ∞ t ∞ ∼']\n",
      "['well-known phenomenon', 'study from', 'ancient times']\n",
      "['kind', 'reset of', 'variables']\n",
      "['kinetic scheme s', 'within susceptible', 's']\n",
      "['k', 'of variation', 'satisfactory reproduces']\n",
      "['ﬂu activity', 'in', 'berlin']\n",
      "['european regions', 'with', 'continental climate']\n",
      "['berlin', 'for light-gray', 'red online lines']\n",
      "['harmonic ﬁlter', 'in', 'eq']\n",
      "['summary', 'in', 'work']\n",
      "['ml', 'support by', 'epsrc leadership fellowship ep/j004022/2']\n",
      "['near-criticality', 'use', 'arguments']\n",
      "['near-criticality', 'use', 'non-rigorous arguments']\n",
      "['infectives', 'observe', 'bimodality']\n",
      "['1 n 1/3', 'with', 'ω growing']\n",
      "['couplings', 'with', 'suitable brownian motions']\n",
      "['n', 'be', 'simple']\n",
      "['n', 'let', 'ds']\n",
      "['∞ e ds', '0 by', 'd6']\n",
      "['d3 s', 'of integrability', 'd2']\n",
      "['ns p ∞', 'in', 'cases']\n",
      "['theorem 2.3', 'of case', 'i']\n",
      "['βn', 'assume → λ as', 'n → ∞']\n",
      "['g', 'with', 'm nλ']\n",
      "['rt', 'increase moreover in', 't']\n",
      "['rt', 'increase in', 't']\n",
      "['o', 'w.h.p', 'remark 3.6']\n",
      "['duration', 'in', 'section 3.4']\n",
      "['l', 'for ωn', 'ωn − 1']\n",
      "['k 6', 'with ˜st', 'k']\n",
      "['˜st', 'with', 'k 6']\n",
      "['lemma 6.1', 'with', 'd']\n",
      "['jensen', 'have', 'cauchy']\n",
      "['t', 'for', 'k']\n",
      "['t', 'for', 'k − knk']\n",
      "['0 ˜αn t', 'in', 'uniformly t']\n",
      "['t ∈', 'for f', 't']\n",
      "['t', 'for f', 't']\n",
      "['y', 'for ns', 'k']\n",
      "['− kpk t', 'in', 't']\n",
      "['martingale', 'with', 'respect']\n",
      "['kns', 'nsα2 since', 'o nαn']\n",
      "['lemma 4.2', 'with', 'b']\n",
      "['ε', 'for yield', 'k2']\n",
      "['nα3 n →', 'since i', '∞']\n",
      "['bt', 'be', 'brownian']\n",
      "['theorem 2.4', 'of z', 'i']\n",
      "['t ∈', 'for 0', '0 δ']\n",
      "['w.h.p m', 'stop with', 'zm']\n",
      "['ym', 'with', 'xm']\n",
      "['m', 'be', 'i.i.d']\n",
      "['m 6 m', 'for k', 'c3n']\n",
      "['¯ ztα − 2', 'in', 'd']\n",
      "['n', 'in', 'theorem 2.3']\n",
      "['theorem 2.3', 'in w.h.p', 'b']\n",
      "['w.h.p', 'in', 'theorem 2.3']\n",
      "['z0', 'since independent', 'p']\n",
      "['argument', 'with', 'minor modiﬁcations']\n",
      "['notation', 'in', 'lemma 4.3']\n",
      "['fraction', 'let', 'χ']\n",
      "['criticality', 'near n', 'di n']\n",
      "['luczak', 'in', 'preparation']\n",
      "['vertices', 'in', 'c1']\n",
      "['lemma 6.3', 'with', 'theorem 3.1']\n",
      "['brownian motion', 'with', 'parabolic drift']\n",
      "['mathematics', 'of department', 'sweden e-mail address']\n",
      "['mathematical sciences', 'of school', 'uk']\n",
      "['mathematics', 'of school', 'uk e-mail address']\n",
      "['mathematics', 'of school', 'm13 9pl']\n",
      "['n', 'for relationship', 'n']\n",
      "['biocomputation', 'for institute', 'bifi']\n",
      "['biocomputation', 'for institute', '50018 zaragoza']\n",
      "['mathematics', '2department of', 'theoretical physics']\n",
      "['mathematics', 'university of', 'strathclyde']\n",
      "['3 institute', 'university of', 'zaragoza']\n",
      "['mathematics', 'university of', 'zaragoza']\n",
      "['s c i', 'be', 'y h p']\n",
      "['application', 'in', 'ecology']\n",
      "['persistence', 'in', 'natural landscapes']\n",
      "['crops', 'cultivate in', 'country ﬁelds']\n",
      "['plants', 'grow naturally in', 'given environment']\n",
      "['entries aij', 'have', '1 0']\n",
      "['g', 'be', 'simple']\n",
      "['n points', 'in', 'unit rectangle']\n",
      "['2 bottom', 'with', 'top']\n",
      "['fig 2', 'of panel', 'b']\n",
      "['rgg', 'in', 'd-dimensional cube']\n",
      "['rgg', 'in', 'd-dimensional cube']\n",
      "['nr2', 'in', 'graphs']\n",
      "['square', 'in', 'rgg']\n",
      "['rrgs', 'with', '1 left']\n",
      "['spectral radius', 'of plot', '30 centre']\n",
      "['rrgs', 'with', '1 left']\n",
      "['rrgs', 'for degree', '30 centre']\n",
      "['radius r', 'with', 'goal checking']\n",
      "['grant', 'through aragón', 'spain']\n",
      "['− 1 ≤ r', '≤', 'dashed line']\n",
      "['complexity', 'in', 'host morphology']\n",
      "['severity', 'in', 'wheat cultivar mixtures']\n",
      "['connectivity', 'in', 'fractal regions']\n",
      "['future contacts', 'in', 'time-varying networks']\n",
      "['conﬁgurations', 'in', 'leftmost column']\n",
      "['p xt', 'x0 by', 'approximate sampling method']\n",
      "['belief propagation technique', 'with', 'partial observation']\n",
      "['heatplots', 'in', 'fig 3']\n",
      "['heatplots', 'in', 'fig 5a']\n",
      "['degree-based observation', 'particularly convenient for', 'heterogenous networks']\n",
      "['observations', 'with', 'few infected/recovered nodes']\n",
      "['nodes', 'in', 'network']\n",
      "['snapshots', 'in', 'panels']\n",
      "['snapshots', 'in', 'panels']\n",
      "['prediction', 'with', 'complete observation']\n",
      "['heat-plots', 'in', 'fig 10b']\n",
      "['comment', 'in', 'thread about b. citation citation']\n",
      "['belief propagation', 'recover only', 'nodes']\n",
      "['belief propagation', 'recover', 'nodes']\n",
      "['short loops', 'in', 'network']\n",
      "['node i', 'infect', 'node j']\n",
      "['xt', 'with', 'ζ t i i']\n",
      "['patient zero', 'in', 'static networks']\n",
      "['heatplots', 'compute by', 'belief propagation']\n",
      "['n', 'of graph', 'µ']\n",
      "['roc curve', 'under area', 'c']\n",
      "['tobs', 'at observation', 'd']\n",
      "['roc curve', 'under area', 'c']\n",
      "['tobs', 'at observation', 'd']\n",
      "['anhui university', 'university of', 'electronic science']\n",
      "['anhui university', 'xi', 'shan']\n",
      "['reasons', 'present in', 'sec']\n",
      "['s f', 'of rate', 'probability']\n",
      "['labeled er network 26', 'with', 'network size n']\n",
      "['er network', 'in', 'fig 2']\n",
      "['ineq', 'accord to ehki', 'x −']\n",
      "['er networks', 'on r', '∞']\n",
      "['ucm networks', 'on r', '∞']\n",
      "['o', 'to proportional', 'ρ2 i']\n",
      "['theory', 'be', 'ideal']\n",
      "['d. j. earn', 'proceed of', 'national academy']\n",
      "['d. j. earn', 'proceed of', 'national academy']\n",
      "['v. a. jansen', 'journal of', 'royal society interface 7 1247']\n",
      "['m. barth', 'elemy', 'physical review e 73 046131']\n",
      "['shaun', 'for model', 'e']\n",
      "['depictions', 'in', 'ﬁlms']\n",
      "['ﬁlms', 'destroy', 'zombie']\n",
      "['interaction', 'with', 'zombie z']\n",
      "['zombie apocalypse', 'take', 'days']\n",
      "['rarity', 'in', 'romero ﬁlms']\n",
      "['romero ﬁlms', 'percent of', 'susceptibles']\n",
      "['ﬁlm', 'have', 'title']\n",
      "['interaction', 'with', 'susceptibles']\n",
      "['zombie', 'have', 'brain']\n",
      "['suicide', 'be', 'commonplace']\n",
      "['ﬁve stubs', 'form', 'lines']\n",
      "['shortest paths', 'of set', 'geodesics']\n",
      "['node', 'how often appear in', 'geodesics']\n",
      "['kirkwood', 'have', 'superposition approximation']\n",
      "['squares', 'with', 'diagonals']\n",
      "['triangle corner', 'with', 'p']\n",
      "['diﬀerences', 'in', 'higher-order structures']\n",
      "['p', 'in', 'p ll']\n",
      "['betweenness centrality', 'rank', 'nodes']\n",
      "['abstract cauchy problem', 'in', 'state-space x']\n",
      "['invariant subset', 'in', 'x']\n",
      "['n 7 → b', 'with', '0 b ∞']\n",
      "['0 b', 'with b', 'n']\n",
      "['0 nonnegative c 1', 'with', 'b']\n",
      "['y → y', 'φ by', 'f']\n",
      "['β', 'with', 'π']\n",
      "['long computation', 'include in', 'appendix']\n",
      "['pde', 'of r', 't']\n",
      "['t ≥ 0', 'for r', 't']\n",
      "['ψ ≥ 0', 'for r', 't']\n",
      "['solution', 'be', 'triple']\n",
      "['proposition 1', 'existence of', 'dfe']\n",
      "['solution', 'with', '˘ν']\n",
      "['3 librium', 'in', 'x']\n",
      "['t s', 'n for', 't']\n",
      "['t', 'for n', 't']\n",
      "['t t ∗', 'with', 'x']\n",
      "['equation', 'with', 't']\n",
      "['dfe', 'about model', 'm1']\n",
      "['zj j ∈ n', 'with', 'hj']\n",
      "['rf', 'to probability', '1 − θ']\n",
      "['dr', '∂ with', 'boundary condition rzmax']\n",
      "['∂', 'with', 'boundary condition rzmax']\n",
      "['q3 x', 'in term', 'p4']\n",
      "['dq kx', 'with', 't1']\n",
      "['w → 0', '≤ for', 'kx − ykx → 0']\n",
      "['incomplete freund', 'with', 'adjuvant hydroxide']\n",
      "['matched components', 'of number', 'r0']\n",
      "['research', 'bridge', 'gap']\n",
      "['coordinates', 'within', 'england']\n",
      "['tweets', 'originate in', 'england']\n",
      "['non-diagonal elements', 'in', 'β']\n",
      "['median r0', 'in', 'ﬂuenza']\n",
      "['d. j. nokes', 'proceed of', 'royal society']\n",
      "['n. ferguson', 'journal of', 'royal society interface']\n",
      "['a. g. mckendrick', 'proceed of', 'royal society']\n",
      "['1 − δ', 'in', 'βa']\n",
      "['βa pi', 'in pi', 'j']\n",
      "['βa pi', 'in pi', 'i']\n",
      "['1 − δ pi', 'in', 'βa pi']\n",
      "['pr', 'tor of', 'markov chain']\n",
      "['fi ∈ r2n +1', 'represent', 'ith unit column vector']\n",
      "['maps', 'deﬁned above follow', 'properties']\n",
      "['ψ', 'with', 'ψi']\n",
      "['∗', 'be', 'zi u']\n",
      "['nonlinear map', 'of point', 'i']\n",
      "['nonlinear map', 'of point', '⇒ p ∗ r']\n",
      "['nonlinear map', 'of point', 'i']\n",
      "['nonlinear map', 'of point', 'i.e p ∗ r']\n",
      "['j', 'neighbor of', 'k']\n",
      "['lemma 2.1', 'in', 'numericcitation']\n",
      "['1 − δ', 'in', 'p ∗ s βa']\n",
      "['yi', 'with', 'change']\n",
      "['decision', 'in ahn', 'cdc']\n",
      "['numericcitation h. j. ahn', 'in', 'decision']\n",
      "['decision', 'in numericcitation', 'cdc']\n",
      "['numericcitation', 'in', 'decision']\n",
      "['column vector', 'with', 'n elements']\n",
      "['diagonal matrix', 'with', 'vertex degrees along its diagonal']\n",
      "['νd', 'be', 'ﬁnite']\n",
      "['nodes', 'with', 'high k-core index']\n",
      "['networks', 'with', 'νd']\n",
      "['networks', 'with', 'νd']\n",
      "['numericcitation', 'network with', 'νd']\n",
      "['absolute errors decrease', 'with', 'n']\n",
      "['spreading processes', 'perform on', 'giant connected clusters']\n",
      "['networks', 'have', 'distinct eigenvector localizations numericcitation']\n",
      "['networks', 'have', 'eigenvector localizations numericcitation']\n",
      "['adjacency', 'of ratio', 'ipr']\n",
      "['c', 'fraction of', 'λmfl']\n",
      "['networks', 'in', 'lhns']\n",
      "['networks', 'in', 'lkns']\n",
      "['microscopic topologies', 'in', 'real-world networks including degree-degree correlations']\n",
      "['networks', 'with', 'negative degree-degree correlations']\n",
      "['networks', 'with', 'low clustering']\n",
      "['networks', 'with', 'high clustering']\n",
      "['numericcitation newman', 'spread on', 'networks']\n",
      "['numericcitation gross', 'dynamics on', 'adaptive network']\n",
      "['numericcitation g', 'omez', 's']\n",
      "['mieghem', 'spread in', 'networks']\n",
      "['numericcitation lokhov', 'ezard', 'm']\n",
      "['numericcitation lokhov', 'l', 'm']\n",
      "['centrality', 'in', 'networks']\n",
      "['spectral redemption', 'in', 'clustering sparse networks']\n",
      "['numericcitation radicchi', 'threshold in', 'networks']\n",
      "['eigenvector localization', 'in', 'networks']\n",
      "['s c i', 'be', 'y h p']\n",
      "['i', 'q', 's']\n",
      "['eye', 'with', 'slopes']\n",
      "['parameter', 'imply', 'ﬁrst-order transition']\n",
      "['τc', 'close to ist', 'τ']\n",
      "['characteristic time-scale', 'in', 'fig 3']\n",
      "['e.g', 'thank', 'h.j.h']\n",
      "['e.g', 'formulate', 'proof']\n",
      "['child health', 'of institute', 'p01-hd31921']\n",
      "['innovations', 'of diﬀusion', 'citation']\n",
      "['security', 'in symposium', '343 1991']\n",
      "['1991 ieee computer society symposium', 'in', 'security']\n",
      "['j. kert', 'esz', 'phys']\n",
      "['m. jacomy', 'conference on', 'weblogs']\n",
      "['international trade network', 'version of', 'database']\n",
      "['subsidiary corporations', 'in', 'country b']\n",
      "['node', 'in', 'network']\n",
      "['nodes', 'in', 'highest shell']\n",
      "['nodes', 'only have', 'k ≥ 2']\n",
      "['shell 9', 'in', 'itn']\n",
      "['goods', 'most by', 'especially china']\n",
      "['headquarters', 'in', 'china']\n",
      "['headquarters', 'in', 'russia']\n",
      "['con', 'for fig', 'c']\n",
      "['itn', 'for fig', 'c']\n",
      "['country', 'overcome', 'crisis']\n",
      "['targeted node', 'of weight', 'strength']\n",
      "['country j', 'infect by', 'country i']\n",
      "['crisis', 'in', 'subsidiary']\n",
      "['error bars', 'in', 'figs 3']\n",
      "['crisis', 'in', 'indonesia']\n",
      "['crisis', 'in', 'greece 2']\n",
      "['correlations', 'in', 'finance']\n",
      "['shell 2', 'in', 'itn']\n",
      "['nucleus', 'from distance', 'con']\n",
      "['nucleus', 'from distance', 'itn']\n",
      "['simulated network', 'number of', 'links per node']\n",
      "['nodes', 'of m.', 'c']\n",
      "['1 worldwide spreading', 'university of', 'thessaloniki']\n",
      "['business', 'of 3faculty', 'ch']\n",
      "['business', 'of 3faculty', '1015 lausanne switzerland']\n",
      "['i', 'result about', 'con']\n",
      "['subsidiary corporations', 'in', 'country b']\n",
      "['propagation', 'in', 'itn']\n",
      "['country', 'in', 'crisis']\n",
      "['bilateral trade', 'with', 'countryi']\n",
      "['error bars', 'show', 'standard deviation']\n",
      "['error bars', 'show', 'standard deviation']\n",
      "['error bars', 'show', 'standard deviation']\n",
      "['rural community', 'in', 'kansas']\n",
      "['overseas territories', 'report', 'laboratory']\n",
      "['la gloria', 'town near', 'veracruz']\n",
      "['hog barn', 'in', 'walworth county']\n",
      "['predictive strategies', 'in', 'rural regions']\n",
      "['rural clay county', 'in', 'kansas']\n",
      "['rural clay', 'in', 'kansas']\n",
      "['sub-weight wx', 'j between', 'node i']\n",
      "['links', 'with', 'strength greater']\n",
      "['k-core', 'be', 'maximal']\n",
      "['nodes', 'with', 'deepest coreness']\n",
      "['various locations', 'in', 'county']\n",
      "['node random selection', 'in', 'key location']\n",
      "['locations', 'be', 'therefore easier']\n",
      "['preventive medicine', 'of journal', 'issue 3']\n",
      "['medicine', 'of journal', 'issue 1']\n",
      "['v.d perminov', 'model for', 'simulation']\n",
      "['aki', 'esz', 'j. kert']\n",
      "['aki', 'generalization of', 'clustering coeﬃcient']\n",
      "['contacts', 'with', 'friends']\n",
      "['medium', 'of percentage', 'percentage']\n",
      "['percentage', 'percentage of', 'fully vaccinated']\n",
      "['locations', 'in', 'clay center']\n",
      "['map', 'courtesy of', 'google']\n",
      "['partition', 'in', 'households']\n",
      "['individuals', 'in', 'school']\n",
      "['nh', 'individual in', 'school']\n",
      "['agreement', 'with', 'theirs']\n",
      "['household members', 'in', 'large school']\n",
      "['vertices', 'represent', 'individuals']\n",
      "['edges', 'be', 'school']\n",
      "['vertices', 'in', 'large cluster']\n",
      "['schools', 'in', 'partition']\n",
      "['n', 'number of', 'households']\n",
      "['j', 'for x', 'j']\n",
      "['n →', 'as p', 'ts nc']\n",
      "['− 1', 'in', 'l − 1']\n",
      "['ancestor', 'with', 'type school']\n",
      "['children', 'to x', 'k']\n",
      "['construction', 'in', 'obvious way']\n",
      "['household contacts', 'in', 'x']\n",
      "['vertices', 'in', 'households']\n",
      "['l2', 'in', 'way']\n",
      "['individuals', 'in', 'cluster']\n",
      "['individuals', 'in', 'type j cluster']\n",
      "['sizes', 'in', 'school']\n",
      "['schools', 'of x', 'x']\n",
      "['households', 'of x', 'x']\n",
      "['individuals', 'in', 'household']\n",
      "['size l', 'note', 'l ≤ l']\n",
      "['vertices', 'in', 'c h']\n",
      "['l vertices', 'in', 'c h']\n",
      "['k', 'note from', 'c i 1']\n",
      "['edge', 'actually end in', 'large household/school cluster']\n",
      "['comparisons', 'probability of', 'large outbreak']\n",
      "['adv', 'in', 'appl']\n",
      "['adv', 'in', 'appl']\n",
      "['adv', 'in', 'appl']\n",
      "['conjunction', 'with', 'parameters estimated empirically']\n",
      "['factors', 'include context', 'quarantine']\n",
      "['peak', 'in', 'intermediate connectivity range']\n",
      "['single focal point', 'dependence of', 'impact']\n",
      "['dependence', 'in', 'r 0 range']\n",
      "['modules', 'each with', 'n']\n",
      "['curves', 'taper asymptotically', 'oﬀ']\n",
      "['3 discussion', 'in', 'paper']\n",
      "['new infection', 'with', 'diﬀerent strain']\n",
      "['advances', 'in', 'biomedical therapies']\n",
      "['delays', 'associate with', '2 immune response']\n",
      "['phenomenon', 'know as', 'herd immunity']\n",
      "['numerical calculations', 'use', 'matlab']\n",
      "['r0', 'of value', 'greater likelihood']\n",
      "['stochastic optimal solution', 'better fulﬁlls measure than', 'deterministic optimum']\n",
      "['incremental increase', 'in', 'allocation']\n",
      "['increase', 'in', 'vaccine allocation']\n",
      "['increase', 'with', 'decreasing r0']\n",
      "['v', 'of function', 'amount']\n",
      "['signiﬁcant increase', 'in', 'gain g']\n",
      "['fundamental tradeoﬀ', 'in', 'resource management']\n",
      "['optimal stochastic solutions', 'in', 'closer detail']\n",
      "['diﬀerence', 'in', 'strategy']\n",
      "['cusp', 'in', 'optimal v 1']\n",
      "['greater peak', 'in', 'gain']\n",
      "['declines', 'in', 'gain']\n",
      "['peak', 'in', 'gain']\n",
      "['worst case protocol', 'reach', 'herd immunity']\n",
      "['color style legends', 'in', 'b']\n",
      "['diﬀerence', 'in', 'e']\n",
      "['deterministic protocol', 'maximize', 'e emax']\n",
      "['diﬀerences', 'in', 'optimal allocations']\n",
      "['result', 'be', 'somewhat surprising']\n",
      "['stochastic protocol', 'outperform', 'deterministic counterpart']\n",
      "['tradeoﬀ', 'in', 'vaccination allocation']\n",
      "['strong assumptions', 'assumption of', 'non-interacting populations']\n",
      "['modest amount', 'in', 'particularly deterministic case']\n",
      "['s c i', 'be', 'y h p']\n",
      "['k ∈ n s', 'if k', 't']\n",
      "['k ∈ n i t', 'if k', 't']\n",
      "['last observation vector', 'deﬁned merely for', 'theoretical completeness']\n",
      "['slots', 'in', 'wk']\n",
      "['t ≥ 0', 'for pk', 't']\n",
      "['objective function', 'in yk', 't']\n",
      "['observation variables yk', 'in', 'objective function']\n",
      "['candidate source', 'give o', 'dtf n']\n",
      "['00.20.40.60.8100.10.20.30.40.5 nnormalizedrankofs', '∗', 'infectionrate00 .20.40.60.8101234567 ieee trans']\n",
      "['herpes infections', 'in', 'human populations']\n",
      "['topics', 'signal', 'process']\n",
      "['topics', 'signal', 'vol']\n",
      "['advances', 'in', 'difference equations']\n",
      "['uncertainty', 'in', 'artiﬁcial intelligence']\n",
      "['arbitrary proportion', 'in', 'long run']\n",
      "['schematic diagrams', 'in', 'figs']\n",
      "['single osn', 'in', 'isolation']\n",
      "['system', 'of', 'equilibria']\n",
      "['point', 'be', 'ﬁxed']\n",
      "['i', '−', '1 − 2i']\n",
      "['equilibrium', 'be', 'so unstable']\n",
      "['initial exponential growth', 'in', 'i t']\n",
      "['initial rightward motion', 'in', 'i direction']\n",
      "['interaction', 'with', 'active user']\n",
      "['system', 'in', 'variables']\n",
      "['schematic representation', 'with', 'demography']\n",
      "['i', 'edge of', 'simplex']\n",
      "['i', '−', 'αi − γ']\n",
      "['long-run stable equilibrium', 'with', 'nonzero osn membership']\n",
      "['memes', 'in', 'digital culture']\n",
      "['ariel cintr', 'on-arias', 'david i kaiser']\n",
      "['3 media lab', 'institute of', 'technology']\n",
      "['simulations', 'in', 'd']\n",
      "['simulations', 'in', 'd']\n",
      "['linear lattice', 'with', 'periodic boundary conditions']\n",
      "['θ', 'measure of', 'relative contributions']\n",
      "['upa', 'show in', 'inset']\n",
      "['mfa', 'for', 'singletons']\n",
      "['science', 'for foundation', 'fct']\n",
      "['r. m. anderson', 'disease of', 'humans']\n",
      "['pontryagin', \"'s principle\", 'pmp']\n",
      "['pontryagin', 'have', 'minimum principle']\n",
      "['export ban', 'in', 'hong kong']\n",
      "['∗', 'u for', 't ∈']\n",
      "['t ∈', 'for u', '0 t ∗']\n",
      "['controls', 'apply at', 's']\n",
      "['0 → umax', 'umax for', 't ∈']\n",
      "['t ∈', 'for ∗', 't']\n",
      "['suitable interval', 'over j', 'τ']\n",
      "['∗', 'be', 'bang-bang']\n",
      "['t', 'for ˙i', 't']\n",
      "['gray regions', 'in', 'fig 2']\n",
      "['strategy', 'be', 'bang-bang']\n",
      "['fig', 'in', 'figs 3 4']\n",
      "['pontryagin', 'accord to', 'minimum principle']\n",
      "['pontryagin', 'accord to', 'minimum principle']\n",
      "['remarks', 'improve', 'manuscript']\n",
      "['qk', 'be', 'invariant']\n",
      "['t →', 'as i', 't']\n",
      "['ψ', 'vanish in', 'open interval b']\n",
      "['contradiction', 'with', 'theorem 1']\n",
      "['consequence', 'as ul', 't']\n",
      "['τs', 'be', 'generic']\n",
      "['τs', 'be', 'generic']\n",
      "['interval', 'on λs', 'λi']\n",
      "['interval', 'on λs', 't']\n",
      "['pontryagin', 'by', 'minimum principle']\n",
      "['τs', 'be', 'generic']\n",
      "['environment', 'for paper', 'uk']\n",
      "['commercial poultry farms', 'in', 'netherlands']\n",
      "['perspectives', 'in', 'agriculture']\n",
      "['n', 'write', 'n']\n",
      "['f', 'with s', 'm ≥ 2']\n",
      "['s', 'with', 'f']\n",
      "['extinction', 'be', 'rare']\n",
      "['qnin', 'in', 'fig 4']\n",
      "['lnht i/n', 'of plot', 'm']\n",
      "['distance', 'η for', 'm']\n",
      "['i', '→', 'r → s']\n",
      "['i', '→', 'sirs']\n",
      "['r', 'become also spontaneously', 's']\n",
      "['r', 'become', 's']\n",
      "['i', 'turn into', 'r']\n",
      "['i', 'turn spontaneously into', 'r']\n",
      "['agreement', 'with', 'conjecture']\n",
      "['parameters', 'in', 'way']\n",
      "['phase diagram', 'show in', 'ﬁgure 1']\n",
      "['i', '→', 'r']\n",
      "['i', '→', 's']\n",
      "['n', 'be', 'population size']\n",
      "['active phase', 'see in', 'ﬁgure 5']\n",
      "['qualitative level', 'in', 'phase diagram']\n",
      "['t. tom', 'e', 'j. math']\n",
      "['oﬀspring distributions', 'of function', 'als']\n",
      "['edges', 'remove', 'self-loops']\n",
      "['parallel edges', 'remove', 'self-loops']\n",
      "['nonnegative random variable x', 'of variant', 'provided e x ∈']\n",
      "['xn ⇒ x', 'denote', 'convergence']\n",
      "['xn ⇒ x', 'denote convergence in', 'distribution']\n",
      "['random undirected multigraphs', 'of sequence', 'n ∈ n']\n",
      "['random undirected multigraphs', 'of sequence', 'ω']\n",
      "['j', 'j ≤', '⌊ αn ⌋']\n",
      "['vertices', 'vi', '∈']\n",
      "['2.4 b j', 'with p', 'b j']\n",
      "['n →', 'as number', 'a ⇒ a']\n",
      "['length 2', 'in', 'a n']\n",
      "['edges', 'of ′', 'n']\n",
      "['vertex vl', 'be', 'chosen']\n",
      "['end vertex', 'in', 'v']\n",
      "['′', 'be', 'chosen']\n",
      "['chosen vertices', 'in', 'v ′']\n",
      "['oﬀspring distribution', 'in', 'generations']\n",
      "['ρ', 'for', 'x ∈']\n",
      "['appendix b 2', 'in equation', 'b 7']\n",
      "['b', 'be', 'branching']\n",
      "['b', 'be', 'single-type']\n",
      "['connection', 'with', 'computation']\n",
      "['g ′', 'delete independently', 'directed edges']\n",
      "['g ′', 'delete', 'directed edges']\n",
      "['ρb ρ', 'in', 'case']\n",
      "['random variable', 'with', 'p tf']\n",
      "['tf', 'be', 'random']\n",
      "['iε ε ⌊ i ε ⌋', 'with', 'convention']\n",
      "['n →', 'as p', 'x ∈ a']\n",
      "['r', 'of member', 'n']\n",
      "['subset', 'in', 'v']\n",
      "['hk', 'be', 'proper']\n",
      "['l → ∞', 'as p', 'hk l']\n",
      "['hk', 'be', 'random']\n",
      "['l ∈ p', 'for →', 'l']\n",
      "['v', 'from v', 'n']\n",
      "['vertices', 'be', 'vertex chosen uniformly']\n",
      "['vertices', 'in', 'l ′']\n",
      "['1 coupling', 'now proceed in', 'obvious way']\n",
      "['vertices', 'in', 'v ′']\n",
      "['above restrictions', 'under v', 'n']\n",
      "['k', 'be', 'random']\n",
      "['pointwise nondecreasing', 'in', 'k']\n",
      "['graph', 'construct from', 'n']\n",
      "['ε 0', 'for o', 'nε']\n",
      "['tn', 'in', 'coupling argument']\n",
      "['distance', 'in', 'graph']\n",
      "['most n', 'in', 'ˆa']\n",
      "['distance 2tn', 'within v', 'n']\n",
      "['distance 2tn', 'within ′', 'n']\n",
      "['ν', 'for ω', 'pω']\n",
      "['ε 0', 'for opν', 'nε']\n",
      "['positions', 'in', 'space']\n",
      "['y', 'with', 'home location']\n",
      "['chosen site', 'in', 'ny']\n",
      "['sλ', 'γ for', 'event']\n",
      "['monotonicity', 'in', 'λ']\n",
      "['≥ proposition 1.1', 'monotonicity in', 'λ']\n",
      "['strict diﬀerence', 'in', 'sense']\n",
      "['types', 'disease among', 'cows']\n",
      "['conjecture 1.1', 'transition in', 'λ']\n",
      "['z2', 'with', 'm']\n",
      "['c c', 'in', 'left-hand picture']\n",
      "['conjecture 1.2', 'transition in', 'γ']\n",
      "['c m', 'in', 'c']\n",
      "['s', 'for γ', 't']\n",
      "['t. sλ', 'γ for', 's']\n",
      "['second claim', 'in', 'proposition 1.1']\n",
      "['particle ξz', 'with', 'home location']\n",
      "['particle ξz', 'visit', 'zact']\n",
      "['ξz', 'position at', 'zact']\n",
      "['ξz', 'once position at', 'zact']\n",
      "['γ', 'let', 'bλ']\n",
      "['initial conﬁguration', 'with', 'fewer particles']\n",
      "['particles', 'in', 'neighboring regions']\n",
      "['given site', 'in', 'zd']\n",
      "['elements', 'in', 's0 ,0']\n",
      "['i', 'β', 'i s n γ i']\n",
      "['i', '˙r', 'r']\n",
      "['y', 'set', 'x']\n"
     ]
    }
   ],
   "source": [
    "# pre-process extracted tuples\n",
    "print('\\nPreprocessing raw relation triples')\n",
    "rdf_obj.preprocess_raw_tuples()\n",
    "\n",
    "# cluster all entities using fast\n",
    "# agglomerative clustering and string distance\n",
    "print('\\nClustering entities from relation triples')\n",
    "rdf_obj.cluster_entities()\n",
    "print('\\n{} entity clusters were found'\n",
    "      .format(len(rdf_obj.entity_cluster_results['cluster_members'])))\n",
    "\n",
    "\n",
    "# output text files that will be used openke for knowledge graph creation\n",
    "# and embedding output .txt files for openke output\n",
    "from datetime import datetime\n",
    "timestamp = datetime.now()\n",
    "print('\\nSaving final tuples to .txt files for input to OpenKE')\n",
    "rdf_obj.output_to_openke(timestamp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2559"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(rdf_obj.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[('repeated vaccination',\n",
       "   'be',\n",
       "   'Thus has discontinued recently on geographical scale')],\n",
       " [('We', 'follow up', 'effects of vaccination in detailed manner'),\n",
       "  ('effects', 'be in', 'detailed manner')],\n",
       " [('baits', 'locally be assign to', 'individual foxes'),\n",
       "  ('baits', 'be', 'locally assigned')],\n",
       " [('controlled area', 'ring-like around', 'outbreak')],\n",
       " [('ring area', 'result in', 'fewer breakouts')],\n",
       " [('Conclusion', 'be', 'favourable')],\n",
       " [('main reserve', 'be in', 'Europe')],\n",
       " [('efficient oral rabies virus vaccine', 'be', 'available NumericCitation')],\n",
       " [('huge efforts', 'be in', 'rabies control')],\n",
       " [('regional scale', 'be in', 'central Europe')],\n",
       " [('repeated vaccination', 'be in', 'regions')],\n",
       " [('local outbreak', 'be in', 'non-immunised wildlife populations'),\n",
       "  ('we',\n",
       "   'develop',\n",
       "   'emergency measures designed for local outbreak in non-immunised wildlife populations')],\n",
       " [('situation', 'be in', 'UK')],\n",
       " [('event', 'be in', 'particular.How'),\n",
       "  ('re-introduction', 'detection of', 'infection within rabies-freed area')],\n",
       " [('1', 'restrict', 'spatial extent of control area')],\n",
       " [('outbreak', 'be', 'when very local at first')],\n",
       " [('appealing counter-measure would fence', 'be in', 'epidemic first')],\n",
       " [('outbreak', 'be', 'hence actually contained')],\n",
       " [('we', 'centre on', 'detected outbreak'),\n",
       "  ('we', 'aim at', 'serious contingency plan')],\n",
       " [('Schematic design', 'be in', 'emergency situation')],\n",
       " [('We', 'compare', 'different spatial designs of vaccination')],\n",
       " [('rabies', 'epidemic inside', 'control area')],\n",
       " [('two spatial designs', 'definition of', 'most-rewarding-point-in-time'),\n",
       "  ('application',\n",
       "   'for design be',\n",
       "   'i.e. definition of most-rewarding-point-in-time change from ring to circle')],\n",
       " [('rabies outbreak', 'be in', 'previously rabies-free region')],\n",
       " [('spread', 'be in', 'foxes'),\n",
       "  ('control', 'provide', 'management support NumericCitation'),\n",
       "  ('control', 'provide', 'useful management support NumericCitation')],\n",
       " [('approach',\n",
       "   'incorporate',\n",
       "   'implicit adjustment of fox density effect NumericCitation')],\n",
       " [('65.536 sq. km', 'be in', 'central European scenario')],\n",
       " [('control area', 'representation of be', 'abstraction')],\n",
       " [('geometric simplification', 'be in', 'model')],\n",
       " [('temporal resolution',\n",
       "   'timing of',\n",
       "   'initial vaccination campaign NumericCitation'),\n",
       "  ('temporal resolution', 'time until', 'detection of epidemic')],\n",
       " [('Fox groups', 'be in', 'field')],\n",
       " [('NumericCitation', 'be with', 'mortality process')],\n",
       " [('Juveniles',\n",
       "   'be',\n",
       "   'subjected to monthly mortality of 12 % until dispersal NumericCitation')],\n",
       " [('they', 'be treat after', 'dispersal')],\n",
       " [('Reproduction Reproduction', 'be schedule in', 'first week of April')],\n",
       " [('Dispersal', 'be with', 'population dynamics')],\n",
       " [('right', 'be with', 'equal probability')],\n",
       " [('state', 'be update accord to', 'weekly time-steps')],\n",
       " [('negative exponential distribution', 'be with', 'minimum of 2 weeks')],\n",
       " [('fox',\n",
       "   'can transmit disease numericcitation during',\n",
       "   'following infectious period of 1 week')],\n",
       " [('infected cubs', 'die of', 'rabies')],\n",
       " [('i.e. approach', 'be in', 'NumericCitation'),\n",
       "  ('infected NumericCitation', 'approach of', 'Infection Communities'),\n",
       "  ('infected NumericCitation', 'approach in', 'NumericCitation')],\n",
       " [('biannual campaigns', 'be with', '18')],\n",
       " [('vaccination campaigns', 'be with', '18'),\n",
       "  ('vaccination campaigns', 'be in', 'field')],\n",
       " [('Biannual vaccination', 'be', 'always performed'),\n",
       "  ('50 %', 'be in', 'spring')],\n",
       " [('immunisation level', 'be in', 'vaccinated area')],\n",
       " [('lower average immunisation level', 'be in', 'circular vaccination areas')],\n",
       " [('distance', 'by level be', '100 repetitions'),\n",
       "  ('3 vaccination campaigns', 'after level be', '100 repetitions'),\n",
       "  ('centre', 'to level be', '100 repetitions')],\n",
       " [('Rabid juvenile foxes',\n",
       "   'will',\n",
       "   'will detected only from August onwards NumericCitation'),\n",
       "  ('Rabid foxes',\n",
       "   'will',\n",
       "   'will detected only from August onwards NumericCitation')],\n",
       " [('PVacArea', 'be', 'set')],\n",
       " [('area', 'scale', 'mean area of fox group home-ranges')],\n",
       " [('NumericCitation',\n",
       "   'amount of',\n",
       "   'baits per campaign used in three scenarios'),\n",
       "  ('mean area', 'be in', 'model')],\n",
       " [('PVacArea', 'accord to 80 be', '104 104 126 cells respectively')],\n",
       " [('inner part',\n",
       "   'should compensate for',\n",
       "   'annual spread of rabies of 30 km NumericCitation')],\n",
       " [('simulations', 'be with', 'strategies')],\n",
       " [('PVacArea', 'have', 'has doubled'),\n",
       "  ('PVacArea', 'have', 'when has doubled')],\n",
       " [('group', 'be with', 'at least one infected fox'),\n",
       "  ('group', 'be with', 'at least one infectious fox'),\n",
       "  ('group', 'be with', 'at least one immune fox')],\n",
       " [('We', 'cover', 'stochastic effects'), ('We', 'cover', 'effects')],\n",
       " [('following spatial configuration', 'be apply in', 'experiment'),\n",
       "  ('surface', 'get double after', 'time lag')],\n",
       " [('area', 'be', 'solid')],\n",
       " [('we',\n",
       "   'hence compare population parameter as',\n",
       "   're-read from model to empirical data')],\n",
       " [('year', 'during density be', 'development of immunisation level'),\n",
       "  ('densities', 'development of', 'immunisation level'),\n",
       "  ('1.5 3 foxes', 'from density be', 'development of immunisation level')],\n",
       " [('Figure 6 Emergency vaccination', 'be with', 'fixed resources')],\n",
       " [('Emergency vaccination', 'be with', 'fixed resources')],\n",
       " [('past control programs', 'be in', 'Europe NumericCitation'),\n",
       "  ('resulting immunisation', 'be in', 'model'),\n",
       "  ('agreement',\n",
       "   'be with',\n",
       "   'recent findings about potential over-baiting during past control programs in Europe NumericCitation')],\n",
       " [('ring strategy', 'perform', 'worse than circle strategy'),\n",
       "  ('Independent', 'be in', 'long run')],\n",
       " [('ring strategy', 'contain', 'rabies')],\n",
       " [('strategy',\n",
       "   'be characterise by',\n",
       "   'increasing risk of breakouts over time')],\n",
       " [('ring rise', 'be with', 'time')],\n",
       " [('shaded areas', 'indicate', 'extent of vaccinated areas')],\n",
       " [('control area', 'beyond', 'outer border')],\n",
       " [('Figure 9', 'illustrate qualitatively', 'risk of breakouts over time')],\n",
       " [('we', 'check', 'prediction')],\n",
       " [('secondary infections', 'be', 'ignored')],\n",
       " [('starting control', 'be with', 'ring vaccination'),\n",
       "  ('right', 'be in', 'Fig. 9')],\n",
       " [('cross point', 'be around', 'third vaccination campaign')],\n",
       " [('Eradication', 'be with', 'increasing resources According Figure 10')],\n",
       " [('resource limitations', 'be', 'overridden'),\n",
       "  ('resource limitations', 'be', 'when overridden')],\n",
       " [('Eradication', 'compare to', 'Fig. 11b')],\n",
       " [('Figure 11 Emergency vaccination', 'be with', 'increasing resources')],\n",
       " [('vaccinated area', 'be', 'when doubled')],\n",
       " [('control success', 'however be shift by', 'time lag')],\n",
       " [('they', 'sample', 'NumericCitation'),\n",
       "  ('they', 'be', 'even difficult'),\n",
       "  ('they', 'sample', 'precisely NumericCitation')],\n",
       " [('management decisions', 'be make', 'NumericCitation'),\n",
       "  ('it', 'overcome', 'uncertainties')],\n",
       " [('one particular configuration',\n",
       "   'substantiate parameter choice with',\n",
       "   'help of logical arguments')],\n",
       " [('initial studies', 'be in', 'literature NumericCitation')],\n",
       " [('model', 'acquire in', 'relation to slight changes in assumed parameters'),\n",
       "  ('slight changes', 'be in', 'assumed parameters')],\n",
       " [('we', 'target', 'useful support for decisions')],\n",
       " [('One',\n",
       "   'refer to',\n",
       "   'immediate control of infection within smaller treated area')],\n",
       " [('more cases', 'be in', 'centre.Simulation of two spatial strategies')],\n",
       " [('ring strategy', 'be in', 'general')],\n",
       " [('Figure 10', 'actually propose', 'switch around third campaign')],\n",
       " [('switching', 'be in', 'experiment 2')],\n",
       " [('Our', 'assessment', 'Fig. 9')],\n",
       " [('our', 'simulation', 'Fig. 4a'),\n",
       "  ('noteworthy agreement', 'be with', 'our simulations')],\n",
       " [('change', 'be in', 'strategy')],\n",
       " [('time', 'be in', 'practice')],\n",
       " [('we', 'test', 'respective models')],\n",
       " [('aim', 'be in', 'fact'), ('we', 'accept rabies inside', 'ring')],\n",
       " [('decreasing gain', 'be in', 'outer radius')],\n",
       " [('Gain', 'be in', 'maximum distance')],\n",
       " [('Presented scenarios', 'be with', 'input parameters framed')],\n",
       " [('cost', 'be mainly link to', 'total number of baits needed for program')],\n",
       " [('eradication', 'until time be', 'Fig. 11b')],\n",
       " [('principal objective NumericCitation', 'be in', 'circle design.The'),\n",
       "  ('remaining public health issue',\n",
       "   'be principal objective numericcitation in',\n",
       "   'circle design.The')],\n",
       " [('it', 'less competitive than', 'compact circle approach'),\n",
       "  ('livestock', 'make', 'it less competitive than compact circle approach')],\n",
       " [('non-viable approach', 'be in', 'terms of eradication')],\n",
       " [('we', 'deal with', 'possible early breakouts')],\n",
       " [('circle strategy', 'be with', 'control area of 10.800 sq. km')],\n",
       " [('DeclarationsAcknowledgementsDE',\n",
       "   'be partly fund by',\n",
       "   'Impfstoffwerke Dessau-Tornau GmbH'),\n",
       "  ('DeclarationsAcknowledgementsDE', 'be', 'partly funded')],\n",
       " [('We', 'gratefully acknowledge', 'improvements')],\n",
       " [('they', 'draft', 'manuscript')],\n",
       " [('DE', 'perform', 'simulation experiments')],\n",
       " [('TM', 'provide', 'background of practical rabies management')],\n",
       " [('Environmental Research Leipzig/Halle', 'institute for', 'Health'),\n",
       "  ('Ecological Modelling',\n",
       "   'ufz-centre for',\n",
       "   'Environmental Research Leipzig/Halle')],\n",
       " [('prevention', 'be in', 'United States')],\n",
       " [('Rabies', 'be in', 'wildlife')],\n",
       " [('Rev', 'infect', 'Dis')],\n",
       " [('Zoonoses', 'be in', 'world')],\n",
       " [('Vorbereitung und Durchführung der ersten Feldversuche',\n",
       "   'be in',\n",
       "   'den ostdeutschen Bundesländern')],\n",
       " [('oral immunisation', 'be in', 'Europe'),\n",
       "  ('Progress', 'be in', 'oral immunisation of foxes against rabies')],\n",
       " [('rabies', 'be in', 'Switzerland'),\n",
       "  ('Switzerland', 'in rabies be', 'German')],\n",
       " [('Belgique', 'en rage be', 'Vulpes vulpes')],\n",
       " [('Oral immunization', 'be in', 'Europe')],\n",
       " [('red foxes', 'be in', 'eastern Ontario')],\n",
       " [('Rabies', 'be in', 'urban foxes'), ('Rabies', 'be in', 'Britain')],\n",
       " [('rabies control', 'be in', 'Britain')],\n",
       " [('HMSO', 'ministry of', 'agriculture fisheries')],\n",
       " [('Modelling rabies control', 'be in', 'UK')],\n",
       " [('post vaccination resurgence', 'be in', 'foxes')],\n",
       " [('WHO Seminar', 'of report be', 'Geneva 2-5 Citation')],\n",
       " [('Advances', 'be in', 'Rabies Research'),\n",
       "  ('Peterborough', 'proceedings of', 'XIIth International'),\n",
       "  ('Erfahrungen mit der Flugzeugbeköderung von Ködern zur oralen Immunisierung der Füchse gegen Tollwut',\n",
       "   'be in',\n",
       "   'Ostdeutschland'),\n",
       "  ('Advances', 'be in', 'Americas')],\n",
       " [('Modeling control', 'be in', 'red fox populations')],\n",
       " [('Population Dynamics', 'be in', 'Wildlife')],\n",
       " [('Individual-based models', 'be in', 'ecology')],\n",
       " [('rabies control', 'be in', 'fox populations')],\n",
       " [('Dispersal distance', 'be in', 'red fox'),\n",
       "  ('red fox', 'in distance be', 'Vulpes vulpes')],\n",
       " [('Movements', 'be in', 'Denmark'),\n",
       "  ('red fox', 'of movement be', 'Vulpes vulpes L.'),\n",
       "  ('Denmark', 'in movement be', 'Vulpes vulpes L.')],\n",
       " [('territoriality', 'diminish with', 'increasing area')],\n",
       " [('Population Dynamics', 'be in', 'Wildlife')],\n",
       " [('costs', 'be in', 'oral rabies vaccination programs')],\n",
       " [('Advances', 'be in', 'Rabies Research'),\n",
       "  ('Advances', 'be in', 'Americas'),\n",
       "  ('Nov. 12-16 28-28', 'peterborough at_time', 'Canada')],\n",
       " [('Variation', 'be in', 'social system of red fox')],\n",
       " [('group living', 'of benefit be', 'Vulpes vulpes'),\n",
       "  ('group living', 'be in', 'red fox')],\n",
       " [('Female cooperation', 'be in', 'red fox')],\n",
       " [('der Oberlausitz', 'in rotfuchses be', 'Vulpes vulpes'),\n",
       "  ('Daten zur Fortpflanzungsbiologie und zur Reproduktionsstrategie des Rotfuchses',\n",
       "   'be in',\n",
       "   'der Oberlausitz')],\n",
       " [('Untersuchungen zur Habitatnutzung des Rotfuches',\n",
       "   'be in',\n",
       "   'der Agrarlandschaft als Grundlage für die'),\n",
       "  ('der Agrarlandschaft als Grundlage für',\n",
       "   'in rotfuches be',\n",
       "   'Vulpes vulpes L. 1758')],\n",
       " [('endemic focus', 'be in', 'Brandenburg')],\n",
       " [('epidemiology', 'be in', 'Europe')],\n",
       " [('Population Dynamics', 'be in', 'Wildlife')],\n",
       " [('Current status', 'be in', 'Europe')],\n",
       " [('Rotfuchspopulationen', 'be in', 'der Schweiz'),\n",
       "  ('der Schweiz', 'in rotfuchspopulationen be', 'Vulpes vulpes')],\n",
       " [('Experiences',\n",
       "   'be with',\n",
       "   'oral immunization of foxes against rabies in Hesse'),\n",
       "  ('oral immunization', 'be in', 'Hesse')],\n",
       " [('population', 'be in', 'Dutch coastal dune area')],\n",
       " [('use', 'be in', 'rabies vaccination campaigns')],\n",
       " [('rabies', 'against fox be', 'Vulpes vulpes')],\n",
       " [('Bait uptake', 'be in', 'urban Melbourne')],\n",
       " [('A field trial', 'be in', 'Belgium')],\n",
       " [('Oral vaccination', 'be with', 'SAD B19 Citation -1998')],\n",
       " [('Die orale Immunisierung von Füchsen gegen Tollwut',\n",
       "   'be in',\n",
       "   'der Schweiz')],\n",
       " [('Review', 'be in', 'Europe')],\n",
       " [('foxes', 'be in', 'France')],\n",
       " [('Tollwutbekämpfung', 'be in', 'Deutschland')],\n",
       " [],\n",
       " [('Mathematical models', 'be in', 'ecology')],\n",
       " [('Pattern-oriented modelling', 'be in', 'population ecology')],\n",
       " [('Rabies', 'be in', 'wild carnivores'),\n",
       "  ('wild carnivores', 'be in', 'central Europe')],\n",
       " [('wave front', 'be in', 'central Europe')],\n",
       " [('case occurrences', 'be in', 'Europe')],\n",
       " [('Population dynamics', 'be in', 'wildlife')],\n",
       " [('summer', 'be with', 'VRG vaccine bait')],\n",
       " [('coyotes', 'be in', 'southern Texas')],\n",
       " [('baiting rates', 'be in', 'Western Australia')],\n",
       " [('cost-efficient oral vaccination strategies', 'be in', 'fox')],\n",
       " [('spatial aggregation',\n",
       "   'be in',\n",
       "   'its natural intermediate host Microtus arvalis')],\n",
       " [('foxes', 'be with', 'immunity')],\n",
       " [('progress', 'be in', 'human rabies prevention')],\n",
       " [('Habitat related differences', 'be in', 'age structure')],\n",
       " [('Open Access article', 'provide', 'original work'),\n",
       "  ('use', 'be in', 'medium')]]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rdf_obj.all_tuples[-2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.2"
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
